--- 
title: "The Epigenomics of Human Ageing"
author: "Richard J. Acton"
orcid: "ORCID: [0000-0002-2574-9611](https://orcid.org/0000-0002-2574-9611)"
date: "`r Sys.Date()`"
site: bookdown::bookdown_site
output: 
  bookdown::gitbook:
    template: my_template2.html
    config:
      toc:
        collapse: subsection
        scroll_highlight: yes
        before: null
        after: null
      toolbar:
        position: fixed
      edit : null
      download: null
      search: yes
      fontsettings:
        theme: night
        family: sans
        size: 2
      info: yes
link-citations: yes
linkcolor: blue
linestretch: 1.5
documentclass: book
bibliography: "library.bib"
csl: "genomebiology.csl"
github-repo: 
description: "PhD Thesis"
subtitle: |

  __Academic Unit:__ Human Development and Health
  
  __Supervisors:__ Chris Bell, Karen Lillycrop and Cyrus Cooper
---

```{r,include=FALSE,echo=FALSE}
suppressPackageStartupMessages({
	#library(DiagrammeR)
	library(tidyverse)
	#library(gt)
	library(kableExtra)
})
```

\newpage

# Research Thesis: Declaration of Authorship {-}

Name: Richard J. Acton

Title of thesis: The Epigenomics of Human Ageing

I declare that this thesis and the work presented in it is my own and has been generated by me as the result of my own original research.

I confirm that:

1. This work was done wholly or mainly while in candidature for a research degree at this University;
2. Where any part of this thesis has previously been submitted for a degree or any other qualification at this University or any other institution, this has been clearly stated;
3. Where I have consulted the published work of others, this is always clearly attributed;
4. Where I have quoted from the work of others, the source is always given. With the exception of such quotations, this thesis is entirely my own work;
5. I have acknowledged all main sources of help;
6. Where the thesis is based on work done by myself jointly with others, I have made clear exactly what was done by others and what I have contributed myself; 
7. Parts of this work have been published as: a $BioR \chi iv$ pre-print [@Acton2020]

Signature: __________________________

Date: `r Sys.Date()`

\newpage

# Abstract {-}

The global population is ageing and age is a primary risk factor for many life threatening diseases including cancer, cardiovascular and neurodegenerative conditions. 
As well as for conditions such as osteoporosis which increase the risk of death indirectly through elevated risk of the fracture of major bones and subsequent complications. 
It is the aim of this work to contribute to the understanding of the epigentic processes involved in ageing and ageing-related disease.
three specific areas were investigated. 

Firstly, the relationship between early life epigenetic state and long term bone health. This investigation took the form of Epigenome wide association studies conducted with umbilical cord blood. 
Which identified candidate CpGs whose DNA methylation state is associated with bone mineral content at 6 years of age ($p < 2.52\times 10^{-8}$, n = 402) and periosteal circumference at 6 years of age ($p < 4.24\times 10^{-8}$, n = 141) respectively.

Secondly, following up on an earlier finding that a human tRNA gene exhibited age-related hypermethylation, the changes in human tRNA gene DNA methylation with age were interogated. 
Human tRNA genes are enriched for age-related DNA hypermethylation and three specific tRNA loci show genome wide significant ($p < 4.34\times10^{-9}$) hypermethylation with age. 
Two of which were validated using at different technology and replicated in an independent cohort, using a third DNA methylation assay.

Thirdly, a DNA methylation based age predictor based on a region of the genome not previously broadly accessible to DNA methylation based age predictors, namely the Alu family of SINE repeat elements. 
Age predictors using the DNA methylation state of Alu repeat elements are able to predict human age with an R of 0.65 and a median absolute error of 8.1 years.
This predicition was possible using MeDIP-seq training data from 774 individuals and validated on an unrelated set of 664 individuals.
This MeDIP-seq dataset also facilitated the investigation of the tRNA gene regions which like the Alu repeat elements are poorly covered by most other genome-wide DNA methylation assays.

In Summary, this work provides a starting point for further investigations into the relationship between early life epigenetic associations to longer term bone health outcomes.
This work highlights age-related epigenetics changes at the tRNA gene loci, a region of the genome responsible for the production of a core component of the apparatus of translation.
Additionally, this work also demonstrates a proof of principle for DNA methylation based age estimators created with the DNA methylation signatures from specific genomic compartments. 
As well as for identifying the genetic associations of biological age indicators such as age acceleration based on such genomic feature specific age predictors.

# Acknowledgements {-}

I would like to acknowledge Nevena Krstic for her work extracting the DNA for the MAVIDOS EPIC array analysis and Dr Millie Parsons for her assistance with with the MAVIDOS sample metadata, as well as Dr Beth Curtis and Professor Nick Harvey of the MRC-LEU for their assistance with framing the research questions for the vitamin D and bone development outcomes work with the MAVIDOS samples.
The MRC-LEU is supported by the Medical Research Council (MRC).
I gratefully acknowledge the individuals from TwinsUK, Mavidos and the Hertfordshrie cohort.
I would like to acknowledge Nikki Graham for her assistance identifying suitable samples for targeted bisulfite sequencing work and her work extracting DNA from those samples. 
TwinsUK received funding from the Wellcome Trust (Ref: 081878/Z/06/Z), European Community’s Seventh Framework Programme (FP7/2007-2013), the National Institute for Health Research (NIHR)-funded BioResource, Clinical Research Facility and Biomedical Research Centre based at Guy’s and St Thomas’ NHS Foundation Trust in partnership with King’s College London. 
Further funding support for the EpiTwin project was obtained from the European Research Council (project number 250157) and BGI. 
SNP Genotyping was performed by The Wellcome Trust Sanger Institute and National Eye Institute via NIH/CIDR.
I acknowledge the use of the IRIDIS High-Performance Computing Facility, and associated support services at the University of Southampton, in the completion of this work. 
I would like to thank the MRC Doctoral fund (1820097) for supporting this work.
Finally, I would like to acknowledge the extensive help and support of my primary supervisor Dr Chris Bell.

