--- 
title: "__The Epigenomics of Human Ageing__"
author: "Richard J. Acton"
orcid: "ORCID: [0000-0002-2574-9611](https://orcid.org/0000-0002-2574-9611)"
date: "`r format(Sys.Date(), '%B %Y')`"
site: bookdown::bookdown_site
output: 
  bookdown::gitbook:
    template: my_template2.html
    config:
      toc:
        collapse: subsection
        scroll_highlight: yes
        before: null
        after: null
      toolbar:
        position: fixed
      edit : null
      download: null
      search: yes
      fontsettings:
        theme: white #night
        family: sans
        size: 2
      info: yes
link-citations: yes
linkcolor: blue
linestretch: 1.5
documentclass: book
bibliography: "library.bib"
csl: "genomebiology.csl"
github-repo: 
description: "Thesis for the degree of Doctor of Philosophy"
subtitle: |
  *_University of Southampton_*
  
  __Academic Unit:__ Human Development and Health
  
  A thesis for the degree of Doctor of Philosophy
abstract: |
  # Abstract {-}
  
  The global population is ageing and age is a primary risk factor for many life threatening diseases including cancer, cardiovascular and neurodegenerative conditions. 
  This also encompasses conditions such as osteoporosis which increase the risk of death indirectly through elevated risk of the fracture of major bones and subsequent complications. 
  The aim of this thesis was to extend the understanding of the epigenetic processes involved in ageing and ageing-related disease.
  Three specific areas were investigated. 

  Firstly, the relationship between early life epigenetic state and long term bone health was analysed.
  This investigation took the form of Epigenome-wide association studies (EWAS) conducted on umbilical cord blood DNA methylation.
  This identified candidate CpGs whose, DNA methylation state is associated with bone mineral content at 6 years of age ($p < 2.52\times 10^{-8}$, n = 402) and periosteal circumference at 6 years of age ($p < 4.24\times 10^{-8}$, n = 141) respectively.

  Secondly, the changes in human tRNA gene DNA methylation with age were interogated. 
  tRNAs permit the look-up of amino acids matching a given codon and as such are an essential core component of the translation of mRNA into protein by the ribosome.
  Human tRNA genes were found to be enriched for age-related DNA hypermethylation and three specific tRNA loci show genome-wide significant ($p < 4.34\times10^{-9}$) hypermethylation with age. 
  Two of which, tRNA-iMet-CAT-1-4 and tRNA-Ser-AGA-2-6, were validated using the 450k Illumina DNA methylation array and replicated in an independent cohort, using targeted bisulfite sequencing.

  Thirdly, a DNA methylation based age predictor based on the Alu family of SINE repeat elements was constructed.
  These repeats comprise a region of the genome not previously broadly accessible to DNA methylation assays used in the construction of age predictors. 
  Age predictors using the DNA methylation state of Alu repeat elements were able to predict human age with an R of 0.65 and a median absolute error of 8.1 years.
  This predicition was possible using MeDIP-seq training data from 774 individuals and validated on an unrelated set of 664 individuals.
  An attempt was made to use ageing-related changes in Alu DNA methylation as a potential measure of ‘biological’ ageing.
  A Genome-wide Association study (GWAS) was performed for age accleration, the difference between predicted and chronological age.
  However, this age acceleration calculation was observed to be still strongly driven by actual age.

  This work brings together epigenomic changes related to ageing from the beginning of the lifecourse through to later life, uniquely examining areas of the DNA methylome not previously studied in depth.
  
  \newpage
  
  # Publications arising from this Thesis {-}
  $BioR\chi iv$ manuscript: The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation

  $Richard J. Acton ^{1,2,3}$, $Wei Yuan ^{4,5}$, $Fei Gao ^{6}$, $Yudong Xia ^{6}$, $Emma Bourne ^{7}$, $Eva Wozniak ^{7}$, $Jordana Bell ^{4}$, $Karen Lillycrop ^{3}$, $Jun Wang ^{6}$, $Elaine Dennison ^{2}$, $Nicholas Harvey ^{2}$, $Charles A. Mein ^{7}$, $Tim D. Spector ^{4}$, $Pirro G. Hysi ^{4}$, $Cyrus Cooper ^{2}$, $Christopher G. Bell ^{1}$

  1. William Harvey Research Institute, Barts & The London School of Medicine and Dentistry, Charterhouse Square, Queen Mary University of London, London, U.K.
  2. MRC Lifecourse Epidemiology Unit, University of Southampton, Southampton, U.K.
  3. Human Development and Health, Institute of Developmental Sciences, University of Southampton, Southampton, U.K.  
  4. Department of Twin Research & Genetic Epidemiology, St Thomas Hospital, King’s College London, London, U.K.
  5. Institute of Cancer Research, Sutton, U.K.
  6. BGI-Shenzhen, Shenzhen, China
  7. Barts & The London Genome Centre, Blizard Institute, Barts & The London School of Medicine and Dentistry, Queen Mary University of London, London, U.K.
  [$bioR \chi iv$](https://doi.org/10.1101/870352) [@Acton2020]

  [Under Consideration at Nature Communications](https://nature-research-under-consideration.nature.com/users/37265-nature-communications/posts/57741-the-genomic-loci-of-specific-human-trna-genes-exhibit-ageing-related-dna-hypermethylation)
  
  \newpage
  
  # Research Thesis: Declaration of Authorship {-}

  Name: Richard J. Acton

  Title of thesis: The Epigenomics of Human Ageing

  I declare that this thesis and the work presented in it is my own and has been generated by me as the result of my own original research.

  I confirm that:

  1. This work was done wholly or mainly while in candidature for a research degree at this   University;
  2. Where any part of this thesis has previously been submitted for a degree or any other qualification at this University or any other institution, this has been clearly stated;
  3. Where I have consulted the published work of others, this is always clearly attributed;
  4. Where I have quoted from the work of others, the source is always given. With the exception   of such quotations, this thesis is entirely my own work;
  5. I have acknowledged all main sources of help;
  6. Where the thesis is based on work done by myself jointly with others, I have made clear exactly what was done by others and what I have contributed myself; 
  7. Parts of this work have been published as: a $BioR \chi iv$ pre-print [@Acton2020]

  Signature: __________________________

  Date: `r Sys.Date()`

---

```{r,include=FALSE,echo=FALSE}
suppressPackageStartupMessages({
	#library(DiagrammeR)
	library(tidyverse)
	#library(gt)
	library(kableExtra)
})
```

\newpage

\listoffigures

\listoftables

\newpage

# Acknowledgements {-}

I would like to acknowledge Dr Nevena Krstic for her work extracting the DNA for the MAVIDOS EPIC array analysis and Dr Millie Parsons for her assistance with with the MAVIDOS sample metadata, as well as Dr. Beth Curtis and Professor Nick Harvey of the MRC-LEU for their assistance with framing the research questions for the vitamin D and bone development outcomes work with the MAVIDOS samples.
The MRC-LEU is supported by the Medical Research Council (MRC).
I gratefully acknowledge the individuals from TwinsUK, Mavidos and the Hertfordshrie cohort.
I would like to thank Dr. Pirro Hysi for his work performing the age acceleration GWAS.
I would like to acknowledge Nikki Graham for her assistance identifying suitable samples for targeted bisulfite sequencing work and her work extracting DNA from those samples. 
I acknowledge the use of the IRIDIS High-Performance Computing Facility, and associated support services at the University of Southampton, in the completion of this work. 
I would like to thank the MRC Doctoral fund for supporting this work.
I thank my family for their invaluable support, encouragement, and occasional proofreading.
Thanks also to Dr Michael Glinka for his advice, encouragement and friendship.
I thank my supervisors Professors Karen Lillycrop and Cyrus Cooper for their input and advice on my project.
Finally, I would like to acknowledge the extensive help and support of my primary supervisor Dr Chris Bell.
