<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Results 3 - | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Results 3 - | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Results 3 - | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-08-04" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="tRNAs.html"/>
<link rel="next" href="discussion.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#illumina-dna-methylation-arrays"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a><ul>
<li class="chapter" data-level="3.1" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.2.1</b> Outline of EWAS</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.3.2</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.3</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.4" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.4</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.4.1</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.4.2</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.4.3</b> Southampton Women’s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-abstract"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Introduction"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#tRNAmethods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Participants"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-methylation-in-cancer"><i class="fa fa-check"></i><b>4.5</b> tRNA Gene Methylation in Cancer</a></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Discussion"><i class="fa fa-check"></i><b>4.6</b> Discussion</a></li>
<li class="chapter" data-level="4.7" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-data-avail"><i class="fa fa-check"></i><b>4.7</b> Data availability</a></li>
<li class="chapter" data-level="4.8" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-code-avail"><i class="fa fa-check"></i><b>4.8</b> Code availablity</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a></li>
<li class="chapter" data-level="5.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="Alus.html"><a href="Alus.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>5.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.4.2" data-path="Alus.html"><a href="Alus.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="5.4.3" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>5.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="5.4.4" data-path="Alus.html"><a href="Alus.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>5.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="Alus.html"><a href="Alus.html#methods-1"><i class="fa fa-check"></i><b>5.6</b> Methods</a><ul>
<li class="chapter" data-level="5.6.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>5.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="Alus.html"><a href="Alus.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>5.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="5.7.1" data-path="Alus.html"><a href="Alus.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>5.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="5.7.2" data-path="Alus.html"><a href="Alus.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>5.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.7.3" data-path="Alus.html"><a href="Alus.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.7.4" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.7.5" data-path="Alus.html"><a href="Alus.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="Alus" class="section level1">
<h1><span class="header-section-number">5</span> Results 3 -</h1>
<div id="abstract" class="section level2">
<h2><span class="header-section-number">5.1</span> Abstract</h2>
</div>
<div id="AluIntro" class="section level2">
<h2><span class="header-section-number">5.2</span> Introduction</h2>
<p>Age is a significant risk factor across the breadth of chronic diseases, including cardiovascular, obesity-related, ophthalmic, musculoskeletal and neurological disorders <span class="citation">[<a href="#ref-Partridge2018" role="doc-biblioref">226</a>]</span>.
The molecular changes that occur with age include alterations to the epigenome <span class="citation">[<a href="#ref-Booth2016" role="doc-biblioref">10</a>]</span> – the chemical modifications and packaging of the genome that indicate or facilitate cell-specific activity <span class="citation">[<a href="#ref-Bird2007" role="doc-biblioref">16</a>]</span>.
This includes age-related changes to DNA methylation, the most common modification of DNA <span class="citation">[<a href="#ref-Lopez-Otin2013" role="doc-biblioref">9</a>]</span>.
This epigenetic mechanism is crucial in repressing the large proportion of the human genome (~45% <span class="citation">[<a href="#ref-Gregory2005" role="doc-biblioref">342</a>]</span>) that is comprised of repetitive or transposable elements <span class="citation">[<a href="#ref-Deniz2019" role="doc-biblioref">343</a>]</span>, <span class="citation">[<a href="#ref-Kazazian2017" role="doc-biblioref">344</a>]</span>.
It also classically leads to robust repression of gene expression through promoter DNA methylation <span class="citation">[<a href="#ref-Deaton2011" role="doc-biblioref">345</a>]</span>, and has more nuanced effects on expression within the gene body <span class="citation">[<a href="#ref-Hellman2007" role="doc-biblioref">346</a>]</span>, as well as influencing splicing <span class="citation">[<a href="#ref-Shukla2011" role="doc-biblioref">347</a>]</span> and transcription factor binding-sites (TFBSs) <span class="citation">[<a href="#ref-Blattler2014" role="doc-biblioref">348</a>]</span>, <span class="citation">[<a href="#ref-Yin2017" role="doc-biblioref">51</a>]</span>.</p>
<p>A significant decrease in DNA methylation with age was first observed with pioneering global measures <span class="citation">[<a href="#ref-Wilson1983" role="doc-biblioref">90</a>]</span>, in similar fashion to the reduced methylation seen in cancer tissue <span class="citation">[<a href="#ref-Gama-Sosa1983" role="doc-biblioref">349</a>]</span>.
This DNA methylation loss was seen to be strongly contributed to by the demethylation of repetitive elements <span class="citation">[<a href="#ref-Bollati2010" role="doc-biblioref">350</a>]</span>.
Random drift in the epigenetic state with age was also observed <span class="citation">[<a href="#ref-Bollati2010" role="doc-biblioref">350</a>]</span>, and more recent studies have identified targeted hypermethylation of specific functional regions, such as the promoters of polycomb target genes <span class="citation">[<a href="#ref-Teschendorff2010" role="doc-biblioref">108</a>]</span>, bivalent chromatin domains <span class="citation">[<a href="#ref-Rakyan2010" role="doc-biblioref">106</a>]</span> and the genomic loci of specific tRNA genes <span class="citation">[<a href="#ref-Acton2020" role="doc-biblioref">225</a>]</span></p>
<p>Alu elements are the most successful transposon in terms of numbers within the human genome, totalling ~1.19 million copies and comprising ~10.7% of the genomic sequence <span class="citation">[<a href="#ref-Dewannieux2003" role="doc-biblioref">351</a>]</span>, <span class="citation">[<a href="#ref-Deininger2011" role="doc-biblioref">352</a>]</span>.
This ~300 bp primate-specific retrotransposon is a member of the short interspersed nuclear elements (SINEs) and is derived from the 7SL RNA gene <span class="citation">[<a href="#ref-Ullu1984" role="doc-biblioref">353</a>]</span>.
Alu elements arose ~65 million years ago (MYA), with a peak in amplification around ~40 MYA <span class="citation">[<a href="#ref-Ade2013" role="doc-biblioref">354</a>]</span>.
Non-LTR retrotransposons, which include Alus as well as L1 and SVA elements, are active in human.
However, Alu elements themselves are non-autonomous and need to commandeer a nearby L1 element’s retrotransposition molecular machinery <span class="citation">[<a href="#ref-Cordaux2009" role="doc-biblioref">355</a>]</span>.
Alu insertion mutagenesis as well as Alu element mediated recombination account for 0.4% of all human genetic disorders <span class="citation">[<a href="#ref-Kazazian2017" role="doc-biblioref">344</a>]</span>, <span class="citation">[<a href="#ref-Belancio2008" role="doc-biblioref">356</a>]</span>.</p>
<p>DNA methylation represses Alu elements and due to their sequences being rich in CpGs they in fact account for ~25% of all the CpGs in human genome (~7.5 million) <span class="citation">[<a href="#ref-Deininger2011" role="doc-biblioref">352</a>]</span>.
Alu repeats are predominately methylated across all normal tissues, and even in cancer cells they are the more resistant to hypomethylation than other repetitive elements, implying a strong pressure to maintain their epigenetic state <span class="citation">[<a href="#ref-Jorda2017" role="doc-biblioref">357</a>]</span>.
These elements are an abundant genetic resource of regulatory sequences with
numerous TFBSs residing within its canonical sequence <span class="citation">[<a href="#ref-Polak2006" role="doc-biblioref">358</a>]</span>.
They can influence gene expression <span class="citation">[<a href="#ref-Chen2017b" role="doc-biblioref">359</a>]</span>, <span class="citation">[<a href="#ref-RobertoFerrari2019" role="doc-biblioref">360</a>]</span> and there is evidence that some may possess enhancer activity <span class="citation">[<a href="#ref-Su2014" role="doc-biblioref">361</a>]</span>.
The deterioration of epigenomic repression can unmask latent regulatory function within repetitive elements <span class="citation">[<a href="#ref-Ward2013" role="doc-biblioref">362</a>]</span>, <span class="citation">[<a href="#ref-Chuong2017" role="doc-biblioref">228</a>]</span>, <span class="citation">[<a href="#ref-Xie2013" role="doc-biblioref">363</a>]</span>.
This has been clearly displayed as a pathogenic mechanism in cancer, with epigenetic reactivation of cryptic cis-regulatory elements (CREs) within transposons acting as oncogenic drivers <span class="citation">[<a href="#ref-Jang2019" role="doc-biblioref">364</a>]</span> , including Alu elements <span class="citation">[<a href="#ref-Rajendiran2016" role="doc-biblioref">365</a>]</span>.
There is the potential for these changes to also influence common non-malignant age-related diseases <span class="citation">[<a href="#ref-Chuong2017" role="doc-biblioref">228</a>]</span>, <span class="citation">[<a href="#ref-Payer2017" role="doc-biblioref">366</a>]</span>.</p>
<p>The identification of specific age-related DNA methylation changes has enabled the construction of ‘DNA methylation clocks‘ <span class="citation">[<a href="#ref-Horvath2018" role="doc-biblioref">149</a>]</span> that can predict an individual’s chronological age, but also capture aspects of biological ageing, such as all-cause mortality <span class="citation">[<a href="#ref-Marioni2015" role="doc-biblioref">135</a>]</span>.
Hypermethylated and hypomethylated cytosines approximately equally drive these DNA methylation clocks, although these age-related increases or decreases in methylation are driven by distinct biological mechanisms <span class="citation">[<a href="#ref-Mozhui2017" role="doc-biblioref">367</a>]</span>, <span class="citation">[<a href="#ref-Wu2017" role="doc-biblioref">60</a>]</span>.
Interestingly, hypomethylating CpGs have been identified to be the most indicative of biological ageing-related all-cause mortality effects in blood <span class="citation">[<a href="#ref-Zhang2017" role="doc-biblioref">140</a>]</span>.
However, due to poor coverage of the repetitive space because of technical issues <span class="citation">[<a href="#ref-Bell2019" role="doc-biblioref">150</a>]</span>, much of the genomic hypomethylation is not well captured by array analysis.</p>
<p>In this study we set about to identify and explore age-related DNA methylation changes in Alu elements in a large (&gt;3,000) immunoprecipitation-derived sequencing-based dataset (MeDIP-seq) from peripheral blood DNA. We constructed an Alu-specific DNA methylation ageing clock and compared how this contrasted with other clocks’ measures. We also investigated genetic influences on age- acceleration measures from this Alu-clock via GWAS, as has been with shown previously with an association in the TERT gene with the Horvath clock <span class="citation">[<a href="#ref-Lu2017" role="doc-biblioref">368</a>]</span>, <span class="citation">[<a href="#ref-Gibson2019" role="doc-biblioref">369</a>]</span>.
These results may reveal mechanisms involved in controlling the epigenetic state of this large portion of the genome.
Finally, we interrogated these genetic associations for any potential connection with common age-related diseases through PheWAS in the UKBiobank dataset <span class="citation">[<a href="#ref-Bycroft2017" role="doc-biblioref">370</a>]</span>.</p>
</div>
<div id="methods" class="section level2">
<h2><span class="header-section-number">5.3</span> Methods</h2>
</div>
<div id="results" class="section level2">
<h2><span class="header-section-number">5.4</span> Results</h2>
<p>Age-related DNA methylation changes in Alu Elements</p>
<p>Study Design - Figure 1</p>
<p>(ref:studyDesignCap)</p>
<ul>
<li>This sequencing-based dataset – MeDIP-seq fragments 200-500bp – size Alu elements 300bp</li>
<li>Remove strongly genetically driven loci
<ul>
<li>Removed known polymorphic Alu in the human population <span class="citation">[<a href="#ref-Payer2017" role="doc-biblioref">366</a>]</span></li>
<li>?Adapted Gap Hunting algorithm (see Methods)</li>
</ul></li>
<li>Genome-wide significant/ Study-wide Significant windows</li>
<li>?Validated Alu methylation with target BiS-seq</li>
<li>Enrichment Analysis - GREAT <span class="citation">[<a href="#ref-McLean2010" role="doc-biblioref">371</a>]</span></li>
<li>?Association with GWAS disease loci</li>
</ul>
<div id="alu-element-specific-dna-methylation-clock" class="section level3">
<h3><span class="header-section-number">5.4.1</span> Alu-element specific DNA methylation Clock</h3>
<ul>
<li>Elastic net regression with R package glmnet <span class="citation">[<a href="#ref-Friedman2010" role="doc-biblioref">120</a>]</span>, <span class="citation">[<a href="#ref-Engebretsen2019" role="doc-biblioref">372</a>]</span></li>
</ul>
</div>
<div id="comparison-of-alu-clock-with-array-derived-dna-methylation-clocks" class="section level3">
<h3><span class="header-section-number">5.4.2</span> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</h3>
<ul>
<li>Compared with
<ul>
<li>Horvath <span class="citation">[<a href="#ref-Horvath2013" role="doc-biblioref">119</a>]</span> ,</li>
<li>Hannum et al. <span class="citation">[<a href="#ref-Hannum2013" role="doc-biblioref">118</a>]</span></li>
<li>PhenoAge <span class="citation">[<a href="#ref-Levine2018" role="doc-biblioref">138</a>]</span></li>
<li>GrimAge <span class="citation">[<a href="#ref-Lu2019" role="doc-biblioref">139</a>]</span></li>
<li>EpiTOC <span class="citation">[<a href="#ref-Yang2016" role="doc-biblioref">310</a>]</span></li>
<li>?Remethylation Window <span class="citation">[<a href="#ref-Zhou2018" role="doc-biblioref">373</a>]</span></li>
</ul></li>
</ul>
</div>
<div id="genetic-association-for-alu-clock-calculated-age-acceleration" class="section level3">
<h3><span class="header-section-number">5.4.3</span> Genetic Association for Alu-Clock calculated Age Acceleration</h3>
<ul>
<li>We performed a GWAS on Age-acceleration similar to Lu et al. 42 calculated in a separate dataset used to construct the Alu-clock</li>
<li>Genome-wide significant &amp; nominally significant results</li>
</ul>
</div>
<div id="disease-association-with-these-variants-enrichment-for-age-related-diseases" class="section level3">
<h3><span class="header-section-number">5.4.4</span> Disease Association with these Variants / Enrichment for Age-Related Diseases</h3>
<ul>
<li>Ageing and genetics shared risk <span class="citation">[<a href="#ref-Zenin2019" role="doc-biblioref">374</a>]</span></li>
<li><a href="https://www.eurekalert.org/pub_releases/2019-01/g-aac012919.php" class="uri">https://www.eurekalert.org/pub_releases/2019-01/g-aac012919.php</a></li>
<li>GWAS results from UKBioBank</li>
<li><a href="http://www.nealelab.is/uk-biobank" class="uri">http://www.nealelab.is/uk-biobank</a></li>
<li>Explore via PheWAS of these Alu-related variants
<ul>
<li>Phewascatelog.org</li>
<li>Denny et al Nat Biotech 2013 <span class="citation">[<a href="#ref-Denny2014" role="doc-biblioref">375</a>]</span></li>
<li>Bush NRG <span class="citation">[<a href="#ref-Bush2016" role="doc-biblioref">376</a>]</span></li>
<li>Wei et al. Circulation 2018 <span class="citation">[<a href="#ref-Wei2018" role="doc-biblioref">377</a>]</span></li>
<li>Other papers (Zhao et al. Plos One 2019; We et al. BioRxiv 462077; Nielsen et al Nat Gen 2018; Giri et al. Nat Gen 2019; Bastarache et al Science (2018))</li>
<li><a href="http://pheweb.sph.umich.edu/" class="uri">http://pheweb.sph.umich.edu/</a></li>
</ul></li>
<li>Enrichment of nominal significant results with Age-related disease cf. Random</li>
</ul>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion" class="section level2">
<h2><span class="header-section-number">5.5</span> Discussion</h2>
<div style="page-break-after: always;"></div>
</div>
<div id="methods-1" class="section level2">
<h2><span class="header-section-number">5.6</span> Methods</h2>
<div id="participants" class="section level3">
<h3><span class="header-section-number">5.6.1</span> Participants</h3>
<p>Human participants were drawn from the TwinsUK Bioresource.
These deeply phenotyped individuals DNA methylomes</p>
<p>MeDIP-seq data as described in <span class="citation">[<a href="#ref-Bell2017a" role="doc-biblioref">67</a>]</span> – Briefly All data is available at</p>
</div>
</div>
<div id="age-related-dna-methylation-changes" class="section level2">
<h2><span class="header-section-number">5.7</span> Age-related DNA methylation changes</h2>
<p>To identify strongly genetically we ?utilised the Gap Hunting algorithm of Andrews et al. <span class="citation">[<a href="#ref-Andrews2016" role="doc-biblioref">166</a>]</span> adapted
We excluded all known
Excluded polymorphic Alus <span class="citation">[<a href="#ref-Payer2017" role="doc-biblioref">366</a>]</span>
?Exclude any methylation overlapping STRs</p>
<div id="validation-with-target-bis-seq" class="section level3">
<h3><span class="header-section-number">5.7.1</span> ?Validation with target BiS-seq</h3>
</div>
<div id="alu-element-specific-dna-methylation-clock-1" class="section level3">
<h3><span class="header-section-number">5.7.2</span> Alu element specific DNA methylation Clock</h3>
</div>
<div id="comparison-with-array-derived-clocks" class="section level3">
<h3><span class="header-section-number">5.7.3</span> Comparison with Array-derived Clocks</h3>
</div>
<div id="genetic-association-for-alu-clock" class="section level3">
<h3><span class="header-section-number">5.7.4</span> Genetic Association for Alu-Clock</h3>
</div>
<div id="association-of-these-variants-with-age-related-disease" class="section level3">
<h3><span class="header-section-number">5.7.5</span> Association of these variants with Age-related Disease</h3>
<div id="data-availability" class="section level4">
<h4><span class="header-section-number">5.7.5.1</span> Data availability</h4>
<p>The MeDIP-seq data supporting the results of this article are available in the EMBL-EBI European Genome-phenome Archive (EGA) under Data set Accession number EGAD00010000983 (<a href="https://www.ebi.ac.uk/ega/datasets/" class="uri">https://www.ebi.ac.uk/ega/datasets/</a>
EGAD00010000983).</p>
</div>
<div id="code-availablity" class="section level4">
<h4><span class="header-section-number">5.7.5.2</span> Code availablity</h4>
<p>Available at <a href="https://github.com/richardjacton">GitHub</a></p>
<div style="page-break-after: always;"></div>
</div>
<div id="acknowledgements" class="section level4">
<h4><span class="header-section-number">5.7.5.3</span> Acknowledgements</h4>
</div>
<div id="author-contributions" class="section level4">
<h4><span class="header-section-number">5.7.5.4</span> Author Contributions</h4>
<div style="page-break-after: always;"></div>

</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Lopez-Otin2013">
<p>9. López-Otín C, Blasco MA, Partridge L, Serrano M, Kroemer G: <strong>The hallmarks of aging.</strong> <em>Cell</em> 2013, <strong>153</strong>:1194–217<a href="https://doi.org/10.1016/j.cell.2013.05.039">10.1016/j.cell.2013.05.039</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23746838">http://www.ncbi.nlm.nih.gov/pubmed/23746838</a>.</p>
</div>
<div id="ref-Booth2016">
<p>10. Booth LN, Brunet A: <strong>The Aging Epigenome</strong>. <em>Molecular Cell</em> 2016, <strong>62</strong>:728–744<a href="https://doi.org/10.1016/j.molcel.2016.05.013">10.1016/j.molcel.2016.05.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2016.05.013">http://dx.doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div id="ref-Bird2007">
<p>16. Bird A: <strong>Perceptions of epigenetics.</strong> <em>Nature</em> 2007, <strong>447</strong>:396–8<a href="https://doi.org/10.1038/nature05913">10.1038/nature05913</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17522671">http://www.ncbi.nlm.nih.gov/pubmed/17522671</a>.</p>
</div>
<div id="ref-Yin2017">
<p>51. Yin Y, Morgunova E, Jolma A, Kaasinen E, Sahu B, Khund-Sayeed S, Das PK, Kivioja T, Dave K, Zhong F, Nitta KR, Taipale M, Popov A, Ginno PA, Domcke S, Yan J, Schübeler D, Vinson C, Taipale J: <strong>Impact of cytosine methylation on DNA binding specificities of human transcription factors.</strong> <em>Science (New York, N.Y.)</em> 2017, <strong>356</strong>:eaaj2239<a href="https://doi.org/10.1126/science.aaj2239">10.1126/science.aaj2239</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28473536">http://www.ncbi.nlm.nih.gov/pubmed/28473536</a>.</p>
</div>
<div id="ref-Wu2017">
<p>60. Wu X, Zhang Y: <strong>TET-mediated active DNA demethylation: mechanism, function and beyond</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:517–534<a href="https://doi.org/10.1038/nrg.2017.33">10.1038/nrg.2017.33</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.33">http://www.nature.com/doifinder/10.1038/nrg.2017.33</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>67. Bell CG, Gao F, Yuan W, Roos L, Acton RJ, Xia Y, Bell J, Ward K, Mangino M, Hysi PG, Wang J, Spector TD: <strong>Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci</strong>. <em>Nature Communications</em> 2018, <strong>9</strong>:8<a href="https://doi.org/10.1038/s41467-017-01586-1">10.1038/s41467-017-01586-1</a>Available: <a href="http://www.nature.com/articles/s41467-017-01586-1">http://www.nature.com/articles/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Wilson1983">
<p>90. Wilson VL, Jones PA: <strong>DNA methylation decreases in aging but not in immortal cells.</strong> <em>Science (New York, N.Y.)</em> 1983, <strong>220</strong>:1055–7Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.</p>
</div>
<div id="ref-Rakyan2010">
<p>106. Rakyan VK, Down TA, Maslau S, Andrew T, Yang TP, Beyan H, Whittaker P, McCann OT, Finer S, Valdes AM, Leslie RD, Deloukas P, Spector TD: <strong>Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains</strong>. <em>Genome Research</em> 2010, <strong>20</strong>:434–439<a href="https://doi.org/10.1101/gr.103101.109">10.1101/gr.103101.109</a>Available: <a href="http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109">http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109</a>.</p>
</div>
<div id="ref-Teschendorff2010">
<p>108. Teschendorff AE, Menon U, Gentry-Maharaj A, Ramus SJ, Weisenberger DJ, Shen H, Campan M, Noushmehr H, Bell CG, Maxwell AP, Savage DA, Mueller-Holzner E, Marth C, Kocjan G, Gayther SA, Jones A, Beck S, Wagner W, Laird PW, Jacobs IJ, Widschwendter M: <strong>Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.</strong> <em>Genome research</em> 2010, <strong>20</strong>:440–6<a href="https://doi.org/10.1101/gr.103606.109">10.1101/gr.103606.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20219944">http://www.ncbi.nlm.nih.gov/pubmed/20219944</a>.</p>
</div>
<div id="ref-Hannum2013">
<p>118. Hannum G, Guinney J, Zhao L, Zhang L, Hughes G, Sadda S, Klotzle B, Bibikova M, Fan J-B, Gao Y, Deconde R, Chen M, Rajapakse I, Friend S, Ideker T, Zhang K: <strong>Genome-wide methylation profiles reveal quantitative views of human aging rates.</strong> <em>Molecular cell</em> 2013, <strong>49</strong>:359–367<a href="https://doi.org/10.1016/j.molcel.2012.10.016">10.1016/j.molcel.2012.10.016</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23177740">http://www.ncbi.nlm.nih.gov/pubmed/23177740</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>119. Horvath S: <strong>DNA methylation age of human tissues and cell types.</strong> <em>Genome biology</em> 2013, <strong>14</strong>:R115<a href="https://doi.org/10.1186/gb-2013-14-10-r115">10.1186/gb-2013-14-10-r115</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24138928">http://www.ncbi.nlm.nih.gov/pubmed/24138928</a>.</p>
</div>
<div id="ref-Friedman2010">
<p>120. Friedman J, Hastie T, Tibshirani R: <strong>Regularization Paths for Generalized Linear Models via Coordinate Descent</strong>. <em>Journal of Statistical Software</em> 2010, <strong>33</strong>:7–10<a href="https://doi.org/10.18637/jss.v033.i01">10.18637/jss.v033.i01</a>Available: <a href="http://www.jstatsoft.org/v33/i01/">http://www.jstatsoft.org/v33/i01/</a>.</p>
</div>
<div id="ref-Marioni2015">
<p>135. Marioni RE, Shah S, McRae AF, Chen BH, Colicino E, Harris SE, Gibson J, Henders AK, Redmond P, Cox SR, Pattie A, Corley J, Murphy L, Martin NG, Montgomery GW, Feinberg AP, Fallin MD, Multhaup ML, Jaffe AE, Joehanes R, Schwartz J, Just AC, Lunetta KL, Murabito JM, Starr JM, Horvath S, Baccarelli AA, Levy D, Visscher PM, Wray NR, et al.: <strong>DNA methylation age of blood predicts all-cause mortality in later life.</strong> <em>Genome biology</em> 2015, <strong>16</strong>:25<a href="https://doi.org/10.1186/s13059-015-0584-6">10.1186/s13059-015-0584-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25633388">http://www.ncbi.nlm.nih.gov/pubmed/25633388</a>.</p>
</div>
<div id="ref-Levine2018">
<p>138. Levine ME, Lu AT, Quach A, Chen BH, Assimes TL, Bandinelli S, Hou L, Baccarelli AA, Stewart JD, Li Y, Whitsel EA, Wilson JG, Reiner AP, Aviv A, Lohman K, Liu Y, Ferrucci L, Horvath S: <strong>An epigenetic biomarker of aging for lifespan and healthspan</strong>. <em>Aging</em> 2018, <strong>10</strong>:573–591<a href="https://doi.org/10.18632/aging.101414">10.18632/aging.101414</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29676998">http://www.ncbi.nlm.nih.gov/pubmed/29676998</a>.</p>
</div>
<div id="ref-Lu2019">
<p>139. Lu AT, Quach A, Wilson JG, Reiner AP, Aviv A, Raj K, Hou L, Baccarelli AA, Li Y, Stewart JD, Whitsel EA, Assimes TL, Ferrucci L, Horvath S: <strong>DNA methylation GrimAge strongly predicts lifespan and healthspan</strong>. <em>Aging</em> 2019, <strong>11</strong>:303–327<a href="https://doi.org/10.18632/aging.101684">10.18632/aging.101684</a>Available: <a href="http://www.aging-us.com/article/101684/text">http://www.aging-us.com/article/101684/text</a>.</p>
</div>
<div id="ref-Zhang2017">
<p>140. Zhang Y, Hapala J, Brenner H, Wagner W: <strong>Individual CpG sites that are associated with age and life expectancy become hypomethylated upon aging</strong>. <em>Clinical Epigenetics</em> 2017, <strong>9</strong>:9<a href="https://doi.org/10.1186/s13148-017-0315-9">10.1186/s13148-017-0315-9</a>Available: <a href="http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9">http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9</a>.</p>
</div>
<div id="ref-Horvath2018">
<p>149. Horvath S, Raj K: <strong>DNA methylation-based biomarkers and the epigenetic clock theory of ageing</strong>. <em>Nature Reviews Genetics</em> 2018, <strong>19</strong>:371–384<a href="https://doi.org/10.1038/s41576-018-0004-3">10.1038/s41576-018-0004-3</a>Available: <a href="http://www.nature.com/articles/s41576-018-0004-3">http://www.nature.com/articles/s41576-018-0004-3</a>.</p>
</div>
<div id="ref-Bell2019">
<p>150. Bell CG, Lowe R, Adams PD, Baccarelli AA, Beck S, Bell JT, Christensen BC, Gladyshev VN, Heijmans BT, Horvath S, Ideker T, Issa J-PJ, Kelsey KT, Marioni RE, Reik W, Relton CL, Schalkwyk LC, Teschendorff AE, Wagner W, Zhang K, Rakyan VK: <strong>DNA methylation aging clocks: challenges and recommendations.</strong> <em>Genome biology</em> 2019, <strong>20</strong>:249<a href="https://doi.org/10.1186/s13059-019-1824-y">10.1186/s13059-019-1824-y</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31767039">http://www.ncbi.nlm.nih.gov/pubmed/31767039</a>.</p>
</div>
<div id="ref-Andrews2016">
<p>166. Andrews SV, Ladd-Acosta C, Feinberg AP, Hansen KD, Fallin MD: <strong>“Gap hunting” to characterize clustered probe signals in Illumina methylation array data</strong>. <em>Epigenetics &amp; Chromatin</em> 2016, <strong>9</strong>:56<a href="https://doi.org/10.1186/s13072-016-0107-z">10.1186/s13072-016-0107-z</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z</a>.</p>
</div>
<div id="ref-Acton2020">
<p>225. Acton RJ, Bourne E, Bell J, Lillycrop K, Wang J, Dennison E, Harvey N, Spector TD, Cooper C, Bell CG: <strong>The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</strong>. <em>bioRxiv</em> 2019,:1–48<a href="https://doi.org/10.1101/870352">10.1101/870352</a>Available: <a href="https://www.biorxiv.org/content/10.1101/870352v1">https://www.biorxiv.org/content/10.1101/870352v1</a>.</p>
</div>
<div id="ref-Partridge2018">
<p>226. Partridge L, Deelen J, Slagboom PE: <strong>Facing up to the global challenges of ageing</strong>. <em>Nature</em> 2018, <strong>561</strong>:45–56<a href="https://doi.org/10.1038/s41586-018-0457-8">10.1038/s41586-018-0457-8</a>Available: <a href="http://www.nature.com/articles/s41586-018-0457-8">http://www.nature.com/articles/s41586-018-0457-8</a>.</p>
</div>
<div id="ref-Chuong2017">
<p>228. Chuong EB, Elde NC, Feschotte C: <strong>Regulatory activities of transposable elements: from conflicts to benefits</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:71–86<a href="https://doi.org/10.1038/nrg.2016.139">10.1038/nrg.2016.139</a>Available: <a href="http://www.nature.com/articles/nrg.2016.139">http://www.nature.com/articles/nrg.2016.139</a>.</p>
</div>
<div id="ref-Yang2016">
<p>310. Yang Z, Wong A, Kuh D, Paul DS, Rakyan VK, Leslie RD, Zheng SC, Widschwendter M, Beck S, Teschendorff AE: <strong>Correlation of an epigenetic mitotic clock with cancer risk</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:205<a href="https://doi.org/10.1186/s13059-016-1064-3">10.1186/s13059-016-1064-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3</a>.</p>
</div>
<div id="ref-Gregory2005">
<p>342. Gregory TR: <strong>Synergy between sequence and size in large-scale genomics</strong>. <em>Nature Reviews Genetics</em> 2005, <strong>6</strong>:699–708<a href="https://doi.org/10.1038/nrg1674">10.1038/nrg1674</a>.</p>
</div>
<div id="ref-Deniz2019">
<p>343. Deniz Ö, Frost JM, Branco MR: <strong>Regulation of transposable elements by DNA modifications</strong>. <em>Nature Reviews Genetics</em> 2019, <strong>20</strong>:417–431<a href="https://doi.org/10.1038/s41576-019-0106-6">10.1038/s41576-019-0106-6</a>.</p>
</div>
<div id="ref-Kazazian2017">
<p>344. Kazazian HH, Moran JV: <strong>Mobile DNA in health and disease</strong>. <em>New England Journal of Medicine</em> 2017, <strong>377</strong>:361–370<a href="https://doi.org/10.1056/NEJMra1510092">10.1056/NEJMra1510092</a>.</p>
</div>
<div id="ref-Deaton2011">
<p>345. Deaton AM, Bird A: <strong>CpG islands and the regulation of transcription</strong>. <em>Genes &amp; development</em> 2011, <strong>25</strong>:1010–22<a href="https://doi.org/10.1101/gad.2037511">10.1101/gad.2037511</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21576262">http://www.ncbi.nlm.nih.gov/pubmed/21576262</a>.</p>
</div>
<div id="ref-Hellman2007">
<p>346. Hellman A, Chess A: <strong>Gene body-specific methylation on the active X chromosome</strong>. <em>Science</em> 2007, <strong>315</strong>:1141–1143<a href="https://doi.org/10.1126/science.1136352">10.1126/science.1136352</a>.</p>
</div>
<div id="ref-Shukla2011">
<p>347. Shukla S, Kavak E, Gregory M, Imashimizu M, Shutinoski B, Kashlev M, Oberdoerffer P, Sandberg R, Oberdoerffer S: <strong>CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing</strong>. <em>Nature</em> 2011, <strong>479</strong>:74–79<a href="https://doi.org/10.1038/nature10442">10.1038/nature10442</a>Available: <a href="http://dx.doi.org/10.1038/nature10442">http://dx.doi.org/10.1038/nature10442</a>.</p>
</div>
<div id="ref-Blattler2014">
<p>348. Blattler A, Yao L, Witt H, Guo Y, Nicolet CM, Berman BP, Farnham PJ: <strong>Global loss of DNA methylation uncovers intronic enhancers in genes showing expression changes</strong>. <em>Genome Biology</em> 2014, <strong>15</strong>:469<a href="https://doi.org/10.1186/s13059-014-0469-0">10.1186/s13059-014-0469-0</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0</a>.</p>
</div>
<div id="ref-Gama-Sosa1983">
<p>349. Gama-Sosa MA, Slagel VA, Trewyn RW, Oxenhandler R, Kuo KC, Gehrke CW, Ehrlich M: <strong>The 5-methylcytosine content of DNA from human tumors</strong>. <em>Nucleic Acids Research</em> 1983, <strong>11</strong>:6883–6894<a href="https://doi.org/10.1093/nar/11.19.6883">10.1093/nar/11.19.6883</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/11.19.6883">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/11.19.6883</a>.</p>
</div>
<div id="ref-Bollati2010">
<p>350. Bollati V, Schwartz J, Wright R, Litonjua A, Tarantini L, Suh H, Sparrow D, Vokonas P, Baccarelli A: <strong>Decline in genomic DNA methylation through aging in a cohort of elderly subjects</strong>. <em>Mechanisms of Ageing and Development</em> 2009, <strong>130</strong>:234–239<a href="https://doi.org/10.1016/j.mad.2008.12.003">10.1016/j.mad.2008.12.003</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820">https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820</a>.</p>
</div>
<div id="ref-Dewannieux2003">
<p>351. Dewannieux M, Esnault C, Heidmann T: <strong>LINE-mediated retrotransposition of marked Alu sequences</strong>. <em>Nature Genetics</em> 2003, <strong>35</strong>:41–48<a href="https://doi.org/10.1038/ng1223">10.1038/ng1223</a>Available: <a href="http://www.nature.com/articles/ng1223">http://www.nature.com/articles/ng1223</a>.</p>
</div>
<div id="ref-Deininger2011">
<p>352. Deininger P: <strong>Alu elements: know the SINEs</strong>. <em>Genome Biology</em> 2011, <strong>12</strong>:236<a href="https://doi.org/10.1186/gb-2011-12-12-236">10.1186/gb-2011-12-12-236</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236">http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236</a>.</p>
</div>
<div id="ref-Ullu1984">
<p>353. Ullu E, Tschudi C: <strong>Alu sequences are processed 7SL RNA genes</strong>. <em>Nature</em> 1984, <strong>312</strong>:171–172<a href="https://doi.org/10.1038/312171a0">10.1038/312171a0</a>Available: <a href="http://www.nature.com/articles/312171a0">http://www.nature.com/articles/312171a0</a>.</p>
</div>
<div id="ref-Ade2013">
<p>354. Ade C, Roy-Engel AM, Deininger PL: <strong>Alu elements: an intrinsic source of human genome instability</strong>. <em>Current Opinion in Virology</em> 2013, <strong>3</strong>:639–645<a href="https://doi.org/10.1016/j.coviro.2013.09.002">10.1016/j.coviro.2013.09.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533">https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533</a>.</p>
</div>
<div id="ref-Cordaux2009">
<p>355. Cordaux R, Batzer MA: <strong>The impact of retrotransposons on human genome evolution</strong>. <em>Nature Reviews Genetics</em> 2009, <strong>10</strong>:691–703<a href="https://doi.org/10.1038/nrg2640">10.1038/nrg2640</a>Available: <a href="http://www.nature.com/articles/nrg2640">http://www.nature.com/articles/nrg2640</a>.</p>
</div>
<div id="ref-Belancio2008">
<p>356. Belancio VP, Hedges DJ, Deininger P: <strong>Mammalian non-LTR retrotransposons: For better or worse, in sickness and in health</strong>. <em>Genome Research</em> 2008, <strong>18</strong>:343–358<a href="https://doi.org/10.1101/gr.5558208">10.1101/gr.5558208</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.5558208">http://www.genome.org/cgi/doi/10.1101/gr.5558208</a>.</p>
</div>
<div id="ref-Jorda2017">
<p>357. Jordà M, Díez-Villanueva A, Mallona I, Martín B, Lois S, Barrera V, Esteller M, Vavouri T, Peinado MA: <strong>The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells</strong>. <em>Genome Research</em> 2017, <strong>27</strong>:118–132<a href="https://doi.org/10.1101/gr.207522.116">10.1101/gr.207522.116</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27999094">http://www.ncbi.nlm.nih.gov/pubmed/27999094</a>.</p>
</div>
<div id="ref-Polak2006">
<p>358. Polak P, Domany E: <strong>Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes.</strong> <em>BMC genomics</em> 2006, <strong>7</strong>:133<a href="https://doi.org/10.1186/1471-2164-7-133">10.1186/1471-2164-7-133</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16740159">http://www.ncbi.nlm.nih.gov/pubmed/16740159</a>.</p>
</div>
<div id="ref-Chen2017b">
<p>359. Chen L-L, Yang L: <strong>ALU ternative Regulation for Gene Expression</strong>. <em>Trends in Cell Biology</em> 2017, <strong>27</strong>:480–490<a href="https://doi.org/10.1016/j.tcb.2017.01.002">10.1016/j.tcb.2017.01.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028">https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028</a>.</p>
</div>
<div id="ref-RobertoFerrari2019">
<p>360. Ferrari R, de Llobet Cucalon LI, Di Vona C, Le Dilly F, Vidal E, Lioutas A, Oliete JQ, Jochem L, Cutts E, Dieci G, Vannini A, Teichmann M, Luna S de la, Beato M: <strong>TFIIIC Binding to Alu Elements Controls Gene Expression via Chromatin Looping and Histone Acetylation</strong>. <em>Molecular Cell</em> 2019,:1–13<a href="https://doi.org/10.1016/j.molcel.2019.10.020">10.1016/j.molcel.2019.10.020</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981">https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981</a>.</p>
</div>
<div id="ref-Su2014">
<p>361. Su M, Han D, Boyd-Kirkup J, Yu X, Han J-DJ: <strong>Evolution of Alu Elements toward Enhancers</strong>. <em>Cell Reports</em> 2014, <strong>7</strong>:376–385<a href="https://doi.org/10.1016/j.celrep.2014.03.011">10.1016/j.celrep.2014.03.011</a>Available: <a href="http://dx.doi.org/10.1016/j.celrep.2014.03.011">http://dx.doi.org/10.1016/j.celrep.2014.03.011</a>.</p>
</div>
<div id="ref-Ward2013">
<p>362. Ward MC, Wilson MD, Barbosa-Morais NL, Schmidt D, Stark R, Pan Q, Schwalie PC, Menon S, Lukk M, Watt S, Thybert D, Kutter C, Kirschner K, Flicek P, Blencowe BJ, Odom DT: <strong>Latent Regulatory Potential of Human-Specific Repetitive Elements</strong>. <em>Molecular Cell</em> 2013, <strong>49</strong>:262–272<a href="https://doi.org/10.1016/j.molcel.2012.11.013">10.1016/j.molcel.2012.11.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2012.11.013">http://dx.doi.org/10.1016/j.molcel.2012.11.013</a>.</p>
</div>
<div id="ref-Xie2013">
<p>363. Xie M, Hong C, Zhang B, Lowdon RF, Xing X, Li D, Zhou X, Lee HJ, Maire CL, Ligon KL, Gascard P, Sigaroudinia M, Tlsty TD, Kadlecek T, Weiss A, O’Geen H, Farnham PJ, Madden PAF, Mungall AJ, Tam A, Kamoh B, Cho S, Moore R, Hirst M, Marra MA, Costello JF, Wang T: <strong>DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape</strong>. <em>Nature Genetics</em> 2013, <strong>45</strong>:836–841<a href="https://doi.org/10.1038/ng.2649">10.1038/ng.2649</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23708189">http://www.ncbi.nlm.nih.gov/pubmed/23708189</a>.</p>
</div>
<div id="ref-Jang2019">
<p>364. Jang HS, Shah NM, Du AY, Dailey ZZ, Pehrsson EC, Godoy PM, Zhang D, Li D, Xing X, Kim S, O’Donnell D, Gordon JI, Wang T: <strong>Transposable elements drive widespread expression of oncogenes in human cancers</strong>. <em>Nature Genetics</em> 2019, <strong>51</strong>:611–617<a href="https://doi.org/10.1038/s41588-019-0373-3">10.1038/s41588-019-0373-3</a>Available: <a href="http://dx.doi.org/10.1038/s41588-019-0373-3">http://dx.doi.org/10.1038/s41588-019-0373-3</a>.</p>
</div>
<div id="ref-Rajendiran2016">
<p>365. Rajendiran S, Gibbs LD, Van Treuren T, Klinkebiel DL, Vishwanatha JK: <strong>MIEN1 is tightly regulated by SINE Alu methylation in its promoter</strong>. <em>Oncotarget</em> 2016, <strong>7</strong>:65307–65319<a href="https://doi.org/10.18632/oncotarget.11675">10.18632/oncotarget.11675</a>Available: <a href="http://www.oncotarget.com/fulltext/11675">http://www.oncotarget.com/fulltext/11675</a>.</p>
</div>
<div id="ref-Payer2017">
<p>366. Payer LM, Steranka JP, Yang WR, Kryatova M, Medabalimi S, Ardeljan D, Liu C, Boeke JD, Avramopoulos D, Burns KH: <strong>Structural variants caused by Alu insertions are associated with risks for many human diseases</strong>. <em>Proceedings of the National Academy of Sciences</em> 2017, <strong>114</strong>:E3984–E3992<a href="https://doi.org/10.1073/pnas.1704117114">10.1073/pnas.1704117114</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114">http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114</a>.</p>
</div>
<div id="ref-Mozhui2017">
<p>367. Mozhui K, Pandey AK: <strong>Conserved effect of aging on DNA methylation and association with EZH2 polycomb protein in mice and humans</strong>. <em>Mechanisms of Ageing and Development</em> 2017, <strong>162</strong>:27–37<a href="https://doi.org/10.1016/j.mad.2017.02.006">10.1016/j.mad.2017.02.006</a>Available: <a href="http://dx.doi.org/10.1016/j.mad.2017.02.006">http://dx.doi.org/10.1016/j.mad.2017.02.006</a>.</p>
</div>
<div id="ref-Lu2017">
<p>368. Lu AT, Hannon E, Levine ME, Crimmins EM, Lunnon K, Mill J, Geschwind DH, Horvath S: <strong>Genetic architecture of epigenetic and neuronal ageing rates in human brain regions</strong>. <em>Nature Communications</em> 2017, <strong>8</strong>:15353<a href="https://doi.org/10.1038/ncomms15353">10.1038/ncomms15353</a>Available: <a href="http://dx.doi.org/10.1038/ncomms15353">http://dx.doi.org/10.1038/ncomms15353</a>.</p>
</div>
<div id="ref-Gibson2019">
<p>369. Gibson J, Russ TC, Clarke T-K, Howard DM, Hillary RF, Evans KL, Walker RM, Bermingham ML, Morris SW, Campbell A, Hayward C, Murray AD, Porteous DJ, Horvath S, Lu AT, McIntosh AM, Whalley HC, Marioni RE: <strong>A meta-analysis of genome-wide association studies of epigenetic age acceleration</strong>. <em>PLOS Genetics</em> 2019, <strong>15</strong>:e1008104<a href="https://doi.org/10.1371/journal.pgen.1008104">10.1371/journal.pgen.1008104</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31738745">http://www.ncbi.nlm.nih.gov/pubmed/31738745</a>.</p>
</div>
<div id="ref-Bycroft2017">
<p>370. Bycroft C, Freeman C, Petkova D, Band G, Elliott LT, Sharp K, Motyer A, Vukcevic D, Delaneau O, O’Connell J, Cortes A, Welsh S, McVean G, Leslie S, Donnelly P, Marchini J: <strong>Genome-wide genetic data on ~500,000 UK Biobank participants</strong>. <em>bioRxiv</em> 2017,:166298<a href="https://doi.org/10.1101/166298">10.1101/166298</a>Available: <a href="https://www.biorxiv.org/content/10.1101/166298v1">https://www.biorxiv.org/content/10.1101/166298v1</a>.</p>
</div>
<div id="ref-McLean2010">
<p>371. McLean CY, Bristor D, Hiller M, Clarke SL, Schaar BT, Lowe CB, Wenger AM, Bejerano G: <strong>GREAT improves functional interpretation of cis-regulatory regions</strong>. <em>Nature Biotechnology</em> 2010, <strong>28</strong>:495–501<a href="https://doi.org/10.1038/nbt.1630">10.1038/nbt.1630</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20436461">http://www.ncbi.nlm.nih.gov/pubmed/20436461</a>.</p>
</div>
<div id="ref-Engebretsen2019">
<p>372. Engebretsen S, Bohlin J: <strong>Statistical predictions with glmnet</strong>. <em>Clinical Epigenetics</em> 2019, <strong>11</strong>:123<a href="https://doi.org/10.1186/s13148-019-0730-1">10.1186/s13148-019-0730-1</a>Available: <a href="https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-019-0730-1">https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-019-0730-1</a>.</p>
</div>
<div id="ref-Zhou2018">
<p>373. Zhou W, Dinh HQ, Ramjan Z, Weisenberger DJ, Nicolet CM, Shen H, Laird PW, Berman BP: <strong>DNA methylation loss in late-replicating domains is linked to mitotic cell division</strong>. <em>Nature Genetics</em> 2018, <strong>50</strong>:591–602<a href="https://doi.org/10.1038/s41588-018-0073-4">10.1038/s41588-018-0073-4</a>Available: <a href="http://dx.doi.org/10.1038/s41588-018-0073-4">http://dx.doi.org/10.1038/s41588-018-0073-4</a>.</p>
</div>
<div id="ref-Zenin2019">
<p>374. Zenin A, Tsepilov Y, Sharapov S, Getmantsev E, Menshikov LI, Fedichev PO, Aulchenko Y: <strong>Identification of 12 genetic loci associated with human healthspan</strong>. <em>Communications Biology</em> 2019, <strong>2</strong>:41<a href="https://doi.org/10.1038/s42003-019-0290-0">10.1038/s42003-019-0290-0</a>Available: <a href="http://dx.doi.org/10.1038/s42003-019-0290-0">http://dx.doi.org/10.1038/s42003-019-0290-0</a>.</p>
</div>
<div id="ref-Denny2014">
<p>375. Denny JC, Bastarache L, Ritchie MD, Carroll RJ, Zink R, Mosley JD, Field JR, Pulley JM, Ramirez AH, Bowton E, Basford M a, Carrell DS, Peissig PL, Kho AN, Pacheco J a, Rasmussen LV, Crosslin DR, Crane PK, Pathak J, Bielinski SJ, Pendergrass SA, Xu H, Hindorff LA, Li R, Manolio TA, Chute CG, Chisholm RL, Larson EB, Jarvik GP, Brilliant MH, et al.: <strong>Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data</strong>. <em>Nature Biotechnology</em> 2013, <strong>31</strong>:1102–1111<a href="https://doi.org/10.1038/nbt.2749">10.1038/nbt.2749</a>Available: <a href="http://www.nature.com/articles/nbt.2749">http://www.nature.com/articles/nbt.2749</a>.</p>
</div>
<div id="ref-Bush2016">
<p>376. Bush WS, Oetjens MT, Crawford DC: <strong>Unravelling the human genome-phenome relationship using phenome-wide association studies</strong>. <em>Nature Reviews Genetics</em> 2016, <strong>17</strong>:129–145<a href="https://doi.org/10.1038/nrg.2015.36">10.1038/nrg.2015.36</a>Available: <a href="http://dx.doi.org/10.1038/nrg.2015.36">http://dx.doi.org/10.1038/nrg.2015.36</a>.</p>
</div>
<div id="ref-Wei2018">
<p>377. Wei W-Q, Li X, Feng Q, Kubo M, Kullo IJ, Peissig PL, Karlson EW, Jarvik GP, Lee MTM, Shang N, Larson EA, Edwards T, Shaffer CM, Mosley JD, Maeda S, Horikoshi M, Ritchie M, Williams MS, Larson EB, Crosslin DR, Bland ST, Pacheco JA, Rasmussen-Torvik LJ, Cronkite D, Hripcsak G, Cox NJ, Wilke RA, Stein CM, Rotter JI, Momozawa Y, et al.: <strong>LPA Variants Are Associated With Residual Cardiovascular Risk in Patients Receiving Statins</strong>. <em>Circulation</em> 2018, <strong>138</strong>:1839–1849<a href="https://doi.org/10.1161/CIRCULATIONAHA.117.031356">10.1161/CIRCULATIONAHA.117.031356</a>Available: <a href="https://www.ahajournals.org/doi/10.1161/CIRCULATIONAHA.117.031356">https://www.ahajournals.org/doi/10.1161/CIRCULATIONAHA.117.031356</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="tRNAs.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/05-Alu.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
