<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Results 3 - DNA methylation clocks in Alu Elements | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Results 3 - DNA methylation clocks in Alu Elements | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Results 3 - DNA methylation clocks in Alu Elements | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-08-30" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="tRNAs.html"/>
<link rel="next" href="discussion.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#GenMethodsMethArray"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a><ul>
<li class="chapter" data-level="3.1" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.2.1</b> Outline of EWAS</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.3.2</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.3</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.4" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.4</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.4.1</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.4.2</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.4.3</b> Southampton Womenâ€™s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-abstract"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Introduction"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#tRNAmethods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Participants"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-methylation-in-cancer"><i class="fa fa-check"></i><b>4.5</b> tRNA Gene Methylation in Cancer</a></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Discussion"><i class="fa fa-check"></i><b>4.6</b> Discussion</a></li>
<li class="chapter" data-level="4.7" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-data-avail"><i class="fa fa-check"></i><b>4.7</b> Data availability</a></li>
<li class="chapter" data-level="4.8" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-code-avail"><i class="fa fa-check"></i><b>4.8</b> Code availablity</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>5</b> Results 3 - DNA methylation clocks in Alu Elements</a><ul>
<li class="chapter" data-level="5.1" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>5.3.1</b> DNA methylome data</a></li>
<li class="chapter" data-level="5.3.2" data-path="Alus.html"><a href="Alus.html#alu-element-annotation-and-pre-filtering"><i class="fa fa-check"></i><b>5.3.2</b> Alu element annotation and pre-filtering</a></li>
<li class="chapter" data-level="5.3.3" data-path="Alus.html"><a href="Alus.html#elastic-net-regression"><i class="fa fa-check"></i><b>5.3.3</b> Elastic net regression</a></li>
<li class="chapter" data-level="5.3.4" data-path="Alus.html"><a href="Alus.html#sample-selection"><i class="fa fa-check"></i><b>5.3.4</b> Sample Selection</a></li>
<li class="chapter" data-level="5.3.5" data-path="Alus.html"><a href="Alus.html#binarisation"><i class="fa fa-check"></i><b>5.3.5</b> Binarisation</a></li>
<li class="chapter" data-level="5.3.6" data-path="Alus.html"><a href="Alus.html#AluMethodsArrays"><i class="fa fa-check"></i><b>5.3.6</b> Comparisons to other clocks in TwinsUK Illumina 450k array data</a></li>
<li class="chapter" data-level="5.3.7" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.3.7</b> Genetic Association for Alu-Clock</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>5.4</b> Results</a></li>
<li class="chapter" data-level="5.5" data-path="Alus.html"><a href="Alus.html#comparison-to-other-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.5</b> Comparison to other DNA methylation clocks</a><ul>
<li class="chapter" data-level="5.5.1" data-path="Alus.html"><a href="Alus.html#alu-age-acceleration"><i class="fa fa-check"></i><b>5.5.1</b> Alu Age Acceleration</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>5.6</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="Alus" class="section level1">
<h1><span class="header-section-number">5</span> Results 3 - DNA methylation clocks in Alu Elements</h1>
<div id="abstract" class="section level2">
<h2><span class="header-section-number">5.1</span> Abstract</h2>
</div>
<div id="AluIntro" class="section level2">
<h2><span class="header-section-number">5.2</span> Introduction</h2>
<p>As we age biological systems from the molecular to the macroscopic undergo functional decline.
This increasingly impaired function with time results in increased risk for a wide variety of chronic disease states and indeed for co-morbidities with these conditions.
Ranging from the cardiovascular, opthalmic, and musculoskeletal through to neurological conditions and obesity, all have age as a primary risk factor <span class="citation">[<a href="#ref-Partridge2018" role="doc-biblioref">226</a>]</span>.
Among the molecular changes that occur with age are alterations to the epigenome <span class="citation">[<a href="#ref-Booth2016" role="doc-biblioref">10</a>]</span>.
The epigenome here refers to: The chemical modifications to the genome and the molecules which package it.
These modifications do not affect the genomic sequence but do impact the accessibility or expression of the genome in different contexts, such as different cell-types <span class="citation">[<a href="#ref-Bird2007" role="doc-biblioref">16</a>]</span>.</p>
<p>Approximately 45% of the human genome is comprised of repetitive or transposable elements <span class="citation">[<a href="#ref-Deniz2019" role="doc-biblioref">341</a>,<a href="#ref-Kazazian2017" role="doc-biblioref">342</a>]</span>.
The repression of these elements to prevent their expansion or the impact of their latent regulatory potential from effecting genomic stability and regulation is a core function of epigenetic mechanisms <span class="citation">[<a href="#ref-Gregory2005" role="doc-biblioref">343</a>]</span>.
The most common DNA modification, 5-methylcytosine (5mC) at CpG loci, exhibits the most well documented age-related changes, and plays a central role in the repression of repetitive and transposable elements <span class="citation">[<a href="#ref-Lopez-Otin2013" role="doc-biblioref">9</a>]</span>.
DNA methylation also classically functions to repress gene expression at promoters <span class="citation">[<a href="#ref-Deaton2011" role="doc-biblioref">344</a>]</span>, similar to the repressive role that is plays at repetitive elements.
Beyond solely repressive effects, DNA methylation in gene bodies can modulate expression <span class="citation">[<a href="#ref-Hellman2007" role="doc-biblioref">345</a>]</span> and influence splicing patterns <span class="citation">[<a href="#ref-Shukla2011" role="doc-biblioref">346</a>]</span>.
Furthermore, DNA methylation levels at transcription factor binding sites (TFBS) modulate the binding affinity of transcriptions factors for these sites <span class="citation">[<a href="#ref-Yin2017" role="doc-biblioref">51</a>,<a href="#ref-Blattler2014" role="doc-biblioref">347</a>]</span>.</p>
<p>Significant decreases in bulk 5mC with age <span class="citation">[<a href="#ref-Wilson1983" role="doc-biblioref">90</a>]</span> and in cancer tissues <span class="citation">[<a href="#ref-Gama-Sosa1983" role="doc-biblioref">348</a>]</span> are among the earliest pioneering measure of global DNAm changes.
Demethylation of repetitive elements was thought to be a major contributor to the global decreases in 5mC <span class="citation">[<a href="#ref-Bollati2010" role="doc-biblioref">349</a>]</span>.
More recent studies using methods able to resolve the genomic position of DNA methylation changes have revealed that some regions of the genome show increasing DNA methylation with age.
Notably, the promoters of polycomb target genes <span class="citation">[<a href="#ref-Teschendorff2010" role="doc-biblioref">108</a>]</span>, bivalent chromatin domains <span class="citation">[<a href="#ref-Rakyan2010" role="doc-biblioref">106</a>]</span> and specific tRNA genes <span class="citation">[<a href="#ref-Acton2020" role="doc-biblioref">225</a>]</span>.</p>
<p>Alu elements are the single most abundant transposon in the human genome with ~1.19 million copies and comprising ~10.7% of the total genomic sequence <span class="citation">[<a href="#ref-Dewannieux2003" role="doc-biblioref">350</a>,<a href="#ref-Deininger2011" role="doc-biblioref">351</a>]</span>.
Alu elements are members of the short interspersed nuclear elements (SINEs) family of retrotransposons, are ~300bp in length, and are derived from the 7SL RNA gene <span class="citation">[<a href="#ref-Ullu1984" role="doc-biblioref">352</a>]</span>.
The Alu elements arose from the 7SL sequence via the FAM sequences, which gave rise to truncated variants, FRAM and FLAM the fusion of which yielded the current dimeric Alu sequence <span class="citation">[<a href="#ref-Quentin1992" role="doc-biblioref">353</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:AluStruct"></span>
<img src="figs/AluStructure.png" alt="Structure of an Alu element A complete Alu element contains the A and B box sequence elements from an RNA polIII promoter, a central A rich region, a short insertion in the right arm, and a poly A region involved in retrotransposition and insertion. There is not an explicit polIII terminator in the Alu sequence so transcription continues through arbitrary flanking sequence until a run of at least 4 Ts, is encountered [354,355]. FLAM sequences are the origin of the Left arm and FRAMs the origin of the Right [[353]." width="90%" />
<p class="caption">
Figure 5.1: <strong>Structure of an Alu element</strong> A complete Alu element contains the A and B box sequence elements from an RNA polIII promoter, a central A rich region, a short insertion in the right arm, and a poly A region involved in retrotransposition and insertion. There is not an explicit polIII terminator in the Alu sequence so transcription continues through arbitrary flanking sequence until a run of at least 4 Ts, is encountered <span class="citation">[<a href="#ref-Chen2008" role="doc-biblioref">354</a>,<a href="#ref-Batzer2002" role="doc-biblioref">355</a>]</span>. FLAM sequences are the origin of the Left arm and FRAMs the origin of the Right [<span class="citation">[<a href="#ref-Quentin1992" role="doc-biblioref">353</a>]</span>.
</p>
</div>

<p>Alus are primate specific and arose ~65 million years ago (MYA), with a peak in amplification around ~40 MYA (Figure <a href="Alus.html#fig:AluAge">5.2</a>) <span class="citation">[<a href="#ref-Ade2013" role="doc-biblioref">356</a>]</span>.
Non-LTR retrotransposons, including Alu, L1 and SVA elements, are active in humans.
However, Alu elements are non-autonomous and require the presence of an L1 element to provide the molecular machinery needed for retrotransposition <span class="citation">[<a href="#ref-Cordaux2009" role="doc-biblioref">357</a>]</span>.
Mutagenesis arising from Alu element insertion and Alu element mediated recombination is responsible for an estimated 0.4% of all human genetic disorders <span class="citation">[<a href="#ref-Kazazian2017" role="doc-biblioref">342</a>,<a href="#ref-Belancio2008" role="doc-biblioref">358</a>]</span></p>
<div class="figure" style="text-align: center"><span id="fig:AluAge"></span>
<img src="figs/Batzer2002_fig2_Alu_age_expansion.png" alt="Alu element Age and history of expansion in primates Mya (milion years ago). Reproduced from Batzer &amp; Deininger 2002 [355]." width="90%" />
<p class="caption">
Figure 5.2: <strong>Alu element Age and history of expansion in primates</strong> Mya (milion years ago). Reproduced from Batzer &amp; Deininger 2002 <span class="citation">[<a href="#ref-Batzer2002" role="doc-biblioref">355</a>]</span>.
</p>
</div>

<p>Alu elements are CpG rich (Figure <a href="Alus.html#fig:AlusWindowsCpGdLVplot">5.3</a>), accounting for ~7.5 million CpGs, or ~25% of all human CpG sites <span class="citation">[<a href="#ref-Deininger2011" role="doc-biblioref">351</a>]</span>.
High levels of DNA methylation predominate at Alu elements across normal tissues.
Furthermore Alu elements are more resistant to hypomethylation than other repetitive elements in cancer cells.
This is indicative of strong pressure to maintain their repressed state <span class="citation">[<a href="#ref-Jorda2017" role="doc-biblioref">359</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:AlusWindowsCpGdLVplot"></span>
<img src="figs/AlusWindowsCpGdLVplot-FAMs.png" alt="Alu elements are CpG dense" width="80%" />
<p class="caption">
Figure 5.3: <strong>Alu elements are CpG dense</strong>
</p>
</div>

<p>Alu elements contain many sequences with the potential to effect gene regulation, and indeed do effect gene expression <span class="citation">[<a href="#ref-Chen2017b" role="doc-biblioref">360</a>,<a href="#ref-RobertoFerrari2019" role="doc-biblioref">361</a>]</span>.
There are numerous TFBSs in the canonical Alu sequence <span class="citation">[<a href="#ref-Polak2006" role="doc-biblioref">362</a>]</span>, and some may possess enhancer activity <span class="citation">[<a href="#ref-Su2014" role="doc-biblioref">363</a>]</span>.
Thus deterioration in the effectiveness of the repression of Alu elements has the potential to disruptively expose this latent regulatory function <span class="citation">[<a href="#ref-Chuong2017" role="doc-biblioref">228</a>,<a href="#ref-Ward2013" role="doc-biblioref">364</a>,<a href="#ref-Xie2013" role="doc-biblioref">365</a>]</span>.
Epigenetic reactivation of cryptic cis-regulatory elements (CREs) within transposons has been show to act as an oncogenic driver <span class="citation">[<a href="#ref-Jang2019" role="doc-biblioref">366</a>]</span>, including specifically from Alu elements <span class="citation">[<a href="#ref-Rajendiran2016" role="doc-biblioref">367</a>]</span>.
De-repression of Alu elements also has the potential to influence common non-malignant diseases <span class="citation">[<a href="#ref-Chuong2017" role="doc-biblioref">228</a>,<a href="#ref-Payer2017" role="doc-biblioref">368</a>]</span>.</p>
<p>Quantitative assays of DNA methylation with high positional resolution which cover a large number of loci, combined with statistical methods for sparse regression analysis have permitted the construction of â€˜DNA methylation clocksâ€˜ which predict chronological age with a high degree of accuracy <span class="citation">[<a href="#ref-Horvath2018" role="doc-biblioref">149</a>,<a href="#ref-Bell2019" role="doc-biblioref">150</a>]</span>.
The difference between chronological and predicted age know as the â€™age accelerationâ€™ has been found to capture aspects of biological ageing in a number of these clocks.
For example, Age acceleration is a strong predictor of all-cause mortality <span class="citation">[<a href="#ref-Marioni2015" role="doc-biblioref">135</a>]</span>.
Whilst DNA hypermethylation and hypomethylation are driven by distinct mechanisms <span class="citation">[<a href="#ref-Wu2017" role="doc-biblioref">60</a>,<a href="#ref-Mozhui2017" role="doc-biblioref">369</a>]</span> CpGs undergoing these processes are approximately equally represented in DNA methylation clocks.
Interestingly however, hypomethylating CpGs were identified by Zhang et al.Â to be the most indicative of biological ageing-related all-cause mortality effects in blood <span class="citation">[<a href="#ref-Zhang2017" role="doc-biblioref">140</a>]</span>.
Most DNA methylation clocks constructed in humans have been constructed using the Illumina bead chip array based DNA methylation assays.
The coverage of these assays is strongly focused on promoters <span class="citation">[<a href="#ref-Bibikova2009" role="doc-biblioref">107</a>]</span>, CpG islands <span class="citation">[<a href="#ref-Bibikova2011" role="doc-biblioref">114</a>]</span>, and more recently enhancers <span class="citation">[<a href="#ref-Moran2015" role="doc-biblioref">154</a>]</span>.
However, array coverage of repetitive elements is lacking due in part to the technical challenges of accurate mapping in these spaces <span class="citation">[<a href="#ref-Bell2019" role="doc-biblioref">150</a>]</span>.
Thus, much of the genomic hypomethylation goes uncovered by the assays most commonly used to construct human epigenetic clocks.</p>
<p>In this study elastic net regression models were used to construct age predictors comprised of the DNA methylation state of Alu elements.
DNA methylation data at Alu elemetns was drawn from a large (&gt;3,000) immunoprecipitation-derived sequencing-based dataset (MeDIP-seq) from peripheral blood DNA.
Alu-specific DNA methylation ageing clocks were constructed and compared to other clocksâ€™ measures. The best performing Alu age predictor had a Pearson correlation coefficient of 0.65 and a median absolute error of 8.1 years.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="methods" class="section level2">
<h2><span class="header-section-number">5.3</span> Methods</h2>
<div id="dna-methylome-data" class="section level3">
<h3><span class="header-section-number">5.3.1</span> DNA methylome data</h3>
<p>The Methylated DNA Immunoprecipitation sequencing (MeDIP-seq) data was processed as previously described <span class="citation">[<a href="#ref-Bell2017a" role="doc-biblioref">67</a>,<a href="#ref-Bell2016" role="doc-biblioref">151</a>]</span> and detailed in <a href="methods.html#MeDIPseq">Methods</a> <a href="methods.html#MeDIPseq">2.2</a>.
These processed data are available from the European Genome-phenome Archive (EGA) (<a href="https://www.ebi.ac.uk/ega" class="uri">https://www.ebi.ac.uk/ega</a>) under study number EGAS00001001910 and dataset EGAD00010000983 and were generated by BGI Shenzen for TwinsUK.</p>
<p><code>MEDIPS</code> (v1.0) was used for the MeDIP-seq analysis <span class="citation">[<a href="#ref-Lienhard2014" role="doc-biblioref">80</a>]</span>.
This produced reads per million base pairs (RPM) values binned into 500bp windows with a 250bp slide in the BED format, resulting in ~12.8 million windows on the genome.</p>
</div>
<div id="alu-element-annotation-and-pre-filtering" class="section level3">
<h3><span class="header-section-number">5.3.2</span> Alu element annotation and pre-filtering</h3>
<p>Alu element annotations were taken from Repeat Masker <span class="citation">[<a href="#ref-Smit2009" role="doc-biblioref">370</a>]</span>, only Alus on the autosomes were considered.
2,758,588 500bp windows with a 250bp slide overlapped the 1,137,653 Alu elements annotated.</p>
<p>Technically problematic and genetically confounded Alu element loci were removed.
Alu elements located in <a href="https://github.com/Boyle-Lab/Blacklist/raw/master/lists/hg19-blacklist.v2.bed.gz">blacklisted</a> 834 regions of the hg19 genome, as identified by Amemiya et al.Â <span class="citation">[<a href="#ref-Amemiya2019" role="doc-biblioref">294</a>]</span> were excluded as candidate loci when constructing DNAm clocks.
Alu elements overlapping <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/integrated_sv_map/ALL.wgs.mergedSV.v8.20130502.svs.genotypes.vcf.gz">known structural variants</a> identified by the 1000 genomes project <span class="citation">[<a href="#ref-Payer2017" role="doc-biblioref">368</a>,<a href="#ref-Fairley2020" role="doc-biblioref">371</a>]</span> were excluded as candidate clock loci.
Genetic variants which impact on the rate of biological ageing could confound DNA methylation ageing signal.
Haplotype specific methylation peaks, in which a common disease associated genetic variant is predictive of DNA methylation state, were also excluded from the clock; Bell et al.Â identified 7,173 such regions <span class="citation">[<a href="#ref-Bell2017a" role="doc-biblioref">67</a>]</span>.
To exclude loci with low coverage which are likely to be uninformative and noisy a zero filter was used.
In quantile normalised data loci in which 0.1% or more of samples have an RPM value of 0 are excluded.</p>
</div>
<div id="elastic-net-regression" class="section level3">
<h3><span class="header-section-number">5.3.3</span> Elastic net regression</h3>
<p><a href="https://privefl.github.io/bigstatsr/"><code>Bigstatsr</code></a> was used to fit penalised regression models <span class="citation">[<a href="#ref-Prive2019" role="doc-biblioref">372</a>]</span>.
Elastic net regression with an <span class="math inline">\(\alpha\)</span> = 0.5, and cross-model selection and averaging (CMSA) with 10 fold cross-validation was used.
This procedure obviates the necessity of selecting a <span class="math inline">\(\lambda\)</span> regularisation hyperparameter.
CMSA generates model coefficient by averaging across coefficients generated by cross-validation <span class="citation">[<a href="#ref-Friedman2010" role="doc-biblioref">120</a>,<a href="#ref-Prive2019" role="doc-biblioref">372</a>,<a href="#ref-Tibshirani2012" role="doc-biblioref">373</a>]</span>.</p>
</div>
<div id="sample-selection" class="section level3">
<h3><span class="header-section-number">5.3.4</span> Sample Selection</h3>
<p>Two training and two replication sets were created from the twin data so as to permit the training and replication sets to be unrelated.
The training sets, which were permitted to contain singletons, contained 1548 individuals and the replication sets contained the 1308 individuals who where twins of those in the replication sets.
The 1548 individuals in the training sets were randomly split into 2 groups of 774 the replication set for each training set was comprised of the twins of the other training set such that two training sets with unrelated replication sets could be constructed (Figure <a href="Alus.html#fig:twinsplit">5.4</a>).
Giving a total of 2856 individuals in all sets.</p>
<div class="figure" style="text-align: center"><span id="fig:twinsplit"></span>
<img src="figs/twin_split.png" alt="Construction of training and replication sets Twin pairs were split one twin in each group. The first group was twins was split into two groups with a random number generator. The twins of the individuals in one random half of the group were assigned to be the replication set for the other half, producing two sets of unrelated training and replication groups. Models were also used to predict the age of related individuals to gauge the effect of genetic factors on the predictive power of the models." width="100%" />
<p class="caption">
Figure 5.4: <strong>Construction of training and replication sets</strong> Twin pairs were split one twin in each group. The first group was twins was split into two groups with a random number generator. The twins of the individuals in one random half of the group were assigned to be the replication set for the other half, producing two sets of unrelated training and replication groups. Models were also used to predict the age of related individuals to gauge the effect of genetic factors on the predictive power of the models.
</p>
</div>

<p>The age distribution of these training and replication sets are highly similar (Figure <a href="Alus.html#fig:setsAgeDist">5.5</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:setsAgeDist"></span>
<img src="figs/setsAgeDist.png" alt="Age distribution of training and replication sets Training 1 (n = 774); Training 2 (n = 774); Replication 1 (n = 664), unrelated to Training 1; Replication 2 (n = 644), unrelated to Training 2." width="90%" />
<p class="caption">
Figure 5.5: <strong>Age distribution of training and replication sets</strong> Training 1 (n = 774); Training 2 (n = 774); Replication 1 (n = 664), unrelated to Training 1; Replication 2 (n = 644), unrelated to Training 2.
</p>
</div>

<p>All sets are comprised of ~2/3 Monozygotic (MZ) twins and ~1/3 Dizygotic (DZ) twins, exact numbers are detailed in (Figure <a href="Alus.html#fig:mzdzsplitplot">5.6</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:mzdzsplitplot"></span>
<img src="figs/mzdzsplitplot.png" alt="Zygosity Make-up of sets Replication 1 (n = 664, 225 DZ, 439 MZ); Replication 2 (n = 644, 204 DZ, 440 MZ), Training 1 (n = 774, 256 DZ, 518 MZ); Training 2 (n = 774, 269 DZ, 505 MZ)." width="90%" />
<p class="caption">
Figure 5.6: <strong>Zygosity Make-up of sets</strong> Replication 1 (n = 664, 225 DZ, 439 MZ); Replication 2 (n = 644, 204 DZ, 440 MZ), Training 1 (n = 774, 256 DZ, 518 MZ); Training 2 (n = 774, 269 DZ, 505 MZ).
</p>
</div>

</div>
<div id="binarisation" class="section level3">
<h3><span class="header-section-number">5.3.5</span> Binarisation</h3>
<p>Binarisation of DNA methylation data was achieved by a simple procedure: assigning windows in a given sample a DNA methylation value of 1 if their DNA methylation value was greater than the median RPM value for this window across samples and a 0 if it was less.</p>
<p>Binarisation of gene expression data yielded increased predictive accuracy in gene expression based ageing clocks <span class="citation">[<a href="#ref-Meyer2020" role="doc-biblioref">374</a>]</span>.
The reasoning behind this approach is that the quantitation of changes in gene expression is noisy and binarising the data may reduce the noise whilst retaining the information important for age prediction.</p>
</div>
<div id="AluMethodsArrays" class="section level3">
<h3><span class="header-section-number">5.3.6</span> Comparisons to other clocks in TwinsUK Illumina 450k array data</h3>
<p>Illumina Infinium DNA methylation 450k arrays ((C) Illumina) were also performed on TwinsUK participants (<a href="methods.html#GenMethodsMethArray">Methods</a> <a href="methods.html#GenMethodsMethArray">2.1</a>).
574 Blood-derived DNA samples had matched MeDIP-seq data not used in the training or replication sets in the Alu clock analysis.
These data were available for this analysis in a pre-processed form, Methylation â€˜betaâ€™ values subject to beta-mixture quantile normalisation (BMIQ) as previously described <span class="citation">[<a href="#ref-Bell2017a" role="doc-biblioref">67</a>,<a href="#ref-Bell2016" role="doc-biblioref">151</a>]</span>.</p>
<p>DNA methylation ages were computed for the Horvath <span class="citation">[<a href="#ref-Horvath2013" role="doc-biblioref">119</a>]</span>, Hannum et al.Â <span class="citation">[<a href="#ref-Hannum2013" role="doc-biblioref">118</a>]</span> and Levine et al.Â â€˜PhenoAgeâ€™ <span class="citation">[<a href="#ref-Levine2018" role="doc-biblioref">138</a>]</span> clocks, using the coefficients, intercept values and transformations specified the original publications, and a model of the form:
<span class="math inline">\(DNAmAge = a\beta_{probe\ 1} ... b\beta_{probe\ n} + intercept\)</span>
Where <em>a</em>, <em>b</em> and <em>intercept</em> are model parameters provided by the authors of the respective DNAm age predictors.
In the absence of an intercept value 0 was used, in the case of the Hannum clock.
Age acceleration values were computed as the predicted age minus the chronological age.</p>
</div>
<div id="genetic-association-for-alu-clock" class="section level3">
<h3><span class="header-section-number">5.3.7</span> Genetic Association for Alu-Clock</h3>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="results" class="section level2">
<h2><span class="header-section-number">5.4</span> Results</h2>
<p>A variety of different initial filtering criteria were applied to the Alu elements included in the models, as well as a number of data transformations.
Pre-filtering steps were performed partially for quality control and partially to see if subsets of the Alu space with biologically interesting differences could yield clocks of equivalent accuracy but which might capture different aspects of biological ageing related to repeat de-repression.</p>
<p>Preliminary analysis were carried out using RPM values, Quantile normalised RPM values, Absolute methylation value estimates from <code>MEDIPS</code> and Binarised DNA methylation data.
Quantile normalised RPM values yielded the best and most consistent results and were used in subsequent analyses.</p>
<p>Whilst Human epigenetic age is logarithmic with time across the lifespan the effects of this are more relevant at younger ages <span class="citation">[<a href="#ref-Snir2019" role="doc-biblioref">375</a>]</span>.
We fitted Alu Clock models on the natural logarithm of age in years this did not improve the quality of predicted to actual age values, likely due in part to the older age distribution of our data.</p>
<p>An Alu element length filter of at least 180bp or approximately two-thirds the length of a full length Alu element (~280bp) was imposed.
This minimises the number of reads with the potential to misalign to other Alu elements, reads which, in this context, could lead to reduced accuracy of positionally specific Alu DNA methylation estimates <span class="citation">[<a href="#ref-Hellen2013" role="doc-biblioref">376</a>]</span>.</p>
<p>CpG density filters were employed to prioritise regions likely to have dynamic and functionally interesting differences in DNA methylation.
Filters were used for CpG densities above the genomic background of 2%, as well as for &gt;5% the approximate threshold for CpG island shores and low methylation regions <span class="citation">[<a href="#ref-Irizarry2009" role="doc-biblioref">43</a>,<a href="#ref-Stadler2011" role="doc-biblioref">50</a>]</span>.</p>
<p>To limit the selection of predictive sites to those likely to have consistent directions of change across samples a clock was constructed with only windows which show nominally significant changes in both members of an MZ twin pair in a consistent direction.
Linear models to predict age based on DNA methylation were fitted for 687 pairs of monozygotic (MZ) twins.
Batch information and blood cell counts (eosinophils, lymphocytes, monocytes, neutrophils) were included as covariates.</p>
<p>To produce a model that specifically captures age as a function of the de-repression of Alu elements clocks were also constructed with only windows which exhibited nominally significant (p &lt; 0.05) decreases in DNAm in both MZ twins.
This filtering was based on the model used in selecting windows with a consistent direction of change.</p>
<p>An age predictor was also constructed using only AluY elements.
AluY is the youngest (Figure <a href="Alus.html#fig:AluAge">5.2</a>), and most CpG dense <a href="Alus.html#fig:AlusWindowsCpGdLVplot">5.3</a> family of Alu elements.
Having a CpG density closer to that of typically de-repressed functional elements may make such elements more readily susceptible to stable de-repression.
In addition the host genome has had less time to adapt to the presence of these elements so their de-repression may be more disruptive than older elements.</p>
<p>Figure <a href="Alus.html#fig:modelSummaryTabsMain">5.7</a> summarises the details of Alu clocks trained with these various pre-filtering criteria applied to their starting sets.
The correlation with the predicted age and the chronological age of samples unrelated to the individuals on which the clock was trained indicate model quality (Figure <a href="Alus.html#fig:corplotsPanelSplit">5.9</a> a).
The Quality of the Alu clock predictions is quite poor in comparison to other DNA methylation clocks with the highest R of 0.65 and a median absolute error of 8.1 years, compared to 0.96 and 3.6 years for Horvathâ€™s multi-tissue clock <span class="citation">[<a href="#ref-Horvath2013" role="doc-biblioref">119</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:modelSummaryTabsMain"></span>
<img src="figs/model_summary_tabs_main.png" alt="Alu Clock Model Summaries A) Models fitted with Training 1 set predicting Replication 1 ages. B) Models fitted with Training 2 set predicting Replication 2 ages. Alu Type Composition Colour code: AluY, AluS, AluJ, FAM, FLAM, FRAM. R is Pearsonâ€™s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="100%" />
<p class="caption">
Figure 5.7: <strong>Alu Clock Model Summaries</strong> <em>A)</em> Models fitted with Training 1 set predicting Replication 1 ages. <em>B)</em> Models fitted with Training 2 set predicting Replication 2 ages. Alu Type Composition Colour code: <span style="     color: #e41a1c !important;">AluY</span>, <span style="     color: #377eb8 !important;">AluS</span>, <span style="     color: #4daf4a !important;">AluJ</span>, <span style="     color: #984ea3 !important;">FAM</span>, <span style="     color: #ff7f00 !important;">FLAM</span>, <span style="     color: #ffff33 !important;">FRAM</span>. R is Pearsonâ€™s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<p>Figure <a href="Alus.html#fig:modelSummaryTabsCtrl">5.8</a> shows the model summaries when correlations are computed for for the related sets comprised of the twins of individuals in the training set.
The correlations are mostly slightly higher for this prediction than for the prediction of unrelated individuals (Figure <a href="Alus.html#fig:corplotsPanelSplit">5.9</a> b).
This suggests that there are not large genetic effects on the models as their predictive efficacy is only marginally better when employed to predict the age of individuals closely related to the training set compared to unrelated individuals.</p>
<div class="figure" style="text-align: center"><span id="fig:modelSummaryTabsCtrl"></span>
<img src="figs/model_summary_tabs_ctrl.png" alt="Summaries of Alu Clock Models Predicting Twin Groups for the purpose of comparing predictive of age models in related and unrelated samples. A) Models fitted with Training 1 set predicting Replication 2 ages. B) Models fitted with Training 2 set predicting Replication 1 ages. Alu Type Composition Colour code: AluY, AluS, AluJ, FAM, FLAM, FRAM. R is Pearsonâ€™s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="100%" />
<p class="caption">
Figure 5.8: <strong>Summaries of Alu Clock Models Predicting Twin Groups</strong> for the purpose of comparing predictive of age models in related and unrelated samples. <em>A)</em> Models fitted with Training 1 set predicting Replication 2 ages. <em>B)</em> Models fitted with Training 2 set predicting Replication 1 ages. Alu Type Composition Colour code: <span style="     color: #e41a1c !important;">AluY</span>, <span style="     color: #377eb8 !important;">AluS</span>, <span style="     color: #4daf4a !important;">AluJ</span>, <span style="     color: #984ea3 !important;">FAM</span>, <span style="     color: #ff7f00 !important;">FLAM</span>, <span style="     color: #ffff33 !important;">FRAM</span>. R is Pearsonâ€™s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:corplotsPanelSplit"></span>
<img src="figs/corplots_panel_split.png" alt="Correlation of Predicted with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="100%" />
<p class="caption">
Figure 5.9: <strong>Correlation of Predicted with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

</div>
<div id="comparison-to-other-dna-methylation-clocks" class="section level2">
<h2><span class="header-section-number">5.5</span> Comparison to other DNA methylation clocks</h2>
<p>To compare the Alu clock results with those of other DNA methylation age clocks DNAm age was predicted for the: Horvath <span class="citation">[<a href="#ref-Horvath2013" role="doc-biblioref">119</a>]</span>, Hannum et al.Â <span class="citation">[<a href="#ref-Hannum2013" role="doc-biblioref">118</a>]</span> and Levine et al.Â â€˜PhenoAgeâ€™ <span class="citation">[<a href="#ref-Levine2018" role="doc-biblioref">138</a>]</span> clocks (<a href="Alus.html#AluMethodsArrays">Methods</a> <a href="Alus.html#AluMethodsArrays">5.3.6</a>) using samples for which both DNA methylation array data and MeDIP-seq data was available (n = 574).
The array based clocks all perform similarly with R values in excess of 0.8, the Alu clock only attains an R of 0.46, though with a narrower error than PhenoAge.
This however, is not entirely surprising given that PhenoAge aims to capture ageing phenotypes and not narrowly chronological age.</p>
<div class="figure" style="text-align: center"><span id="fig:ageVsPredage"></span>
<img src="figs/age_vs_pred_age.png" alt="Correlation of Alu Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set." width="90%" />
<p class="caption">
Figure 5.10: <strong>Correlation of Alu Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:aluclockageVsPredageSingle"></span>
<img src="figs/alu_clock_age_vs_pred_age_single.png" alt="Correlation of Alu Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set." width="90%" />
<p class="caption">
Figure 5.11: <strong>Correlation of Alu Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set.
</p>
</div>

<div id="alu-age-acceleration" class="section level3">
<h3><span class="header-section-number">5.5.1</span> Alu Age Acceleration</h3>
<p>The age acceleration values (Chronological - Predicted Age) for the Alu clocks are strongly correlated with chronological age (Figure <a href="Alus.html#fig:accelCorplotsPanelSplit">5.12</a>), unlike the age acceleration of the Horvath DNA methylation clock and others (Figure <a href="Alus.html#fig:ageVsAgeAccelAluVsOther">5.13</a>).
The older the in individual and more prone their age is to be overestimated and vice versa.
This means that Alu clock age acceleration values cannot be used independently of chronological age to make statements about biological age.</p>
<div class="figure" style="text-align: center"><span id="fig:accelCorplotsPanelSplit"></span>
<img src="figs/accel_corplots_panel_split.png" alt="Correlation of Alu Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="90%" />
<p class="caption">
Figure 5.12: <strong>Correlation of Alu Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ageVsAgeAccelAluVsOther"></span>
<img src="figs/age_vs_age_accel_alu_vs_other.png" alt="Correlation of Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="90%" />
<p class="caption">
Figure 5.13: <strong>Correlation of Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<p>Age acceleration in the Alu clocks is not strongly correlated with age acceleration in the Horvath, Hannun or PhenoAge clocks.</p>
<div class="figure" style="text-align: center"><span id="fig:aluClockAccelVsOthers"></span>
<img src="figs/alu_clock_accel_vs_others.png" alt="Correlation of Alu Age Accelerations with Age Acceleration from Other Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="60%" />
<p class="caption">
Figure 5.14: <strong>Correlation of Alu Age Accelerations with Age Acceleration from Other Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set. â€˜+â€™ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion" class="section level2">
<h2><span class="header-section-number">5.6</span> Discussion</h2>
<p>The criteria for being called an epigenetic clock layed out by Horvath &amp; Raj <span class="citation">[<a href="#ref-Horvath2018" role="doc-biblioref">149</a>]</span> are that an r &gt; 0.8 should be achieved by the clock.
By this metric the Alu age predictors generated here should not be considered epigenetic clocks.</p>
<p>The age acceleration of the Alu age predictors in strongly correlate with age.
This may be due the the relatively poor ability of MeDIP-seq to provide absolute quantitation of DNA methylation.
There is substantial technical noise in the value representing DNA methylation at a given locus.
The observed systematic bais towards underestimating the ages of the young and overesimating the ages of the old may be explained by a broadly consistent direction with an inconsistent magnitude.
Simply scaling the value of given locus by a fixed model coefficient when there is a large amount of variation leads to outsized of high magnitude outliers on the age estimates.</p>
<p>It was an aim of this work to investigate the genetic influences on the Alu clock based measures of age-acceleration, as has been with shown previously with an association in the TERT gene with the Horvath clock <span class="citation">[<a href="#ref-Lu2017" role="doc-biblioref">377</a>]</span>, <span class="citation">[<a href="#ref-Gibson2019" role="doc-biblioref">378</a>]</span>.
These results had the potential to reveal mechanisms involved in controlling the epigenetic state of this large portion of the genome.
However the strong relationship with Age acceleration and Chronological age in the Alu clock measures of age acceleration renders this difficult as age would be a potent confounder of age acceleration in this instance.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Lopez-Otin2013">
<p>9. LÃ³pez-OtÃ­n C, Blasco MA, Partridge L, Serrano M, Kroemer G: <strong>The hallmarks of aging.</strong> <em>Cell</em> 2013, <strong>153</strong>:1194â€“217<a href="https://doi.org/10.1016/j.cell.2013.05.039">10.1016/j.cell.2013.05.039</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23746838">http://www.ncbi.nlm.nih.gov/pubmed/23746838</a>.</p>
</div>
<div id="ref-Booth2016">
<p>10. Booth LN, Brunet A: <strong>The Aging Epigenome</strong>. <em>Molecular Cell</em> 2016, <strong>62</strong>:728â€“744<a href="https://doi.org/10.1016/j.molcel.2016.05.013">10.1016/j.molcel.2016.05.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2016.05.013">http://dx.doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div id="ref-Bird2007">
<p>16. Bird A: <strong>Perceptions of epigenetics.</strong> <em>Nature</em> 2007, <strong>447</strong>:396â€“8<a href="https://doi.org/10.1038/nature05913">10.1038/nature05913</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17522671">http://www.ncbi.nlm.nih.gov/pubmed/17522671</a>.</p>
</div>
<div id="ref-Irizarry2009">
<p>43. Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P, Cui H, Gabo K, Rongione M, Webster M, Ji H, Potash J, Sabunciyan S, Feinberg AP: <strong>The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.</strong> <em>Nature genetics</em> 2009, <strong>41</strong>:178â€“186<a href="https://doi.org/10.1038/ng.298">10.1038/ng.298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151715">http://www.ncbi.nlm.nih.gov/pubmed/19151715</a>.</p>
</div>
<div id="ref-Stadler2011">
<p>50. Stadler MB, Murr R, Burger L, Ivanek R, Lienert F, SchÃ¶ler A, Nimwegen E van, Wirbelauer C, Oakeley EJ, Gaidatzis D, Tiwari VK, SchÃ¼beler D: <strong>DNA-binding factors shape the mouse methylome at distal regulatory regions.</strong> <em>Nature</em> 2011, <strong>480</strong>:490â€“5<a href="https://doi.org/10.1038/nature10716">10.1038/nature10716</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22170606">http://www.ncbi.nlm.nih.gov/pubmed/22170606</a>.</p>
</div>
<div id="ref-Yin2017">
<p>51. Yin Y, Morgunova E, Jolma A, Kaasinen E, Sahu B, Khund-Sayeed S, Das PK, Kivioja T, Dave K, Zhong F, Nitta KR, Taipale M, Popov A, Ginno PA, Domcke S, Yan J, SchÃ¼beler D, Vinson C, Taipale J: <strong>Impact of cytosine methylation on DNA binding specificities of human transcription factors.</strong> <em>Science (New York, N.Y.)</em> 2017, <strong>356</strong>:eaaj2239<a href="https://doi.org/10.1126/science.aaj2239">10.1126/science.aaj2239</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28473536">http://www.ncbi.nlm.nih.gov/pubmed/28473536</a>.</p>
</div>
<div id="ref-Wu2017">
<p>60. Wu X, Zhang Y: <strong>TET-mediated active DNA demethylation: mechanism, function and beyond</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:517â€“534<a href="https://doi.org/10.1038/nrg.2017.33">10.1038/nrg.2017.33</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.33">http://www.nature.com/doifinder/10.1038/nrg.2017.33</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>67. Bell CG, Gao F, Yuan W, Roos L, Acton RJ, Xia Y, Bell J, Ward K, Mangino M, Hysi PG, Wang J, Spector TD: <strong>Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci</strong>. <em>Nature Communications</em> 2018, <strong>9</strong>:8<a href="https://doi.org/10.1038/s41467-017-01586-1">10.1038/s41467-017-01586-1</a>Available: <a href="http://www.nature.com/articles/s41467-017-01586-1">http://www.nature.com/articles/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Lienhard2014">
<p>80. Lienhard M, Grimm C, Morkel M, Herwig R, Chavez L: <strong>MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments</strong>. <em>Bioinformatics</em> 2014, <strong>30</strong>:284â€“286<a href="https://doi.org/10.1093/bioinformatics/btt650">10.1093/bioinformatics/btt650</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24227674">http://www.ncbi.nlm.nih.gov/pubmed/24227674</a>.</p>
</div>
<div id="ref-Wilson1983">
<p>90. Wilson VL, Jones PA: <strong>DNA methylation decreases in aging but not in immortal cells.</strong> <em>Science (New York, N.Y.)</em> 1983, <strong>220</strong>:1055â€“7Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.</p>
</div>
<div id="ref-Rakyan2010">
<p>106. Rakyan VK, Down TA, Maslau S, Andrew T, Yang TP, Beyan H, Whittaker P, McCann OT, Finer S, Valdes AM, Leslie RD, Deloukas P, Spector TD: <strong>Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains</strong>. <em>Genome Research</em> 2010, <strong>20</strong>:434â€“439<a href="https://doi.org/10.1101/gr.103101.109">10.1101/gr.103101.109</a>Available: <a href="http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109">http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109</a>.</p>
</div>
<div id="ref-Bibikova2009">
<p>107. Bibikova M, Le J, Barnes B, Saedinia-Melnyk S, Zhou L, Shen R, Gunderson KL: <strong>Genome-wide DNA methylation profiling using Infinium assay</strong>. <em>Epigenomics</em> 2009, <strong>1</strong>:177â€“200<a href="https://doi.org/10.2217/epi.09.14">10.2217/epi.09.14</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi.09.14">https://www.futuremedicine.com/doi/10.2217/epi.09.14</a>.</p>
</div>
<div id="ref-Teschendorff2010">
<p>108. Teschendorff AE, Menon U, Gentry-Maharaj A, Ramus SJ, Weisenberger DJ, Shen H, Campan M, Noushmehr H, Bell CG, Maxwell AP, Savage DA, Mueller-Holzner E, Marth C, Kocjan G, Gayther SA, Jones A, Beck S, Wagner W, Laird PW, Jacobs IJ, Widschwendter M: <strong>Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.</strong> <em>Genome research</em> 2010, <strong>20</strong>:440â€“6<a href="https://doi.org/10.1101/gr.103606.109">10.1101/gr.103606.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20219944">http://www.ncbi.nlm.nih.gov/pubmed/20219944</a>.</p>
</div>
<div id="ref-Bibikova2011">
<p>114. Bibikova M, Barnes B, Tsan C, Ho V, Klotzle B, Le JM, Delano D, Zhang L, Schroth GP, Gunderson KL, Fan J-B, Shen R: <strong>High density DNA methylation array with single CpG site resolution.</strong> <em>Genomics</em> 2011, <strong>98</strong>:288â€“95<a href="https://doi.org/10.1016/j.ygeno.2011.07.007">10.1016/j.ygeno.2011.07.007</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21839163">http://www.ncbi.nlm.nih.gov/pubmed/21839163</a>.</p>
</div>
<div id="ref-Hannum2013">
<p>118. Hannum G, Guinney J, Zhao L, Zhang L, Hughes G, Sadda S, Klotzle B, Bibikova M, Fan J-B, Gao Y, Deconde R, Chen M, Rajapakse I, Friend S, Ideker T, Zhang K: <strong>Genome-wide methylation profiles reveal quantitative views of human aging rates.</strong> <em>Molecular cell</em> 2013, <strong>49</strong>:359â€“367<a href="https://doi.org/10.1016/j.molcel.2012.10.016">10.1016/j.molcel.2012.10.016</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23177740">http://www.ncbi.nlm.nih.gov/pubmed/23177740</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>119. Horvath S: <strong>DNA methylation age of human tissues and cell types.</strong> <em>Genome biology</em> 2013, <strong>14</strong>:R115<a href="https://doi.org/10.1186/gb-2013-14-10-r115">10.1186/gb-2013-14-10-r115</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24138928">http://www.ncbi.nlm.nih.gov/pubmed/24138928</a>.</p>
</div>
<div id="ref-Friedman2010">
<p>120. Friedman J, Hastie T, Tibshirani R: <strong>Regularization Paths for Generalized Linear Models via Coordinate Descent</strong>. <em>Journal of Statistical Software</em> 2010, <strong>33</strong>:7â€“10<a href="https://doi.org/10.18637/jss.v033.i01">10.18637/jss.v033.i01</a>Available: <a href="http://www.jstatsoft.org/v33/i01/">http://www.jstatsoft.org/v33/i01/</a>.</p>
</div>
<div id="ref-Marioni2015">
<p>135. Marioni RE, Shah S, McRae AF, Chen BH, Colicino E, Harris SE, Gibson J, Henders AK, Redmond P, Cox SR, Pattie A, Corley J, Murphy L, Martin NG, Montgomery GW, Feinberg AP, Fallin MD, Multhaup ML, Jaffe AE, Joehanes R, Schwartz J, Just AC, Lunetta KL, Murabito JM, Starr JM, Horvath S, Baccarelli AA, Levy D, Visscher PM, Wray NR, et al.: <strong>DNA methylation age of blood predicts all-cause mortality in later life.</strong> <em>Genome biology</em> 2015, <strong>16</strong>:25<a href="https://doi.org/10.1186/s13059-015-0584-6">10.1186/s13059-015-0584-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25633388">http://www.ncbi.nlm.nih.gov/pubmed/25633388</a>.</p>
</div>
<div id="ref-Levine2018">
<p>138. Levine ME, Lu AT, Quach A, Chen BH, Assimes TL, Bandinelli S, Hou L, Baccarelli AA, Stewart JD, Li Y, Whitsel EA, Wilson JG, Reiner AP, Aviv A, Lohman K, Liu Y, Ferrucci L, Horvath S: <strong>An epigenetic biomarker of aging for lifespan and healthspan</strong>. <em>Aging</em> 2018, <strong>10</strong>:573â€“591<a href="https://doi.org/10.18632/aging.101414">10.18632/aging.101414</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29676998">http://www.ncbi.nlm.nih.gov/pubmed/29676998</a>.</p>
</div>
<div id="ref-Zhang2017">
<p>140. Zhang Y, Hapala J, Brenner H, Wagner W: <strong>Individual CpG sites that are associated with age and life expectancy become hypomethylated upon aging</strong>. <em>Clinical Epigenetics</em> 2017, <strong>9</strong>:9<a href="https://doi.org/10.1186/s13148-017-0315-9">10.1186/s13148-017-0315-9</a>Available: <a href="http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9">http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9</a>.</p>
</div>
<div id="ref-Horvath2018">
<p>149. Horvath S, Raj K: <strong>DNA methylation-based biomarkers and the epigenetic clock theory of ageing</strong>. <em>Nature Reviews Genetics</em> 2018, <strong>19</strong>:371â€“384<a href="https://doi.org/10.1038/s41576-018-0004-3">10.1038/s41576-018-0004-3</a>Available: <a href="http://www.nature.com/articles/s41576-018-0004-3">http://www.nature.com/articles/s41576-018-0004-3</a>.</p>
</div>
<div id="ref-Bell2019">
<p>150. Bell CG, Lowe R, Adams PD, Baccarelli AA, Beck S, Bell JT, Christensen BC, Gladyshev VN, Heijmans BT, Horvath S, Ideker T, Issa J-PJ, Kelsey KT, Marioni RE, Reik W, Relton CL, Schalkwyk LC, Teschendorff AE, Wagner W, Zhang K, Rakyan VK: <strong>DNA methylation aging clocks: challenges and recommendations.</strong> <em>Genome biology</em> 2019, <strong>20</strong>:249<a href="https://doi.org/10.1186/s13059-019-1824-y">10.1186/s13059-019-1824-y</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31767039">http://www.ncbi.nlm.nih.gov/pubmed/31767039</a>.</p>
</div>
<div id="ref-Bell2016">
<p>151. Bell CG, Xia Y, Yuan W, Gao F, Ward K, Roos L, Mangino M, Hysi PG, Bell J, Wang J, Spector TD: <strong>Novel regional age-associated DNA methylation changes within human common disease-associated loci</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:193<a href="https://doi.org/10.1186/s13059-016-1051-8">10.1186/s13059-016-1051-8</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27663977">http://www.ncbi.nlm.nih.gov/pubmed/27663977</a>.</p>
</div>
<div id="ref-Moran2015">
<p>154. Moran S, Arribas C, Esteller M: <strong>Validation of a DNA methylation microarray for 850,000 CpG sites of the human genome enriched in enhancer sequences.</strong> <em>Epigenomics</em> 2016, <strong>8</strong>:389â€“99<a href="https://doi.org/10.2217/epi.15.114">10.2217/epi.15.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26673039">http://www.ncbi.nlm.nih.gov/pubmed/26673039</a>.</p>
</div>
<div id="ref-Acton2020">
<p>225. Acton RJ, Bourne E, Bell J, Lillycrop K, Wang J, Dennison E, Harvey N, Spector TD, Cooper C, Bell CG: <strong>The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</strong>. <em>bioRxiv</em> 2019,:1â€“48<a href="https://doi.org/10.1101/870352">10.1101/870352</a>Available: <a href="https://www.biorxiv.org/content/10.1101/870352v1">https://www.biorxiv.org/content/10.1101/870352v1</a>.</p>
</div>
<div id="ref-Partridge2018">
<p>226. Partridge L, Deelen J, Slagboom PE: <strong>Facing up to the global challenges of ageing</strong>. <em>Nature</em> 2018, <strong>561</strong>:45â€“56<a href="https://doi.org/10.1038/s41586-018-0457-8">10.1038/s41586-018-0457-8</a>Available: <a href="http://www.nature.com/articles/s41586-018-0457-8">http://www.nature.com/articles/s41586-018-0457-8</a>.</p>
</div>
<div id="ref-Chuong2017">
<p>228. Chuong EB, Elde NC, Feschotte C: <strong>Regulatory activities of transposable elements: from conflicts to benefits</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:71â€“86<a href="https://doi.org/10.1038/nrg.2016.139">10.1038/nrg.2016.139</a>Available: <a href="http://www.nature.com/articles/nrg.2016.139">http://www.nature.com/articles/nrg.2016.139</a>.</p>
</div>
<div id="ref-Amemiya2019">
<p>294. Amemiya HM, Kundaje A, Boyle AP: <strong>The ENCODE Blacklist: Identification of Problematic Regions of the Genome</strong>. <em>Scientific Reports</em> 2019, <strong>9</strong>:9354<a href="https://doi.org/10.1038/s41598-019-45839-z">10.1038/s41598-019-45839-z</a>Available: <a href="http://www.nature.com/articles/s41598-019-45839-z">http://www.nature.com/articles/s41598-019-45839-z</a>.</p>
</div>
<div id="ref-Deniz2019">
<p>341. Deniz Ã–, Frost JM, Branco MR: <strong>Regulation of transposable elements by DNA modifications</strong>. <em>Nature Reviews Genetics</em> 2019, <strong>20</strong>:417â€“431<a href="https://doi.org/10.1038/s41576-019-0106-6">10.1038/s41576-019-0106-6</a>.</p>
</div>
<div id="ref-Kazazian2017">
<p>342. Kazazian HH, Moran JV: <strong>Mobile DNA in health and disease</strong>. <em>New England Journal of Medicine</em> 2017, <strong>377</strong>:361â€“370<a href="https://doi.org/10.1056/NEJMra1510092">10.1056/NEJMra1510092</a>.</p>
</div>
<div id="ref-Gregory2005">
<p>343. Gregory TR: <strong>Synergy between sequence and size in large-scale genomics</strong>. <em>Nature Reviews Genetics</em> 2005, <strong>6</strong>:699â€“708<a href="https://doi.org/10.1038/nrg1674">10.1038/nrg1674</a>.</p>
</div>
<div id="ref-Deaton2011">
<p>344. Deaton AM, Bird A: <strong>CpG islands and the regulation of transcription</strong>. <em>Genes &amp; development</em> 2011, <strong>25</strong>:1010â€“22<a href="https://doi.org/10.1101/gad.2037511">10.1101/gad.2037511</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21576262">http://www.ncbi.nlm.nih.gov/pubmed/21576262</a>.</p>
</div>
<div id="ref-Hellman2007">
<p>345. Hellman A, Chess A: <strong>Gene body-specific methylation on the active X chromosome</strong>. <em>Science</em> 2007, <strong>315</strong>:1141â€“1143<a href="https://doi.org/10.1126/science.1136352">10.1126/science.1136352</a>.</p>
</div>
<div id="ref-Shukla2011">
<p>346. Shukla S, Kavak E, Gregory M, Imashimizu M, Shutinoski B, Kashlev M, Oberdoerffer P, Sandberg R, Oberdoerffer S: <strong>CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing</strong>. <em>Nature</em> 2011, <strong>479</strong>:74â€“79<a href="https://doi.org/10.1038/nature10442">10.1038/nature10442</a>Available: <a href="http://dx.doi.org/10.1038/nature10442">http://dx.doi.org/10.1038/nature10442</a>.</p>
</div>
<div id="ref-Blattler2014">
<p>347. Blattler A, Yao L, Witt H, Guo Y, Nicolet CM, Berman BP, Farnham PJ: <strong>Global loss of DNA methylation uncovers intronic enhancers in genes showing expression changes</strong>. <em>Genome Biology</em> 2014, <strong>15</strong>:469<a href="https://doi.org/10.1186/s13059-014-0469-0">10.1186/s13059-014-0469-0</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0</a>.</p>
</div>
<div id="ref-Gama-Sosa1983">
<p>348. Gama-Sosa MA, Slagel VA, Trewyn RW, Oxenhandler R, Kuo KC, Gehrke CW, Ehrlich M: <strong>The 5-methylcytosine content of DNA from human tumors</strong>. <em>Nucleic Acids Research</em> 1983, <strong>11</strong>:6883â€“6894<a href="https://doi.org/10.1093/nar/11.19.6883">10.1093/nar/11.19.6883</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/11.19.6883">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/11.19.6883</a>.</p>
</div>
<div id="ref-Bollati2010">
<p>349. Bollati V, Schwartz J, Wright R, Litonjua A, Tarantini L, Suh H, Sparrow D, Vokonas P, Baccarelli A: <strong>Decline in genomic DNA methylation through aging in a cohort of elderly subjects</strong>. <em>Mechanisms of Ageing and Development</em> 2009, <strong>130</strong>:234â€“239<a href="https://doi.org/10.1016/j.mad.2008.12.003">10.1016/j.mad.2008.12.003</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820">https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820</a>.</p>
</div>
<div id="ref-Dewannieux2003">
<p>350. Dewannieux M, Esnault C, Heidmann T: <strong>LINE-mediated retrotransposition of marked Alu sequences</strong>. <em>Nature Genetics</em> 2003, <strong>35</strong>:41â€“48<a href="https://doi.org/10.1038/ng1223">10.1038/ng1223</a>Available: <a href="http://www.nature.com/articles/ng1223">http://www.nature.com/articles/ng1223</a>.</p>
</div>
<div id="ref-Deininger2011">
<p>351. Deininger P: <strong>Alu elements: know the SINEs</strong>. <em>Genome Biology</em> 2011, <strong>12</strong>:236<a href="https://doi.org/10.1186/gb-2011-12-12-236">10.1186/gb-2011-12-12-236</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236">http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236</a>.</p>
</div>
<div id="ref-Ullu1984">
<p>352. Ullu E, Tschudi C: <strong>Alu sequences are processed 7SL RNA genes</strong>. <em>Nature</em> 1984, <strong>312</strong>:171â€“172<a href="https://doi.org/10.1038/312171a0">10.1038/312171a0</a>Available: <a href="http://www.nature.com/articles/312171a0">http://www.nature.com/articles/312171a0</a>.</p>
</div>
<div id="ref-Quentin1992">
<p>353. Quentin Y: <strong>Origin of the Alu family:</strong> <em>Nucleic Acids Research</em> 1992, <strong>20</strong>:3397â€“3401.</p>
</div>
<div id="ref-Chen2008">
<p>354. Chen L-L, Carmichael GG: <strong>Gene regulation by SINES and inosines: biological consequences of A-to-I editing of Alu element inverted repeats</strong>. <em>Cell Cycle</em> 2008, <strong>7</strong>:3294â€“3301<a href="https://doi.org/10.4161/cc.7.21.6927">10.4161/cc.7.21.6927</a>Available: <a href="http://www.tandfonline.com/doi/abs/10.4161/cc.7.21.6927">http://www.tandfonline.com/doi/abs/10.4161/cc.7.21.6927</a>.</p>
</div>
<div id="ref-Batzer2002">
<p>355. Batzer MA, Deininger PL: <strong>Alu repeats and human genomic diversity</strong>. <em>Nature Reviews Genetics</em> 2002, <strong>3</strong>:370â€“379<a href="https://doi.org/10.1038/nrg798">10.1038/nrg798</a>Available: <a href="http://www.nature.com/articles/nrg798">http://www.nature.com/articles/nrg798</a>.</p>
</div>
<div id="ref-Ade2013">
<p>356. Ade C, Roy-Engel AM, Deininger PL: <strong>Alu elements: an intrinsic source of human genome instability</strong>. <em>Current Opinion in Virology</em> 2013, <strong>3</strong>:639â€“645<a href="https://doi.org/10.1016/j.coviro.2013.09.002">10.1016/j.coviro.2013.09.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533">https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533</a>.</p>
</div>
<div id="ref-Cordaux2009">
<p>357. Cordaux R, Batzer MA: <strong>The impact of retrotransposons on human genome evolution</strong>. <em>Nature Reviews Genetics</em> 2009, <strong>10</strong>:691â€“703<a href="https://doi.org/10.1038/nrg2640">10.1038/nrg2640</a>Available: <a href="http://www.nature.com/articles/nrg2640">http://www.nature.com/articles/nrg2640</a>.</p>
</div>
<div id="ref-Belancio2008">
<p>358. Belancio VP, Hedges DJ, Deininger P: <strong>Mammalian non-LTR retrotransposons: For better or worse, in sickness and in health</strong>. <em>Genome Research</em> 2008, <strong>18</strong>:343â€“358<a href="https://doi.org/10.1101/gr.5558208">10.1101/gr.5558208</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.5558208">http://www.genome.org/cgi/doi/10.1101/gr.5558208</a>.</p>
</div>
<div id="ref-Jorda2017">
<p>359. JordÃ  M, DÃ­ez-Villanueva A, Mallona I, MartÃ­n B, Lois S, Barrera V, Esteller M, Vavouri T, Peinado MA: <strong>The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells</strong>. <em>Genome Research</em> 2017, <strong>27</strong>:118â€“132<a href="https://doi.org/10.1101/gr.207522.116">10.1101/gr.207522.116</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27999094">http://www.ncbi.nlm.nih.gov/pubmed/27999094</a>.</p>
</div>
<div id="ref-Chen2017b">
<p>360. Chen L-L, Yang L: <strong>ALU ternative Regulation for Gene Expression</strong>. <em>Trends in Cell Biology</em> 2017, <strong>27</strong>:480â€“490<a href="https://doi.org/10.1016/j.tcb.2017.01.002">10.1016/j.tcb.2017.01.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028">https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028</a>.</p>
</div>
<div id="ref-RobertoFerrari2019">
<p>361. Ferrari R, de Llobet Cucalon LI, Di Vona C, Le Dilly F, Vidal E, Lioutas A, Oliete JQ, Jochem L, Cutts E, Dieci G, Vannini A, Teichmann M, Luna S de la, Beato M: <strong>TFIIIC Binding to Alu Elements Controls Gene Expression via Chromatin Looping and Histone Acetylation</strong>. <em>Molecular Cell</em> 2019,:1â€“13<a href="https://doi.org/10.1016/j.molcel.2019.10.020">10.1016/j.molcel.2019.10.020</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981">https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981</a>.</p>
</div>
<div id="ref-Polak2006">
<p>362. Polak P, Domany E: <strong>Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes.</strong> <em>BMC genomics</em> 2006, <strong>7</strong>:133<a href="https://doi.org/10.1186/1471-2164-7-133">10.1186/1471-2164-7-133</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16740159">http://www.ncbi.nlm.nih.gov/pubmed/16740159</a>.</p>
</div>
<div id="ref-Su2014">
<p>363. Su M, Han D, Boyd-Kirkup J, Yu X, Han J-DJ: <strong>Evolution of Alu Elements toward Enhancers</strong>. <em>Cell Reports</em> 2014, <strong>7</strong>:376â€“385<a href="https://doi.org/10.1016/j.celrep.2014.03.011">10.1016/j.celrep.2014.03.011</a>Available: <a href="http://dx.doi.org/10.1016/j.celrep.2014.03.011">http://dx.doi.org/10.1016/j.celrep.2014.03.011</a>.</p>
</div>
<div id="ref-Ward2013">
<p>364. Ward MC, Wilson MD, Barbosa-Morais NL, Schmidt D, Stark R, Pan Q, Schwalie PC, Menon S, Lukk M, Watt S, Thybert D, Kutter C, Kirschner K, Flicek P, Blencowe BJ, Odom DT: <strong>Latent Regulatory Potential of Human-Specific Repetitive Elements</strong>. <em>Molecular Cell</em> 2013, <strong>49</strong>:262â€“272<a href="https://doi.org/10.1016/j.molcel.2012.11.013">10.1016/j.molcel.2012.11.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2012.11.013">http://dx.doi.org/10.1016/j.molcel.2012.11.013</a>.</p>
</div>
<div id="ref-Xie2013">
<p>365. Xie M, Hong C, Zhang B, Lowdon RF, Xing X, Li D, Zhou X, Lee HJ, Maire CL, Ligon KL, Gascard P, Sigaroudinia M, Tlsty TD, Kadlecek T, Weiss A, Oâ€™Geen H, Farnham PJ, Madden PAF, Mungall AJ, Tam A, Kamoh B, Cho S, Moore R, Hirst M, Marra MA, Costello JF, Wang T: <strong>DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape</strong>. <em>Nature Genetics</em> 2013, <strong>45</strong>:836â€“841<a href="https://doi.org/10.1038/ng.2649">10.1038/ng.2649</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23708189">http://www.ncbi.nlm.nih.gov/pubmed/23708189</a>.</p>
</div>
<div id="ref-Jang2019">
<p>366. Jang HS, Shah NM, Du AY, Dailey ZZ, Pehrsson EC, Godoy PM, Zhang D, Li D, Xing X, Kim S, Oâ€™Donnell D, Gordon JI, Wang T: <strong>Transposable elements drive widespread expression of oncogenes in human cancers</strong>. <em>Nature Genetics</em> 2019, <strong>51</strong>:611â€“617<a href="https://doi.org/10.1038/s41588-019-0373-3">10.1038/s41588-019-0373-3</a>Available: <a href="http://dx.doi.org/10.1038/s41588-019-0373-3">http://dx.doi.org/10.1038/s41588-019-0373-3</a>.</p>
</div>
<div id="ref-Rajendiran2016">
<p>367. Rajendiran S, Gibbs LD, Van Treuren T, Klinkebiel DL, Vishwanatha JK: <strong>MIEN1 is tightly regulated by SINE Alu methylation in its promoter</strong>. <em>Oncotarget</em> 2016, <strong>7</strong>:65307â€“65319<a href="https://doi.org/10.18632/oncotarget.11675">10.18632/oncotarget.11675</a>Available: <a href="http://www.oncotarget.com/fulltext/11675">http://www.oncotarget.com/fulltext/11675</a>.</p>
</div>
<div id="ref-Payer2017">
<p>368. Payer LM, Steranka JP, Yang WR, Kryatova M, Medabalimi S, Ardeljan D, Liu C, Boeke JD, Avramopoulos D, Burns KH: <strong>Structural variants caused by Alu insertions are associated with risks for many human diseases</strong>. <em>Proceedings of the National Academy of Sciences</em> 2017, <strong>114</strong>:E3984â€“E3992<a href="https://doi.org/10.1073/pnas.1704117114">10.1073/pnas.1704117114</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114">http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114</a>.</p>
</div>
<div id="ref-Mozhui2017">
<p>369. Mozhui K, Pandey AK: <strong>Conserved effect of aging on DNA methylation and association with EZH2 polycomb protein in mice and humans</strong>. <em>Mechanisms of Ageing and Development</em> 2017, <strong>162</strong>:27â€“37<a href="https://doi.org/10.1016/j.mad.2017.02.006">10.1016/j.mad.2017.02.006</a>Available: <a href="http://dx.doi.org/10.1016/j.mad.2017.02.006">http://dx.doi.org/10.1016/j.mad.2017.02.006</a>.</p>
</div>
<div id="ref-Smit2009">
<p>370. Smit A, Hubley R, Green: <strong>Repeat Masker</strong>. 2009, Available: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>.</p>
</div>
<div id="ref-Fairley2020">
<p>371. Fairley S, Lowy-Gallego E, Perry E, Flicek P: <strong>The International Genome Sample Resource (IGSR) collection of open human genomic variation resources</strong>. <em>Nucleic Acids Research</em> 2020, <strong>48</strong>:D941â€“D947<a href="https://doi.org/10.1093/nar/gkz836">10.1093/nar/gkz836</a>Available: <a href="https://academic.oup.com/nar/article/48/D1/D941/5580898">https://academic.oup.com/nar/article/48/D1/D941/5580898</a>.</p>
</div>
<div id="ref-Prive2019">
<p>372. PrivÃ© F, Aschard H, Blum MGB: <strong>Efficient Implementation of Penalized Regression for Genetic Risk Prediction</strong>. <em>Genetics</em> 2019, <strong>212</strong>:65â€“74<a href="https://doi.org/10.1534/genetics.119.302019">10.1534/genetics.119.302019</a>Available: <a href="http://www.genetics.org/lookup/doi/10.1534/genetics.119.302019%20https://figshare.com/articles/code/7178750%20https://gsajournals.figshare.com/articles/Supplemental%7B\_%7DMaterial%7B\_%7Dfor%7B\_%7DPriv%7B\_%7DAschard%7B\_%7Dand%7B\_%7DBlum%7B\_%7D2019/7851470">http://www.genetics.org/lookup/doi/10.1534/genetics.119.302019 https://figshare.com/articles/code/7178750 https://gsajournals.figshare.com/articles/Supplemental{\_}Material{\_}for{\_}Priv{\_}Aschard{\_}and{\_}Blum{\_}2019/7851470</a>.</p>
</div>
<div id="ref-Tibshirani2012">
<p>373. Tibshirani R, Bien J, Friedman J, Hastie T, Simon N, Taylor J, Tibshirani RJ: <strong>Strong rules for discarding predictors in lasso-type problems</strong>. <em>Journal of the Royal Statistical Society. Series B: Statistical Methodology</em> 2012, <strong>74</strong>:245â€“266<a href="https://doi.org/10.1111/j.1467-9868.2011.01004.x">10.1111/j.1467-9868.2011.01004.x</a>Available: <a href="http://arxiv.org/abs/1011.2234">http://arxiv.org/abs/1011.2234</a>.</p>
</div>
<div id="ref-Meyer2020">
<p>374. Meyer DH: <strong>A transcriptome based aging clock near the theoretical limit of accuracy</strong>. 2020, <a href="https://doi.org/10.1101/2020.05.29.123430">10.1101/2020.05.29.123430</a>Available: <a href="https://doi.org/10.1101/2020.05.29.123430">https://doi.org/10.1101/2020.05.29.123430</a>.</p>
</div>
<div id="ref-Snir2019">
<p>375. Snir S, Farrell C, Pellegrini M: <strong>Human epigenetic ageing is logarithmic with time across the entire lifespan</strong>. <em>Epigenetics</em> 2019, <strong>0</strong>:1â€“15<a href="https://doi.org/10.1080/15592294.2019.1623634">10.1080/15592294.2019.1623634</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634">https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634</a>.</p>
</div>
<div id="ref-Hellen2013">
<p>376. Hellen EH, Brookfield JF: <strong>Alu elements in primates are preferentially lost from areas of high GC content</strong>. <em>PeerJ</em> 2013, <strong>1</strong>:e78<a href="https://doi.org/10.7717/peerj.78">10.7717/peerj.78</a>Available: <a href="https://peerj.com/articles/78">https://peerj.com/articles/78</a>.</p>
</div>
<div id="ref-Lu2017">
<p>377. Lu AT, Hannon E, Levine ME, Crimmins EM, Lunnon K, Mill J, Geschwind DH, Horvath S: <strong>Genetic architecture of epigenetic and neuronal ageing rates in human brain regions</strong>. <em>Nature Communications</em> 2017, <strong>8</strong>:15353<a href="https://doi.org/10.1038/ncomms15353">10.1038/ncomms15353</a>Available: <a href="http://dx.doi.org/10.1038/ncomms15353">http://dx.doi.org/10.1038/ncomms15353</a>.</p>
</div>
<div id="ref-Gibson2019">
<p>378. Gibson J, Russ TC, Clarke T-K, Howard DM, Hillary RF, Evans KL, Walker RM, Bermingham ML, Morris SW, Campbell A, Hayward C, Murray AD, Porteous DJ, Horvath S, Lu AT, McIntosh AM, Whalley HC, Marioni RE: <strong>A meta-analysis of genome-wide association studies of epigenetic age acceleration</strong>. <em>PLOS Genetics</em> 2019, <strong>15</strong>:e1008104<a href="https://doi.org/10.1371/journal.pgen.1008104">10.1371/journal.pgen.1008104</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31738745">http://www.ncbi.nlm.nih.gov/pubmed/31738745</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="tRNAs.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/05-Alu.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
