<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.19 and GitBook 2.6.7" />

  <meta property="og:title" content="3 Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-06-27" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="methods.html"/>
<link rel="next" href="tRNAs.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#illumina-dna-methylation-arrays"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a><ul>
<li class="chapter" data-level="3.1" data-path="arrays.html"><a href="arrays.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="arrays.html"><a href="arrays.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.2.1</b> Outline of EWAS</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="arrays.html"><a href="arrays.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.3.2</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.3</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.4" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.4</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.4.1</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.4.2</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.4.3</b> Southampton Women’s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#Methods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#participants-1"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#discussion-1"><i class="fa fa-check"></i><b>4.6</b> Discussion</a></li>
<li class="chapter" data-level="4.7" data-path="tRNAs.html"><a href="tRNAs.html#data-availability"><i class="fa fa-check"></i><b>4.7</b> Data availability</a></li>
<li class="chapter" data-level="4.8" data-path="tRNAs.html"><a href="tRNAs.html#code-availablity"><i class="fa fa-check"></i><b>4.8</b> Code availablity</a></li>
<li class="chapter" data-level="4.9" data-path="tRNAs.html"><a href="tRNAs.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.9</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.10" data-path="tRNAs.html"><a href="tRNAs.html#author-contributions"><i class="fa fa-check"></i><b>4.10</b> Author Contributions</a></li>
<li class="chapter" data-level="" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="4.10.1" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-figures"><i class="fa fa-check"></i><b>4.10.1</b> Supplementary Figures</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="Alus.html"><a href="Alus.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="Alus.html"><a href="Alus.html#methods-2"><i class="fa fa-check"></i><b>5.3</b> Methods</a></li>
<li class="chapter" data-level="5.4" data-path="Alus.html"><a href="Alus.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="Alus.html"><a href="Alus.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>5.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.4.2" data-path="Alus.html"><a href="Alus.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="5.4.3" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>5.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="5.4.4" data-path="Alus.html"><a href="Alus.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>5.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="Alus.html"><a href="Alus.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="Alus.html"><a href="Alus.html#methods-3"><i class="fa fa-check"></i><b>5.6</b> Methods</a><ul>
<li class="chapter" data-level="5.6.1" data-path="Alus.html"><a href="Alus.html#participants-2"><i class="fa fa-check"></i><b>5.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="Alus.html"><a href="Alus.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>5.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="5.7.1" data-path="Alus.html"><a href="Alus.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>5.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="5.7.2" data-path="Alus.html"><a href="Alus.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>5.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.7.3" data-path="Alus.html"><a href="Alus.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.7.4" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.7.5" data-path="Alus.html"><a href="Alus.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="arrays" class="section level1">
<h1><span class="header-section-number">3</span> Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</h1>
<div id="abstract-1" class="section level2">
<h2><span class="header-section-number">3.1</span> Abstract</h2>
<p>Long term bone health and fracture risk is strongly influenced by the peak bone mass that is attained in early adulthood, this has its origins in development from late pregnancy through early childhood.
The availability of vitamin D to facilitate the uptake of calcium and phosphate is essential to healthy bone development, in addition to the role of vitamin D as a hormone mediating this process.
Greater mechanistic understanding of this process is desirable to permit the design of more effective interventions to reduce the substantial health burden of osteoporotic fractures in the elderly.
Maternal vitamin D levels during pregnancy have previously been shown to impact on the methylation status of specific genes associated with vitamin D signalling and metabolism, such as <em>RXRA</em>.
Long-term bone health outcomes have also been associated with both maternal vitamin D during pregnancy and methylation of the <em>CDKN2A</em> gene.
To attempt to identify other loci, the epigenetic regulation of which is pertinent to the impact of maternal vitamin D levels on long-term bone health, Epigenome-Wide Association Studies (EWAS) were performed with DNA methylation data from the Illumina EPIC &amp; 450k methylation bead chip arrays.
We tentatively identify two CpGs whose DNA methylation state is associated with bone mineral content at 6 years of age and periosteal circumference at 6 years of age respectively.</p>
</div>
<div id="introduction-1" class="section level2">
<h2><span class="header-section-number">3.2</span> Introduction</h2>
<p>In England and Wales a survey of general practice data indicates that 53% of women and 21% of men will experience a bone fracture in their remaining lifetime from the age of 50 years <span class="citation">[<a href="#ref-VanStaa2001" role="doc-biblioref">178</a>]</span> (Figure <a href="arrays.html#fig:VanStaa2001">3.1</a>).
Hip fractures in particular are associated with a reduction in survival of an estimated 10-20% with most deaths occurring in the first 6 months, vertebral fractures though approximately an order of magnitude less common carry a similar mortality risk <span class="citation">[<a href="#ref-VanStaa2001" role="doc-biblioref">178</a>,<a href="#ref-Streubel2011" role="doc-biblioref">179</a>]</span>.
The national osteoporosis foundation estimated the societal cost of osteoporosis at 58 Billion USD for 2018 in the USA alone <span class="citation">[<a href="#ref-Thompson2018" role="doc-biblioref">180</a>]</span> with serious negative implications for the quality of life of those with this condition.
Thus interventions to increase bone health and reduce the rate of osteoporosis as well as developing a mechanistic understanding of the development of this condition are of considerable medical interest.</p>
<div class="figure" style="text-align: center"><span id="fig:VanStaa2001"></span>
<img src="figs/adapted_from_VanStaa2001_Fig1-2.png" alt="Overall Fracture Incidence Increases With Age Data from 5 million adults in the General Practice Research Database 1988-1998. A Incidence of all fractures at any site by sex and age. B Fractures by sex and age at select sites, pelvis, Femur/Hip &amp; Vertebra confer the greatest increased mortality rates and Radius/Ulna has the greatest sexual dimorphism. (Adapted from van Staa et al. [178].)" width="80%" />
<p class="caption">
Figure 3.1: <strong>Overall Fracture Incidence Increases With Age</strong> Data from 5 million adults in the General Practice Research Database 1988-1998. <strong>A</strong> Incidence of all fractures at any site by sex and age. <strong>B</strong> Fractures by sex and age at select sites, pelvis, Femur/Hip &amp; Vertebra confer the greatest increased mortality rates and Radius/Ulna has the greatest sexual dimorphism. (Adapted from van Staa et al. <span class="citation">[<a href="#ref-VanStaa2001" role="doc-biblioref">178</a>]</span>.)
</p>
</div>

<p>Modelling work indicates that the largest determinant of osteoporosis risk is peak bone mineral density (BMD) with a 10% change in peak BMD leading to a predicted 13 year delay in the development of osteoporosis <span class="citation">[<a href="#ref-Hernandez2003" role="doc-biblioref">181</a>]</span>.
Though the baseline BMD is a better predictor of fracture risk, the rate of loss also independently predicts risk <span class="citation">[<a href="#ref-Nguyen2005" role="doc-biblioref">182</a>–<a href="#ref-Hui1990" role="doc-biblioref">184</a>]</span>.
Thus factors altering the peak bone mass will be important for modulating osteoporosis risk.
Bone mass increases from intrauterine development through to a peak in early adulthood (Figure <a href="arrays.html#fig:Harvey2014bfig1">3.2</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:Harvey2014bfig1"></span>
<img src="figs/Harvey2014bfig1.png" alt="Bone Mass Over The Lifecourse Bone mass increases from intrauterine development through to a peak in early adulthood. Intervention in early life to modulate growth trajectory and increase peak mass may provide a higher starting point and delay bone loss in later life. (Reproduced from [185].)" width="80%" />
<p class="caption">
Figure 3.2: <strong>Bone Mass Over The Lifecourse</strong> Bone mass increases from intrauterine development through to a peak in early adulthood. Intervention in early life to modulate growth trajectory and increase peak mass may provide a higher starting point and delay bone loss in later life. (Reproduced from <span class="citation">[<a href="#ref-Harvey2014b" role="doc-biblioref">185</a>]</span>.)
</p>
</div>

<p>Calcium, phosphate and vitamin D are needed for bone development <span class="citation">[<a href="#ref-Bikle2012" role="doc-biblioref">186</a>]</span> and maintenance.
Vitamin D is needed for sufficient intestinal absorption of calcium <span class="citation">[<a href="#ref-Christakos2011" role="doc-biblioref">187</a>]</span> and phosphate <span class="citation">[<a href="#ref-Fukumoto2014" role="doc-biblioref">188</a>]</span> in addition to the hormonal role it plays in the regulation of bone development.
Calcium and vitamin D intake are modifiable factors and thus constitute a reasonable starting point for interventions to improve bone health.
This raises the question of how and when they can best be deployed to maximise their impact on bone health.</p>
<p>Birth weight predicts bone mass in adulthood <span class="citation">[<a href="#ref-Baird2011" role="doc-biblioref">189</a>]</span>.
A longitudinal study with long term follow up at Helsinki University Central Hospital linked low childhood growth rate with greater fracture risk in later life <span class="citation">[<a href="#ref-Cooper2001" role="doc-biblioref">190</a>]</span>.
Data from the Southampton Women’s Survey (SWS)<span class="citation">[<a href="#ref-Inskip2006" role="doc-biblioref">191</a>]</span> suggest that intervention in the interuterine period and infancy may afford the greatest opportunity to impact on long term bone health.
Rate of growth in late pregnancy (19-34 weeks) and early post natal growth (up to ~2 years) predict bone mass at 4 years of age <span class="citation">[<a href="#ref-Harvey2010a" role="doc-biblioref">192</a>]</span>, and estimates of hip strength at 6 years <span class="citation">[<a href="#ref-Harvey2013" role="doc-biblioref">193</a>]</span>.
The proportion of individuals crossing between tertiles of the distribution of growth in length decreases with age, indicating that the variability and thus potential malleability of growth is greatest during the period from 11 weeks gestation to ~2 years.</p>
<p>Observational data indicated a relationship between maternal vitamin D status and bone outcomes in children
<span class="citation">[<a href="#ref-Mahon2010a" role="doc-biblioref">194</a>,<a href="#ref-Viljakainen2010" role="doc-biblioref">195</a>]</span>.
Prospective studies showed the effects of maternal vitamin D status on bone growth in newborns <span class="citation">[<a href="#ref-Viljakainen2011" role="doc-biblioref">196</a>]</span> and at 20 years of age <span class="citation">[<a href="#ref-Zhu2014a" role="doc-biblioref">197</a>]</span>.
MAVIDOS (Maternal Vitamin D Osteoporosis Study) is a randomised, double-blind, placebo-controlled trial of oral vitamin D supplementation (1000 IU cholecalciferol/day or placebo) from 14 weeks gestation in women with initial circulating 25(OH)-vitamin D levels of 25-100 nmol/l <span class="citation">[<a href="#ref-Harvey2012a" role="doc-biblioref">152</a>,<a href="#ref-Cooper2016" role="doc-biblioref">153</a>]</span>.
The primary outcome of the study was whole body bone mineral content, assessed by Dual Energy X-ray absorptiometry (DXA) within 14 days of birth, and follow up DXA at 4 years of age.
MAVIDOS did not find significant differences in neonatal bone outcomes overall but there were significant differences between intervention and placebo for births taking place during the winter months.</p>
<p><span class="math inline">\(1,25(OH)_2-Vitamin~D\)</span> has been implicated in the control of Plasma membrane <span class="math inline">\(Ca^{2+}\)</span>-ATPases (PMCAs).
The expression of the <em>PMCA3</em> gene in the placenta has been associated with umbilical cord calcium concentration and intrauterine accrual of bone mineral content <span class="citation">[<a href="#ref-Kip2004" role="doc-biblioref">198</a>]</span>.
This provides part of a candidate mechanism linking intrauterine vitamin D availability to bone outcomes, but much remains to be elucidated about how maternal vitamin D status may impact bone development.
Work in rat models has demonstrated the ability of maternal nutritional exposure during pregnancy to influence the epigenetic state and expression of genes in their offspring, specifically a change in DNA methylation was identified <span class="citation">[<a href="#ref-Burdge2007a" role="doc-biblioref">199</a>–<a href="#ref-Burdge2007" role="doc-biblioref">201</a>]</span>.
Consequently, the DNA methylation status of target genes of interest for vitamin D and bone development related pathways have been examined <span class="citation">[<a href="#ref-Holroyd2012" role="doc-biblioref">202</a>]</span>.</p>
<p>Retinoid-X-receptor-alpha (RXRA), which forms a heterodimer with the vitamin D receptor is essential for the nuclear action of <span class="math inline">\(1,25(OH)_2-Vitamin~D\)</span>.
DNA methylation at four of ten CpG sites in the <em>RXRA</em> gene were significantly (<span class="math inline">\(p\le0.05\)</span>) lower in the umbilical cord of offspring from cholecalciferol supplemented mothers compared to placebo with a mean difference in methylation of -1.98% (n=447, 95% CI -3.65 to -0.32, p=0.01).
One CpG site in the gene is related to estimated free 25(OH)D levels <span class="citation">[<a href="#ref-Harvey2014d" role="doc-biblioref">203</a>]</span>.</p>
<p>Methylation of cyclin-dependent kinase inhibitor 2A (CDKN2A) in umbilical cord tissue has been implicated in bone cell activity mediating skeletal development and homeostasis.
The <em>CDKN2A</em> locus has quite complex biology encoding two cell cycle inhibitors <span class="math inline">\(p14^{ARF}\)</span> and <span class="math inline">\(p16^{INK4a}\)</span> as well as the long non-coding RNA <em>ANRIL</em> which inhibits <span class="math inline">\(p16^{INK4a}\)</span>.
Targeted analysis of nine CpGs in a 300bp stretch of the <em>ANRIL</em> promoter region of <em>CDKN2A</em> was carried out.
Methylation at several of these sites showed an inverse correlation with bone size, mineral content, and mineral density at age four years <span class="citation">[<a href="#ref-Curtis2017" role="doc-biblioref">204</a>]</span>.</p>
<p>Epigenetic states are a product of both genetics and environment, they exist on a continuum from complete or obligatory genetic determination through facilitative variation influenced by genetics, to the purely epigenetic subject entirely to environmental factors.
This picture is complicated by the observation that genetics can not merely influence epigenetic state but also the extent to which that state will vary depending on environmental factors <span class="citation">[<a href="#ref-Holland2017" role="doc-biblioref">205</a>]</span>.
Epigenome-wide association studies have been performed on DNA methylation data for many traits including age (Section <a href="introduction.html#DNAmAgeReview">1.5.1</a>), smoking status <span class="citation">[<a href="#ref-Dogan2017" role="doc-biblioref">206</a>]</span>, exposure to atmospheric particulate matter <span class="citation">[<a href="#ref-Dai2016" role="doc-biblioref">207</a>]</span>, and obesity <span class="citation">[<a href="#ref-Wahl2016" role="doc-biblioref">208</a>]</span>.
Some robust and reproducible DNA methylation changes have been identified with EWAS, particularly for traits such as age and smoking.
Unlike genome-wide association studies (GWAS) associations found by EWAS can be due to reverse causation.
That is to say, a genetic variant associated with a disease is very unlikely to have been caused by the disease state whereas an epigenetic association can be the consequence of a disease state.
Whilst this can complicate the dissection of disease aetiology it can also be a boon in unpicking the biology that follows on from a particular disease or environmental exposure by revealing what biological networks are affected.</p>
<p>The previous work identifying DNA methylation changes at the <em>RXRA</em> <span class="citation">[<a href="#ref-Harvey2014d" role="doc-biblioref">203</a>]</span> and <em>CDKN2A/ANRIL</em> <span class="citation">[<a href="#ref-Curtis2017" role="doc-biblioref">204</a>]</span> loci associated with childhood bone outcomes lead to an interest in establishing if DNA methylation changes were occuring at other loci.
EWAS using the Illumina 450k and EPIC array technologies were performed for maternal Vitamin D status and childhood bone outcomes in data from the MAVIDOS and SWS cohorts.</p>
<div id="outline-of-ewas" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Outline of EWAS</h3>
<p>Three sets of EWAS analyses were performed:
Phase I of the MAVIDOS EPIC arrays and comparison with 450k array results.
Phase II of the MAVIDOS EPIC analysis with additional samples.
Analysis of the SWS cord blood EPIC array data.</p>
<ul>
<li>MAVIDOS phase I results (EPIC n = 140, 450k n = 60), EWAS outcomes:
<ul>
<li>Intervention Vs. Placebo</li>
<li>Bone Mineral Content at Birth (g), measured by DXA</li>
<li>Maternal circulating 25(OH)-vitamin D levels (nmol/l) at 34 weeks</li>
<li>Change in Maternal Vitamin D form 11 to 34 weeks</li>
</ul></li>
<li>MAVIDOS phase II results (n = 237), EWAS outcomes:
<ul>
<li>4 year Total BMC (g), without heads , measured by DXA</li>
</ul></li>
<li>SWS cord blood, EWAS outcomes:
<ul>
<li>8 year Total BMC (kg), without heads, adjusted for sex and age, measured by DXA (n = 408)</li>
<li>6 year Total BMC (kg), without heads, adjusted for sex and age, measured by DXA (n = 402)</li>
<li>6 year Periosteal circumference of the Tibia at 38% from the distal end (mm), measured by Peripheral Quantitative Computed Tomography (PQCT) (n = 141)</li>
<li>6 year Cortical density of the Tibia at 38% from the distal end (mg/cm3), measured by Peripheral PQCT (n = 141)</li>
</ul></li>
</ul>
</div>
</div>
<div id="methods-1" class="section level2">
<h2><span class="header-section-number">3.3</span> Methods</h2>
<p>Analysis of the Illumina EPIC and 450k methylation arrays was carried out in the <code>R</code> statistical programming language (v3.5.2) using the <a href="https://github.com/perishky/meffil/wiki"><code>meffil</code></a> package <span class="citation">[<a href="#ref-Min2017" role="doc-biblioref">209</a>]</span>, which was chosen as it is capable of performing functional normalisation in a more memory efficient fashion than alternatives such as <code>minfi</code>.</p>
<div id="functional-normalisation" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Functional Normalisation</h3>
<p>Functional normalisation is an approach to removing unwanted variation associated with ‘batch’ effects such as the date on which a sample was analysed or which slide a sample is on which was developed by Fortin et al. <span class="citation">[<a href="#ref-Fortin2014" role="doc-biblioref">163</a>]</span>.
This noise in the data masks the signal associated with the underlying biological effect of interest.</p>
<p>Functional normalisation makes use of control probes on the arrays which are designed to capture only technical variation as surrogates for the sources of unwanted variation.
The control probes are processed into 42 summary measures, and principal components analysis is performed on the control probe summary matrices for all samples.
The top <em>m</em> principal components (PCs) are used as the surrogates for technical variation going forward.
The number, <em>m</em>, of PCs from the control probe summary matrices used is informed by the amount of residual variation remaining after normalisation.
Picking the number of PCs which correspond to the last steep drop in residual variation is the approach recommended by the implementers of <code>meffil</code>, and Fortin et al. <span class="citation">[<a href="#ref-Fortin2014" role="doc-biblioref">163</a>]</span> recommend an <em>m</em> of 2 as performing well across a variety of analyses.
(See supplementary material from <span class="citation">[<a href="#ref-Fortin2014" role="doc-biblioref">163</a>]</span> for details of the control probe summary process.)</p>
<p>The process used by functional normalisation is a variant on quantile normalisation.
Instead of forcing the empirical marginal distributions of the samples to be identical at each site across arrays. To produce the normalised values it constructs a quantile function which only removes variation arising from surrogates for batch variation.
This approach is effective even when comparing samples with large global differences in methylation levels such as between normal and cancer samples, but cannot overcome high degrees of confounding <span class="citation">[<a href="#ref-Fortin2014" role="doc-biblioref">163</a>]</span>.</p>
</div>
<div id="genetically-confounded-and-multi-mapping-probes" class="section level3">
<h3><span class="header-section-number">3.3.2</span> Genetically Confounded and Multi-mapping Probes</h3>
<p>Probes from different locations in the genome with similar sequences, especially following the reduction in sequence complexity associated with bisulfite conversion, can be cross-reactive on the arrays leading to erroneous signals and are thus commonly excluded from analysis.
DNAm is often strongly influenced by genetic factors, the effect is especially pronounced when variants alter the sequence at CpG sites themselves as the site can then no longer be methylated.
Thus, sites that have common genetic variants at the probe site are also excluded from the analysis, as well as some sites whose methylation is known to be under strong genetic influence by common genetic variation.</p>
<p>43,254 probes on the EPIC array have been identified as multi-mapping, DNA binding to the probes may be derived from other locations in the genome invalidating these probes as a measure of methylation at their intended loci <span class="citation">[<a href="#ref-Pidsley2016" role="doc-biblioref">164</a>]</span>.
12,510 probes were identified as having genetic variants at the CpG locus they are intending to assay, this can produce misleading results as mutant bases can resemble the products of bisulfite conversion <span class="citation">[<a href="#ref-Pidsley2016" role="doc-biblioref">164</a>]</span>.
1,812 probes where found t overlap regions exhibiting haplotype-specific methylation associated with common non-SNP genetic variants (CNVs, Indels, STRs) and regional in-phase clustering of CpG-SNPs <span class="citation">[<a href="#ref-Bell2017a" role="doc-biblioref">67</a>]</span>.
Zhou et al. <span class="citation">[<a href="#ref-Zhou2017" role="doc-biblioref">165</a>]</span> provided a list of probes which they recommend ‘masking’ from the 450k array due to multi-mapping issues, genetic variants overlapping the CpG sites, and other factors which may render results from these probes problematic.</p>
<p>In order to identify any potential additional sources of genetic confounding in the phase I MAVIDOS analysis, Probes with methylation values which cluster into distinct groups were identified using the ‘gap hunting’ method developed by Andrews et al. <span class="citation">[<a href="#ref-Andrews2016" role="doc-biblioref">166</a>]</span>.
Such distinct clusters of methylation can arise from genetic variants which influence methylation levels being present in homozygous and heterozygous forms in the study population, see Figure <a href="arrays.html#fig:MAVIDOSgaphunterEgMonApr162018">3.15</a>.
As the sensitivity and specificity of ‘gap hunting’ is limited, it is the advice of the authors not to exclude probes flagged by <code>gaphunter()</code> prior to performing EWAS.
It is instead advised to check if any of the results appear in this list after the fact and examine the possibility of a genetic effect if they are.
In phase II of the MAVIDOS analysis and in the SWS analyses only probes with specific technical QC issues were excluded prior to normalisation and EWAS.
Flagging of problematic probes occurred after EWAS on any significant results.</p>
</div>
<div id="ewas-models" class="section level3">
<h3><span class="header-section-number">3.3.3</span> EWAS Models</h3>
<div id="cell-type-correction" class="section level4">
<h4><span class="header-section-number">3.3.3.1</span> Cell-Type Correction</h4>
<p>Cell-type composition is a known confounder in epigenetic studies <span class="citation">[<a href="#ref-Jaffe2014" role="doc-biblioref">145</a>,<a href="#ref-Houseman2012" role="doc-biblioref">146</a>]</span>.
Observed variation DNA methylation can be cell-type intrinsic, where changes in DNA methylation are not driven by changes in the cell-type composition of the tissue; or cell-type extrinsic, where changes in DNA methylation are due to changes in the cell-type composition of the population sampled (Figure <a href="arrays.html#fig:IntrinsicVsExtrinsicDNAmChangeGeneral">3.3</a>).
An established approach for addressing this potential source of confounding is to add terms to the regression model which reflect the cell-type composition of each sample.
Cell-type composition can be ascertained through three main approaches: Direct cell count data; Estimating cell counts using the experimental data and models fitted on DNA methylation data from reference panels of known cell-type composition <span class="citation">[<a href="#ref-Houseman2012" role="doc-biblioref">146</a>]</span>; or reference free approaches which make use of mathematical methods to identify sources of confounding variation such as cell-type heterogeneity.
Whilst there are several reference samples available for cord blood <span class="citation">[<a href="#ref-Cardenas2016" role="doc-biblioref">210</a>–<a href="#ref-Gervin2016" role="doc-biblioref">213</a>]</span>, none were available for cord tissue at the time of the initial analysis, and consequently a reference free technique was used in phase I of the MAVIDOS analysis.
Models were fitted using Surrogate Variable Analysis (SVA) <span class="citation">[<a href="#ref-Leek2007" role="doc-biblioref">214</a>]</span> and independent Surrogate Variable Analysis (iSVA) <span class="citation">[<a href="#ref-Teschendorff2011" role="doc-biblioref">215</a>]</span>.
The focus of these results is on SVA as minimal differences between the two methods were observed and SVA was recommended based on comparisons of the performance of various cell-type heterogeneity correction methods <span class="citation">[<a href="#ref-McGregor2016" role="doc-biblioref">216</a>,<a href="#ref-Teschendorff2017" role="doc-biblioref">217</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:IntrinsicVsExtrinsicDNAmChangeGeneral"></span>
<img src="figs/IntrinsicVsExtrinsicDNAmChangeGeneral.png" alt="Diagrammatic representation of DNAm change arising from extrinsic or intrinsic changes in DNA methylation. Extrinsic changes are due to shifts in cell-type composition. Intrinsic changes in DNA methylation occur without changes in the proportions of cell-types. These two modes of change are of course not mutually exclusive and both can be occurring." width="80%" />
<p class="caption">
Figure 3.3: Diagrammatic representation of DNAm change arising from extrinsic or intrinsic changes in DNA methylation. Extrinsic changes are due to shifts in cell-type composition. Intrinsic changes in DNA methylation occur without changes in the proportions of cell-types. These two modes of change are of course not mutually exclusive and both can be occurring.
</p>
</div>

<p>A reference panel for cord tissue samples was recently published <span class="citation">[<a href="#ref-Lin2018" role="doc-biblioref">218</a>]</span>.
This cell-type reference had not yet been integrated into the <code>meffil</code> R package used to perform the EWAS analyses so I created <a href="https://github.com/RichardJActon/meffil/blob/LinAndKarnaniCordTissueRef/data-raw/lin-karnani-reference.r">a fork of <code>meffil</code> including this reference panel in the required format</a>, this can be installed directly from github.
The code I used to add this reference to the package data before building the R package can be found in this <a href="https://gist.github.com/RichardJActon/703ae70438e865a37998cd608bd12cd3">gist</a>.
Cell-type correction based on this reference was used in phase II of the MAVIDOS analysis.</p>
</div>
<div id="structure-of-models-fitted-for-each-ewas" class="section level4">
<h4><span class="header-section-number">3.3.3.2</span> Structure of models fitted for each EWAS</h4>
<p>By default EWAS in <code>meffil</code> are run with four different models:</p>
<ol style="list-style-type: decimal">
<li>No covariates, attempting to predict methylation with the variable of interest.</li>
<li>All covariates, attempting to predict methylation with the variable of interest plus a user-supplied list of covariates.</li>
<li>Surrogate Variables + all covariates, attempting to predict methylation with user-supplied covariates and surrogate variables generated from SVA.</li>
<li>Independent Surrogate Variables + all covariates, attempting to predict methylation with user-supplied covariates and independent surrogate variables generated from iSVA.</li>
</ol>
<p>Running EWAS with multiple models permits the effects of adding the various covariates on the results of the analysis to be seen.</p>
</div>
</div>
<div id="concordance-of-epic-and-450k-ewas-results" class="section level3">
<h3><span class="header-section-number">3.3.4</span> Concordance of EPIC and 450k EWAS Results</h3>
<p>In order to ascertain if the results from the EWAS in the 450k and EPIC arrays were producing similar sets of probes in the top ranking positions when ordered by p-value, the concordance (% overlap) between the top k probes, where <span class="math inline">\(k = 1 .. 100,000\)</span> was calculated.
Only probes in common between the two arrays were used and k was incremented in steps of 50.</p>
</div>
</div>
<div id="results" class="section level2">
<h2><span class="header-section-number">3.4</span> Results</h2>
<div id="mavidos-phase-i" class="section level3">
<h3><span class="header-section-number">3.4.1</span> MAVIDOS phase I</h3>
<p>DNA methylation at none of the probes were significantly associated with any of the four variables of interest for each EWAS performed in either the EPIC or 450k datasets.
The concordance between the probes with the top-ranked p-values in common between the EPIC and 450k data was at the level of chance.</p>
<div id="whole-array-qc" class="section level4">
<h4><span class="header-section-number">3.4.1.1</span> Whole array QC</h4>
<div id="epic-arrays" class="section level5">
<h5><span class="header-section-number">3.4.1.1.1</span> EPIC arrays</h5>
<p>The predicted sex of the samples generated using sex chromosome probe intensities was checked against that in the sample annotation and two mismatches were found.
These were MAVIDOS IDs 206 and 63 and the associated arrays were excluded from further analysis, (Figure <a href="arrays.html#fig:MAVIDOSqcEPICsexMismatch">3.4</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICsexMismatch"></span>
<img src="figs/MAVIDOSqcEPICsexMismatch.png" alt="Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotations. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.4: Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotations. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<p>The dataset also contained four samples for which there were two technical replicates, only the first replicate from each was used (144 arrays run for 140 individuals).
Array 201516310023 (MAVIDOS ID 95) was excluded as its median methylated signal was more than <span class="math inline">\(3\sigma\)</span> from the expected value, (Figure <a href="arrays.html#fig:MAVIDOSqcEPICmethVsUnmeth">3.5</a>).
No samples were excluded for having a higher than expected proportion of undetected probes (proportion of probes with detection p-value &gt; 0.01 is &gt; 0.1) (Figure <a href="arrays.html#fig:MAVIDOSqcEPICpropSites">3.6</a>).
No samples were excluded for having a high proportion of probes with low bead counts (proportion of probes with bead number &lt; 3 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqcEPICbeadNum">3.7</a>).
In total 3 of the EPIC arrays were excluded from the analysis for failing quality control leaving an n=137.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICmethVsUnmeth"></span>
<img src="figs/MAVIDOSqcEPICmethVsUnmeth.png" alt="Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing \(3\sigma\) from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.5: Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing <span class="math inline">\(3\sigma\)</span> from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICpropSites"></span>
<img src="figs/MAVIDOSqcEPICpropSites.png" alt="Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.6: Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICbeadNum"></span>
<img src="figs/MAVIDOSqcEPICbeadNum.png" alt="Proportion of probes with a bead count of &lt; 3 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.7: Proportion of probes with a bead count of &lt; 3 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

</div>
<div id="k-arrays" class="section level5">
<h5><span class="header-section-number">3.4.1.1.2</span> 450k Arrays</h5>
<p>There were no mismatches between predicted and annotated sex (Figure <a href="arrays.html#fig:MAVIDOSqc450ksexMismatch">3.8</a>).
There were not any samples with outliers in their methylated / unmethylated probe proportions (Figure <a href="arrays.html#fig:MAVIDOSqc450kmethVsUnmeth">3.9</a>).
No samples were excluded for having a higher than expected proportion of undetected probes (proportion of probes with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqc450kpropSites">3.10</a>).
No samples were excluded for having a high proportion of probes with low bead counts (proportion of probes with bead number &lt; 3 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqc450kbeadNum">3.11</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450ksexMismatch"></span>
<img src="figs/MAVIDOSqc450ksexMismatch.png" alt="Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the 450k array data. No predicted sex values differ from their annotations. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.8: Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the 450k array data. No predicted sex values differ from their annotations. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kmethVsUnmeth"></span>
<img src="figs/MAVIDOSqc450kmethVsUnmeth.png" alt="Median methylated signal vs unmethylated signal per sample for the 450k array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing \(3\sigma\) from the expected mean. Samples outside the expected range would be indicated in the legend. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.9: Median methylated signal vs unmethylated signal per sample for the 450k array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing <span class="math inline">\(3\sigma\)</span> from the expected mean. Samples outside the expected range would be indicated in the legend. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kpropSites"></span>
<img src="figs/MAVIDOSqc450kpropSites.png" alt="Proportion of probes with detection p-values &gt;0.01 by sample for the 450k array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.10: Proportion of probes with detection p-values &gt;0.01 by sample for the 450k array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kbeadNum"></span>
<img src="figs/MAVIDOSqc450kbeadNum.png" alt="Proportion of probes with a bead count of &lt; 3 by sample for the 450k array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.11: Proportion of probes with a bead count of &lt; 3 by sample for the 450k array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

</div>
</div>
<div id="probe-qc" class="section level4">
<h4><span class="header-section-number">3.4.1.2</span> Probe QC</h4>
<div id="probe-qc---epic-arrays" class="section level5">
<h5><span class="header-section-number">3.4.1.2.1</span> Probe QC - EPIC Arrays</h5>
<p>There were no outliers within the control probes Figure <a href="arrays.html#fig:MAVIDOSqcEPICctrlProbes">3.12</a>.
1,626 probes were excluded for having high background signal in a high proportion of samples (proportion of samples with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqcEPICpropSamples">3.13</a>).
162 probes were excluded for having low bead count in a high proportion of samples (proportion of samples with bead number &lt; 3 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqcEPICbeadNumAcross">3.14</a>).
Probes with poor technical quality were excluded from the analysis prior to functional normalisation.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICctrlProbes"></span>
<img src="figs/MAVIDOSqcEPICctrlProbes.png" alt="Control probe signal by sample for each summary group for the EPIC data. Outliers would be circled in black. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.12: Control probe signal by sample for each summary group for the EPIC data. Outliers would be circled in black. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICpropSamples"></span>
<img src="figs/MAVIDOSqcEPICpropSamples.png" alt="Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.13: Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICbeadNumAcross"></span>
<img src="figs/MAVIDOSqcEPICbeadNumAcross.png" alt="Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.14: Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<p>Problematic probes identified by Pidsley et al. <span class="citation">[<a href="#ref-Pidsley2016" role="doc-biblioref">164</a>]</span> and those overlapping the regions identified by Bell et al. <span class="citation">[<a href="#ref-Bell2017a" role="doc-biblioref">67</a>]</span> were excluded from subsequent analysis after functional normalisation.
This, including the poor quality probes, is a total of 57,396 unique probes excluded from the analysis (~6.62% of the total number of probes).
Gap hunter identified a further 77,398 probes (8.9% of all probes) which might be subject to genetic confounding (Figure <a href="arrays.html#fig:MAVIDOSgaphunterEgMonApr162018">3.15</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSgaphunterEgMonApr162018"></span>
<img src="figs/MAVIDOSgaphunterEgMonApr162018.png" alt="An example of the DNAm distribution for a result from the gaphunter() function. This is an example chosen to best exemplify the sort of result which is strongly suggestive of a genetic variant with an impact on methylation status acting on this site. It is unrepresentative of typical results from gaphunter() in that the groups have a relatively even membership, many results have a small number of individuals in one or more groups making it hard to distinguish methylation outliers caused by rarer genetic variants from those with other causes." width="80%" />
<p class="caption">
Figure 3.15: An example of the DNAm distribution for a result from the <code>gaphunter()</code> function. This is an example chosen to best exemplify the sort of result which is strongly suggestive of a genetic variant with an impact on methylation status acting on this site. It is unrepresentative of typical results from <code>gaphunter()</code> in that the groups have a relatively even membership, many results have a small number of individuals in one or more groups making it hard to distinguish methylation outliers caused by rarer genetic variants from those with other causes.
</p>
</div>

</div>
<div id="probe-qc---450k-arrays" class="section level5">
<h5><span class="header-section-number">3.4.1.2.2</span> Probe QC - 450k Arrays</h5>
<p>509 probes were excluded for having high background signal in a high proportion of samples (proportion of samples with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqc450kpropSamples">3.17</a>).
1037 probes were excluded for having low bead count in a high proportion of samples (proportion of samples with bead number &lt; 3 is &gt; 0.1) (Figure <a href="arrays.html#fig:MAVIDOSqc450kbeadNumAcross">3.18</a>).
There was one sample (MAVIDOS ID 2183) with an outlier within the control probes, a dinitrophenyl labelled staining control probe, thus it was not excluded as only outliers in dye bias and bisulfite conversion control probes were deemed sufficient grounds for excluding a sample (Figure <a href="arrays.html#fig:MAVIDOSqc450kctrlProbes">3.16</a>, <a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/infinium_assays/infinium_hd_methylation/infinium-hd-methylation-guide-15019519-01.pdf">Detailed contol probe desciption</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kctrlProbes"></span>
<img src="figs/MAVIDOSqc450kctrlProbes.png" alt="Control probe signal by sample for each summary group for the 450k data. Outliers would be circled in black. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.16: Control probe signal by sample for each summary group for the 450k data. Outliers would be circled in black. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kpropSamples"></span>
<img src="figs/MAVIDOSqc450kpropSamples.png" alt="Undetectable probes across samples for 450k data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.17: Undetectable probes across samples for 450k data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kbeadNumAcross"></span>
<img src="figs/MAVIDOSqc450kbeadNumAcross.png" alt="Low bead count probes across samples for 450k data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.18: Low bead count probes across samples for 450k data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<p>All probes on the ‘general mask’ list from Zhou et al. <span class="citation">[<a href="#ref-Zhou2017" role="doc-biblioref">165</a>]</span> were excluded from the analysis following functional normalisation, leaving a total of 418,632 probes for subsequent analysis.</p>
</div>
</div>
<div id="functional-normalisation-1" class="section level4">
<h4><span class="header-section-number">3.4.1.3</span> Functional Normalisation</h4>
<p>An <em>m</em> of 6 was chosen for the EPIC arrays as this value produced the last steep drop in residual variation, see Figure <a href="arrays.html#fig:MAVIDOSfunnormPCsEPICMonApr162018">3.19</a>.
An <em>m</em> of 6 was chosen for the 450k arrays as this value produced the last steep drop in residual variation, see Figure <a href="arrays.html#fig:MAVIDOSfunnormPCs450kMonApr162018">3.20</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSfunnormPCsEPICMonApr162018"></span>
<img src="figs/MAVIDOSfunnormPCsEPICMonApr162018.png" alt="Residual variation remaining after functional normalisation of the top 20,000 most variable probes with m PCs from the control probe summary matrices for the EPIC array samples (n=137), for M = methylated and U = unmethylated probes." width="80%" />
<p class="caption">
Figure 3.19: Residual variation remaining after functional normalisation of the top 20,000 most variable probes with <em>m</em> PCs from the control probe summary matrices for the EPIC array samples (n=137), for M = methylated and U = unmethylated probes.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSfunnormPCs450kMonApr162018"></span>
<img src="figs/MAVIDOSfunnormPCs450kMonApr162018.png" alt="Residual variation remaining after functional normalisation of the top 20,000 most variable probes with m PCs from the control probe summary matrices for the 450k array samples (n=60), for M = methylated and U = unmethylated probes." width="80%" />
<p class="caption">
Figure 3.20: Residual variation remaining after functional normalisation of the top 20,000 most variable probes with <em>m</em> PCs from the control probe summary matrices for the 450k array samples (n=60), for M = methylated and U = unmethylated probes.
</p>
</div>

</div>
<div id="ewass" class="section level4">
<h4><span class="header-section-number">3.4.1.4</span> EWASs</h4>
<p>All EWAS performed below were also performed in exactly the same fashion for the 60 450k arrays, none of these results were significant and they are not included here.</p>
<div id="neonatal-bone-mineral-content" class="section level5">
<h5><span class="header-section-number">3.4.1.4.1</span> Neonatal Bone Mineral Content</h5>
<p>Figure <a href="arrays.html#fig:MAVIDOSbmc">3.21</a> illustrates the distribution of Neonatal Bone Mineral Content, the outcome on which this EWAS was performed.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSbmc"></span>
<img src="figs/MAVIDOSbmc.png" alt="Distribution of Neonatal Bone Mineral Content (g) for individuals in the EWAS." width="80%" />
<p class="caption">
Figure 3.21: Distribution of Neonatal Bone Mineral Content (g) for individuals in the EWAS.
</p>
</div>

<p>No probes fell below the Bonferroni corrected significance threshold for an association between DNA methylation at that locus and neonatal bone mineral content, Figure <a href="arrays.html#fig:MAVIDOSbmcEPICewasManhattanSVA">3.22</a>.
Sex and sample age at DXA were included as covariates in the ‘all’ model.
SVA generated 5 significant surrogate variables which were additionally used in the SVA model.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSbmcEPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSbmcEPICewasManhattanSVA.png" alt="Results of EWAS for neonatal bone mineral content with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)." width="80%" />
<p class="caption">
Figure 3.22: Results of EWAS for neonatal bone mineral content with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>.
</p>
</div>

</div>
<div id="intervention-placebo" class="section level5">
<h5><span class="header-section-number">3.4.1.4.2</span> Intervention / Placebo</h5>
<p>No probes fell below the Bonferroni corrected significance threshold for an association between DNA methylation at that locus and Intervention/placebo group status, Figure <a href="arrays.html#fig:MAVIDOSccEPICewasManhattanSVA">3.23</a>.
Sex and sample age at DXA were included as covariates in the ‘all’ model.
SVA generated 5 significant surrogate variables which were additionally used in the SVA model.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSccEPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSccEPICewasManhattanSVA.png" alt="Results of EWAS for intervention/placebo group status with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)." width="80%" />
<p class="caption">
Figure 3.23: Results of EWAS for intervention/placebo group status with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>.
</p>
</div>

</div>
<div id="maternal-vitamin-d-34wks" class="section level5">
<h5><span class="header-section-number">3.4.1.4.3</span> Maternal Vitamin D (34wks)</h5>
<p>Maternal Vitamin D levels remain substantially overlapping between intervention and placebo groups at 34wks, see Figure <a href="arrays.html#fig:MAVIDOSvitDviolinMonApr162018">3.24</a>.
Thus maternal vitamin D at 34wks may prove a more useful variable to model than intervention/placebo status.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSvitDviolinMonApr162018"></span>
<img src="figs/MAVIDOSvitDviolinMonApr162018.png" alt="Maternal circulating 25(OH)-vitamin D levels (nmol/l) at 11 and 34wks gestation, supplementation with 1000 IU/day cholecalciferol began at week 14. Each participant is shown at both time points linked by a line to indicate the direction of change. The violin plots indicate the density of the distribution of vitamin D values at each time point with the \(25^{th}\), \(50^{th}\), \(75^{th}\) quantiles indicated with horizontal black lines. The colour indicates Intervention (Red) / Placebo (Blue) group" width="80%" />
<p class="caption">
Figure 3.24: Maternal circulating 25(OH)-vitamin D levels (nmol/l) at 11 and 34wks gestation, supplementation with 1000 IU/day cholecalciferol began at week 14. Each participant is shown at both time points linked by a line to indicate the direction of change. The violin plots indicate the density of the distribution of vitamin D values at each time point with the <span class="math inline">\(25^{th}\)</span>, <span class="math inline">\(50^{th}\)</span>, <span class="math inline">\(75^{th}\)</span> quantiles indicated with horizontal black lines. The colour indicates Intervention (Red) / Placebo (Blue) group
</p>
</div>

<p>No probes fell below the Bonferroni corrected significance threshold for an association between DNAm at that locus and maternal vitamin D levels at 34wks gestation, (Figure <a href="arrays.html#fig:MAVIDOSmVitD34EPICewasManhattanSVA">3.25</a>).
Sex and sample age at DXA were included as covariates in the ‘all’ model.
SVA generated 5 significant surrogate variables which were additionally used in the SVA model.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSmVitD34EPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSmVitD34EPICewasManhattanSVA.png" alt="Results of EWAS for maternal circulating 25(OH)-vitamin D levels (nmol/l) levels at 34wks gestation with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented by the largest and least transleucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)." width="80%" />
<p class="caption">
Figure 3.25: Results of EWAS for maternal circulating 25(OH)-vitamin D levels (nmol/l) levels at 34wks gestation with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented by the largest and least transleucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>.
</p>
</div>

</div>
<div id="change-in-maternal-vitamin-d" class="section level5">
<h5><span class="header-section-number">3.4.1.4.4</span> Change in Maternal Vitamin D</h5>
<p>Figure <a href="arrays.html#fig:MAVIDOSdifVD">3.26</a> illustrates the change in maternal vitamin D from 11 to 34 weeks gestation.
No probes fell below the Bonferroni corrected significance threshold for an association between DNAm at that locus and change in maternal vitamin D from 11 to 34wks gestation, Figure <a href="arrays.html#fig:MAVIDOSdifVDEPICewasManhattanSVA">3.27</a>.
SVA generated 5 significant surrogate variables which were additionally used in the SVA model.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSdifVD"></span>
<img src="figs/MAVIDOSdifVD.png" alt="Distribution of the changes in maternal circulating 25(OH)-vitamin D levels (nmol/l) levels from 11 to 34 weeks gestation." width="80%" />
<p class="caption">
Figure 3.26: Distribution of the changes in maternal circulating 25(OH)-vitamin D levels (nmol/l) levels from 11 to 34 weeks gestation.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSdifVDEPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSdifVDEPICewasManhattanSVA.png" alt="Results of EWAS for change in maternal circulating 25(OH)-vitamin D levels (nmol/l) levels from 11 to 34wks gestation with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)." width="80%" />
<p class="caption">
Figure 3.27: Results of EWAS for change in maternal circulating 25(OH)-vitamin D levels (nmol/l) levels from 11 to 34wks gestation with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>.
</p>
</div>

</div>
</div>
<div id="concordance-of-epic-and-450k-ewas-results-1" class="section level4">
<h4><span class="header-section-number">3.4.1.5</span> Concordance of EPIC and 450k EWAS results</h4>
<p>Concordance between the rankings of probes would suggest that the EWAS may be capturing a ‘real’ signal that is simply below the significance threshold with the statistical sensitivity/power that is available in this dataset.
The concordance between the EPIC and 450k datasets (Figures <a href="arrays.html#fig:EPIC450kEWASconcordMAVIDOSMonApr162018">3.28</a> &amp; <a href="arrays.html#fig:EPIC450kEWASconcordMAVIDOSMonApr162018zoom">3.29</a>) appears to be at roughly the level expected by chance.
This does not lend support to the possibility that there are associations between the variables of interest and DNAm that are beneath the current sensitivity of the study, it does not, however, rule this out.
In the absence of any probes above the significance threshold and with poor concordance between the 450k and EPIC array p-value rankings no further analyses such as gene set enrichment and differentially methylated region (DMR) calling have been carried out at this time.</p>
<div class="figure" style="text-align: center"><span id="fig:EPIC450kEWASconcordMAVIDOSMonApr162018"></span>
<img src="figs/EPIC450kEWASconcordMAVIDOSMonApr162018.png" alt="Concordance between the top 100,000 probes in common between the EWASs run on the EPIC (n=137) and 450k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Change in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concordance expected by chance (intersects 50% at 387,511, the number of shared probes)." width="80%" />
<p class="caption">
Figure 3.28: Concordance between the top 100,000 probes in common between the EWASs run on the EPIC (n=137) and 450k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Change in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concordance expected by chance (intersects 50% at 387,511, the number of shared probes).
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:EPIC450kEWASconcordMAVIDOSMonApr162018zoom"></span>
<img src="figs/EPIC450kEWASconcordMAVIDOSMonApr162018zoom.png" alt="Concordance between the top 10,000 probes in common between the EWASs run on the EPIC (n=137) and 405k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Chance in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concordance expected by chance (intersects 50% at 387,511, the number of shared probes)." width="80%" />
<p class="caption">
Figure 3.29: Concordance between the top 10,000 probes in common between the EWASs run on the EPIC (n=137) and 405k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Chance in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concordance expected by chance (intersects 50% at 387,511, the number of shared probes).
</p>
</div>

</div>
</div>
<div id="mavidos-phase-ii" class="section level3">
<h3><span class="header-section-number">3.4.2</span> MAVIDOS phase II</h3>
<p>DNA methylation at none of the probes was significantly associated with bone mineral content at 4 years.</p>
<div id="whole-array-qc-1" class="section level4">
<h4><span class="header-section-number">3.4.2.1</span> Whole Array QC</h4>
<p>The predicted sex of the samples generated using sex chromosome probe intensities was checked against that in the sample annotation and 5 mismatches were found (Figure <a href="arrays.html#fig:MAVIDOSIIqcEPICsexMismatch">3.30</a>).
Two samples did not have sex information, 2 had predicted sex discordant with their annotated sex and 1 was ambiguous these were excluded from further analysis.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIqcEPICsexMismatch"></span>
<img src="figs/MAVIDOS-II-qcEPICsexMismatch.png" alt="Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotations. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.30: Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotations. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<p>Arrays: 201516310023 (mavid: 1490), 201516320022 (mavid: 1672), 201530430013 (mavvid: 4090), 201530430015 (mavid: 1903), &amp; 202410280028 (mavid: 2078) were excluded as their median methylated signal was more than <span class="math inline">\(3\sigma\)</span> from the expected value, (Figure <a href="arrays.html#fig:MAVIDOSIIqcEPICmethVsUnmeth">3.31</a>).
No samples were excluded for having a higher than expected proportion of undetected probes (proportion of probes with detection p-value &gt; 0.01 is &gt; 0.1) (Figure <a href="arrays.html#fig:MAVIDOSIIqcEPICpropSites">3.32</a>).
No samples were excluded for having a high proportion of probes with low bead counts (proportion of probes with bead number &lt; 3 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqcEPICbeadNum">3.7</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIqcEPICmethVsUnmeth"></span>
<img src="figs/MAVIDOS-II-qcEPICmethVsUnmeth.png" alt="Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing \(3\sigma\) from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.31: Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing <span class="math inline">\(3\sigma\)</span> from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIqcEPICpropSites"></span>
<img src="figs/MAVIDOS-II-qcEPICpropSites.png" alt="Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.32: Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIqcEPICbeadNum"></span>
<img src="figs/MAVIDOS-II-qcEPICbeadNum.png" alt="Proportion of probes with a bead count of &lt; 3 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.33: Proportion of probes with a bead count of &lt; 3 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

</div>
<div id="probe-qc-1" class="section level4">
<h4><span class="header-section-number">3.4.2.2</span> Probe QC</h4>
<p>There was one outlier within the control probes, in a non-critical specificity control probe for detecting non-specific methylation detection over an unmethylated background (Figure <a href="arrays.html#fig:MAVIDOSqcEPICctrlProbes">3.12</a>).
1,317 probes were excluded for having high background signal in a high proportion of samples (proportion of samples with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqcEPICpropSamples">3.13</a>).
220 probes were excluded for having low bead count in a high proportion of samples (proportion of samples with bead number &lt; 3 is &gt; 0.1), (Figure <a href="arrays.html#fig:MAVIDOSqcEPICbeadNumAcross">3.14</a>).
Probes with poor technical quality were excluded from the analysis prior to functional normalisation.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIqcEPICctrlProbes"></span>
<img src="figs/MAVIDOS-II-qcEPICctrlProbes.png" alt="Control probe signal by sample for each summary group for the EPIC data. Outliers would be circled in black. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.34: Control probe signal by sample for each summary group for the EPIC data. Outliers would be circled in black. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIqcEPICpropSamples"></span>
<img src="figs/MAVIDOS-II-qcEPICpropSamples.png" alt="Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.35: Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIqcEPICbeadNumAcross"></span>
<img src="figs/MAVIDOS-II-qcEPICbeadNumAcross.png" alt="Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.36: Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

</div>
<div id="functional-normalisation-2" class="section level4">
<h4><span class="header-section-number">3.4.2.3</span> Functional Normalisation</h4>
<p>An <em>m</em> of 6 was chosen as this value produced the last steep drop in residual variation, see Figure <a href="arrays.html#fig:MAVIDOSIIfunnormPCsEPIC">3.37</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIIfunnormPCsEPIC"></span>
<img src="figs/MAVIDOS-II-funnormPCsEPIC.png" alt="Residual variation remaining after functional normalisation of the top 20,000 most variable probes with m PCs from the control probe summary matrices for the EPIC array samples (n=237), for M = methylated and U = unmethylated probes." width="80%" />
<p class="caption">
Figure 3.37: Residual variation remaining after functional normalisation of the top 20,000 most variable probes with <em>m</em> PCs from the control probe summary matrices for the EPIC array samples (n=237), for M = methylated and U = unmethylated probes.
</p>
</div>

</div>
<div id="ewas" class="section level4">
<h4><span class="header-section-number">3.4.2.4</span> EWAS</h4>
<div id="bone-mineral-content-at-4-years" class="section level5">
<h5><span class="header-section-number">3.4.2.4.1</span> Bone Mineral Content at 4 years</h5>
<p>The model for this EWAS attempted to predict whole body (minus head) bone mineral content (g) (Figure <a href="arrays.html#fig:MAVIDOSIInsubtotbmc">3.38</a>) correcting for: Cell type composition (B-cells, CD4+T, CD8+T, Granulocytes, Monocytes, Natural Killer cells, Endothelial cells, Epithelial cells, Stromal cells), Mother’s age at birth, Sex, Mother’s BMI at 11 weeks gestation, whether the mother smoked during pregnancy, &amp; Gestational Age.
The EWAS was performed on 237 individuals.
surrogate variable analysis identified 19 significant surrogate variables which were included in the model.
No probes fell below the Bonferroni corrected significance threshold for an association between DNAm at that locus and bone mineral content, in any of the models including SVA (Figure <a href="arrays.html#fig:MAVIDOSIInsubtotbmcEPICewasManhattanSVA">3.39</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIInsubtotbmc"></span>
<img src="figs/MAVIDOSIInsubtotbmc.png" alt="Distribution of whole body (minus head) bone mineral content in grams at 4 years of age." width="80%" />
<p class="caption">
Figure 3.38: Distribution of whole body (minus head) bone mineral content in grams at 4 years of age.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSIInsubtotbmcEPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSIInsubtotbmcEPICewasManhattanSVA.png" alt="Results of EWAS for whole body minus head bone mineral content (g) with SVA model (n = 237). Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)." width="80%" />
<p class="caption">
Figure 3.39: Results of EWAS for whole body minus head bone mineral content (g) with SVA model (n = 237). Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented by the largest and least translucent points. x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>.
</p>
</div>

</div>
</div>
</div>
<div id="southampton-womens-survey-sws" class="section level3">
<h3><span class="header-section-number">3.4.3</span> Southampton Women’s Survey (SWS)</h3>
<p>DNA methylation at two probes were significantly associated with total bone mineral content at 6 years and periosteal circumference at 6 years respectively.
Probe cg26559250 located at Chr6:157,653,445-157,653,447 at the ZDHHC14 (zinc finger DHHC-type palmitoyltransferase 14) gene showed an increase of DNA methylation with increasing bone mineral content with a significance of <span class="math inline">\(2.52\times 10^{-8}\)</span>
Probe cg22570676 located at Chr19:2,527,492-2,527,494 at the GNG7 (G protein subunit gamma 7) gene showed an increase of DNA methylation with increasing periosteal circumference with a significance of <span class="math inline">\(4.24\times 10^{-8}\)</span>.
Neither probe is flagged for known technical issues or genetic confounding <span class="citation">[<a href="#ref-Zhou2017" role="doc-biblioref">165</a>]</span>.</p>
<div id="whole-array-qc-2" class="section level4">
<h4><span class="header-section-number">3.4.3.1</span> Whole Array QC</h4>
<p>The predicted sex of the samples generated using sex chromosome probe intensities was checked against that in the sample annotation and no mismatches were found (Figure <a href="arrays.html#fig:SWSqcEPICsexMismatch">3.40</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:SWSqcEPICsexMismatch"></span>
<img src="figs/SWSqcEPICsexMismatch.png" alt="Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotations. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.40: Predicted sex of each sample based on the sex chromosome copy numbers inferred from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotations. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<p>No samples were excluded for having a median methylated signal that was more than <span class="math inline">\(3\sigma\)</span> from the expected value, (Figure <a href="arrays.html#fig:SWSqcEPICmethVsUnmeth">3.41</a>).
No samples were excluded for having a higher than expected proportion of undetected probes (proportion of probes with detection p-value &gt; 0.01 is &gt; 0.1) (Figure <a href="arrays.html#fig:SWSqcEPICpropSites">3.42</a>).
No samples were excluded for having a high proportion of probes with low bead counts (proportion of probes with bead number &lt; 3 is &gt; 0.1), (Figure <a href="arrays.html#fig:SWSqcEPICbeadNum">3.43</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:SWSqcEPICmethVsUnmeth"></span>
<img src="figs/SWSqcEPICmethVsUnmeth.png" alt="Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing \(3\sigma\) from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.41: Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing <span class="math inline">\(3\sigma\)</span> from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:SWSqcEPICpropSites"></span>
<img src="figs/SWSqcEPICpropSites.png" alt="Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.42: Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:SWSqcEPICbeadNum"></span>
<img src="figs/SWSqcEPICbeadNum.png" alt="Proportion of probes with a bead count of &lt; 3 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.43: Proportion of probes with a bead count of &lt; 3 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

</div>
<div id="probe-qc-2" class="section level4">
<h4><span class="header-section-number">3.4.3.2</span> Probe QC</h4>
<p>There were no outliers within the control probes (Figure <a href="arrays.html#fig:SWSqcEPICctrlProbes">3.44</a>).
833 probes were excluded for having high background signal in a high proportion of samples (proportion of samples with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="arrays.html#fig:SWSqcEPICpropSamples">3.45</a>).
127 probes were excluded for having low bead count in a high proportion of samples (proportion of samples with bead number &lt; 3 is &gt; 0.1), (Figure <a href="arrays.html#fig:SWSqcEPICbeadNumAcross">3.46</a>).
Probes with poor technical quality were excluded from the analysis prior to functional normalisation.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSqcEPICctrlProbes"></span>
<img src="figs/SWSqcEPICctrlProbes.png" alt="Control probe signal by sample for each summary group for the EPIC data. Outliers would be circled in black. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.44: Control probe signal by sample for each summary group for the EPIC data. Outliers would be circled in black. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:SWSqcEPICpropSamples"></span>
<img src="figs/SWSqcEPICpropSamples.png" alt="Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.45: Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:SWSqcEPICbeadNumAcross"></span>
<img src="figs/SWSqcEPICbeadNumAcross.png" alt="Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.46: Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by <code>meffil</code> QC report.
</p>
</div>

</div>
<div id="functional-normalisation-3" class="section level4">
<h4><span class="header-section-number">3.4.3.3</span> Functional Normalisation</h4>
<p>An <em>m</em> of 12 was chosen as this value produced the last steep drop in residual variation, see Figure <a href="arrays.html#fig:SWSfunnormPCsEPIC">3.47</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSfunnormPCsEPIC"></span>
<img src="figs/SWSfunnormPCsEPIC.png" alt="Residual variation remaining after functional normalisation of the top 20,000 most variable probes with m PCs from the control probe summary matrices for the EPIC array samples (n=464), for M = methylated and U = unmethylated probes." width="80%" />
<p class="caption">
Figure 3.47: Residual variation remaining after functional normalisation of the top 20,000 most variable probes with <em>m</em> PCs from the control probe summary matrices for the EPIC array samples (n=464), for M = methylated and U = unmethylated probes.
</p>
</div>

</div>
<div id="ewas-1" class="section level4">
<h4><span class="header-section-number">3.4.3.4</span> EWAS</h4>
<p>For all the EWAS, blood cell-type counts were estimated using the Houseman method <span class="citation">[<a href="#ref-Houseman2012" role="doc-biblioref">146</a>]</span> and the cord blood cell-type reference panel from Bakulski et al. <span class="citation">[<a href="#ref-Bakulski2016" role="doc-biblioref">212</a>]</span>.
The Cell-types estimated were: B cells, CD4+ T cells, CD8+ T Cells, Granulocytes, Monocytes, Natural Killer cells, &amp; Erythrocytes.
In addition to the estimated blood cell counts all models included as covariates: Maternal Age at time of birth (years), Sex, maternal BMI at 11 weeks gestation, parity, whether or not the mother smoked during pregnancy, and gestational age.</p>
<div id="total-bone-mineral-conent-at-8-years" class="section level5">
<h5><span class="header-section-number">3.4.3.4.1</span> Total Bone Mineral Conent at 8 years</h5>
<p>Figure <a href="arrays.html#fig:SWSttotbmcwhasa">3.48</a>, illustrates the distribution of bone mineral content at 8 years in the 408 individuals in this EWAS.
Surrogate variable analysis identified 95 significant surrogate variables, this is likely an overestimate stemming from small amounts of variation remaining unaccounted for by the manual model thus the Manhattan plots based on the manual model were included.
No probes fell below the Bonferroni corrected significance threshold (<span class="math inline">\(5.92\times10^{-8}\)</span>) for an association between DNA methylation at that locus and total bone mineral content minus head at 8 years adjusted for age and sex, Figure <a href="arrays.html#fig:SWSttotbmcwhasaEPICewasManhattanSVA">3.49</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSttotbmcwhasa"></span>
<img src="figs/SWS8yrBMCdist.png" alt="Distribution of whole body (minus head) bone mineral content in kg at 8 years of age (n = 408), adjusted for sex and age, as measured by DXA." width="80%" />
<p class="caption">
Figure 3.48: Distribution of whole body (minus head) bone mineral content in kg at 8 years of age (n = 408), adjusted for sex and age, as measured by DXA.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:SWSttotbmcwhasaEPICewasManhattanSVA"></span>
<img src="figs/SWSttotbmcwhasaEPICewasManhattanAll.png" alt="Results of EWAS for whole body (minus head) bone mineral content in kg at 8 years of age (n = 408), adjusted for sex and age. The ‘all’ model results are shown here. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(5.92\times10^{-8}\)." width="80%" />
<p class="caption">
Figure 3.49: Results of EWAS for whole body (minus head) bone mineral content in kg at 8 years of age (n = 408), adjusted for sex and age. The ‘all’ model results are shown here. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(5.92\times10^{-8}\)</span>.
</p>
</div>

</div>
<div id="total-bone-mineral-conent-at-6-years" class="section level5">
<h5><span class="header-section-number">3.4.3.4.2</span> Total Bone Mineral Conent at 6 years</h5>
<p>Figure <a href="arrays.html#fig:SWSstotbmcwhasa">3.50</a>, illustrates the distribution of bone mineral content at 6 years in the 402 individuals in this EWAS.
Surrogate variable analysis identified 97 significant surrogate variables, this is likely an overestimate stemming from small amounts of variation remaining unaccounted for by the manual model thus Manhattan plots based on the manual model have been included.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSstotbmcwhasa"></span>
<img src="figs/SWS6yrBMCdist.png" alt="Distribution of whole body (minus head) bone mineral content in kg at 8 years of age (n = 402), adjusted for sex and age, as measured by DXA." width="80%" />
<p class="caption">
Figure 3.50: Distribution of whole body (minus head) bone mineral content in kg at 8 years of age (n = 402), adjusted for sex and age, as measured by DXA.
</p>
</div>

<p>One probe fell below the Bonferroni corrected significance threshold (<span class="math inline">\(5.92\times10^{-8}\)</span>) for an association between DNA methylation at that locus and total bone mineral content minus head at 6 years adjusted for age and sex, Figure <a href="arrays.html#fig:SWSstotbmcwhasaEPICewasManhattanSVA">3.51</a>. This probe was cg26559250 which is located at Chr6:157,653,445-157,653,447 adjacent to the ZDHHC14 (zinc finger DHHC-type palmitoyltransferase 14) gene. cg26559250 was significant (p = <span class="math inline">\(2.52\times 10^{-8}\)</span>, increase of 1.46% per kg) in the ‘all’ model and was also Bonferroni significant in the uncorrected model. However, cg26559250 was not significant in the SVA or iSVA models suggesting that it may be attributable to batch or cell-type effects.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSstotbmcwhasaEPICewasManhattanSVA"></span>
<img src="figs/SWSstotbmcwhasaEPICewasManhattanAll.png" alt="Results of EWAS for whole body (minus head) bone mineral content in kg at 6 years of age (n = 402), adjusted for sex and age. The ‘all’ model results are shown here. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(5.92\times10^{-8}\)." width="80%" />
<p class="caption">
Figure 3.51: Results of EWAS for whole body (minus head) bone mineral content in kg at 6 years of age (n = 402), adjusted for sex and age. The ‘all’ model results are shown here. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(5.92\times10^{-8}\)</span>.
</p>
</div>

</div>
<div id="periosteal-circumference-at-6-years" class="section level5">
<h5><span class="header-section-number">3.4.3.4.3</span> Periosteal Circumference at 6 years</h5>
<p>Figure <a href="arrays.html#fig:SWSs3peric">3.52</a>, illustrates the distribution of periosteal circumference at 38% from the distal end of the tibia at 6 years (mm) in the 141 individuals in this EWAS.
Surrogate variable analysis identified 37 significant surrogate variables, this is likely an overestimate stemming from small amounts of variation remaining unaccounted for by the manual model thus Manhattan plots based on the manual model have been included.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSs3peric"></span>
<img src="figs/SWSs3peric.png" alt="Distribution of periosteal circumference at 38% from the distal end of the tibia (mm) at 6 years of age (n = 141), adjusted for sex and age, as measured by PQCT." width="80%" />
<p class="caption">
Figure 3.52: Distribution of periosteal circumference at 38% from the distal end of the tibia (mm) at 6 years of age (n = 141), adjusted for sex and age, as measured by PQCT.
</p>
</div>

<p>One probe fell below the Bonferroni corrected significance threshold for an association between DNA methylation at that locus and periosteal circumference at 38% from the distal end of the tibia at 6 years (mm) adjusted for age and sex, Figure <a href="arrays.html#fig:SWSs3pericEPICewasManhattanSVA">3.53</a>. This probes was cg22570676 which is located at Chr19:2,527,492-2,527,494 at the GNG7 (G protein subunit gamma 7) gene. cg22570676 was significant (p = <span class="math inline">\(4.24\times 10^{-8}\)</span>, increase of 0.370% per mm) in the ‘all’ model and was also Bonferroni significant in the uncorrected model. However, cg22570676 was not significant in the SVA or iSVA models suggesting that it may be attributable to batch or cell-type effects.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSs3pericEPICewasManhattanSVA"></span>
<img src="figs/SWSs3pericEPICewasManhattanAll.png" alt="Results of EWAS for periosteal circumference at 38% from the distal end of the tibia (mm) at 6 years of age (n = 141), adjusted for sex and age. The ‘all’ model results are shown here. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(5.92\times10^{-8}\)" width="80%" />
<p class="caption">
Figure 3.53: Results of EWAS for periosteal circumference at 38% from the distal end of the tibia (mm) at 6 years of age (n = 141), adjusted for sex and age. The ‘all’ model results are shown here. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(5.92\times10^{-8}\)</span>
</p>
</div>

</div>
<div id="cortical-denisty-at-6-years" class="section level5">
<h5><span class="header-section-number">3.4.3.4.4</span> Cortical Denisty at 6 years</h5>
<p>Figure <a href="arrays.html#fig:SWSs3crtden">3.54</a>, illustrates the distribution of cortical density at 38% from the distal end of the tibia at 6 years (<span class="math inline">\(mg~cm^{-3}\)</span>) in the 141 individuals in this EWAS.
No probes fell below the Bonferroni corrected significance threshold (<span class="math inline">\(5.92\times10^{-8}\)</span>) for an association between DNA methylation at that locus and cortical density at 6 years, Figure <a href="arrays.html#fig:SWSs3crtdenEPICewasManhattanSVA">3.55</a>.
Surrogate variable analysis identified 37 significant surrogate variables, this is likely an overestimate stemming from small amounts of variation remaining unaccounted for by the manual model thus Manhattan plots based on the manual model.</p>
<div class="figure" style="text-align: center"><span id="fig:SWSs3crtden"></span>
<img src="figs/SWSs3crtden.png" alt="Distribution of cortical density at 38% from the distal end of the tibia (\(mg~cm^{-3}\)) at 6 years of age (n = 141), as measured by PQCT." width="80%" />
<p class="caption">
Figure 3.54: Distribution of cortical density at 38% from the distal end of the tibia (<span class="math inline">\(mg~cm^{-3}\)</span>) at 6 years of age (n = 141), as measured by PQCT.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:SWSs3crtdenEPICewasManhattanSVA"></span>
<img src="figs/SWSs3crtdenEPICewasManhattanAll.png" alt="Results of EWAS for cortical density at 38% from the distal end of the tibia (\(mg~cm^{-3}\)) at 6 years of age (n = 141). The ‘all’ model results are shown here. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of \(5.92\times10^{-8}\)" width="80%" />
<p class="caption">
Figure 3.55: Results of EWAS for cortical density at 38% from the distal end of the tibia (<span class="math inline">\(mg~cm^{-3}\)</span>) at 6 years of age (n = 141). The ‘all’ model results are shown here. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. The x axis represents chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(5.92\times10^{-8}\)</span>
</p>
</div>

</div>
</div>
</div>
</div>
<div id="discussion" class="section level2">
<h2><span class="header-section-number">3.5</span> Discussion</h2>
<p>EWAS for 9 outcomes were carried out across 3 sets of samples from MAVIDOS and SWS using the EPIC and 450K array platforms.
No significant results were found in either the first or second phase of the MAVIDOS analysis.
Two possible results for bone outcomes at 6 years were identified in the SWS data but did not remain in models including surrogate variables for possible confounding effects.
The effect observed at probe cg26559250, adjacent to the ZDHHC14 gene, in EWAS for bone mineral content at 6 years was not seen at the 8 year time point.
Furthermore both this result and the finding for probe cg22570676 and periosteal circumference were of small effect sizes 1.46% per kg, 0.370% per mm respectively.
Consequently these results should be treated with considerable caution, they are in need of replication before they can be considered reliable.
No Bonferroni significant DNA methylation changes at the <em>CDKN2A</em> and <em>RXRA</em> loci which have been previously associated with maternal vitamin D and bone phenotypes were identified, despite the presence of 95 and 75 probes annotated as being in the vicinity of these genes respectively.</p>
<p>Whilst the calculation of power for EWAS is a complex and rather understudied problem <span class="citation">[<a href="#ref-Wang2011" role="doc-biblioref">219</a>–<a href="#ref-Graw2019" role="doc-biblioref">221</a>]</span> it is possible to achieve some approximations using Cohen’s methods <span class="citation">[<a href="#ref-Cohen1988" role="doc-biblioref">222</a>]</span>.
To achieve the modest goal of 80% power for a small effect size (<span class="math inline">\(r^2 = 0.02\)</span>) in a linear regression analysis (F-test) with 7 covariates at a significance level suitable for the EPIC array of p = <span class="math inline">\(5.92\times10^{-8}\)</span> an n of 2607 is needed.
Seven was the smallest number of covariates used in an SVA model in these analyses, models in the SWS analyses had 13 manually specified variables.
When considering what is for EWAS a very large effect size (<span class="math inline">\(r^2 = 0.15\)</span>) with the 13 covariates used in the SWS models it is possible to achieve 80% power for an n of 374.
This is an effect size in line with the effects of smoking on DNA methylation at some loci <span class="citation">[<a href="#ref-Joubert2016" role="doc-biblioref">223</a>,<a href="#ref-Breton2017" role="doc-biblioref">224</a>]</span>.
This would make the two largest EWAS performed here for BMC at 8 and 6 years (n = 408, n = 402 respectively) powered only to find large effect sizes with just over 80% probability.
The most generous set of parameters (80% power, <span class="math inline">\(r^2 = 0.15\)</span>, p = <span class="math inline">\(5.92\times10^{-8}\)</span>, &amp; 2 covariates) yield an n of 259, more realistic numbers (90% power, <span class="math inline">\(r^2 = 0.02\)</span>, p = <span class="math inline">\(5.92\times10^{-8}\)</span>, &amp; 13 covariates) yield an n of 3370.
EWAS have identified biologically relevant changes associated with environmental exposures in DNA methylation with magnitudes of less than a single percentage point, and percentage changes in the low single digits are not uncommon in EWAS <span class="citation">[<a href="#ref-Breton2017" role="doc-biblioref">224</a>]</span>.
This makes all of the EWAS performed here underpowered to identify small DNA methylation changes which might be expected to occur.
A collaboration with colleagues in Bristol is underway to perform a meta-analysis to include these data with similar results from other cohorts in order to increase the power of these analyses.
The covariates included in the models for the second phase of MAVIDOS analysis and the SWS analysis are matched to those being used by our collaborators to maximise the comparability of our results.</p>
<p>Given that the effect of maternal vitamin D on neonatal bone mass appears to be seasonal, with only babies born in the winter months showing statistically significant benefits of supplementation <span class="citation">[<a href="#ref-Cooper2016" role="doc-biblioref">153</a>]</span>, it would be interesting to perform seasonally stratified EWAS were sufficient numbers available to do so with reasonable power.</p>
<p>Attempting to identify small changes in the overall DNA methylation state of complex populations of cells like blood and umbilical cord tissue which are associated with phenotypes such as circulating maternal vitamin D is a technically challenging undertaking.
This work provides two candidates for further analysis for associations between DNA methylation and bone health outcomes at 6 years of age.
Furthermore, these results have contributed to a larger meta analysis with greater power to detect associations with metrics of bone health.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Bell2017a">
<p>67. Bell CG, Gao F, Yuan W, Roos L, Acton RJ, Xia Y, Bell J, Ward K, Mangino M, Hysi PG, Wang J, Spector TD: <strong>Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci</strong>. <em>Nature Communications</em> 2018, <strong>9</strong>:8<a href="https://doi.org/10.1038/s41467-017-01586-1">10.1038/s41467-017-01586-1</a>Available: <a href="http://www.nature.com/articles/s41467-017-01586-1">http://www.nature.com/articles/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Jaffe2014">
<p>145. Jaffe AE, Irizarry RA: <strong>Accounting for cellular heterogeneity is critical in epigenome-wide association studies.</strong> <em>Genome biology</em> 2014, <strong>15</strong>:R31<a href="https://doi.org/10.1186/gb-2014-15-2-r31">10.1186/gb-2014-15-2-r31</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24495553">http://www.ncbi.nlm.nih.gov/pubmed/24495553</a>.</p>
</div>
<div id="ref-Houseman2012">
<p>146. Houseman EA, Accomando WP, Koestler DC, Christensen BC, Marsit CJ, Nelson HH, Wiencke JK, Kelsey KT: <strong>DNA methylation arrays as surrogate measures of cell mixture distribution</strong>. <em>BMC Bioinformatics</em> 2012, <strong>13</strong>:86<a href="https://doi.org/10.1186/1471-2105-13-86">10.1186/1471-2105-13-86</a>Available: <a href="http://www.biomedcentral.com/1471-2105/13/86">http://www.biomedcentral.com/1471-2105/13/86</a>.</p>
</div>
<div id="ref-Harvey2012a">
<p>152. Harvey NC, Javaid K, Bishop N, Kennedy S, Papageorghiou AT, Fraser R, Gandhi SV, Schoenmakers I, Prentice A, Cooper C: <strong>MAVIDOS Maternal Vitamin D Osteoporosis Study: study protocol for a randomized controlled trial. The MAVIDOS Study Group</strong>. <em>Trials</em> 2012, <strong>13</strong>:13<a href="https://doi.org/10.1186/1745-6215-13-13">10.1186/1745-6215-13-13</a>Available: <a href="http://trialsjournal.biomedcentral.com/articles/10.1186/1745-6215-13-13">http://trialsjournal.biomedcentral.com/articles/10.1186/1745-6215-13-13</a>.</p>
</div>
<div id="ref-Cooper2016">
<p>153. Cooper C, Harvey NC, Bishop NJ, Kennedy S, Papageorghiou AT, Schoenmakers I, Fraser R, Gandhi SV, Carr A, D’Angelo S, Crozier SR, Moon RJ, Arden NK, Dennison EM, Godfrey KM, Inskip HM, Prentice A, Mughal MZ, Eastell R, Reid DM, Javaid MK: <strong>Maternal gestational vitamin D supplementation and offspring bone health (MAVIDOS): a multicentre, double-blind, randomised placebo-controlled trial</strong>. <em>The Lancet Diabetes &amp; Endocrinology</em> 2016, <strong>4</strong>:393–402<a href="https://doi.org/10.1016/S2213-8587(16)00044-9">10.1016/S2213-8587(16)00044-9</a>Available: <a href="http://dx.doi.org/10.1016/S2213-8587(16)00044-9">http://dx.doi.org/10.1016/S2213-8587(16)00044-9</a>.</p>
</div>
<div id="ref-Fortin2014">
<p>163. Fortin J-P, Labbe A, Lemire M, Zanke BW, Hudson TJ, Fertig EJ, Greenwood CMT, Hansen KD: <strong>Functional normalization of 450k methylation array data improves replication in large cancer studies</strong>. <em>Genome Biology</em> 2014, <strong>15</strong>:503<a href="https://doi.org/10.1186/s13059-014-0503-2">10.1186/s13059-014-0503-2</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0503-2">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0503-2</a>.</p>
</div>
<div id="ref-Pidsley2016">
<p>164. Pidsley R, Zotenko E, Peters TJ, Lawrence MG, Risbridger GP, Molloy P, Van Djik S, Muhlhausler B, Stirzaker C, Clark SJ: <strong>Critical evaluation of the Illumina MethylationEPIC BeadChip microarray for whole-genome DNA methylation profiling.</strong> <em>Genome biology</em> 2016, <strong>17</strong>:208<a href="https://doi.org/10.1186/s13059-016-1066-1">10.1186/s13059-016-1066-1</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27717381">http://www.ncbi.nlm.nih.gov/pubmed/27717381</a>.</p>
</div>
<div id="ref-Zhou2017">
<p>165. Zhou W, Laird PW, Shen H: <strong>Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes</strong>. <em>Nucleic Acids Research</em> 2016, <strong>45</strong>:gkw967<a href="https://doi.org/10.1093/nar/gkw967">10.1093/nar/gkw967</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27924034">http://www.ncbi.nlm.nih.gov/pubmed/27924034</a>.</p>
</div>
<div id="ref-Andrews2016">
<p>166. Andrews SV, Ladd-Acosta C, Feinberg AP, Hansen KD, Fallin MD: <strong>“Gap hunting” to characterize clustered probe signals in Illumina methylation array data</strong>. <em>Epigenetics &amp; Chromatin</em> 2016, <strong>9</strong>:56<a href="https://doi.org/10.1186/s13072-016-0107-z">10.1186/s13072-016-0107-z</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z</a>.</p>
</div>
<div id="ref-VanStaa2001">
<p>178. Staa TP van, Dennison EM, Leufkens HGM, Cooper C: <strong>Epidemiology of fractures in England and Wales</strong>. <em>Bone</em> 2001, <strong>29</strong>:517–522<a href="https://doi.org/10.1016/S8756-3282(01)00614-7">10.1016/S8756-3282(01)00614-7</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S8756328201006147">https://linkinghub.elsevier.com/retrieve/pii/S8756328201006147</a>.</p>
</div>
<div id="ref-Streubel2011">
<p>179. Streubel PN, Ricci WM, Wong A, Gardner MJ: <strong>Mortality After Distal Femur Fractures in Elderly Patients</strong>. <em>Clinical Orthopaedics and Related Research</em> 2011, <strong>469</strong>:1188–1196<a href="https://doi.org/10.1007/s11999-010-1530-2">10.1007/s11999-010-1530-2</a>Available: <a href="http://link.springer.com/10.1007/s11999-010-1530-2">http://link.springer.com/10.1007/s11999-010-1530-2</a>.</p>
</div>
<div id="ref-Thompson2018">
<p>180. Thompson E, Greenspan S: <em>National Osteoporosis Foundation Annual Report (2018)</em>. National Osteoporosis Foundation; 2018 Available: <a href="https://cdn.nof.org/wp-content/uploads/2018%7B\_%7DNOF%7B\_%7DAnnual%7B\_%7Dreport%7B\_%7DFINAL.pdf">https://cdn.nof.org/wp-content/uploads/2018{\_}NOF{\_}Annual{\_}report{\_}FINAL.pdf</a>.</p>
</div>
<div id="ref-Hernandez2003">
<p>181. Hernandez CJ, Beaupré GS, Carter DR: <strong>A theoretical analysis of the relative influences of peak BMD, age-related bone loss and menopause on the development of osteoporosis</strong>. <em>Osteoporosis International</em> 2003, <strong>14</strong>:843–847<a href="https://doi.org/10.1007/s00198-003-1454-8">10.1007/s00198-003-1454-8</a>.</p>
</div>
<div id="ref-Nguyen2005">
<p>182. Nguyen TV, Center JR, Eisman JA: <strong>Femoral Neck Bone Loss Predicts Fracture Risk Independent of Baseline BMD</strong>. <em>Journal of Bone and Mineral Research</em> 2005, <strong>20</strong>:1195–1201<a href="https://doi.org/10.1359/JBMR.050215">10.1359/JBMR.050215</a>Available: <a href="http://doi.wiley.com/10.1359/JBMR.050215">http://doi.wiley.com/10.1359/JBMR.050215</a>.</p>
</div>
<div id="ref-Hui1990">
<p>184. Hui SL, Slemenda CW, Johnston CC: <strong>The contribution of bone loss to postmenopausal osteoporosis</strong>. <em>Osteoporosis International</em> 1990, <strong>1</strong>:30–34<a href="https://doi.org/10.1007/BF01880413">10.1007/BF01880413</a>.</p>
</div>
<div id="ref-Harvey2014b">
<p>185. Harvey N, Dennison E, Cooper C: <strong>Osteoporosis: A Lifecourse Approach</strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:1917–1925<a href="https://doi.org/10.1002/jbmr.2286">10.1002/jbmr.2286</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2286">http://doi.wiley.com/10.1002/jbmr.2286</a>.</p>
</div>
<div id="ref-Bikle2012">
<p>186. Bikle DD: <strong>Vitamin D and Bone</strong>. <em>Current Osteoporosis Reports</em> 2012, <strong>10</strong>:151–159<a href="https://doi.org/10.1007/s11914-012-0098-z">10.1007/s11914-012-0098-z</a>Available: <a href="http://link.springer.com/10.1007/s11914-012-0098-z">http://link.springer.com/10.1007/s11914-012-0098-z</a>.</p>
</div>
<div id="ref-Christakos2011">
<p>187. Christakos S, Dhawan P, Porta A, Mady LJ, Seth T: <strong>Vitamin D and intestinal calcium absorption</strong>. <em>Molecular and Cellular Endocrinology</em> 2011, <strong>347</strong>:25–29<a href="https://doi.org/10.1016/j.mce.2011.05.038">10.1016/j.mce.2011.05.038</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0303720711002930">https://linkinghub.elsevier.com/retrieve/pii/S0303720711002930</a>.</p>
</div>
<div id="ref-Fukumoto2014">
<p>188. Fukumoto S: <strong>Phosphate metabolism and vitamin D</strong>. <em>BoneKEy Reports</em> 2014, <strong>3</strong>:1–5<a href="https://doi.org/10.1038/bonekey.2013.231">10.1038/bonekey.2013.231</a>Available: <a href="http://dx.doi.org/10.1038/bonekey.2013.231">http://dx.doi.org/10.1038/bonekey.2013.231</a>.</p>
</div>
<div id="ref-Baird2011">
<p>189. Baird J, Kurshid MA, Kim M, Harvey N, Dennison E, Cooper C: <strong>Does birthweight predict bone mass in adulthood? A systematic review and meta-analysis</strong>. <em>Osteoporosis International</em> 2011, <strong>22</strong>:1323–1334<a href="https://doi.org/10.1007/s00198-010-1344-9">10.1007/s00198-010-1344-9</a>.</p>
</div>
<div id="ref-Cooper2001">
<p>190. Cooper C, Eriksson JG, Forsén T, Osmond C, Tuomilehto J, Barker DJP: <strong>Maternal height, childhood growth and risk of hip fracture in later life: A longitudinal study</strong>. <em>Osteoporosis International</em> 2001, <strong>12</strong>:623–629<a href="https://doi.org/10.1007/s001980170061">10.1007/s001980170061</a>.</p>
</div>
<div id="ref-Inskip2006">
<p>191. Inskip HM, Godfrey KM, Robinson SM, Law CM, Jp D: <strong>Europe PMC Funders Group Cohort Profile : The Southampton Women ’ s Survey</strong>. <em>International Journal of Epidemiology</em> 2006, <strong>35</strong>:42–48<a href="https://doi.org/10.1093/ije/dyi202">10.1093/ije/dyi202</a>.</p>
</div>
<div id="ref-Harvey2010a">
<p>192. Harvey NC, Mahon PA, Robinson SM, Nisbet CE, Javaid MK, Crozier SR, Inskip HM, Godfrey KM, Arden NK, Dennison EM, Cooper C, Taylor P, Greenaway LJ, Hanson M, Barker DJP, Law CM: <strong>Different indices of fetal growth predict bone size and volumetric density at 4 years of age</strong>. <em>Journal of Bone and Mineral Research</em> 2010, <strong>25</strong>:920–927<a href="https://doi.org/10.1359/jbmr.091022">10.1359/jbmr.091022</a>.</p>
</div>
<div id="ref-Harvey2013">
<p>193. Harvey NC, Cole ZA, Crozier SR, Ntani G, Mahon PA, Robinson SM, Inskip HM, Godfrey KM, Dennison EM, Cooper C: <strong>Fetal and infant growth predict hip geometry at 6 y old: Findings from the Southampton Women’s Survey</strong>. <em>Pediatric Research</em> 2013, <strong>74</strong>:450–456<a href="https://doi.org/10.1038/pr.2013.119">10.1038/pr.2013.119</a>.</p>
</div>
<div id="ref-Mahon2010a">
<p>194. Mahon P, Harvey N, Crozier S, Inskip H, Robinson S, Arden N, Swaminathan R, Cooper C, Godfrey K: <strong>Low maternal vitamin D status and fetal bone development: Cohort study</strong>. <em>Journal of Bone and Mineral Research</em> 2010, <strong>25</strong>:14–19<a href="https://doi.org/10.1359/jbmr.090701">10.1359/jbmr.090701</a>Available: <a href="http://doi.wiley.com/10.1359/jbmr.090701">http://doi.wiley.com/10.1359/jbmr.090701</a>.</p>
</div>
<div id="ref-Viljakainen2010">
<p>195. Viljakainen HT, Saarnio E, Hytinantti T, Miettinen M, Surcel H, Mäkitie O, Andersson S, Laitinen K, Lamberg-Allardt C: <strong>Maternal Vitamin D Status Determines Bone Variables in the Newborn</strong>. <em>The Journal of Clinical Endocrinology &amp; Metabolism</em> 2010, <strong>95</strong>:1749–1757<a href="https://doi.org/10.1210/jc.2009-1391">10.1210/jc.2009-1391</a>Available: <a href="https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2009-1391">https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2009-1391</a>.</p>
</div>
<div id="ref-Viljakainen2011">
<p>196. Viljakainen HT, Korhonen T, Hytinantti T, Laitinen EKA, Andersson S, Mäkitie O, Lamberg-Allardt C: <strong>Maternal vitamin D status affects bone growth in early childhood—a prospective cohort study</strong>. <em>Osteoporosis International</em> 2011, <strong>22</strong>:883–891<a href="https://doi.org/10.1007/s00198-010-1499-4">10.1007/s00198-010-1499-4</a>Available: <a href="http://link.springer.com/10.1007/s00198-010-1499-4">http://link.springer.com/10.1007/s00198-010-1499-4</a>.</p>
</div>
<div id="ref-Zhu2014a">
<p>197. Zhu K, Whitehouse AJ, Hart PH, Kusel M, Mountain J, Lye S, Pennell C, Walsh JP: <strong>Maternal Vitamin D Status During Pregnancy and Bone Mass in Offspring at 20 Years of Age: A Prospective Cohort Study</strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:1088–1095<a href="https://doi.org/10.1002/jbmr.2138">10.1002/jbmr.2138</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2138">http://doi.wiley.com/10.1002/jbmr.2138</a>.</p>
</div>
<div id="ref-Kip2004">
<p>198. Kip SN, Strehler EE: <strong>Vitamin D 3 upregulates plasma membrane Ca 2+ -ATPase expression and potentiates apico-basal Ca 2+ flux in MDCK cells</strong>. <em>American Journal of Physiology-Renal Physiology</em> 2004, <strong>286</strong>:F363–F369<a href="https://doi.org/10.1152/ajprenal.00076.2003">10.1152/ajprenal.00076.2003</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/14583431">http://www.ncbi.nlm.nih.gov/pubmed/14583431</a>.</p>
</div>
<div id="ref-Burdge2007a">
<p>199. Burdge GC, Slater-Jefferies J, Torrens C, Phillips ES, Hanson MA, Lillycrop KA: <strong>Dietary protein restriction of pregnant rats in the F 0 generation induces altered methylation of hepatic gene promoters in the adult male offspring in the F 1 and F 2 generations</strong>. <em>British Journal of Nutrition</em> 2007, <strong>97</strong>:435–439<a href="https://doi.org/10.1017/S0007114507352392">10.1017/S0007114507352392</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507352392/type/journal%7B\_%7Darticle">https://www.cambridge.org/core/product/identifier/S0007114507352392/type/journal{\_}article</a>.</p>
</div>
<div id="ref-Burdge2007">
<p>201. Burdge GC, Hanson MA, Slater-Jefferies JL, Lillycrop KA: <strong>Epigenetic regulation of transcription: a mechanism for inducing variations in phenotype (fetal programming) by differences in nutrition during early life?</strong> <em>British Journal of Nutrition</em> 2007, <strong>97</strong>:1036–1046<a href="https://doi.org/10.1017/S0007114507682920">10.1017/S0007114507682920</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507682920/type/journal%7B\_%7Darticle">https://www.cambridge.org/core/product/identifier/S0007114507682920/type/journal{\_}article</a>.</p>
</div>
<div id="ref-Holroyd2012">
<p>202. Holroyd C, Harvey N, Dennison E, Cooper C: <strong>Epigenetic influences in the developmental origins of osteoporosis</strong>. <em>Osteoporosis International</em> 2012, <strong>23</strong>:401–410<a href="https://doi.org/10.1007/s00198-011-1671-5">10.1007/s00198-011-1671-5</a>.</p>
</div>
<div id="ref-Harvey2014d">
<p>203. Harvey NC, Sheppard A, Godfrey KM, McLean C, Garratt E, Ntani G, Davies L, Murray R, Inskip HM, Gluckman PD, Hanson MA, Lillycrop KA, Cooper C: <strong>Childhood Bone Mineral Content Is Associated With Methylation Status of the RXRA Promoter at Birth</strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:600–607<a href="https://doi.org/10.1002/jbmr.2056">10.1002/jbmr.2056</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2056">http://doi.wiley.com/10.1002/jbmr.2056</a>.</p>
</div>
<div id="ref-Curtis2017">
<p>204. Curtis EM, Murray R, Titcombe P, Cook E, Clarke-Harris R, Costello P, Garratt E, Holbrook JD, Barton S, Inskip H, Godfrey KM, Bell CG, Cooper C, Lillycrop KA, Harvey NC: <strong>Perinatal DNA Methylation at CDKN2A Is Associated With Offspring Bone Mass: Findings From the Southampton Women’s Survey</strong>. <em>Journal of Bone and Mineral Research</em> 2017, <strong>32</strong>:2030–2040<a href="https://doi.org/10.1002/jbmr.3153">10.1002/jbmr.3153</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.3153">http://doi.wiley.com/10.1002/jbmr.3153</a>.</p>
</div>
<div id="ref-Holland2017">
<p>205. Holland ML, Lowe R, Caton PW, Gemma C, Carbajosa G, Danson AF, Carpenter AAM, Loche E, Ozanne SE, Rakyan VK: <strong>Early-life nutrition modulates the epigenetic state of specific rDNA genetic variants in mice</strong>. <em>Science</em> 2016, <strong>353</strong>:495–498<a href="https://doi.org/10.1126/science.aaf7040">10.1126/science.aaf7040</a>Available: <a href="http://www.sciencemag.org/lookup/doi/10.1126/science.aaf7040">http://www.sciencemag.org/lookup/doi/10.1126/science.aaf7040</a>.</p>
</div>
<div id="ref-Dogan2017">
<p>206. Dogan MV, Beach SRH, Philibert RA: <strong>Genetically contextual effects of smoking on genome wide DNA methylation</strong>. <em>American Journal of Medical Genetics Part B: Neuropsychiatric Genetics</em> 2017, <strong>174</strong>:595–607<a href="https://doi.org/10.1002/ajmg.b.32565">10.1002/ajmg.b.32565</a>Available: <a href="http://doi.wiley.com/10.1002/ajmg.b.32565">http://doi.wiley.com/10.1002/ajmg.b.32565</a>.</p>
</div>
<div id="ref-Dai2016">
<p>207. Dai L, Mehta A, Mordukhovich I, Just AC, Shen J, Hou L, Koutrakis P, Sparrow D, Vokonas PS, Baccarelli AA, Schwartz JD: <strong>Differential DNA methylation and PM 2.5 species in a 450K epigenome-wide association study</strong>. <em>Epigenetics</em> 2017, <strong>12</strong>:139–148<a href="https://doi.org/10.1080/15592294.2016.1271853">10.1080/15592294.2016.1271853</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2016.1271853">https://www.tandfonline.com/doi/full/10.1080/15592294.2016.1271853</a>.</p>
</div>
<div id="ref-Wahl2016">
<p>208. Wahl S, Drong A, Lehne B, Loh M, Scott WR, Kunze S, Tsai P-C, Ried JS, Zhang W, Yang Y, Tan S, Fiorito G, Franke L, Guarrera S, Kasela S, Kriebel J, Richmond RC, Adamo M, Afzal U, Ala-Korpela M, Albetti B, Ammerpohl O, Apperley JF, Beekman M, Bertazzi PA, Black SL, Blancher C, Bonder M-J, Brosch M, Carstensen-Kirberg M, et al.: <strong>Epigenome-wide association study of body mass index, and the adverse outcomes of adiposity.</strong> <em>Nature</em> 2017, <strong>541</strong>:81–86<a href="https://doi.org/10.1038/nature20784">10.1038/nature20784</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28002404">http://www.ncbi.nlm.nih.gov/pubmed/28002404</a>.</p>
</div>
<div id="ref-Min2017">
<p>209. Min J, Hemani G, Smith GD, Relton CL, Suderman M: <strong>Meffil: efficient normalisation and analysis of very large DNA methylation samples</strong>. <em>bioRxiv</em> 2017, <strong>44</strong>:125963<a href="https://doi.org/10.1101/125963">10.1101/125963</a>Available: <a href="https://www.biorxiv.org/content/early/2017/04/27/125963">https://www.biorxiv.org/content/early/2017/04/27/125963</a>.</p>
</div>
<div id="ref-Cardenas2016">
<p>210. Cardenas A, Allard C, Doyon M, Houseman EA, Bakulski KM, Perron P, Bouchard L, Hivert M-F: <strong>Validation of a DNA methylation reference panel for the estimation of nucleated cells types in cord blood.</strong> <em>Epigenetics</em> 2016, <strong>11</strong>:773–779<a href="https://doi.org/10.1080/15592294.2016.1233091">10.1080/15592294.2016.1233091</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27668573">http://www.ncbi.nlm.nih.gov/pubmed/27668573</a>.</p>
</div>
<div id="ref-Bakulski2016">
<p>212. Bakulski KM, Feinberg JI, Andrews SV, Yang J, Brown S, L. McKenney S, Witter F, Walston J, Feinberg AP, Fallin MD: <strong>DNA methylation of cord blood cell types: Applications for mixed cell birth studies</strong>. <em>Epigenetics</em> 2016, <strong>11</strong>:354–362<a href="https://doi.org/10.1080/15592294.2016.1161875">10.1080/15592294.2016.1161875</a>Available: <a href="http://dx.doi.org/10.1080/15592294.2016.1161875">http://dx.doi.org/10.1080/15592294.2016.1161875</a>.</p>
</div>
<div id="ref-Gervin2016">
<p>213. Gervin K, Page CM, Aass HCD, Jansen MA, Fjeldstad HE, Andreassen BK, Duijts L, Meurs JB van, Zelm MC van, Jaddoe VW, Nordeng H, Knudsen GP, Magnus P, Nystad W, Staff AC, Felix JF, Lyle R: <strong>Cell type specific DNA methylation in cord blood: A 450K-reference data set and cell count-based validation of estimated cell type composition</strong>. <em>Epigenetics</em> 2016, <strong>11</strong>:690–698<a href="https://doi.org/10.1080/15592294.2016.1214782">10.1080/15592294.2016.1214782</a>Available: <a href="http://dx.doi.org/10.1080/15592294.2016.1214782">http://dx.doi.org/10.1080/15592294.2016.1214782</a>.</p>
</div>
<div id="ref-Leek2007">
<p>214. Leek JT, Storey JD: <strong>Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis</strong>. <em>PLoS Genetics</em> 2007, <strong>3</strong>:e161<a href="https://doi.org/10.1371/journal.pgen.0030161">10.1371/journal.pgen.0030161</a>Available: <a href="https://dx.plos.org/10.1371/journal.pgen.0030161">https://dx.plos.org/10.1371/journal.pgen.0030161</a>.</p>
</div>
<div id="ref-Teschendorff2011">
<p>215. Teschendorff AE, Zhuang J, Widschwendter M: <strong>Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies</strong>. <em>Bioinformatics</em> 2011, <strong>27</strong>:1496–1505<a href="https://doi.org/10.1093/bioinformatics/btr171">10.1093/bioinformatics/btr171</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr171">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr171</a>.</p>
</div>
<div id="ref-McGregor2016">
<p>216. McGregor K, Bernatsky S, Colmegna I, Hudson M, Pastinen T, Labbe A, Greenwood CMT: <strong>An evaluation of methods correcting for cell-type heterogeneity in DNA methylation studies</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:84<a href="https://doi.org/10.1186/s13059-016-0935-y">10.1186/s13059-016-0935-y</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0935-y">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0935-y</a>.</p>
</div>
<div id="ref-Teschendorff2017">
<p>217. Teschendorff AE, Zheng SC: <strong>Cell-type deconvolution in epigenome-wide association studies: a review and recommendations</strong>. <em>Epigenomics</em> 2017, <strong>9</strong>:757–768<a href="https://doi.org/10.2217/epi-2016-0153">10.2217/epi-2016-0153</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi-2016-0153">https://www.futuremedicine.com/doi/10.2217/epi-2016-0153</a>.</p>
</div>
<div id="ref-Lin2018">
<p>218. Lin X, Tan JYL, Teh AL, Lim IY, Liew SJ, MacIsaac JL, Chong YS, Gluckman PD, Kobor MS, Cheong CY, Karnani N: <strong>Cell type-specific DNA methylation in neonatal cord tissue and cord blood: a 850K-reference panel and comparison of cell types</strong>. <em>Epigenetics</em> 2018, <strong>13</strong>:941–958<a href="https://doi.org/10.1080/15592294.2018.1522929">10.1080/15592294.2018.1522929</a>Available: <a href="https://doi.org/10.1080/15592294.2018.1522929">https://doi.org/10.1080/15592294.2018.1522929</a>.</p>
</div>
<div id="ref-Wang2011">
<p>219. Wang S: <strong>Method to detect differentially methylated loci with case-control designs using Illumina arrays</strong>. <em>Genetic Epidemiology</em> 2011, <strong>35</strong>:686–694<a href="https://doi.org/10.1002/gepi.20619">10.1002/gepi.20619</a>Available: <a href="http://doi.wiley.com/10.1002/gepi.20619">http://doi.wiley.com/10.1002/gepi.20619</a>.</p>
</div>
<div id="ref-Graw2019">
<p>221. Graw S, Henn R, Thompson JA, Koestler DC: <strong>PwrEWAS: A user-friendly tool for comprehensive power estimation for epigenome wide association studies (EWAS)</strong>. <em>BMC Bioinformatics</em> 2019, <strong>20</strong>:1–11<a href="https://doi.org/10.1186/s12859-019-2804-7">10.1186/s12859-019-2804-7</a>.</p>
</div>
<div id="ref-Cohen1988">
<p>222. Cohen J: <em>Statisticsl Power Analysis for the Behavioural Sciences</em>. 2nd ed. New York: Lawrence Erlbaum Associates; 1988.</p>
</div>
<div id="ref-Joubert2016">
<p>223. Joubert BR, Felix JF, Yousefi P, Bakulski KM, Just AC, Breton C, Reese SE, Markunas CA, Richmond RC, Xu C-J, Küpers LK, Oh SS, Hoyo C, Gruzieva O, Söderhäll C, Salas LA, Baïz N, Zhang H, Lepeule J, Ruiz C, Ligthart S, Wang T, Taylor JA, Duijts L, Sharp GC, Jankipersadsing SA, Nilsen RM, Vaez A, Fallin MD, Hu D, et al.: <strong>DNA Methylation in Newborns and Maternal Smoking in Pregnancy: Genome-wide Consortium Meta-analysis</strong>. <em>The American Journal of Human Genetics</em> 2016, <strong>98</strong>:680–696<a href="https://doi.org/10.1016/j.ajhg.2016.02.019">10.1016/j.ajhg.2016.02.019</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27040690">http://www.ncbi.nlm.nih.gov/pubmed/27040690</a>.</p>
</div>
<div id="ref-Breton2017">
<p>224. Breton CV, Marsit CJ, Faustman E, Nadeau K, Goodrich JM, Dolinoy DC, Herbstman J, Holland N, LaSalle JM, Schmidt R, Yousefi P, Perera F, Joubert BR, Wiemels J, Taylor M, Yang IV, Chen R, Hew KM, Freeland DMH, Miller R, Murphy SK: <strong>Small-Magnitude Effect Sizes in Epigenetic End Points are Important in Children’s Environmental Health Studies: The Children’s Environmental Health and Disease Prevention Research Center’s Epigenetics Working Group</strong>. <em>Environmental Health Perspectives</em> 2017, <strong>125</strong>:511–526<a href="https://doi.org/10.1289/EHP595">10.1289/EHP595</a>Available: <a href="https://ehp.niehs.nih.gov/doi/10.1289/EHP595">https://ehp.niehs.nih.gov/doi/10.1289/EHP595</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="methods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="tRNAs.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/03-EWAS.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
