<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 Epigenomics - Overview | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="3 Epigenomics - Overview | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 Epigenomics - Overview | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="the-biology-of-ageing.html"/>
<link rel="next" href="fundamentals-of-dna-methylation.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="the-ageing-population-and-the-burden-of-disease.html"><a href="the-ageing-population-and-the-burden-of-disease.html"><i class="fa fa-check"></i><b>1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="2" data-path="the-biology-of-ageing.html"><a href="the-biology-of-ageing.html"><i class="fa fa-check"></i><b>2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="3" data-path="epigenomics-overview.html"><a href="epigenomics-overview.html"><i class="fa fa-check"></i><b>3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="4" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html"><i class="fa fa-check"></i><b>4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="4.1" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#structure-and-context"><i class="fa fa-check"></i><b>4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="4.2" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="4.3" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="4.4" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="4.5" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#dna-methylation-assays"><i class="fa fa-check"></i><b>4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html"><i class="fa fa-check"></i><b>5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="5.1" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#DNAmAgeReview"><i class="fa fa-check"></i><b>5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="5.2" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="5.3" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="5.4" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="part"><span><b>II Methods</b></span></li>
<li class="chapter" data-level="7" data-path="MeDIPseq.html"><a href="MeDIPseq.html"><i class="fa fa-check"></i><b>7</b> MeDIP-seq</a></li>
<li class="chapter" data-level="8" data-path="illumina-methylation-arrays.html"><a href="illumina-methylation-arrays.html"><i class="fa fa-check"></i><b>8</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="9" data-path="whole-genome.html"><a href="whole-genome.html"><i class="fa fa-check"></i><b>9</b> Whole Genome</a></li>
<li class="chapter" data-level="10" data-path="targeted-bisulfite-sequencing.html"><a href="targeted-bisulfite-sequencing.html"><i class="fa fa-check"></i><b>10</b> Targeted Bisulfite Sequencing</a></li>
<li class="chapter" data-level="11" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>11</b> Discussion</a></li>
<li class="part"><span><b>III Results 1</b></span></li>
<li class="chapter" data-level="12" data-path="MAVIDOS.html"><a href="MAVIDOS.html"><i class="fa fa-check"></i><b>12</b> Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="12.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#abstract-1"><i class="fa fa-check"></i><b>12.1</b> Abstract</a></li>
<li class="chapter" data-level="12.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#introduction"><i class="fa fa-check"></i><b>12.2</b> Introduction</a><ul>
<li class="chapter" data-level="12.2.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#background"><i class="fa fa-check"></i><b>12.2.1</b> Background</a></li>
<li class="chapter" data-level="12.2.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>12.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#methods"><i class="fa fa-check"></i><b>12.3</b> Methods</a><ul>
<li class="chapter" data-level="12.3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation"><i class="fa fa-check"></i><b>12.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="12.3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-models"><i class="fa fa-check"></i><b>12.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="12.3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>12.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="12.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#results"><i class="fa fa-check"></i><b>12.4</b> Results</a><ul>
<li class="chapter" data-level="12.4.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#summary-of-results"><i class="fa fa-check"></i><b>12.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="12.4.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#quality-control-qc"><i class="fa fa-check"></i><b>12.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="12.4.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation-1"><i class="fa fa-check"></i><b>12.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="12.4.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewass"><i class="fa fa-check"></i><b>12.4.4</b> EWASs</a></li>
<li class="chapter" data-level="12.4.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>12.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="12.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>12.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="12.5.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#section"><i class="fa fa-check"></i><b>12.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="12.6" data-path="MAVIDOS.html"><a href="MAVIDOS.html#discussion-1"><i class="fa fa-check"></i><b>12.6</b> Discussion</a><ul>
<li class="chapter" data-level="12.6.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#future-work"><i class="fa fa-check"></i><b>12.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>IV Results 2</b></span></li>
<li class="chapter" data-level="13" data-path="tRNApaper.html"><a href="tRNApaper.html"><i class="fa fa-check"></i><b>13</b> The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="13.1" data-path="tRNApaper.html"><a href="tRNApaper.html#abstract-2"><i class="fa fa-check"></i><b>13.1</b> Abstract</a></li>
<li class="chapter" data-level="13.2" data-path="tRNApaper.html"><a href="tRNApaper.html#introduction-1"><i class="fa fa-check"></i><b>13.2</b> Introduction</a></li>
<li class="chapter" data-level="13.3" data-path="tRNApaper.html"><a href="tRNApaper.html#Methods"><i class="fa fa-check"></i><b>13.3</b> Methods</a><ul>
<li class="chapter" data-level="13.3.1" data-path="tRNApaper.html"><a href="tRNApaper.html#participants"><i class="fa fa-check"></i><b>13.3.1</b> Participants</a></li>
<li class="chapter" data-level="13.3.2" data-path="tRNApaper.html"><a href="tRNApaper.html#tRNAanno"><i class="fa fa-check"></i><b>13.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="13.3.3" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylome-data"><i class="fa fa-check"></i><b>13.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="13.3.4" data-path="tRNApaper.html"><a href="tRNApaper.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>13.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="13.3.5" data-path="tRNApaper.html"><a href="tRNApaper.html#sortedbloodmethods"><i class="fa fa-check"></i><b>13.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="13.3.6" data-path="tRNApaper.html"><a href="tRNApaper.html#TCGAmethods"><i class="fa fa-check"></i><b>13.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="13.3.7" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmapmethods"><i class="fa fa-check"></i><b>13.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="13.3.8" data-path="tRNApaper.html"><a href="tRNApaper.html#mousemethods"><i class="fa fa-check"></i><b>13.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="13.3.9" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-figures"><i class="fa fa-check"></i><b>13.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="13.3.10" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>13.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="tRNApaper.html"><a href="tRNApaper.html#res"><i class="fa fa-check"></i><b>13.4</b> Results</a><ul>
<li class="chapter" data-level="13.4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>13.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="13.4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>13.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="13.4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>13.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="13.4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>13.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="13.4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>13.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="13.5" data-path="tRNApaper.html"><a href="tRNApaper.html#discussion-2"><i class="fa fa-check"></i><b>13.5</b> Discussion</a></li>
<li class="chapter" data-level="13.6" data-path="tRNApaper.html"><a href="tRNApaper.html#data-availability"><i class="fa fa-check"></i><b>13.6</b> Data availability</a></li>
<li class="chapter" data-level="13.7" data-path="tRNApaper.html"><a href="tRNApaper.html#code-availablity"><i class="fa fa-check"></i><b>13.7</b> Code availablity</a></li>
<li class="chapter" data-level="13.8" data-path="tRNApaper.html"><a href="tRNApaper.html#acknowledgements-1"><i class="fa fa-check"></i><b>13.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="13.9" data-path="tRNApaper.html"><a href="tRNApaper.html#author-contributions"><i class="fa fa-check"></i><b>13.9</b> Author Contributions</a></li>
<li class="chapter" data-level="13.10" data-path="tRNApaper.html"><a href="tRNApaper.html#references"><i class="fa fa-check"></i><b>13.10</b> References</a></li>
</ul></li>
<li class="part"><span><b>V Results 3</b></span></li>
<li class="chapter" data-level="14" data-path="abstract-3.html"><a href="abstract-3.html"><i class="fa fa-check"></i><b>14</b> Abstract</a></li>
<li class="chapter" data-level="15" data-path="AluIntro.html"><a href="AluIntro.html"><i class="fa fa-check"></i><b>15</b> Introduction</a></li>
<li class="chapter" data-level="16" data-path="methods-1.html"><a href="methods-1.html"><i class="fa fa-check"></i><b>16</b> Methods</a></li>
<li class="chapter" data-level="17" data-path="results-1.html"><a href="results-1.html"><i class="fa fa-check"></i><b>17</b> Results</a><ul>
<li class="chapter" data-level="17.1" data-path="results-1.html"><a href="results-1.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>17.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="17.2" data-path="results-1.html"><a href="results-1.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>17.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="17.3" data-path="results-1.html"><a href="results-1.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>17.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="17.4" data-path="results-1.html"><a href="results-1.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>17.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>18</b> Discussion</a></li>
<li class="chapter" data-level="19" data-path="methods-2.html"><a href="methods-2.html"><i class="fa fa-check"></i><b>19</b> Methods</a><ul>
<li class="chapter" data-level="19.1" data-path="methods-2.html"><a href="methods-2.html#participants-1"><i class="fa fa-check"></i><b>19.1</b> Participants</a></li>
<li class="chapter" data-level="19.2" data-path="methods-2.html"><a href="methods-2.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>19.2</b> Age-related DNA methylation changes</a></li>
<li class="chapter" data-level="19.3" data-path="methods-2.html"><a href="methods-2.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>19.3</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="19.4" data-path="methods-2.html"><a href="methods-2.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>19.4</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="19.5" data-path="methods-2.html"><a href="methods-2.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>19.5</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="19.6" data-path="methods-2.html"><a href="methods-2.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>19.6</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="19.7" data-path="methods-2.html"><a href="methods-2.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>19.7</b> Association of these variants with Age-related Disease</a><ul>
<li class="chapter" data-level="19.7.1" data-path="methods-2.html"><a href="methods-2.html#data-availability-1"><i class="fa fa-check"></i><b>19.7.1</b> Data availability</a></li>
<li class="chapter" data-level="19.7.2" data-path="methods-2.html"><a href="methods-2.html#code-availablity-1"><i class="fa fa-check"></i><b>19.7.2</b> Code availablity</a></li>
<li class="chapter" data-level="19.7.3" data-path="methods-2.html"><a href="methods-2.html#acknowledgements-2"><i class="fa fa-check"></i><b>19.7.3</b> Acknowledgements</a></li>
<li class="chapter" data-level="19.7.4" data-path="methods-2.html"><a href="methods-2.html#author-contributions-1"><i class="fa fa-check"></i><b>19.7.4</b> Author Contributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="20" data-path="final-words.html"><a href="final-words.html"><i class="fa fa-check"></i><b>20</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="epigenomics---overview" class="section level1">
<h1><span class="header-section-number">3</span> Epigenomics - Overview</h1>
<p>Epigenetics generally refers to modifications to DNA and chromatin which do not affect the primary sequence of DNA bases, but which are to varying degrees stable and heritable.
The term derives from epigenesis and genetics originating with Conrad Waddington <span class="citation">(Waddington <a href="#ref-Waddington1942" role="doc-biblioref">2012</a>)</span>.
Epigenesis refers to the idea that organisms develop through the progressive differentiation of cells from the egg into adult tissues.
As modern genetics revealed that every cell contained a complete copy of the genome which was differentially utilised by the cells of adult organisms, the terms were fused to reflect the study of how this occurs.
The usage has evolved a little further since molecular biology began to elucidate the mechanisms involved in this process and the term now frequently refers to the study of these mechanisms and their effects in less explicitly developmental contexts.
Arthur Riggs et al. considered that heritability should be a criterion for a mark to be considered epigenetic, but this excludes many phenomena now commonly referred to using the term <span class="citation">(Russo, Martienssen, and Riggs <a href="#ref-Russo1996" role="doc-biblioref">1996</a>)</span>.
Requiring heritability results in a further definitional dispute over degrees of heritability; mitotic, meiotic, intergenerational, transgenerational and to what degree of fidelity?
Adrian Bird proposed the definition: “the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.” as a useful compromise <span class="citation">(Bird <a href="#ref-Bird2007" role="doc-biblioref">2007</a>)</span>.
Epigenomics refers to the totality of the epigenetic modifications present in a particular cell, tissue type or genome.
This defintition appears to have caught on with at least some as: ‘These days, “epigenetics” more generally refers to all non-genomic information storage in cells including gene networks, chromatin structure and post-translational modifications to histones.’, according to Alice Kane and David Sinclair <span class="citation">(Kane and Sinclair <a href="#ref-Kane2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>Whilst an organism can generally be thought of as having a single genome, with exceptions such as Somatic mutations and Chimerism, it will have at least as many epigenomes as it has cell types.
There are on the order of <span class="math inline">\(10^{13}\)</span> cells in the human body <span class="citation">(Bianconi et al. <a href="#ref-Bianconi2013" role="doc-biblioref">2013</a>)</span>.
Estimates of the number of cell types vary with resolution at which one deems cells functionally distinct <span class="citation">(Regev et al. <a href="#ref-Regev2017" role="doc-biblioref">2017</a>)</span>.
At this point in time, however, there is no clear definition of what constitutes a distinct cell type, indeed they are beginning to be defined by the distinct patterns of gene expression and epigenetic modifications they exhibit <span class="citation">(CellSystemsVoices <a href="#ref-CellSystemsCellType2017" role="doc-biblioref">2017</a>)</span>.
Consequently, the number epigenomes that could be considered distinct will likely ultimately approximate to the number of epigenomes which can usefully distinguish between sub-populations of cells.
A consortium has been established to produce a <a href="www.humancellatlas.org">Human Cell Atlas</a> <span class="citation">(Regev et al. <a href="#ref-Regev2017" role="doc-biblioref">2017</a>)</span> which aims to define all human cell types, and a cell type ontology <span class="citation">(CellOntology <a href="#ref-CellOntologyOBO" role="doc-biblioref">2017</a>)</span> exists.</p>
<p>There are a number of epigenetic modifications which can be subdivided into four broad categories:</p>
<ol style="list-style-type: decimal">
<li><p><strong>DNA modifications</strong></p>
<p>DNA methylation primarily takes place on Cytosine residues.
Cytosine methylation occurs principally in an mCpG sequence context but can also occur in mCpH, mCHG and mCHH (H=A/C/T) contexts; particularly in cells of the nervous system <span class="citation">(Guo et al. <a href="#ref-Guo2014a" role="doc-biblioref">2014</a>)</span> and embryonic stem cells where as many as 25% of all cytosines can be methylated in non-canonical contexts.
In contrast to methylation in a differentiated non-neuronal tissue such as foetal lung fibroblasts which is 99.98% in CpG context <span class="citation">(Lister et al. <a href="#ref-Lister2009" role="doc-biblioref">2009</a>; Schultz et al. <a href="#ref-Schultz2015" role="doc-biblioref">2015</a>)</span>.
5-methylcytosine (5mc) can be oxidised to produce and another modified DNA base 5-hydroxymethylcytosine (5hmC).
5hmC may act as an intermediate to demethylation and potentially has regulatory functions in its own right, reviewed in <span class="citation">(Pfeifer, Kadam, and Jin <a href="#ref-Pfeifer2013" role="doc-biblioref">2013</a>)</span>.
5hmC can be oxidised further to formylcytosine (5fC) and carboxylcytosine (5caC).
The extent to which these modifications are stable and functional is still being explored.
Other DNA bases can be methylated such as N6-methyladenine, but this occurs at a substantially lower frequency that 5mC and much less is known about their potential functions <span class="citation">(Wu et al. <a href="#ref-Wu2016" role="doc-biblioref">2016</a>)</span>.</p></li>
<li><p><strong>Histone tail modifications</strong></p>
<p>Histone proteins form disk-shaped octamers around which ~150bp of DNA can be wrapped to form a nucleosome.
The ‘tails’ are generally the N termini of the histone proteins, outside of the core globular domains, which protrude from the nucleosome structure.
Histone tails can be subject to post-translational modification which commonly takes place at lysine residues, other residues are subject to modifications but lysines are among the best characterised.
Examples of modifications include methylations with between one and three methyl groups per lysine, Acetylations, Ubiquitylation, SUMOylations, and phosphorylations (Reviewed in <span class="citation">Bannister and Kouzarides (<a href="#ref-Bannister2011" role="doc-biblioref">2011</a>)</span>).
Many of these marks can be generally classified as permissive or repressive but are frequently found in combinations of marks of opposing effect, rendering the interpretation of the ‘Histone Code’ extremely challenging <span class="citation">(Voigt et al. <a href="#ref-Voigt2012" role="doc-biblioref">2012</a>)</span>.
One approach to interpreting chromatin state is segmentation.
Segmentation makes use of pattern discovery techniques to divide the genome into discrete sections assigning these sections to a set of categories which can then be compared to existing annotations to provide functional labels <span class="citation">(Hoffman et al. <a href="#ref-Hoffman2012" role="doc-biblioref">2012</a>; Carrillo-de-Santa-Pau et al. <a href="#ref-Carrillo-de-Santa-Pau2017" role="doc-biblioref">2017</a>)</span>.
When successful the pattern discovery algorithm independently recapitulates our existing ontology of functional elements and hopefully provides new insights by for example highlighting regions not previously considered to be in a given functional category.
This is a “top-down” approach making use of the “sum” of available chromatin state data rather than trying to interpret the significance of a single type of mark.</p></li>
<li><p><strong>Histone Variants</strong></p>
<p>Alternatives to the canonical histones can be substituted in the nucleosome, examples of such alternative histones include H2A.Z and H3.3.
Alternative histones can alter chromatin structure and dynamics by altering nucleosome stability, binding different factors, and presenting a different substrate for histone modifications, thus altering their pattern. (reviewed in <span class="citation">Weber and Henikoff (<a href="#ref-Weber2014" role="doc-biblioref">2014</a>)</span>.)</p></li>
<li><p><strong>Some non-coding RNAs (ncRNA)</strong></p>
<p>A subset of ncRNAs, particularly those which persist in close association with chromatin can fall within the definition of epigenetic features (reviewed in <span class="citation">Kaikkonen, Lam, and Glass (<a href="#ref-Kaikkonen2011" role="doc-biblioref">2011</a>)</span>).
An example of such a long non-coding RNA (lncRNA) is Xist which is a key regulator of X inactivation <span class="citation">(Cerase et al. <a href="#ref-Cerase2015" role="doc-biblioref">2015</a>)</span>.</p></li>
</ol>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Bannister2011">
<p>Bannister, Andrew J, and Tony Kouzarides. 2011. “Regulation of chromatin by histone modifications.” <em>Cell Research</em> 21 (3): 381–95. <a href="https://doi.org/10.1038/cr.2011.22">https://doi.org/10.1038/cr.2011.22</a>.</p>
</div>
<div id="ref-Bianconi2013">
<p>Bianconi, Eva, Allison Piovesan, Federica Facchin, Alina Beraudi, Raffaella Casadei, Flavia Frabetti, Lorenza Vitale, et al. 2013. “An estimation of the number of cells in the human body.” <em>Annals of Human Biology</em> 40 (6): 463–71. <a href="https://doi.org/10.3109/03014460.2013.807878">https://doi.org/10.3109/03014460.2013.807878</a>.</p>
</div>
<div id="ref-Bird2007">
<p>Bird, Adrian. 2007. “Perceptions of epigenetics.” <em>Nature</em> 447 (7143): 396–8. <a href="https://doi.org/10.1038/nature05913">https://doi.org/10.1038/nature05913</a>.</p>
</div>
<div id="ref-Carrillo-de-Santa-Pau2017">
<p>Carrillo-de-Santa-Pau, Enrique, David Juan, Vera Pancaldi, Felipe Were, Ignacio Martin-Subero, Daniel Rico, and Alfonso Valencia. 2017. “Automatic identification of informative regions with epigenomic changes associated to hematopoiesis.” <em>Nucleic Acids Research</em> 45 (16): 9244–59. <a href="https://doi.org/10.1093/nar/gkx618">https://doi.org/10.1093/nar/gkx618</a>.</p>
</div>
<div id="ref-CellOntologyOBO">
<p>CellOntology. 2017. “Cell Ontology.” <a href="https://doi.org/10.5281/zenodo.168254">https://doi.org/10.5281/zenodo.168254</a>.</p>
</div>
<div id="ref-CellSystemsCellType2017">
<p>CellSystemsVoices. 2017. “What Is Your Conceptual Definition of ‘Cell Type’ in the Context of a Mature Organism?” <em>Cell Systems</em> 4 (3): 255–59. <a href="https://doi.org/10.1016/j.cels.2017.03.006">https://doi.org/10.1016/j.cels.2017.03.006</a>.</p>
</div>
<div id="ref-Cerase2015">
<p>Cerase, Andrea, Greta Pintacuda, Anna Tattermusch, and Philip Avner. 2015. “Xist localization and function: new insights from multiple levels.” <em>Genome Biology</em> 16 (1): 166. <a href="https://doi.org/10.1186/s13059-015-0733-y">https://doi.org/10.1186/s13059-015-0733-y</a>.</p>
</div>
<div id="ref-Guo2014a">
<p>Guo, Junjie U, Yijing Su, Joo Heon Shin, Jaehoon Shin, Hongda Li, Bin Xie, Chun Zhong, et al. 2014. “Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain.” <em>Nature Neuroscience</em> 17 (2): 215–22. <a href="https://doi.org/10.1038/nn.3607">https://doi.org/10.1038/nn.3607</a>.</p>
</div>
<div id="ref-Hoffman2012">
<p>Hoffman, Michael M, Orion J Buske, Jie Wang, Zhiping Weng, Jeff a Bilmes, and William Stafford Noble. 2012. “Unsupervised pattern discovery in human chromatin structure through genomic segmentation.” <em>Nature Methods</em> 9 (5): 473–6. <a href="https://doi.org/10.1038/nmeth.1937">https://doi.org/10.1038/nmeth.1937</a>.</p>
</div>
<div id="ref-Kaikkonen2011">
<p>Kaikkonen, Minna U., Michael T Y Lam, and Christopher K. Glass. 2011. “Non-coding RNAs as regulators of gene expression and epigenetics.” <em>Cardiovascular Research</em> 90 (3): 430–40. <a href="https://doi.org/10.1093/cvr/cvr097">https://doi.org/10.1093/cvr/cvr097</a>.</p>
</div>
<div id="ref-Kane2019">
<p>Kane, Alice E., and David A. Sinclair. 2019. “Epigenetic changes during aging and their reprogramming potential.” <em>Critical Reviews in Biochemistry and Molecular Biology</em> 54 (1): 61–83. <a href="https://doi.org/10.1080/10409238.2019.1570075">https://doi.org/10.1080/10409238.2019.1570075</a>.</p>
</div>
<div id="ref-Lister2009">
<p>Lister, Ryan, Mattia Pelizzola, Robert H Dowen, R David Hawkins, Gary Hon, Julian Tonti-Filippini, Joseph R Nery, et al. 2009. “Human DNA methylomes at base resolution show widespread epigenomic differences.” <em>Nature</em> 462 (7271): 315–22. <a href="https://doi.org/10.1038/nature08514">https://doi.org/10.1038/nature08514</a>.</p>
</div>
<div id="ref-Pfeifer2013">
<p>Pfeifer, Gerd P, Swati Kadam, and Seung-Gi Jin. 2013. “5-Hydroxymethylcytosine and Its Potential Roles in Development and Cancer.” <em>Epigenetics &amp; Chromatin</em> 6 (1): 10. <a href="https://doi.org/10.1186/1756-8935-6-10">https://doi.org/10.1186/1756-8935-6-10</a>.</p>
</div>
<div id="ref-Regev2017">
<p>Regev, Aviv, Sarah A. Teichmann, Eric S. Lander, Ido Amit, Christophe Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. “The Human Cell Atlas.” <em>eLife</em> 6 (December): 1–30. <a href="https://doi.org/10.7554/eLife.27041">https://doi.org/10.7554/eLife.27041</a>.</p>
</div>
<div id="ref-Russo1996">
<p>Russo, Vincenzo, Robert Martienssen, and Arthur D Riggs. 1996. <em>Epigenetic Mechanisms of Gene Regulation</em>. Cold Spring Harbor Laboratory Press. <a href="https://books.google.co.uk/books/about/Epigenetic%7B\_%7DMechanisms%7B\_%7Dof%7B\_%7DGene%7B\_%7DRegulation.html?id=clnwAAAAMAAJ%20http://genesdev.cshlp.org/content/11/11/local/front-matter.pdf%20https://cshmonographs.org/index.php/monographs/issue/view/087969490.32">https://books.google.co.uk/books/about/Epigenetic{\_}Mechanisms{\_}of{\_}Gene{\_}Regulation.html?id=clnwAAAAMAAJ http://genesdev.cshlp.org/content/11/11/local/front-matter.pdf https://cshmonographs.org/index.php/monographs/issue/view/087969490.32</a>.</p>
</div>
<div id="ref-Schultz2015">
<p>Schultz, Matthew D, Yupeng He, John W Whitaker, Manoj Hariharan, Eran A Mukamel, Danny Leung, Nisha Rajagopal, et al. 2015. “Human body epigenome maps reveal noncanonical DNA methylation variation.” <em>Nature</em> 523 (7559): 212–6. <a href="https://doi.org/10.1038/nature14465">https://doi.org/10.1038/nature14465</a>.</p>
</div>
<div id="ref-Voigt2012">
<p>Voigt, Philipp, Gary LeRoy, William J. Drury, Barry M. Zee, Jinsook Son, David B. Beck, Nicolas L. Young, Benjamin A. Garcia, and Danny Reinberg. 2012. “Asymmetrically modified nucleosomes.” <em>Cell</em> 151 (1): 181–93. <a href="https://doi.org/10.1016/j.cell.2012.09.002">https://doi.org/10.1016/j.cell.2012.09.002</a>.</p>
</div>
<div id="ref-Waddington1942">
<p>Waddington, C. H. 2012. “The epigenotype. 1942.” <em>International Journal of Epidemiology</em> 41 (1): 10–13. <a href="https://doi.org/10.1093/ije/dyr184">https://doi.org/10.1093/ije/dyr184</a>.</p>
</div>
<div id="ref-Weber2014">
<p>Weber, Christopher M, and Steven Henikoff. 2014. “Histone variants: dynamic punctuation in transcription.” <em>Genes &amp; Development</em> 28 (7): 672–82. <a href="https://doi.org/10.1101/gad.238873.114">https://doi.org/10.1101/gad.238873.114</a>.</p>
</div>
<div id="ref-Wu2016">
<p>Wu, Tao P., Tao Wang, Matthew G. Seetin, Yongquan Lai, Shijia Zhu, Kaixuan Lin, Yifei Liu, et al. 2016. “DNA methylation on N(6)-adenine in mammalian embryonic stem cells.” <em>Nature</em> 532 (7599): 329–33. <a href="https://doi.org/10.1038/nature17640">https://doi.org/10.1038/nature17640</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="the-biology-of-ageing.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="fundamentals-of-dna-methylation.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/01-intro.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
