<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4 Fundamentals of DNA Methylation | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="4 Fundamentals of DNA Methylation | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4 Fundamentals of DNA Methylation | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="epigenomics-overview.html"/>
<link rel="next" href="dna-methylation-and-ageing.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="the-ageing-population-and-the-burden-of-disease.html"><a href="the-ageing-population-and-the-burden-of-disease.html"><i class="fa fa-check"></i><b>1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="2" data-path="the-biology-of-ageing.html"><a href="the-biology-of-ageing.html"><i class="fa fa-check"></i><b>2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="3" data-path="epigenomics-overview.html"><a href="epigenomics-overview.html"><i class="fa fa-check"></i><b>3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="4" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html"><i class="fa fa-check"></i><b>4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="4.1" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#structure-and-context"><i class="fa fa-check"></i><b>4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="4.2" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="4.3" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="4.4" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="4.5" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#dna-methylation-assays"><i class="fa fa-check"></i><b>4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html"><i class="fa fa-check"></i><b>5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="5.1" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#DNAmAgeReview"><i class="fa fa-check"></i><b>5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="5.2" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="5.3" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="5.4" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="part"><span><b>II Methods</b></span></li>
<li class="chapter" data-level="7" data-path="MeDIPseq.html"><a href="MeDIPseq.html"><i class="fa fa-check"></i><b>7</b> MeDIP-seq</a></li>
<li class="chapter" data-level="8" data-path="illumina-methylation-arrays.html"><a href="illumina-methylation-arrays.html"><i class="fa fa-check"></i><b>8</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="9" data-path="whole-genome.html"><a href="whole-genome.html"><i class="fa fa-check"></i><b>9</b> Whole Genome</a></li>
<li class="chapter" data-level="10" data-path="targeted-bisulfite-sequencing.html"><a href="targeted-bisulfite-sequencing.html"><i class="fa fa-check"></i><b>10</b> Targeted Bisulfite Sequencing</a></li>
<li class="chapter" data-level="11" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>11</b> Discussion</a></li>
<li class="part"><span><b>III Results 1</b></span></li>
<li class="chapter" data-level="12" data-path="MAVIDOS.html"><a href="MAVIDOS.html"><i class="fa fa-check"></i><b>12</b> Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="12.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#abstract-1"><i class="fa fa-check"></i><b>12.1</b> Abstract</a></li>
<li class="chapter" data-level="12.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#introduction"><i class="fa fa-check"></i><b>12.2</b> Introduction</a><ul>
<li class="chapter" data-level="12.2.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#background"><i class="fa fa-check"></i><b>12.2.1</b> Background</a></li>
<li class="chapter" data-level="12.2.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>12.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#methods"><i class="fa fa-check"></i><b>12.3</b> Methods</a><ul>
<li class="chapter" data-level="12.3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation"><i class="fa fa-check"></i><b>12.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="12.3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-models"><i class="fa fa-check"></i><b>12.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="12.3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>12.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="12.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#results"><i class="fa fa-check"></i><b>12.4</b> Results</a><ul>
<li class="chapter" data-level="12.4.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#summary-of-results"><i class="fa fa-check"></i><b>12.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="12.4.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#quality-control-qc"><i class="fa fa-check"></i><b>12.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="12.4.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation-1"><i class="fa fa-check"></i><b>12.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="12.4.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewass"><i class="fa fa-check"></i><b>12.4.4</b> EWASs</a></li>
<li class="chapter" data-level="12.4.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>12.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="12.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>12.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="12.5.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#section"><i class="fa fa-check"></i><b>12.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="12.6" data-path="MAVIDOS.html"><a href="MAVIDOS.html#discussion-1"><i class="fa fa-check"></i><b>12.6</b> Discussion</a><ul>
<li class="chapter" data-level="12.6.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#future-work"><i class="fa fa-check"></i><b>12.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>IV Results 2</b></span></li>
<li class="chapter" data-level="13" data-path="tRNApaper.html"><a href="tRNApaper.html"><i class="fa fa-check"></i><b>13</b> The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="13.1" data-path="tRNApaper.html"><a href="tRNApaper.html#abstract-2"><i class="fa fa-check"></i><b>13.1</b> Abstract</a></li>
<li class="chapter" data-level="13.2" data-path="tRNApaper.html"><a href="tRNApaper.html#introduction-1"><i class="fa fa-check"></i><b>13.2</b> Introduction</a></li>
<li class="chapter" data-level="13.3" data-path="tRNApaper.html"><a href="tRNApaper.html#Methods"><i class="fa fa-check"></i><b>13.3</b> Methods</a><ul>
<li class="chapter" data-level="13.3.1" data-path="tRNApaper.html"><a href="tRNApaper.html#participants"><i class="fa fa-check"></i><b>13.3.1</b> Participants</a></li>
<li class="chapter" data-level="13.3.2" data-path="tRNApaper.html"><a href="tRNApaper.html#tRNAanno"><i class="fa fa-check"></i><b>13.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="13.3.3" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylome-data"><i class="fa fa-check"></i><b>13.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="13.3.4" data-path="tRNApaper.html"><a href="tRNApaper.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>13.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="13.3.5" data-path="tRNApaper.html"><a href="tRNApaper.html#sortedbloodmethods"><i class="fa fa-check"></i><b>13.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="13.3.6" data-path="tRNApaper.html"><a href="tRNApaper.html#TCGAmethods"><i class="fa fa-check"></i><b>13.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="13.3.7" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmapmethods"><i class="fa fa-check"></i><b>13.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="13.3.8" data-path="tRNApaper.html"><a href="tRNApaper.html#mousemethods"><i class="fa fa-check"></i><b>13.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="13.3.9" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-figures"><i class="fa fa-check"></i><b>13.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="13.3.10" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>13.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="tRNApaper.html"><a href="tRNApaper.html#res"><i class="fa fa-check"></i><b>13.4</b> Results</a><ul>
<li class="chapter" data-level="13.4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>13.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="13.4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>13.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="13.4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>13.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="13.4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>13.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="13.4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>13.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="13.5" data-path="tRNApaper.html"><a href="tRNApaper.html#discussion-2"><i class="fa fa-check"></i><b>13.5</b> Discussion</a></li>
<li class="chapter" data-level="13.6" data-path="tRNApaper.html"><a href="tRNApaper.html#data-availability"><i class="fa fa-check"></i><b>13.6</b> Data availability</a></li>
<li class="chapter" data-level="13.7" data-path="tRNApaper.html"><a href="tRNApaper.html#code-availablity"><i class="fa fa-check"></i><b>13.7</b> Code availablity</a></li>
<li class="chapter" data-level="13.8" data-path="tRNApaper.html"><a href="tRNApaper.html#acknowledgements-1"><i class="fa fa-check"></i><b>13.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="13.9" data-path="tRNApaper.html"><a href="tRNApaper.html#author-contributions"><i class="fa fa-check"></i><b>13.9</b> Author Contributions</a></li>
<li class="chapter" data-level="13.10" data-path="tRNApaper.html"><a href="tRNApaper.html#references"><i class="fa fa-check"></i><b>13.10</b> References</a></li>
</ul></li>
<li class="part"><span><b>V Results 3</b></span></li>
<li class="chapter" data-level="14" data-path="abstract-3.html"><a href="abstract-3.html"><i class="fa fa-check"></i><b>14</b> Abstract</a></li>
<li class="chapter" data-level="15" data-path="AluIntro.html"><a href="AluIntro.html"><i class="fa fa-check"></i><b>15</b> Introduction</a></li>
<li class="chapter" data-level="16" data-path="methods-1.html"><a href="methods-1.html"><i class="fa fa-check"></i><b>16</b> Methods</a></li>
<li class="chapter" data-level="17" data-path="results-1.html"><a href="results-1.html"><i class="fa fa-check"></i><b>17</b> Results</a><ul>
<li class="chapter" data-level="17.1" data-path="results-1.html"><a href="results-1.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>17.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="17.2" data-path="results-1.html"><a href="results-1.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>17.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="17.3" data-path="results-1.html"><a href="results-1.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>17.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="17.4" data-path="results-1.html"><a href="results-1.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>17.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>18</b> Discussion</a></li>
<li class="chapter" data-level="19" data-path="methods-2.html"><a href="methods-2.html"><i class="fa fa-check"></i><b>19</b> Methods</a><ul>
<li class="chapter" data-level="19.1" data-path="methods-2.html"><a href="methods-2.html#participants-1"><i class="fa fa-check"></i><b>19.1</b> Participants</a></li>
<li class="chapter" data-level="19.2" data-path="methods-2.html"><a href="methods-2.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>19.2</b> Age-related DNA methylation changes</a></li>
<li class="chapter" data-level="19.3" data-path="methods-2.html"><a href="methods-2.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>19.3</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="19.4" data-path="methods-2.html"><a href="methods-2.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>19.4</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="19.5" data-path="methods-2.html"><a href="methods-2.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>19.5</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="19.6" data-path="methods-2.html"><a href="methods-2.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>19.6</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="19.7" data-path="methods-2.html"><a href="methods-2.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>19.7</b> Association of these variants with Age-related Disease</a><ul>
<li class="chapter" data-level="19.7.1" data-path="methods-2.html"><a href="methods-2.html#data-availability-1"><i class="fa fa-check"></i><b>19.7.1</b> Data availability</a></li>
<li class="chapter" data-level="19.7.2" data-path="methods-2.html"><a href="methods-2.html#code-availablity-1"><i class="fa fa-check"></i><b>19.7.2</b> Code availablity</a></li>
<li class="chapter" data-level="19.7.3" data-path="methods-2.html"><a href="methods-2.html#acknowledgements-2"><i class="fa fa-check"></i><b>19.7.3</b> Acknowledgements</a></li>
<li class="chapter" data-level="19.7.4" data-path="methods-2.html"><a href="methods-2.html#author-contributions-1"><i class="fa fa-check"></i><b>19.7.4</b> Author Contributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="20" data-path="final-words.html"><a href="final-words.html"><i class="fa fa-check"></i><b>20</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="fundamentals-of-dna-methylation" class="section level1">
<h1><span class="header-section-number">4</span> Fundamentals of DNA Methylation</h1>
<div id="structure-and-context" class="section level2">
<h2><span class="header-section-number">4.1</span> Structure and Context</h2>
<p>DNA methylation is the most well-studied epigenetic modification <span class="citation">(Jenuwein <a href="#ref-Haluskova2010" role="doc-biblioref">2001</a>)</span>.
Specifically CpG methylation, the addition of a Methyl group to the 5 carbon of a cytosine base (figure <a href="fundamentals-of-dna-methylation.html#fig:5mCvsC">4.1</a>), in a CpG dinucleotide context (figure <a href="fundamentals-of-dna-methylation.html#fig:CpGvsGpC">4.2</a>).
(hereafter ‘DNA Methylation’ or ‘DNAm’ refers to 5mC CpG methylation unless stated otherwise.)</p>
<p>DNA methylation is stable and relatively easy to characterise from biological samples by comparison with other epigenetic marks, the methylation status of ancient DNA has even been characterised, see for example:
DNAm profiles have been produced from DNA extracted from mammoth and neandertal remains <span class="citation">(Briggs et al. <a href="#ref-Briggs2010" role="doc-biblioref">2010</a>)</span> and from native American remains ranging in age from 230 to 4500 years <span class="citation">(Smith, Monroe, and Bolnick <a href="#ref-Smith2015" role="doc-biblioref">2015</a>)</span>.
5mC is quite chemically stable such that for most tissue sample preservation techniques for which DNA can be extracted from the sample DNAm can also be characterised.
DNA extracted with normal laboratory methods can be assayed for DNAm.
Histone modification status, on the other hand, is less robust against environmental stresses on samples and often requires considerably more laborious sample preparation to characterise <span class="citation">(Bauden et al. <a href="#ref-Bauden2017" role="doc-biblioref">2017</a>)</span>.
DNA methylation also offers the appearance of relative simplicity in comparison to the complex picture of the ‘histone code’ giving the impression that its useful interpretation may be more readily tractable.
However, interaction between DNAm and histone modifications is well documented <span class="citation">(Hashimshony et al. <a href="#ref-Hashimshony2003" role="doc-biblioref">2003</a>; Estève et al. <a href="#ref-Esteve2006" role="doc-biblioref">2006</a>; Rose and Klose <a href="#ref-Rose2014" role="doc-biblioref">2014</a>)</span> consequently, a fuller understanding of DNAm will likely require an understanding of histone modifications and their potential to feedback on and thus affect DNAm.</p>
<div class="figure" style="text-align: center"><span id="fig:5mCvsC"></span>
<img src="figs/5mC_vs_C.png" alt="The structure of 5-methylcytosine contrasted with Cytosine. (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 4.1: The structure of 5-methylcytosine contrasted with Cytosine. (Figure created by the Author.)
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:CpGvsGpC"></span>
<img src="figs/CpG_vs_GpC.png" alt="A schematic representation showing a Methylated CpG dinucleotide in its structural context. This is contrasted with a GpC dinucleotide, which cannot be superimposed on a CpG. upper left: methylated cytosine residues in a CpG dinucleotide, showing 5mC is 5’ of G on both DNA strands. upper right: unmethylated cytosine residues in a GpC dinucleotide, showing G is 5’ of C on both DNA strands. lower: superimposition of a GpC and CpG dinucleotide illustrating the mirrored nature of these structures and the impossibility of superimposing them through rotation. (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 4.2: A schematic representation showing a Methylated <strong>CpG</strong> dinucleotide in its structural context. This is contrasted with a <strong>GpC</strong> dinucleotide, which cannot be superimposed on a <strong>CpG</strong>. upper left: methylated cytosine residues in a <strong>CpG</strong> dinucleotide, showing 5mC is 5’ of G on both DNA strands. upper right: unmethylated cytosine residues in a <strong>GpC</strong> dinucleotide, showing G is 5’ of C on both DNA strands. lower: superimposition of a <strong>GpC</strong> and <strong>CpG</strong> dinucleotide illustrating the mirrored nature of these structures and the impossibility of superimposing them through rotation. (Figure created by the Author.)
</p>
</div>

</div>
<div id="distribution-and-global-trends" class="section level2">
<h2><span class="header-section-number">4.2</span> Distribution and Global Trends</h2>
<p>There are 28,299,634 CpGs <span class="citation">(Luo, Lu, and Xie <a href="#ref-Luo2014" role="doc-biblioref">2014</a>)</span> in the hg19 assembly of the human genome <span class="citation">(Lander et al. <a href="#ref-Lander2001" role="doc-biblioref">2001</a>)</span>.
Given that the GC content of the human genome is <span class="math inline">\(42\%\)</span> the prior probability of getting a CpG dinucleotide is: <span class="math inline">\(0.21 \times 0.21 = 0.0441\)</span>, <span class="math inline">\(4.41\%\)</span>.
CpGs represent <span class="math inline">\(\sim1.8\%\)</span> of the dinucleotides in the human genome ( <span class="math inline">\(28,299,634 \div (3.23\times10^9\div2) \approx 0.0175\)</span>).
Making them <span class="math inline">\(\sim2.5\)</span> fold less frequent than would be expected <em>a priori</em>.
Methylated cytosines are prone to deamination to thymines, resulting in mismatch lesions <span class="citation">(Duncan and Miller <a href="#ref-Duncan1980" role="doc-biblioref">1980</a>)</span>.
This increased mutagenic potential generally means they are selected against, accounting for at least some part of their under-representation in the genome.
The three major classes of repeat elements SINEs, LINEs and LTRs contain some <span class="math inline">\(46\%\)</span> of all CpG sites, with a further <span class="math inline">\(5\%\)</span> in other repetitive elements <span class="citation">(Luo, Lu, and Xie <a href="#ref-Luo2014" role="doc-biblioref">2014</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:CGIs"></span>
<img src="figs/CGI-Shores-Shelves-Seas_2.png" alt="Schematic representation of CpG Islands, shores, shelves and seas. CpG density declines from shores to seas and methylation increases (CpG density and methylation proportion not to scale). (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 4.3: Schematic representation of CpG Islands, shores, shelves and seas. CpG density declines from shores to seas and methylation increases (CpG density and methylation proportion not to scale). (Figure created by the Author.)
</p>
</div>

<p>CpGs are not uniformly distributed in the genome, they occur at higher frequency in some regions.
“CpG islands” or CGIs are regions of high CpG density.
The total number of CpGs in the UCSC repeat masked CGI annotation list of 28,691 CGIs is 1,990,729.
Therefore, these CpGs comprise <span class="math inline">\(\sim7.0\%\)</span> of the total number of CpGs the genome. The mean percentage of the sequence of these CGIs that is comprised of CpG dinucleotides is <span class="math inline">\(\sim18.5\%\)</span> and their mean length is 761bp.
Irizarry et al. found that the 2kb regions flanking CpG islands which they termed “CpG island shores” exhibited greater tissue-specific differential methylation than the islands themselves <span class="citation">(Irizarry et al. <a href="#ref-Irizarry2009" role="doc-biblioref">2009</a>)</span>.
This nomenclature has subsequently been expanded further with “CpG island shelves” which are 2kb - 4kb from the CGIs, and “seas” referring to the rest of the genome, see figure <a href="fundamentals-of-dna-methylation.html#fig:CGIs">4.3</a>.</p>
<p>CGIs overlap the promoters of <span class="math inline">\(\sim70\%\)</span> of genes <span class="citation">(Saxonov, Berg, and Brutlag <a href="#ref-Saxonov2006" role="doc-biblioref">2006</a>)</span>, this, however, leaves roughly half of all CGIs as “orphans” not associated with a known transcription site.
These orphan CGIs frequently constitute cell-type specific enhancers <span class="citation">(Bell and Vertino <a href="#ref-Bell2017" role="doc-biblioref">2017</a>)</span> and alternate promoters <span class="citation">(Illingworth et al. <a href="#ref-Illingworth2010" role="doc-biblioref">2010</a>; Maunakea et al. <a href="#ref-Maunakea2010" role="doc-biblioref">2010</a>)</span>.
They are found in both intergenic and intragenic regions.
CpGs are distributed quite sparsely through the genome occurring at low density in “seas” and at increasing density in CGIs and their flanking regions, see figure <a href="fundamentals-of-dna-methylation.html#fig:CGIs">4.3</a>.
Early work showed 70-80% of CpGs are constitutively methylated <span class="citation">(Ehrlich et al. <a href="#ref-Ehrlich1982" role="doc-biblioref">1982</a>)</span> (reviewed <span class="citation">Bird (<a href="#ref-Bird2002" role="doc-biblioref">2002</a>)</span>) Stadler et al. produced a more detailed picture of the distribution of CpG methylation in mouse embryonic stem cells (ESCs) <span class="citation">(Stadler et al. <a href="#ref-Stadler2011" role="doc-biblioref">2011</a>)</span> see figure <a href="fundamentals-of-dna-methylation.html#fig:CpGfreqVsDNAm">4.4</a>.
It is a characteristic of CGIs that they are generally unmethylated.</p>
<div class="figure" style="text-align: center"><span id="fig:CpGfreqVsDNAm"></span>
<img src="figs/CpGfreqVsDNAm.png" alt="The fraction of CpGs with a given methylation level. Data from Whole-Genome Bisulfite Sequencing (WGBS) of Mouse embryonic stem cells. (Figure reproduced from Stadler et al. 2011 (Stadler et al. 2011) figure 1a)" width="80%" />
<p class="caption">
Figure 4.4: The fraction of CpGs with a given methylation level. Data from Whole-Genome Bisulfite Sequencing (WGBS) of Mouse embryonic stem cells. (Figure reproduced from Stadler et al. 2011 <span class="citation">(Stadler et al. <a href="#ref-Stadler2011" role="doc-biblioref">2011</a>)</span> figure 1a)
</p>
</div>

<p>The regulatory role played by DNA methylation varies with context and the scale at which it is examined.
For example methylation at a single CpG in a transcription factor binding site can determine whether or not the factor can bind at that site.
There are methylation-sensitive transcription factors which can have their affinity for DNA either increased or decreased by methylation <span class="citation">(Yin et al. <a href="#ref-Yin2017" role="doc-biblioref">2017</a>; Zuo et al. <a href="#ref-Zuo2017" role="doc-biblioref">2017</a>)</span>.
In addition, larger scale changes in DNAm can through interactions with chromatin modifiers alter DNA compaction and more general accessibility of DNA regions for binding, and/or impacting on their topological organisation.
DNAm also varies with nucleosome occupancy with lower methylation in linker sequences than on nucleosome-associated DNA <span class="citation">(Kelly et al. <a href="#ref-Kelly2012" role="doc-biblioref">2012</a>; Huff and Zilberman <a href="#ref-Huff2014" role="doc-biblioref">2014</a>)</span>.
The oxidation products of 5mC can also affect chromatin organisation, 5-Formylcytosine can impact nucleosome positioning through covalently binding to histones <span class="citation">(Raiber et al. <a href="#ref-Raiber2017" role="doc-biblioref">2018</a>)</span>.
This raises the question if some DNA methylation occurs specifically to act as an intermediate step for regulatory mechanisms which utilise its oxidation products.</p>
</div>
<div id="pathways-of-dna-methylation-and-demethylation" class="section level2">
<h2><span class="header-section-number">4.3</span> Pathways of DNA methylation and demethylation</h2>
<p>CpG methylation is produced and maintained by DNA methyltransferase (DNMT) enzymes.
All the DNMT enzymes use S-adenosylmethionine as the source of the methyl donor group.
DNMTs form a covalent intermediate between a conserved cysteine residue and the target base, through a nucleophilic attack on the C6 position in the cytosine ring.
This is followed by the transfer of the S-adenosylmethionine methyl group to C5, and deprotonation of the C5 to reform the double bond between C5 and C6; which is mediated by a base provided by the enzyme (Figure <a href="fundamentals-of-dna-methylation.html#fig:Lyko2017DNAmMechanism">4.5</a>).</p>
<p>DNMT1 is associated with the replication machinery <span class="citation">(Vertino et al. <a href="#ref-Vertino2002" role="doc-biblioref">2002</a>)</span> and reproduces the methylation state of the parent strand on the daughter strand during replication.
The largest of the DNMT family DNMT1 contains a ‘replication foci targeting sequence’ (RFTS) domain required for its targeting to replication forks.
DNMT1 specialises in recognising hemimethylated DNA and methylating the unmethylated C in a palindromic CpG dinucleotide site, the UHRF1 protein assists in the recognition of these sequences <span class="citation">(Bostick et al. <a href="#ref-Bostick2007" role="doc-biblioref">2007</a>)</span>.
DNMT3a and DNMT3b are responsible for <em>de novo</em> DNA methylation along with DNMT3L a catalytically inactive, but DNA-binding subunit <span class="citation">(Jia et al. <a href="#ref-Jia2009" role="doc-biblioref">2007</a>)</span>.
The location of <em>de novo</em> methylation by the DNMT3s is influenced by a number of factors including the Chromatin state and other DNA binding factors <span class="citation">(Lyko <a href="#ref-Lyko2017" role="doc-biblioref">2017</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Lyko2017DNAmMechanism"></span>
<img src="figs/Lyko2017DNAmMechanism.png" alt="Motif VI (ENV/PCQ) refers to the conserved catalytic motif in DNMTs. ‘B:’ Represents the enzyme supplied base (Figure reproduced from the review of DNMT biology by Lyko (2017) (fig. 1b))" width="80%" />
<p class="caption">
Figure 4.5: Motif VI (ENV/PCQ) refers to the conserved catalytic motif in DNMTs. ‘B:’ Represents the enzyme supplied base (Figure reproduced from the review of DNMT biology by <span class="citation">Lyko (<a href="#ref-Lyko2017" role="doc-biblioref">2017</a>)</span> (fig. 1b))
</p>
</div>

<p>DNMT2 is unlike the other members of the DNMT family in that it targets an RNA substrate.
DNMT2 is a tRNA methyltransferase which methylates a specific subset of tRNA genes, mostly Asp isoacceptors, at a site adjacent to the anticodon which protects them from endonucleolytic cleavage under stress conditions.
Unprotected tRNAs produce fragments which compete with small interfering RNAs interfering with their signalling pathways.</p>
<p>The mechanisms of the demethylation of DNA were more recently characterised than those which govern its methylation.
They are more complex and less well understood.
A distinction is drawn between active and passive demethylation, in passive demethylation, 5mC bases are diluted out in the process of DNA replication.
In the leading model of active demethylation, they are oxidised one or more times by an enzyme from the TET (Ten-Eleven-translocase) family.
They are then either passively removed by DNA replication or actively removed by a DNA glycosylase (thymine DNA glycosylase TGD) to create an apurinic site which is restored to a C by the Base Excision Repair (BER) pathway. This cycle of cytosine methylation and demethylation is illustrated in Figure <a href="fundamentals-of-dna-methylation.html#fig:Wu2017TETsDNAmCycle">4.6</a> <span class="citation">(Wu and Zhang <a href="#ref-Wu2017" role="doc-biblioref">2017</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Wu2017TETsDNAmCycle"></span>
<img src="figs/Wu2017TETsDNAmCycle.png" alt="BER = Bases Excision Repair; AM = Active Modification; AR = Active Removal, PD = Passive Dilution (Figure reproduced from the review of TET mediated active demethylation by Wu and Zhang (2017) (fig. 1a))" width="60%" />
<p class="caption">
Figure 4.6: BER = Bases Excision Repair; AM = Active Modification; AR = Active Removal, PD = Passive Dilution (Figure reproduced from the review of TET mediated active demethylation by <span class="citation">Wu and Zhang (<a href="#ref-Wu2017" role="doc-biblioref">2017</a>)</span> (fig. 1a))
</p>
</div>

</div>
<div id="maintenance-and-fidelity" class="section level2">
<h2><span class="header-section-number">4.4</span> Maintenance and Fidelity</h2>
<p>DNA methylation is highly dependent on the underlying DNA sequence.
Sequence features of particular importance to determining DNA methylation status are; transcription factor and other DNA binding protein recognition motifs, and CpG density <span class="citation">(Lienert et al. <a href="#ref-Lienert2011" role="doc-biblioref">2011</a>; Ziller et al. <a href="#ref-Ziller2013" role="doc-biblioref">2013</a>)</span>.
Alterations in underlying DNA sequence such as SNPs and copy number variants (CNVs) can have a significant impact on methylation level and the susceptibility of the methylation level to change.
In addition CNVs can result in dosage effects on measures of DNAm, causing regions to appear, respectively, substantially more or less methylated when fewer or greater copies than expected are present <span class="citation">(Boks et al. <a href="#ref-Boks2009" role="doc-biblioref">2009</a>; Kerkel et al. <a href="#ref-Kerkel2008" role="doc-biblioref">2008</a>; Schalkwyk et al. <a href="#ref-Schalkwyk2010" role="doc-biblioref">2010</a>; Shoemaker et al. <a href="#ref-Shoemaker2010" role="doc-biblioref">2010</a>; Bell et al. <a href="#ref-Bell2017a" role="doc-biblioref">2018</a>)</span>.
Despite the strong influence of sequence on methylation, global CpG methylation exhibits change over developmental time as illustrated in figure <a href="fundamentals-of-dna-methylation.html#fig:DNAmDevTime">4.7</a> and exhibits tissue-specific changes in distribution and amount <span class="citation">(Ziller et al. <a href="#ref-Ziller2013" role="doc-biblioref">2013</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:DNAmDevTime"></span>
<img src="figs/Ciernia2016f3DNAmTime.PNG" alt="5mC levels over developmental time. 5mCH or 5-methylcytosine-(A, T or C) levels are represented on a separate (purple) Axis from the main 5mCG axis. PGC = Primordial Germ Cell. (Figure reproduced from Ciernia and LaSalle (2016) figure 3)" width="80%" />
<p class="caption">
Figure 4.7: 5mC levels over developmental time. 5mCH or 5-methylcytosine-(A, T or C) levels are represented on a separate (purple) Axis from the main 5mCG axis. PGC = Primordial Germ Cell. (Figure reproduced from <span class="citation">Ciernia and LaSalle (<a href="#ref-Ciernia2016" role="doc-biblioref">2016</a>)</span> figure 3)
</p>
</div>

<p>In mitosis, DNA methylation is inherited by daughter cells with an error rate on the order of <span class="math inline">\(1\times10^{-3}\)</span> per site per generation <span class="citation">(Ushijima et al. <a href="#ref-Ushijima2005" role="doc-biblioref">2005</a>)</span>, several orders of magnitude less than that of DNA replication (error rate of <span class="math inline">\(1\times10^{-7}\)</span> - <span class="math inline">\(1\times10^{-8}\)</span> <span class="citation">(Kunkel <a href="#ref-Kunkel2004a" role="doc-biblioref">2004</a>)</span>).
The fidelity of DNA methylation copying can be assayed by Hairpin-Bisulfite PCR (Polymerase Chain Reaction) <span class="citation">(Laird et al. <a href="#ref-Laird2004" role="doc-biblioref">2004</a>)</span>.
Laird et al. looked at two alleles of a portion of the CpG island from the human <em>FMR1</em> gene in uncultured lymphocytes, one hypermethylated and one hypomethylated.
In the hypermethylated allele, they found that 96% of sites methylated in the parent strand remained methylated in the daughter strand and 86% of unmethylated sites remained unmethylated.
By contrast in the hypomethylated allele, there were no methylated sites to be retained and &gt;99% of unmethylated sites remained so following replication.
Laird et al.’s work suggests a higher degree of overall methylation fidelity for hypomethylated DNA and a propensity for unmethylated sites in hypermethylated DNA to become methylated.</p>
<p>Change in DNAm levels over time (divisions) can be modelled using the differential equations <span class="citation">(Pfeifer et al. <a href="#ref-Pfeifer1990a" role="doc-biblioref">1990</a>)</span>, which predict that a fully methylated site and a fully unmethylated site will converge on an equilibrium level.
This level is determined by the probability of maintenance of the methylation state and of <em>de novo</em> methylation for a given locus.
This stochastic model of DNAm is in agreement with experimental findings <span class="citation">(Laird et al. <a href="#ref-Laird2004" role="doc-biblioref">2004</a>; Riggs and Xiong <a href="#ref-Riggs2004a" role="doc-biblioref">2004</a>)</span>, Figure <a href="fundamentals-of-dna-methylation.html#fig:Riggs2004PNASfig2DNAmFidelity">4.8</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:Riggs2004PNASfig2DNAmFidelity"></span>
<img src="figs/Riggs2004PNASfig2DNAmFidelity.png" alt="\(E_m\) = probability of methylation maintenance, \(E_m\) = probability of de novo methylation. \(M\) &amp; \(U\) = the number of methylated and unmethylated molecules at specific CpG sites respectively. Modelling (Grey points) \(E_m,E_d\) values of 0.90,0.10; 0.95,0.05; 0.97,0.03 (outermost to innermost, two curves for each \(E_m,E_d\) set, one starting at \(M=1,U=0\) and one a \(M=0,U=1\). Experimental data (Black points) from the HpaII locus (site H3) in 17 clones of mouse cell line BML-2 which has a known methylation level of 50%. (Reproduced from (Riggs and Xiong 2004))" width="70%" />
<p class="caption">
Figure 4.8: <span class="math inline">\(E_m\)</span> = probability of methylation maintenance, <span class="math inline">\(E_m\)</span> = probability of <em>de novo</em> methylation. <span class="math inline">\(M\)</span> &amp; <span class="math inline">\(U\)</span> = the number of methylated and unmethylated molecules at specific CpG sites respectively. Modelling (<strong>Grey points</strong>) <span class="math inline">\(E_m,E_d\)</span> values of 0.90,0.10; 0.95,0.05; 0.97,0.03 (outermost to innermost, two curves for each <span class="math inline">\(E_m,E_d\)</span> set, one starting at <span class="math inline">\(M=1,U=0\)</span> and one a <span class="math inline">\(M=0,U=1\)</span>. Experimental data (<strong>Black points</strong>) from the <em>HpaII</em> locus (site H3) in 17 clones of mouse cell line BML-2 which has a known methylation level of 50%. (Reproduced from <span class="citation">(Riggs and Xiong <a href="#ref-Riggs2004a" role="doc-biblioref">2004</a>)</span>)
</p>
</div>

<p>Jenkinson et al. <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> used an information-theoretic approach, modelling DNAm as a binary communications channel using a 1-dimensional Ising model from the field of statistical physics.
This permitted them to examine properties of DNAm not accessible to conventional means of analysis which typically capture the mean methylation level and perhaps the variability for a given locus.
When considering methylation fidelity using this lens the maintenance of a given methylation state can be seen as an information processing task which requires the consumption of free energy in order to reduce the probability of error in transmission of that methylation state.
Thus Relative Dissipated Energy (RDE) can serve as a measure of the work expended by a cell at a given locus in order to preserve the current methylation state of that locus.</p>
<p>Zhao et al <span class="citation">(Zhao et al. <a href="#ref-Zhao2014" role="doc-biblioref">2014</a>)</span> applied Hairpin-Bisulfite PCR genome-wide in mouse embryonic stem cells (ESCs).
They also found high degrees of methylation fidelity in hypomethylated regions such as CGIs and Promoters, as well as a high degree of fidelity in sites bound by transcription factors.
This is in agreement with the findings of Jenkinson et al. <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> who noted that entropy (methylation stochasticity) was lower and more variable in CGIs and TSS (transcription start sites).</p>
<p>Methylation inheritance fidelity is reduced in cancer <span class="citation">(Ushijima et al. <a href="#ref-Ushijima2005" role="doc-biblioref">2005</a>)</span> and increases with differentiation <span class="citation">(Zhao et al. <a href="#ref-Zhao2014" role="doc-biblioref">2014</a>)</span>.
The RDE (relative dissipated energy) at CGIs and TSSs is higher in differentiated tissues such as the brain, implying low entropy, and lower in embryonic stem cells, implying greater entropy <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> (see Figure <a href="fundamentals-of-dna-methylation.html#fig:Zhao2014fig3ab">4.9</a>).
In addition, the correlation between CpG sites increases in cancer <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span>, suggesting reduced higher level regulatory control and tendency to fall back on lower level feedback mechanisms leading to a return to the baseline equilibrium described above as the cell is expending less energy to maintain an out of equilibrium methylation state.</p>
<div class="figure" style="text-align: center"><span id="fig:Zhao2014fig3ab"></span>
<img src="figs/Zhao2014-fig3-ab.png" alt="Bimodal Distribution of DNA methylation level and fidelity. Methylation fidelity exhibits a bimodal distribution with most and least methylated regions exhibiting the highest fidelity. Additionally, fidelity for methylation levels of 10-50% is considerably better than fidelity for levels of 50-90%. These data are from the mouse cell line ES-E14TG2a which is capable of self-renewal when cultured with Leukaemia Inhibitory Factor (LIF) and spontaneous differentiation upon removal of LIF (denoted as E14-d0 at day 0 and E14-d6 at day 6 after the withdrawal of LIF, respectively). Fidelity represents the percentage of symmetrically methylated or unmethylated CpG dyads for a given position as determined by Hairpin-Bisulfite PCR. (Reproduced from Zhao et al. 2014 (Zhao et al. 2014) (figure 3 a and b).)" width="80%" />
<p class="caption">
Figure 4.9: Bimodal Distribution of DNA methylation level and fidelity. Methylation fidelity exhibits a bimodal distribution with most and least methylated regions exhibiting the highest fidelity. Additionally, fidelity for methylation levels of 10-50% is considerably better than fidelity for levels of 50-90%. These data are from the mouse cell line ES-E14TG2a which is capable of self-renewal when cultured with Leukaemia Inhibitory Factor (LIF) and spontaneous differentiation upon removal of LIF (denoted as E14-d0 at day 0 and E14-d6 at day 6 after the withdrawal of LIF, respectively). Fidelity represents the percentage of symmetrically methylated or unmethylated CpG dyads for a given position as determined by Hairpin-Bisulfite PCR. (Reproduced from Zhao et al. 2014 <span class="citation">(Zhao et al. <a href="#ref-Zhao2014" role="doc-biblioref">2014</a>)</span> (figure 3 a and b).)
</p>
</div>

</div>
<div id="dna-methylation-assays" class="section level2">
<h2><span class="header-section-number">4.5</span> DNA Methylation Assays</h2>
<p>There are numerous DNA methylation assays.
Some are designed to assay global DNAm levels producing a single global measurement of the amount of 5mC present in a sample, others assay DNAm in a manner traceble to a specific genomic locus.
These methods can be further subdivided into targeted methods where the loci to be examined are known in advance, and those where the loci covered are quasi-randomly sampled.
Both the targeted and untargeted methods vary in their granularity, from individual CpGs to large regions of the genome, on the order of megabases.
These assays also vary in their coverage of the genome from locus-specific approaches looking at small numbers of individual loci to whole genome methods providing methylation information on all ~28 million CpG sites.
Between these two extremes are ‘epigenome-wide’ methods which are focused with varying degrees of specificity on genomic regions of interest.</p>
<p>These DNA methylation assays generally exploit one of four following things:</p>
<ol style="list-style-type: decimal">
<li><p><strong>Methylation sensitive restriction digestion</strong>
A variety of methods exploit enzymes which differentially cut methylated and unmethylated DNA.
An example of such enzymes is: HpaII and MspI which are isoschizomers for the sequence: 5’-C^CGG-3’ but only HpaII is methylation sensitive and unable to cut the sequence if the internal cytosine is methylated.
The methods which use such enzymes include: Restriction landmark genomic scanning, which gives a roughly megabase resolution indicators of DNAm levels by 2D electrophoresis <span class="citation">(Hatada et al. <a href="#ref-Hatada1991" role="doc-biblioref">1991</a>)</span>.
MRE-seq, which makes use of methylation-sensitive restriction digests to enrich for unmethylated DNA <span class="citation">(Maunakea et al. <a href="#ref-Maunakea2010" role="doc-biblioref">2010</a>)</span>.
This is then sequenced to provide the genomic location of this unmethylated signal.
Reduced Representation Bisulfite Sequencing (RRBS), which uses methylation sensitive restriction digestion to enrich for unmethylated sequences which are then subject to bisulfite sequencing, (see point 3 below).</p></li>
<li><p><strong>DNA methylation-sensitive binding of DNA by antibodies or other proteins selectively which bind methylated DNA</strong>
Anti-5mC antibodies were first used to isolate methylated DNA in 1985 and were subsequently paired with array-based technologies to assay DNAm levels at specific loci in 2005 <span class="citation">(Weber et al. <a href="#ref-Weber2005" role="doc-biblioref">2005</a>; Harrison and Parle-McDermott <a href="#ref-Harrison2011" role="doc-biblioref">2011</a>)</span>.
This was followed by MeDIP-seq (methylated DNA immuno-precipitation and sequencing) <span class="citation">(Down et al. <a href="#ref-Down2009" role="doc-biblioref">2008</a>; Lienhard et al. <a href="#ref-Lienhard2014" role="doc-biblioref">2014</a>)</span>, which enriches methylated DNA that is then sequenced and the number of reads mapping to a locus is indicative of the relative methylation level.
MBD-seq uses a recombinant Methyl Binding Domain (MBD) protein to enrich for methylated double-stranded DNA, prior to sequencing <span class="citation">(Serre, Lee, and Ting <a href="#ref-Serre2009a" role="doc-biblioref">2010</a>)</span>.
Inferring the absolute methylation levels form the number of reads mapping to a locus enriched by one of these pull-down methods is complicated by variation in CpG density in the genome and the fact that CpG dense regions tend to have low methylation levels, and thus tend not be pulled down by antibodies binding 5mC or MBD proteins.
The best results for estimating absolute methylation levels with these methods come from pairing them with a complementary method to enrich for unmethylated sites such as MRE-seq (described above) <span class="citation">(Li et al. <a href="#ref-Li2015" role="doc-biblioref">2015</a>)</span>.
Though relative methylation levels remain effective in identifying differentially methylated regions when using these methods alone <span class="citation">(Bock et al. <a href="#ref-Bock2010" role="doc-biblioref">2010</a>; Maunakea et al. <a href="#ref-Maunakea2010" role="doc-biblioref">2010</a>)</span>.</p></li>
<li><p><strong>Sodium bisulfite conversion of unmethylated cytosines to uracil</strong>
Conversion of cytosine to uracil changes the base complementary to this site from G to A <span class="citation">(Hayatsu et al. <a href="#ref-Hayatsu1970" role="doc-biblioref">1970</a>)</span>.
The basic chemistry of the conversion process is illustrated in Figure <a href="fundamentals-of-dna-methylation.html#fig:BisulfiteConversionNEB">4.10</a>.
This conversion can be detected with a variety of technologies such as:
The Illumina bead chip methylation arrays; WGBS; and Targeted bisulfite PCR or sequencing to examine a small number of selected loci.
It can also be preformed on a larger number of loci with microfuidic multiplexing such as that provided by the Fluidigm access array <span class="citation">(Adamowicz, Maratou, and Aitman <a href="#ref-Adamowicz2018" role="doc-biblioref">2018</a>)</span>.
It is worth noting that bisulfite conversion cannot differentiate between methylated and hydroxymethylated cytosines.
Because 5hmC represents a small fraction of modified bases compared to 5mC many analyses have made the working assumption that unconverted bases are methylated, but it is beginning to be recognised that distinguishing between the two may be biologically important especially in tissues where 5hmC is more prevalent such as neurons.
A variant of bisulfite conversion, oxidative bisulfite conversion, exists which can permit 5hmC to be distinguished from 5mC <span class="citation">(Skvortsova et al. <a href="#ref-Skvortsova2017" role="doc-biblioref">2017</a>)</span>.
Additionally New England Biolabs has recently developed an enzymatic alternative to chemical bisulfite conversion <span class="citation">(Williams et al. <a href="#ref-Williams2019" role="doc-biblioref">2019</a>)</span></p></li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:BisulfiteConversionNEB"></span>
<img src="figs/BisulfiteConversionNEB.jpg" alt="Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs)." width="80%" />
<p class="caption">
Figure 4.10: Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs).
</p>
</div>

<ol start="4" style="list-style-type: decimal">
<li><strong>Physical differences in the methylated base</strong>
These are exploited by the not yet widely used ‘3rd generation’ sequencing technologies such as nanopore and single-molecule real-time sequencing.
These methods respectively exploit the effects of modified bases on ion flow through the nanopore and impact on incorporation time of new bases whilst in the polymerase <span class="citation">(Rhoads and Au <a href="#ref-Rhoads2015" role="doc-biblioref">2015</a>; Simpson et al. <a href="#ref-Simpson2017" role="doc-biblioref">2017</a>)</span>.</li>
</ol>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Adamowicz2018">
<p>Adamowicz, Martyna, Klio Maratou, and Timothy J Aitman. 2018. “Multiplexed DNA Methylation Analysis of Target Regions Using Microfluidics (Fluidigm).” In <em>Methods in Molecular Biology (Clifton, N.j.)</em>, 1708:349–63. <a href="https://doi.org/10.1007/978-1-4939-7481-8_18">https://doi.org/10.1007/978-1-4939-7481-8_18</a>.</p>
</div>
<div id="ref-Bauden2017">
<p>Bauden, Monika, Theresa Kristl, Roland Andersson, György Marko-Varga, and Daniel Ansari. 2017. “Characterization of histone-related chemical modifications in formalin-fixed paraffin-embedded and fresh-frozen human pancreatic cancer xenografts using LC-MS/MS.” <em>Laboratory Investigation</em> 97 (3): 279–88. <a href="https://doi.org/10.1038/labinvest.2016.134">https://doi.org/10.1038/labinvest.2016.134</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>Bell, Christopher G., Fei Gao, Wei Yuan, Leonie Roos, Richard J. Acton, Yudong Xia, Jordana Bell, et al. 2018. “Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci.” <em>Nature Communications</em> 9 (1): 8. <a href="https://doi.org/10.1038/s41467-017-01586-1">https://doi.org/10.1038/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Bell2017">
<p>Bell, Joshua S. K., and Paula M. Vertino. 2017. “Orphan CpG islands define a novel class of highly active enhancers.” <em>Epigenetics</em> 12 (6): 449–64. <a href="https://doi.org/10.1080/15592294.2017.1297910">https://doi.org/10.1080/15592294.2017.1297910</a>.</p>
</div>
<div id="ref-Bird2002">
<p>Bird, Adrian. 2002. “DNA methylation patterns and epigenetic memory.” <em>Genes &amp; Development</em> 16 (1): 6–21. <a href="https://doi.org/10.1101/gad.947102">https://doi.org/10.1101/gad.947102</a>.</p>
</div>
<div id="ref-Bock2010">
<p>Bock, Christoph, Eleni M Tomazou, Arie B Brinkman, Fabian Müller, Femke Simmer, Hongcang Gu, Natalie Jäger, Andreas Gnirke, Hendrik G Stunnenberg, and Alexander Meissner. 2010. “Quantitative comparison of genome-wide DNA methylation mapping technologies.” <em>Nature Biotechnology</em> 28 (10): 1106–14. <a href="https://doi.org/10.1038/nbt.1681">https://doi.org/10.1038/nbt.1681</a>.</p>
</div>
<div id="ref-Boks2009">
<p>Boks, Marco P., Eske M. Derks, Daniel J. Weisenberger, Erik Strengman, Esther Janson, Iris E. Sommer, René S. Kahn, and Roel A. Ophoff. 2009. “The relationship of DNA methylation with age, gender and genotype in twins and healthy controls.” Edited by Joseph Najbauer. <em>PloS One</em> 4 (8): e6767. <a href="https://doi.org/10.1371/journal.pone.0006767">https://doi.org/10.1371/journal.pone.0006767</a>.</p>
</div>
<div id="ref-Bostick2007">
<p>Bostick, M., J. K. Kim, P.-O. Esteve, A. Clark, S. Pradhan, and S. E. Jacobsen. 2007. “UHRF1 Plays a Role in Maintaining DNA Methylation in Mammalian Cells.” <em>Science</em> 317 (5845): 1760–4. <a href="https://doi.org/10.1126/science.1147939">https://doi.org/10.1126/science.1147939</a>.</p>
</div>
<div id="ref-Briggs2010">
<p>Briggs, Adrian W., Udo Stenzel, Matthias Meyer, Johannes Krause, Martin Kircher, and Svante Pääbo. 2010. “Removal of deaminated cytosines and detection of in vivo methylation in ancient DNA.” <em>Nucleic Acids Research</em> 38 (6): e87. <a href="https://doi.org/10.1093/nar/gkp1163">https://doi.org/10.1093/nar/gkp1163</a>.</p>
</div>
<div id="ref-Ciernia2016">
<p>Ciernia, Annie Vogel, and Janine LaSalle. 2016. “The landscape of DNA methylation amid a perfect storm of autism aetiologies.” <em>Nature Reviews Neuroscience</em> 17 (7): 411–23. <a href="https://doi.org/10.1038/nrn.2016.41">https://doi.org/10.1038/nrn.2016.41</a>.</p>
</div>
<div id="ref-Down2009">
<p>Down, Thomas A, Vardhman K Rakyan, Daniel J Turner, Paul Flicek, Heng Li, Eugene Kulesha, Stefan Gräf, et al. 2008. “A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis.” <em>Nature Biotechnology</em> 26 (7): 779–85. <a href="https://doi.org/10.1038/nbt1414">https://doi.org/10.1038/nbt1414</a>.</p>
</div>
<div id="ref-Duncan1980">
<p>Duncan, B K, and J H Miller. 1980. “Mutagenic deamination of cytosine residues in DNA.” <em>Nature</em> 287 (5782): 560–1. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6999365">http://www.ncbi.nlm.nih.gov/pubmed/6999365</a>.</p>
</div>
<div id="ref-Ehrlich1982">
<p>Ehrlich, Melanie, Miguel A. Gama-Sosa, Lan-Hsiang Huang, Rose Marie Midgett, Kenneth C. Kuo, Roy A. McCune, and Charles Gehrke. 1982. “Amount and distribution of 5-methylcytosine in human DNA from different types of tissues or cells.” <em>Nucleic Acids Research</em> 10 (8): 2709–21. <a href="https://doi.org/10.1093/nar/10.8.2709">https://doi.org/10.1093/nar/10.8.2709</a>.</p>
</div>
<div id="ref-Esteve2006">
<p>Estève, Pierre-Olivier, Hang Gyeong Chin, Andrea Smallwood, George R. Feehery, Omkaram Gangisetty, Adam R. Karpf, Michael F. Carey, and Sriharsa Pradhan. 2006. “Direct interaction between DNMT1 and G9a coordinates DNA and histone methylation during replication.” <em>Genes &amp; Development</em> 20 (22): 3089–3103. <a href="https://doi.org/10.1101/gad.1463706">https://doi.org/10.1101/gad.1463706</a>.</p>
</div>
<div id="ref-Harrison2011">
<p>Harrison, Alan, and Anne Parle-McDermott. 2011. “DNA Methylation: A Timeline of Methods and Applications.” <em>Frontiers in Genetics</em> 2 (3): 632–49. <a href="https://doi.org/10.3389/fgene.2011.00074">https://doi.org/10.3389/fgene.2011.00074</a>.</p>
</div>
<div id="ref-Hashimshony2003">
<p>Hashimshony, Tamar, Jianmin Zhang, Ilana Keshet, Michael Bustin, and Howard Cedar. 2003. “The role of DNA methylation in setting up chromatin structure during development.” <em>Nature Genetics</em> 34 (2): 187–92. <a href="https://doi.org/10.1038/ng1158">https://doi.org/10.1038/ng1158</a>.</p>
</div>
<div id="ref-Hatada1991">
<p>Hatada, I, Y Hayashizaki, S Hirotsune, H Komatsubara, and T Mukai. 1991. “A genomic scanning method for higher organisms using restriction sites as landmarks.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 88 (21): 9523–7. <a href="https://doi.org/10.1073/pnas.88.21.9523">https://doi.org/10.1073/pnas.88.21.9523</a>.</p>
</div>
<div id="ref-Hayatsu1970">
<p>Hayatsu, Hikoya, Yusuke Wataya, Kazushige Kai, and Shigeru Iida. 1970. “Reaction of sodium bisulfite with uracil, cytosine, and their derivatives.” <em>Biochemistry</em> 9 (14): 2858–65. <a href="https://doi.org/10.1021/bi00816a016">https://doi.org/10.1021/bi00816a016</a>.</p>
</div>
<div id="ref-Huff2014">
<p>Huff, Jason T., and Daniel Zilberman. 2014. “Dnmt1-Independent CG Methylation Contributes to Nucleosome Positioning in Diverse Eukaryotes.” <em>Cell</em> 156 (6): 1286–97. <a href="https://doi.org/10.1016/j.cell.2014.01.029">https://doi.org/10.1016/j.cell.2014.01.029</a>.</p>
</div>
<div id="ref-Illingworth2010">
<p>Illingworth, Robert S., Ulrike Gruenewald-Schneider, Shaun Webb, Alastair R W Kerr, Keith D. James, Daniel J. Turner, Colin Smith, David J. Harrison, Robert Andrews, and Adrian P. Bird. 2010. “Orphan CpG islands identify numerous conserved promoters in the mammalian genome.” Edited by Wolf Reik. <em>PLoS Genetics</em> 6 (9): e1001134. <a href="https://doi.org/10.1371/journal.pgen.1001134">https://doi.org/10.1371/journal.pgen.1001134</a>.</p>
</div>
<div id="ref-Irizarry2009">
<p>Irizarry, Rafael A, Christine Ladd-Acosta, Bo Wen, Zhijin Wu, Carolina Montano, Patrick Onyango, Hengmi Cui, et al. 2009. “The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.” <em>Nature Genetics</em> 41 (2): 178–86. <a href="https://doi.org/10.1038/ng.298">https://doi.org/10.1038/ng.298</a>.</p>
</div>
<div id="ref-Jenkinson2017">
<p>Jenkinson, Garrett, Elisabet Pujadas, John Goutsias, and Andrew P Feinberg. 2017. “Potential energy landscapes identify the information-theoretic nature of the epigenome.” <em>Nature Genetics</em> 49 (5): 719–29. <a href="https://doi.org/10.1038/ng.3811">https://doi.org/10.1038/ng.3811</a>.</p>
</div>
<div id="ref-Haluskova2010">
<p>Jenuwein, T. 2001. “Translating the Histone Code.” <em>Science</em> 293 (5532): 1074–80. <a href="https://doi.org/10.1126/science.1063127">https://doi.org/10.1126/science.1063127</a>.</p>
</div>
<div id="ref-Jia2009">
<p>Jia, Da, Renata Z. Jurkowska, Xing Zhang, Albert Jeltsch, and Xiaodong Cheng. 2007. “Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation.” <em>Nature</em> 449 (7159): 248–51. <a href="https://doi.org/10.1038/nature06146">https://doi.org/10.1038/nature06146</a>.</p>
</div>
<div id="ref-Kelly2012">
<p>Kelly, Theresa K, Yaping Liu, Fides D Lay, Gangning Liang, Benjamin P Berman, and Peter a Jones. 2012. “Genome-wide mapping of nucleosome positioning and DNA methylation within individual DNA molecules.” <em>Genome Research</em> 22 (12): 2497–2506. <a href="https://doi.org/10.1101/gr.143008.112">https://doi.org/10.1101/gr.143008.112</a>.</p>
</div>
<div id="ref-Kerkel2008">
<p>Kerkel, Kristi, Alexandra Spadola, Eric Yuan, Jolanta Kosek, Le Jiang, Eldad Hod, Kerry Li, et al. 2008. “Genomic surveys by methylation-sensitive SNP analysis identify sequence-dependent allele-specific DNA methylation.” <em>Nature Genetics</em> 40 (7): 904–8. <a href="https://doi.org/10.1038/ng.174">https://doi.org/10.1038/ng.174</a>.</p>
</div>
<div id="ref-Kunkel2004a">
<p>Kunkel, Thomas A. 2004. “DNA Replication Fidelity.” <em>Journal of Biological Chemistry</em> 279 (17): 16895–8. <a href="https://doi.org/10.1074/jbc.R400006200">https://doi.org/10.1074/jbc.R400006200</a>.</p>
</div>
<div id="ref-Laird2004">
<p>Laird, Charles D, Nicole D Pleasant, Aaron D Clark, Jessica L Sneeden, K M Anwarul Hassan, Nathan C Manley, Jay C Vary, Todd Morgan, R Scott Hansen, and Reinhard Stöger. 2004. “Hairpin-bisulfite PCR: assessing epigenetic methylation patterns on complementary strands of individual DNA molecules.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 101 (1): 204–9. <a href="https://doi.org/10.1073/pnas.2536758100">https://doi.org/10.1073/pnas.2536758100</a>.</p>
</div>
<div id="ref-Lander2001">
<p>Lander, E S, L M Linton, B Birren, C Nusbaum, M C Zody, J Baldwin, K Devon, et al. 2001. “Initial sequencing and analysis of the human genome.” <em>Nature</em> 409 (6822): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.</p>
</div>
<div id="ref-Li2015">
<p>Li, Jingqiu, Jie Yang, Ping Zhou, Yanping Le, Chengwei Zhou, Shaomin Wang, Dazhi Xu, Hui-Kuan Lin, and Zhaohui Gong. 2015. “Circular RNAs in cancer: novel insights into origins, properties, functions and implications.” <em>American Journal of Cancer Research</em> 5 (2): 472–80. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25973291">http://www.ncbi.nlm.nih.gov/pubmed/25973291</a>.</p>
</div>
<div id="ref-Lienert2011">
<p>Lienert, Florian, Christiane Wirbelauer, Indrani Som, Ann Dean, Fabio Mohn, and Dirk Schübeler. 2011. “Identification of genetic elements that autonomously determine DNA methylation states.” <em>Nature Genetics</em> 43 (11): 1091–7. <a href="https://doi.org/10.1038/ng.946">https://doi.org/10.1038/ng.946</a>.</p>
</div>
<div id="ref-Lienhard2014">
<p>Lienhard, Matthias, Christina Grimm, Markus Morkel, Ralf Herwig, and Lukas Chavez. 2014. “MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments.” <em>Bioinformatics</em> 30 (2): 284–86. <a href="https://doi.org/10.1093/bioinformatics/btt650">https://doi.org/10.1093/bioinformatics/btt650</a>.</p>
</div>
<div id="ref-Luo2014">
<p>Luo, Yanting, Xuemei Lu, and Hehuang Xie. 2014. “Dynamic Alu Methylation during Normal Development, Aging, and Tumorigenesis.” <em>BioMed Research International</em> 2014: 1–12. <a href="https://doi.org/10.1155/2014/784706">https://doi.org/10.1155/2014/784706</a>.</p>
</div>
<div id="ref-Lyko2017">
<p>Lyko, Frank. 2017. “The DNA methyltransferase family: a versatile toolkit for epigenetic regulation.” <em>Nature Reviews Genetics</em> 19 (2): 81–92. <a href="https://doi.org/10.1038/nrg.2017.80">https://doi.org/10.1038/nrg.2017.80</a>.</p>
</div>
<div id="ref-Maunakea2010">
<p>Maunakea, Alika K, Raman P Nagarajan, Mikhail Bilenky, Tracy J Ballinger, Cletus D’Souza, Shaun D Fouse, Brett E Johnson, et al. 2010. “Conserved role of intragenic DNA methylation in regulating alternative promoters.” <em>Nature</em> 466 (7303): 253–7. <a href="https://doi.org/10.1038/nature09165">https://doi.org/10.1038/nature09165</a>.</p>
</div>
<div id="ref-Pfeifer1990a">
<p>Pfeifer, G P, S D Steigerwald, R S Hansen, S M Gartler, and A D Riggs. 1990. “Polymerase chain reaction-aided genomic sequencing of an X chromosome-linked CpG island: methylation patterns suggest clonal inheritance, CpG site autonomy, and an explanation of activity state stability.” <em>Proceedings of the National Academy of Sciences</em> 87 (21): 8252–6. <a href="https://doi.org/10.1073/pnas.87.21.8252">https://doi.org/10.1073/pnas.87.21.8252</a>.</p>
</div>
<div id="ref-Raiber2017">
<p>Raiber, Eun-ang, Guillem Portella, Sergio Martínez Cuesta, Robyn Hardisty, Pierre Murat, Zhe Li, Mario Iurlaro, et al. 2018. “5-Formylcytosine organizes nucleosomes and forms Schiff base interactions with histones in mouse embryonic stem cells.” <em>Nature Chemistry</em> 10 (12): 1258–66. <a href="https://doi.org/10.1038/s41557-018-0149-x">https://doi.org/10.1038/s41557-018-0149-x</a>.</p>
</div>
<div id="ref-Rhoads2015">
<p>Rhoads, Anthony, and Kin Fai Au. 2015. “PacBio Sequencing and Its Applications.” <em>Genomics, Proteomics &amp; Bioinformatics</em> 13 (5): 278–89. <a href="https://doi.org/10.1016/j.gpb.2015.08.002">https://doi.org/10.1016/j.gpb.2015.08.002</a>.</p>
</div>
<div id="ref-Riggs2004a">
<p>Riggs, Arthur D, and Zhenggang Xiong. 2004. “Methylation and epigenetic fidelity.” <em>Proceedings of the National Academy of Sciences</em> 101 (1): 4–5. <a href="https://doi.org/10.1073/pnas.0307781100">https://doi.org/10.1073/pnas.0307781100</a>.</p>
</div>
<div id="ref-Rose2014">
<p>Rose, Nathan R., and Robert J. Klose. 2014. “Understanding the relationship between DNA methylation and histone lysine methylation.” <em>Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</em> 1839 (12): 1362–72. <a href="https://doi.org/10.1016/j.bbagrm.2014.02.007">https://doi.org/10.1016/j.bbagrm.2014.02.007</a>.</p>
</div>
<div id="ref-Saxonov2006">
<p>Saxonov, Serge, Paul Berg, and Douglas L Brutlag. 2006. “A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters.” <em>Proceedings of the National Academy of Sciences</em> 103 (5): 1412–7. <a href="https://doi.org/10.1073/pnas.0510310103">https://doi.org/10.1073/pnas.0510310103</a>.</p>
</div>
<div id="ref-Schalkwyk2010">
<p>Schalkwyk, Leonard C., Emma L. Meaburn, Rebecca Smith, Emma L. Dempster, Aaron R. Jeffries, Matthew N. Davies, Robert Plomin, and Jonathan Mill. 2010. “Allelic skewing of DNA methylation is widespread across the genome.” <em>American Journal of Human Genetics</em> 86 (2): 196–212. <a href="https://doi.org/10.1016/j.ajhg.2010.01.014">https://doi.org/10.1016/j.ajhg.2010.01.014</a>.</p>
</div>
<div id="ref-Serre2009a">
<p>Serre, David, Byron H. Lee, and Angela H. Ting. 2010. “MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome.” <em>Nucleic Acids Research</em> 38 (2): 391–99. <a href="https://doi.org/10.1093/nar/gkp992">https://doi.org/10.1093/nar/gkp992</a>.</p>
</div>
<div id="ref-Shoemaker2010">
<p>Shoemaker, Robert, Jie Deng, Wei Wang, and Kun Zhang. 2010. “Allele-specific methylation is prevalent and is contributed by CpG-SNPs in the human genome.” <em>Genome Research</em> 20 (7): 883–9. <a href="https://doi.org/10.1101/gr.104695.109">https://doi.org/10.1101/gr.104695.109</a>.</p>
</div>
<div id="ref-Simpson2017">
<p>Simpson, Jared T, Rachael E Workman, P C Zuzarte, Matei David, L J Dursi, and Winston Timp. 2017. “Detecting DNA cytosine methylation using nanopore sequencing.” <em>Nature Methods</em> 14 (4): 407–10. <a href="https://doi.org/10.1038/nmeth.4184">https://doi.org/10.1038/nmeth.4184</a>.</p>
</div>
<div id="ref-Skvortsova2017">
<p>Skvortsova, Ksenia, Elena Zotenko, Phuc-Loi Luu, Cathryn M. Gould, Shalima S. Nair, Susan J. Clark, and Clare Stirzaker. 2017. “Comprehensive evaluation of genome-wide 5-hydroxymethylcytosine profiling approaches in human DNA.” <em>Epigenetics &amp; Chromatin</em> 10 (1): 16. <a href="https://doi.org/10.1186/s13072-017-0123-7">https://doi.org/10.1186/s13072-017-0123-7</a>.</p>
</div>
<div id="ref-Smith2015">
<p>Smith, Rick W A, Cara Monroe, and Deborah A. Bolnick. 2015. “Detection of Cytosine methylation in ancient DNA from five native american populations using bisulfite sequencing.” Edited by Ludovic Orlando. <em>PloS One</em> 10 (5): e0125344. <a href="https://doi.org/10.1371/journal.pone.0125344">https://doi.org/10.1371/journal.pone.0125344</a>.</p>
</div>
<div id="ref-Stadler2011">
<p>Stadler, Michael B., Rabih Murr, Lukas Burger, Robert Ivanek, Florian Lienert, Anne Schöler, Erik van Nimwegen, et al. 2011. “DNA-binding factors shape the mouse methylome at distal regulatory regions.” <em>Nature</em> 480 (7378): 490–5. <a href="https://doi.org/10.1038/nature10716">https://doi.org/10.1038/nature10716</a>.</p>
</div>
<div id="ref-Ushijima2005">
<p>Ushijima, Toshikazu, Naoko Watanabe, Kimiko Shimizu, Kazuaki Miyamoto, Takashi Sugimura, and Atsushi Kaneda. 2005. “Decreased fidelity in replicating CpG methylation patterns in cancer cells.” <em>Cancer Research</em> 65 (1): 11–17. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15665274">http://www.ncbi.nlm.nih.gov/pubmed/15665274</a>.</p>
</div>
<div id="ref-Vertino2002">
<p>Vertino, Paula M., Jennifer A. Sekowski, Jennifer M. Coll, Nancy Applegren, Suhua Han, Robert J. Hickey, and Linda H. Malkas. 2002. “DNMT1 is a component of a multiprotein DNA replication complex.” <em>Cell Cycle (Georgetown, Tex.)</em> 1 (6): 416–23. <a href="https://doi.org/10.4161/cc.1.6.270">https://doi.org/10.4161/cc.1.6.270</a>.</p>
</div>
<div id="ref-Weber2005">
<p>Weber, Michael, Jonathan J Davies, David Wittig, Edward J Oakeley, Michael Haase, Wan L Lam, and Dirk Schübeler. 2005. “Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells.” <em>Nature Genetics</em> 37 (8): 853–62. <a href="https://doi.org/10.1038/ng1598">https://doi.org/10.1038/ng1598</a>.</p>
</div>
<div id="ref-Williams2019">
<p>Williams, Louise, Yanxia Bei, Heidi E Church, Nan Dai, Eileen T Dimalanta, Laurence M Ettwiller, Thomas C Evans, et al. 2019. “Enzymatic Methyl-seq: The Next Generation of Methylome Analysis BISULFITE SEQUENCING ALTERNATIVE METHODS FOR DETECTING 5mC AND 5hmC.” <a href="https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis">https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis</a>.</p>
</div>
<div id="ref-Wu2017">
<p>Wu, Xiaoji, and Yi Zhang. 2017. “TET-mediated active DNA demethylation: mechanism, function and beyond.” <em>Nature Reviews Genetics</em> 18 (9): 517–34. <a href="https://doi.org/10.1038/nrg.2017.33">https://doi.org/10.1038/nrg.2017.33</a>.</p>
</div>
<div id="ref-Yin2017">
<p>Yin, Yimeng, Ekaterina Morgunova, Arttu Jolma, Eevi Kaasinen, Biswajyoti Sahu, Syed Khund-Sayeed, Pratyush K. Das, et al. 2017. “Impact of cytosine methylation on DNA binding specificities of human transcription factors.” <em>Science (New York, N.Y.)</em> 356 (6337): eaaj2239. <a href="https://doi.org/10.1126/science.aaj2239">https://doi.org/10.1126/science.aaj2239</a>.</p>
</div>
<div id="ref-Zhao2014">
<p>Zhao, Lei, Ming-an Sun, Zejuan Li, Xue Bai, Miao Yu, Min Wang, Liji Liang, et al. 2014. “The dynamics of DNA methylation fidelity during mouse embryonic stem cell self-renewal and differentiation.” <em>Genome Research</em> 24 (8): 1296–1307. <a href="https://doi.org/10.1101/gr.163147.113">https://doi.org/10.1101/gr.163147.113</a>.</p>
</div>
<div id="ref-Ziller2013">
<p>Ziller, Michael J, Hongcang Gu, Fabian Müller, Julie Donaghey, Linus T-Y Tsai, Oliver Kohlbacher, Philip L De Jager, et al. 2013. “Charting a dynamic DNA methylation landscape of the human genome.” <em>Nature</em> 500 (7463): 477–81. <a href="https://doi.org/10.1038/nature12433">https://doi.org/10.1038/nature12433</a>.</p>
</div>
<div id="ref-Zuo2017">
<p>Zuo, Zheng, Basab Roy, Yiming Kenny Chang, David Granas, and Gary D Stormo. 2017. “Measuring quantitative effects of methylation on transcription factor–DNA binding affinity.” <em>Science Advances</em> 3 (11): eaao1799. <a href="https://doi.org/10.1126/sciadv.aao1799">https://doi.org/10.1126/sciadv.aao1799</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="epigenomics-overview.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="dna-methylation-and-ageing.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/01-intro.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
