<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>8 Illumina Methylation arrays | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="8 Illumina Methylation arrays | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="8 Illumina Methylation arrays | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="MeDIPseq.html"/>
<link rel="next" href="whole-genome.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="aims.html"><a href="aims.html"><i class="fa fa-check"></i>Aims</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="the-ageing-population-and-the-burden-of-disease.html"><a href="the-ageing-population-and-the-burden-of-disease.html"><i class="fa fa-check"></i><b>1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="2" data-path="the-biology-of-ageing.html"><a href="the-biology-of-ageing.html"><i class="fa fa-check"></i><b>2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="3" data-path="epigenomics-overview.html"><a href="epigenomics-overview.html"><i class="fa fa-check"></i><b>3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="4" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html"><i class="fa fa-check"></i><b>4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="4.1" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#structure-and-context"><i class="fa fa-check"></i><b>4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="4.2" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="4.3" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="4.4" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="4.5" data-path="fundamentals-of-dna-methylation.html"><a href="fundamentals-of-dna-methylation.html#dna-methylation-assays"><i class="fa fa-check"></i><b>4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html"><i class="fa fa-check"></i><b>5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="5.1" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#DNAmAgeReview"><i class="fa fa-check"></i><b>5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="5.2" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="5.3" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="5.4" data-path="dna-methylation-and-ageing.html"><a href="dna-methylation-and-ageing.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="part"><span><b>II Methods</b></span></li>
<li class="chapter" data-level="7" data-path="MeDIPseq.html"><a href="MeDIPseq.html"><i class="fa fa-check"></i><b>7</b> MeDIP-seq</a></li>
<li class="chapter" data-level="8" data-path="illumina-methylation-arrays.html"><a href="illumina-methylation-arrays.html"><i class="fa fa-check"></i><b>8</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="9" data-path="whole-genome.html"><a href="whole-genome.html"><i class="fa fa-check"></i><b>9</b> Whole Genome</a></li>
<li class="chapter" data-level="10" data-path="targeted-bisulfite-sequencing.html"><a href="targeted-bisulfite-sequencing.html"><i class="fa fa-check"></i><b>10</b> Targeted Bisulfite Sequencing</a></li>
<li class="chapter" data-level="11" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>11</b> Discussion</a></li>
<li class="part"><span><b>III Results 1</b></span></li>
<li class="chapter" data-level="12" data-path="MAVIDOS.html"><a href="MAVIDOS.html"><i class="fa fa-check"></i><b>12</b> Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="12.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#abstract-1"><i class="fa fa-check"></i><b>12.1</b> Abstract</a></li>
<li class="chapter" data-level="12.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#introduction"><i class="fa fa-check"></i><b>12.2</b> Introduction</a><ul>
<li class="chapter" data-level="12.2.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#background"><i class="fa fa-check"></i><b>12.2.1</b> Background</a></li>
<li class="chapter" data-level="12.2.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>12.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#methods"><i class="fa fa-check"></i><b>12.3</b> Methods</a><ul>
<li class="chapter" data-level="12.3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation"><i class="fa fa-check"></i><b>12.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="12.3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-models"><i class="fa fa-check"></i><b>12.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="12.3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>12.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="12.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#results"><i class="fa fa-check"></i><b>12.4</b> Results</a><ul>
<li class="chapter" data-level="12.4.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#summary-of-results"><i class="fa fa-check"></i><b>12.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="12.4.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#quality-control-qc"><i class="fa fa-check"></i><b>12.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="12.4.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation-1"><i class="fa fa-check"></i><b>12.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="12.4.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewass"><i class="fa fa-check"></i><b>12.4.4</b> EWASs</a></li>
<li class="chapter" data-level="12.4.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>12.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="12.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>12.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="12.5.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#section"><i class="fa fa-check"></i><b>12.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="12.6" data-path="MAVIDOS.html"><a href="MAVIDOS.html#discussion-1"><i class="fa fa-check"></i><b>12.6</b> Discussion</a><ul>
<li class="chapter" data-level="12.6.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#future-work"><i class="fa fa-check"></i><b>12.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>IV Results 2</b></span></li>
<li class="chapter" data-level="13" data-path="tRNApaper.html"><a href="tRNApaper.html"><i class="fa fa-check"></i><b>13</b> The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="13.1" data-path="tRNApaper.html"><a href="tRNApaper.html#abstract-2"><i class="fa fa-check"></i><b>13.1</b> Abstract</a></li>
<li class="chapter" data-level="13.2" data-path="tRNApaper.html"><a href="tRNApaper.html#introduction-1"><i class="fa fa-check"></i><b>13.2</b> Introduction</a></li>
<li class="chapter" data-level="13.3" data-path="tRNApaper.html"><a href="tRNApaper.html#Methods"><i class="fa fa-check"></i><b>13.3</b> Methods</a><ul>
<li class="chapter" data-level="13.3.1" data-path="tRNApaper.html"><a href="tRNApaper.html#participants"><i class="fa fa-check"></i><b>13.3.1</b> Participants</a></li>
<li class="chapter" data-level="13.3.2" data-path="tRNApaper.html"><a href="tRNApaper.html#tRNAanno"><i class="fa fa-check"></i><b>13.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="13.3.3" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylome-data"><i class="fa fa-check"></i><b>13.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="13.3.4" data-path="tRNApaper.html"><a href="tRNApaper.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>13.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="13.3.5" data-path="tRNApaper.html"><a href="tRNApaper.html#sortedbloodmethods"><i class="fa fa-check"></i><b>13.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="13.3.6" data-path="tRNApaper.html"><a href="tRNApaper.html#TCGAmethods"><i class="fa fa-check"></i><b>13.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="13.3.7" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmapmethods"><i class="fa fa-check"></i><b>13.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="13.3.8" data-path="tRNApaper.html"><a href="tRNApaper.html#mousemethods"><i class="fa fa-check"></i><b>13.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="13.3.9" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-figures"><i class="fa fa-check"></i><b>13.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="13.3.10" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>13.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="tRNApaper.html"><a href="tRNApaper.html#res"><i class="fa fa-check"></i><b>13.4</b> Results</a><ul>
<li class="chapter" data-level="13.4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>13.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="13.4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>13.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="13.4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>13.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="13.4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>13.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="13.4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>13.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="13.5" data-path="tRNApaper.html"><a href="tRNApaper.html#discussion-2"><i class="fa fa-check"></i><b>13.5</b> Discussion</a></li>
<li class="chapter" data-level="13.6" data-path="tRNApaper.html"><a href="tRNApaper.html#data-availability"><i class="fa fa-check"></i><b>13.6</b> Data availability</a></li>
<li class="chapter" data-level="13.7" data-path="tRNApaper.html"><a href="tRNApaper.html#code-availablity"><i class="fa fa-check"></i><b>13.7</b> Code availablity</a></li>
<li class="chapter" data-level="13.8" data-path="tRNApaper.html"><a href="tRNApaper.html#acknowledgements-1"><i class="fa fa-check"></i><b>13.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="13.9" data-path="tRNApaper.html"><a href="tRNApaper.html#author-contributions"><i class="fa fa-check"></i><b>13.9</b> Author Contributions</a></li>
<li class="chapter" data-level="13.10" data-path="tRNApaper.html"><a href="tRNApaper.html#references"><i class="fa fa-check"></i><b>13.10</b> References</a></li>
</ul></li>
<li class="part"><span><b>V Results 3</b></span></li>
<li class="chapter" data-level="14" data-path="abstract-3.html"><a href="abstract-3.html"><i class="fa fa-check"></i><b>14</b> Abstract</a></li>
<li class="chapter" data-level="15" data-path="AluIntro.html"><a href="AluIntro.html"><i class="fa fa-check"></i><b>15</b> Introduction</a></li>
<li class="chapter" data-level="16" data-path="methods-1.html"><a href="methods-1.html"><i class="fa fa-check"></i><b>16</b> Methods</a></li>
<li class="chapter" data-level="17" data-path="results-1.html"><a href="results-1.html"><i class="fa fa-check"></i><b>17</b> Results</a><ul>
<li class="chapter" data-level="17.1" data-path="results-1.html"><a href="results-1.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>17.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="17.2" data-path="results-1.html"><a href="results-1.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>17.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="17.3" data-path="results-1.html"><a href="results-1.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>17.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="17.4" data-path="results-1.html"><a href="results-1.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>17.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>18</b> Discussion</a></li>
<li class="chapter" data-level="19" data-path="methods-2.html"><a href="methods-2.html"><i class="fa fa-check"></i><b>19</b> Methods</a><ul>
<li class="chapter" data-level="19.1" data-path="methods-2.html"><a href="methods-2.html#participants-1"><i class="fa fa-check"></i><b>19.1</b> Participants</a></li>
<li class="chapter" data-level="19.2" data-path="methods-2.html"><a href="methods-2.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>19.2</b> Age-related DNA methylation changes</a></li>
<li class="chapter" data-level="19.3" data-path="methods-2.html"><a href="methods-2.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>19.3</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="19.4" data-path="methods-2.html"><a href="methods-2.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>19.4</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="19.5" data-path="methods-2.html"><a href="methods-2.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>19.5</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="19.6" data-path="methods-2.html"><a href="methods-2.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>19.6</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="19.7" data-path="methods-2.html"><a href="methods-2.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>19.7</b> Association of these variants with Age-related Disease</a><ul>
<li class="chapter" data-level="19.7.1" data-path="methods-2.html"><a href="methods-2.html#data-availability-1"><i class="fa fa-check"></i><b>19.7.1</b> Data availability</a></li>
<li class="chapter" data-level="19.7.2" data-path="methods-2.html"><a href="methods-2.html#code-availablity-1"><i class="fa fa-check"></i><b>19.7.2</b> Code availablity</a></li>
<li class="chapter" data-level="19.7.3" data-path="methods-2.html"><a href="methods-2.html#acknowledgements-2"><i class="fa fa-check"></i><b>19.7.3</b> Acknowledgements</a></li>
<li class="chapter" data-level="19.7.4" data-path="methods-2.html"><a href="methods-2.html#author-contributions-1"><i class="fa fa-check"></i><b>19.7.4</b> Author Contributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="20" data-path="final-words.html"><a href="final-words.html"><i class="fa fa-check"></i><b>20</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="illumina-methylation-arrays" class="section level1">
<h1><span class="header-section-number">8</span> Illumina Methylation arrays</h1>
<p>The Illumina methylation bead chip arrays make use of 50 bp long probes with sequences corresponding to the loci of interest in the genome.<br />
These probes are attached by their 5’ ends to beads which are stuck to the surface of a glass slide, several beads represent each locus at a single spot on the slide.
The positions of the bead spots are known and used to identify the probes.
The CpG of interest is located at the 3’ end of the probes.
Samples are prepared for the array assay by fragmentation, bisulfite conversion and amplification.
These sample fragments anneal to complementary probes on the array and unbound DNA is washed away.
A single DNA polymerase-mediated elongation step is carried out to permit a fluorescently labelled base to be incorporated at the end of each probe.
C and G are tagged with one fluorophore and A and T with another.
There are two probe designs one which makes use of two distinct bead types for each locus, the type I probes, and one which makes use of a single type of bead at each locus, the type II.
A scanner excites the two fluorophore channels with a laser and their respective fluorescence intensities at each spot are recorded with an imaging system. The ‘colour’ of the fluorophore incorporated signifies the methylation state and the intensity of the proportion of sites which are in that state.
The logic of which ‘colour’ corresponds to methylated or unmethylated varies with the probe type and Figure <a href="illumina-methylation-arrays.html#fig:Bibikova2011IlluminaMethylationBeadChip">8.1</a> illustrates and explains this in depth.</p>
<p>These arrays have now undergone several iterations <span class="citation">(Bibikova et al. <a href="#ref-Bibikova2009" role="doc-biblioref">2009</a>, <a href="#ref-Bibikova2011" role="doc-biblioref">2011</a>; Sandoval, Heyn, Moran, Serra-Musach, Pujana, et al. <a href="#ref-Moran2015" role="doc-biblioref">2011</a><a href="#ref-Moran2015" role="doc-biblioref">a</a>)</span>.
The current array probe design was preceded by the ‘golden gate’ array which used a variant on the SNP probe design with methylation-specific PCR, This array contained ~1.5k probes covering 371 genes with a strong focus of cancer-related genes <span class="citation">(Bibikova <a href="#ref-Bibikova2006" role="doc-biblioref">2006</a>)</span>.
The first array using the probe design outlined above was the ‘Infinium’ array which contained 27k type I probes focused on the promoter regions for 14,475 consensus coding genes with 110 miRNA promoters <span class="citation">(Bibikova et al. <a href="#ref-Bibikova2009" role="doc-biblioref">2009</a>)</span>.
The 450k array which had ~480k mixed type I and II probes covering 21,231 RefSeq genes, 26,658 CpG islands, 80,538 predicted enhancer regions and a variety of other features, including the MHC region <span class="citation">(Bibikova et al. <a href="#ref-Bibikova2011" role="doc-biblioref">2011</a>)</span>.
The ‘EPIC’ array has ~850k probes including &gt;90% of those on the 450k with greatly expanded coverage of loci with more dynamic methylation states than promoters and CGIs which tend to have relatively low methylation variability aiming to capture more functional methylation variation at regions such as enhancers <span class="citation">(Sandoval, Heyn, Moran, Serra-Musach, Pujana, et al. <a href="#ref-Moran2015" role="doc-biblioref">2011</a><a href="#ref-Moran2015" role="doc-biblioref">a</a>; Ziller et al. <a href="#ref-Ziller2013" role="doc-biblioref">2013</a>)</span>.
The ‘EPIC’ array also contains both Type I and Type II probes, with many of the new sites being type II.
The type I probes have greater dynamic range than the type II but take up twice as much space on the array so there is some trade-off between maximising the number of sites covered and the quality of data at those sites.</p>
<div class="figure" style="text-align: center"><span id="fig:Bibikova2011IlluminaMethylationBeadChip"></span>
<img src="figs/Bibikova2011IlluminaMethylationBeadChip.png" alt="Illumina Methylation Bead Chip Array Probe Designs The probes on the array are 50bp in length. In the type I probes unlike the type II probes the base that is incorporated (or not) in the elongation step is the base that follows the CpG site. The probes on the array are bound by the amplification products derived from the bisulfite converted DNA not direct conversion products. Thus, Cs in the sample DNA represent methylated positions and Ts unmethylated positions, as Ts take the place of Us create by bisulfite conversion during the aplification process. Type I: Panel A shows the type I probe design. Each locus is represented on two beads, a methylated and an unmethylated bead. Methylated Sample: When sample DNA binds a methylated probe the G at the second position in the CpG on the methylated probe will be complementary to the C marking the location of the 5mC. Extension will then be able to occur incorporating a fluorescently labeled base complementary to the base just 5’ of the 5mC position in the probe. On the unmethylated probe the C marking the position of the 5mC will not be complementary to the to the A base in the probe and thus extension cannot occur. No fluorescent base will be incorporated at the unmethylated probe. Unmethylated Sample: When sample DNA binds the methylated probe the T marking the position of the unmethylated C is not complementary to the G at the second position in the CpG site on the probe. Thus, no fluorescent base is incorporated. Whereas, on the unmethylated probe the T marking the position of the unmethylated C will be complementary to the A at the second position in the CpG site on the probe. Thus, a fluorescently labelled base can be incorporated. Type II: Panel B shows the Type II probe design. For the Type II probes only one probe represents each locus. In the Type II probes the incorporated base is at the second position in the CpG site. A fluorescently labelled A is always incorporated opposite a T in the sample DNA marking the position of an unmethylated C, and a fluorescently labelled G is always incorporated opposite a C marking the position of an unconverted 5mC in the sample DNA. For the Type II probes, in contrast to the Type I probes, methylation is always signalled in the green channel and unmethylation in the red channel. (figure reproduced from (Bibikova et al. 2011).)" width="90%" />
<p class="caption">
Figure 8.1: <strong>Illumina Methylation Bead Chip Array Probe Designs</strong> The probes on the array are 50bp in length. In the type I probes unlike the type II probes the base that is incorporated (or not) in the elongation step is the base that <strong>follows</strong> the CpG site. The probes on the array are bound by the amplification products derived from the bisulfite converted DNA not direct conversion products. Thus, Cs in the sample DNA represent methylated positions and Ts unmethylated positions, as Ts take the place of Us create by bisulfite conversion during the aplification process. <strong>Type I</strong>: Panel A shows the type I probe design. Each locus is represented on two beads, a methylated and an unmethylated bead. <em>Methylated Sample</em>: When sample DNA binds a methylated probe the G at the second position in the CpG on the methylated probe will be complementary to the C marking the location of the 5mC. Extension will then be able to occur incorporating a fluorescently labeled base complementary to the base just 5’ of the 5mC position in the probe. On the unmethylated probe the C marking the position of the 5mC will not be complementary to the to the A base in the probe and thus extension cannot occur. No fluorescent base will be incorporated at the unmethylated probe. <em>Unmethylated Sample</em>: When sample DNA binds the methylated probe the T marking the position of the unmethylated C is not complementary to the G at the second position in the CpG site on the probe. Thus, no fluorescent base is incorporated. Whereas, on the unmethylated probe the T marking the position of the unmethylated C will be complementary to the A at the second position in the CpG site on the probe. Thus, a fluorescently labelled base can be incorporated. <strong>Type II</strong>: Panel B shows the Type II probe design. For the Type II probes only one probe represents each locus. In the Type II probes the incorporated base is at the second position in the CpG site. A fluorescently labelled A is always incorporated opposite a T in the sample DNA marking the position of an unmethylated C, and a fluorescently labelled G is always incorporated opposite a C marking the position of an unconverted 5mC in the sample DNA. For the Type II probes, in contrast to the Type I probes, methylation is always signalled in the green channel and unmethylation in the red channel. (figure reproduced from <span class="citation">(Bibikova et al. <a href="#ref-Bibikova2011" role="doc-biblioref">2011</a>)</span>.)
</p>
</div>

<p>Methylation at each site on the array is commonly reported as a <span class="math inline">\(\beta\)</span> value, which correspond to the proportion of the sample DNA which was methylated at that site.</p>
<p><span class="math display">\[\frac{\beta=intensity[M]}{intensity[U] + intensity[M]}\]</span></p>
<p>Where <span class="math inline">\(M\)</span> = methylated and <span class="math inline">\(U\)</span> = unmethylated.</p>
<ul>
<li>masks?</li>
</ul>
<hr />
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Bibikova2006">
<p>Bibikova, Marina. 2006. “High-throughput DNA methylation profiling using universal bead arrays.” <em>Genome Research</em> 16 (3): 383–93. <a href="https://doi.org/10.1101/gr.4410706">https://doi.org/10.1101/gr.4410706</a>.</p>
</div>
<div id="ref-Bibikova2011">
<p>Bibikova, Marina, Bret Barnes, Chan Tsan, Vincent Ho, Brandy Klotzle, Jennie M. Le, David Delano, et al. 2011. “High density DNA methylation array with single CpG site resolution.” <em>Genomics</em> 98 (4): 288–95. <a href="https://doi.org/10.1016/j.ygeno.2011.07.007">https://doi.org/10.1016/j.ygeno.2011.07.007</a>.</p>
</div>
<div id="ref-Bibikova2009">
<p>Bibikova, Marina, Jennie Le, Bret Barnes, Shadi Saedinia-Melnyk, Lixin Zhou, Richard Shen, and Kevin L Gunderson. 2009. “Genome-wide DNA methylation profiling using Infinium assay.” <em>Epigenomics</em> 1 (1): 177–200. <a href="https://doi.org/10.2217/epi.09.14">https://doi.org/10.2217/epi.09.14</a>.</p>
</div>
<div id="ref-Moran2015">
<p>Sandoval, Juan, Holger Heyn, Sebastian Moran, Jordi Serra-Musach, Miguel A Pujana, Marina Bibikova, and Manel Esteller. 2011a. “Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome.” <em>Epigenetics</em> 6 (6): 692–702. <a href="https://doi.org/10.2217/epi.15.114">https://doi.org/10.2217/epi.15.114</a>.</p>
</div>
<div id="ref-Ziller2013">
<p>Ziller, Michael J, Hongcang Gu, Fabian Müller, Julie Donaghey, Linus T-Y Tsai, Oliver Kohlbacher, Philip L De Jager, et al. 2013. “Charting a dynamic DNA methylation landscape of the human genome.” <em>Nature</em> 500 (7463): 477–81. <a href="https://doi.org/10.1038/nature12433">https://doi.org/10.1038/nature12433</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="MeDIPseq.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="whole-genome.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/02-methods.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
