<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1 Introduction | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="1 Introduction | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1 Introduction | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="acknowledgements.html"/>
<link rel="next" href="methods.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#illumina-methylation-arrays"><i class="fa fa-check"></i><b>2.1</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="3.1" data-path="arrays.html"><a href="arrays.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="arrays.html"><a href="arrays.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#background"><i class="fa fa-check"></i><b>3.2.1</b> Background</a></li>
<li class="chapter" data-level="3.2.2" data-path="arrays.html"><a href="arrays.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>3.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="arrays.html"><a href="arrays.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#summary-of-results"><i class="fa fa-check"></i><b>3.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#quality-control-qc"><i class="fa fa-check"></i><b>3.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#functional-normalisation-1"><i class="fa fa-check"></i><b>3.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.4.4" data-path="arrays.html"><a href="arrays.html#ewass"><i class="fa fa-check"></i><b>3.4.4</b> EWASs</a></li>
<li class="chapter" data-level="3.4.5" data-path="arrays.html"><a href="arrays.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>3.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>3.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="3.5.1" data-path="arrays.html"><a href="arrays.html#section"><i class="fa fa-check"></i><b>3.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.6</b> Discussion</a><ul>
<li class="chapter" data-level="3.6.1" data-path="arrays.html"><a href="arrays.html#future-work"><i class="fa fa-check"></i><b>3.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#Methods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#participants-1"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="4.3.9" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-figures"><i class="fa fa-check"></i><b>4.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.3.10" data-path="tRNAs.html"><a href="tRNAs.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>4.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#discussion-1"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#data-availability"><i class="fa fa-check"></i><b>4.6</b> Data availability</a></li>
<li class="chapter" data-level="4.7" data-path="tRNAs.html"><a href="tRNAs.html#code-availablity"><i class="fa fa-check"></i><b>4.7</b> Code availablity</a></li>
<li class="chapter" data-level="4.8" data-path="tRNAs.html"><a href="tRNAs.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.9" data-path="tRNAs.html"><a href="tRNAs.html#author-contributions"><i class="fa fa-check"></i><b>4.9</b> Author Contributions</a></li>
<li class="chapter" data-level="4.10" data-path="tRNAs.html"><a href="tRNAs.html#references"><i class="fa fa-check"></i><b>4.10</b> References</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results-3-.html"><a href="results-3-.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="results-3-.html"><a href="results-3-.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="results-3-.html"><a href="results-3-.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="results-3-.html"><a href="results-3-.html#methods-2"><i class="fa fa-check"></i><b>5.3</b> Methods</a></li>
<li class="chapter" data-level="5.4" data-path="results-3-.html"><a href="results-3-.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>5.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.4.2" data-path="results-3-.html"><a href="results-3-.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="5.4.3" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>5.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="5.4.4" data-path="results-3-.html"><a href="results-3-.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>5.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="results-3-.html"><a href="results-3-.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="results-3-.html"><a href="results-3-.html#methods-3"><i class="fa fa-check"></i><b>5.6</b> Methods</a><ul>
<li class="chapter" data-level="5.6.1" data-path="results-3-.html"><a href="results-3-.html#participants-2"><i class="fa fa-check"></i><b>5.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="results-3-.html"><a href="results-3-.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>5.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="5.7.1" data-path="results-3-.html"><a href="results-3-.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>5.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="5.7.2" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>5.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.7.3" data-path="results-3-.html"><a href="results-3-.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.7.4" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.7.5" data-path="results-3-.html"><a href="results-3-.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<div id="the-ageing-population-and-the-burden-of-disease" class="section level2">
<h2><span class="header-section-number">1.1</span> The Ageing Population and The Burden of Disease</h2>
<p>The global population is ageing as is evident from the changing shape of the world population pyramid (figure <a href="introduction.html#fig:poppyramid">1.1</a>).
The increase in average lifespan underlying this demographic shift is projected to continue in industrialised nations with a probability of &gt;65% for women and 85% for men <span class="citation">(Kontis et al. <a href="#ref-Kontis2017" role="doc-biblioref">2017</a>)</span> (figure <a href="introduction.html#fig:Kontis2017-fig1">1.2</a>).
Ageing is the main risk factor for cancer, cardiovascular, neurodegenerative diseases, and many other conditions including osteoarthritis <span class="citation">(Niccoli and Partridge <a href="#ref-Niccoli2012" role="doc-biblioref">2012</a>)</span>. Furthermore, mortality from these conditions increases according to a logistic function with age <span class="citation">(Belikov <a href="#ref-Belikov2019" role="doc-biblioref">2019</a>)</span> (figure <a href="introduction.html#fig:Belikov2019-fig1">1.3</a>).
In addition to the non-infectious diseases, immunosenescence and frailty in old age contribute to increased mortality from infectious diseases, as exemplified by dramatically higher hospital admissions and deaths for respiratory disease during seasonal flu epidemics in the over 75s <span class="citation">(Fleming and Elliot <a href="#ref-Fleming2005" role="doc-biblioref">2005</a>)</span> (figure <a href="introduction.html#fig:Fleming2005-fig3">1.4</a>).
Ageing is a common underlying risk factor for many conditions and as this would lead one to expect the number of concomitant disorders and the proportion of persons with multiple morbidities increases with age <span class="citation">(Barnett et al. <a href="#ref-Barnett2012" role="doc-biblioref">2012</a>)</span> (figure <a href="introduction.html#fig:Barnett2012-fig1">1.5</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:poppyramid"></span>
<img src="figs/population-pryamids-1950-2019.png" alt="The Population is Getting Older Population pryamids for global population for A) 1950 and B) 2019 from populationpyramid.net" width="95%" />
<p class="caption">
Figure 1.1: <strong>The Population is Getting Older</strong> Population pryamids for global population for A) 1950 and B) 2019 from <a href="populationpyramid.net">populationpyramid.net</a>
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Kontis2017-fig1"></span>
<img src="figs/Kontis2017_fig1.jpg" alt="The Population is Likely to Continue Getting Older Median projected increase in life expectancy from birth in the period 2010 to 2030 for 35 industrial nations reproduced from Kontis et al. 2017 figure 1 (Kontis et al. 2017)." width="95%" />
<p class="caption">
Figure 1.2: <strong>The Population is Likely to Continue Getting Older</strong> Median projected increase in life expectancy from birth in the period 2010 to 2030 for 35 industrial nations reproduced from Kontis et al. 2017 figure 1 <span class="citation">(Kontis et al. <a href="#ref-Kontis2017" role="doc-biblioref">2017</a>)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Belikov2019-fig1"></span>
<img src="figs/Belikov2019_fig1.jpg" alt="Mortality from Age-Related Conditions Increases According to a Logistic Function with Age Morality rate in deaths per 100,000 at different ages for Atherosclerosis, Essential Hypertension, Insulin-dependent diabetes, Non-insulin-dependent diabetes, Alzheimer’s disease, and Parkinson’s disease. Showing Model fits for exponential, logistic and the sum of 2 logistic functions. Underlying Cause of Death data (1999-2015) were form the Centers for Disease Control and Prevention, Wide-ranging OnLine Data for Epidemiologic Research (CDC WONDER) database. Reproduced from Belikov 2019 figure 1 (Belikov 2019)." width="80%" />
<p class="caption">
Figure 1.3: <strong>Mortality from Age-Related Conditions Increases According to a Logistic Function with Age</strong> Morality rate in deaths per 100,000 at different ages for Atherosclerosis, Essential Hypertension, Insulin-dependent diabetes, Non-insulin-dependent diabetes, Alzheimer’s disease, and Parkinson’s disease. Showing Model fits for exponential, logistic and the sum of 2 logistic functions. Underlying Cause of Death data (1999-2015) were form the Centers for Disease Control and Prevention, Wide-ranging OnLine Data for Epidemiologic Research (CDC WONDER) database. Reproduced from Belikov 2019 figure 1 <span class="citation">(Belikov <a href="#ref-Belikov2019" role="doc-biblioref">2019</a>)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Fleming2005-fig3"></span>
<img src="figs/Fleming2005_fig3.jpg" alt="Mortality from Infectious Diseases Increases With Age Visits to General practitioners, hospital admissions and deaths from respiratory disease in persons aged: (a) 45-64, (b) 65-74 and (c) 75 years and over. Reproduced from Fleming et al. 2005 figure 3 (Fleming and Elliot 2005)." width="90%" />
<p class="caption">
Figure 1.4: <strong>Mortality from Infectious Diseases Increases With Age</strong> Visits to General practitioners, hospital admissions and deaths from respiratory disease in persons aged: (a) 45-64, (b) 65-74 and (c) 75 years and over. Reproduced from Fleming et al. 2005 figure 3 <span class="citation">(Fleming and Elliot <a href="#ref-Fleming2005" role="doc-biblioref">2005</a>)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Barnett2012-fig1"></span>
<img src="figs/Barnett2012_fig1.jpg" alt="Number and Frequency of Comorbidities increase with Age Number of chronic disorders by age group. reproduced from Barnett et al. 2012 figure 1 (Barnett et al. 2012)." width="80%" />
<p class="caption">
Figure 1.5: <strong>Number and Frequency of Comorbidities increase with Age</strong> Number of chronic disorders by age group. reproduced from Barnett et al. 2012 figure 1 <span class="citation">(Barnett et al. <a href="#ref-Barnett2012" role="doc-biblioref">2012</a>)</span>.
</p>
</div>

<p>Given the rapid increases in the rates of numerous diseases and the rate of death from those diseases as we age, there is a clear impetus to develop interventions to prevent and treat age-related diseases.
The common factor among all these conditions is their relationship to age, but age is not merely a shared risk factor but frequently the largest; outside of rare genetic conditions which predispose to early-onset.
Thus understanding the underlying biological mechanisms of the ageing process and intervening in those mechanisms, has the potential to be vastly more effective and economical <span class="citation">(Goldman <a href="#ref-Goldman2016" role="doc-biblioref">2016</a>)</span> than attempting to tackle diseases individually.
The discovery in 1993 by Cynthia Kenyon and others <span class="citation">(Kenyon et al. <a href="#ref-Kenyon1993" role="doc-biblioref">1993</a>)</span> that a single mutation in the <em>daf-2</em> gene of <em>C. elegans</em> could double the lifespan of this organism is often cited as an inflexion point in the confidence of the ageing research field that interventions in the process of ageing itself may be a realistic therapeutic target. What then do we know about the underlying biology of ageing?</p>
</div>
<div id="the-biology-of-ageing" class="section level2">
<h2><span class="header-section-number">1.2</span> The biology of Ageing</h2>
<p>Ageing is characterised by a progressive deterioration of physiological integrity with time, resulting in an increased risk of mortality and morbidity.
Theories of how and why this occurs are numerous with a history in the scientific literature stretching back to the late 1800s <span class="citation">(Medvedev <a href="#ref-Medvedev1990" role="doc-biblioref">1990</a>)</span>, setting aside, for now, theoretical frameworks for the process of ageing we will first consider the nature of these changes.
Changes at the organismal level have their origins in changes at the molecular and cellular scales. Lopez-Otin et al. 2013 <span class="citation">(López-Otín et al. <a href="#ref-Lopez-Otin2013" role="doc-biblioref">2013</a>)</span> identified nine hallmarks of ageing in their 2013 review.
These hallmarks meet the criteria of:
Manifesting during normal ageing.
Accelerating ageing when aggravated.
Retarding ageing when ameliorated.
The hallmarks are further subdivided and hierarchically arranged into primary, antagonistic and integrative categories.
Primary hallmarks are of unequivocally negative effect.
Antagonistic hallmarks arise as a response to the primary and are initially protective against these changes but become problems in their own right as they occur at higher levels over time.
Lastly, the integrative hallmarks directly impact on tissue function and homoeostasis.
The hallmarks are as follows:</p>
<p><strong>Primary Hallmarks - Causes of Damage</strong></p>
<ol style="list-style-type: decimal">
<li><p><em>Genomic instability</em>:
Damage to the genomes of our cells occurs over time and accumulates as we age, this damage ranges from point mutations to large structural mutations, as well as mutations in the mitochondrial genome.</p></li>
<li><p><em>Telomere attrition</em>:
Telomere shortening occurs during the ageing process, somatic cells do not generally express telomerase so with each division telomeres grow shorter.
Telomere exhaustion limits the replicative capacity of cells inducing replicative senescence, limiting regenerative potential but also protecting from the uncontrolled growth of cancer.</p></li>
<li><p><strong>Epigenetic alterations</strong>:
Patterns of DNA methylation and of a number of histone modifications exhibit age-related changes. These contribute to the dysregulation of gene expression and derepression of heterochromatin regions which also increases the risk of genomic instability.</p></li>
<li><p><em>Loss of proteostasis</em>:
production of protein folding chaperones in response to stress decreases with age, as does the activity of the two major proteolytic systems, the ubiquitin-proteasome system and the autophagy-lysosome system.
This leads to increased prevalence of misfolded, aberrant and aggregated proteins with age.</p></li>
</ol>
<p><strong>Antagonistic Hallmarks - Responses to damage</strong></p>
<ol start="5" style="list-style-type: decimal">
<li><p><em>Deregulated nutrient sensing</em>:
The IIS (insulin &amp; insulin-like growth signalling) and TOR (Target Of Rapamycin) pathways signal nutrient abundance, promoting anabolism.
The AMPK (5’ adenosine monophosphate-activated protein kinase) and Sirtuins signal nutrient scarcity, promoting catabolism.
Broadly speaking inhibiting the pro-anabolic pathways and activating the pro-catabolic pathways can extend lifespan.</p></li>
<li><p><em>Mitochondrial dysfunction</em>:
Mitochondrial ATP production drops off with and ROS (Reactive Oxygen Species) production increases. Elevated ROS do not appear to accelerate ageing nor do elevated antioxidants retard it.
Indicating the primary function of ROS in ageing is not straightforwardly a result of oxidative damage, but may result from its role in stress signalling.
Dysfunctional mitochondria exhibit an increased propensity to polarise in response to stress, impacting on apoptotic signalling and inflammatory responses.
This contributes to the increases in senescent cells and their chronic inflammatory phenotype.</p></li>
<li><p><em>Cellular senescence</em>:
Senescent cells are in stable cell-cycle arrest induced by shortened telomeres, other forms of DNA damage, INK4/ARF derepression, or a variety of mitogenic/oncogenic signals.
Induction of a senescent state in pre-cancerous cells protects against cancer and senescent cells are initially cleared by the immune system.
However, these cells accumulate in tissue as we age without contributing to function and with a pro-inflammatory secretory phenotype that promotes chronic inflammation.</p></li>
</ol>
<p><strong>Integrative Hallmarks - Proximal Causes of the Ageing phenotype</strong></p>
<ol start="8" style="list-style-type: decimal">
<li><p><em>Stem cell exhaustion</em>:
Adult stem cells capable of producing new cells for the regeneration of tissues undergo asymmetric divisions to replace themselves and produce replicative progenitor cells to replenish tissue.
Stem cells may accumulate damage and cease replicating or leave their quiescent state becoming overly replicative and potentially becoming senescent.
This disrupts the supply of new cells for the renewal of tissue.</p></li>
<li><p><em>Altered intercellular communication</em>:
The aforementioned cell-autonomous alterations alter external signalling behaviour of cells across intercellular signalling paradigms.
This can lead to feedback loops that exacerbate the problem. The high background of pro-inflammatory signals from senescent cells dilutes already declining immune response and fewer senescent cells are cleared.</p></li>
</ol>
<p>This work and therefore the following review of the literature focuses on the 3rd hallmark, epigenetic alterations, specifically the role of DNA methylation in ageing.
Booth &amp; Brunet <span class="citation">(Booth and Brunet <a href="#ref-Booth2016" role="doc-biblioref">2016</a>)</span> contend that epigenetic processes are a hub through which all of the other hallmarks of ageing are mediated and feed-back on one another (Figure <a href="introduction.html#fig:Booth2016fig3">1.6</a>).
One of these interactions is between DNA damage and chromatin state including DNAm.
Sinclair and Oberdoerffer <span class="citation">(Sinclair and Oberdoerffer <a href="#ref-Sinclair2009" role="doc-biblioref">2009</a>)</span> make the case that DNA damage and repair processes are disruptive to the epigenome and, with Hayano et al., link double-stranded DNA breaks to accelerated epigenetic ageing in a recent pre-print <span class="citation">(Hayano et al. <a href="#ref-Hayano2019" role="doc-biblioref">2019</a>)</span>.
Kane &amp; Sinclair <span class="citation">(Kane and Sinclair <a href="#ref-Kane2019" role="doc-biblioref">2019</a>)</span> make the case that reprogramming of the epigenome to a ‘younger’ state is a promising mode of intervention in the ageing process.
Whilst loss of genetic information is effectively irreversible, loss of epigenetic information is not necessarily so.
Dysregulation from the loss of epigenetic information is arguably proximate to age-related changes and may run ahead of significant genetic information loss such that intervening in the former may have beneficial effects in its own right and stave off the latter.</p>
<div class="figure" style="text-align: center"><span id="fig:Booth2016fig3"></span>
<img src="figs/Booth2016_F3_epigenomeHubAgingHallmarks_alpha.png" alt="Epigenetic Changes as a hub for the hallmarks of ageing. (reproduced from (Booth and Brunet 2016), figure 3)" width="60%" />
<p class="caption">
Figure 1.6: Epigenetic Changes as a hub for the hallmarks of ageing. (reproduced from <span class="citation">(Booth and Brunet <a href="#ref-Booth2016" role="doc-biblioref">2016</a>)</span>, figure 3)
</p>
</div>

<p>Prior to examining the specific relationship of epigenetic changes to ageing we take a step back to define and outline epigenomics and examine the fundamentals of DNA methylation biology.</p>
</div>
<div id="epigenomics---overview" class="section level2">
<h2><span class="header-section-number">1.3</span> Epigenomics - Overview</h2>
<p>Epigenetics generally refers to modifications to DNA and chromatin which do not affect the primary sequence of DNA bases, but which are to varying degrees stable and heritable.
The term derives from epigenesis and genetics originating with Conrad Waddington <span class="citation">(Waddington <a href="#ref-Waddington1942" role="doc-biblioref">2012</a>)</span>.
Epigenesis refers to the idea that organisms develop through the progressive differentiation of cells from the egg into adult tissues.
As modern genetics revealed that every cell contained a complete copy of the genome which was differentially utilised by the cells of adult organisms, the terms were fused to reflect the study of how this occurs.
The usage has evolved a little further since molecular biology began to elucidate the mechanisms involved in this process and the term now frequently refers to the study of these mechanisms and their effects in less explicitly developmental contexts.
Arthur Riggs et al. considered that heritability should be a criterion for a mark to be considered epigenetic, but this excludes many phenomena now commonly referred to using the term <span class="citation">(Russo, Martienssen, and Riggs <a href="#ref-Russo1996" role="doc-biblioref">1996</a>)</span>.
Requiring heritability results in a further definitional dispute over degrees of heritability; mitotic, meiotic, intergenerational, transgenerational and to what degree of fidelity?
Adrian Bird proposed the definition: “the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.” as a useful compromise <span class="citation">(Bird <a href="#ref-Bird2007" role="doc-biblioref">2007</a>)</span>.
Epigenomics refers to the totality of the epigenetic modifications present in a particular cell, tissue type or genome.
This definition appears to have caught on with at least some as: ‘These days, “epigenetics” more generally refers to all non-genomic information storage in cells including gene networks, chromatin structure and post-translational modifications to histones.’, according to Alice Kane and David Sinclair <span class="citation">(Kane and Sinclair <a href="#ref-Kane2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>Whilst an organism can generally be thought of as having a single genome, with exceptions such as Somatic mutations and Chimerism, it will have at least as many epigenomes as it has cell types.
There are on the order of <span class="math inline">\(10^{13}\)</span> cells in the human body <span class="citation">(Bianconi et al. <a href="#ref-Bianconi2013" role="doc-biblioref">2013</a>)</span>.
Estimates of the number of cell types vary with the resolution at which one deems cells functionally distinct <span class="citation">(Regev et al. <a href="#ref-Regev2017" role="doc-biblioref">2017</a>)</span>.
At this point in time, however, there is no clear definition of what constitutes a distinct cell type, indeed they are beginning to be defined by the distinct patterns of gene expression and epigenetic modifications they exhibit <span class="citation">(CellSystemsVoices <a href="#ref-CellSystemsCellType2017" role="doc-biblioref">2017</a>)</span>.
Consequently, the number epigenomes that could be considered distinct will likely ultimately approximate to the number of epigenomes which can usefully distinguish between sub-populations of cells.
A consortium has been established to produce a <a href="www.humancellatlas.org">Human Cell Atlas</a> <span class="citation">(Regev et al. <a href="#ref-Regev2017" role="doc-biblioref">2017</a>)</span> which aims to define all human cell types, and a cell type ontology <span class="citation">(CellOntology <a href="#ref-CellOntologyOBO" role="doc-biblioref">2017</a>)</span> exists.</p>
<p>There are a number of epigenetic modifications which can be subdivided into four broad categories:</p>
<ol style="list-style-type: decimal">
<li><p><strong>DNA modifications</strong></p>
<p>DNA methylation primarily takes place on Cytosine residues.
Cytosine methylation occurs principally in an mCpG sequence context but can also occur in mCpH, mCHG and mCHH (H=A/C/T) contexts; particularly in cells of the nervous system <span class="citation">(Guo et al. <a href="#ref-Guo2014a" role="doc-biblioref">2014</a>)</span> and embryonic stem cells where as many as 25% of all cytosines can be methylated in non-canonical contexts.
In contrast to methylation in a differentiated non-neuronal tissue such as foetal lung fibroblasts which is 99.98% in CpG context <span class="citation">(Lister et al. <a href="#ref-Lister2009" role="doc-biblioref">2009</a>; Schultz et al. <a href="#ref-Schultz2015" role="doc-biblioref">2015</a>)</span>.
5-methylcytosine (5mc) can be oxidised to produce and another modified DNA base 5-hydroxymethylcytosine (5hmC).
5hmC may act as an intermediate to demethylation and potentially has regulatory functions in its own right, reviewed in <span class="citation">(Pfeifer, Kadam, and Jin <a href="#ref-Pfeifer2013" role="doc-biblioref">2013</a>)</span>.
5hmC can be oxidised further to formylcytosine (5fC) and carboxylcytosine (5caC).
The extent to which these modifications are stable and functional is still being explored.
Other DNA bases can be methylated such as N6-methyladenine, but this occurs at a substantially lower frequency that 5mC and much less is known about their potential functions <span class="citation">(Wu et al. <a href="#ref-Wu2016" role="doc-biblioref">2016</a>)</span>.</p></li>
<li><p><strong>Histone tail modifications</strong></p>
<p>Histone proteins form disk-shaped octamers around which ~150bp of DNA can be wrapped to form a nucleosome.
The ‘tails’ are generally the N termini of the histone proteins, outside of the core globular domains, which protrude from the nucleosome structure.
Histone tails can be subject to post-translational modification which commonly takes place at lysine residues, other residues are subject to modifications but lysines are among the best characterised.
Examples of modifications include methylations with between one and three methyl groups per lysine, Acetylations, Ubiquitylation, SUMOylations, and phosphorylations (Reviewed in <span class="citation">Bannister and Kouzarides (<a href="#ref-Bannister2011" role="doc-biblioref">2011</a>)</span>).
Many of these marks can be generally classified as permissive or repressive but are frequently found in combinations of marks of opposing effect, rendering the interpretation of the ‘Histone Code’ extremely challenging <span class="citation">(Voigt et al. <a href="#ref-Voigt2012" role="doc-biblioref">2012</a>)</span>.
One approach to interpreting chromatin state is segmentation.
Segmentation makes use of pattern discovery techniques to divide the genome into discrete sections assigning these sections to a set of categories which can then be compared to existing annotations to provide functional labels <span class="citation">(Hoffman et al. <a href="#ref-Hoffman2012" role="doc-biblioref">2012</a>; Carrillo-de-Santa-Pau et al. <a href="#ref-Carrillo-de-Santa-Pau2017" role="doc-biblioref">2017</a>)</span>.
When successful the pattern discovery algorithm independently recapitulates our existing ontology of functional elements and hopefully provides new insights by for example highlighting regions not previously considered to be in a given functional category.
This is a “top-down” approach making use of the “sum” of available chromatin state data rather than trying to interpret the significance of a single type of mark.</p></li>
<li><p><strong>Histone Variants</strong></p>
<p>Alternatives to the canonical histones can be substituted in the nucleosome, examples of such alternative histones include H2A.Z and H3.3.
Alternative histones can alter chromatin structure and dynamics by altering nucleosome stability, binding different factors, and presenting a different substrate for histone modifications, thus altering their pattern. (reviewed in <span class="citation">Weber and Henikoff (<a href="#ref-Weber2014" role="doc-biblioref">2014</a>)</span>.)</p></li>
<li><p><strong>Some non-coding RNAs (ncRNA)</strong></p>
<p>A subset of ncRNAs, particularly those which persist in close association with chromatin can fall within the definition of epigenetic features (reviewed in <span class="citation">Kaikkonen, Lam, and Glass (<a href="#ref-Kaikkonen2011" role="doc-biblioref">2011</a>)</span>).
An example of such a long non-coding RNA (lncRNA) is Xist which is a key regulator of X inactivation <span class="citation">(Cerase et al. <a href="#ref-Cerase2015" role="doc-biblioref">2015</a>)</span>.</p></li>
</ol>
</div>
<div id="fundamentals-of-dna-methylation" class="section level2">
<h2><span class="header-section-number">1.4</span> Fundamentals of DNA Methylation</h2>
<div id="structure-and-context" class="section level3">
<h3><span class="header-section-number">1.4.1</span> Structure and Context</h3>
<p>DNA methylation is the most well-studied epigenetic modification <span class="citation">(Jenuwein <a href="#ref-Haluskova2010" role="doc-biblioref">2001</a>)</span>.
Specifically CpG methylation, the addition of a Methyl group to the 5 carbon of a cytosine base (figure <a href="introduction.html#fig:5mCvsC">1.7</a>), in a CpG dinucleotide context (figure <a href="introduction.html#fig:CpGvsGpC">1.8</a>).
(hereafter ‘DNA Methylation’ or ‘DNAm’ refers to 5mC CpG methylation unless stated otherwise.)</p>
<p>DNA methylation is stable and relatively easy to characterise from biological samples by comparison with other epigenetic marks, the methylation status of ancient DNA has even been characterised, see for example:
DNAm profiles have been produced from DNA extracted from mammoth and neandertal remains <span class="citation">(Briggs et al. <a href="#ref-Briggs2010" role="doc-biblioref">2010</a>)</span> and native American remains ranging in age from 230 to 4500 years <span class="citation">(Smith, Monroe, and Bolnick <a href="#ref-Smith2015" role="doc-biblioref">2015</a>)</span>.
5mC is quite chemically stable such that for most tissue sample preservation techniques for which DNA can be extracted from the sample DNAm can also be characterised.
DNA extracted with normal laboratory methods can be assayed for DNAm.
Histone modification status, on the other hand, is less robust against environmental stresses on samples and often requires considerably more laborious sample preparation to characterise <span class="citation">(Bauden et al. <a href="#ref-Bauden2017" role="doc-biblioref">2017</a>)</span>.
DNA methylation also offers the appearance of relative simplicity in comparison to the complex picture of the ‘histone code’ giving the impression that its useful interpretation may be more readily tractable.
However, interaction between DNAm and histone modifications is well documented <span class="citation">(Hashimshony et al. <a href="#ref-Hashimshony2003" role="doc-biblioref">2003</a>; Estève et al. <a href="#ref-Esteve2006" role="doc-biblioref">2006</a>; Rose and Klose <a href="#ref-Rose2014" role="doc-biblioref">2014</a>)</span> consequently, a fuller understanding of DNAm will likely require an understanding of histone modifications and their potential to feedback on and thus affect DNAm.</p>
<div class="figure" style="text-align: center"><span id="fig:5mCvsC"></span>
<img src="figs/5mC_vs_C.png" alt="The structure of 5-methylcytosine contrasted with Cytosine. (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 1.7: The structure of 5-methylcytosine contrasted with Cytosine. (Figure created by the Author.)
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:CpGvsGpC"></span>
<img src="figs/CpG_vs_GpC.png" alt="A schematic representation showing a Methylated CpG dinucleotide in its structural context. This is contrasted with a GpC dinucleotide, which cannot be superimposed on a CpG. upper left: methylated cytosine residues in a CpG dinucleotide, showing 5mC is 5’ of G on both DNA strands. upper right: unmethylated cytosine residues in a GpC dinucleotide, showing G is 5’ of C on both DNA strands. lower: superimposition of a GpC and CpG dinucleotide illustrating the mirrored nature of these structures and the impossibility of superimposing them through rotation. (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 1.8: A schematic representation showing a Methylated <strong>CpG</strong> dinucleotide in its structural context. This is contrasted with a <strong>GpC</strong> dinucleotide, which cannot be superimposed on a <strong>CpG</strong>. upper left: methylated cytosine residues in a <strong>CpG</strong> dinucleotide, showing 5mC is 5’ of G on both DNA strands. upper right: unmethylated cytosine residues in a <strong>GpC</strong> dinucleotide, showing G is 5’ of C on both DNA strands. lower: superimposition of a <strong>GpC</strong> and <strong>CpG</strong> dinucleotide illustrating the mirrored nature of these structures and the impossibility of superimposing them through rotation. (Figure created by the Author.)
</p>
</div>

</div>
<div id="distribution-and-global-trends" class="section level3">
<h3><span class="header-section-number">1.4.2</span> Distribution and Global Trends</h3>
<p>There are 28,299,634 CpGs <span class="citation">(Luo, Lu, and Xie <a href="#ref-Luo2014" role="doc-biblioref">2014</a>)</span> in the hg19 assembly of the human genome <span class="citation">(Lander et al. <a href="#ref-Lander2001" role="doc-biblioref">2001</a>)</span>.
Given that the GC content of the human genome is <span class="math inline">\(42\%\)</span> the prior probability of getting a CpG dinucleotide is: <span class="math inline">\(0.21 \times 0.21 = 0.0441\)</span>, <span class="math inline">\(4.41\%\)</span>.
CpGs represent <span class="math inline">\(\sim1.8\%\)</span> of the dinucleotides in the human genome ( <span class="math inline">\(28,299,634 \div (3.23\times10^9\div2) \approx 0.0175\)</span>).
Making them <span class="math inline">\(\sim2.5\)</span> fold less frequent than would be expected <em>a priori</em>.
Methylated cytosines are prone to deamination to thymines, resulting in mismatch lesions <span class="citation">(Duncan and Miller <a href="#ref-Duncan1980" role="doc-biblioref">1980</a>)</span>.
This increased mutagenic potential generally means they are selected against, accounting for at least some part of their under-representation in the genome.
The three major classes of repeat elements SINEs, LINEs and LTRs contain some <span class="math inline">\(46\%\)</span> of all CpG sites, with a further <span class="math inline">\(5\%\)</span> in other repetitive elements <span class="citation">(Luo, Lu, and Xie <a href="#ref-Luo2014" role="doc-biblioref">2014</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:CGIs"></span>
<img src="figs/CGI-Shores-Shelves-Seas_2.png" alt="Schematic representation of CpG Islands, shores, shelves and seas. CpG density declines from shores to seas and methylation increases (CpG density and methylation proportion not to scale). (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 1.9: Schematic representation of CpG Islands, shores, shelves and seas. CpG density declines from shores to seas and methylation increases (CpG density and methylation proportion not to scale). (Figure created by the Author.)
</p>
</div>

<p>CpGs are not uniformly distributed in the genome, they occur at higher frequency in some regions.
“CpG islands” or CGIs are regions of high CpG density.
The total number of CpGs in the UCSC repeat masked CGI annotation list of 28,691 CGIs is 1,990,729.
Therefore, these CpGs comprise <span class="math inline">\(\sim7.0\%\)</span> of the total number of CpGs the genome. The mean percentage of the sequence of these CGIs that is comprised of CpG dinucleotides is <span class="math inline">\(\sim18.5\%\)</span> and their mean length is 761bp.
Irizarry et al. found that the 2kb regions flanking CpG islands which they termed “CpG island shores” exhibited greater tissue-specific differential methylation than the islands themselves <span class="citation">(Irizarry et al. <a href="#ref-Irizarry2009" role="doc-biblioref">2009</a>)</span>.
This nomenclature has subsequently been expanded further with “CpG island shelves” which are 2kb - 4kb from the CGIs, and “seas” referring to the rest of the genome, see figure <a href="introduction.html#fig:CGIs">1.9</a>.</p>
<p>CGIs overlap the promoters of <span class="math inline">\(\sim70\%\)</span> of genes <span class="citation">(Saxonov, Berg, and Brutlag <a href="#ref-Saxonov2006" role="doc-biblioref">2006</a>)</span>, this, however, leaves roughly half of all CGIs as “orphans” not associated with a known transcription site.
These orphan CGIs frequently constitute cell-type specific enhancers <span class="citation">(Bell and Vertino <a href="#ref-Bell2017" role="doc-biblioref">2017</a>)</span> and alternate promoters <span class="citation">(Illingworth et al. <a href="#ref-Illingworth2010" role="doc-biblioref">2010</a>; Maunakea et al. <a href="#ref-Maunakea2010" role="doc-biblioref">2010</a>)</span>.
They are found in both intergenic and intragenic regions.
CpGs are distributed quite sparsely through the genome occurring at low density in “seas” and at increasing density in CGIs and their flanking regions, see figure <a href="introduction.html#fig:CGIs">1.9</a>.
Early work showed 70-80% of CpGs are constitutively methylated <span class="citation">(Ehrlich et al. <a href="#ref-Ehrlich1982" role="doc-biblioref">1982</a>)</span> (reviewed <span class="citation">Bird (<a href="#ref-Bird2002" role="doc-biblioref">2002</a>)</span>) Stadler et al. produced a more detailed picture of the distribution of CpG methylation in mouse embryonic stem cells (ESCs) <span class="citation">(Stadler et al. <a href="#ref-Stadler2011" role="doc-biblioref">2011</a>)</span> see figure <a href="introduction.html#fig:CpGfreqVsDNAm">1.10</a>.
It is a characteristic of CGIs that they are generally unmethylated.</p>
<div class="figure" style="text-align: center"><span id="fig:CpGfreqVsDNAm"></span>
<img src="figs/CpGfreqVsDNAm.png" alt="The fraction of CpGs with a given methylation level. Data from Whole-Genome Bisulfite Sequencing (WGBS) of Mouse embryonic stem cells. (Figure reproduced from Stadler et al. 2011 (Stadler et al. 2011) figure 1a)" width="80%" />
<p class="caption">
Figure 1.10: The fraction of CpGs with a given methylation level. Data from Whole-Genome Bisulfite Sequencing (WGBS) of Mouse embryonic stem cells. (Figure reproduced from Stadler et al. 2011 <span class="citation">(Stadler et al. <a href="#ref-Stadler2011" role="doc-biblioref">2011</a>)</span> figure 1a)
</p>
</div>

<p>The regulatory role played by DNA methylation varies with context and the scale at which it is examined.
For example methylation at a single CpG in a transcription factor binding site can determine whether or not the factor can bind at that site.
There are methylation-sensitive transcription factors which can have their affinity for DNA either increased or decreased by methylation <span class="citation">(Yin et al. <a href="#ref-Yin2017" role="doc-biblioref">2017</a>; Zuo et al. <a href="#ref-Zuo2017" role="doc-biblioref">2017</a>)</span>.
In addition, larger-scale changes in DNAm can through interactions with chromatin modifiers alter DNA compaction and more general accessibility of DNA regions for binding, and/or impacting on their topological organisation.
DNAm also varies with nucleosome occupancy with lower methylation in linker sequences than on nucleosome-associated DNA <span class="citation">(Kelly et al. <a href="#ref-Kelly2012" role="doc-biblioref">2012</a>; Huff and Zilberman <a href="#ref-Huff2014" role="doc-biblioref">2014</a>)</span>.
The oxidation products of 5mC can also affect chromatin organisation, 5-Formylcytosine can impact nucleosome positioning through covalently binding to histones <span class="citation">(Raiber et al. <a href="#ref-Raiber2017" role="doc-biblioref">2018</a>)</span>.
This raises the question if some DNA methylation occurs specifically to act as an intermediate step for regulatory mechanisms which utilise its oxidation products.</p>
</div>
<div id="pathways-of-dna-methylation-and-demethylation" class="section level3">
<h3><span class="header-section-number">1.4.3</span> Pathways of DNA methylation and demethylation</h3>
<p>CpG methylation is produced and maintained by DNA methyltransferase (DNMT) enzymes.
All the DNMT enzymes use S-adenosylmethionine as the source of the methyl donor group.
DNMTs form a covalent intermediate between a conserved cysteine residue and the target base, through a nucleophilic attack on the C6 position in the cytosine ring.
This is followed by the transfer of the S-adenosylmethionine methyl group to C5, and deprotonation of the C5 to reform the double bond between C5 and C6; which is mediated by a base provided by the enzyme (Figure <a href="introduction.html#fig:Lyko2017DNAmMechanism">1.11</a>).</p>
<p>DNMT1 is associated with the replication machinery <span class="citation">(Vertino et al. <a href="#ref-Vertino2002" role="doc-biblioref">2002</a>)</span> and reproduces the methylation state of the parent strand on the daughter strand during replication.
The largest of the DNMT family DNMT1 contains a ‘replication foci targeting sequence’ (RFTS) domain required for its targeting to replication forks.
DNMT1 specialises in recognising hemimethylated DNA and methylating the unmethylated C in a palindromic CpG dinucleotide site, the UHRF1 protein assists in the recognition of these sequences <span class="citation">(Bostick et al. <a href="#ref-Bostick2007" role="doc-biblioref">2007</a>)</span>.
DNMT3a and DNMT3b are responsible for <em>de novo</em> DNA methylation along with DNMT3L a catalytically inactive, but DNA-binding subunit <span class="citation">(Jia et al. <a href="#ref-Jia2009" role="doc-biblioref">2007</a>)</span>.
The location of <em>de novo</em> methylation by the DNMT3s is influenced by a number of factors including the Chromatin state and other DNA binding factors <span class="citation">(Lyko <a href="#ref-Lyko2017" role="doc-biblioref">2017</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Lyko2017DNAmMechanism"></span>
<img src="figs/Lyko2017DNAmMechanism.png" alt="Motif VI (ENV/PCQ) refers to the conserved catalytic motif in DNMTs. ‘B:’ Represents the enzyme supplied base (Figure reproduced from the review of DNMT biology by Lyko (2017) (fig. 1b))" width="80%" />
<p class="caption">
Figure 1.11: Motif VI (ENV/PCQ) refers to the conserved catalytic motif in DNMTs. ‘B:’ Represents the enzyme supplied base (Figure reproduced from the review of DNMT biology by <span class="citation">Lyko (<a href="#ref-Lyko2017" role="doc-biblioref">2017</a>)</span> (fig. 1b))
</p>
</div>

<p>DNMT2 is unlike the other members of the DNMT family in that it targets an RNA substrate.
DNMT2 is a tRNA methyltransferase which methylates a specific subset of tRNA genes, mostly Asp isoacceptors, at a site adjacent to the anticodon which protects them from endonucleolytic cleavage under stress conditions.
Unprotected tRNAs produce fragments which compete with small interfering RNAs interfering with their signalling pathways.</p>
<p>The mechanisms of the demethylation of DNA were more recently characterised than those which govern its methylation.
They are more complex and less well understood.
A distinction is drawn between active and passive demethylation, in passive demethylation, 5mC bases are diluted out in the process of DNA replication.
In the leading model of active demethylation, they are oxidised one or more times by an enzyme from the TET (Ten-Eleven-translocase) family.
They are then either passively removed by DNA replication or actively removed by a DNA glycosylase (thymine DNA glycosylase TGD) to create an apurinic site which is restored to a C by the Base Excision Repair (BER) pathway. This cycle of cytosine methylation and demethylation is illustrated in Figure <a href="introduction.html#fig:Wu2017TETsDNAmCycle">1.12</a> <span class="citation">(Wu and Zhang <a href="#ref-Wu2017" role="doc-biblioref">2017</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Wu2017TETsDNAmCycle"></span>
<img src="figs/Wu2017TETsDNAmCycle.png" alt="BER = Bases Excision Repair; AM = Active Modification; AR = Active Removal, PD = Passive Dilution (Figure reproduced from the review of TET mediated active demethylation by Wu and Zhang (2017) (fig. 1a))" width="60%" />
<p class="caption">
Figure 1.12: BER = Bases Excision Repair; AM = Active Modification; AR = Active Removal, PD = Passive Dilution (Figure reproduced from the review of TET mediated active demethylation by <span class="citation">Wu and Zhang (<a href="#ref-Wu2017" role="doc-biblioref">2017</a>)</span> (fig. 1a))
</p>
</div>

</div>
<div id="maintenance-and-fidelity" class="section level3">
<h3><span class="header-section-number">1.4.4</span> Maintenance and Fidelity</h3>
<p>DNA methylation is highly dependent on the underlying DNA sequence.
Sequence features of particular importance to determining DNA methylation status are; transcription factor and other DNA binding protein recognition motifs, and CpG density <span class="citation">(Lienert et al. <a href="#ref-Lienert2011" role="doc-biblioref">2011</a>; Ziller et al. <a href="#ref-Ziller2013" role="doc-biblioref">2013</a>)</span>.
Alterations in underlying DNA sequence such as SNPs and copy number variants (CNVs) can have a significant impact on methylation level and the susceptibility of the methylation level to change.
In addition, CNVs can result in dosage effects on measures of DNAm, causing regions to appear, respectively, substantially more or less methylated when fewer or greater copies than expected are present <span class="citation">(Boks et al. <a href="#ref-Boks2009" role="doc-biblioref">2009</a>; Kerkel et al. <a href="#ref-Kerkel2008" role="doc-biblioref">2008</a>; Schalkwyk et al. <a href="#ref-Schalkwyk2010" role="doc-biblioref">2010</a>; Shoemaker et al. <a href="#ref-Shoemaker2010" role="doc-biblioref">2010</a>; Bell et al. <a href="#ref-Bell2017a" role="doc-biblioref">2018</a>)</span>.
Despite the strong influence of sequence on methylation, global CpG methylation exhibits change over developmental time as illustrated in figure <a href="introduction.html#fig:DNAmDevTime">1.13</a> and exhibits tissue-specific changes in distribution and amount <span class="citation">(Ziller et al. <a href="#ref-Ziller2013" role="doc-biblioref">2013</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:DNAmDevTime"></span>
<img src="figs/Ciernia2016f3DNAmTime.PNG" alt="5mC levels over developmental time. 5mCH or 5-methylcytosine-(A, T or C) levels are represented on a separate (purple) Axis from the main 5mCG axis. PGC = Primordial Germ Cell. (Figure reproduced from Ciernia and LaSalle (2016) figure 3)" width="80%" />
<p class="caption">
Figure 1.13: 5mC levels over developmental time. 5mCH or 5-methylcytosine-(A, T or C) levels are represented on a separate (purple) Axis from the main 5mCG axis. PGC = Primordial Germ Cell. (Figure reproduced from <span class="citation">Ciernia and LaSalle (<a href="#ref-Ciernia2016" role="doc-biblioref">2016</a>)</span> figure 3)
</p>
</div>

<p>In mitosis, DNA methylation is inherited by daughter cells with an error rate on the order of <span class="math inline">\(1\times10^{-3}\)</span> per site per generation <span class="citation">(Ushijima et al. <a href="#ref-Ushijima2005" role="doc-biblioref">2005</a>)</span>, several orders of magnitude less than that of DNA replication (error rate of <span class="math inline">\(1\times10^{-7}\)</span> - <span class="math inline">\(1\times10^{-8}\)</span> <span class="citation">(Kunkel <a href="#ref-Kunkel2004a" role="doc-biblioref">2004</a>)</span>).
The fidelity of DNA methylation copying can be assayed by Hairpin-Bisulfite PCR (Polymerase Chain Reaction) <span class="citation">(Laird et al. <a href="#ref-Laird2004" role="doc-biblioref">2004</a>)</span>.
Laird et al. looked at two alleles of a portion of the CpG island from the human <em>FMR1</em> gene in uncultured lymphocytes, one hypermethylated and one hypomethylated.
In the hypermethylated allele, they found that 96% of sites methylated in the parent strand remained methylated in the daughter strand and 86% of unmethylated sites remained unmethylated.
By contrast in the hypomethylated allele, there were no methylated sites to be retained and &gt;99% of unmethylated sites remained so following replication.
Laird et al.’s work suggests a higher degree of overall methylation fidelity for hypomethylated DNA and a propensity for unmethylated sites in hypermethylated DNA to become methylated.</p>
<p>Change in DNAm levels over time (divisions) can be modelled using the differential equations <span class="citation">(Pfeifer et al. <a href="#ref-Pfeifer1990a" role="doc-biblioref">1990</a>)</span>, which predict that a fully methylated site and a fully unmethylated site will converge on an equilibrium level.
This level is determined by the probability of maintenance of the methylation state and of <em>de novo</em> methylation for a given locus.
This stochastic model of DNAm is in agreement with experimental findings <span class="citation">(Laird et al. <a href="#ref-Laird2004" role="doc-biblioref">2004</a>; Riggs and Xiong <a href="#ref-Riggs2004a" role="doc-biblioref">2004</a>)</span>, Figure <a href="introduction.html#fig:Riggs2004PNASfig2DNAmFidelity">1.14</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:Riggs2004PNASfig2DNAmFidelity"></span>
<img src="figs/Riggs2004PNASfig2DNAmFidelity.png" alt="\(E_m\) = probability of methylation maintenance, \(E_m\) = probability of de novo methylation. \(M\) &amp; \(U\) = the number of methylated and unmethylated molecules at specific CpG sites respectively. Modelling (Grey points) \(E_m,E_d\) values of 0.90,0.10; 0.95,0.05; 0.97,0.03 (outermost to innermost, two curves for each \(E_m,E_d\) set, one starting at \(M=1,U=0\) and one a \(M=0,U=1\). Experimental data (Black points) from the HpaII locus (site H3) in 17 clones of mouse cell line BML-2 which has a known methylation level of 50%. (Reproduced from (Riggs and Xiong 2004))" width="70%" />
<p class="caption">
Figure 1.14: <span class="math inline">\(E_m\)</span> = probability of methylation maintenance, <span class="math inline">\(E_m\)</span> = probability of <em>de novo</em> methylation. <span class="math inline">\(M\)</span> &amp; <span class="math inline">\(U\)</span> = the number of methylated and unmethylated molecules at specific CpG sites respectively. Modelling (<strong>Grey points</strong>) <span class="math inline">\(E_m,E_d\)</span> values of 0.90,0.10; 0.95,0.05; 0.97,0.03 (outermost to innermost, two curves for each <span class="math inline">\(E_m,E_d\)</span> set, one starting at <span class="math inline">\(M=1,U=0\)</span> and one a <span class="math inline">\(M=0,U=1\)</span>. Experimental data (<strong>Black points</strong>) from the <em>HpaII</em> locus (site H3) in 17 clones of mouse cell line BML-2 which has a known methylation level of 50%. (Reproduced from <span class="citation">(Riggs and Xiong <a href="#ref-Riggs2004a" role="doc-biblioref">2004</a>)</span>)
</p>
</div>

<p>Jenkinson et al. <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> used an information-theoretic approach, modelling DNAm as a binary communications channel using a 1-dimensional Ising model from the field of statistical physics.
This permitted them to examine properties of DNAm not accessible to conventional means of analysis which typically capture the mean methylation level and perhaps the variability for a given locus.
When considering methylation fidelity using this lens the maintenance of a given methylation state can be seen as an information processing task which requires the consumption of free energy in order to reduce the probability of error in the transmission of that methylation state.
Thus Relative Dissipated Energy (RDE) can serve as a measure of the work expended by a cell at a given locus in order to preserve the current methylation state of that locus.</p>
<p>Zhao et al <span class="citation">(Zhao et al. <a href="#ref-Zhao2014" role="doc-biblioref">2014</a>)</span> applied Hairpin-Bisulfite PCR genome-wide in mouse embryonic stem cells (ESCs).
They also found high degrees of methylation fidelity in hypomethylated regions such as CGIs and Promoters, as well as a high degree of fidelity in sites bound by transcription factors.
This is in agreement with the findings of Jenkinson et al. <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> who noted that entropy (methylation stochasticity) was lower and more variable in CGIs and TSS (transcription start sites).</p>
<p>Methylation inheritance fidelity is reduced in cancer <span class="citation">(Ushijima et al. <a href="#ref-Ushijima2005" role="doc-biblioref">2005</a>)</span> and increases with differentiation <span class="citation">(Zhao et al. <a href="#ref-Zhao2014" role="doc-biblioref">2014</a>)</span>.
The RDE (relative dissipated energy) at CGIs and TSSs is higher in differentiated tissues such as the brain, implying low entropy, and lower in embryonic stem cells, implying greater entropy <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> (see Figure <a href="introduction.html#fig:Zhao2014fig3ab">1.15</a>).
In addition, the correlation between CpG sites increases in cancer <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span>, suggesting reduced higher-level regulatory control and tendency to fall back on lower level feedback mechanisms leading to a return to the baseline equilibrium described above as the cell is expending less energy to maintain an out of equilibrium methylation state.</p>
<div class="figure" style="text-align: center"><span id="fig:Zhao2014fig3ab"></span>
<img src="figs/Zhao2014-fig3-ab.png" alt="Bimodal Distribution of DNA methylation level and fidelity. Methylation fidelity exhibits a bimodal distribution with most and least methylated regions exhibiting the highest fidelity. Additionally, fidelity for methylation levels of 10-50% is considerably better than fidelity for levels of 50-90%. These data are from the mouse cell line ES-E14TG2a which is capable of self-renewal when cultured with Leukaemia Inhibitory Factor (LIF) and spontaneous differentiation upon removal of LIF (denoted as E14-d0 at day 0 and E14-d6 at day 6 after the withdrawal of LIF, respectively). Fidelity represents the percentage of symmetrically methylated or unmethylated CpG dyads for a given position as determined by Hairpin-Bisulfite PCR. (Reproduced from Zhao et al. 2014 (Zhao et al. 2014) (figure 3 a and b).)" width="80%" />
<p class="caption">
Figure 1.15: Bimodal Distribution of DNA methylation level and fidelity. Methylation fidelity exhibits a bimodal distribution with most and least methylated regions exhibiting the highest fidelity. Additionally, fidelity for methylation levels of 10-50% is considerably better than fidelity for levels of 50-90%. These data are from the mouse cell line ES-E14TG2a which is capable of self-renewal when cultured with Leukaemia Inhibitory Factor (LIF) and spontaneous differentiation upon removal of LIF (denoted as E14-d0 at day 0 and E14-d6 at day 6 after the withdrawal of LIF, respectively). Fidelity represents the percentage of symmetrically methylated or unmethylated CpG dyads for a given position as determined by Hairpin-Bisulfite PCR. (Reproduced from Zhao et al. 2014 <span class="citation">(Zhao et al. <a href="#ref-Zhao2014" role="doc-biblioref">2014</a>)</span> (figure 3 a and b).)
</p>
</div>

</div>
<div id="dna-methylation-assays" class="section level3">
<h3><span class="header-section-number">1.4.5</span> DNA Methylation Assays</h3>
<p>There are numerous DNA methylation assays.
Some are designed to assay global DNAm levels producing a single global measurement of the amount of 5mC present in a sample, others assay DNAm in a manner traceable to a specific genomic locus.
These methods can be further subdivided into targeted methods where the loci to be examined are known in advance, and those where the loci covered are quasi-randomly sampled.
Both the targeted and untargeted methods vary in their granularity, from individual CpGs to large regions of the genome, on the order of megabases.
These assays also vary in their coverage of the genome from locus-specific approaches looking at small numbers of individual loci to whole-genome methods providing methylation information on all ~28 million CpG sites.
Between these two extremes are ‘epigenome-wide’ methods which are focused with varying degrees of specificity on genomic regions of interest.</p>
<p>These DNA methylation assays generally exploit one of four following things:</p>
<ol style="list-style-type: decimal">
<li><p><strong>Methylation sensitive restriction digestion</strong>
A variety of methods exploit enzymes which differentially cut methylated and unmethylated DNA.
An example of such enzymes is: HpaII and MspI which are isoschizomers for the sequence: 5’-C^CGG-3’ but only HpaII is methylation-sensitive and unable to cut the sequence if the internal cytosine is methylated.
The methods which use such enzymes include: Restriction landmark genomic scanning, which gives a roughly megabase resolution indicators of DNAm levels by 2D electrophoresis <span class="citation">(Hatada et al. <a href="#ref-Hatada1991" role="doc-biblioref">1991</a>)</span>.
MRE-seq, which makes use of methylation-sensitive restriction digests to enrich for unmethylated DNA <span class="citation">(Maunakea et al. <a href="#ref-Maunakea2010" role="doc-biblioref">2010</a>)</span>.
This is then sequenced to provide the genomic location of this unmethylated signal.
Reduced Representation Bisulfite Sequencing (RRBS), which uses methylation-sensitive restriction digestion to enrich for unmethylated sequences which are then subject to bisulfite sequencing, (see point 3 below).</p></li>
<li><p><strong>DNA methylation-sensitive binding of DNA by antibodies or other proteins selectively which bind methylated DNA</strong>
Anti-5mC antibodies were first used to isolate methylated DNA in 1985 and were subsequently paired with array-based technologies to assay DNAm levels at specific loci in 2005 <span class="citation">(Weber et al. <a href="#ref-Weber2005" role="doc-biblioref">2005</a>; Harrison and Parle-McDermott <a href="#ref-Harrison2011" role="doc-biblioref">2011</a>)</span>.
This was followed by MeDIP-seq (methylated DNA immunoprecipitation and sequencing) <span class="citation">(Down et al. <a href="#ref-Down2009" role="doc-biblioref">2008</a>; Lienhard et al. <a href="#ref-Lienhard2014" role="doc-biblioref">2014</a>)</span>, which enriches methylated DNA that is then sequenced and the number of reads mapping to a locus is indicative of the relative methylation level.
MBD-seq uses a recombinant Methyl Binding Domain (MBD) protein to enrich for methylated double-stranded DNA, prior to sequencing <span class="citation">(Serre, Lee, and Ting <a href="#ref-Serre2009a" role="doc-biblioref">2010</a>)</span>.
Inferring the absolute methylation levels form the number of reads mapping to a locus enriched by one of these pull-down methods is complicated by variation in CpG density in the genome and the fact that CpG dense regions tend to have low methylation levels, and thus tend not to be pulled down by antibodies binding 5mC or MBD proteins.
The best results for estimating absolute methylation levels with these methods come from pairing them with a complementary method to enrich for unmethylated sites such as MRE-seq (described above) <span class="citation">(Li et al. <a href="#ref-Li2015" role="doc-biblioref">2015</a>)</span>.
Though relative methylation levels remain effective in identifying differentially methylated regions when using these methods alone <span class="citation">(Bock et al. <a href="#ref-Bock2010" role="doc-biblioref">2010</a>; Maunakea et al. <a href="#ref-Maunakea2010" role="doc-biblioref">2010</a>)</span>.</p></li>
<li><p><strong>Sodium bisulfite conversion of unmethylated cytosines to uracil</strong>
Conversion of cytosine to uracil changes the base complementary to this site from G to A <span class="citation">(Hayatsu et al. <a href="#ref-Hayatsu1970" role="doc-biblioref">1970</a>)</span>.
The basic chemistry of the conversion process is illustrated in Figure <a href="introduction.html#fig:BisulfiteConversionNEB">1.16</a>.
This conversion can be detected with a variety of technologies such as:
The Illumina bead chip methylation arrays; WGBS; and Targeted bisulfite PCR or sequencing to examine a small number of selected loci.
It can also be performed on a larger number of loci with microfluidic multiplexing such as that provided by the Fluidigm access array <span class="citation">(Adamowicz, Maratou, and Aitman <a href="#ref-Adamowicz2018" role="doc-biblioref">2018</a>)</span>.
It is worth noting that bisulfite conversion cannot differentiate between methylated and hydroxymethylated cytosines.
Because 5hmC represents a small fraction of modified bases compared to 5mC many analyses have made the working assumption that unconverted bases are methylated, but it is beginning to be recognised that distinguishing between the two may be biologically important especially in tissues where 5hmC is more prevalent such as neurons.
A variant of bisulfite conversion, oxidative bisulfite conversion, exists which can permit 5hmC to be distinguished from 5mC <span class="citation">(Skvortsova et al. <a href="#ref-Skvortsova2017" role="doc-biblioref">2017</a>)</span>.
Additionally, New England Biolabs has recently developed an enzymatic alternative to chemical bisulfite conversion <span class="citation">(Williams et al. <a href="#ref-Williams2019" role="doc-biblioref">2019</a>)</span></p></li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:BisulfiteConversionNEB"></span>
<img src="figs/BisulfiteConversionNEB.jpg" alt="Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs)." width="80%" />
<p class="caption">
Figure 1.16: Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs).
</p>
</div>

<ol start="4" style="list-style-type: decimal">
<li><strong>Physical differences in the methylated base</strong>
These are exploited by the not yet widely used ‘3rd generation’ sequencing technologies such as nanopore and single-molecule real-time sequencing.
These methods respectively exploit the effects of modified bases on ion flow through the nanopore and impact on incorporation time of new bases whilst in the polymerase <span class="citation">(Rhoads and Au <a href="#ref-Rhoads2015" role="doc-biblioref">2015</a>; Simpson et al. <a href="#ref-Simpson2017" role="doc-biblioref">2017</a>)</span>.</li>
</ol>
</div>
</div>
<div id="dna-methylation-and-ageing" class="section level2">
<h2><span class="header-section-number">1.5</span> DNA Methylation and Ageing</h2>
<p>Returning now to the relationship between DNA methylation and ageing we will review a little of the history of that field, the roles thought to be played by DNAm in ageing and the utility of DNAm clocks as biomarkers of ageing.</p>
<p>Early work on DNA methylation and ageing used biochemical, chromatographic and radiolabeling techniques to assay global changes in the amount of 5mC.
These studies found differences in 5mC composition with cell type <span class="citation">(Ehrlich et al. <a href="#ref-Ehrlich1982" role="doc-biblioref">1982</a>)</span> but did not see changes with age.
Other work around that time however found differences in 5mC levels with the age of cells in culture <span class="citation">(Wilson and Jones <a href="#ref-Wilson1983" role="doc-biblioref">1983</a>)</span>.
Wilson et al. noted that DNA methylation decreased across several tens of generations of cell lines in culture, but not in immortal cell lines.
Immortal cell lines had lower absolute levels of 5mC to start with but remained constant over time.
Wilson et al. also cite earlier work by Romanov at al. 1981 <span class="citation">(Romanov and Vanyushin <a href="#ref-Romanov1981" role="doc-biblioref">1981</a>)</span> and Berdyshev el al. 1967 <span class="citation">(Berdyshev et al. <a href="#ref-Berdyshev1967" role="doc-biblioref">1967</a>)</span> which documented decreases in 5mC with age in cows and salmon, respectively.</p>
<p>Later work by Wilson et al. 1987 <span class="citation">(Wilson et al. <a href="#ref-Wilson1987" role="doc-biblioref">1987</a>)</span> in mice <em>P. leucopus</em> and <em>M. musculus</em> found decreases in 5mC with age and that the rate of decline was less in the longer-lived <em>P. leucopus</em> than in <em>M. musculus</em>.
Interestingly, a recent study using different methods by Cole et al. found no global differences between young and old mice in short and long-lived strains, however, their other observations would seem to corroborate these initial trends.
For the assayed sites long-lived mice exhibited 10x more hypermethylation than wild-type (WT), and WT mice had 3x more Differentially Methylated Regions significantly associated with age (DMRs/aDMRs) than did long-lived mice.
Notably, the WT and long-lived mice shared many of the same aDMRs which differed in their degree of methylation as opposed to affecting different sites in the genome <span class="citation">(Cole et al. <a href="#ref-Cole2017" role="doc-biblioref">2017</a>)</span>.
The lack of apparent global changes may be due to the biases of the reduced representation bisulfite sequencing (RRBS) method used.
RRBS uses a restriction enzyme-based approach to enrich for regions with high GC content such as CGIs which tend to have low levels of methylation, and consequently may not be sensitive to loss of methylation in generally hypermethylated regions which could contribute to a global trend <span class="citation">(Meissner et al. <a href="#ref-Meissner2005" role="doc-biblioref">2005</a>)</span>.</p>
<p>Wilson et al. also noted that the mitotic index of tissues did not relate to the loss of DNA methylation with age in tension with their earlier observations <em>in vitro</em>.
The persistent loss of 5mC over time and the dramatic changes in methylation seen in cancer cells lead Wilson et al. to suggest that dysregulation of DNA methylation may have a substantial role to play in the age-dependency of cancer risk and ageing more generally (reviewed by Feinberg and Tycko <span class="citation">(Feinberg and Tycko <a href="#ref-Feinberg2004" role="doc-biblioref">2004</a>)</span>).
It has been suggested that “Epimutations” may be able to substitute for mutations in the multi-hit model of carcinogenesis <span class="citation">(Knudson <a href="#ref-Knudson1971" role="doc-biblioref">1971</a>)</span>.
For example; hypermethylation of the <em>BRCA1</em> promoter <span class="citation">(Esteller et al. <a href="#ref-Esteller2000" role="doc-biblioref">2000</a>)</span>, Or, the development of Wilms’ tumour due to Beckwith–Wiedemann syndrome, a disorder arising from loss of imprinting of the gene encoding insulin-like growth factor 2 (IGF-2) leading to a double dose of IGF-2 protein <span class="citation">(Feinberg <a href="#ref-Feinberg2018" role="doc-biblioref">2018</a>)</span>.
Furthermore, the loss of methylation and increased entropy of closed chromatin regions may lead to increased susceptibility to structural mutations commonly found in cancer cells through the exposure of homologous sequences <span class="citation">(Feinberg and Tycko <a href="#ref-Feinberg2004" role="doc-biblioref">2004</a>; Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span>.</p>
<p>Methods which allowed the examination of changes in DNA methylation at known loci in the genome permitted a more nuanced picture of changes in DNA methylation over time to develop.
Fraga et al. <span class="citation">(Fraga et al. <a href="#ref-Fraga2005" role="doc-biblioref">2005</a>)</span> introduced the concept of “epigenetic drift” being the divergence of DNA methylation and other epigenetic modification patterns with time.
Fraga et al. looked at the divergence in DNA methylation along with global Histone H3 and H4 acetylation patterns between monozygotic (MZ) twins over a wide range of ages.
They observed that older twins had greater epigenetic differences with time, in DNA methylation as well as H3, H4 Acetylation.
Fraga et al. also noted that the divergence in epigenetic state was greater in twins who had lived longer apart and had different medical histories.
Figure <a href="introduction.html#fig:Fraga2005fig3">1.17</a> is a useful visual encapsulation of Fraga et al.’s DNA methylation results.
Slieker et al. <span class="citation">(Slieker et al. <a href="#ref-Slieker2016" role="doc-biblioref">2016</a>)</span> identified 6366 CpGs whose methylation variability increased with age using the Illumina 450k array platform on whole-blood from 3295 individuals.
Both the increase in variability with age and the increasing divergence of twins support a narrative of epigenetic dysregulation and increasing entropy with age.</p>
<div class="figure" style="text-align: center"><span id="fig:Fraga2005fig3"></span>
<img src="figs/Fraga2005fig3.png" alt="Mapping Chromosomal Regions of Differential DNA Methylation. Examples from Chromosomes 1, 3, 12 &amp; 17 are shown for a 3 and 50-year-old twin pair. Methylation status is shown by competitive hybridization of AIMS (amplification of inter-methylated sites) products to metaphase chromosomes. Green and Red signals indicate hypermethylation and hypomethylation events between twins, Yellow indicates little difference. Red and Green blocks adjacent to ideograms indicate areas of significant DNA methylation change. (Reproduced from (Fraga et al. 2005) (figure 3).)" width="80%" />
<p class="caption">
Figure 1.17: Mapping Chromosomal Regions of Differential DNA Methylation. Examples from Chromosomes 1, 3, 12 &amp; 17 are shown for a 3 and 50-year-old twin pair. Methylation status is shown by competitive hybridization of AIMS (amplification of inter-methylated sites) products to metaphase chromosomes. Green and Red signals indicate hypermethylation and hypomethylation events between twins, Yellow indicates little difference. Red and Green blocks adjacent to ideograms indicate areas of significant DNA methylation change. (Reproduced from <span class="citation">(Fraga et al. <a href="#ref-Fraga2005" role="doc-biblioref">2005</a>)</span> (figure 3).)
</p>
</div>

<div id="DNAmAgeReview" class="section level3">
<h3><span class="header-section-number">1.5.1</span> Age-Related Differential Methylation and Epigenetic Clocks</h3>
<p>With the advent of array-based techniques which permitted the measurement of the methylation status of specific CpGs placed strategically throughout the genome, researchers were able to undertake a more fine-grained analysis of the changes in DNA methylation with age.
This has allowed the prediction of chronological age from a metric of “DNA methylation age” (DNAm age), and interpretation of the differences in predicted and chronological age in terms of the pathophysiology of ageing.</p>
<p>The Illumina Golden gate array <span class="citation">(Bibikova <a href="#ref-Bibikova2006" role="doc-biblioref">2006</a>)</span> with 1505 probes targeted to cancer-related genes was the first of these.
Bjornsson et al. observed DNAm changes with age as well as noting substantial intra-individual differences.
In contrast earlier work by Eckhardt et al. <span class="citation">(Eckhardt et al. <a href="#ref-Eckhardt2006" role="doc-biblioref">2006</a>)</span> and Ehrlich et al. <span class="citation">(Ehrlich et al. <a href="#ref-Ehrlich1982" role="doc-biblioref">1982</a>)</span> found no age-related changes using averages across individuals.
Bjornsson et al. <span class="citation">(Bjornsson <a href="#ref-Bjornsson2008" role="doc-biblioref">2008</a>)</span> also found that the intra-individual differences in differential methylation with ageing were highly heritable by use of familial clustering.
Work by Boks et al. <span class="citation">(Boks et al. <a href="#ref-Boks2009" role="doc-biblioref">2009</a>)</span> also using the golden gate array corroborated Bjornsson et al.’s findings in monozygotic twins.</p>
<p>The Bjornsson and Boks studies used the golden gate array on peripheral blood samples, whereas Christensen et al. <span class="citation">(Christensen et al. <a href="#ref-Christensen2009" role="doc-biblioref">2009</a>)</span> used the golden gate array to look for age-related changes in DNAm in several tissue types.
CpG sites that are differentially methylated with age when searching across tissues were frequently specific to a small number of tissue types.
In addition, CpGs in CGIs tended to be hypermethylated with age and <em>vice versa</em>.
Christensen et al. also found that the methylation profiles for different tissues were highly predictive of tissue type.</p>
<p>Rakyan et al. <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>)</span> looked at age-related changes in DNAm using the Illumina 27k array <span class="citation">(Bibikova et al. <a href="#ref-Bibikova2009" role="doc-biblioref">2009</a>)</span> with whole blood samples in a discovery set.
They replicated their findings in sorted cell fractions to see if they could capture changes in DNAm that could be attributed to changes in blood cell-type composition over time, which they did not find.
Teschendorff et al. <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> found that promoters of targets of the polycomb group proteins (PCGTs) were more likely than non-PCGTs to become methylated with age <span class="citation">(Jung and Pfeifer <a href="#ref-Jung2015" role="doc-biblioref">2015</a>)</span>.
Repression of PCGTs is required for Stem cells to differentiate.
PCGTs whose methylation status was associated with age were associated with pre-neoplastic conditions in a large cohort using the 27k array with blood and epithelial cell samples.</p>
<p>Bocklandt et al. <span class="citation">(Bocklandt et al. <a href="#ref-Bocklandt2011" role="doc-biblioref">2011</a>)</span> also using the 27k array but with saliva samples created the first “epigenetic clock” used to predict the chronological age of donors based on the DNAm landscape of their cells, the mean error was 5.2 years.
Bocklandt et al. were able to achieve similar predictive accuracy with as few as 3 CpG sites in their model.
Koch et al. <span class="citation">(Koch and Wagner <a href="#ref-Koch2011" role="doc-biblioref">2011</a>)</span> used publicly available 27k array datasets from a variety of different tissues to train a model using a different statistical method but only achieved an error of 11 years.
They did, however, identify a number of CpGs also identified by Bocklandt et al.
Bell et al. <span class="citation">(J. T. Bell et al. <a href="#ref-Bell2012" role="doc-biblioref">2012</a>)</span> used 27k array data to look for correlations between differential methylation and age-related phenotypes as well as chronological age.
They found that few age-related phenotypes were correlated with differential methylation, only 5 CpG sites were identified for the 16 age-related phenotypes examined. Whereas 490 significant CpGs were associated with chronological age.
Many of the CpGs they identified persisted across tissue types and replicated in a second cohort, several had been identified previously by Rakyan et al. <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>)</span> and Bocklandt et al. <span class="citation">(Bocklandt et al. <a href="#ref-Bocklandt2011" role="doc-biblioref">2011</a>)</span>.</p>
<p>Heyn et al. <span class="citation">(Heyn et al. <a href="#ref-Heyn2012" role="doc-biblioref">2012</a>)</span> examined the DNA methylation status of a newborn and a centenarian in CD4+ T cells using whole-genome bisulfite sequencing (WGBS), along with a group of newborns and nonagenarians using the Illumina 450k array <span class="citation">(Bibikova et al. <a href="#ref-Bibikova2011" role="doc-biblioref">2011</a>)</span>.
Heyn et al. observed a global decrease in methylation from newborn to centenarian, as well as an intermediate level of methylation on a sample of intermediate age (see figure <a href="introduction.html#fig:Heyn2012F1b">1.18</a>).
This observation was replicated in 450k data.
In addition, Heyn et al. noted that adjacent CpGs normally exhibit a substantial degree of correlation in methylation status and that they were less well correlated with their neighbours with increasing age.
Interestingly Jenkinson et al. <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> found that correlation among nearby CpGs increased in cancer tissues.
Modelling work done by Affinito et al. <span class="citation">(Affinito et al. <a href="#ref-Affinito2016" role="doc-biblioref">2016</a>)</span> agrees with this correlation between neighbouring CpGs under physiological conditions, which is particularly pronounced in CpG dense regions like CGIs.
The greater physical proximity of CpGs in CpG dense regions means that greater correlation is expected.
Due to the fact that these CpGs are more likely to be affected by the same proteins and regulatory features than CpGs with greater distance separating them <span class="citation">(Haerter et al. <a href="#ref-Haerter2014" role="doc-biblioref">2014</a>)</span>.
Garagnani et al. used 450k array data from a cohort of 64 subjects aged 9-83 to identify those CpGs most well correlated with age as had been done with previous array technologies <span class="citation">(Garagnani et al. <a href="#ref-Garagnani2012" role="doc-biblioref">2012</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Heyn2012F1b"></span>
<img src="figs/Heyn2012F1b.png" alt="(ref:Heyn2012F1b-Cap)" width="80%" />
<p class="caption">
Figure 1.18: (ref:Heyn2012F1b-Cap)
</p>
</div>
<p>(ref:Heyn2012F1b-Cap) Global Hypomethylation with Age. NB = Newborn, Y26 = 26 year old, Y103 = 103 year old.
(Reproduced from <span class="citation">Heyn et al. (<a href="#ref-Heyn2012" role="doc-biblioref">2012</a>)</span> figure 1 b.)</p>
<p>In January of 2013 Hannum at al. <span class="citation">(Hannum et al. <a href="#ref-Hannum2013" role="doc-biblioref">2013</a>)</span> published a study using the Illumina 450k array and whole blood samples for 656 participants aged 19-101.
Their optimised model was able to predict age with an error of 3.9 years.
In December of the same year, Horvath <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span> published a study using multiple datasets, including Hannum’s data, some produced on the 27k and some the 450k array.
Both used ‘elastic net’ penalised multivariate regression models to identify CpGs which collectively provide the best predictor of DNAm age rather than the CpGs which are individually most highly correlated with age <span class="citation">(Friedman, Hastie, and Tibshirani <a href="#ref-Friedman2010" role="doc-biblioref">2010</a>)</span>.</p>
<p>Florath et al. <span class="citation">(Florath et al. <a href="#ref-Florath2014" role="doc-biblioref">2014</a>)</span> identified some additional age-associated CpGs from two large cohorts totalling N = 898 and an N = 67 8yr follow-up longitudinal cohort.
More than 3/4 of the CpG sites they identified began as hypomethylated and increased in methylation with age, a proportion likely skewed by the large number of 450k probes in CGIs and other typically low methylation regions.
Bacalini et al. <span class="citation">(Bacalini et al. <a href="#ref-Bacalini2015" role="doc-biblioref">2015</a>)</span> performed a meta-analysis of existing DNAm age datasets and employed a “region-centric” approach to try to identify loci larger than single CpGs which they anticipate will be more biologically meaningful than lone CpGs.
They found that their approach increased the number of common features identified using the Hannum et al. and Heyn et al. datasets.
Zaghlool et al. <span class="citation">(Zaghlool et al. <a href="#ref-Zaghlool2015" role="doc-biblioref">2015</a>)</span> performed 450k array DNAm age study in Qatari population they identified 12/88, 23/490 and 102/162 of the CpGs found by Bocklandt et al, Bell et al. and Florath et al. respectively. Zaghlool et al. also found that Horvath’s age predictor had an error of 3.7 years in their dataset.
This is in agreement with existing findings and indicates that ethnicity has no major effects on the DNAm age signature. Benton et al. <span class="citation">(Benton et al. <a href="#ref-Benton2017" role="doc-biblioref">2017</a>)</span> looked at changes in DNA methylation with age in a genetically isolated population on Norfolk Island with similar results to previous studies but identifying some novel age-associated CpGs.</p>
<p>Most DNAm relationships with age noted in previous studies have been linear, Johnson et al. <span class="citation">(Johnson et al. <a href="#ref-Johnson2017" role="doc-biblioref">2017</a>)</span> used the 450k array in peripheral blood samples.
They identified 21 CpGs whose DNAm changes at a rate that changes with age from an initial pool of 27,723 CpGs which were differentially methylated with age.
Two sites exhibited an increasing rate of increase in DNAm with age, and 18 sites a decreasing rate of increase.</p>
</div>
<div id="genetic-influences-on-dna-methylation" class="section level3">
<h3><span class="header-section-number">1.5.2</span> Genetic Influences on DNA Methylation</h3>
<p>Epigenetic variation falls on a continuum of genetic influence that can be summarised by three categories:</p>
<ol style="list-style-type: decimal">
<li><p><strong>pure</strong> - DNA sequence has no predictive value for epigenetic state.</p></li>
<li><p><strong>facilitated</strong> - DNA sequence biases epigenetic state.</p></li>
<li><p><strong>obligatory</strong> - DNA sequence permits exact prediction of epigenetic state.</p></li>
</ol>
<p>Regional methylation state is strongly influenced by genotype, by single nucleotide polymorphisms (SNPs) <span class="citation">(Smith et al. <a href="#ref-Smith2014a" role="doc-biblioref">2014</a>)</span> and by structural variants <span class="citation">(Bell et al. <a href="#ref-Bell2017a" role="doc-biblioref">2018</a>)</span>.
An example of an obligatory effect on methylation is a point mutation at a CpG site, a C to T transition precludes methylation at that site in future.
The effect of structural variants on methylation can be hard to determine as changes in sequence dosage often lead to measurement artefacts.
Efforts have been made to correct for the influence of genetic factors in EWAS, when searching for purely epigenetic effects but the potential interaction of the somatic mutations known to accumulate with age (Figure <a href="introduction.html#fig:Jaiswal2014fig1">1.19</a>) and the changes in DNAm with age remain largely unexplored.
This is of particular relevance to DNAm as the profile of the types of mutation which accumulate with age (Figure <a href="introduction.html#fig:Jaiswal2014fig2c">1.20</a>) distinctly favours C to T transition mutations which can disrupt CpG dinucleotides <span class="citation">(Jaiswal et al. <a href="#ref-Jaiswal2014c" role="doc-biblioref">2014</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Jaiswal2014fig1"></span>
<img src="figs/Jaiswal2014fig1_somaticMutationAge.png" alt="Somatic mutations increase with Age. (Reproduced from (Jaiswal et al. 2014), figure 1)" width="60%" />
<p class="caption">
Figure 1.19: Somatic mutations increase with Age. (Reproduced from <span class="citation">(Jaiswal et al. <a href="#ref-Jaiswal2014c" role="doc-biblioref">2014</a>)</span>, figure 1)
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Jaiswal2014fig2c"></span>
<img src="figs/Jaiswal2014fig2c.png" alt="C to T transitions are the most common substitutions that occur with age. (Reproduced from (Jaiswal et al. 2014), figure2 c)" width="60%" />
<p class="caption">
Figure 1.20: C to T transitions are the most common substitutions that occur with age. (Reproduced from <span class="citation">(Jaiswal et al. <a href="#ref-Jaiswal2014c" role="doc-biblioref">2014</a>)</span>, figure2 c)
</p>
</div>

</div>
<div id="dnam-age-as-a-biomarker-of-ageing" class="section level3">
<h3><span class="header-section-number">1.5.3</span> DNAm Age as a Biomarker of Ageing</h3>
<p>We have seen that DNAm can be a fairly accurate predictor of age, however, there is more to being a good biomarker of ageing than predictive accuracy.
Weidner et al. <span class="citation">(Weidner et al. <a href="#ref-Weidner2014" role="doc-biblioref">2014</a>)</span> conducted a study with the specific intent of identifying CpG sites that would serve as the best biomarkers of ageing.
Below is a list of specific criteria for a high-quality biomarker of ageing laid out by Johnson et al. <span class="citation">(Johnson <a href="#ref-Johnson2006" role="doc-biblioref">2006</a>)</span>.</p>
<ol style="list-style-type: decimal">
<li>It must predict the rate of ageing. In other words, it would tell exactly where a person is in their total lifespan. It must be a better predictor of lifespan than chronological age.</li>
<li>It must monitor a basic process that underlies the ageing process, not the effects of disease.</li>
<li>It must be able to be tested repeatedly without harming the person. For example, a blood test or an imaging technique.</li>
<li>It must be something that works in humans and laboratory animals, such as mice. This is so that it can be tested in lab animals before being validated in humans.</li>
</ol>
<p>We have seen that DNA methylation clocks can be good predictors of chronological age, we will revisit the question of whether or not they are superior to chronological age in their ability to predict lifespan below.
Bell et al. <span class="citation">(J. T. Bell et al. <a href="#ref-Bell2012" role="doc-biblioref">2012</a>)</span> found DNAm did not generally correlate well with ageing phenotypes but did correlate well with chronological age and Teschendorff et al. <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> found that their age-associated DNAm signature remained fairly constant across several disease states including ovarian cancer and type 1 diabetes.
Suggesting that DNAm age prediction does indeed: “monitor a basic process that underlies the ageing process, not the effects of disease”, meeting criterion number 2.
Several of the DNAm clocks discussed so far have been based on whole peripheral blood samples meeting criterion number 3. DNAm age clocks have been shown to work in Chimpanzees <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span>, Dogs/Wolves <span class="citation">(Thompson et al. <a href="#ref-Thompson2017" role="doc-biblioref">2017</a>)</span>, Mice <span class="citation">(Stubbs et al. <a href="#ref-Stubbs2017" role="doc-biblioref">2017</a>)</span>, Naked Mole Rats <span class="citation">(Lowe et al. <a href="#ref-Lowe2020" role="doc-biblioref">2020</a>)</span>, Rhesus Macaques, Humpback Whales <span class="citation">(Lowe et al. <a href="#ref-Lowe2018" role="doc-biblioref">2018</a>)</span> and are likely to work in other mammalian model organisms.
Furthermore, a cross-species clock based on conserved ribosomal RNA genes has been developed <span class="citation">(Wang and Lemos <a href="#ref-Wang2019" role="doc-biblioref">2019</a>)</span>.
It should be noted that Horvath’s Human clock uses array data and that the Dog/Wolf and Mouse clocks use RRBS.
Model organisms such as <em>C. elegans</em> which essentially lack DNA methylation would not, however, be likely to be tractable for DNAm based age prediction.
Consequently DNAm age prediction at least partially meets criterion number 4.</p>
<p>Is DNA methylation a better predictor of lifespan than chronological age?
Both Hannum and Horvath identified the difference between DNAm age and chronological age or <span class="math inline">\(\Delta_{Age}\)</span> as a potential indicator of disproportionate biological ageing.
Marioni et al. <span class="citation">(Marioni et al. <a href="#ref-Marioni2015" role="doc-biblioref">2015</a>)</span> explicitly set forth to test the value of DNAm age as a predictor of mortality.
Adjusting for age, sex, childhood IQ, education, social class, hypertension, diabetes, cardiovascular disease, and <em>APOE</em> e4 status a <span class="math inline">\(\Delta_{Age}\)</span> of +5yr is associated with a 16% increased mortality risk (See figure <a href="introduction.html#fig:Marioni2015f3">1.21</a>).
A longitudinal study of twins found 3.2 fold increase in the risk of dying first per 5yr <span class="math inline">\(\Delta_{Age}\)</span> within twin pairs <span class="citation">(Christiansen et al. <a href="#ref-Christiansen2016" role="doc-biblioref">2016</a>)</span>.
A meta-analysis of DNAm age predictors by Chen et al. <span class="citation">(Chen et al. <a href="#ref-Chen2016a" role="doc-biblioref">2016</a>)</span> also found that measures of age acceleration based on Hannum and Horvath clocks were superior predictors of mortality than chronological age before and after correction for various potentially confounding factors.
Chen et al. also noted that correction for blood cell composition improved predictive power.
Indicating that DNAm age is indeed superior to chronological age as a predictor of lifespan meeting criterion number 1 for an ageing biomarker.</p>
<p>A new DNAm age clock explicitly designed to capture phenotypic age and outperform age acceleration as a biomarker of ageing has recently been produced by Levine et al. <span class="citation">(Levine et al. <a href="#ref-Levine2018" role="doc-biblioref">2018</a>)</span>.
The ‘PhenoAge’ metric was created by selecting nine biomarkers of ageing from 42 possible metrics using a proportional hazards penalised regression model and combining these with chronological age.
(The nine biomarkers are: Albumin, Creatinine, Serum glucose, C-reactive protein, Lymphocyte per cent, Mean red cell volume, Red cell distribution width, Alkaline phosphatase, White blood cell count.)
DNAm data was then regressed against PhenoAge using elastic-net regression to produce a DNAm based PhenoAge predictor which made use of 513 CpGs.
The PhenoAge predictor outperformed the Horvath and Hannum clocks at predicting all-cause mortality, comorbidities, coronary heart disease risk, and measures of physical functioning.
41 of the 513 CpGs in the PhenoAge clock were present in the original 353 CpG sites used in the Horvath clock.</p>
<p>The ‘GrimAge’ clock <span class="citation">(Lu et al. <a href="#ref-Lu2019" role="doc-biblioref">2019</a>)</span> takes a different approach to previous epigenetic clocks in that it has a 2-tiered model predicting first seven surrogates for a variety of other biomarkers of ageing then fitting a model to predict time to death using these surrogates.
‘GrimAge’ outperforms age accelerations from the Horvath, Hannum and PhenoAge clocks at predicting time to death.</p>
<div class="figure" style="text-align: center"><span id="fig:Marioni2015f3"></span>
<img src="figs/Marioni2015f3.png" alt="Survival probability by quartiles of \(\Delta_{Age}\) in LBC 1921 adjusted for sex, and chronological age. LBC = Lothian Birth Cohort. Using the Hannum and Horvath predictors’ values for \(\Delta_{Age}\). (Reproduced from Marioni et al. (2015) figure 3.)" width="80%" />
<p class="caption">
Figure 1.21: Survival probability by quartiles of <span class="math inline">\(\Delta_{Age}\)</span> in LBC 1921 adjusted for sex, and chronological age. LBC = Lothian Birth Cohort. Using the Hannum and Horvath predictors’ values for <span class="math inline">\(\Delta_{Age}\)</span>. (Reproduced from <span class="citation">Marioni et al. (<a href="#ref-Marioni2015" role="doc-biblioref">2015</a>)</span> figure 3.)
</p>
</div>

</div>
<div id="discussion-and-analysis-of-the-findings-of-dnam-age-studies" class="section level3">
<h3><span class="header-section-number">1.5.4</span> Discussion and Analysis of the Findings of DNAm Age Studies</h3>
<p>Interestingly CpGs whose methylation status has been identified as correlating with age in various studies show relatively little overlap. Despite this, changes in DNAm with age are probably the most robust and reproducible large-scale epigenetic change yet captured.
This lack of agreement in absolute terms, however, presents a challenge about how to interpret DNAm changes with age.</p>
<p>The result of the various DNAm age studies display some seemingly contradictory trends.
For example, there is the global loss of methylation with age <span class="citation">(Wilson et al. <a href="#ref-Wilson1987" role="doc-biblioref">1987</a>; Fraga et al. <a href="#ref-Fraga2005" role="doc-biblioref">2005</a>; Heyn et al. <a href="#ref-Heyn2012" role="doc-biblioref">2012</a>)</span> and yet the majority of the highly significant age predicting CpGs in clocks are undergoing hypermethylation by a ratio of very roughly 2:1 <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>; Bocklandt et al. <a href="#ref-Bocklandt2011" role="doc-biblioref">2011</a>; Koch and Wagner <a href="#ref-Koch2011" role="doc-biblioref">2011</a>; Garagnani et al. <a href="#ref-Garagnani2012" role="doc-biblioref">2012</a>; Florath et al. <a href="#ref-Florath2014" role="doc-biblioref">2014</a>)</span> (see figure <a href="introduction.html#fig:Florath2014F4abc">1.22</a>).
Interestingly when expanded to look at all age-associated CpGs and not just the most highly correlated CpGs a slightly different picture emerges. Teschendorff et al. <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> noted in their original study and Zhang et al. <span class="citation">(Zhang et al. <a href="#ref-Zhang2017" role="doc-biblioref">2017</a>)</span> noted in a subsequent analysis if more loosely age correlated sites are included hypomethylation with age is more prevalent.
Zhang et al. state roughly 60% of sites are hypomethylated and 40% hypermethylated, they also observed that all of the highly age-predictive CpGs which overlapped between the studies they examined were hypomethylated.</p>
<p>It is worth noting that the technologies used to measure the DNAm introduce their own biases into global changes in DNAm levels.
The more Global measures such as WGBS and biochemical measures seem to favour hypomethylation with age <span class="citation">(Wilson et al. <a href="#ref-Wilson1987" role="doc-biblioref">1987</a>; Fraga et al. <a href="#ref-Fraga2005" role="doc-biblioref">2005</a>; Heyn et al. <a href="#ref-Heyn2012" role="doc-biblioref">2012</a>)</span>.
The arrays seem to have agreed more with these global measures as they have increased in size.
RRBS also seems to skew in favour of hypermethylation with age <span class="citation">(Cole et al. <a href="#ref-Cole2017" role="doc-biblioref">2017</a>)</span>.
These more narrow technologies are biased towards promoters and CGIs with their higher CpG density and lower baseline methylation levels making the prior probability of observing a methylation event greater than demethylation by dint of having a dearth of methylated sites to start with.</p>
<p>Sites which are hypomethylated as their “ground state” are more likely to be in CGIs than sites which are hypermethylated as their “ground state”, and thus hypomethylated sites are more likely to be variable in their methylation state and subject to tight regulatory control than are hypermethylated sites.
Expanding the pool to look at CpG sites more poorly correlated with age is likely to introduce more sites whose hypomethylation with age is due to increased stochasticity with age.
However, from the background information, one would expect that hypermethylated sites in CpG dense regions like CGIs whose methylation level declines with age would make good quality indicators.
Given that CpGs in CGIs have a high prior probability of being hypomethylated, it follows that one would be more likely to observe a hypermethylation event by chance.
Consequently, a CpG in a CGI that exhibits hypomethylation with age is less likely to be a result of noise than its converse, this may explain the higher reproducibility of these hypomethylated sites.</p>
<div class="figure" style="text-align: center"><span id="fig:Florath2014F4abc"></span>
<img src="figs/Florath2014_F4abc.png" alt="Mean methylation level by genomic region showing proportions negatively correlated with age, i.e. become hypomethylated as age increases. (Reproduced from Florath et al. (2014) figure4 a, b, c.)" width="80%" />
<p class="caption">
Figure 1.22: Mean methylation level by genomic region showing proportions negatively correlated with age, i.e. become hypomethylated as age increases. (Reproduced from <span class="citation">Florath et al. (<a href="#ref-Florath2014" role="doc-biblioref">2014</a>)</span> figure4 a, b, c.)
</p>
</div>

<p>In a review of DNAm and ageing Jones et al. <span class="citation">(Jones, Goodman, and Kobor <a href="#ref-Jones2015" role="doc-biblioref">2015</a>)</span> drew a distinction between epigenetic drift changes and epigenetic clock changes with age wherein they defined drift as “the collection of DNA methylation changes that are associated with age within an individual but are not common across individuals.”, which is slightly different from the definition used by Fraga et al. <span class="citation">(Fraga et al. <a href="#ref-Fraga2005" role="doc-biblioref">2005</a>)</span>.
Clock changes are defined as “those sites that are associated with age across individuals and can thus in some cases be used to predict chronological age”.
Drift constitutes stochastic change over time not driven by some process which strongly biases the resulting changes in DNAm in direction or to specific sites.
Clock changes are the converse, some underlying process biases DNAm changes in direction and/or position.</p>
<p>This picture is complicated by the tissue specificity of DNAm changes.
Whilst it is evidently the case that multi-tissue DNAm age prediction is possible, it also seems to be the case that an age predictor trained on samples from a specific tissue will outperform one trained on multiple or other tissues when predicting the age of a sample of matching tissue type.
This is supported by the out-performance of the Horvath predictor by the Hannum predictor in blood samples <span class="citation">(Marioni et al. <a href="#ref-Marioni2015" role="doc-biblioref">2015</a>)</span> and differentially predicted ages of tissues such as breast noted by Horvath <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span>.
DNAm was highly correlated with age for a variety of tissue-specific sites in many studies, suggesting that the definition of clock sites is frequently tissue-dependent.
This may also be true of drift changes, stochastic changes may occur at sites in one tissue that would be tightly controlled in another.
One could restrict the definition of clock changes further to only those which are present in all/most tissue types to make a multi-tissue predictor.</p>
<p>Apparent changes in the DNAm state of a mixed cell population could be due not to global changes in that population but rather its composition, blood exhibits changes in the cellular composition with age <span class="citation">(Rimmelé et al. <a href="#ref-Rimmele2014" role="doc-biblioref">2014</a>; Izzo et al. <a href="#ref-Izzo2020" role="doc-biblioref">2020</a>)</span>.
As haematopoietic stem cells age their numbers increase and their regenerative potential declines.
Their output becomes skewed towards the myeloid lineage but produces poorer quality cells.
The number of naive T-cells decreases with age, memory and effector T-cells accumulate as to memory B cells limiting the diversity in the cell-types and contributing to immunosenescence (reviewed in <span class="citation">(Geiger, Haan, and Florian <a href="#ref-Geiger2013" role="doc-biblioref">2013</a>)</span>).
Cellular composition of tissues can have a substantial impact on epigenome-wide association studies (EWAS), especially when dealing with small effect sizes <span class="citation">(Jaffe and Irizarry <a href="#ref-Jaffe2014" role="doc-biblioref">2014</a>)</span>.
Thus shifts in cell composition such as those in blood during ageing can be substantial sources of confounding when searching for cell-intrinsic changes in DNA.
Some attempts to assay and/or correct for cell-type composition have been made, for example, Rakyan et al <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>)</span> isolated <span class="math inline">\(CD4^+\)</span> T-cells and found that they had a 60% overlap with age-related CpGs in whole blood. Chen et al. <span class="citation">(Chen et al. <a href="#ref-Chen2016a" role="doc-biblioref">2016</a>)</span> used the method developed by Houseman et al. <span class="citation">(Houseman et al. <a href="#ref-Houseman2012" role="doc-biblioref">2012</a>)</span> to correct for cell-type composition and found that it did improve the performance of the Horvath and Hannum age predictors.
The changes were significant but the effect of this correction was relatively modest.
Whilst cell composition has an effect, DNAm Age definitely seems to be capturing something beyond shifts in cell-type composition.
However, the age acceleration computed with Hannum’s method, which is trained on blood data only, was more strongly correlated with cell-type composition than Horvath’s method, indicating that extrinsic changes in cell-type composition likely have some impact the model.</p>
<p>Jenkinson et al. <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> took an information-theoretic approach to DNA methylation maintenance, modelling the binary methylation state of CpGs as a noisy communications channel.
Permitting them to compute the information capacity (CAP), methylation entropy (ENT), and the relative dissipated energy (RDE) of these channels.
High capacity channels represent reliable transmission of methylation state, consistent with high RDE as more energy is needed to ensure reliability and low entropy as methylation is more ordered at sites under tighter control than at less highly controlled sites.
They found that the Transcription Start Sites (TSS) exhibited high levels of RDE and high information capacity as well as low levels of entropy.
These values trended in the opposite direction as they moved outwards to CGIs, shores and seas.
This information-theoretic framing provides a useful lens through which to view drift and clock loci.</p>
<p>It could be hypothesised that as a cell differentiates those sites the methylation status of which are most important for successful differentiation will be the focus of the expenditure of cellular resources to set and maintain their methylation state, and will thus exhibit high capacity and low entropy.
Whatever mechanism is focusing the resources of the cell on the maintenance of the methylation at these sites may be responsible for the tissue-specific biases in DNAm changes over time i.e. their targets will be clock sites.
Those sites not involved in the maintenance of the specific differentiated state would be a lower priority for the expenditure of cellular resources and thus display lower information capacity and higher entropy i.e. be drift sites.
This would also lead one to predict that multi-tissue age predicting loci would be enriched for factors common to the maintenance of a differentiated state.
The polycomb and Trithorax group proteins are very broadly evolutionarily conserved and play a key role in developmental cell fate choice and particularly in long-term stable maintenance of epigenetic state <span class="citation">(Schuettengruber et al. <a href="#ref-Schuettengruber2017" role="doc-biblioref">2007</a>)</span>.
Consistent with this hypothesis Teschendorff et al. <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> observed that PCGTs are enriched among CpGs associated with age and Levine et al. <span class="citation">(Levine et al. <a href="#ref-Levine2018" role="doc-biblioref">2018</a>)</span> also noted that their 513 clock CpGs were enriched for PCGTs.
This is also complementary to the idea that DNAm dysregulation with age promotes carcinogenesis by stabilising stem cell character but with reduced capacity for differentiation as proposed by Teschendorff et al. <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> and Rakyan et al. <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>)</span>.</p>
<p>In his paper <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span> Horvath proposed that DNAm age is a measure of the work done by an “Epigenetic Maintenance System” (EMS) which maintains epigenetic stability over time.
Considering work in its physical sense of the integral of power over time, Horvath considered power, the rate of change in the use of energy by the EMS as the ‘tick-rate’ of the epigenetic clock.
He proposed that during development power would be high due to the high level of stress on the system and that power would drop to a consistent level once maturity was reached.
Horvath laid out four predictions of this EMS model and argued that there was evidence in support of each of them. The predictions were:</p>
<ol style="list-style-type: decimal">
<li>Cancer tissue should show signs of accelerated age, reflecting the protective actions of the EMS</li>
<li>Many mitogens, genomic aberrations, and oncogenes, which trigger the response of the EMS, should be associated with accelerated DNAm age.</li>
<li>High age acceleration of cancer tissue should be associated with fewer somatic mutations given the protective role of the EMS</li>
<li>Mutations in TP53 should be associated with a lower age acceleration of cancer tissue if one further assumes that p53 signalling helps trigger the EMS</li>
</ol>
<p>I would propose that a fifth prediction of this model is that DNA methylation fidelity should be elevated during development and differentiation as a reflection of the greater amount of work done by the EMS during this period.
This is borne out by the results of Zhao et al. <span class="citation">(Zhao et al. <a href="#ref-Zhao2014" role="doc-biblioref">2014</a>)</span> who noted a global increase in DNA methylation fidelity during the early differentiation of mouse embryonic stems cells (see figure <a href="introduction.html#fig:Zhao2014fig3ab">1.15</a>).</p>
<p>If Horvath’s Concept of an EMS is accurate and Jenkinson et al.’s RDE captures the energy expended by the cell on the maintenance of the methylation state of a locus, one would expect Horvath’s concept of an EMS to tie in quite nicely with Jenkinson et al.’s methylation channel model.
With the Relative dissipated energy (RDE) for a CpG channel corresponding to the work done by Horvath’s EMS at a given locus.
Jenkinson et al. did indeed observe high levels of RDE in stem cells and a decline of RDE with age.
Interestingly, however, some cancers exhibited decreased RDE, CAP and increased entropy which is consistent with an accelerated ageing profile but seemingly less so with elevated activity of an EMS.
It is possible that whilst the distribution of RDE at various loci in the genome shifts giving a lower mean RDE that the total amount of energy expended is increasing but becoming less well targeted such that whilst there is increased energy flux through the EMS it is not being properly directed.
There may also be a plateau effect wherein the energy flux through the EMS is maxed out and cannot keep up with demand from many loci.
One would also predict more rapid equilibration of DNAm levels at loci with high RDEs, which could be examined by looking at the second derivatives for the stochastic model of DNA methylation described by Pfeifer et al. <span class="citation">(Pfeifer et al. <a href="#ref-Pfeifer1990a" role="doc-biblioref">1990</a>)</span> using the experimental systems Laird et al. <span class="citation">(Laird et al. <a href="#ref-Laird2004" role="doc-biblioref">2004</a>)</span> and Riggs et al. <span class="citation">(Riggs and Xiong <a href="#ref-Riggs2004a" role="doc-biblioref">2004</a>)</span> used which supported the original model.</p>
<p>Jenkinson et al. <span class="citation">(Jenkinson et al. <a href="#ref-Jenkinson2017" role="doc-biblioref">2017</a>)</span> found a global increase in entropy with age, but not with cell passage in culture suggesting an increase in entropy is associated with epigenetic age independent of mitotic age.
This observation is in agreement with the finding that epigenetic state becomes more stochastic and diverges with age <span class="citation">(Fraga et al. <a href="#ref-Fraga2005" role="doc-biblioref">2005</a>; Slieker et al. <a href="#ref-Slieker2016" role="doc-biblioref">2016</a>)</span>, indeed genes whose methylation were most divergent with age were enriched for ageing associations.
They also observed a loss of entropic sensitivity with age.
Entropic sensitivity is an indicator of how plastic DNAm state is to extrinsic effects.
Jenkinson et al. noted that there is a general loss of phenotypic plasticity with age, but cited no specific instance of loss of DNAm plasticity to environmental effects with age.
Hahn et al. <span class="citation">(Hahn et al. <a href="#ref-Hahn2017" role="doc-biblioref">2017</a>)</span> provided a possible example of this effect when they reported that dietary restriction caused fewer differences in the methylation state of older than younger mice.</p>
<p>In summary, DNAm age is a high-quality biomarker of ageing which can be tailored to the particular requirements of the context, be it estimating the chronological age of individuals, measuring age acceleration/deceleration of individuals relative to their chronological age or estimating age disparity of tissues <span class="citation">(Horvath and Raj <a href="#ref-Horvath2018" role="doc-biblioref">2018</a>; Bell et al. <a href="#ref-Bell2019" role="doc-biblioref">2019</a>)</span>.
There is predictably a cost/accuracy trade-off.
DNAm age is to some degree capturing “biological age” and likely not being driven by changes due to specific ageing-related diseases as illustrated by lack of relationships with specific pathologies and phenotypes of ageing noted by Bell et al. <span class="citation">(J. T. Bell et al. <a href="#ref-Bell2012" role="doc-biblioref">2012</a>)</span>.
The PhenoAge clock is a better predictor of morbidity and mortality than age acceleration but may not be a better predictor of underlying biological age if the risk it is capturing is driven more by disease than that captured by age acceleration.
PhenoAge is strongly associated with comorbidity count, more so than either the Hannun or Horvath clocks, interestingly zero comorbidities are associated with a negative PhenoAge score suggesting some of the PhenoAge signal is derived from disease status.
Disease may, of course, interact with biological age causing an acceleration thereof, making them difficult to disentangle.
Neither Age acceleration nor PhenoAge perfectly captures biological age but both are highly informative on the subject</p>
<p>Global DNA methylation decreases with age, DNA methylation becomes more disordered and less plastic to environmental influences with age.
A set of loci, not necessarily individual CpGs but rather some functional unit of methylation exhibit consistent predictable changes with age.
Which loci are in this set varies with tissue type; there is, however, a subset which seems to be common to most tissues.
The preponderance of these loci are in areas of high to moderate CpG density and become hypermethylated with age.
The high frequency of CpGs and the hypomethylated “ground state” of CGIs would lead one to expect a bias towards being located in CGIs and becoming hypermethylated.
Thus it is unclear if changes in DNA methylation over time are biased towards regions of high CpG density and hypermethylation at a level that is greater or less than one would predict with the prior information.</p>
</div>
</div>
<div id="aims" class="section level2">
<h2><span class="header-section-number">1.6</span> Aims</h2>
<p>Going forward this work covers several aspects of change in DNA methylation related to ageing phenotypes.
Starting at the beginning of the human life-course with EWAS for <em>in utero</em> markers and interventions to impact on long-term bone health through the lens of the developmental origins of health and disease hypothesis.
Continuing with an examination of the age-related changes in the methylation state of a key functional region of the genome, whose methylation state has not previously been characterised in detail, namely the tRNA genes.
Lastly, the generation of new DNAm clocks based on a family of repetitive elements, specifically the Alu repeat family. Making use of another region of the genome the DNAm state of which has not previously been characterised in detail, but which represents a substantial proportion of all CpG sites and the derepression of which has long been considered to play a role in ageing.</p>
<ol style="list-style-type: decimal">
<li><p>Identify Age-related changes in DNA methylation in regions of the genome characterised in the TwinsUK MeDIP-seq dataset and poorly covered by or covered in small samples by other technologies.</p>
<ol style="list-style-type: lower-alpha">
<li>The tRNA genes, following up on the previous finding of age-related DNA hypermethylation at tRNA-iMet-CAT-1-4 <span class="citation">(Bell et al. <a href="#ref-Bell2016" role="doc-biblioref">2016</a>)</span>. tRNA genes have a core role in cellular metabolism, many emerging regulatory functions both structural and as signalling molecules. tRNA genes also interface with many systems the modulation of which impact longevity, making any age-related changes in their epigenetic state potentially very consequential.</li>
<li>Alu repeat elements, a primate-specific family of SINEs present in over 1 million copies in the human genome. We aim to construct a DNA methylation clock using only these elements. We reason that age acceleration based on this clock will yield different information about biological ageing than from previous clocks as the effects of DNA methylation changes differ between repressed repetitive elements and regulatory sequences.</li>
</ol></li>
<li><p>Identify epigenetic associations with bone health outcomes and of vitamin D supplementation during pregnancy as an intervention to improve bone outcomes <span class="citation">(Harvey et al. <a href="#ref-Harvey2012a" role="doc-biblioref">2012</a>; Cooper et al. <a href="#ref-Cooper2016" role="doc-biblioref">2016</a>)</span>. Provide some groundwork for mechanistic studies to extend understanding of what systems influence bone health by identifying what genomic features change epigenetically with bone traits and interventions intended to affect these traits.</p></li>
</ol>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Adamowicz2018">
<p>Adamowicz, Martyna, Klio Maratou, and Timothy J Aitman. 2018. “Multiplexed DNA Methylation Analysis of Target Regions Using Microfluidics (Fluidigm).” In <em>Methods in Molecular Biology (Clifton, N.j.)</em>, 1708:349–63. <a href="https://doi.org/10.1007/978-1-4939-7481-8_18">https://doi.org/10.1007/978-1-4939-7481-8_18</a>.</p>
</div>
<div id="ref-Affinito2016">
<p>Affinito, Ornella, Giovanni Scala, Domenico Palumbo, Ermanno Florio, Antonella Monticelli, Gennaro Miele, Vittorio Enrico Avvedimento, Alessandro Usiello, Lorenzo Chiariotti, and Sergio Cocozza. 2016. “Modeling DNA methylation by analyzing the individual configurations of single molecules.” <em>Epigenetics</em> 11 (12): 881–88. <a href="https://doi.org/10.1080/15592294.2016.1246108">https://doi.org/10.1080/15592294.2016.1246108</a>.</p>
</div>
<div id="ref-Bacalini2015">
<p>Bacalini, Maria Giulia, Alessio Boattini, Davide Gentilini, Enrico Giampieri, Chiara Pirazzini, Cristina Giuliani, Elisa Fontanesi, et al. 2015. “A meta-analysis on age-associated changes in blood DNA methylation: results from an original analysis pipeline for Infinium 450k data.” <em>Aging</em> 7 (2): 97–109. <a href="https://doi.org/10.18632/aging.100718">https://doi.org/10.18632/aging.100718</a>.</p>
</div>
<div id="ref-Bannister2011">
<p>Bannister, Andrew J, and Tony Kouzarides. 2011. “Regulation of chromatin by histone modifications.” <em>Cell Research</em> 21 (3): 381–95. <a href="https://doi.org/10.1038/cr.2011.22">https://doi.org/10.1038/cr.2011.22</a>.</p>
</div>
<div id="ref-Barnett2012">
<p>Barnett, Karen, Stewart W. Mercer, Michael Norbury, Graham Watt, Sally Wyke, and Bruce Guthrie. 2012. “Epidemiology of multimorbidity and implications for health care, research, and medical education: a cross-sectional study.” <em>The Lancet</em> 380 (9836): 37–43. <a href="https://doi.org/10.1016/S0140-6736(12)60240-2">https://doi.org/10.1016/S0140-6736(12)60240-2</a>.</p>
</div>
<div id="ref-Bauden2017">
<p>Bauden, Monika, Theresa Kristl, Roland Andersson, György Marko-Varga, and Daniel Ansari. 2017. “Characterization of histone-related chemical modifications in formalin-fixed paraffin-embedded and fresh-frozen human pancreatic cancer xenografts using LC-MS/MS.” <em>Laboratory Investigation</em> 97 (3): 279–88. <a href="https://doi.org/10.1038/labinvest.2016.134">https://doi.org/10.1038/labinvest.2016.134</a>.</p>
</div>
<div id="ref-Belikov2019">
<p>Belikov, Aleksey V. 2019. “Age-related diseases as vicious cycles.” <em>Ageing Research Reviews</em> 49 (November 2018): 11–26. <a href="https://doi.org/10.1016/j.arr.2018.11.002">https://doi.org/10.1016/j.arr.2018.11.002</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>Bell, Christopher G., Fei Gao, Wei Yuan, Leonie Roos, Richard J. Acton, Yudong Xia, Jordana Bell, et al. 2018. “Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci.” <em>Nature Communications</em> 9 (1): 8. <a href="https://doi.org/10.1038/s41467-017-01586-1">https://doi.org/10.1038/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Bell2019">
<p>Bell, Christopher G, Robert Lowe, Peter D Adams, Andrea A Baccarelli, Stephan Beck, Jordana T Bell, Brock C. Christensen, et al. 2019. “DNA methylation aging clocks: challenges and recommendations.” <em>Genome Biology</em> 20 (1): 249. <a href="https://doi.org/10.1186/s13059-019-1824-y">https://doi.org/10.1186/s13059-019-1824-y</a>.</p>
</div>
<div id="ref-Bell2016">
<p>Bell, Christopher G., Yudong Xia, Wei Yuan, Fei Gao, Kirsten Ward, Leonie Roos, Massimo Mangino, et al. 2016. “Novel regional age-associated DNA methylation changes within human common disease-associated loci.” <em>Genome Biology</em> 17 (1): 193. <a href="https://doi.org/10.1186/s13059-016-1051-8">https://doi.org/10.1186/s13059-016-1051-8</a>.</p>
</div>
<div id="ref-Bell2012">
<p>Bell, Jordana T., Pei-Chien Tsai, Tsun-Po Yang, Ruth Pidsley, James Nisbet, Daniel Glass, Massimo Mangino, et al. 2012. “Epigenome-wide scans identify differentially methylated regions for age and age-related phenotypes in a healthy ageing population.” Edited by Jun Li. <em>PLoS Genetics</em> 8 (4): e1002629. <a href="https://doi.org/10.1371/journal.pgen.1002629">https://doi.org/10.1371/journal.pgen.1002629</a>.</p>
</div>
<div id="ref-Bell2017">
<p>Bell, Joshua S. K., and Paula M. Vertino. 2017. “Orphan CpG islands define a novel class of highly active enhancers.” <em>Epigenetics</em> 12 (6): 449–64. <a href="https://doi.org/10.1080/15592294.2017.1297910">https://doi.org/10.1080/15592294.2017.1297910</a>.</p>
</div>
<div id="ref-Benton2017">
<p>Benton, Miles C, Heidi G Sutherland, Donia Macartney-Coxson, Larisa M Haupt, Rodney A. Lea, and Lyn R Griffiths. 2017. “Methylome-wide association study of whole blood DNA in the Norfolk Island isolate identifies robust loci associated with age.” <em>Aging</em> 9 (3): 753–68. <a href="https://doi.org/10.18632/aging.101187">https://doi.org/10.18632/aging.101187</a>.</p>
</div>
<div id="ref-Berdyshev1967">
<p>Berdyshev, G D, G K Korotaev, G V Boiarskikh, and B F Vaniushin. 1967. “Nucleotide composition of DNA and RNA from somatic tissues of humpback and its changes during spawning.” <em>Biokhimiia (Moscow, Russia)</em> 32 (5): 988–93. <a href="http://www.ncbi.nlm.nih.gov/pubmed/5628601">http://www.ncbi.nlm.nih.gov/pubmed/5628601</a>.</p>
</div>
<div id="ref-Bianconi2013">
<p>Bianconi, Eva, Allison Piovesan, Federica Facchin, Alina Beraudi, Raffaella Casadei, Flavia Frabetti, Lorenza Vitale, et al. 2013. “An estimation of the number of cells in the human body.” <em>Annals of Human Biology</em> 40 (6): 463–71. <a href="https://doi.org/10.3109/03014460.2013.807878">https://doi.org/10.3109/03014460.2013.807878</a>.</p>
</div>
<div id="ref-Bibikova2006">
<p>Bibikova, Marina. 2006. “High-throughput DNA methylation profiling using universal bead arrays.” <em>Genome Research</em> 16 (3): 383–93. <a href="https://doi.org/10.1101/gr.4410706">https://doi.org/10.1101/gr.4410706</a>.</p>
</div>
<div id="ref-Bibikova2011">
<p>Bibikova, Marina, Bret Barnes, Chan Tsan, Vincent Ho, Brandy Klotzle, Jennie M. Le, David Delano, et al. 2011. “High density DNA methylation array with single CpG site resolution.” <em>Genomics</em> 98 (4): 288–95. <a href="https://doi.org/10.1016/j.ygeno.2011.07.007">https://doi.org/10.1016/j.ygeno.2011.07.007</a>.</p>
</div>
<div id="ref-Bibikova2009">
<p>Bibikova, Marina, Jennie Le, Bret Barnes, Shadi Saedinia-Melnyk, Lixin Zhou, Richard Shen, and Kevin L Gunderson. 2009. “Genome-wide DNA methylation profiling using Infinium assay.” <em>Epigenomics</em> 1 (1): 177–200. <a href="https://doi.org/10.2217/epi.09.14">https://doi.org/10.2217/epi.09.14</a>.</p>
</div>
<div id="ref-Bird2002">
<p>Bird, Adrian. 2002. “DNA methylation patterns and epigenetic memory.” <em>Genes &amp; Development</em> 16 (1): 6–21. <a href="https://doi.org/10.1101/gad.947102">https://doi.org/10.1101/gad.947102</a>.</p>
</div>
<div id="ref-Bird2007">
<p>Bird, Adrian. 2007. “Perceptions of epigenetics.” <em>Nature</em> 447 (7143): 396–8. <a href="https://doi.org/10.1038/nature05913">https://doi.org/10.1038/nature05913</a>.</p>
</div>
<div id="ref-Bjornsson2008">
<p>Bjornsson, Hans T. 2008. “Intra-individual Change Over Time in DNA Methylation With Familial Clustering.” <em>JAMA</em> 299 (24): 2877. <a href="https://doi.org/10.1001/jama.299.24.2877">https://doi.org/10.1001/jama.299.24.2877</a>.</p>
</div>
<div id="ref-Bock2010">
<p>Bock, Christoph, Eleni M Tomazou, Arie B Brinkman, Fabian Müller, Femke Simmer, Hongcang Gu, Natalie Jäger, Andreas Gnirke, Hendrik G Stunnenberg, and Alexander Meissner. 2010. “Quantitative comparison of genome-wide DNA methylation mapping technologies.” <em>Nature Biotechnology</em> 28 (10): 1106–14. <a href="https://doi.org/10.1038/nbt.1681">https://doi.org/10.1038/nbt.1681</a>.</p>
</div>
<div id="ref-Bocklandt2011">
<p>Bocklandt, Sven, Wen Lin, Mary E. Sehl, Francisco J. Sánchez, Janet S. Sinsheimer, Steve Horvath, and Eric Vilain. 2011. “Epigenetic Predictor of Age.” Edited by Nicoletta Landsberger. <em>PLoS ONE</em> 6 (6): e14821. <a href="https://doi.org/10.1371/journal.pone.0014821">https://doi.org/10.1371/journal.pone.0014821</a>.</p>
</div>
<div id="ref-Boks2009">
<p>Boks, Marco P., Eske M. Derks, Daniel J. Weisenberger, Erik Strengman, Esther Janson, Iris E. Sommer, René S. Kahn, and Roel A. Ophoff. 2009. “The relationship of DNA methylation with age, gender and genotype in twins and healthy controls.” Edited by Joseph Najbauer. <em>PloS One</em> 4 (8): e6767. <a href="https://doi.org/10.1371/journal.pone.0006767">https://doi.org/10.1371/journal.pone.0006767</a>.</p>
</div>
<div id="ref-Booth2016">
<p>Booth, Lauren N., and Anne Brunet. 2016. “The Aging Epigenome.” <em>Molecular Cell</em> 62 (5): 728–44. <a href="https://doi.org/10.1016/j.molcel.2016.05.013">https://doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div id="ref-Bostick2007">
<p>Bostick, M., J. K. Kim, P.-O. Esteve, A. Clark, S. Pradhan, and S. E. Jacobsen. 2007. “UHRF1 Plays a Role in Maintaining DNA Methylation in Mammalian Cells.” <em>Science</em> 317 (5845): 1760–4. <a href="https://doi.org/10.1126/science.1147939">https://doi.org/10.1126/science.1147939</a>.</p>
</div>
<div id="ref-Briggs2010">
<p>Briggs, Adrian W., Udo Stenzel, Matthias Meyer, Johannes Krause, Martin Kircher, and Svante Pääbo. 2010. “Removal of deaminated cytosines and detection of in vivo methylation in ancient DNA.” <em>Nucleic Acids Research</em> 38 (6): e87. <a href="https://doi.org/10.1093/nar/gkp1163">https://doi.org/10.1093/nar/gkp1163</a>.</p>
</div>
<div id="ref-Carrillo-de-Santa-Pau2017">
<p>Carrillo-de-Santa-Pau, Enrique, David Juan, Vera Pancaldi, Felipe Were, Ignacio Martin-Subero, Daniel Rico, and Alfonso Valencia. 2017. “Automatic identification of informative regions with epigenomic changes associated to hematopoiesis.” <em>Nucleic Acids Research</em> 45 (16): 9244–59. <a href="https://doi.org/10.1093/nar/gkx618">https://doi.org/10.1093/nar/gkx618</a>.</p>
</div>
<div id="ref-CellOntologyOBO">
<p>CellOntology. 2017. “Cell Ontology.” <a href="https://doi.org/10.5281/zenodo.168254">https://doi.org/10.5281/zenodo.168254</a>.</p>
</div>
<div id="ref-CellSystemsCellType2017">
<p>CellSystemsVoices. 2017. “What Is Your Conceptual Definition of ‘Cell Type’ in the Context of a Mature Organism?” <em>Cell Systems</em> 4 (3): 255–59. <a href="https://doi.org/10.1016/j.cels.2017.03.006">https://doi.org/10.1016/j.cels.2017.03.006</a>.</p>
</div>
<div id="ref-Cerase2015">
<p>Cerase, Andrea, Greta Pintacuda, Anna Tattermusch, and Philip Avner. 2015. “Xist localization and function: new insights from multiple levels.” <em>Genome Biology</em> 16 (1): 166. <a href="https://doi.org/10.1186/s13059-015-0733-y">https://doi.org/10.1186/s13059-015-0733-y</a>.</p>
</div>
<div id="ref-Chen2016a">
<p>Chen, Brian H, Riccardo E Marioni, Elena Colicino, Marjolein J Peters, Cavin K Ward-Caviness, Pei-Chien Tsai, Nicholas S Roetker, et al. 2016. “DNA methylation-based measures of biological age: meta-analysis predicting time to death.” <em>Aging</em> 8 (9): 1844–65. <a href="https://doi.org/10.18632/aging.101020">https://doi.org/10.18632/aging.101020</a>.</p>
</div>
<div id="ref-Christensen2009">
<p>Christensen, Brock C., E. Andres Houseman, Carmen J. Marsit, Shichun Zheng, Margaret R. Wrensch, Joseph L. Wiemels, Heather H. Nelson, et al. 2009. “Aging and Environmental Exposures Alter Tissue-Specific DNA Methylation Dependent upon CpG Island Context.” Edited by Dirk Schübeler. <em>PLoS Genetics</em> 5 (8): e1000602. <a href="https://doi.org/10.1371/journal.pgen.1000602">https://doi.org/10.1371/journal.pgen.1000602</a>.</p>
</div>
<div id="ref-Christiansen2016">
<p>Christiansen, Lene, Adam Lenart, Qihua Tan, James W. Vaupel, Abraham Aviv, Matt McGue, and Kaare Christensen. 2016. “DNA methylation age is associated with mortality in a longitudinal Danish twin study.” <em>Aging Cell</em> 15 (1): 149–54. <a href="https://doi.org/10.1111/acel.12421">https://doi.org/10.1111/acel.12421</a>.</p>
</div>
<div id="ref-Ciernia2016">
<p>Ciernia, Annie Vogel, and Janine LaSalle. 2016. “The landscape of DNA methylation amid a perfect storm of autism aetiologies.” <em>Nature Reviews Neuroscience</em> 17 (7): 411–23. <a href="https://doi.org/10.1038/nrn.2016.41">https://doi.org/10.1038/nrn.2016.41</a>.</p>
</div>
<div id="ref-Cole2017">
<p>Cole, John Joseph, Neil A. Robertson, Mohammed Iqbal Rather, John P. Thomson, Tony McBryan, Duncan Sproul, Tina Wang, et al. 2017. “Diverse interventions that extend mouse lifespan suppress shared age-associated epigenetic changes at critical gene regulatory regions.” <em>Genome Biology</em> 18 (1): 58. <a href="https://doi.org/10.1186/s13059-017-1185-3">https://doi.org/10.1186/s13059-017-1185-3</a>.</p>
</div>
<div id="ref-Cooper2016">
<p>Cooper, Cyrus, Nicholas C. Harvey, Nicholas J. Bishop, Stephen Kennedy, Aris T. Papageorghiou, Inez Schoenmakers, Robert Fraser, et al. 2016. “Maternal gestational vitamin D supplementation and offspring bone health (MAVIDOS): a multicentre, double-blind, randomised placebo-controlled trial.” <em>The Lancet Diabetes &amp; Endocrinology</em> 4 (5): 393–402. <a href="https://doi.org/10.1016/S2213-8587(16)00044-9">https://doi.org/10.1016/S2213-8587(16)00044-9</a>.</p>
</div>
<div id="ref-Down2009">
<p>Down, Thomas A, Vardhman K Rakyan, Daniel J Turner, Paul Flicek, Heng Li, Eugene Kulesha, Stefan Gräf, et al. 2008. “A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis.” <em>Nature Biotechnology</em> 26 (7): 779–85. <a href="https://doi.org/10.1038/nbt1414">https://doi.org/10.1038/nbt1414</a>.</p>
</div>
<div id="ref-Duncan1980">
<p>Duncan, B K, and J H Miller. 1980. “Mutagenic deamination of cytosine residues in DNA.” <em>Nature</em> 287 (5782): 560–1. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6999365">http://www.ncbi.nlm.nih.gov/pubmed/6999365</a>.</p>
</div>
<div id="ref-Eckhardt2006">
<p>Eckhardt, Florian, Joern Lewin, Rene Cortese, Vardhman K Rakyan, John Attwood, Matthias Burger, John Burton, et al. 2006. “DNA methylation profiling of human chromosomes 6, 20 and 22.” <em>Nature Genetics</em> 38 (12): 1378–85. <a href="https://doi.org/10.1038/ng1909">https://doi.org/10.1038/ng1909</a>.</p>
</div>
<div id="ref-Ehrlich1982">
<p>Ehrlich, Melanie, Miguel A. Gama-Sosa, Lan-Hsiang Huang, Rose Marie Midgett, Kenneth C. Kuo, Roy A. McCune, and Charles Gehrke. 1982. “Amount and distribution of 5-methylcytosine in human DNA from different types of tissues or cells.” <em>Nucleic Acids Research</em> 10 (8): 2709–21. <a href="https://doi.org/10.1093/nar/10.8.2709">https://doi.org/10.1093/nar/10.8.2709</a>.</p>
</div>
<div id="ref-Esteller2000">
<p>Esteller, M, J M Silva, G Dominguez, F Bonilla, X Matias-Guiu, E Lerma, E Bussaglia, et al. 2000. “Promoter hypermethylation and BRCA1 inactivation in sporadic breast and ovarian tumors.” <em>Journal of the National Cancer Institute</em> 92 (7): 564–9. <a href="https://doi.org/10.1093/jnci/92.7.564">https://doi.org/10.1093/jnci/92.7.564</a>.</p>
</div>
<div id="ref-Esteve2006">
<p>Estève, Pierre-Olivier, Hang Gyeong Chin, Andrea Smallwood, George R. Feehery, Omkaram Gangisetty, Adam R. Karpf, Michael F. Carey, and Sriharsa Pradhan. 2006. “Direct interaction between DNMT1 and G9a coordinates DNA and histone methylation during replication.” <em>Genes &amp; Development</em> 20 (22): 3089–3103. <a href="https://doi.org/10.1101/gad.1463706">https://doi.org/10.1101/gad.1463706</a>.</p>
</div>
<div id="ref-Feinberg2018">
<p>Feinberg, Andrew P. 2018. “The Key Role of Epigenetics in Human Disease Prevention and Mitigation.” Edited by Dan L. Longo. <em>New England Journal of Medicine</em> 378 (14): 1323–34. <a href="https://doi.org/10.1056/NEJMra1402513">https://doi.org/10.1056/NEJMra1402513</a>.</p>
</div>
<div id="ref-Feinberg2004">
<p>Feinberg, Andrew P, and Benjamin Tycko. 2004. “The history of cancer epigenetics.” <em>Nature Reviews Cancer</em> 4 (2): 143–53. <a href="https://doi.org/10.1038/nrc1279">https://doi.org/10.1038/nrc1279</a>.</p>
</div>
<div id="ref-Fleming2005">
<p>Fleming, Douglas M., and Alex J. Elliot. 2005. “The impact of influenza on the health and health care utilisation of elderly people.” <em>Vaccine</em> 23 (SUPPL. 1): S1. <a href="https://doi.org/10.1016/j.vaccine.2005.04.018">https://doi.org/10.1016/j.vaccine.2005.04.018</a>.</p>
</div>
<div id="ref-Florath2014">
<p>Florath, Ines, Katja Butterbach, H. Muller, M. Bewerunge-Hudler, and Hermann Brenner. 2014. “Cross-sectional and longitudinal changes in DNA methylation with age: an epigenome-wide analysis revealing over 60 novel age-associated CpG sites.” <em>Human Molecular Genetics</em> 23 (5): 1186–1201. <a href="https://doi.org/10.1093/hmg/ddt531">https://doi.org/10.1093/hmg/ddt531</a>.</p>
</div>
<div id="ref-Fraga2005">
<p>Fraga, Mario F, E. Ballestar, Maria F Paz, Santiago Ropero, Fernando Setien, Maria L Ballestar, D. Heine-Suner, et al. 2005. “From The Cover: Epigenetic differences arise during the lifetime of monozygotic twins.” <em>Proceedings of the National Academy of Sciences</em> 102 (30): 10604–9. <a href="https://doi.org/10.1073/pnas.0500398102">https://doi.org/10.1073/pnas.0500398102</a>.</p>
</div>
<div id="ref-Friedman2010">
<p>Friedman, Jerome, Trevor Hastie, and Robert Tibshirani. 2010. “Regularization Paths for Generalized Linear Models via Coordinate Descent.” <em>Journal of Statistical Software</em> 33 (1): 7–10. <a href="https://doi.org/10.18637/jss.v033.i01">https://doi.org/10.18637/jss.v033.i01</a>.</p>
</div>
<div id="ref-Garagnani2012">
<p>Garagnani, Paolo, Maria G. Bacalini, Chiara Pirazzini, Davide Gori, Cristina Giuliani, Daniela Mari, Anna M. Di Blasio, et al. 2012. “Methylation of ELOVL2 gene as a new epigenetic marker of age.” <em>Aging Cell</em> 11 (6): 1132–4. <a href="https://doi.org/10.1111/acel.12005">https://doi.org/10.1111/acel.12005</a>.</p>
</div>
<div id="ref-Geiger2013">
<p>Geiger, Hartmut, Gerald de Haan, and M. Carolina Florian. 2013. “The ageing haematopoietic stem cell compartment.” <em>Nature Reviews Immunology</em> 13 (5): 376–89. <a href="https://doi.org/10.1038/nri3433">https://doi.org/10.1038/nri3433</a>.</p>
</div>
<div id="ref-Goldman2016">
<p>Goldman, Dana. 2016. “The Economic Promise of Delayed Aging.” <em>Cold Spring Harbor Perspectives in Medicine</em> 6 (2): a025072. <a href="https://doi.org/10.1101/cshperspect.a025072">https://doi.org/10.1101/cshperspect.a025072</a>.</p>
</div>
<div id="ref-Guo2014a">
<p>Guo, Junjie U, Yijing Su, Joo Heon Shin, Jaehoon Shin, Hongda Li, Bin Xie, Chun Zhong, et al. 2014. “Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain.” <em>Nature Neuroscience</em> 17 (2): 215–22. <a href="https://doi.org/10.1038/nn.3607">https://doi.org/10.1038/nn.3607</a>.</p>
</div>
<div id="ref-Haerter2014">
<p>Haerter, Jan O., Cecilia Lövkvist, Ian B. Dodd, and Kim Sneppen. 2014. “Collaboration between CpG sites is needed for stable somatic inheritance of DNA methylation states.” <em>Nucleic Acids Research</em> 42 (4): 2235–44. <a href="https://doi.org/10.1093/nar/gkt1235">https://doi.org/10.1093/nar/gkt1235</a>.</p>
</div>
<div id="ref-Hahn2017">
<p>Hahn, Oliver, Sebastian Grönke, Thomas M. Stubbs, Gabriella Ficz, Oliver Hendrich, Felix Krueger, Simon Andrews, et al. 2017. “Dietary restriction protects from age-associated DNA methylation and induces epigenetic reprogramming of lipid metabolism.” <em>Genome Biology</em> 18 (1): 56. <a href="https://doi.org/10.1186/s13059-017-1187-1">https://doi.org/10.1186/s13059-017-1187-1</a>.</p>
</div>
<div id="ref-Hannum2013">
<p>Hannum, Gregory, Justin Guinney, Ling Zhao, Li Zhang, Guy Hughes, SriniVas Sadda, Brandy Klotzle, et al. 2013. “Genome-wide methylation profiles reveal quantitative views of human aging rates.” <em>Molecular Cell</em> 49 (2): 359–67. <a href="https://doi.org/10.1016/j.molcel.2012.10.016">https://doi.org/10.1016/j.molcel.2012.10.016</a>.</p>
</div>
<div id="ref-Harrison2011">
<p>Harrison, Alan, and Anne Parle-McDermott. 2011. “DNA Methylation: A Timeline of Methods and Applications.” <em>Frontiers in Genetics</em> 2 (3): 632–49. <a href="https://doi.org/10.3389/fgene.2011.00074">https://doi.org/10.3389/fgene.2011.00074</a>.</p>
</div>
<div id="ref-Harvey2012a">
<p>Harvey, Nicholas C, Kassim Javaid, Nicholas Bishop, Stephen Kennedy, Aris T Papageorghiou, Robert Fraser, Saurabh V Gandhi, Inez Schoenmakers, Ann Prentice, and Cyrus Cooper. 2012. “MAVIDOS Maternal Vitamin D Osteoporosis Study: study protocol for a randomized controlled trial. The MAVIDOS Study Group.” <em>Trials</em> 13 (1): 13. <a href="https://doi.org/10.1186/1745-6215-13-13">https://doi.org/10.1186/1745-6215-13-13</a>.</p>
</div>
<div id="ref-Hashimshony2003">
<p>Hashimshony, Tamar, Jianmin Zhang, Ilana Keshet, Michael Bustin, and Howard Cedar. 2003. “The role of DNA methylation in setting up chromatin structure during development.” <em>Nature Genetics</em> 34 (2): 187–92. <a href="https://doi.org/10.1038/ng1158">https://doi.org/10.1038/ng1158</a>.</p>
</div>
<div id="ref-Hatada1991">
<p>Hatada, I, Y Hayashizaki, S Hirotsune, H Komatsubara, and T Mukai. 1991. “A genomic scanning method for higher organisms using restriction sites as landmarks.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 88 (21): 9523–7. <a href="https://doi.org/10.1073/pnas.88.21.9523">https://doi.org/10.1073/pnas.88.21.9523</a>.</p>
</div>
<div id="ref-Hayano2019">
<p>Hayano, Motoshi, Jae-Hyun Yang, Michael S. Bonkowski, Joao A. Amorim, Jaime M. Ross, Giuseppe Coppotelli, Patrick Griffin, et al. 2019. “DNA Break-Induced Epigenetic Drift as a Cause of Mammalian Aging.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/808659">https://doi.org/10.1101/808659</a>.</p>
</div>
<div id="ref-Hayatsu1970">
<p>Hayatsu, Hikoya, Yusuke Wataya, Kazushige Kai, and Shigeru Iida. 1970. “Reaction of sodium bisulfite with uracil, cytosine, and their derivatives.” <em>Biochemistry</em> 9 (14): 2858–65. <a href="https://doi.org/10.1021/bi00816a016">https://doi.org/10.1021/bi00816a016</a>.</p>
</div>
<div id="ref-Heyn2012">
<p>Heyn, Holger, Ning Li, Humberto J Ferreira, Sebastian Moran, David G Pisano, Antonio Gomez, Javier Diez, et al. 2012. “Distinct DNA methylomes of newborns and centenarians.” <em>Proceedings of the National Academy of Sciences</em> 109 (26): 10522–7. <a href="https://doi.org/10.1073/pnas.1120658109">https://doi.org/10.1073/pnas.1120658109</a>.</p>
</div>
<div id="ref-Hoffman2012">
<p>Hoffman, Michael M, Orion J Buske, Jie Wang, Zhiping Weng, Jeff a Bilmes, and William Stafford Noble. 2012. “Unsupervised pattern discovery in human chromatin structure through genomic segmentation.” <em>Nature Methods</em> 9 (5): 473–6. <a href="https://doi.org/10.1038/nmeth.1937">https://doi.org/10.1038/nmeth.1937</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>Horvath, Steve. 2013. “DNA methylation age of human tissues and cell types.” <em>Genome Biology</em> 14 (10): R115. <a href="https://doi.org/10.1186/gb-2013-14-10-r115">https://doi.org/10.1186/gb-2013-14-10-r115</a>.</p>
</div>
<div id="ref-Horvath2018">
<p>Horvath, Steve, and Kenneth Raj. 2018. “DNA methylation-based biomarkers and the epigenetic clock theory of ageing.” <em>Nature Reviews Genetics</em> 19 (6): 371–84. <a href="https://doi.org/10.1038/s41576-018-0004-3">https://doi.org/10.1038/s41576-018-0004-3</a>.</p>
</div>
<div id="ref-Houseman2012">
<p>Houseman, Eugene Andres, William P Accomando, Devin C Koestler, Brock C Christensen, Carmen J Marsit, Heather H Nelson, John K Wiencke, and Karl T Kelsey. 2012. “DNA methylation arrays as surrogate measures of cell mixture distribution.” <em>BMC Bioinformatics</em> 13 (1): 86. <a href="https://doi.org/10.1186/1471-2105-13-86">https://doi.org/10.1186/1471-2105-13-86</a>.</p>
</div>
<div id="ref-Huff2014">
<p>Huff, Jason T., and Daniel Zilberman. 2014. “Dnmt1-Independent CG Methylation Contributes to Nucleosome Positioning in Diverse Eukaryotes.” <em>Cell</em> 156 (6): 1286–97. <a href="https://doi.org/10.1016/j.cell.2014.01.029">https://doi.org/10.1016/j.cell.2014.01.029</a>.</p>
</div>
<div id="ref-Illingworth2010">
<p>Illingworth, Robert S., Ulrike Gruenewald-Schneider, Shaun Webb, Alastair R W Kerr, Keith D. James, Daniel J. Turner, Colin Smith, David J. Harrison, Robert Andrews, and Adrian P. Bird. 2010. “Orphan CpG islands identify numerous conserved promoters in the mammalian genome.” Edited by Wolf Reik. <em>PLoS Genetics</em> 6 (9): e1001134. <a href="https://doi.org/10.1371/journal.pgen.1001134">https://doi.org/10.1371/journal.pgen.1001134</a>.</p>
</div>
<div id="ref-Irizarry2009">
<p>Irizarry, Rafael A, Christine Ladd-Acosta, Bo Wen, Zhijin Wu, Carolina Montano, Patrick Onyango, Hengmi Cui, et al. 2009. “The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.” <em>Nature Genetics</em> 41 (2): 178–86. <a href="https://doi.org/10.1038/ng.298">https://doi.org/10.1038/ng.298</a>.</p>
</div>
<div id="ref-Izzo2020">
<p>Izzo, Franco, Stanley C. Lee, Asaf Poran, Ronan Chaligne, Federico Gaiti, Baptiste Gross, Rekha R. Murali, et al. 2020. “DNA methylation disruption reshapes the hematopoietic differentiation landscape.” <em>Nature Genetics</em> 52 (April). <a href="https://doi.org/10.1038/s41588-020-0595-4">https://doi.org/10.1038/s41588-020-0595-4</a>.</p>
</div>
<div id="ref-Jaffe2014">
<p>Jaffe, Andrew E, and Rafael A Irizarry. 2014. “Accounting for cellular heterogeneity is critical in epigenome-wide association studies.” <em>Genome Biology</em> 15 (2): R31. <a href="https://doi.org/10.1186/gb-2014-15-2-r31">https://doi.org/10.1186/gb-2014-15-2-r31</a>.</p>
</div>
<div id="ref-Jaiswal2014c">
<p>Jaiswal, Siddhartha, Pierre Fontanillas, Jason Flannick, Alisa Manning, Peter V. Grauman, Brenton G. Mar, R. Coleman Lindsley, et al. 2014. “Age-Related Clonal Hematopoiesis Associated with Adverse Outcomes.” <em>New England Journal of Medicine</em> 371 (26): 2488–98. <a href="https://doi.org/10.1056/NEJMoa1408617">https://doi.org/10.1056/NEJMoa1408617</a>.</p>
</div>
<div id="ref-Jenkinson2017">
<p>Jenkinson, Garrett, Elisabet Pujadas, John Goutsias, and Andrew P Feinberg. 2017. “Potential energy landscapes identify the information-theoretic nature of the epigenome.” <em>Nature Genetics</em> 49 (5): 719–29. <a href="https://doi.org/10.1038/ng.3811">https://doi.org/10.1038/ng.3811</a>.</p>
</div>
<div id="ref-Haluskova2010">
<p>Jenuwein, T. 2001. “Translating the Histone Code.” <em>Science</em> 293 (5532): 1074–80. <a href="https://doi.org/10.1126/science.1063127">https://doi.org/10.1126/science.1063127</a>.</p>
</div>
<div id="ref-Jia2009">
<p>Jia, Da, Renata Z. Jurkowska, Xing Zhang, Albert Jeltsch, and Xiaodong Cheng. 2007. “Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation.” <em>Nature</em> 449 (7159): 248–51. <a href="https://doi.org/10.1038/nature06146">https://doi.org/10.1038/nature06146</a>.</p>
</div>
<div id="ref-Johnson2017">
<p>Johnson, Nicholas D, Howard W Wiener, Alicia K Smith, Shota Nishitani, Devin M Absher, Donna K Arnett, Stella Aslibekyan, and Karen N Conneely. 2017. “Non-linear patterns in age-related DNA methylation may reflect CD4 + T cell differentiation.” <em>Epigenetics</em> 12 (6): 492–503. <a href="https://doi.org/10.1080/15592294.2017.1314419">https://doi.org/10.1080/15592294.2017.1314419</a>.</p>
</div>
<div id="ref-Johnson2006">
<p>Johnson, Thomas E. 2006. “Recent results: Biomarkers of aging.” <em>Experimental Gerontology</em> 41 (12): 1243–6. <a href="https://doi.org/10.1016/j.exger.2006.09.006">https://doi.org/10.1016/j.exger.2006.09.006</a>.</p>
</div>
<div id="ref-Jones2015">
<p>Jones, Meaghan J., Sarah J. Goodman, and Michael S. Kobor. 2015. “DNA methylation and healthy human aging.” <em>Aging Cell</em> 14 (6): 924–32. <a href="https://doi.org/10.1111/acel.12349">https://doi.org/10.1111/acel.12349</a>.</p>
</div>
<div id="ref-Jung2015">
<p>Jung, Marc, and Gerd P Pfeifer. 2015. “Aging and DNA methylation.” <em>BMC Biology</em> 13 (1): 7. <a href="https://doi.org/10.1186/s12915-015-0118-4">https://doi.org/10.1186/s12915-015-0118-4</a>.</p>
</div>
<div id="ref-Kaikkonen2011">
<p>Kaikkonen, Minna U., Michael T Y Lam, and Christopher K. Glass. 2011. “Non-coding RNAs as regulators of gene expression and epigenetics.” <em>Cardiovascular Research</em> 90 (3): 430–40. <a href="https://doi.org/10.1093/cvr/cvr097">https://doi.org/10.1093/cvr/cvr097</a>.</p>
</div>
<div id="ref-Kane2019">
<p>Kane, Alice E., and David A. Sinclair. 2019. “Epigenetic changes during aging and their reprogramming potential.” <em>Critical Reviews in Biochemistry and Molecular Biology</em> 54 (1): 61–83. <a href="https://doi.org/10.1080/10409238.2019.1570075">https://doi.org/10.1080/10409238.2019.1570075</a>.</p>
</div>
<div id="ref-Kelly2012">
<p>Kelly, Theresa K, Yaping Liu, Fides D Lay, Gangning Liang, Benjamin P Berman, and Peter a Jones. 2012. “Genome-wide mapping of nucleosome positioning and DNA methylation within individual DNA molecules.” <em>Genome Research</em> 22 (12): 2497–2506. <a href="https://doi.org/10.1101/gr.143008.112">https://doi.org/10.1101/gr.143008.112</a>.</p>
</div>
<div id="ref-Kenyon1993">
<p>Kenyon, Cynthia, Jean Chang, Erin Gensch, Adam Rudner, and Ramon Tabtiang. 1993. “A C. elegans mutant that lives twice as long as wild type.” <em>Nature</em> 366 (6454): 461–64. <a href="https://doi.org/10.1038/366461a0">https://doi.org/10.1038/366461a0</a>.</p>
</div>
<div id="ref-Kerkel2008">
<p>Kerkel, Kristi, Alexandra Spadola, Eric Yuan, Jolanta Kosek, Le Jiang, Eldad Hod, Kerry Li, et al. 2008. “Genomic surveys by methylation-sensitive SNP analysis identify sequence-dependent allele-specific DNA methylation.” <em>Nature Genetics</em> 40 (7): 904–8. <a href="https://doi.org/10.1038/ng.174">https://doi.org/10.1038/ng.174</a>.</p>
</div>
<div id="ref-Knudson1971">
<p>Knudson, A G. 1971. “Mutation and cancer: statistical study of retinoblastoma.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 68 (4): 820–3. <a href="https://doi.org/10.1073/pnas.68.4.820">https://doi.org/10.1073/pnas.68.4.820</a>.</p>
</div>
<div id="ref-Koch2011">
<p>Koch, Carmen M., and Wolfgang Wagner. 2011. “Epigenetic-aging-signature to determine age in different tissues.” <em>Aging</em> 3 (10): 1018–27. <a href="https://doi.org/10.18632/aging.100395">https://doi.org/10.18632/aging.100395</a>.</p>
</div>
<div id="ref-Kontis2017">
<p>Kontis, Vasilis, James E. Bennett, Colin D. Mathers, Guangquan Li, Kyle Foreman, and Majid Ezzati. 2017. “Future life expectancy in 35 industrialised countries: projections with a Bayesian model ensemble.” <em>The Lancet</em> 389 (10076): 1323–35. <a href="https://doi.org/10.1016/S0140-6736(16)32381-9">https://doi.org/10.1016/S0140-6736(16)32381-9</a>.</p>
</div>
<div id="ref-Kunkel2004a">
<p>Kunkel, Thomas A. 2004. “DNA Replication Fidelity.” <em>Journal of Biological Chemistry</em> 279 (17): 16895–8. <a href="https://doi.org/10.1074/jbc.R400006200">https://doi.org/10.1074/jbc.R400006200</a>.</p>
</div>
<div id="ref-Laird2004">
<p>Laird, Charles D, Nicole D Pleasant, Aaron D Clark, Jessica L Sneeden, K M Anwarul Hassan, Nathan C Manley, Jay C Vary, Todd Morgan, R Scott Hansen, and R. Stoger. 2004. “Hairpin-bisulfite PCR: Assessing epigenetic methylation patterns on complementary strands of individual DNA molecules.” <em>Proceedings of the National Academy of Sciences</em> 101 (1): 204–9. <a href="https://doi.org/10.1073/pnas.2536758100">https://doi.org/10.1073/pnas.2536758100</a>.</p>
</div>
<div id="ref-Lander2001">
<p>Lander, E S, L M Linton, B Birren, C Nusbaum, M C Zody, J Baldwin, K Devon, et al. 2001. “Initial sequencing and analysis of the human genome.” <em>Nature</em> 409 (6822): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.</p>
</div>
<div id="ref-Levine2018">
<p>Levine, Morgan E, Ake T Lu, Austin Quach, Brian H Chen, Themistocles L Assimes, Stefania Bandinelli, Lifang Hou, et al. 2018. “An epigenetic biomarker of aging for lifespan and healthspan.” <em>Aging</em> 10 (4): 573–91. <a href="https://doi.org/10.18632/aging.101414">https://doi.org/10.18632/aging.101414</a>.</p>
</div>
<div id="ref-Li2015">
<p>Li, Jingqiu, Jie Yang, Ping Zhou, Yanping Le, Chengwei Zhou, Shaomin Wang, Dazhi Xu, Hui-Kuan Lin, and Zhaohui Gong. 2015. “Circular RNAs in cancer: novel insights into origins, properties, functions and implications.” <em>American Journal of Cancer Research</em> 5 (2): 472–80. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25973291">http://www.ncbi.nlm.nih.gov/pubmed/25973291</a>.</p>
</div>
<div id="ref-Lienert2011">
<p>Lienert, Florian, Christiane Wirbelauer, Indrani Som, Ann Dean, Fabio Mohn, and Dirk Schübeler. 2011. “Identification of genetic elements that autonomously determine DNA methylation states.” <em>Nature Genetics</em> 43 (11): 1091–7. <a href="https://doi.org/10.1038/ng.946">https://doi.org/10.1038/ng.946</a>.</p>
</div>
<div id="ref-Lienhard2014">
<p>Lienhard, Matthias, Christina Grimm, Markus Morkel, Ralf Herwig, and Lukas Chavez. 2014. “MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments.” <em>Bioinformatics</em> 30 (2): 284–86. <a href="https://doi.org/10.1093/bioinformatics/btt650">https://doi.org/10.1093/bioinformatics/btt650</a>.</p>
</div>
<div id="ref-Lister2009">
<p>Lister, Ryan, Mattia Pelizzola, Robert H Dowen, R David Hawkins, Gary Hon, Julian Tonti-Filippini, Joseph R Nery, et al. 2009. “Human DNA methylomes at base resolution show widespread epigenomic differences.” <em>Nature</em> 462 (7271): 315–22. <a href="https://doi.org/10.1038/nature08514">https://doi.org/10.1038/nature08514</a>.</p>
</div>
<div id="ref-Lowe2018">
<p>Lowe, Robert, Carl Barton, Christopher A. Jenkins, Christina Ernst, Oliver Forman, Denise S. Fernandez-Twinn, Christoph Bock, et al. 2018. “Ageing-associated DNA methylation dynamics are a molecular readout of lifespan variation among mammalian species.” <em>Genome Biology</em> 19 (1): 22. <a href="https://doi.org/10.1186/s13059-018-1397-1">https://doi.org/10.1186/s13059-018-1397-1</a>.</p>
</div>
<div id="ref-Lowe2020">
<p>Lowe, Robert, Amy F Danson, Vardhman K Rakyan, Selin Yildizoglu, Frédéric Saldmann, Melanie Viltard, Gérard Friedlander, and Chris G Faulkes. 2020. “DNA methylation clocks as a predictor for ageing and age estimation in naked mole-rats , Heterocephalus glaber” 12 (March): 1–13. <a href="https://doi.org/10.18632/aging.102892">https://doi.org/10.18632/aging.102892</a>.</p>
</div>
<div id="ref-Lopez-Otin2013">
<p>López-Otín, Carlos, Maria A. Blasco, Linda Partridge, Manuel Serrano, and Guido Kroemer. 2013. “The hallmarks of aging.” <em>Cell</em> 153 (6): 1194–1217. <a href="https://doi.org/10.1016/j.cell.2013.05.039">https://doi.org/10.1016/j.cell.2013.05.039</a>.</p>
</div>
<div id="ref-Lu2019">
<p>Lu, Ake T., Austin Quach, James G. Wilson, Alex P. Reiner, Abraham Aviv, Kenneth Raj, Lifang Hou, et al. 2019. “DNA methylation GrimAge strongly predicts lifespan and healthspan.” <em>Aging</em> 11 (2): 303–27. <a href="https://doi.org/10.18632/aging.101684">https://doi.org/10.18632/aging.101684</a>.</p>
</div>
<div id="ref-Luo2014">
<p>Luo, Yanting, Xuemei Lu, and Hehuang Xie. 2014. “Dynamic Alu Methylation during Normal Development, Aging, and Tumorigenesis.” <em>BioMed Research International</em> 2014: 1–12. <a href="https://doi.org/10.1155/2014/784706">https://doi.org/10.1155/2014/784706</a>.</p>
</div>
<div id="ref-Lyko2017">
<p>Lyko, Frank. 2017. “The DNA methyltransferase family: a versatile toolkit for epigenetic regulation.” <em>Nature Reviews Genetics</em> 19 (2): 81–92. <a href="https://doi.org/10.1038/nrg.2017.80">https://doi.org/10.1038/nrg.2017.80</a>.</p>
</div>
<div id="ref-Marioni2015">
<p>Marioni, Riccardo E, Sonia Shah, Allan F McRae, Brian H Chen, Elena Colicino, Sarah E Harris, Jude Gibson, et al. 2015. “DNA methylation age of blood predicts all-cause mortality in later life.” <em>Genome Biology</em> 16 (1): 25. <a href="https://doi.org/10.1186/s13059-015-0584-6">https://doi.org/10.1186/s13059-015-0584-6</a>.</p>
</div>
<div id="ref-Maunakea2010">
<p>Maunakea, Alika K, Raman P Nagarajan, Mikhail Bilenky, Tracy J Ballinger, Cletus D’Souza, Shaun D Fouse, Brett E Johnson, et al. 2010. “Conserved role of intragenic DNA methylation in regulating alternative promoters.” <em>Nature</em> 466 (7303): 253–7. <a href="https://doi.org/10.1038/nature09165">https://doi.org/10.1038/nature09165</a>.</p>
</div>
<div id="ref-Medvedev1990">
<p>Medvedev, Zhores A. 1990. “An Attempt at a Rational Classification of Theories of Ageing.” <em>Biological Reviews</em> 65 (3): 375–98. <a href="https://doi.org/10.1111/j.1469-185X.1990.tb01428.x">https://doi.org/10.1111/j.1469-185X.1990.tb01428.x</a>.</p>
</div>
<div id="ref-Meissner2005">
<p>Meissner, Alexander, Andreas Gnirke, George W Bell, Bernard Ramsahoye, Eric S Lander, and Rudolf Jaenisch. 2005. “Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis.” <em>Nucleic Acids Research</em> 33 (18): 5868–77. <a href="https://doi.org/10.1093/nar/gki901">https://doi.org/10.1093/nar/gki901</a>.</p>
</div>
<div id="ref-Niccoli2012">
<p>Niccoli, Teresa, and Linda Partridge. 2012. “Ageing as a risk factor for disease.” <em>Current Biology</em> 22 (17): R741–R752. <a href="https://doi.org/10.1016/j.cub.2012.07.024">https://doi.org/10.1016/j.cub.2012.07.024</a>.</p>
</div>
<div id="ref-Pfeifer2013">
<p>Pfeifer, Gerd P, Swati Kadam, and Seung-Gi Jin. 2013. “5-Hydroxymethylcytosine and Its Potential Roles in Development and Cancer.” <em>Epigenetics &amp; Chromatin</em> 6 (1): 10. <a href="https://doi.org/10.1186/1756-8935-6-10">https://doi.org/10.1186/1756-8935-6-10</a>.</p>
</div>
<div id="ref-Pfeifer1990a">
<p>Pfeifer, G P, S D Steigerwald, R S Hansen, S M Gartler, and A D Riggs. 1990. “Polymerase chain reaction-aided genomic sequencing of an X chromosome-linked CpG island: methylation patterns suggest clonal inheritance, CpG site autonomy, and an explanation of activity state stability.” <em>Proceedings of the National Academy of Sciences</em> 87 (21): 8252–6. <a href="https://doi.org/10.1073/pnas.87.21.8252">https://doi.org/10.1073/pnas.87.21.8252</a>.</p>
</div>
<div id="ref-Raiber2017">
<p>Raiber, Eun-ang, Guillem Portella, Sergio Martínez Cuesta, Robyn Hardisty, Pierre Murat, Zhe Li, Mario Iurlaro, et al. 2018. “5-Formylcytosine organizes nucleosomes and forms Schiff base interactions with histones in mouse embryonic stem cells.” <em>Nature Chemistry</em> 10 (12): 1258–66. <a href="https://doi.org/10.1038/s41557-018-0149-x">https://doi.org/10.1038/s41557-018-0149-x</a>.</p>
</div>
<div id="ref-Rakyan2010">
<p>Rakyan, Vardhman K., Thomas A. Down, Siarhei Maslau, Toby Andrew, Tsun Po Yang, Huriya Beyan, Pamela Whittaker, et al. 2010. “Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains.” <em>Genome Research</em> 20 (4): 434–39. <a href="https://doi.org/10.1101/gr.103101.109">https://doi.org/10.1101/gr.103101.109</a>.</p>
</div>
<div id="ref-Regev2017">
<p>Regev, Aviv, Sarah A. Teichmann, Eric S. Lander, Ido Amit, Christophe Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. “The Human Cell Atlas.” <em>eLife</em> 6 (December): 1–30. <a href="https://doi.org/10.7554/eLife.27041">https://doi.org/10.7554/eLife.27041</a>.</p>
</div>
<div id="ref-Rhoads2015">
<p>Rhoads, Anthony, and Kin Fai Au. 2015. “PacBio Sequencing and Its Applications.” <em>Genomics, Proteomics &amp; Bioinformatics</em> 13 (5): 278–89. <a href="https://doi.org/10.1016/j.gpb.2015.08.002">https://doi.org/10.1016/j.gpb.2015.08.002</a>.</p>
</div>
<div id="ref-Riggs2004a">
<p>Riggs, Arthur D, and Zhenggang Xiong. 2004. “Methylation and epigenetic fidelity.” <em>Proceedings of the National Academy of Sciences</em> 101 (1): 4–5. <a href="https://doi.org/10.1073/pnas.0307781100">https://doi.org/10.1073/pnas.0307781100</a>.</p>
</div>
<div id="ref-Rimmele2014">
<p>Rimmelé, Pauline, Carolina L. Bigarella, Raymond Liang, Brigitte Izac, Rebeca Dieguez-Gonzalez, Gaetan Barbet, Michael Donovan, et al. 2014. “Aging-like phenotype and defective lineage specification in SIRT1-deleted hematopoietic stem and progenitor cells.” <em>Stem Cell Reports</em> 3 (1): 44–59. <a href="https://doi.org/10.1016/j.stemcr.2014.04.015">https://doi.org/10.1016/j.stemcr.2014.04.015</a>.</p>
</div>
<div id="ref-Romanov1981">
<p>Romanov, G A, and B F Vanyushin. 1981. “Methylation of reiterated sequences in mammalian DNAs. Effects of the tissue type, age, malignancy and hormonal induction.” <em>Biochimica et Biophysica Acta</em> 653 (2): 204–18. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7225396">http://www.ncbi.nlm.nih.gov/pubmed/7225396</a>.</p>
</div>
<div id="ref-Rose2014">
<p>Rose, Nathan R., and Robert J. Klose. 2014. “Understanding the relationship between DNA methylation and histone lysine methylation.” <em>Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</em> 1839 (12): 1362–72. <a href="https://doi.org/10.1016/j.bbagrm.2014.02.007">https://doi.org/10.1016/j.bbagrm.2014.02.007</a>.</p>
</div>
<div id="ref-Russo1996">
<p>Russo, Vincenzo, Robert Martienssen, and Arthur D Riggs. 1996. <em>Epigenetic Mechanisms of Gene Regulation</em>. Cold Spring Harbor Laboratory Press. <a href="https://cshmonographs.org/index.php/monographs/issue/view/087969490.32">https://cshmonographs.org/index.php/monographs/issue/view/087969490.32</a>.</p>
</div>
<div id="ref-Saxonov2006">
<p>Saxonov, Serge, Paul Berg, and Douglas L Brutlag. 2006. “A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters.” <em>Proceedings of the National Academy of Sciences</em> 103 (5): 1412–7. <a href="https://doi.org/10.1073/pnas.0510310103">https://doi.org/10.1073/pnas.0510310103</a>.</p>
</div>
<div id="ref-Schalkwyk2010">
<p>Schalkwyk, Leonard C., Emma L. Meaburn, Rebecca Smith, Emma L. Dempster, Aaron R. Jeffries, Matthew N. Davies, Robert Plomin, and Jonathan Mill. 2010. “Allelic skewing of DNA methylation is widespread across the genome.” <em>American Journal of Human Genetics</em> 86 (2): 196–212. <a href="https://doi.org/10.1016/j.ajhg.2010.01.014">https://doi.org/10.1016/j.ajhg.2010.01.014</a>.</p>
</div>
<div id="ref-Schuettengruber2017">
<p>Schuettengruber, Bernd, Daniel Chourrout, Michel Vervoort, Benjamin Leblanc, and Giacomo Cavalli. 2007. “Genome Regulation by Polycomb and Trithorax Proteins.” <em>Cell</em> 128 (4): 735–45. <a href="https://doi.org/10.1016/j.cell.2007.02.009">https://doi.org/10.1016/j.cell.2007.02.009</a>.</p>
</div>
<div id="ref-Schultz2015">
<p>Schultz, Matthew D, Yupeng He, John W Whitaker, Manoj Hariharan, Eran A Mukamel, Danny Leung, Nisha Rajagopal, et al. 2015. “Human body epigenome maps reveal noncanonical DNA methylation variation.” <em>Nature</em> 523 (7559): 212–6. <a href="https://doi.org/10.1038/nature14465">https://doi.org/10.1038/nature14465</a>.</p>
</div>
<div id="ref-Serre2009a">
<p>Serre, David, Byron H. Lee, and Angela H. Ting. 2010. “MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome.” <em>Nucleic Acids Research</em> 38 (2): 391–99. <a href="https://doi.org/10.1093/nar/gkp992">https://doi.org/10.1093/nar/gkp992</a>.</p>
</div>
<div id="ref-Shoemaker2010">
<p>Shoemaker, Robert, Jie Deng, Wei Wang, and Kun Zhang. 2010. “Allele-specific methylation is prevalent and is contributed by CpG-SNPs in the human genome.” <em>Genome Research</em> 20 (7): 883–9. <a href="https://doi.org/10.1101/gr.104695.109">https://doi.org/10.1101/gr.104695.109</a>.</p>
</div>
<div id="ref-Simpson2017">
<p>Simpson, Jared T, Rachael E Workman, P C Zuzarte, Matei David, L J Dursi, and Winston Timp. 2017. “Detecting DNA cytosine methylation using nanopore sequencing.” <em>Nature Methods</em> 14 (4): 407–10. <a href="https://doi.org/10.1038/nmeth.4184">https://doi.org/10.1038/nmeth.4184</a>.</p>
</div>
<div id="ref-Sinclair2009">
<p>Sinclair, David A., and Philipp Oberdoerffer. 2009. “The ageing epigenome: Damaged beyond repair?” <em>Ageing Research Reviews</em> 8 (3): 189–98. <a href="https://doi.org/10.1016/j.arr.2009.04.004">https://doi.org/10.1016/j.arr.2009.04.004</a>.</p>
</div>
<div id="ref-Skvortsova2017">
<p>Skvortsova, Ksenia, Elena Zotenko, Phuc-Loi Luu, Cathryn M. Gould, Shalima S. Nair, Susan J. Clark, and Clare Stirzaker. 2017. “Comprehensive evaluation of genome-wide 5-hydroxymethylcytosine profiling approaches in human DNA.” <em>Epigenetics &amp; Chromatin</em> 10 (1): 16. <a href="https://doi.org/10.1186/s13072-017-0123-7">https://doi.org/10.1186/s13072-017-0123-7</a>.</p>
</div>
<div id="ref-Slieker2016">
<p>Slieker, Roderick C., Maarten van Iterson, René Luijk, Marian Beekman, Daria V Zhernakova, Matthijs H. Moed, Hailiang Mei, et al. 2016. “Age-related accrual of methylomic variability is linked to fundamental ageing mechanisms.” <em>Genome Biology</em> 17 (1): 191. <a href="https://doi.org/10.1186/s13059-016-1053-6">https://doi.org/10.1186/s13059-016-1053-6</a>.</p>
</div>
<div id="ref-Smith2014a">
<p>Smith, Alicia K., Varun Kilaru, Mehmet Kocak, Lynn M. Almli, Kristina B. Mercer, Kerry J. Ressler, Frances A. Tylavsky, and Karen N. Conneely. 2014. “Methylation quantitative trait loci (meQTLs) are consistently detected across ancestry, developmental stage, and tissue type.” <em>BMC Genomics</em> 15 (1): 145. <a href="https://doi.org/10.1186/1471-2164-15-145">https://doi.org/10.1186/1471-2164-15-145</a>.</p>
</div>
<div id="ref-Smith2015">
<p>Smith, Rick W A, Cara Monroe, and Deborah A. Bolnick. 2015. “Detection of Cytosine methylation in ancient DNA from five native american populations using bisulfite sequencing.” Edited by Ludovic Orlando. <em>PloS One</em> 10 (5): e0125344. <a href="https://doi.org/10.1371/journal.pone.0125344">https://doi.org/10.1371/journal.pone.0125344</a>.</p>
</div>
<div id="ref-Stadler2011">
<p>Stadler, Michael B., Rabih Murr, Lukas Burger, Robert Ivanek, Florian Lienert, Anne Schöler, Erik van Nimwegen, et al. 2011. “DNA-binding factors shape the mouse methylome at distal regulatory regions.” <em>Nature</em> 480 (7378): 490–5. <a href="https://doi.org/10.1038/nature10716">https://doi.org/10.1038/nature10716</a>.</p>
</div>
<div id="ref-Stubbs2017">
<p>Stubbs, Thomas M, Marc Jan Bonder, Anne-Katrien Stark, Felix Krueger, Ferdinand von Meyenn, Oliver Stegle, and Wolf Reik. 2017. “Multi-tissue DNA methylation age predictor in mouse.” <em>Genome Biology</em> 18 (1): 68. <a href="https://doi.org/10.1186/s13059-017-1203-5">https://doi.org/10.1186/s13059-017-1203-5</a>.</p>
</div>
<div id="ref-Teschendorff2010">
<p>Teschendorff, Andrew E., Usha Menon, Aleksandra Gentry-Maharaj, Susan J. Ramus, Daniel J. Weisenberger, Hui Shen, Mihaela Campan, et al. 2010. “Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.” <em>Genome Research</em> 20 (4): 440–6. <a href="https://doi.org/10.1101/gr.103606.109">https://doi.org/10.1101/gr.103606.109</a>.</p>
</div>
<div id="ref-Thompson2017">
<p>Thompson, Michael J, Bridgett VonHoldt, Steve Horvath, and Matteo Pellegrini. 2017. “An epigenetic aging clock for dogs and wolves.” <em>Aging</em> 9 (3): 1055–68. <a href="https://doi.org/10.18632/aging.101211">https://doi.org/10.18632/aging.101211</a>.</p>
</div>
<div id="ref-Ushijima2005">
<p>Ushijima, Toshikazu, Naoko Watanabe, Kimiko Shimizu, Kazuaki Miyamoto, Takashi Sugimura, and Atsushi Kaneda. 2005. “Decreased fidelity in replicating CpG methylation patterns in cancer cells.” <em>Cancer Research</em> 65 (1): 11–17. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15665274">http://www.ncbi.nlm.nih.gov/pubmed/15665274</a>.</p>
</div>
<div id="ref-Vertino2002">
<p>Vertino, Paula M., Jennifer A. Sekowski, Jennifer M. Coll, Nancy Applegren, Suhua Han, Robert J. Hickey, and Linda H. Malkas. 2002. “DNMT1 is a component of a multiprotein DNA replication complex.” <em>Cell Cycle (Georgetown, Tex.)</em> 1 (6): 416–23. <a href="https://doi.org/10.4161/cc.1.6.270">https://doi.org/10.4161/cc.1.6.270</a>.</p>
</div>
<div id="ref-Voigt2012">
<p>Voigt, Philipp, Gary LeRoy, William J. Drury, Barry M. Zee, Jinsook Son, David B. Beck, Nicolas L. Young, Benjamin A. Garcia, and Danny Reinberg. 2012. “Asymmetrically modified nucleosomes.” <em>Cell</em> 151 (1): 181–93. <a href="https://doi.org/10.1016/j.cell.2012.09.002">https://doi.org/10.1016/j.cell.2012.09.002</a>.</p>
</div>
<div id="ref-Waddington1942">
<p>Waddington, C. H. 2012. “The epigenotype. 1942.” <em>International Journal of Epidemiology</em> 41 (1): 10–13. <a href="https://doi.org/10.1093/ije/dyr184">https://doi.org/10.1093/ije/dyr184</a>.</p>
</div>
<div id="ref-Wang2019">
<p>Wang, Meng, and Bernardo Lemos. 2019. “Ribosomal DNA harbors an evolutionarily conserved clock of biological aging.” <em>Genome Research</em> 29 (3): 325–33. <a href="https://doi.org/10.1101/gr.241745.118">https://doi.org/10.1101/gr.241745.118</a>.</p>
</div>
<div id="ref-Weber2014">
<p>Weber, Christopher M, and Steven Henikoff. 2014. “Histone variants: dynamic punctuation in transcription.” <em>Genes &amp; Development</em> 28 (7): 672–82. <a href="https://doi.org/10.1101/gad.238873.114">https://doi.org/10.1101/gad.238873.114</a>.</p>
</div>
<div id="ref-Weber2005">
<p>Weber, Michael, Jonathan J Davies, David Wittig, Edward J Oakeley, Michael Haase, Wan L Lam, and Dirk Schübeler. 2005. “Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells.” <em>Nature Genetics</em> 37 (8): 853–62. <a href="https://doi.org/10.1038/ng1598">https://doi.org/10.1038/ng1598</a>.</p>
</div>
<div id="ref-Weidner2014">
<p>Weidner, Carola Ingrid, Qiong Lin, Carmen Maike Koch, Lewin Eisele, Fabian Beier, Patrick Ziegler, Dirk Olaf Bauerschlag, et al. 2014. “Aging of blood can be tracked by DNA methylation changes at just three CpG sites.” <em>Genome Biology</em> 15 (2): R24. <a href="https://doi.org/10.1186/gb-2014-15-2-r24">https://doi.org/10.1186/gb-2014-15-2-r24</a>.</p>
</div>
<div id="ref-Williams2019">
<p>Williams, Louise, Yanxia Bei, Heidi E Church, Nan Dai, Eileen T Dimalanta, Laurence M Ettwiller, Thomas C Evans, et al. 2019. “Enzymatic Methyl-seq: The Next Generation of Methylome Analysis BISULFITE SEQUENCING ALTERNATIVE METHODS FOR DETECTING 5mC AND 5hmC.” <a href="https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis">https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis</a>.</p>
</div>
<div id="ref-Wilson1983">
<p>Wilson, V L, and P A Jones. 1983. “DNA methylation decreases in aging but not in immortal cells.” <em>Science (New York, N.Y.)</em> 220 (4601): 1055–7. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.</p>
</div>
<div id="ref-Wilson1987">
<p>Wilson, V. L., R. A. Smith, S. Ma, and R. G. Cutler. 1987. “Genomic 5-methyldeoxycytidine decreases with age.” <em>The Journal of Biological Chemistry</em> 262 (21): 9948–51. <a href="http://www.ncbi.nlm.nih.gov/pubmed/3611071">http://www.ncbi.nlm.nih.gov/pubmed/3611071</a>.</p>
</div>
<div id="ref-Wu2016">
<p>Wu, Tao P., Tao Wang, Matthew G. Seetin, Yongquan Lai, Shijia Zhu, Kaixuan Lin, Yifei Liu, et al. 2016. “DNA methylation on N(6)-adenine in mammalian embryonic stem cells.” <em>Nature</em> 532 (7599): 329–33. <a href="https://doi.org/10.1038/nature17640">https://doi.org/10.1038/nature17640</a>.</p>
</div>
<div id="ref-Wu2017">
<p>Wu, Xiaoji, and Yi Zhang. 2017. “TET-mediated active DNA demethylation: mechanism, function and beyond.” <em>Nature Reviews Genetics</em> 18 (9): 517–34. <a href="https://doi.org/10.1038/nrg.2017.33">https://doi.org/10.1038/nrg.2017.33</a>.</p>
</div>
<div id="ref-Yin2017">
<p>Yin, Yimeng, Ekaterina Morgunova, Arttu Jolma, Eevi Kaasinen, Biswajyoti Sahu, Syed Khund-Sayeed, Pratyush K. Das, et al. 2017. “Impact of cytosine methylation on DNA binding specificities of human transcription factors.” <em>Science (New York, N.Y.)</em> 356 (6337): eaaj2239. <a href="https://doi.org/10.1126/science.aaj2239">https://doi.org/10.1126/science.aaj2239</a>.</p>
</div>
<div id="ref-Zaghlool2015">
<p>Zaghlool, Shaza B, Mashael Al-Shafai, Wadha A Al Muftah, Pankaj Kumar, Mario Falchi, and Karsten Suhre. 2015. “Association of DNA methylation with age, gender, and smoking in an Arab population.” <em>Clinical Epigenetics</em> 7 (1): 6. <a href="https://doi.org/10.1186/s13148-014-0040-6">https://doi.org/10.1186/s13148-014-0040-6</a>.</p>
</div>
<div id="ref-Zhang2017">
<p>Zhang, Yan, Jan Hapala, Hermann Brenner, and Wolfgang Wagner. 2017. “Individual CpG sites that are associated with age and life expectancy become hypomethylated upon aging.” <em>Clinical Epigenetics</em> 9 (1): 9. <a href="https://doi.org/10.1186/s13148-017-0315-9">https://doi.org/10.1186/s13148-017-0315-9</a>.</p>
</div>
<div id="ref-Zhao2014">
<p>Zhao, Lei, Ming-an Sun, Zejuan Li, Xue Bai, Miao Yu, Min Wang, Liji Liang, et al. 2014. “The dynamics of DNA methylation fidelity during mouse embryonic stem cell self-renewal and differentiation.” <em>Genome Research</em> 24 (8): 1296–1307. <a href="https://doi.org/10.1101/gr.163147.113">https://doi.org/10.1101/gr.163147.113</a>.</p>
</div>
<div id="ref-Ziller2013">
<p>Ziller, Michael J, Hongcang Gu, Fabian Müller, Julie Donaghey, Linus T-Y Tsai, Oliver Kohlbacher, Philip L De Jager, et al. 2013. “Charting a dynamic DNA methylation landscape of the human genome.” <em>Nature</em> 500 (7463): 477–81. <a href="https://doi.org/10.1038/nature12433">https://doi.org/10.1038/nature12433</a>.</p>
</div>
<div id="ref-Zuo2017">
<p>Zuo, Zheng, Basab Roy, Yiming Kenny Chang, David Granas, and Gary D Stormo. 2017. “Measuring quantitative effects of methylation on transcription factor–DNA binding affinity.” <em>Science Advances</em> 3 (11): eaao1799. <a href="https://doi.org/10.1126/sciadv.aao1799">https://doi.org/10.1126/sciadv.aao1799</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="acknowledgements.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/01-intro.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
