<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="3 Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="methods.html"/>
<link rel="next" href="tRNApaper.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.1</b> MeDIP-seq</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#illumina-methylation-arrays"><i class="fa fa-check"></i><b>2.2</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#whole-genome"><i class="fa fa-check"></i><b>2.3</b> Whole Genome</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.4</b> Targeted Bisulfite sequencing</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#discussion"><i class="fa fa-check"></i><b>2.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="MAVIDOS.html"><a href="MAVIDOS.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#background"><i class="fa fa-check"></i><b>3.2.1</b> Background</a></li>
<li class="chapter" data-level="3.2.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>3.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-models"><i class="fa fa-check"></i><b>3.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#summary-of-results"><i class="fa fa-check"></i><b>3.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="3.4.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#quality-control-qc"><i class="fa fa-check"></i><b>3.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="3.4.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation-1"><i class="fa fa-check"></i><b>3.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.4.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewass"><i class="fa fa-check"></i><b>3.4.4</b> EWASs</a></li>
<li class="chapter" data-level="3.4.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>3.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>3.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="3.5.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#section"><i class="fa fa-check"></i><b>3.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="MAVIDOS.html"><a href="MAVIDOS.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a><ul>
<li class="chapter" data-level="3.6.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#future-work"><i class="fa fa-check"></i><b>3.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNApaper.html"><a href="tRNApaper.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#Methods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNApaper.html"><a href="tRNApaper.html#participants"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNApaper.html"><a href="tRNApaper.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNApaper.html"><a href="tRNApaper.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNApaper.html"><a href="tRNApaper.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNApaper.html"><a href="tRNApaper.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNApaper.html"><a href="tRNApaper.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="4.3.9" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-figures"><i class="fa fa-check"></i><b>4.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.3.10" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>4.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#discussion-2"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="tRNApaper.html"><a href="tRNApaper.html#data-availability"><i class="fa fa-check"></i><b>4.6</b> Data availability</a></li>
<li class="chapter" data-level="4.7" data-path="tRNApaper.html"><a href="tRNApaper.html#code-availablity"><i class="fa fa-check"></i><b>4.7</b> Code availablity</a></li>
<li class="chapter" data-level="4.8" data-path="tRNApaper.html"><a href="tRNApaper.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.9" data-path="tRNApaper.html"><a href="tRNApaper.html#author-contributions"><i class="fa fa-check"></i><b>4.9</b> Author Contributions</a></li>
<li class="chapter" data-level="4.10" data-path="tRNApaper.html"><a href="tRNApaper.html#references"><i class="fa fa-check"></i><b>4.10</b> References</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results-3-.html"><a href="results-3-.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="results-3-.html"><a href="results-3-.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="results-3-.html"><a href="results-3-.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="results-3-.html"><a href="results-3-.html#methods-2"><i class="fa fa-check"></i><b>5.3</b> Methods</a></li>
<li class="chapter" data-level="5.4" data-path="results-3-.html"><a href="results-3-.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>5.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.4.2" data-path="results-3-.html"><a href="results-3-.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="5.4.3" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>5.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="5.4.4" data-path="results-3-.html"><a href="results-3-.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>5.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="results-3-.html"><a href="results-3-.html#discussion-3"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="results-3-.html"><a href="results-3-.html#methods-3"><i class="fa fa-check"></i><b>5.6</b> Methods</a><ul>
<li class="chapter" data-level="5.6.1" data-path="results-3-.html"><a href="results-3-.html#participants-1"><i class="fa fa-check"></i><b>5.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="results-3-.html"><a href="results-3-.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>5.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="5.7.1" data-path="results-3-.html"><a href="results-3-.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>5.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="5.7.2" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>5.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.7.3" data-path="results-3-.html"><a href="results-3-.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.7.4" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.7.5" data-path="results-3-.html"><a href="results-3-.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion-4.html"><a href="discussion-4.html"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="MAVIDOS" class="section level1">
<h1><span class="header-section-number">3</span> Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</h1>
<div id="abstract-1" class="section level2">
<h2><span class="header-section-number">3.1</span> Abstract</h2>
<p>Maternal vitamin D levels during pregnancy have previously been shown to impact on the methylation status of specific genes associated with vitamin D signalling and metabolism, such as <em>RXRA</em>.
Long-term bone health outcomes have also been associated with both maternal vitamin D during pregnancy and methylation of the <em>CDKN2A</em> gene.
To attempt to identify other loci the epigenetic regulation of which is pertinent to the impact of maternal vitamin D levels on long-term bone health Epigenome-Wide Association Studies (EWAS) were performed with DNA methylation data from the Illumina EPIC &amp; 450k methylation bead chip arrays (n=137 &amp; n=60, respectively).
The impact of intervention vs placebo status in a trial of vitamin D supplementation in pregnancy, neonatal bone mineral content, Maternal Vitamin D levels (34wks) and change in maternal Vitamin D levels from 11-34wks, on methylation were examined.
No loci were identified as changing significantly in methylation status in any of these EWAS, however, the power of this study was such that changes with small to medium effect sizes may have gone undetected.
It is our intention to extend this analysis with additional samples to increase the power.
We also intend to make use of this data to assess epigenetic estimates of biological and gestational age.</p>
</div>
<div id="introduction-1" class="section level2">
<h2><span class="header-section-number">3.2</span> Introduction</h2>
<div id="background" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Background</h3>
<p>Epigenetic states are a product of both genetics and environment, they exist on a continuum from complete or obligatory genetic influence through facultative variation influenced by genetics, to the purely epigenetic subject entirely to environmental factors.
This picture is complicated by the observation that genetics can not merely influence epigenetic state but also the extent to which that state will vary depending on environmental factors <span class="citation">(Holland et al. <a href="#ref-Holland2017" role="doc-biblioref">2016</a>)</span>.
Epigenome-wide association studies have been performed for many traits including age (Section <a href="introduction.html#DNAmAgeReview">1.5.1</a>), smoking status <span class="citation">(Dogan, Beach, and Philibert <a href="#ref-Dogan2017" role="doc-biblioref">2017</a>)</span>, exposure to atmospheric particulate matter [Dai2016], and obesity <span class="citation">(Wahl et al. <a href="#ref-Wahl2016" role="doc-biblioref">2017</a>)</span>.
Some robust and reproducible epigenetic changes have been identified with EWAS, particularly for traits such as a age and smoking.
Unlike genome-wide association studies (GWAS) associations found by EWAS can be due to reverse causation.
That is to say, a genetic variant associated with a disease is very unlikely to have been caused by the disease state whereas an epigenetic association can be the consequence of a disease state.
Whilst this can complicate the dissection of disease aetiology it can also be a boon in unpicking the biology that follows on from a particular disease or environmental exposure by revealing what biological networks are affected.</p>
<p>Work in rat models has demonstrated the ability of maternal nutritional exposure during pregnancy to influence the methylation and expression of genes in their offspring <span class="citation">(G. C. Burdge et al. <a href="#ref-Burdge2007a" role="doc-biblioref">2007</a>; Lillycrop et al. <a href="#ref-Lillycrop2008" role="doc-biblioref">2008</a>)</span> (reviewed in <span class="citation">(Burdge et al. <a href="#ref-Burdge2007" role="doc-biblioref">2007</a>)</span>).
Consequently, the methylation status of target genes of interest for vitamin D and bone development related pathways have been examined.</p>
<p>Retinoid-X-receptor-alpha (RXRA), which forms a heterodimer with the vitamin D receptor is essential for the nuclear action of <span class="math inline">\(1,25(OH)_2-Vitamin~D\)</span>.
DNA methylation at four of ten CpG sites in the <em>RXRA</em> gene were significantly (<span class="math inline">\(p\le0.05\)</span>) lower in the umbilical cord of offspring from cholecalciferol supplemented mothers compared to placebo with a mean difference in methylation of -1.98% (n=447, 95% CI -3.65 to -0.32, p=0.01).
One CpG site in the gene is related to estimated free 25(OH)D levels <span class="citation">(Harvey et al. <a href="#ref-Harvey2014d" role="doc-biblioref">2014</a>)</span>.</p>
<p>Methylation of cyclin-dependent kinase inhibitor 2A (CDKN2A) in umbilical cord tissue has been implicated in bone cell activity mediating skeletal development and homeostasis.
The <em>CDKN2A</em> locus has quite complex biology encoding two cell cycle inhibitors <span class="math inline">\(p14^{ARF}\)</span> and <span class="math inline">\(p16^{INK4a}\)</span> as well as the long non-coding RNA <em>ANRIL</em> which inhibits <span class="math inline">\(p16^{INK4a}\)</span>.
Targeted analysis of nine CpGs in a 300bp stretch of the <em>ANRIL</em> promoter region of <em>CDKN2A</em> was carried out.
Methylation at several of these sites showed an inverse correlation with bone size, mineral content, and mineral density at age four years <span class="citation">(Curtis et al. <a href="#ref-Curtis2017" role="doc-biblioref">2017</a>)</span>.</p>
<p>The MAVIDOS trial is a randomised, double-blind, placebo-controlled trial of vitamin D supplementation in pregnancy focused on bone development outcomes <span class="citation">(Harvey et al. <a href="#ref-Harvey2012a" role="doc-biblioref">2012</a>; Cooper et al. <a href="#ref-Cooper2016" role="doc-biblioref">2016</a>)</span>.
Motivated by findings that low levels of maternal 25(OH)-Vitamin D during gestation have been associated with reduced bone mass, extending to peak bone mass <span class="citation">(Zhu et al. <a href="#ref-Zhu2014a" role="doc-biblioref">2014</a>)</span> and altered femoral morphology; leading to increased risk of osteoporosis in later life <span class="citation">(Mahon et al. <a href="#ref-Mahon2010a" role="doc-biblioref">2010</a>; Viljakainen et al. <a href="#ref-Viljakainen2010" role="doc-biblioref">2010</a>, <a href="#ref-Viljakainen2011" role="doc-biblioref">2011</a>)</span>.
<span class="math inline">\(1,25(OH)_2-Vitamin~D\)</span> has been implicated in the control of Plasma membrane <span class="math inline">\(Ca^{2+}\)</span>-ATPases (PMCAs).
The expression of the <em>PMCA3</em> gene in the placenta has been associated with umbilical cord calcium concentration and intrauterine accrual of bone mineral content <span class="citation">(Kip and Strehler <a href="#ref-Kip2004" role="doc-biblioref">2004</a>)</span>.
This provides part of a candidate mechanism linking vitamin D to bone outcomes.</p>
</div>
<div id="ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes" class="section level3">
<h3><span class="header-section-number">3.2.2</span> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</h3>
<p>This work sought to identify additional loci with changes in DNAm associated with bone development outcomes and vitamin D, beyond those identified in previous work targeted to specific loci.
An ‘epigenome-wide’ approach examining ~850k sites in the genome was carried out using the Illumina EPIC methylation bead chip array on umbilical cord tissue samples from participants in the MAVIDOS study (n=140).
Results were compared with additional samples run on the Illumina 450k array (n=60), which as the name suggests covers only ~450k sites.</p>
<p>Epigenome-Wide Association Studies (EWAS) were performed for four variables of interest: intervention vs placebo status, neonatal bone mineral content(g) as determined by dual-energy X-ray absorptiometry (DXA) scans, Maternal Vitamin D levels (34wks) and change in maternal Vitamin D levels from 11 to 34wks.</p>
<blockquote>
<p>Subsequent analyses planned with this data include: Testing the effectiveness of epigenetic predictors of gestational age developed in cord blood <span class="citation">(Bohlin et al. <a href="#ref-Bohlin2016" role="doc-biblioref">2016</a>; Knight et al. <a href="#ref-Knight2016" role="doc-biblioref">2016</a>)</span> in cord tissue samples.
Potentially testing if ‘phenotypic age’ as predicted by DNA methylation <span class="citation">(Levine et al. <a href="#ref-Levine2018" role="doc-biblioref">2018</a>)</span> is related to gestational age, unlike DNA methylation age <span class="citation">(Simpkin et al. <a href="#ref-Simpkin2016" role="doc-biblioref">2016</a>; Andrew J Simpkin et al. <a href="#ref-Simpkin2017" role="doc-biblioref">2017</a>)</span>.</p>
</blockquote>
</div>
</div>
<div id="methods-1" class="section level2">
<h2><span class="header-section-number">3.3</span> Methods</h2>
<p>Analysis of the Illumina EPIC and 450k methylation arrays was carried out in the R language using the <a href="https://github.com/perishky/meffil/wiki">‘meffil’</a> package <span class="citation">(Min et al. <a href="#ref-Min2017" role="doc-biblioref">2017</a>)</span>, which is capable of performing functional normalisation in a more memory efficient fashion than alternatives such as ‘minfi’.</p>
<div id="functional-normalisation" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Functional Normalisation</h3>
<p>Functional normalisation is an approach to removing unwanted variation associated with ‘batch’ effects such as the date on which a sample was analysed or which slide a sample is on developed by <span class="citation">Fortin et al. (<a href="#ref-Fortin2014" role="doc-biblioref">2014</a>)</span>.
This noise in the data masks the signal associated with the underlying biological effect of interest.</p>
<p>Functional normalisation makes use of control probes on the arrays which are designed to capture only technical variation as surrogates for the sources of unwanted variation.
The control probes are processed into 42 summary measures, and principal components analysis is performed on the control probe summary matrices for all samples.
The top <em>m</em> principal components (PCs) are used as the surrogates for technical variation going forward.
The number, <em>m</em>, of PCs from the control probe summary matrices used is informed by the amount of residual variation remaining after normalisation.
Picking the number of PCs which correspond to the last steep drop in residual variation is the approach recommended by the implementers of meffil, and <span class="citation">Fortin et al. (<a href="#ref-Fortin2014" role="doc-biblioref">2014</a>)</span> recommend an <em>m</em> of 2 as performing well across a variety of analyses.
(See supplementary material from <span class="citation">(Fortin et al. <a href="#ref-Fortin2014" role="doc-biblioref">2014</a>)</span> for details of the control probe summary process.)</p>
<p>The process used by functional normalisation is a variant on quantile normalisation.
Instead of forcing the empirical marginal distributions of the samples to be identical at each site across arrays it constructs quantile function which only removes variation arising from surrogates for batch variation to produce the normalised values.
This approach is effective even when comparing samples with large global differences in methylation levels such as between normal and cancer samples, but cannot overcome high degrees of confounding <span class="citation">(Fortin et al. <a href="#ref-Fortin2014" role="doc-biblioref">2014</a>)</span>.</p>
</div>
<div id="ewas-models" class="section level3">
<h3><span class="header-section-number">3.3.2</span> EWAS Models</h3>
<div id="cell-type-correction" class="section level4">
<h4><span class="header-section-number">3.3.2.1</span> Cell-Type Correction</h4>
<p>Blood cell composition is a known confounder in epigenetic studies <span class="citation">(Jaffe and Irizarry <a href="#ref-Jaffe2014" role="doc-biblioref">2014</a>; Houseman et al. <a href="#ref-Houseman2012" role="doc-biblioref">2012</a>)</span>.
Observed variation DNA methylation can be cell-type intrinsic, changes in DNAm not driven by changes in the cell-type composition of the tissue, or cell-type extrinsic, due to changes in the cell-type composition of the population sampled (Figure <a href="MAVIDOS.html#fig:IntrinsicVsExtrinsicDNAmChangeGeneral">3.1</a>).
An established approach for addressing this potential source of confounding it is to add terms to the regression model which reflect the cell type composition of each sample.
Cell type composition can be ascertained through three main approaches: Direct cell count data; predicted cell-count based on comparing experimental methylation data to methylation data from reference samples of the tissue in question <span class="citation">(Houseman et al. <a href="#ref-Houseman2012" role="doc-biblioref">2012</a>)</span>; or reference free approaches which make use of mathematical methods to identify sources of confounding variation such as cell-type heterogeneity.
Whilst there are several reference samples available for cord blood <span class="citation">(Cardenas et al. <a href="#ref-Cardenas2016" role="doc-biblioref">2016</a>; Goede et al. <a href="#ref-DeGoede2015" role="doc-biblioref">2015</a>; Bakulski et al. <a href="#ref-Bakulski2016" role="doc-biblioref">2016</a>; Gervin et al. <a href="#ref-Gervin2016" role="doc-biblioref">2016</a>)</span>, and one for cord tissue reference was recently published <span class="citation">(Lin et al. <a href="#ref-Lin2018" role="doc-biblioref">2018</a>)</span>.
This cell-type reference had not yet been integrated into the <code>meffil</code> R package I was using to perform the EWAS analyses so I created <a href="https://github.com/RichardJActon/meffil/blob/LinAndKarnaniCordTissueRef/data-raw/lin-karnani-reference.r">a fork of <code>meffil</code> including this reference panel in the required format</a>, this can be installed directly from github.
The code necessary to add this reference to the package data before building the R package can be found in this <a href="https://gist.github.com/RichardJActon/703ae70438e865a37998cd608bd12cd3">gist</a>.</p>
<p><del>none were found for cord tissue samples, consequently, we made use of reference free methods for cell type correction.</del></p>
<p>I performed EWAS using Surrogate Variable Analysis (SVA) <span class="citation">(Leek and Storey <a href="#ref-Leek2007" role="doc-biblioref">2007</a>)</span> and Independent surrogate variable analysis (iSVA) <span class="citation">(Teschendorff, Zhuang, and Widschwendter <a href="#ref-Teschendorff2011" role="doc-biblioref">2011</a>)</span>, the results displayed here are those generated using SVA as this method was recommended based on comparisons of the performance of various cell-type heterogeneity correction methods <span class="citation">(McGregor et al. <a href="#ref-McGregor2016" role="doc-biblioref">2016</a>; Teschendorff and Zheng <a href="#ref-Teschendorff2017" role="doc-biblioref">2017</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:IntrinsicVsExtrinsicDNAmChangeGeneral"></span>
<img src="figs/IntrinsicVsExtrinsicDNAmChangeGeneral.png" alt="Diagramatic representation of DNAm change arrising from extrinsic, cell-type composition change and intrinic, DNAm of cells changes without the proportion of cell-types in the population changing. These two modes of change are of course not mutually exclusive and both can be occuring." width="80%" />
<p class="caption">
Figure 3.1: Diagramatic representation of DNAm change arrising from extrinsic, cell-type composition change and intrinic, DNAm of cells changes without the proportion of cell-types in the population changing. These two modes of change are of course not mutually exclusive and both can be occuring.
</p>
</div>

</div>
<div id="custom-cord-tissue-ref-add-new-results" class="section level4">
<h4><span class="header-section-number">3.3.2.2</span> !!! custom cord tissue ref ~ Add new results</h4>
</div>
<div id="covariates-included-in-the-models" class="section level4">
<h4><span class="header-section-number">3.3.2.3</span> Covariates Included in the Models</h4>
<p>By default EWAS in meffil are run with four different models:</p>
<ol style="list-style-type: decimal">
<li>No covariates, attempting to predict methylation with the variable of interest.</li>
<li>All covariates, attempting to predict methylation with the variable of interest plus a user-supplied list of covariates.</li>
<li>SVs + all covariates, attempting to predict methylation with user-supplied covariates and surrogate variables generated from SVA.</li>
<li>iSVs + all covariates, attempting to predict methylation with user-supplied covariates and independent surrogate variables generated from iSVA.</li>
</ol>
<p>In all the EWAS run here age at which bone measurements were made and sex were supplied as covariates.</p>
<p>Running EWAS with multiple models permits the effects of adding the various covariates on the results of the analysis to be seen.
We observed no major differences in the results when using models with and without covariates so have reported here the results of model 3 above.
That is to say, the covariates used were the surrogate variables produced from SVA, Age (days) and Sex.</p>
</div>
</div>
<div id="concordance-of-epic-and-450k-ewas-results" class="section level3">
<h3><span class="header-section-number">3.3.3</span> Concordance of EPIC and 450k EWAS Results</h3>
<blockquote>
<p>Concordance between the top ranking probes by p-value with SVA correction was determined using the code which can be found in Section xxx: concordanceCode.</p>
</blockquote>
</div>
</div>
<div id="results" class="section level2">
<h2><span class="header-section-number">3.4</span> Results</h2>
<div id="summary-of-results" class="section level3">
<h3><span class="header-section-number">3.4.1</span> Summary of Results</h3>
<p>DNAm at none of the probes were significantly associated with the any of the four variables of interest for which EWAS were performed in either the EPIC or 450k datasets.
The concordance between the probes with the top-ranked p-values in common between the EPIC and 450k data was at the level of chance.
3 of the EPIC arrays were excluded from the analysis for failing quality control leaving an n=137.</p>
</div>
<div id="quality-control-qc" class="section level3">
<h3><span class="header-section-number">3.4.2</span> Quality Control (QC)</h3>
<div id="whole-array-qc" class="section level4">
<h4><span class="header-section-number">3.4.2.1</span> Whole array QC</h4>
<div id="epic-arrays" class="section level5">
<h5><span class="header-section-number">3.4.2.1.1</span> EPIC arrays</h5>
<p>The predicted sex of the samples generated using sex chromosome probe intensities was checked against that in the sample annotation and two mismatches were found.
These were MAVIDOS IDs 206 and 63 and the associated arrays were excluded from further analysis, (Figure <a href="MAVIDOS.html#fig:MAVIDOSqcEPICsexMismatch">3.2</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICsexMismatch"></span>
<img src="figs/MAVIDOSqcEPICsexMismatch.png" alt="Predicted sex of each sample based on the sex chromosome copy numbers infered from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotions. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.2: Predicted sex of each sample based on the sex chromosome copy numbers infered from probe intensities for the EPIC array data. Mismatches between the predicted sex and that asserted in the sample annotation metadata are shown in red. Two predicted sex values differ from their annotions. Plot generated by meffil QC report.
</p>
</div>

<p>The dataset also contained four samples for which there were two technical replicates, only the first replicate from each was used (144 arrays run for 140 individuals).
Array 201516310023 (MAVIDOS ID 95) was excluded as its median methylated signal was more than <span class="math inline">\(3\sigma\)</span> from the expected value, (Figure <a href="MAVIDOS.html#fig:MAVIDOSqcEPICmethVsUnmeth">3.3</a>).
No samples were excluded for having a higher than expected proportion of undetected probes (proportion of probes with detection p-value &gt; 0.01 is &gt; 0.1) (Figure <a href="MAVIDOS.html#fig:MAVIDOSqcEPICpropSites">3.4</a>).
No samples were excluded for having a high proportion of probes with low bead counts (proportion of probes with bead number &lt; 3 is &gt; 0.1), (Figure <a href="MAVIDOS.html#fig:MAVIDOSqcEPICbeadNum">3.5</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICmethVsUnmeth"></span>
<img src="figs/MAVIDOSqcEPICmethVsUnmeth.png" alt="Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing \(3\sigma\) from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.3: Median methylated signal vs unmethylated signal per sample for the EPIC array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing <span class="math inline">\(3\sigma\)</span> from the expected mean. Samples outside the expected range are indicated in the legend. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICpropSites"></span>
<img src="figs/MAVIDOSqcEPICpropSites.png" alt="Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.4: Proportion of probes with detection p-values &gt;0.01 by sample for the EPIC array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICbeadNum"></span>
<img src="figs/MAVIDOSqcEPICbeadNum.png" alt="(ref:MAVIDOSqcEPICbeadNum-Cap)" width="80%" />
<p class="caption">
Figure 3.5: (ref:MAVIDOSqcEPICbeadNum-Cap)
</p>
</div>
<p>(ref:MAVIDOSqcEPICbeadNum-Cap) Proportion of probes with a bead count of &lt; 3 by sample for the EPIC array data.
Black line indicates the exclusion threshold of 0.1.
Plot generated by meffil QC report.</p>
</div>
<div id="k-arrays" class="section level5">
<h5><span class="header-section-number">3.4.2.1.2</span> 450k Arrays</h5>
<p>There were no mismatches between predicted and annotated sex (Figure <a href="MAVIDOS.html#fig:MAVIDOSqc450ksexMismatch">3.6</a>).
There were not any samples with outliers in their methylated / unmethylated probe proportions (Figure <a href="MAVIDOS.html#fig:MAVIDOSqc450kmethVsUnmeth">3.7</a>).
No samples were excluded for having a higher than expected proportion of undetected probes (proportion of probes with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="MAVIDOS.html#fig:MAVIDOSqc450kpropSites">3.8</a>).
No samples were excluded for having a high proportion of probes with low bead counts (proportion of probes with bead number &lt; 3 is &gt; 0.1), (Figure <a href="MAVIDOS.html#fig:MAVIDOSqc450kbeadNum">3.9</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450ksexMismatch"></span>
<img src="figs/MAVIDOSqc450ksexMismatch.png" alt="Predicted sex of each sample based on the sex chromosome copy numbers infered from probe intensities for the 450k array data. No predicted sex values differ from their annotions. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.6: Predicted sex of each sample based on the sex chromosome copy numbers infered from probe intensities for the 450k array data. No predicted sex values differ from their annotions. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kmethVsUnmeth"></span>
<img src="figs/MAVIDOSqc450kmethVsUnmeth.png" alt="Median methylated signal vs unmethylated signal per sample for the 450k array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing \(3\sigma\) from the expected mean. Samples outside the expected range would be indicated in the legend. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.7: Median methylated signal vs unmethylated signal per sample for the 450k array data, solid red line indicates linear regression of median methylated signal vs median unmethylated signal with dotted red lines representing <span class="math inline">\(3\sigma\)</span> from the expected mean. Samples outside the expected range would be indicated in the legend. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kpropSites"></span>
<img src="figs/MAVIDOSqc450kpropSites.png" alt="Proportion of probes with detection p-values &gt;0.01 by sample for the 450k array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.8: Proportion of probes with detection p-values &gt;0.01 by sample for the 450k array data. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kbeadNum"></span>
<img src="figs/MAVIDOSqc450kbeadNum.png" alt="(ref:MAVIDOSqc450kbeadNum-Cap)" width="80%" />
<p class="caption">
Figure 3.9: (ref:MAVIDOSqc450kbeadNum-Cap)
</p>
</div>
<p>(ref:MAVIDOSqc450kbeadNum-Cap) Proportion of probes with a bead count of &lt; 3 by sample for the 450k array data.
Black line indicates the exclusion threshold of 0.1.
Plot generated by meffil QC report.</p>
</div>
</div>
<div id="probe-qc" class="section level4">
<h4><span class="header-section-number">3.4.2.2</span> Probe QC</h4>
<div id="probe-qc---epic-arrays" class="section level5">
<h5><span class="header-section-number">3.4.2.2.1</span> Probe QC - EPIC Arrays</h5>
<div id="probe-quality" class="section level6">
<h6><span class="header-section-number">3.4.2.2.1.1</span> Probe Quality</h6>
<p>There were no outliers within the control probes Figure <a href="MAVIDOS.html#fig:MAVIDOSqcEPICctrlProbes">3.10</a>.
1,626 probes were excluded for having high background signal in a high proportion of samples (proportion of samples with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="MAVIDOS.html#fig:MAVIDOSqcEPICpropSamples">3.11</a>).
162 probes were excluded for having low bead count in a high proportion of samples (proportion of samples with bead number &lt; 3 is &gt; 0.1), (Figure <a href="MAVIDOS.html#fig:MAVIDOSqcEPICbeadNumAcross">3.12</a>).
Probes with poor technical quality were excluded from the analysis prior to functional normalisation.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICctrlProbes"></span>
<img src="figs/MAVIDOSqcEPICctrlProbes.png" alt="(ref:MAVIDOSqcEPICctrlProbes-Cap)" width="80%" />
<p class="caption">
Figure 3.10: (ref:MAVIDOSqcEPICctrlProbes-Cap)
</p>
</div>
<p>(ref:MAVIDOSqcEPICctrlProbes-Cap) Control probe signal by sample for each summary group for the EPIC data.
Outliers would be circled in black. Plot generated by meffil QC report.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICpropSamples"></span>
<img src="figs/MAVIDOSqcEPICpropSamples.png" alt="Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.11: Undetectable probes across samples for EPIC data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqcEPICbeadNumAcross"></span>
<img src="figs/MAVIDOSqcEPICbeadNumAcross.png" alt="Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.12: Low bead count probes across samples for EPIC data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report.
</p>
</div>

</div>
<div id="multi-mapping-and-genetically-confounded-probes" class="section level6">
<h6><span class="header-section-number">3.4.2.2.1.2</span> Multi-mapping and Genetically Confounded Probes</h6>
<p>Probes from different locations in the genome with similar sequences, especially following the reduction in sequence complexity associated with bisulfite conversion, can be cross-reactive on the arrays leading to erroneous signals and are thus commonly excluded from analysis.
DNAm is often strongly influenced by genetic factors, the effect is especially pronounced when variants alter the sequence at CpG sites themselves as the site can then no longer be methylated.
Thus, sites that have common genetic variants at the probe sites are also excluded from the analysis, as well as some sites whose methylation is known to be under strong genetic influence by common genetic variation.</p>
<p>43,254 probes on the EPIC array have been identified as multi-mapping, DNA binding to the probes may be derived from other locations in the genome invalidating these probes as a measure of methylation at their intended loci <span class="citation">(Pidsley et al. <a href="#ref-Pidsley2016" role="doc-biblioref">2016</a>)</span>.
12,510 probes were identified as having genetic variants at the CpG locus they are intending to assay, this can produce misleading results as mutant bases can resemble the products of bisulfite conversion <span class="citation">(Pidsley et al. <a href="#ref-Pidsley2016" role="doc-biblioref">2016</a>)</span>.
1,812 probes overlapping regions exhibiting haplotype-specific methylation associated with common non-SNP genetic variants (CNVs, Indels, STRs) and regional in-phase clustering of CpG-SNPs <span class="citation">(Bell et al. <a href="#ref-Bell2017a" role="doc-biblioref">2018</a>)</span>.
These were excluded from subsequent analysis after functional normalisation.
This, including the poor quality probes, is a total of 57,396 unique probes excluded from the analysis (~6.62% of the total number of probes)</p>
<p>In order to identify any potential additional sources of genetic confounding, we looked for probes with methylation values which cluster into distinct groups using the ‘gap hunting’ method developed by <span class="citation">Andrews et al. (<a href="#ref-Andrews2016" role="doc-biblioref">2016</a>)</span>.
Such distinct clusters of methylation can arise from genetic variants which influence methylation levels being present in homozygous and heterozygous forms in the study population, see Figure <a href="MAVIDOS.html#fig:MAVIDOSgaphunterEgMonApr162018">3.13</a>.
As the sensitivity and specificity of ‘gap hunting’ is limited, it is the advice of the authors not to exclude probes flagged by ‘gaphunter()’ prior to performing EWAS.
It is instead advised to check if any of the results appear in this list after the fact and proceed with caution if they are.
Gap hunter identified a further 77,398 probes (8.9% of all probes) which might be subject to genetic confounding.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSgaphunterEgMonApr162018"></span>
<img src="figs/MAVIDOSgaphunterEgMonApr162018.png" alt="An example of the DNAm distribution for a result from the ‘gaphunter()’ function. This is an example chosen to best exemplify the sort of result which is strongly suggestive of a genetic variant with an impact on methylation status acting on this site. It is unrepresentative of typical results from ‘gaphunter()’ in that the groups have a relatively even membership, many results have a small number of individuals in one or more groups making it hard to distinguish methylation outliers caused by rarer genetic variants from those with other causes." width="80%" />
<p class="caption">
Figure 3.13: An example of the DNAm distribution for a result from the ‘gaphunter()’ function. This is an example chosen to best exemplify the sort of result which is strongly suggestive of a genetic variant with an impact on methylation status acting on this site. It is unrepresentative of typical results from ‘gaphunter()’ in that the groups have a relatively even membership, many results have a small number of individuals in one or more groups making it hard to distinguish methylation outliers caused by rarer genetic variants from those with other causes.
</p>
</div>

</div>
</div>
<div id="probe-qc---450k-arrays" class="section level5">
<h5><span class="header-section-number">3.4.2.2.2</span> Probe QC - 450k Arrays</h5>
<div id="probe-quality-1" class="section level6">
<h6><span class="header-section-number">3.4.2.2.2.1</span> Probe Quality</h6>
<p>There was one sample (MAVIDOS ID 2183) with an outlier within the control probes, the ‘stain.R’ probe, (Figure <a href="MAVIDOS.html#fig:MAVIDOSqc450kctrlProbes">3.14</a>).
509 probes were excluded for having high background signal in a high proportion of samples (proportion of samples with detection p-value &gt; 0.01 is &gt; 0.1), (Figure <a href="MAVIDOS.html#fig:MAVIDOSqc450kpropSamples">3.15</a>).
1037 probes were excluded for having low bead count in a high proportion of samples (proportion of samples with bead number &lt; 3 is &gt; 0.1) (Figure <a href="MAVIDOS.html#fig:MAVIDOSqc450kbeadNumAcross">3.16</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kctrlProbes"></span>
<img src="figs/MAVIDOSqc450kctrlProbes.png" alt="Control probe signal by sample for each summary group for the 450k data. Outliers would be circled in black. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.14: Control probe signal by sample for each summary group for the 450k data. Outliers would be circled in black. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kpropSamples"></span>
<img src="figs/MAVIDOSqc450kpropSamples.png" alt="Undetectable probes across samples for 450k data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.15: Undetectable probes across samples for 450k data. Manhattan plot showing proportion of samples (y) in which a given probe (x) is not distinguishable from background noise, i.e. a detection p-value of &gt; 0.01. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSqc450kbeadNumAcross"></span>
<img src="figs/MAVIDOSqc450kbeadNumAcross.png" alt="Low bead count probes across samples for 450k data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report." width="80%" />
<p class="caption">
Figure 3.16: Low bead count probes across samples for 450k data. Manhattan plot showing the proportion of samples (y) in which a given probe (x) has a bead count of &lt;3. Black line indicates the exclusion threshold of 0.1. Plot generated by meffil QC report.
</p>
</div>

</div>
<div id="multi-mapping-and-genetically-confounded-probes-1" class="section level6">
<h6><span class="header-section-number">3.4.2.2.2.2</span> Multi-mapping and Genetically Confounded Probes</h6>
<p><span class="citation">Zhou, Laird, and Shen (<a href="#ref-Zhou2017" role="doc-biblioref">2016</a>)</span> provided a list of probes which they recommend ‘masking’ from the 450k array due to, multi-mapping issues, genetic variants overlapping the CpG sites and other factors which may render results from these probes problematic.
All probes on this list were excluded from the analysis following functional normalisation, leaving a total of 418,632 probes for subsequent analysis.</p>
</div>
</div>
</div>
</div>
<div id="functional-normalisation-1" class="section level3">
<h3><span class="header-section-number">3.4.3</span> Functional Normalisation</h3>
<p>An <em>m</em> of 6 was chosen for the EPIC arrays as this value produced the last steep drop in residual variation, see Figure <a href="MAVIDOS.html#fig:MAVIDOSfunnormPCsEPICMonApr162018">3.17</a>.
An <em>m</em> of 6 was chosen for the 450k arrays as this value produced the last steep drop in residual variation, see Figure <a href="MAVIDOS.html#fig:MAVIDOSfunnormPCs450kMonApr162018">3.18</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSfunnormPCsEPICMonApr162018"></span>
<img src="figs/MAVIDOSfunnormPCsEPICMonApr162018.png" alt="Residual variation remaining after functional normalisation of the top 20,000 most variable probes with m PCs from the control probe summary matrices for the EPIC array samples (n=137), for M = methylated and U = unmethylated probes." width="80%" />
<p class="caption">
Figure 3.17: Residual variation remaining after functional normalisation of the top 20,000 most variable probes with <em>m</em> PCs from the control probe summary matrices for the EPIC array samples (n=137), for M = methylated and U = unmethylated probes.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSfunnormPCs450kMonApr162018"></span>
<img src="figs/MAVIDOSfunnormPCs450kMonApr162018.png" alt="Residual variation remaining after functional normalisation of the top 20,000 most variable probes with m PCs from the control probe summary matrices for the 450k array samples (n=60), for M = methylated and U = unmethylated probes." width="80%" />
<p class="caption">
Figure 3.18: Residual variation remaining after functional normalisation of the top 20,000 most variable probes with <em>m</em> PCs from the control probe summary matrices for the 450k array samples (n=60), for M = methylated and U = unmethylated probes.
</p>
</div>

</div>
<div id="ewass" class="section level3">
<h3><span class="header-section-number">3.4.4</span> EWASs</h3>
<div id="neonatal-bone-mineral-content" class="section level4">
<h4><span class="header-section-number">3.4.4.1</span> Neonatal Bone Mineral Content</h4>
<p>No probes fell below the Bonferroni corrected significance threshold for an association between DNAm at that locus and neonatal bone mineral content, Figure <a href="MAVIDOS.html#fig:MAVIDOSbmcEPICewasManhattanSVA">3.19</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSbmcEPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSbmcEPICewasManhattanSVA.png" alt="Results of EWAS for neonatal bone mineral content with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)" width="80%" />
<p class="caption">
Figure 3.19: Results of EWAS for neonatal bone mineral content with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>
</p>
</div>

</div>
<div id="intervention-placebo" class="section level4">
<h4><span class="header-section-number">3.4.4.2</span> Intervention / Placebo</h4>
<p>No probes fell below the Bonferroni corrected significance threshold for an association between DNAm at that locus and Intervention/placebo group status, Figure <a href="MAVIDOS.html#fig:MAVIDOSccEPICewasManhattanSVA">3.20</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSccEPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSccEPICewasManhattanSVA.png" alt="Results of EWAS for intervention/placebo group status with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)" width="80%" />
<p class="caption">
Figure 3.20: Results of EWAS for intervention/placebo group status with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>
</p>
</div>

</div>
<div id="maternal-vitamin-d-34wks" class="section level4">
<h4><span class="header-section-number">3.4.4.3</span> Maternal Vitamin D (34wks)</h4>
<p>Maternal Vitamin D levels remain substantially overlapping between intervention and placebo groups at 34wks, see Figure <a href="MAVIDOS.html#fig:MAVIDOSvitDviolinMonApr162018">3.21</a>.
Thus maternal vitamin D at 34wks may prove a more useful variable to model than intervention/placebo status.
No probes fell below the Bonferroni corrected significance threshold for an association between DNAm at that locus and maternal vitamin D levels at 34wks gestation, (Figure <a href="MAVIDOS.html#fig:MAVIDOSmVitD34EPICewasManhattanSVA">3.22</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSvitDviolinMonApr162018"></span>
<img src="figs/MAVIDOSvitDviolinMonApr162018.png" alt="Maternal Vitamin D levels at 11 and 34wks gestation, supplemtation with 1000 IU/day cholecalciferol began at week 14. Each participant is shown at both time points linked by a line to indicate the direction of change. The violin plots indicate the density of the distribution of vitamin D values at each time point with the \(25^{th}\), \(50^{th}\), \(75^{th}\) quantiles indicated with horizontal black lines. The colour indicates Intervention (Red) / Placebo (Blue) group" width="80%" />
<p class="caption">
Figure 3.21: Maternal Vitamin D levels at 11 and 34wks gestation, supplemtation with 1000 IU/day cholecalciferol began at week 14. Each participant is shown at both time points linked by a line to indicate the direction of change. The violin plots indicate the density of the distribution of vitamin D values at each time point with the <span class="math inline">\(25^{th}\)</span>, <span class="math inline">\(50^{th}\)</span>, <span class="math inline">\(75^{th}\)</span> quantiles indicated with horizontal black lines. The colour indicates Intervention (Red) / Placebo (Blue) group
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:MAVIDOSmVitD34EPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSmVitD34EPICewasManhattanSVA.png" alt="Results of EWAS for maternal vitamin D levels at 34wks gestation with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)" width="80%" />
<p class="caption">
Figure 3.22: Results of EWAS for maternal vitamin D levels at 34wks gestation with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>
</p>
</div>

</div>
<div id="change-in-maternal-vitamin-d" class="section level4">
<h4><span class="header-section-number">3.4.4.4</span> Change in Maternal Vitamin D</h4>
<p>No probes fell below the Bonferroni corrected significance threshold for an association between DNAm at that locus and change in maternal vitamin D from 11 to 34wks gestation, Figure <a href="MAVIDOS.html#fig:MAVIDOSdifVDEPICewasManhattanSVA">3.23</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:MAVIDOSdifVDEPICewasManhattanSVA"></span>
<img src="figs/MAVIDOSdifVDEPICewasManhattanSVA.png" alt="Results of EWAS for Change in maternal vitamin D levels from 11 to 34wks gestation with SVA model. Bidirectional Manhattan plot on which \(-log_{10}(p-value)\) is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with \(-log_{10}(p-value)\) such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of \(6.18\times10^{-8}~(0.05\div808,585)\)" width="80%" />
<p class="caption">
Figure 3.23: Results of EWAS for Change in maternal vitamin D levels from 11 to 34wks gestation with SVA model. Bidirectional Manhattan plot on which <span class="math inline">\(-log_{10}(p-value)\)</span> is plotted on the y axis and the sign of this value represents the direction of change. Size and transparency of points increases with <span class="math inline">\(-log_{10}(p-value)\)</span> such that the most significant probes are represented the largest and least transleucent points. x axis represent chromosomes and position thereupon. Red line indicates the significance threshold of <span class="math inline">\(6.18\times10^{-8}~(0.05\div808,585)\)</span>
</p>
</div>

</div>
</div>
<div id="keep-concordance-of-epic-and-850k-ewas-results" class="section level3">
<h3><span class="header-section-number">3.4.5</span> !!! keep? Concordance of EPIC and 850k EWAS results !!!</h3>
<p>In order to ascertain if the results from the EWAS in the 450k and EPIC arrays were producing similar sets of probes in the top ranking positions when ordered by p-value the concordance (% overlap) between the top k probes, where k = 1 ..
100,000 was calculated.
Only probes in common between the two arrays were used and k was incremented in steps of 50.</p>
<p>Concordance between the rankings of probes would suggest that the EWAS may be capturing a ‘real’ signal that is simply below the significance threshold with the statistical sensitivity/power that is available in this dataset.
The concordance between the EPIC and 450k datasets (Figures <a href="MAVIDOS.html#fig:EPIC450kEWASconcordMAVIDOSMonApr162018">3.24</a> &amp; <a href="MAVIDOS.html#fig:EPIC450kEWASconcordMAVIDOSMonApr162018zoom">3.25</a>) appears to be at roughly the level expected by chance.
This does not lend support to the possibility that there are associations between the variables of interest and DNAm that are beneath the current sensitivity of the study, it does not, however, rule this out.
In the absence of any probes above the significance threshold and with poor concordance between the 450k and EPIC array p-value rankings no further analyses such as gene set enrichment and differentially methylated region (DMR) calling have been carried out at this time.</p>
<div class="figure" style="text-align: center"><span id="fig:EPIC450kEWASconcordMAVIDOSMonApr162018"></span>
<img src="figs/EPIC450kEWASconcordMAVIDOSMonApr162018.png" alt="Concordance between the top 100,000 probes in common between the EWASs run on the EPIC (n=137) and 405k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Chance in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concorcance expected by chance (intersects 50% at 387,511, the number of shared probes)" width="80%" />
<p class="caption">
Figure 3.24: Concordance between the top 100,000 probes in common between the EWASs run on the EPIC (n=137) and 405k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Chance in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concorcance expected by chance (intersects 50% at 387,511, the number of shared probes)
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:EPIC450kEWASconcordMAVIDOSMonApr162018zoom"></span>
<img src="figs/EPIC450kEWASconcordMAVIDOSMonApr162018zoom.png" alt="Concordance between the top 10,000 probes in common between the EWASs run on the EPIC (n=137) and 405k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Chance in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concorcance expected by chance (intersects 50% at 387,511, the number of shared probes)" width="80%" />
<p class="caption">
Figure 3.25: Concordance between the top 10,000 probes in common between the EWASs run on the EPIC (n=137) and 405k (n=60) data sets. bmc = bone mineral content, cc = Intervention / Placebo, difVD = Chance in Vitamin D from 11 to 34wks, ExHighConc = Example of High Concordance generated using SVA vs iSVA results for the 450k intervention/placebo EWAS. Dotted line denotes concorcance expected by chance (intersects 50% at 387,511, the number of shared probes)
</p>
</div>

</div>
</div>
<div id="swsnew-mavidos-ewas" class="section level2">
<h2><span class="header-section-number">3.5</span> !!SWS/New MAVIDOS EWAS!!</h2>
<div id="section" class="section level3">
<h3><span class="header-section-number">3.5.1</span> </h3>
</div>
</div>
<div id="discussion-1" class="section level2">
<h2><span class="header-section-number">3.6</span> Discussion</h2>
<p>We have not been able to detect any changes in DNA methylation associated with, neonatal bone mineral content, maternal vitamin D at 34wks gestation, change in maternal vitamin D from 11 to 34wks gestation or intervention/placebo group membership.
However, at this juncture, we lack the necessary statistical power to rule out the presence of such changes, at effect sizes which may be biologically relevant.
EWAS have identified biologically relevant changes associated with environmental exposures in DNAm with magnitudes of less than a single percentage point, and percentage changes in the low single digits are not uncommon in EWAS <span class="citation">(Breton et al. <a href="#ref-Breton2017" role="doc-biblioref">2017</a>)</span>.
To achieve 80% power for the 450k array to detect moderate effect sizes (Cohen’s f2 = 0.15) requires &gt;300 samples, to do so with small effect sizes (Cohen’s f2 = 0.02) requires &gt;2500 samples, the additional multiple testing correction introduced when using the ~850k sites on the EPIC array increases these sample size requirements still further <span class="citation">(Cohen <a href="#ref-Cohen1988" role="doc-biblioref">1988</a>)</span>.</p>
<p>Previous studies have found relationships between bone outcomes in childhood and later life rather than in neonates, we may see an association between DNAm and bone outcomes at subsequent time points.</p>
<p>The EPIC array covers many regions not present on the 450k array, in particular, EPIC covers many additional enhancer regions.
The lack of concordance between the EPIC and 450k data applies only to those sites that are present on both arrays.
Thus, there may be results on the 850k array which are at present false negatives due to the lack of statistical power to detect them but which would have consistently highly ranked p-values were we able to compare them to an additional set of EPIC data.</p>
<div id="future-work" class="section level3">
<h3><span class="header-section-number">3.6.1</span> Future Work</h3>
<p>We are intending to repeat this analysis with additional samples to increase the power.
In addition, we may be getting data on the cord blood vitamin D levels corresponding to our samples, as the EWAS for maternal vitamin D levels at 34wks produced some of the lowest p-values we reason that a more direct measure of vitamin D levels for the sample in question may yield stronger results.
We intend to alter the model construction for subsequent analyses to bring the covariates included in line with those used in other analyses performed on MAVIDOS data, for example including gestational age.
We also intend to combine the EPIC and 450k array data and perform an analysis with all the available data to maximise the power.</p>
<p>At present we likely have insufficient power to perform EWAS on subsets, e.g. 
only births in a particular season as seasonal is know to impact Vitamin D levels <span class="citation">(Sachs et al. <a href="#ref-Sachs2013" role="doc-biblioref">2013</a>)</span>.
It may also be instructive to compare the concordance of results of the four EPIC array for which there are technical replicates to see if there is agreement between these replicates, despite the small size of this group.
It would also be interesting to check if any of the CpGs in <em>CDKN2A</em> identified as correlating with bone outcomes are present on the arrays and possibly to perform a power analysis to ascertain if we would have predicted that we would be able to replicate these findings with this data.</p>
<p>The effect of maternal smoking on cord blood methylation is a very strong and well established EWAS result <span class="citation">(Joubert et al. <a href="#ref-Joubert2016" role="doc-biblioref">2016</a>, @Breton2017)</span>.
We have maternal smoking data so it would be interesting to see if we can recapitulate the smoking EWAS result in cord tissue.</p>
<div id="never-look-at-gestational-age-__-dna-methylation-age-analyses" class="section level4">
<h4><span class="header-section-number">3.6.1.1</span> !!! never look at gestational age __ DNA Methylation Age analyses</h4>
<blockquote>
<p>Epigenetic clocks for gestational age have been developed <span class="citation">(Bohlin et al. <a href="#ref-Bohlin2016" role="doc-biblioref">2016</a>; Knight et al. <a href="#ref-Knight2016" role="doc-biblioref">2016</a>)</span> but their estimates did not correlate as well with gestational age in a replication set <span class="citation">(Andrew J. Simpkin, Suderman, and Howe <a href="#ref-Simpkin2017a" role="doc-biblioref">2017</a>)</span>.
This was potentially due to a bias towards pre-term infants in the initial training set which was quite small.
They were initially trained on cord blood and not cord tissue samples, we intend to assess their effectiveness at predicting gestational age in our cord tissue dataset.
DNAm age estimates at birth with Horvath’s clocks have minimal correlations with gestational age at delivery <span class="citation">(Simpkin et al. <a href="#ref-Simpkin2016" role="doc-biblioref">2016</a>; Andrew J Simpkin et al. <a href="#ref-Simpkin2017" role="doc-biblioref">2017</a>)</span>, we also intend to attempt to replicate this finding.
Once the PhenoAge calculator becomes available we also intend to assess if PhenoAge is similarly uncorrelated with gestational age <span class="citation">(Levine et al. <a href="#ref-Levine2018" role="doc-biblioref">2018</a>)</span>.
Another potentially interesting question is if there is any association with any of the epigenetic age estimates and bone outcomes.</p>
</blockquote>

</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Andrews2016">
<p>Andrews, Shan V, Christine Ladd-Acosta, Andrew P Feinberg, Kasper D Hansen, and M Daniele Fallin. 2016. “‘Gap hunting’ to characterize clustered probe signals in Illumina methylation array data.” <em>Epigenetics &amp; Chromatin</em> 9 (1): 56. <a href="https://doi.org/10.1186/s13072-016-0107-z">https://doi.org/10.1186/s13072-016-0107-z</a>.</p>
</div>
<div id="ref-Bakulski2016">
<p>Bakulski, Kelly M., Jason I. Feinberg, Shan V. Andrews, Jack Yang, Shannon Brown, Stephanie L. McKenney, Frank Witter, Jeremy Walston, Andrew P. Feinberg, and M. Daniele Fallin. 2016. “DNA methylation of cord blood cell types: Applications for mixed cell birth studies.” <em>Epigenetics</em> 11 (5): 354–62. <a href="https://doi.org/10.1080/15592294.2016.1161875">https://doi.org/10.1080/15592294.2016.1161875</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>Bell, Christopher G., Fei Gao, Wei Yuan, Leonie Roos, Richard J. Acton, Yudong Xia, Jordana Bell, et al. 2018. “Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci.” <em>Nature Communications</em> 9 (1): 8. <a href="https://doi.org/10.1038/s41467-017-01586-1">https://doi.org/10.1038/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Bohlin2016">
<p>Bohlin, J., S. E. Håberg, P. Magnus, S. E. Reese, H. K. Gjessing, M. C. Magnus, C. L. Parr, C. M. Page, S. J. London, and W. Nystad. 2016. “Prediction of gestational age based on genome-wide differentially methylated regions.” <em>Genome Biology</em> 17 (1): 207. <a href="https://doi.org/10.1186/s13059-016-1063-4">https://doi.org/10.1186/s13059-016-1063-4</a>.</p>
</div>
<div id="ref-Breton2017">
<p>Breton, Carrie V., Carmen J. Marsit, Elaine Faustman, Kari Nadeau, Jaclyn M. Goodrich, Dana C. Dolinoy, Julie Herbstman, et al. 2017. “Small-Magnitude Effect Sizes in Epigenetic End Points are Important in Children’s Environmental Health Studies: The Children’s Environmental Health and Disease Prevention Research Center’s Epigenetics Working Group.” <em>Environmental Health Perspectives</em> 125 (4): 511–26. <a href="https://doi.org/10.1289/EHP595">https://doi.org/10.1289/EHP595</a>.</p>
</div>
<div id="ref-Burdge2007">
<p>Burdge, Graham C., Mark A. Hanson, Jo L. Slater-Jefferies, and Karen A. Lillycrop. 2007. “Epigenetic regulation of transcription: a mechanism for inducing variations in phenotype (fetal programming) by differences in nutrition during early life?” <em>British Journal of Nutrition</em> 97 (6): 1036–46. <a href="https://doi.org/10.1017/S0007114507682920">https://doi.org/10.1017/S0007114507682920</a>.</p>
</div>
<div id="ref-Burdge2007a">
<p>Burdge, Graham C, Jo Slater-Jefferies, Christopher Torrens, Emma S Phillips, Mark A. Hanson, and Karen A. Lillycrop. 2007. “Dietary protein restriction of pregnant rats in the F 0 generation induces altered methylation of hepatic gene promoters in the adult male offspring in the F 1 and F 2 generations.” <em>British Journal of Nutrition</em> 97 (3): 435–39. <a href="https://doi.org/10.1017/S0007114507352392">https://doi.org/10.1017/S0007114507352392</a>.</p>
</div>
<div id="ref-Cardenas2016">
<p>Cardenas, Andres, Catherine Allard, Myriam Doyon, E Andres Houseman, Kelly M Bakulski, Patrice Perron, Luigi Bouchard, and Marie-France Hivert. 2016. “Validation of a DNA methylation reference panel for the estimation of nucleated cells types in cord blood.” <em>Epigenetics</em> 11 (11): 773–79. <a href="https://doi.org/10.1080/15592294.2016.1233091">https://doi.org/10.1080/15592294.2016.1233091</a>.</p>
</div>
<div id="ref-Cohen1988">
<p>Cohen, Jacob. 1988. <em>Statisticsl Power Analysis for the Behavioural Sciences</em>. 2nd ed. New York: Lawrence Erlbaum Associates.</p>
</div>
<div id="ref-Cooper2016">
<p>Cooper, Cyrus, Nicholas C. Harvey, Nicholas J. Bishop, Stephen Kennedy, Aris T. Papageorghiou, Inez Schoenmakers, Robert Fraser, et al. 2016. “Maternal gestational vitamin D supplementation and offspring bone health (MAVIDOS): a multicentre, double-blind, randomised placebo-controlled trial.” <em>The Lancet Diabetes &amp; Endocrinology</em> 4 (5): 393–402. <a href="https://doi.org/10.1016/S2213-8587(16)00044-9">https://doi.org/10.1016/S2213-8587(16)00044-9</a>.</p>
</div>
<div id="ref-Curtis2017">
<p>Curtis, Elizabeth M., Robert Murray, Philip Titcombe, Eloïse Cook, Rebecca Clarke-Harris, Paula Costello, Emma Garratt, et al. 2017. “Perinatal DNA Methylation at CDKN2A Is Associated With Offspring Bone Mass: Findings From the Southampton Women’s Survey.” <em>Journal of Bone and Mineral Research</em> 32 (10): 2030–40. <a href="https://doi.org/10.1002/jbmr.3153">https://doi.org/10.1002/jbmr.3153</a>.</p>
</div>
<div id="ref-Dogan2017">
<p>Dogan, Meeshanthini V., Steven R. H. Beach, and Robert A. Philibert. 2017. “Genetically contextual effects of smoking on genome wide DNA methylation.” <em>American Journal of Medical Genetics Part B: Neuropsychiatric Genetics</em> 174 (6): 595–607. <a href="https://doi.org/10.1002/ajmg.b.32565">https://doi.org/10.1002/ajmg.b.32565</a>.</p>
</div>
<div id="ref-Fortin2014">
<p>Fortin, Jean-Philippe, Aurélie Labbe, Mathieu Lemire, Brent W. Zanke, Thomas J. Hudson, Elana J. Fertig, Celia M. T. Greenwood, and Kasper D. Hansen. 2014. “Functional normalization of 450k methylation array data improves replication in large cancer studies.” <em>Genome Biology</em> 15 (11): 503. <a href="https://doi.org/10.1186/s13059-014-0503-2">https://doi.org/10.1186/s13059-014-0503-2</a>.</p>
</div>
<div id="ref-Gervin2016">
<p>Gervin, Kristina, Christian Magnus Page, Hans Christian D. Aass, Michelle A. Jansen, Heidi Elisabeth Fjeldstad, Bettina Kulle Andreassen, Liesbeth Duijts, et al. 2016. “Cell type specific DNA methylation in cord blood: A 450K-reference data set and cell count-based validation of estimated cell type composition.” <em>Epigenetics</em> 11 (9): 690–98. <a href="https://doi.org/10.1080/15592294.2016.1214782">https://doi.org/10.1080/15592294.2016.1214782</a>.</p>
</div>
<div id="ref-DeGoede2015">
<p>Goede, Olivia M. de, Hamid R. Razzaghian, E. Magda Price, Meaghan J. Jones, Michael S. Kobor, Wendy P. Robinson, and Pascal M. Lavoie. 2015. “Nucleated red blood cells impact DNA methylation and expression analyses of cord blood hematopoietic cells.” <em>Clinical Epigenetics</em> 7 (1): 95. <a href="https://doi.org/10.1186/s13148-015-0129-6">https://doi.org/10.1186/s13148-015-0129-6</a>.</p>
</div>
<div id="ref-Harvey2012a">
<p>Harvey, Nicholas C, Kassim Javaid, Nicholas Bishop, Stephen Kennedy, Aris T Papageorghiou, Robert Fraser, Saurabh V Gandhi, Inez Schoenmakers, Ann Prentice, and Cyrus Cooper. 2012. “MAVIDOS Maternal Vitamin D Osteoporosis Study: study protocol for a randomized controlled trial. The MAVIDOS Study Group.” <em>Trials</em> 13 (1): 13. <a href="https://doi.org/10.1186/1745-6215-13-13">https://doi.org/10.1186/1745-6215-13-13</a>.</p>
</div>
<div id="ref-Harvey2014d">
<p>Harvey, Nicholas C., Allan Sheppard, Keith M. Godfrey, Cameron McLean, Emma Garratt, Georgia Ntani, Lucy Davies, et al. 2014. “Childhood Bone Mineral Content Is Associated With Methylation Status of the RXRA Promoter at Birth.” <em>Journal of Bone and Mineral Research</em> 29 (3): 600–607. <a href="https://doi.org/10.1002/jbmr.2056">https://doi.org/10.1002/jbmr.2056</a>.</p>
</div>
<div id="ref-Holland2017">
<p>Holland, Michelle L, Robert Lowe, Paul W Caton, Carolina Gemma, Guillermo Carbajosa, Amy F Danson, Asha A M Carpenter, Elena Loche, Susan E Ozanne, and Vardhman K Rakyan. 2016. “Early-life nutrition modulates the epigenetic state of specific rDNA genetic variants in mice.” <em>Science</em> 353 (6298): 495–98. <a href="https://doi.org/10.1126/science.aaf7040">https://doi.org/10.1126/science.aaf7040</a>.</p>
</div>
<div id="ref-Houseman2012">
<p>Houseman, Eugene Andres, William P Accomando, Devin C Koestler, Brock C Christensen, Carmen J Marsit, Heather H Nelson, John K Wiencke, and Karl T Kelsey. 2012. “DNA methylation arrays as surrogate measures of cell mixture distribution.” <em>BMC Bioinformatics</em> 13 (1): 86. <a href="https://doi.org/10.1186/1471-2105-13-86">https://doi.org/10.1186/1471-2105-13-86</a>.</p>
</div>
<div id="ref-Jaffe2014">
<p>Jaffe, Andrew E, and Rafael A Irizarry. 2014. “Accounting for cellular heterogeneity is critical in epigenome-wide association studies.” <em>Genome Biology</em> 15 (2): R31. <a href="https://doi.org/10.1186/gb-2014-15-2-r31">https://doi.org/10.1186/gb-2014-15-2-r31</a>.</p>
</div>
<div id="ref-Joubert2016">
<p>Joubert, Bonnie R., Janine F. Felix, Paul Yousefi, Kelly M. Bakulski, Allan C. Just, Carrie Breton, Sarah E. Reese, et al. 2016. “DNA Methylation in Newborns and Maternal Smoking in Pregnancy: Genome-wide Consortium Meta-analysis.” <em>The American Journal of Human Genetics</em> 98 (4): 680–96. <a href="https://doi.org/10.1016/j.ajhg.2016.02.019">https://doi.org/10.1016/j.ajhg.2016.02.019</a>.</p>
</div>
<div id="ref-Kip2004">
<p>Kip, Sertac N, and Emanuel E Strehler. 2004. “Vitamin D 3 upregulates plasma membrane Ca 2+ -ATPase expression and potentiates apico-basal Ca 2+ flux in MDCK cells.” <em>American Journal of Physiology-Renal Physiology</em> 286 (2): F363–F369. <a href="https://doi.org/10.1152/ajprenal.00076.2003">https://doi.org/10.1152/ajprenal.00076.2003</a>.</p>
</div>
<div id="ref-Knight2016">
<p>Knight, Anna K., Jeffrey M. Craig, Christiane Theda, Marie Bækvad-Hansen, Jonas Bybjerg-Grauholm, Christine S. Hansen, Mads V. Hollegaard, et al. 2016. “An epigenetic clock for gestational age at birth based on blood methylation data.” <em>Genome Biology</em> 17 (1): 206. <a href="https://doi.org/10.1186/s13059-016-1068-z">https://doi.org/10.1186/s13059-016-1068-z</a>.</p>
</div>
<div id="ref-Leek2007">
<p>Leek, Jeffrey T., and John D. Storey. 2007. “Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis.” <em>PLoS Genetics</em> 3 (9): e161. <a href="https://doi.org/10.1371/journal.pgen.0030161">https://doi.org/10.1371/journal.pgen.0030161</a>.</p>
</div>
<div id="ref-Levine2018">
<p>Levine, Morgan E, Ake T Lu, Austin Quach, Brian H Chen, Themistocles L Assimes, Stefania Bandinelli, Lifang Hou, et al. 2018. “An epigenetic biomarker of aging for lifespan and healthspan.” <em>Aging</em> 10 (4): 573–91. <a href="https://doi.org/10.18632/aging.101414">https://doi.org/10.18632/aging.101414</a>.</p>
</div>
<div id="ref-Lillycrop2008">
<p>Lillycrop, Karen A., Emma S. Phillips, Christopher Torrens, Mark A. Hanson, Alan A. Jackson, and Graham C. Burdge. 2008. “Feeding pregnant rats a protein-restricted diet persistently alters the methylation of specific cytosines in the hepatic PPAR<span class="math inline">\(\alpha\)</span> promoter of the offspring.” <em>British Journal of Nutrition</em> 100 (2): 278–82. <a href="https://doi.org/10.1017/S0007114507894438">https://doi.org/10.1017/S0007114507894438</a>.</p>
</div>
<div id="ref-Lin2018">
<p>Lin, Xinyi, Jane Yi Lin Tan, Ai Ling Teh, Ives Yubin Lim, Samantha J. Liew, Julia L. MacIsaac, Yap Seng Chong, et al. 2018. “Cell type-specific DNA methylation in neonatal cord tissue and cord blood: a 850K-reference panel and comparison of cell types.” <em>Epigenetics</em> 13 (9): 941–58. <a href="https://doi.org/10.1080/15592294.2018.1522929">https://doi.org/10.1080/15592294.2018.1522929</a>.</p>
</div>
<div id="ref-Mahon2010a">
<p>Mahon, Pamela, Nicholas Harvey, Sarah Crozier, Hazel Inskip, Sian Robinson, Nigel Arden, Rama Swaminathan, Cyrus Cooper, and Keith Godfrey. 2010. “Low maternal vitamin D status and fetal bone development: Cohort study.” <em>Journal of Bone and Mineral Research</em> 25 (1): 14–19. <a href="https://doi.org/10.1359/jbmr.090701">https://doi.org/10.1359/jbmr.090701</a>.</p>
</div>
<div id="ref-McGregor2016">
<p>McGregor, Kevin, Sasha Bernatsky, Ines Colmegna, Marie Hudson, Tomi Pastinen, Aurélie Labbe, and Celia M. T. Greenwood. 2016. “An evaluation of methods correcting for cell-type heterogeneity in DNA methylation studies.” <em>Genome Biology</em> 17 (1): 84. <a href="https://doi.org/10.1186/s13059-016-0935-y">https://doi.org/10.1186/s13059-016-0935-y</a>.</p>
</div>
<div id="ref-Min2017">
<p>Min, Josine, Gibran Hemani, George Davey Smith, Caroline L Relton, and Matthew Suderman. 2017. “Meffil: efficient normalisation and analysis of very large DNA methylation samples.” <em>bioRxiv</em> 44 (0): 125963. <a href="https://doi.org/10.1101/125963">https://doi.org/10.1101/125963</a>.</p>
</div>
<div id="ref-Pidsley2016">
<p>Pidsley, Ruth, Elena Zotenko, Timothy J. Peters, Mitchell G. Lawrence, Gail P. Risbridger, Peter Molloy, Susan Van Djik, Beverly Muhlhausler, Clare Stirzaker, and Susan J. Clark. 2016. “Critical evaluation of the Illumina MethylationEPIC BeadChip microarray for whole-genome DNA methylation profiling.” <em>Genome Biology</em> 17 (1): 208. <a href="https://doi.org/10.1186/s13059-016-1066-1">https://doi.org/10.1186/s13059-016-1066-1</a>.</p>
</div>
<div id="ref-Sachs2013">
<p>Sachs, Michael C, Abigail Shoben, Gregory P Levin, Cassy Robinson-Cohen, Andrew N Hoofnagle, Nancy Swords-Jenny, Joachim H Ix, et al. 2013. “Estimating mean annual 25-hydroxyvitamin D concentrations from single measurements: the Multi-Ethnic Study of Atherosclerosis.” <em>The American Journal of Clinical Nutrition</em> 97 (6): 1243–51. <a href="https://doi.org/10.3945/ajcn.112.054502">https://doi.org/10.3945/ajcn.112.054502</a>.</p>
</div>
<div id="ref-Simpkin2016">
<p>Simpkin, Andrew J., Gibran Hemani, Matthew Suderman, Tom R. Gaunt, Oliver Lyttleton, Wendy L. Mcardle, Susan M. Ring, et al. 2016. “Prenatal and early life influences on epigenetic age in children: a study of mother–offspring pairs from two cohort studies.” <em>Human Molecular Genetics</em> 25 (1): 191–201. <a href="https://doi.org/10.1093/hmg/ddv456">https://doi.org/10.1093/hmg/ddv456</a>.</p>
</div>
<div id="ref-Simpkin2017">
<p>Simpkin, Andrew J, Laura D Howe, Kate Tilling, Tom R Gaunt, Oliver Lyttleton, Wendy L McArdle, Susan M Ring, Steve Horvath, George Davey Smith, and Caroline L Relton. 2017. “The epigenetic clock and physical development during childhood and adolescence: longitudinal analysis from a UK birth cohort.” <em>International Journal of Epidemiology</em> 46 (2): dyw307. <a href="https://doi.org/10.1093/ije/dyw307">https://doi.org/10.1093/ije/dyw307</a>.</p>
</div>
<div id="ref-Simpkin2017a">
<p>Simpkin, Andrew J., Matthew Suderman, and Laura D. Howe. 2017. “Epigenetic clocks for gestational age: statistical and study design considerations.” <em>Clinical Epigenetics</em> 9 (1): 100. <a href="https://doi.org/10.1186/s13148-017-0402-y">https://doi.org/10.1186/s13148-017-0402-y</a>.</p>
</div>
<div id="ref-Teschendorff2017">
<p>Teschendorff, Andrew E, and Shijie C Zheng. 2017. “Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.” <em>Epigenomics</em> 9 (5): 757–68. <a href="https://doi.org/10.2217/epi-2016-0153">https://doi.org/10.2217/epi-2016-0153</a>.</p>
</div>
<div id="ref-Teschendorff2011">
<p>Teschendorff, Andrew E., Joanna Zhuang, and Martin Widschwendter. 2011. “Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies.” <em>Bioinformatics</em> 27 (11): 1496–1505. <a href="https://doi.org/10.1093/bioinformatics/btr171">https://doi.org/10.1093/bioinformatics/btr171</a>.</p>
</div>
<div id="ref-Viljakainen2011">
<p>Viljakainen, H. T., T. Korhonen, T. Hytinantti, E. K. A. Laitinen, S. Andersson, O. Mäkitie, and C. Lamberg-Allardt. 2011. “Maternal vitamin D status affects bone growth in early childhood—a prospective cohort study.” <em>Osteoporosis International</em> 22 (3): 883–91. <a href="https://doi.org/10.1007/s00198-010-1499-4">https://doi.org/10.1007/s00198-010-1499-4</a>.</p>
</div>
<div id="ref-Viljakainen2010">
<p>Viljakainen, H. T., E. Saarnio, T. Hytinantti, M. Miettinen, H. Surcel, O. Mäkitie, S. Andersson, K. Laitinen, and C. Lamberg-Allardt. 2010. “Maternal Vitamin D Status Determines Bone Variables in the Newborn.” <em>The Journal of Clinical Endocrinology &amp; Metabolism</em> 95 (4): 1749–57. <a href="https://doi.org/10.1210/jc.2009-1391">https://doi.org/10.1210/jc.2009-1391</a>.</p>
</div>
<div id="ref-Wahl2016">
<p>Wahl, Simone, Alexander Drong, Benjamin Lehne, Marie Loh, William R Scott, Sonja Kunze, Pei-Chien Tsai, et al. 2017. “Epigenome-wide association study of body mass index, and the adverse outcomes of adiposity.” <em>Nature</em> 541 (7635): 81–86. <a href="https://doi.org/10.1038/nature20784">https://doi.org/10.1038/nature20784</a>.</p>
</div>
<div id="ref-Zhou2017">
<p>Zhou, Wanding, Peter W. Laird, and Hui Shen. 2016. “Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes.” <em>Nucleic Acids Research</em> 45 (4): gkw967. <a href="https://doi.org/10.1093/nar/gkw967">https://doi.org/10.1093/nar/gkw967</a>.</p>
</div>
<div id="ref-Zhu2014a">
<p>Zhu, Kun, Andrew JO Whitehouse, Prue H Hart, Merci Kusel, Jenny Mountain, Stephen Lye, Craig Pennell, and John P Walsh. 2014. “Maternal Vitamin D Status During Pregnancy and Bone Mass in Offspring at 20 Years of Age: A Prospective Cohort Study.” <em>Journal of Bone and Mineral Research</em> 29 (5): 1088–95. <a href="https://doi.org/10.1002/jbmr.2138">https://doi.org/10.1002/jbmr.2138</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="methods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="tRNApaper.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/03-EWAS.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
