<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Methods | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.17.1 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Methods | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Methods | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J.Acton" />


<meta name="date" content="2020-02-28" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="the-ageing-poplation-and-burden-of-disease.html"/>
<link rel="next" href="MAVIDOS.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> The Epigenomics of Human Ageing</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#aims-context"><i class="fa fa-check"></i><b>1.1</b> Aims &amp; Context</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html"><i class="fa fa-check"></i><b>2</b> The Ageing Poplation and Burden of Disease</a><ul>
<li class="chapter" data-level="2.1" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#epigenomics---overview"><i class="fa fa-check"></i><b>2.1</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="2.2" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#dna-methylation"><i class="fa fa-check"></i><b>2.2</b> DNA Methylation</a><ul>
<li class="chapter" data-level="2.2.1" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#structure-and-context"><i class="fa fa-check"></i><b>2.2.1</b> Structure and Context</a></li>
<li class="chapter" data-level="2.2.2" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>2.2.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="2.2.3" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>2.2.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="2.2.4" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#maintenence-and-fidelity"><i class="fa fa-check"></i><b>2.2.4</b> Maintenence and Fidelity</a></li>
<li class="chapter" data-level="2.2.5" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#dna-methylation-assays"><i class="fa fa-check"></i><b>2.2.5</b> === DNA methylation assays</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#ageing"><i class="fa fa-check"></i><b>2.3</b> Ageing</a><ul>
<li class="chapter" data-level="2.3.1" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#hallmarks"><i class="fa fa-check"></i><b>2.3.1</b> Hallmarks</a></li>
<li class="chapter" data-level="2.3.2" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>2.3.2</b> DNA Methylation and Ageing</a></li>
<li class="chapter" data-level="2.3.3" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#DNAmAgeReview"><i class="fa fa-check"></i><b>2.3.3</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="2.3.4" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>2.3.4</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="2.3.5" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>2.3.5</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="2.3.6" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>2.3.6</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
<li class="chapter" data-level="2.3.7" data-path="the-ageing-poplation-and-burden-of-disease.html"><a href="the-ageing-poplation-and-burden-of-disease.html#conclusions"><i class="fa fa-check"></i><b>2.3.7</b> Conclusions</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II Methods</b></span></li>
<li class="chapter" data-level="3" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>3</b> Methods</a><ul>
<li class="chapter" data-level="3.1" data-path="methods.html"><a href="methods.html#dna-methylation-assays-1"><i class="fa fa-check"></i><b>3.1</b> DNA methylation assays</a><ul>
<li class="chapter" data-level="3.1.1" data-path="methods.html"><a href="methods.html#overview"><i class="fa fa-check"></i><b>3.1.1</b> Overview</a></li>
<li class="chapter" data-level="3.1.2" data-path="methods.html"><a href="methods.html#methods-used-in-this-work"><i class="fa fa-check"></i><b>3.1.2</b> Methods used in this work</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="methods.html"><a href="methods.html#discussion"><i class="fa fa-check"></i><b>3.2</b> Discussion</a></li>
</ul></li>
<li class="part"><span><b>III Results 1</b></span></li>
<li class="chapter" data-level="4" data-path="MAVIDOS.html"><a href="MAVIDOS.html"><i class="fa fa-check"></i><b>4</b> Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="4.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#introduction"><i class="fa fa-check"></i><b>4.2</b> Introduction</a><ul>
<li class="chapter" data-level="4.2.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#background"><i class="fa fa-check"></i><b>4.2.1</b> Background</a></li>
<li class="chapter" data-level="4.2.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>4.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#methods-1"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation"><i class="fa fa-check"></i><b>4.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="4.3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-models"><i class="fa fa-check"></i><b>4.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="4.3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>4.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#results"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#summary-of-results"><i class="fa fa-check"></i><b>4.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="4.4.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#quality-control-qc"><i class="fa fa-check"></i><b>4.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="4.4.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation-1"><i class="fa fa-check"></i><b>4.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="4.4.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewass"><i class="fa fa-check"></i><b>4.4.4</b> EWASs</a></li>
<li class="chapter" data-level="4.4.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>4.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>4.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="4.5.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#section"><i class="fa fa-check"></i><b>4.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="MAVIDOS.html"><a href="MAVIDOS.html#discussion-1"><i class="fa fa-check"></i><b>4.6</b> Discussion</a><ul>
<li class="chapter" data-level="4.6.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#future-work"><i class="fa fa-check"></i><b>4.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>IV Results 2</b></span></li>
<li class="chapter" data-level="5" data-path="tRNApaper.html"><a href="tRNApaper.html"><i class="fa fa-check"></i><b>5</b> The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="5.1" data-path="tRNApaper.html"><a href="tRNApaper.html#abstract-2"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="tRNApaper.html"><a href="tRNApaper.html#introduction-1"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="tRNApaper.html"><a href="tRNApaper.html#Methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="tRNApaper.html"><a href="tRNApaper.html#participants"><i class="fa fa-check"></i><b>5.3.1</b> Participants</a></li>
<li class="chapter" data-level="5.3.2" data-path="tRNApaper.html"><a href="tRNApaper.html#tRNAanno"><i class="fa fa-check"></i><b>5.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="5.3.3" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylome-data"><i class="fa fa-check"></i><b>5.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="5.3.4" data-path="tRNApaper.html"><a href="tRNApaper.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>5.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="5.3.5" data-path="tRNApaper.html"><a href="tRNApaper.html#sortedbloodmethods"><i class="fa fa-check"></i><b>5.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="5.3.6" data-path="tRNApaper.html"><a href="tRNApaper.html#TCGAmethods"><i class="fa fa-check"></i><b>5.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="5.3.7" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmapmethods"><i class="fa fa-check"></i><b>5.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="5.3.8" data-path="tRNApaper.html"><a href="tRNApaper.html#mousemethods"><i class="fa fa-check"></i><b>5.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="5.3.9" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-figures"><i class="fa fa-check"></i><b>5.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="5.3.10" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>5.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="tRNApaper.html"><a href="tRNApaper.html#res"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>5.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="5.4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>5.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="5.4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>5.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="5.4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>5.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="5.4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>5.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="tRNApaper.html"><a href="tRNApaper.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="tRNApaper.html"><a href="tRNApaper.html#data-availability"><i class="fa fa-check"></i><b>5.6</b> Data availability</a></li>
<li class="chapter" data-level="5.7" data-path="tRNApaper.html"><a href="tRNApaper.html#code-availablity"><i class="fa fa-check"></i><b>5.7</b> Code availablity</a></li>
<li class="chapter" data-level="5.8" data-path="tRNApaper.html"><a href="tRNApaper.html#acknowledgements-1"><i class="fa fa-check"></i><b>5.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="5.9" data-path="tRNApaper.html"><a href="tRNApaper.html#author-contributions"><i class="fa fa-check"></i><b>5.9</b> Author Contributions</a></li>
<li class="chapter" data-level="5.10" data-path="tRNApaper.html"><a href="tRNApaper.html#references"><i class="fa fa-check"></i><b>5.10</b> References</a></li>
</ul></li>
<li class="part"><span><b>V Results 3</b></span></li>
<li class="chapter" data-level="6" data-path="abstract-3.html"><a href="abstract-3.html"><i class="fa fa-check"></i><b>6</b> Abstract</a></li>
<li class="chapter" data-level="7" data-path="AluIntro.html"><a href="AluIntro.html"><i class="fa fa-check"></i><b>7</b> Introduction</a></li>
<li class="chapter" data-level="8" data-path="methods-2.html"><a href="methods-2.html"><i class="fa fa-check"></i><b>8</b> Methods</a></li>
<li class="chapter" data-level="9" data-path="results-1.html"><a href="results-1.html"><i class="fa fa-check"></i><b>9</b> Results</a><ul>
<li class="chapter" data-level="9.1" data-path="results-1.html"><a href="results-1.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>9.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="9.2" data-path="results-1.html"><a href="results-1.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>9.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="9.3" data-path="results-1.html"><a href="results-1.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>9.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="9.4" data-path="results-1.html"><a href="results-1.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>9.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>10</b> Discussion</a></li>
<li class="chapter" data-level="11" data-path="methods-3.html"><a href="methods-3.html"><i class="fa fa-check"></i><b>11</b> Methods</a><ul>
<li class="chapter" data-level="11.1" data-path="methods-3.html"><a href="methods-3.html#participants-1"><i class="fa fa-check"></i><b>11.1</b> Participants</a></li>
<li class="chapter" data-level="11.2" data-path="methods-3.html"><a href="methods-3.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>11.2</b> Age-related DNA methylation changes</a></li>
<li class="chapter" data-level="11.3" data-path="methods-3.html"><a href="methods-3.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>11.3</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="11.4" data-path="methods-3.html"><a href="methods-3.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>11.4</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="11.5" data-path="methods-3.html"><a href="methods-3.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>11.5</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="11.6" data-path="methods-3.html"><a href="methods-3.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>11.6</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="11.7" data-path="methods-3.html"><a href="methods-3.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>11.7</b> Association of these variants with Age-related Disease</a><ul>
<li class="chapter" data-level="11.7.1" data-path="methods-3.html"><a href="methods-3.html#data-availability-1"><i class="fa fa-check"></i><b>11.7.1</b> Data availability</a></li>
<li class="chapter" data-level="11.7.2" data-path="methods-3.html"><a href="methods-3.html#code-availablity-1"><i class="fa fa-check"></i><b>11.7.2</b> Code availablity</a></li>
<li class="chapter" data-level="11.7.3" data-path="methods-3.html"><a href="methods-3.html#acknowledgements-2"><i class="fa fa-check"></i><b>11.7.3</b> Acknowledgements</a></li>
<li class="chapter" data-level="11.7.4" data-path="methods-3.html"><a href="methods-3.html#author-contributions-1"><i class="fa fa-check"></i><b>11.7.4</b> Author Contributions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="final-words.html"><a href="final-words.html"><i class="fa fa-check"></i><b>12</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methods" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Methods</h1>
<div id="dna-methylation-assays-1" class="section level2">
<h2><span class="header-section-number">3.1</span> DNA methylation assays</h2>
<div id="overview" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Overview</h3>
<p>There are numerous DNA methylation assays. Some are designed to assay global DNAm levels producing a single global measurement of the amount of 5mC present in a sample, others assay DNAm in a manner traceble to a specific genomic locus. These methods can be further subdivided into targeted methods where the loci to be examined are known in advance, and those where the loci covered are quasi-randomly sampled. Both the targeted and untargeted methods vary in their granularity, from individual CpGs to large regions of the genome, on the order of megabases. These assays also vary in their coverage of the genome from locus-specific approaches looking at small numbers of individual loci to whole genome methods providing methylation information on all ~28 million CpG sites. Between these two extremes are ‘epigenome-wide’ methods which are focused with varying degrees of specificity on genomic regions of interest.</p>
<p>These DNA methylation assays generally exploit one of four following things:</p>
<ol style="list-style-type: decimal">
<li><p><strong>Methylation sensitive restriction digestion</strong>
A variety of methods exploit enzymes which differentially cut methylated and unmethylated DNA. An example of such enzymes is: HpaII and MspI which are isoschizomers for the sequence: 5’-C^CGG-3’ but only HpaII methylation sensitive and unable to cut the sequence if the internal cytosine is methylated. The methods which use such enzymes include: Restriction landmark genomic scanning, which gives a roughly megabase resolution indicators of DNAm levels by 2D electrophoresis <span class="citation">[<a href="#ref-Hatada1991" role="doc-biblioref">135</a>]</span>. MRE-seq, which makes use of methylation-sensitive restriction digests to enrich for unmethylated DNA <span class="citation">[<a href="#ref-Maunakea2010" role="doc-biblioref">48</a>]</span>. This is then is sequenced to provide the genomic location of this unmethylated signal. Reduced Representation Bisulfite Sequencing (RRBS) which using methylation senstitive restriction digestion to enrich for unmethylated sequences which are then subject to bisilfite sequencing, (see point 3 below).</p></li>
<li><p><strong>DNA methylation-sensitive binding of DNA by antibodies or other proteins which bind methylated DNA</strong>
Anti-5mC antibodies were first used to isolate methylated DNA in 1985 and were subsequently paired with array-based technologies to assay DNAm levels at specific loci in 2005 <span class="citation">[<a href="#ref-Weber2005" role="doc-biblioref">136</a>]</span>. This was followed by MeDIP-seq (methylated DNA immuno-precipitation and sequencing) <span class="citation">[<a href="#ref-Down2009" role="doc-biblioref">138</a>]</span>, which enriches methylated DNA that is then sequenced and the number of reads mapping to a locus is indicative of the relative methylation level. MBD-seq uses a recombinant Methyl Binding Domain (MBD) protein to enrich for methylated double-stranded DNA, prior to sequencing <span class="citation">[<a href="#ref-Serre2009a" role="doc-biblioref">140</a>]</span>. Inferring the absolute methylation levels form the number of reads mapping to a locus enriched by one of these pull-down methods is complicated by variation in CpG density in the genome and the fact that CpG dense regions tend to have low methylation levels, and thus tend not be pulled down by antibodies binding 5mC or MBD proteins. The best results for estimating absolute methylation levels with these methods come from pairing them with a complementary method to enrich for unmethylated sites such as MRE-seq (described above) <span class="citation">[<a href="#ref-Li2015" role="doc-biblioref">141</a>]</span>. Though relative methylation levels remain effective in identifying differentially methylated regions when using these methods alone <span class="citation">[<a href="#ref-Bock2010" role="doc-biblioref">142</a>]</span>.</p></li>
<li><p><strong>Sodium bisulfite conversion of unmethylated cytosines to uracil</strong>
Conversion of cytosine to uracil changes the base complementary to this site from G to A <span class="citation">[<a href="#ref-Hayatsu1970" role="doc-biblioref">143</a>]</span>. This basic chemistry of the conversion process is illustrated in Figure <a href="methods.html#fig:BisulfiteConversionNEB">3.1</a>. This conversion can be detected with a variety of other technologies from targeted bisulfite PCR to examine a small number of selected loci, to the Illumina bead chip methylation arrays or WGBS. Targeted bisilfite PCR and be preformed on a larger number of loci with microfuidic multiplexing such as that provided by the Fluidigm access array <span class="citation">[<a href="#ref-Adamowicz2018" role="doc-biblioref">144</a>]</span>. It is worth noting that bisulfite conversion cannot differentiate between methylated and hydroxymethylated cytosines. Because 5hmC represents a small fraction of modified bases compared to 5mC many analyses have made the working assumption that unconverted bases are methylated, but it is beginning to be recognised that distinguishing between the two may be biologicall important especially in tussues where 5hmC is more prevalant such as neurons. A variant of bisulfite conversion, oxidative bisulfite conversion, exists which can permit 5hmC to be distinguished from 5mC <span class="citation">[<a href="#ref-Skvortsova2017" role="doc-biblioref">145</a>]</span>.</p></li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:BisulfiteConversionNEB"></span>
<img src="figs/BisulfiteConversionNEB.jpg" alt="Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs)." width="80%" />
<p class="caption">
Figure 3.1: Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs).
</p>
</div>

<ol start="4" style="list-style-type: decimal">
<li><strong>Physical differences in the methylated base</strong>
These are exploited by the not yet widely used ‘3rd generation’ sequencing technologies such as nanopore and single-molecule real-time sequencing. These methods respectively exploit the effects of modified bases on ion flow through the nanopore and impact on incorporation time of new bases whilst in the polymerase <span class="citation">[<a href="#ref-Rhoads2015" role="doc-biblioref">146</a>]</span>.</li>
</ol>
</div>
<div id="methods-used-in-this-work" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Methods used in this work</h3>
<div id="MeDIPseq" class="section level4">
<h4><span class="header-section-number">3.1.2.1</span> MeDIP-seq</h4>
<p>MeDIP-seq uses a monoclonal anti-5mC antibody to bind denatured fragmented genomic DNA at methylated CpG sites, this antibody-bound fraction of DNA is isolated and sequenced <span class="citation">[<a href="#ref-Down2009" role="doc-biblioref">138</a>]</span>, Figure <a href="methods.html#fig:MeDIPseqReaction">3.2</a> outlines the workflow. Unlike bisulfite conversion approaches this method permits 5mC to be differentiated from 5hmC. The methylation level across the CpG sites in the region of the genome to which the resultant sequencing reads map can subsequently be estimated by counting the number of reads and accounting for the CpG density with software tools such as the MEDIPS R package <span class="citation">[<a href="#ref-Lienhard2014" role="doc-biblioref">139</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:MeDIPseqReaction"></span>
<img src="figs/MeDIPseqReaction.png" alt="Graphical Summary of the MeDIP-seq process." width="100%" />
<p class="caption">
Figure 3.2: Graphical Summary of the MeDIP-seq process.
</p>
</div>

</div>
<div id="bisulfite-conversion-methods" class="section level4">
<h4><span class="header-section-number">3.1.2.2</span> Bisulfite Conversion methods</h4>
<div id="illumina-methylation-arrays" class="section level5">
<h5><span class="header-section-number">3.1.2.2.1</span> Illumina Methylation arrays</h5>
<p>The Illumina methylation bead chip arrays make use of 50 bp long probes with sequences corresponding to the loci of interest in the genome. These probes are attached by their 5’ ends to beads which are stuck to the surface of a glass slide, several beads represent each locus at a single spot on the slide. The positions of the bead spots are known and used to identify the probes. The CpG of interest is located at the 3’ end of the probes. Samples are prepared for the array assay by fragmentation, bisulfite conversion and amplification. These sample fragments anneal to complementary probes on the array and unbound DNA is washed away. A single DNA polymerase-mediated elongation step is carried out to permit a fluorescently labelled base to be incorporated at the end of each probe. C and G are tagged with one fluorophore and A and T with another. There are two probe designs one which makes use of two distinct bead types for each locus, the type I probes, and one which makes use of a single type of bead at each locus, the type II. A scanner excites the two fluorophore channels with a laser and their respective fluorescence intensities at each spot are recorded with an imaging system. The ‘colour’ of the fluorophore incorporated signifies the methylation state and the intensity of the proportion of sites which are in that state. The logic of which ‘colour’ corresponds to methylated or unmethylated varies with the probe type and Figure <a href="methods.html#fig:Bibikova2011IlluminaMethylationBeadChip">3.3</a> illustrates and explains this in depth.</p>
<p>These arrays have now undergone several iterations <span class="citation">[<a href="#ref-Bibikova2009" role="doc-biblioref">95</a>]</span>. The current array probe design was preceded by the ‘golden gate’ array which used a variant on the SNP probe design with methylation-specific PCR, This array contained ~1.5k probes covering 371 genes with a strong focus of cancer-related genes <span class="citation">[<a href="#ref-Bibikova2006" role="doc-biblioref">90</a>]</span>. The first array using the probe design outlined above was the ‘Infinium’ array which contained 27k type I probes focused on the promoter regions for 14,475 consensus coding genes with 110 miRNA promoters <span class="citation">[<a href="#ref-Bibikova2009" role="doc-biblioref">95</a>]</span>. The 450k array which had ~480k mixed type I and II probes covering 21,231 RefSeq genes, 26,658 CpG islands, 80,538 predicted enhancer regions and a variety of other features, including the MHC region <span class="citation">[<a href="#ref-Bibikova2011" role="doc-biblioref">101</a>]</span>. The ‘EPIC’ array has ~850k probes including &gt;90% of those on the 450k with greatly expanded coverage of loci with more dynamic methylation states than promoters and CGIs which tend to have relatively low methylation variability aiming to capture more functional methylation variation at regions such as enhancers <span class="citation">[<a href="#ref-Moran2015" role="doc-biblioref">148</a>]</span>. The ‘EPIC’ array also contains both Type I and Type II probes, with many of the new sites being type II. The type I probes have greater dynamic range than the type II but take up twice as much space on the array so there is some trade-off between maximising the number of sites covered and the quality of data at those sites.</p>
<div class="figure" style="text-align: center"><span id="fig:Bibikova2011IlluminaMethylationBeadChip"></span>
<img src="figs/Bibikova2011IlluminaMethylationBeadChip.png" alt="(ref:Bibikova2011IlluminaMethylationBeadChip-Cap)" width="90%" />
<p class="caption">
Figure 3.3: (ref:Bibikova2011IlluminaMethylationBeadChip-Cap)
</p>
</div>
<p>(ref:Bibikova2011IlluminaMethylationBeadChip-Cap) <strong>Illumina Methylation Bead Chip Array Probe Designs</strong>
The probes on the array are 50bp in length. In the type I probes unlike the type II probes the base that is incorporated (or not) in the elongation step is the base that <strong>follows</strong> the CpG site. The probes on the array are bound by the amplification products derived from the bisulfite converted DNA not direct conversion products. Thus, Cs in the sample DNA represent methylated positions and Ts unmethylated positions, as Ts take the place of Us create by bisulfite conversion during the aplification process.
<strong>Type I</strong>: Panel A shows the type I probe design. Each locus is represented on two beads, a methylated and an unmethylated bead. <em>Methylated Sample</em>: When sample DNA binds a methylated probe the G at the second position in the CpG on the methylated probe will be complementary to the C marking the location of the 5mC. Extension will then be able to occur incorporating a fluorescently labeled base complementary to the base just 5’ of the 5mC position in the probe. On the unmethylated probe the C marking the position of the 5mC will not be complementary to the to the A base in the probe and thus extension cannot occur. No fluorescent base will be incorporated at the unmethylated probe. <em>Unmethylated Sample</em>: When sample DNA binds the methylated probe the T marking the position of the unmethylated C is not complementary to the G at the second position in the CpG site on the probe. Thus, no fluorescent base is incorporated. Whereas, on the unmethylated probe the T marking the position of the unmethylated C will be complementary to the A at the second position in the CpG site on the probe. Thus, a fluorescently labelled base can be incorporated. <strong>Type II</strong>: Panel B shows the Type II probe design. For the Type II probes only one probe represents each locus. In the Type II probes the incorporated base is at the second position in the CpG site. A fluorescently labelled A is always incorporated opposite a T in the sample DNA marking the position of an unmethylated C, and a fluorescently labelled G is always incorporated opposite a C marking the position of an unconverted 5mC in the sample DNA. For the Type II probes, in contrast to the Type I probes, methylation is always signalled in the green channel and unmethylation in the red channel. (figure reproduced from <span class="citation">[<a href="#ref-Bibikova2011" role="doc-biblioref">101</a>]</span>.)</p>
<p>Methylation at each site on the array is commonly reported as a <span class="math inline">\(\beta\)</span> value, which correspond to the proportion of the sample DNA which was methylated at that site.</p>
<p><span class="math display">\[\frac{\beta=intensity[M]}{intensity[U] + intensity[M]}\]</span></p>
<p>Where <span class="math inline">\(M\)</span> = methylated and <span class="math inline">\(U\)</span> = unmethylated.</p>
</div>
<div id="sequencing-methods" class="section level5">
<h5><span class="header-section-number">3.1.2.2.2</span> Sequencing Methods</h5>
<div id="whole-genome" class="section level6">
<h6><span class="header-section-number">3.1.2.2.2.1</span> Whole Genome</h6>
<p>Whole Genome Bisulfite Sequencing (WGBS) theoretically permits all ~28 million CpGs in the genome to have their methylation state assayed (excluding unmappable and other problematic regions) and thus represents a gold standard for characterising the methylome. However, WGBS is highly inefficient with 70-80% of the reads generated providing no information about methylation as many reads map to loci without CpG sites <span class="citation">[<a href="#ref-Ziller2013" role="doc-biblioref">63</a>]</span>. In addition, to reliably call DNA methylation levels at all sites with a dynamic range equivalent to other popular technologies such as the Illumina array would require reads depths on the order of 85x <span class="citation">[<a href="#ref-Libertini2016" role="doc-biblioref">149</a>]</span>. Given that read depth is semi-randomly distributed this means that it will likely not always be possible to get reliable calls at sites in which investigators are interested, without using what are a present prohibitively expensive read depths.</p>
<p>WGBS is very similar in principle to ordinary whole genome sequencing but preceded by a bisulfite conversion step as illustrated in Figure <a href="methods.html#fig:BisulfiteConversionNEB">3.1</a> <span class="citation">[<a href="#ref-Bock2010" role="doc-biblioref">142</a>]</span>. However, bisulfite conversion has several implications for downstream processing. Bisulfite library prep can be either directional, sequencing reads correspond to a bisulfite-converted version of either the forward or reverse strand, or non-directional in which sequencing reads correspond to bisulfite-converted versions of either strand giving a total of four bisulfite converted sequences with the strand of origin unknown. If a library is directional there are only two bisulfite converted sequences.</p>
<p>Mapping is usually done with customised software wrapper to implementations of existing alignment tools. Bismark is a popular example of such as wrapper which uses the bowtie alignment tool and calls site methylation levels <span class="citation">[<a href="#ref-Tran2014" role="doc-biblioref">150</a>]</span>. Alignment is performed with <em>in silico</em> bisulfite converted versions of the genome and reads, with G to A conversions for reverse strand reads. This requires four parallel instances of bowtie for forward and reverse G to A and C to T conversions, which are combined to determine the unique best alignment. (Figure <a href="methods.html#fig:Krueger2011BismarkAlignment">3.4</a> panel A). Bisulfite conversion results in loss of sequence complexity as many Cs effectively become Ts meaning short reads can be challenging to map. Bismark produces methylation calls either combined or by strand and which can be filtered by methylation context CpG, CHG or CHH.</p>
<p>Information theoretic analysis techniques which exploit the correlation of methylation levels between neighbouring CpG sites <span class="citation">[<a href="#ref-Haerter2014" role="doc-biblioref">103</a>]</span> to call methylation levels in small regions have recently been developed. They also provide additional insight into the variability of methylation at a given site and reliability with which the methylation state will be maintained even in relatively low coverage WGBS data <span class="citation">[<a href="#ref-Jenkinson2018" role="doc-biblioref">151</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Krueger2011BismarkAlignment"></span>
<img src="figs/Krueger2011BismarkAlignment.png" alt="[152]" width="90%" />
<p class="caption">
Figure 3.4: <span class="citation">[<a href="#ref-Krueger2011" role="doc-biblioref">152</a>]</span>
</p>
</div>

</div>
<div id="targeted" class="section level6">
<h6><span class="header-section-number">3.1.2.2.2.2</span> Targeted</h6>
<p>Targeted bisulfite sequencing can be carried out with a variety of methods for isolating the sequences of interest. Among them is bisulfite PCR, where the genome is bisulfite converted and PCR carried out with specially designed primers complementary to the regions flanking bisulfite converted sequence of interest. Tools for designing such primers are available, for example: ‘methPrimer’ <span class="citation">[<a href="#ref-Li2002" role="doc-biblioref">153</a>]</span>. Subsequent analysis is similar to that for WGBS, the PCR products are sequenced. The resulting reads aligned and methylation levels called with a tool such as Bismark.</p>
</div>
</div>
</div>
</div>
</div>
<div id="discussion" class="section level2">
<h2><span class="header-section-number">3.2</span> Discussion</h2>
<p>In summary, there are a wide range of DNA methylation assays and analytical techniques available, each with strengths and weaknesses. Thus, the choice of assay and analysis will need to be informed by the biological question being asked. For example, the 450k array would not be well suited to study the methylation of repetitive elements as its coverage in these regions is low. WGBS may be prohibitively expensive and MeDIP-seq may represent a good compromise, as it would provide good coverage of these regions and permit accurate detection of differential methylation. With the caveat that MeDIP-seq can be highly sensitive to genetic variation affecting CpG numbers <span class="citation">[<a href="#ref-Okitsu2015a" role="doc-biblioref">154</a>]</span>. This approach could potentially be improved by coupling MeDIP-seq with MRE-seq, as in congunction they would provide an improved ability to discern absolute methylation levels compared to MeDIP-seq alone <span class="citation">[<a href="#ref-Clark2012" role="doc-biblioref">155</a>]</span>. Unlike the 450k array MeDIP-seq captures regional methylation levels, whilst this may seem like a disadvantage due to lower resolution it can result in a more accurate and biologically relevant picture of regional methylation especially in highly methylated regions. The 450k array systematically underestimates the methylation level in highly methylated regions <span class="citation">[<a href="#ref-Clark2012" role="doc-biblioref">155</a>]</span>, this may well be because it samples only single sites, and highly methylated regions are often not fully methylated. Modeling work by <span class="citation">[<a href="#ref-Affinito2016" role="doc-biblioref">102</a>]</span> suggests that once a threshold level of DNAm is reached in a region further methyation becomes less likely, meaning that sampling a single site is likely to produce a systematic underestimate. MeDIP-seq can also cover much more the methylome than does the 450k array. The 450k array captures ~1.6% of the 28 million CpGs in the genome. At near saturation coverage of ~40 million reads per sample MeDIP-seq can cover ~60% of all CpGs with at least 1 read, this is allmost all methylated CpGs. At half that total read count MeDIP-seq still covers ~50% of CpGs at <span class="math inline">\(\ge1\times\)</span> and ~20% at <span class="math inline">\(\ge5\times\)</span> <span class="citation">[<a href="#ref-Taiwo2012" role="doc-biblioref">156</a>]</span>.</p>

</div>
</div>



<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Maunakea2010">
<p>48. Maunakea AK, Nagarajan RP, Bilenky M, Ballinger TJ, D’Souza C, Fouse SD, Johnson BE, Hong C, Nielsen C, Zhao Y, Turecki G, Delaney A, Varhol R, Thiessen N, Shchors K, Heine VM, Rowitch DH, Xing X, Fiore C, Schillebeeckx M, Jones SJM, Haussler D, Marra MA, Hirst M, Wang T, Costello JF: <strong>Conserved role of intragenic DNA methylation in regulating alternative promoters.</strong> <em>Nature</em> 2010, <strong>466</strong>:253–7<a href="https://doi.org/10.1038/nature09165">10.1038/nature09165</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20613842">http://www.ncbi.nlm.nih.gov/pubmed/20613842</a>.</p>
</div>
<div id="ref-Ziller2013">
<p>63. Ziller MJ, Gu H, Müller F, Donaghey J, Tsai LT-Y, Kohlbacher O, De Jager PL, Rosen ED, Bennett DA, Bernstein BE, Gnirke A, Meissner A: <strong>Charting a dynamic DNA methylation landscape of the human genome.</strong> <em>Nature</em> 2013, <strong>500</strong>:477–81<a href="https://doi.org/10.1038/nature12433">10.1038/nature12433</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23925113">http://www.ncbi.nlm.nih.gov/pubmed/23925113</a>.</p>
</div>
<div id="ref-Bibikova2006">
<p>90. Bibikova M: <strong>High-throughput DNA methylation profiling using universal bead arrays</strong>. <em>Genome Research</em> 2006, <strong>16</strong>:383–393<a href="https://doi.org/10.1101/gr.4410706">10.1101/gr.4410706</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.4410706">http://www.genome.org/cgi/doi/10.1101/gr.4410706</a>.</p>
</div>
<div id="ref-Bibikova2009">
<p>95. Bibikova M, Le J, Barnes B, Saedinia-Melnyk S, Zhou L, Shen R, Gunderson KL: <strong>Genome-wide DNA methylation profiling using Infinium assay</strong>. <em>Epigenomics</em> 2009, <strong>1</strong>:177–200<a href="https://doi.org/10.2217/epi.09.14">10.2217/epi.09.14</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi.09.14">https://www.futuremedicine.com/doi/10.2217/epi.09.14</a>.</p>
</div>
<div id="ref-Bibikova2011">
<p>101. Bibikova M, Barnes B, Tsan C, Ho V, Klotzle B, Le JM, Delano D, Zhang L, Schroth GP, Gunderson KL, Fan J-B, Shen R: <strong>High density DNA methylation array with single CpG site resolution.</strong> <em>Genomics</em> 2011, <strong>98</strong>:288–95<a href="https://doi.org/10.1016/j.ygeno.2011.07.007">10.1016/j.ygeno.2011.07.007</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21839163">http://www.ncbi.nlm.nih.gov/pubmed/21839163</a>.</p>
</div>
<div id="ref-Affinito2016">
<p>102. Affinito O, Scala G, Palumbo D, Florio E, Monticelli A, Miele G, Avvedimento VE, Usiello A, Chiariotti L, Cocozza S: <strong>Modeling DNA methylation by analyzing the individual configurations of single molecules.</strong> <em>Epigenetics</em> 2016, <strong>11</strong>:881–888<a href="https://doi.org/10.1080/15592294.2016.1246108">10.1080/15592294.2016.1246108</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27748645">http://www.ncbi.nlm.nih.gov/pubmed/27748645</a>.</p>
</div>
<div id="ref-Haerter2014">
<p>103. Haerter JO, Lövkvist C, Dodd IB, Sneppen K: <strong>Collaboration between CpG sites is needed for stable somatic inheritance of DNA methylation states.</strong> <em>Nucleic acids research</em> 2014, <strong>42</strong>:2235–44<a href="https://doi.org/10.1093/nar/gkt1235">10.1093/nar/gkt1235</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24288373%20http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3936770">http://www.ncbi.nlm.nih.gov/pubmed/24288373 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3936770</a>.</p>
</div>
<div id="ref-Hatada1991">
<p>135. Hatada I, Hayashizaki Y, Hirotsune S, Komatsubara H, Mukai T: <strong>A genomic scanning method for higher organisms using restriction sites as landmarks.</strong> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 1991, <strong>88</strong>:9523–7<a href="https://doi.org/10.1073/pnas.88.21.9523">10.1073/pnas.88.21.9523</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1946366">http://www.ncbi.nlm.nih.gov/pubmed/1946366</a>.</p>
</div>
<div id="ref-Weber2005">
<p>136. Weber M, Davies JJ, Wittig D, Oakeley EJ, Haase M, Lam WL, Schübeler D: <strong>Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells</strong>. <em>Nature Genetics</em> 2005, <strong>37</strong>:853–862<a href="https://doi.org/10.1038/ng1598">10.1038/ng1598</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16007088">http://www.ncbi.nlm.nih.gov/pubmed/16007088</a>.</p>
</div>
<div id="ref-Down2009">
<p>138. Down TA, Rakyan VK, Turner DJ, Flicek P, Li H, Kulesha E, Gräf S, Johnson N, Herrero J, Tomazou EM, Thorne NP, Bäckdahl L, Herberth M, Howe KL, Jackson DK, Miretti MM, Marioni JC, Birney E, Hubbard TJP, Durbin R, Tavaré S, Beck S: <strong>A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis</strong>. <em>Nature Biotechnology</em> 2008, <strong>26</strong>:779–785<a href="https://doi.org/10.1038/nbt1414">10.1038/nbt1414</a>Available: <a href="http://www.nature.com/articles/nbt1414">http://www.nature.com/articles/nbt1414</a>.</p>
</div>
<div id="ref-Lienhard2014">
<p>139. Lienhard M, Grimm C, Morkel M, Herwig R, Chavez L: <strong>MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments</strong>. <em>Bioinformatics</em> 2014, <strong>30</strong>:284–286<a href="https://doi.org/10.1093/bioinformatics/btt650">10.1093/bioinformatics/btt650</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24227674">http://www.ncbi.nlm.nih.gov/pubmed/24227674</a>.</p>
</div>
<div id="ref-Serre2009a">
<p>140. Serre D, Lee BH, Ting AH: <strong>MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome</strong>. <em>Nucleic Acids Research</em> 2010, <strong>38</strong>:391–399<a href="https://doi.org/10.1093/nar/gkp992">10.1093/nar/gkp992</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp992">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp992</a>.</p>
</div>
<div id="ref-Li2015">
<p>141. Li J, Yang J, Zhou P, Le Y, Zhou C, Wang S, Xu D, Lin H-K, Gong Z: <strong>Circular RNAs in cancer: novel insights into origins, properties, functions and implications.</strong> <em>American journal of cancer research</em> 2015, <strong>5</strong>:472–80Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25973291">http://www.ncbi.nlm.nih.gov/pubmed/25973291</a>.</p>
</div>
<div id="ref-Bock2010">
<p>142. Bock C, Tomazou EM, Brinkman AB, Müller F, Simmer F, Gu H, Jäger N, Gnirke A, Stunnenberg HG, Meissner A: <strong>Quantitative comparison of genome-wide DNA methylation mapping technologies</strong>. <em>Nature Biotechnology</em> 2010, <strong>28</strong>:1106–1114<a href="https://doi.org/10.1038/nbt.1681">10.1038/nbt.1681</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20852634">http://www.ncbi.nlm.nih.gov/pubmed/20852634</a>.</p>
</div>
<div id="ref-Hayatsu1970">
<p>143. Hayatsu H, Wataya Y, Kai K, Iida S: <strong>Reaction of sodium bisulfite with uracil, cytosine, and their derivatives</strong>. <em>Biochemistry</em> 1970, <strong>9</strong>:2858–2865<a href="https://doi.org/10.1021/bi00816a016">10.1021/bi00816a016</a>Available: <a href="http://pubs.acs.org/doi/abs/10.1021/bi00816a016">http://pubs.acs.org/doi/abs/10.1021/bi00816a016</a>.</p>
</div>
<div id="ref-Adamowicz2018">
<p>144. Adamowicz M, Maratou K, Aitman TJ: <strong>Multiplexed DNA Methylation Analysis of Target Regions Using Microfluidics (Fluidigm)</strong>. In <em>Methods in molecular biology (clifton, n.j.)</em> Vol. 1708 2018:349–363. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29224153">http://www.ncbi.nlm.nih.gov/pubmed/29224153</a>.</p>
</div>
<div id="ref-Skvortsova2017">
<p>145. Skvortsova K, Zotenko E, Luu P-L, Gould CM, Nair SS, Clark SJ, Stirzaker C: <strong>Comprehensive evaluation of genome-wide 5-hydroxymethylcytosine profiling approaches in human DNA</strong>. <em>Epigenetics &amp; Chromatin</em> 2017, <strong>10</strong>:16<a href="https://doi.org/10.1186/s13072-017-0123-7">10.1186/s13072-017-0123-7</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-017-0123-7">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-017-0123-7</a>.</p>
</div>
<div id="ref-Rhoads2015">
<p>146. Rhoads A, Au KF: <strong>PacBio Sequencing and Its Applications</strong>. <em>Genomics, Proteomics &amp; Bioinformatics</em> 2015, <strong>13</strong>:278–289<a href="https://doi.org/10.1016/j.gpb.2015.08.002">10.1016/j.gpb.2015.08.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1672022915001345">https://linkinghub.elsevier.com/retrieve/pii/S1672022915001345</a>.</p>
</div>
<div id="ref-Moran2015">
<p>148. Sandoval J, Heyn H, Moran S, Serra-Musach J, Pujana MA, Bibikova M, Esteller M: <strong>Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome.</strong> <em>Epigenetics</em> 2011, <strong>6</strong>:692–702<a href="https://doi.org/10.2217/epi.15.114">10.2217/epi.15.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21593595">http://www.ncbi.nlm.nih.gov/pubmed/21593595</a>.</p>
</div>
<div id="ref-Libertini2016">
<p>149. Libertini E, Heath SC, Hamoudi RA, Gut M, Ziller MJ, Herrero J, Czyz A, Ruotti V, Stunnenberg HG, Frontini M, Ouwehand WH, Meissner A, Gut IG, Beck S: <strong>Saturation analysis for whole-genome bisulfite sequencing data.</strong> <em>Nature biotechnology</em> 2016, <strong>34</strong>:691–693<a href="https://doi.org/10.1038/nbt.3524">10.1038/nbt.3524</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27347755">http://www.ncbi.nlm.nih.gov/pubmed/27347755</a>.</p>
</div>
<div id="ref-Tran2014">
<p>150. Tran H, Porter J, Sun M-a, Xie H, Zhang L: <strong>Objective and Comprehensive Evaluation of Bisulfite Short Read Mapping Tools</strong>. <em>Advances in Bioinformatics</em> 2014, <strong>2014</strong>:1–11<a href="https://doi.org/10.1155/2014/472045">10.1155/2014/472045</a>Available: <a href="http://www.hindawi.com/journals/abi/2014/472045/">http://www.hindawi.com/journals/abi/2014/472045/</a>.</p>
</div>
<div id="ref-Jenkinson2018">
<p>151. Jenkinson G, Abante J, Feinberg AP, Goutsias J: <strong>An information-theoretic approach to the modeling and analysis of whole-genome bisulfite sequencing data</strong>. <em>BMC Bioinformatics</em> 2018, <strong>19</strong>:87<a href="https://doi.org/10.1186/s12859-018-2086-5">10.1186/s12859-018-2086-5</a>Available: <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2086-5">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2086-5</a>.</p>
</div>
<div id="ref-Krueger2011">
<p>152. Krueger F, Andrews SR: <strong>Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications</strong>. <em>Bioinformatics</em> 2011, <strong>27</strong>:1571–1572<a href="https://doi.org/10.1093/bioinformatics/btr167">10.1093/bioinformatics/btr167</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr167">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr167</a>.</p>
</div>
<div id="ref-Li2002">
<p>153. Li L-C, Dahiya R: <strong>MethPrimer: designing primers for methylation PCRs.</strong> <em>Bioinformatics (Oxford, England)</em> 2002, <strong>18</strong>:1427–31Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12424112">http://www.ncbi.nlm.nih.gov/pubmed/12424112</a>.</p>
</div>
<div id="ref-Okitsu2015a">
<p>154. Okitsu CY, Hsieh C-L: <strong>Sensitivity and specificity of immunoprecipitation of DNA containing 5-Methylcytosine</strong>. <em>BMC Research Notes</em> 2015, <strong>8</strong>:102<a href="https://doi.org/10.1186/s13104-015-1069-0">10.1186/s13104-015-1069-0</a>Available: <a href="http://www.biomedcentral.com/1756-0500/8/102">http://www.biomedcentral.com/1756-0500/8/102</a>.</p>
</div>
<div id="ref-Clark2012">
<p>155. Clark C, Palta P, Joyce CJ, Scott C, Grundberg E, Deloukas P, Palotie A, Coffey AJ: <strong>A Comparison of the Whole Genome Approach of MeDIP-Seq to the Targeted Approach of the Infinium HumanMethylation450 BeadChip for Methylome Profiling</strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e50233<a href="https://doi.org/10.1371/journal.pone.0050233">10.1371/journal.pone.0050233</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0050233">https://dx.plos.org/10.1371/journal.pone.0050233</a>.</p>
</div>
<div id="ref-Taiwo2012">
<p>156. Taiwo O, Wilson GA, Morris T, Seisenberger S, Reik W, Pearce D, Beck S, Butcher LM: <strong>Methylome analysis using MeDIP-seq with low DNA concentrations</strong>. <em>Nature Protocols</em> 2012, <strong>7</strong>:617–636<a href="https://doi.org/10.1038/nprot.2012.012">10.1038/nprot.2012.012</a>Available: <a href="http://www.nature.com/articles/nprot.2012.012">http://www.nature.com/articles/nprot.2012.012</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="the-ageing-poplation-and-burden-of-disease.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="MAVIDOS.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/02-methods.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
