<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="References | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="discussion-3.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#illumina-methylation-arrays"><i class="fa fa-check"></i><b>2.1</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="3.1" data-path="arrays.html"><a href="arrays.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="arrays.html"><a href="arrays.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#background"><i class="fa fa-check"></i><b>3.2.1</b> Background</a></li>
<li class="chapter" data-level="3.2.2" data-path="arrays.html"><a href="arrays.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>3.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="arrays.html"><a href="arrays.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#summary-of-results"><i class="fa fa-check"></i><b>3.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#quality-control-qc"><i class="fa fa-check"></i><b>3.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#functional-normalisation-1"><i class="fa fa-check"></i><b>3.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.4.4" data-path="arrays.html"><a href="arrays.html#ewass"><i class="fa fa-check"></i><b>3.4.4</b> EWASs</a></li>
<li class="chapter" data-level="3.4.5" data-path="arrays.html"><a href="arrays.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>3.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>3.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="3.5.1" data-path="arrays.html"><a href="arrays.html#section"><i class="fa fa-check"></i><b>3.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.6</b> Discussion</a><ul>
<li class="chapter" data-level="3.6.1" data-path="arrays.html"><a href="arrays.html#future-work"><i class="fa fa-check"></i><b>3.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#Methods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#participants-1"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="4.3.9" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-figures"><i class="fa fa-check"></i><b>4.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.3.10" data-path="tRNAs.html"><a href="tRNAs.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>4.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#discussion-1"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#data-availability"><i class="fa fa-check"></i><b>4.6</b> Data availability</a></li>
<li class="chapter" data-level="4.7" data-path="tRNAs.html"><a href="tRNAs.html#code-availablity"><i class="fa fa-check"></i><b>4.7</b> Code availablity</a></li>
<li class="chapter" data-level="4.8" data-path="tRNAs.html"><a href="tRNAs.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.9" data-path="tRNAs.html"><a href="tRNAs.html#author-contributions"><i class="fa fa-check"></i><b>4.9</b> Author Contributions</a></li>
<li class="chapter" data-level="4.10" data-path="tRNAs.html"><a href="tRNAs.html#references"><i class="fa fa-check"></i><b>4.10</b> References</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results-3-.html"><a href="results-3-.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="results-3-.html"><a href="results-3-.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="results-3-.html"><a href="results-3-.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="results-3-.html"><a href="results-3-.html#methods-2"><i class="fa fa-check"></i><b>5.3</b> Methods</a></li>
<li class="chapter" data-level="5.4" data-path="results-3-.html"><a href="results-3-.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>5.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.4.2" data-path="results-3-.html"><a href="results-3-.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="5.4.3" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>5.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="5.4.4" data-path="results-3-.html"><a href="results-3-.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>5.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="results-3-.html"><a href="results-3-.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="results-3-.html"><a href="results-3-.html#methods-3"><i class="fa fa-check"></i><b>5.6</b> Methods</a><ul>
<li class="chapter" data-level="5.6.1" data-path="results-3-.html"><a href="results-3-.html#participants-2"><i class="fa fa-check"></i><b>5.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="results-3-.html"><a href="results-3-.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>5.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="5.7.1" data-path="results-3-.html"><a href="results-3-.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>5.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="5.7.2" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>5.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.7.3" data-path="results-3-.html"><a href="results-3-.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.7.4" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.7.5" data-path="results-3-.html"><a href="results-3-.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references-1" class="section level1 unnumbered">
<h1>References</h1>

<div id="refs" class="references">
<div>
<p>Acton, Richard J, Emma Bourne, Jordana Bell, Karen Lillycrop, Jun Wang, Elaine Dennison, Nicholas Harvey, Tim D Spector, Cyrus Cooper, and Christopher G Bell. 2019. “The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation.” <em>bioRxiv</em>, 1–48. <a href="https://doi.org/10.1101/870352">https://doi.org/10.1101/870352</a>.</p>
</div>
<div>
<p>Adamowicz, Martyna, Klio Maratou, and Timothy J Aitman. 2018. “Multiplexed DNA Methylation Analysis of Target Regions Using Microfluidics (Fluidigm).” In <em>Methods in Molecular Biology (Clifton, N.j.)</em>, 1708:349–63. <a href="https://doi.org/10.1007/978-1-4939-7481-8_18">https://doi.org/10.1007/978-1-4939-7481-8_18</a>.</p>
</div>
<div>
<p>Ade, Catherine, Astrid M. Roy-Engel, and Prescott L. Deininger. 2013. “Alu elements: an intrinsic source of human genome instability.” <em>Current Opinion in Virology</em> 3 (6): 639–45. <a href="https://doi.org/10.1016/j.coviro.2013.09.002">https://doi.org/10.1016/j.coviro.2013.09.002</a>.</p>
</div>
<div>
<p>Affinito, Ornella, Giovanni Scala, Domenico Palumbo, Ermanno Florio, Antonella Monticelli, Gennaro Miele, Vittorio Enrico Avvedimento, Alessandro Usiello, Lorenzo Chiariotti, and Sergio Cocozza. 2016. “Modeling DNA methylation by analyzing the individual configurations of single molecules.” <em>Epigenetics</em> 11 (12): 881–88. <a href="https://doi.org/10.1080/15592294.2016.1246108">https://doi.org/10.1080/15592294.2016.1246108</a>.</p>
</div>
<div>
<p>Amemiya, Haley M., Anshul Kundaje, and Alan P. Boyle. 2019. “The ENCODE Blacklist: Identification of Problematic Regions of the Genome.” <em>Scientific Reports</em> 9 (1): 9354. <a href="https://doi.org/10.1038/s41598-019-45839-z">https://doi.org/10.1038/s41598-019-45839-z</a>.</p>
</div>
<div>
<p>Andrews, S. 2010. “FastQC.” Cambridge: Babraham Bioinformatics. <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</p>
</div>
<div>
<p>Andrews, Shan V, Christine Ladd-Acosta, Andrew P Feinberg, Kasper D Hansen, and M Daniele Fallin. 2016. “‘Gap hunting’ to characterize clustered probe signals in Illumina methylation array data.” <em>Epigenetics &amp; Chromatin</em> 9 (1): 56. <a href="https://doi.org/10.1186/s13072-016-0107-z">https://doi.org/10.1186/s13072-016-0107-z</a>.</p>
</div>
<div>
<p>Bacalini, Maria Giulia, Alessio Boattini, Davide Gentilini, Enrico Giampieri, Chiara Pirazzini, Cristina Giuliani, Elisa Fontanesi, et al. 2015. “A meta-analysis on age-associated changes in blood DNA methylation: results from an original analysis pipeline for Infinium 450k data.” <em>Aging</em> 7 (2): 97–109. <a href="https://doi.org/10.18632/aging.100718">https://doi.org/10.18632/aging.100718</a>.</p>
</div>
<div>
<p>Bakulski, Kelly M., Jason I. Feinberg, Shan V. Andrews, Jack Yang, Shannon Brown, Stephanie L. McKenney, Frank Witter, Jeremy Walston, Andrew P. Feinberg, and M. Daniele Fallin. 2016. “DNA methylation of cord blood cell types: Applications for mixed cell birth studies.” <em>Epigenetics</em> 11 (5): 354–62. <a href="https://doi.org/10.1080/15592294.2016.1161875">https://doi.org/10.1080/15592294.2016.1161875</a>.</p>
</div>
<div>
<p>Bannister, Andrew J, and Tony Kouzarides. 2011. “Regulation of chromatin by histone modifications.” <em>Cell Research</em> 21 (3): 381–95. <a href="https://doi.org/10.1038/cr.2011.22">https://doi.org/10.1038/cr.2011.22</a>.</p>
</div>
<div>
<p>Barnett, Karen, Stewart W. Mercer, Michael Norbury, Graham Watt, Sally Wyke, and Bruce Guthrie. 2012. “Epidemiology of multimorbidity and implications for health care, research, and medical education: a cross-sectional study.” <em>The Lancet</em> 380 (9836): 37–43. <a href="https://doi.org/10.1016/S0140-6736(12)60240-2">https://doi.org/10.1016/S0140-6736(12)60240-2</a>.</p>
</div>
<div>
<p>Bauden, Monika, Theresa Kristl, Roland Andersson, György Marko-Varga, and Daniel Ansari. 2017. “Characterization of histone-related chemical modifications in formalin-fixed paraffin-embedded and fresh-frozen human pancreatic cancer xenografts using LC-MS/MS.” <em>Laboratory Investigation</em> 97 (3): 279–88. <a href="https://doi.org/10.1038/labinvest.2016.134">https://doi.org/10.1038/labinvest.2016.134</a>.</p>
</div>
<div>
<p>Belancio, Victoria P., Dale J. Hedges, and Prescott Deininger. 2008. “Mammalian non-LTR retrotransposons: For better or worse, in sickness and in health.” <em>Genome Research</em> 18 (3): 343–58. <a href="https://doi.org/10.1101/gr.5558208">https://doi.org/10.1101/gr.5558208</a>.</p>
</div>
<div>
<p>Belikov, Aleksey V. 2019. “Age-related diseases as vicious cycles.” <em>Ageing Research Reviews</em> 49 (November 2018): 11–26. <a href="https://doi.org/10.1016/j.arr.2018.11.002">https://doi.org/10.1016/j.arr.2018.11.002</a>.</p>
</div>
<div>
<p>Bell, Christopher G., Fei Gao, Wei Yuan, Leonie Roos, Richard J. Acton, Yudong Xia, Jordana Bell, et al. 2018. “Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci.” <em>Nature Communications</em> 9 (1): 8. <a href="https://doi.org/10.1038/s41467-017-01586-1">https://doi.org/10.1038/s41467-017-01586-1</a>.</p>
</div>
<div>
<p>Bell, Christopher G, Robert Lowe, Peter D Adams, Andrea A Baccarelli, Stephan Beck, Jordana T Bell, Brock C. Christensen, et al. 2019. “DNA methylation aging clocks: challenges and recommendations.” <em>Genome Biology</em> 20 (1): 249. <a href="https://doi.org/10.1186/s13059-019-1824-y">https://doi.org/10.1186/s13059-019-1824-y</a>.</p>
</div>
<div>
<p>Bell, Christopher G., Gareth A. Wilson, Lee M. Butcher, Christian Roos, Lutz Walter, and Stephan Beck. 2012. “Human-specific CpG "beacons" identify loci associated with human-specific traits and disease.” <em>Epigenetics</em> 7 (10): 1188–99. <a href="https://doi.org/10.4161/epi.22127">https://doi.org/10.4161/epi.22127</a>.</p>
</div>
<div>
<p>Bell, Christopher G., Yudong Xia, Wei Yuan, Fei Gao, Kirsten Ward, Leonie Roos, Massimo Mangino, et al. 2016. “Novel regional age-associated DNA methylation changes within human common disease-associated loci.” <em>Genome Biology</em> 17 (1): 193. <a href="https://doi.org/10.1186/s13059-016-1051-8">https://doi.org/10.1186/s13059-016-1051-8</a>.</p>
</div>
<div>
<p>Bell, Jordana T., Pei-Chien Tsai, Tsun-Po Yang, Ruth Pidsley, James Nisbet, Daniel Glass, Massimo Mangino, et al. 2012. “Epigenome-wide scans identify differentially methylated regions for age and age-related phenotypes in a healthy ageing population.” Edited by Jun Li. <em>PLoS Genetics</em> 8 (4): e1002629. <a href="https://doi.org/10.1371/journal.pgen.1002629">https://doi.org/10.1371/journal.pgen.1002629</a>.</p>
</div>
<div>
<p>Bell, Joshua S. K., and Paula M. Vertino. 2017. “Orphan CpG islands define a novel class of highly active enhancers.” <em>Epigenetics</em> 12 (6): 449–64. <a href="https://doi.org/10.1080/15592294.2017.1297910">https://doi.org/10.1080/15592294.2017.1297910</a>.</p>
</div>
<div>
<p>Benton, Miles C, Heidi G Sutherland, Donia Macartney-Coxson, Larisa M Haupt, Rodney A. Lea, and Lyn R Griffiths. 2017. “Methylome-wide association study of whole blood DNA in the Norfolk Island isolate identifies robust loci associated with age.” <em>Aging</em> 9 (3): 753–68. <a href="https://doi.org/10.18632/aging.101187">https://doi.org/10.18632/aging.101187</a>.</p>
</div>
<div>
<p>Berdyshev, G D, G K Korotaev, G V Boiarskikh, and B F Vaniushin. 1967. “Nucleotide composition of DNA and RNA from somatic tissues of humpback and its changes during spawning.” <em>Biokhimiia (Moscow, Russia)</em> 32 (5): 988–93. <a href="http://www.ncbi.nlm.nih.gov/pubmed/5628601">http://www.ncbi.nlm.nih.gov/pubmed/5628601</a>.</p>
</div>
<div>
<p>Besser, Daniel, Frank Götz, Kai Schulze-Forster, Herbert Wagner, Hans Kröger, and Dietrich Simon. 1990. “DNA methylation inhibits transcription by RNA polymerase III of a tRNA gene, but not of a 5S rRNA gene.” <em>FEBS Letters</em> 269 (2): 358–62. <a href="https://doi.org/10.1016/0014-5793(90)81193-R">https://doi.org/10.1016/0014-5793(90)81193-R</a>.</p>
</div>
<div>
<p>Bianconi, Eva, Allison Piovesan, Federica Facchin, Alina Beraudi, Raffaella Casadei, Flavia Frabetti, Lorenza Vitale, et al. 2013. “An estimation of the number of cells in the human body.” <em>Annals of Human Biology</em> 40 (6): 463–71. <a href="https://doi.org/10.3109/03014460.2013.807878">https://doi.org/10.3109/03014460.2013.807878</a>.</p>
</div>
<div>
<p>Bibikova, Marina. 2006. “High-throughput DNA methylation profiling using universal bead arrays.” <em>Genome Research</em> 16 (3): 383–93. <a href="https://doi.org/10.1101/gr.4410706">https://doi.org/10.1101/gr.4410706</a>.</p>
</div>
<div>
<p>Bibikova, Marina, Bret Barnes, Chan Tsan, Vincent Ho, Brandy Klotzle, Jennie M. Le, David Delano, et al. 2011. “High density DNA methylation array with single CpG site resolution.” <em>Genomics</em> 98 (4): 288–95. <a href="https://doi.org/10.1016/j.ygeno.2011.07.007">https://doi.org/10.1016/j.ygeno.2011.07.007</a>.</p>
</div>
<div>
<p>Bibikova, Marina, Jennie Le, Bret Barnes, Shadi Saedinia-Melnyk, Lixin Zhou, Richard Shen, and Kevin L Gunderson. 2009. “Genome-wide DNA methylation profiling using Infinium assay.” <em>Epigenomics</em> 1 (1): 177–200. <a href="https://doi.org/10.2217/epi.09.14">https://doi.org/10.2217/epi.09.14</a>.</p>
</div>
<div>
<p>Bird, Adrian. 2002. “DNA methylation patterns and epigenetic memory.” <em>Genes &amp; Development</em> 16 (1): 6–21. <a href="https://doi.org/10.1101/gad.947102">https://doi.org/10.1101/gad.947102</a>.</p>
</div>
<div>
<p>———. 2007. “Perceptions of epigenetics.” <em>Nature</em> 447 (7143): 396–8. <a href="https://doi.org/10.1038/nature05913">https://doi.org/10.1038/nature05913</a>.</p>
</div>
<div>
<p>Bjornsson, Hans T. 2008. “Intra-individual Change Over Time in DNA Methylation With Familial Clustering.” <em>JAMA</em> 299 (24): 2877. <a href="https://doi.org/10.1001/jama.299.24.2877">https://doi.org/10.1001/jama.299.24.2877</a>.</p>
</div>
<div>
<p>Blattler, Adam, Lijing Yao, Heather Witt, Yu Guo, Charles M Nicolet, Benjamin P Berman, and Peggy J Farnham. 2014. “Global loss of DNA methylation uncovers intronic enhancers in genes showing expression changes.” <em>Genome Biology</em> 15 (9): 469. <a href="https://doi.org/10.1186/s13059-014-0469-0">https://doi.org/10.1186/s13059-014-0469-0</a>.</p>
</div>
<div>
<p>Bock, Christoph, Eleni M Tomazou, Arie B Brinkman, Fabian Müller, Femke Simmer, Hongcang Gu, Natalie Jäger, Andreas Gnirke, Hendrik G Stunnenberg, and Alexander Meissner. 2010. “Quantitative comparison of genome-wide DNA methylation mapping technologies.” <em>Nature Biotechnology</em> 28 (10): 1106–14. <a href="https://doi.org/10.1038/nbt.1681">https://doi.org/10.1038/nbt.1681</a>.</p>
</div>
<div>
<p>Bocklandt, Sven, Wen Lin, Mary E. Sehl, Francisco J. Sánchez, Janet S. Sinsheimer, Steve Horvath, and Eric Vilain. 2011. “Epigenetic Predictor of Age.” Edited by Nicoletta Landsberger. <em>PLoS ONE</em> 6 (6): e14821. <a href="https://doi.org/10.1371/journal.pone.0014821">https://doi.org/10.1371/journal.pone.0014821</a>.</p>
</div>
<div>
<p>Bohlin, J., S. E. Håberg, P. Magnus, S. E. Reese, H. K. Gjessing, M. C. Magnus, C. L. Parr, C. M. Page, S. J. London, and W. Nystad. 2016. “Prediction of gestational age based on genome-wide differentially methylated regions.” <em>Genome Biology</em> 17 (1): 207. <a href="https://doi.org/10.1186/s13059-016-1063-4">https://doi.org/10.1186/s13059-016-1063-4</a>.</p>
</div>
<div>
<p>Boks, Marco P., Eske M. Derks, Daniel J. Weisenberger, Erik Strengman, Esther Janson, Iris E. Sommer, René S. Kahn, and Roel A. Ophoff. 2009. “The relationship of DNA methylation with age, gender and genotype in twins and healthy controls.” Edited by Joseph Najbauer. <em>PloS One</em> 4 (8): e6767. <a href="https://doi.org/10.1371/journal.pone.0006767">https://doi.org/10.1371/journal.pone.0006767</a>.</p>
</div>
<div>
<p>Bollati, Valentina, Joel Schwartz, Robert Wright, Augusto Litonjua, Letizia Tarantini, Helen Suh, David Sparrow, Pantel Vokonas, and Andrea Baccarelli. 2009. “Decline in genomic DNA methylation through aging in a cohort of elderly subjects.” <em>Mechanisms of Ageing and Development</em> 130 (4): 234–39. <a href="https://doi.org/10.1016/j.mad.2008.12.003">https://doi.org/10.1016/j.mad.2008.12.003</a>.</p>
</div>
<div>
<p>Booth, Lauren N., and Anne Brunet. 2016. “The Aging Epigenome.” <em>Molecular Cell</em> 62 (5): 728–44. <a href="https://doi.org/10.1016/j.molcel.2016.05.013">https://doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div>
<p>Bostick, M., J. K. Kim, P.-O. Esteve, A. Clark, S. Pradhan, and S. E. Jacobsen. 2007. “UHRF1 Plays a Role in Maintaining DNA Methylation in Mammalian Cells.” <em>Science</em> 317 (5845): 1760–4. <a href="https://doi.org/10.1126/science.1147939">https://doi.org/10.1126/science.1147939</a>.</p>
</div>
<div>
<p>Breton, Carrie V., Carmen J. Marsit, Elaine Faustman, Kari Nadeau, Jaclyn M. Goodrich, Dana C. Dolinoy, Julie Herbstman, et al. 2017. “Small-Magnitude Effect Sizes in Epigenetic End Points are Important in Children’s Environmental Health Studies: The Children’s Environmental Health and Disease Prevention Research Center’s Epigenetics Working Group.” <em>Environmental Health Perspectives</em> 125 (4): 511–26. <a href="https://doi.org/10.1289/EHP595">https://doi.org/10.1289/EHP595</a>.</p>
</div>
<div>
<p>Briggs, Adrian W., Udo Stenzel, Matthias Meyer, Johannes Krause, Martin Kircher, and Svante Pääbo. 2010. “Removal of deaminated cytosines and detection of in vivo methylation in ancient DNA.” <em>Nucleic Acids Research</em> 38 (6): e87. <a href="https://doi.org/10.1093/nar/gkp1163">https://doi.org/10.1093/nar/gkp1163</a>.</p>
</div>
<div>
<p>Burdge, Graham C., Mark A. Hanson, Jo L. Slater-Jefferies, and Karen A. Lillycrop. 2007. “Epigenetic regulation of transcription: a mechanism for inducing variations in phenotype (fetal programming) by differences in nutrition during early life?” <em>British Journal of Nutrition</em> 97 (6): 1036–46. <a href="https://doi.org/10.1017/S0007114507682920">https://doi.org/10.1017/S0007114507682920</a>.</p>
</div>
<div>
<p>Burdge, Graham C, Jo Slater-Jefferies, Christopher Torrens, Emma S Phillips, Mark A. Hanson, and Karen A. Lillycrop. 2007. “Dietary protein restriction of pregnant rats in the F 0 generation induces altered methylation of hepatic gene promoters in the adult male offspring in the F 1 and F 2 generations.” <em>British Journal of Nutrition</em> 97 (3): 435–39. <a href="https://doi.org/10.1017/S0007114507352392">https://doi.org/10.1017/S0007114507352392</a>.</p>
</div>
<div>
<p>Bush, William S., Matthew T. Oetjens, and Dana C. Crawford. 2016. “Unravelling the human genome-phenome relationship using phenome-wide association studies.” <em>Nature Reviews Genetics</em> 17 (3): 129–45. <a href="https://doi.org/10.1038/nrg.2015.36">https://doi.org/10.1038/nrg.2015.36</a>.</p>
</div>
<div>
<p>Bycroft, Clare, Colin Freeman, Desislava Petkova, Gavin Band, Lloyd T. Elliott, Kevin Sharp, Allan Motyer, et al. 2017. “Genome-wide genetic data on ~500,000 UK Biobank participants.” <em>bioRxiv</em>, 166298. <a href="https://doi.org/10.1101/166298">https://doi.org/10.1101/166298</a>.</p>
</div>
<div>
<p>Campisi, Judith, Pankaj Kapahi, Gordon J. Lithgow, Simon Melov, John C. Newman, and Eric Verdin. 2019. “From discoveries in ageing research to therapeutics for healthy ageing.” <em>Nature</em> 571 (7764): 183–92. <a href="https://doi.org/10.1038/s41586-019-1365-2">https://doi.org/10.1038/s41586-019-1365-2</a>.</p>
</div>
<div>
<p>Canella, Donatella, Viviane Praz, Jaime H. Reina, Pascal Cousin, and Nouria Hernandez. 2010. “Defining the RNA polymerase III transcriptome: Genome-wide localization of the RNA polymerase III transcription machinery in human cells.” <em>Genome Research</em> 20 (6): 710–21. <a href="https://doi.org/10.1101/gr.101337.109">https://doi.org/10.1101/gr.101337.109</a>.</p>
</div>
<div>
<p>Cardenas, Andres, Catherine Allard, Myriam Doyon, E Andres Houseman, Kelly M Bakulski, Patrice Perron, Luigi Bouchard, and Marie-France Hivert. 2016. “Validation of a DNA methylation reference panel for the estimation of nucleated cells types in cord blood.” <em>Epigenetics</em> 11 (11): 773–79. <a href="https://doi.org/10.1080/15592294.2016.1233091">https://doi.org/10.1080/15592294.2016.1233091</a>.</p>
</div>
<div>
<p>Carrillo-de-Santa-Pau, Enrique, David Juan, Vera Pancaldi, Felipe Were, Ignacio Martin-Subero, Daniel Rico, and Alfonso Valencia. 2017. “Automatic identification of informative regions with epigenomic changes associated to hematopoiesis.” <em>Nucleic Acids Research</em> 45 (16): 9244–59. <a href="https://doi.org/10.1093/nar/gkx618">https://doi.org/10.1093/nar/gkx618</a>.</p>
</div>
<div>
<p>CellOntology. 2017. “Cell Ontology.” <a href="https://doi.org/10.5281/zenodo.168254">https://doi.org/10.5281/zenodo.168254</a>.</p>
</div>
<div>
<p>CellSystemsVoices. 2017. “What Is Your Conceptual Definition of ‘Cell Type’ in the Context of a Mature Organism?” <em>Cell Systems</em> 4 (3): 255–59. <a href="https://doi.org/10.1016/j.cels.2017.03.006">https://doi.org/10.1016/j.cels.2017.03.006</a>.</p>
</div>
<div>
<p>Cerase, Andrea, Greta Pintacuda, Anna Tattermusch, and Philip Avner. 2015. “Xist localization and function: new insights from multiple levels.” <em>Genome Biology</em> 16 (1): 166. <a href="https://doi.org/10.1186/s13059-015-0733-y">https://doi.org/10.1186/s13059-015-0733-y</a>.</p>
</div>
<div>
<p>Chan, Patricia P., and Todd M. Lowe. 2009. “GtRNAdb: a database of transfer RNA genes detected in genomic sequence.” <em>Nucleic Acids Research</em> 37 (Database issue): D93–7. <a href="https://doi.org/10.1093/nar/gkn787">https://doi.org/10.1093/nar/gkn787</a>.</p>
</div>
<div>
<p>Chen, Brian H, Riccardo E Marioni, Elena Colicino, Marjolein J Peters, Cavin K Ward-Caviness, Pei-Chien Tsai, Nicholas S Roetker, et al. 2016. “DNA methylation-based measures of biological age: meta-analysis predicting time to death.” <em>Aging</em> 8 (9): 1844–65. <a href="https://doi.org/10.18632/aging.101020">https://doi.org/10.18632/aging.101020</a>.</p>
</div>
<div>
<p>Chen, Ling-Ling, and Li Yang. 2017. “ALU ternative Regulation for Gene Expression.” <em>Trends in Cell Biology</em> 27 (7): 480–90. <a href="https://doi.org/10.1016/j.tcb.2017.01.002">https://doi.org/10.1016/j.tcb.2017.01.002</a>.</p>
</div>
<div>
<p>Christensen, Brock C., E. Andres Houseman, Carmen J. Marsit, Shichun Zheng, Margaret R. Wrensch, Joseph L. Wiemels, Heather H. Nelson, et al. 2009. “Aging and Environmental Exposures Alter Tissue-Specific DNA Methylation Dependent upon CpG Island Context.” Edited by Dirk Schübeler. <em>PLoS Genetics</em> 5 (8): e1000602. <a href="https://doi.org/10.1371/journal.pgen.1000602">https://doi.org/10.1371/journal.pgen.1000602</a>.</p>
</div>
<div>
<p>Christiansen, Lene, Adam Lenart, Qihua Tan, James W. Vaupel, Abraham Aviv, Matt McGue, and Kaare Christensen. 2016. “DNA methylation age is associated with mortality in a longitudinal Danish twin study.” <em>Aging Cell</em> 15 (1): 149–54. <a href="https://doi.org/10.1111/acel.12421">https://doi.org/10.1111/acel.12421</a>.</p>
</div>
<div>
<p>Chuong, Edward B., Nels C. Elde, and Cédric Feschotte. 2017. “Regulatory activities of transposable elements: from conflicts to benefits.” <em>Nature Reviews Genetics</em> 18 (2): 71–86. <a href="https://doi.org/10.1038/nrg.2016.139">https://doi.org/10.1038/nrg.2016.139</a>.</p>
</div>
<div>
<p>Ciernia, Annie Vogel, and Janine LaSalle. 2016. “The landscape of DNA methylation amid a perfect storm of autism aetiologies.” <em>Nature Reviews Neuroscience</em> 17 (7): 411–23. <a href="https://doi.org/10.1038/nrn.2016.41">https://doi.org/10.1038/nrn.2016.41</a>.</p>
</div>
<div>
<p>Clark, Christine, Priit Palta, Christopher J. Joyce, Carol Scott, Elin Grundberg, Panos Deloukas, Aarno Palotie, and Alison J. Coffey. 2012. “A Comparison of the Whole Genome Approach of MeDIP-Seq to the Targeted Approach of the Infinium HumanMethylation450 BeadChip for Methylome Profiling.” Edited by Esteban Ballestar. <em>PLoS ONE</em> 7 (11): e50233. <a href="https://doi.org/10.1371/journal.pone.0050233">https://doi.org/10.1371/journal.pone.0050233</a>.</p>
</div>
<div>
<p>Cohen, Jacob. 1988. <em>Statisticsl Power Analysis for the Behavioural Sciences</em>. 2nd ed. New York: Lawrence Erlbaum Associates.</p>
</div>
<div>
<p>Cole, John Joseph, Neil A. Robertson, Mohammed Iqbal Rather, John P. Thomson, Tony McBryan, Duncan Sproul, Tina Wang, et al. 2017. “Diverse interventions that extend mouse lifespan suppress shared age-associated epigenetic changes at critical gene regulatory regions.” <em>Genome Biology</em> 18 (1): 58. <a href="https://doi.org/10.1186/s13059-017-1185-3">https://doi.org/10.1186/s13059-017-1185-3</a>.</p>
</div>
<div>
<p>Cooper, Cyrus, Nicholas C. Harvey, Nicholas J. Bishop, Stephen Kennedy, Aris T. Papageorghiou, Inez Schoenmakers, Robert Fraser, et al. 2016. “Maternal gestational vitamin D supplementation and offspring bone health (MAVIDOS): a multicentre, double-blind, randomised placebo-controlled trial.” <em>The Lancet Diabetes &amp; Endocrinology</em> 4 (5): 393–402. <a href="https://doi.org/10.1016/S2213-8587(16)00044-9">https://doi.org/10.1016/S2213-8587(16)00044-9</a>.</p>
</div>
<div>
<p>Cordaux, Richard, and Mark A. Batzer. 2009. “The impact of retrotransposons on human genome evolution.” <em>Nature Reviews Genetics</em> 10 (10): 691–703. <a href="https://doi.org/10.1038/nrg2640">https://doi.org/10.1038/nrg2640</a>.</p>
</div>
<div>
<p>Crighton, Diane. 2003. “p53 represses RNA polymerase III transcription by targeting TBP and inhibiting promoter occupancy by TFIIIB.” <em>The EMBO Journal</em> 22 (11): 2810–20. <a href="https://doi.org/10.1093/emboj/cdg265">https://doi.org/10.1093/emboj/cdg265</a>.</p>
</div>
<div>
<p>Cruickshanks, Hazel A, Tony McBryan, David M Nelson, Nathan D. VanderKraats, Parisha P Shah, John van Tuyn, Taranjit Singh Rai, et al. 2013. “Senescent cells harbour features of the cancer epigenome.” <em>Nature Cell Biology</em> 15 (12): 1495–1506. <a href="https://doi.org/10.1038/ncb2879">https://doi.org/10.1038/ncb2879</a>.</p>
</div>
<div>
<p>Curtis, Elizabeth M., Robert Murray, Philip Titcombe, Eloïse Cook, Rebecca Clarke-Harris, Paula Costello, Emma Garratt, et al. 2017. “Perinatal DNA Methylation at CDKN2A Is Associated With Offspring Bone Mass: Findings From the Southampton Women’s Survey.” <em>Journal of Bone and Mineral Research</em> 32 (10): 2030–40. <a href="https://doi.org/10.1002/jbmr.3153">https://doi.org/10.1002/jbmr.3153</a>.</p>
</div>
<div>
<p>Darrow, Emily M., and Brian P. Chadwick. 2014. “A novel tRNA variable number tandem repeat at human chromosome 1q23.3 is implicated as a boundary element based on conservation of a CTCF motif in mouse.” <em>Nucleic Acids Research</em> 42 (10): 6421–35. <a href="https://doi.org/10.1093/nar/gku280">https://doi.org/10.1093/nar/gku280</a>.</p>
</div>
<div>
<p>Deaton, Aimée M., and Adrian Bird. 2011. “CpG islands and the regulation of transcription.” <em>Genes &amp; Development</em> 25 (10): 1010–22. <a href="https://doi.org/10.1101/gad.2037511">https://doi.org/10.1101/gad.2037511</a>.</p>
</div>
<div>
<p>Dedeurwaerder, Sarah, Matthieu Defrance, Emilie Calonne, Hélène Denis, Christos Sotiriou, and François Fuks. 2011. “Evaluation of the Infinium Methylation 450K technology.” <em>Epigenomics</em> 3 (6): 771–84. <a href="https://doi.org/10.2217/epi.11.105">https://doi.org/10.2217/epi.11.105</a>.</p>
</div>
<div>
<p>Deininger, Prescott. 2011. “Alu elements: know the SINEs.” <em>Genome Biology</em> 12 (12): 236. <a href="https://doi.org/10.1186/gb-2011-12-12-236">https://doi.org/10.1186/gb-2011-12-12-236</a>.</p>
</div>
<div>
<p>Deniz, Özgen, Jennifer M. Frost, and Miguel R. Branco. 2019. “Regulation of transposable elements by DNA modifications.” <em>Nature Reviews Genetics</em> 20 (7): 417–31. <a href="https://doi.org/10.1038/s41576-019-0106-6">https://doi.org/10.1038/s41576-019-0106-6</a>.</p>
</div>
<div>
<p>Denny, Joshua C, Lisa Bastarache, Marylyn D Ritchie, Robert J Carroll, Raquel Zink, Jonathan D Mosley, Julie R Field, et al. 2013. “Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data.” <em>Nature Biotechnology</em> 31 (12): 1102–11. <a href="https://doi.org/10.1038/nbt.2749">https://doi.org/10.1038/nbt.2749</a>.</p>
</div>
<div>
<p>Derrien, Thomas, Jordi Estellé, Santiago Marco Sola, David G. Knowles, Emanuele Raineri, Roderic Guigó, and Paolo Ribeca. 2012. “Fast Computation and Applications of Genome Mappability.” Edited by Christos A. Ouzounis. <em>PLoS ONE</em> 7 (1): e30377. <a href="https://doi.org/10.1371/journal.pone.0030377">https://doi.org/10.1371/journal.pone.0030377</a>.</p>
</div>
<div>
<p>Dewannieux, Marie, Cécile Esnault, and Thierry Heidmann. 2003. “LINE-mediated retrotransposition of marked Alu sequences.” <em>Nature Genetics</em> 35 (1): 41–48. <a href="https://doi.org/10.1038/ng1223">https://doi.org/10.1038/ng1223</a>.</p>
</div>
<div>
<p>Dhahbi, Joseph M, Stephen R Spindler, Hani Atamna, Amy Yamakawa, Dario Boffelli, Patricia Mote, and David IK Martin. 2013. “5’ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction.” <em>BMC Genomics</em> 14 (1): 298. <a href="https://doi.org/10.1186/1471-2164-14-298">https://doi.org/10.1186/1471-2164-14-298</a>.</p>
</div>
<div>
<p>Dittmar, Kimberly A., Jeffrey M. Goodenbour, and Tao Pan. 2006. “Tissue-specific differences in human transfer RNA expression.” <em>PLoS Genetics</em> 2 (12): e221. <a href="https://doi.org/10.1371/journal.pgen.0020221">https://doi.org/10.1371/journal.pgen.0020221</a>.</p>
</div>
<div>
<p>Dogan, Meeshanthini V., Steven R. H. Beach, and Robert A. Philibert. 2017. “Genetically contextual effects of smoking on genome wide DNA methylation.” <em>American Journal of Medical Genetics Part B: Neuropsychiatric Genetics</em> 174 (6): 595–607. <a href="https://doi.org/10.1002/ajmg.b.32565">https://doi.org/10.1002/ajmg.b.32565</a>.</p>
</div>
<div>
<p>Down, Thomas A, Vardhman K Rakyan, Daniel J Turner, Paul Flicek, Heng Li, Eugene Kulesha, Stefan Gräf, et al. 2008. “A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis.” <em>Nature Biotechnology</em> 26 (7): 779–85. <a href="https://doi.org/10.1038/nbt1414">https://doi.org/10.1038/nbt1414</a>.</p>
</div>
<div>
<p>Du, Qian, Saul A. Bert, Nicola J. Armstrong, C. Elizabeth Caldon, Jenny Z. Song, Shalima S. Nair, Cathryn M. Gould, et al. 2019. “Replication timing and epigenome remodelling are associated with the nature of chromosomal rearrangements in cancer.” <em>Nature Communications</em> 10 (1): 416. <a href="https://doi.org/10.1038/s41467-019-08302-1">https://doi.org/10.1038/s41467-019-08302-1</a>.</p>
</div>
<div>
<p>Duncan, B K, and J H Miller. 1980. “Mutagenic deamination of cytosine residues in DNA.” <em>Nature</em> 287 (5782): 560–1. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6999365">http://www.ncbi.nlm.nih.gov/pubmed/6999365</a>.</p>
</div>
<div>
<p>Eckhardt, Florian, Joern Lewin, Rene Cortese, Vardhman K Rakyan, John Attwood, Matthias Burger, John Burton, et al. 2006. “DNA methylation profiling of human chromosomes 6, 20 and 22.” <em>Nature Genetics</em> 38 (12): 1378–85. <a href="https://doi.org/10.1038/ng1909">https://doi.org/10.1038/ng1909</a>.</p>
</div>
<div>
<p>Ehrlich, Melanie. 2019. “DNA hypermethylation in disease: mechanisms and clinical relevance.” <em>Epigenetics</em> 14 (12): 1141–63. <a href="https://doi.org/10.1080/15592294.2019.1638701">https://doi.org/10.1080/15592294.2019.1638701</a>.</p>
</div>
<div>
<p>Ehrlich, Melanie, Miguel A. Gama-Sosa, Lan-Hsiang Huang, Rose Marie Midgett, Kenneth C. Kuo, Roy A. McCune, and Charles Gehrke. 1982. “Amount and distribution of 5-methylcytosine in human DNA from different types of tissues or cells.” <em>Nucleic Acids Research</em> 10 (8): 2709–21. <a href="https://doi.org/10.1093/nar/10.8.2709">https://doi.org/10.1093/nar/10.8.2709</a>.</p>
</div>
<div>
<p>Eigen, Manfred, B. Lindemann, Manfred Tietze, R Winkler-Oswatitsch, Andreas Dress, and A von Haeseler. 1989. “How old is the genetic code? Statistical geometry of tRNA provides an answer.” <em>Science</em> 244 (4905): 673–79. <a href="https://doi.org/10.1126/science.2497522">https://doi.org/10.1126/science.2497522</a>.</p>
</div>
<div>
<p>Engebretsen, Solveig, and Jon Bohlin. 2019. “Statistical predictions with glmnet.” <em>Clinical Epigenetics</em> 11 (1): 123. <a href="https://doi.org/10.1186/s13148-019-0730-1">https://doi.org/10.1186/s13148-019-0730-1</a>.</p>
</div>
<div>
<p>Ernst, Jason, Pouya Kheradpour, Tarjei S Mikkelsen, Noam Shoresh, Lucas D Ward, Charles B Epstein, Xiaolan Zhang, et al. 2011. “Mapping and analysis of chromatin state dynamics in nine human cell types.” <em>Nature</em> 473 (7345): 43–49. <a href="https://doi.org/10.1038/nature09906">https://doi.org/10.1038/nature09906</a>.</p>
</div>
<div>
<p>Esteller, M, J M Silva, G Dominguez, F Bonilla, X Matias-Guiu, E Lerma, E Bussaglia, et al. 2000. “Promoter hypermethylation and BRCA1 inactivation in sporadic breast and ovarian tumors.” <em>Journal of the National Cancer Institute</em> 92 (7): 564–9. <a href="https://doi.org/10.1093/jnci/92.7.564">https://doi.org/10.1093/jnci/92.7.564</a>.</p>
</div>
<div>
<p>Estève, Pierre-Olivier, Hang Gyeong Chin, Andrea Smallwood, George R. Feehery, Omkaram Gangisetty, Adam R. Karpf, Michael F. Carey, and Sriharsa Pradhan. 2006. “Direct interaction between DNMT1 and G9a coordinates DNA and histone methylation during replication.” <em>Genes &amp; Development</em> 20 (22): 3089–3103. <a href="https://doi.org/10.1101/gad.1463706">https://doi.org/10.1101/gad.1463706</a>.</p>
</div>
<div>
<p>Ewels, Philip, Måns Magnusson, Sverker Lundin, and Max Käller. 2016. “MultiQC: summarize analysis results for multiple tools and samples in a single report.” <em>Bioinformatics (Oxford, England)</em> 32 (19): 3047–8. <a href="https://doi.org/10.1093/bioinformatics/btw354">https://doi.org/10.1093/bioinformatics/btw354</a>.</p>
</div>
<div>
<p>Feinberg, Andrew P. 2018. “The Key Role of Epigenetics in Human Disease Prevention and Mitigation.” Edited by Dan L. Longo. <em>New England Journal of Medicine</em> 378 (14): 1323–34. <a href="https://doi.org/10.1056/NEJMra1402513">https://doi.org/10.1056/NEJMra1402513</a>.</p>
</div>
<div>
<p>Feinberg, Andrew P, and Benjamin Tycko. 2004. “The history of cancer epigenetics.” <em>Nature Reviews Cancer</em> 4 (2): 143–53. <a href="https://doi.org/10.1038/nrc1279">https://doi.org/10.1038/nrc1279</a>.</p>
</div>
<div>
<p>Ferrari, Roberto, Lara Isabel de Llobet Cucalon, Chiara Di Vona, François Le Dilly, Enrique Vidal, Antonios Lioutas, Javier Quilez Oliete, et al. 2019. “TFIIIC Binding to Alu Elements Controls Gene Expression via Chromatin Looping and Histone Acetylation.” <em>Molecular Cell</em>, November, 1–13. <a href="https://doi.org/10.1016/j.molcel.2019.10.020">https://doi.org/10.1016/j.molcel.2019.10.020</a>.</p>
</div>
<div>
<p>Field, Adam E., Neil A. Robertson, Tina Wang, Aaron Havas, Trey Ideker, and Peter D. Adams. 2018. “DNA Methylation Clocks in Aging: Categories, Causes, and Consequences.” <em>Molecular Cell</em> 71 (6): 882–95. <a href="https://doi.org/10.1016/j.molcel.2018.08.008">https://doi.org/10.1016/j.molcel.2018.08.008</a>.</p>
</div>
<div>
<p>Filer, Danny, Maximillian A. Thompson, Vakil Takhaveev, Adam J. Dobson, Ilektra Kotronaki, James W. M. Green, Matthias Heinemann, Jennifer M. A. Tullet, and Nazif Alic. 2017. “RNA polymerase III limits longevity downstream of TORC1.” <em>Nature</em> 552 (7684): 263–67. <a href="https://doi.org/10.1038/nature25007">https://doi.org/10.1038/nature25007</a>.</p>
</div>
<div>
<p>Fleming, Douglas M., and Alex J. Elliot. 2005. “The impact of influenza on the health and health care utilisation of elderly people.” <em>Vaccine</em> 23 (SUPPL. 1): S1. <a href="https://doi.org/10.1016/j.vaccine.2005.04.018">https://doi.org/10.1016/j.vaccine.2005.04.018</a>.</p>
</div>
<div>
<p>Florath, Ines, Katja Butterbach, H. Muller, M. Bewerunge-Hudler, and Hermann Brenner. 2014. “Cross-sectional and longitudinal changes in DNA methylation with age: an epigenome-wide analysis revealing over 60 novel age-associated CpG sites.” <em>Human Molecular Genetics</em> 23 (5): 1186–1201. <a href="https://doi.org/10.1093/hmg/ddt531">https://doi.org/10.1093/hmg/ddt531</a>.</p>
</div>
<div>
<p>Fortin, Jean-Philippe, Aurélie Labbe, Mathieu Lemire, Brent W. Zanke, Thomas J. Hudson, Elana J. Fertig, Celia M. T. Greenwood, and Kasper D. Hansen. 2014. “Functional normalization of 450k methylation array data improves replication in large cancer studies.” <em>Genome Biology</em> 15 (11): 503. <a href="https://doi.org/10.1186/s13059-014-0503-2">https://doi.org/10.1186/s13059-014-0503-2</a>.</p>
</div>
<div>
<p>Fortin, Jean-Philippe, Timothy J. Triche, and Kasper D. Hansen. 2016. “Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi.” <em>Bioinformatics</em>, December, btw691. <a href="https://doi.org/10.1093/bioinformatics/btw691">https://doi.org/10.1093/bioinformatics/btw691</a>.</p>
</div>
<div>
<p>Fraga, Mario F, E. Ballestar, Maria F Paz, Santiago Ropero, Fernando Setien, Maria L Ballestar, D. Heine-Suner, et al. 2005. “From The Cover: Epigenetic differences arise during the lifetime of monozygotic twins.” <em>Proceedings of the National Academy of Sciences</em> 102 (30): 10604–9. <a href="https://doi.org/10.1073/pnas.0500398102">https://doi.org/10.1073/pnas.0500398102</a>.</p>
</div>
<div>
<p>Friedman, Jerome, Trevor Hastie, and Robert Tibshirani. 2010. “Regularization Paths for Generalized Linear Models via Coordinate Descent.” <em>Journal of Statistical Software</em> 33 (1): 7–10. <a href="https://doi.org/10.18637/jss.v033.i01">https://doi.org/10.18637/jss.v033.i01</a>.</p>
</div>
<div>
<p>Gama-Sosa, Miguel A., Valerie A. Slagel, Ronald W. Trewyn, Ronald Oxenhandler, Kenneth C. Kuo, Charles W. Gehrke, and Melanie Ehrlich. 1983. “The 5-methylcytosine content of DNA from human tumors.” <em>Nucleic Acids Research</em> 11 (19): 6883–94. <a href="https://doi.org/10.1093/nar/11.19.6883">https://doi.org/10.1093/nar/11.19.6883</a>.</p>
</div>
<div>
<p>Garagnani, Paolo, Maria G. Bacalini, Chiara Pirazzini, Davide Gori, Cristina Giuliani, Daniela Mari, Anna M. Di Blasio, et al. 2012. “Methylation of ELOVL2 gene as a new epigenetic marker of age.” <em>Aging Cell</em> 11 (6): 1132–4. <a href="https://doi.org/10.1111/acel.12005">https://doi.org/10.1111/acel.12005</a>.</p>
</div>
<div>
<p>Geiger, Hartmut, Gerald de Haan, and M. Carolina Florian. 2013. “The ageing haematopoietic stem cell compartment.” <em>Nature Reviews Immunology</em> 13 (5): 376–89. <a href="https://doi.org/10.1038/nri3433">https://doi.org/10.1038/nri3433</a>.</p>
</div>
<div>
<p>Gervin, Kristina, Christian Magnus Page, Hans Christian D. Aass, Michelle A. Jansen, Heidi Elisabeth Fjeldstad, Bettina Kulle Andreassen, Liesbeth Duijts, et al. 2016. “Cell type specific DNA methylation in cord blood: A 450K-reference data set and cell count-based validation of estimated cell type composition.” <em>Epigenetics</em> 11 (9): 690–98. <a href="https://doi.org/10.1080/15592294.2016.1214782">https://doi.org/10.1080/15592294.2016.1214782</a>.</p>
</div>
<div>
<p>Geslain, Renaud, and Tao Pan. 2010. “Functional Analysis of Human tRNA Isodecoders.” <em>Journal of Molecular Biology</em> 396 (3): 821–31. <a href="https://doi.org/10.1016/j.jmb.2009.12.018">https://doi.org/10.1016/j.jmb.2009.12.018</a>.</p>
</div>
<div>
<p>Gibson, Jude, Tom C Russ, Toni-Kim Clarke, David M Howard, Robert F Hillary, Kathryn L Evans, Rosie M Walker, et al. 2019. “A meta-analysis of genome-wide association studies of epigenetic age acceleration.” Edited by John M. Greally. <em>PLOS Genetics</em> 15 (11): e1008104. <a href="https://doi.org/10.1371/journal.pgen.1008104">https://doi.org/10.1371/journal.pgen.1008104</a>.</p>
</div>
<div>
<p>Gingold, Hila, Orna Dahan, and Yitzhak Pilpel. 2012. “Dynamic changes in translational efficiency are deduced from codon usage of the transcriptome.” <em>Nucleic Acids Research</em> 40 (20): 10053–63. <a href="https://doi.org/10.1093/nar/gks772">https://doi.org/10.1093/nar/gks772</a>.</p>
</div>
<div>
<p>Gingold, Hila, Disa Tehler, Nanna R. Christoffersen, Morten M. Nielsen, Fazila Asmar, Susanne M. Kooistra, Nicolaj S. Christophersen, et al. 2014. “A Dual Program for Translation Regulation in Cellular Proliferation and Differentiation.” <em>Cell</em> 158 (6): 1281–92. <a href="https://doi.org/10.1016/j.cell.2014.08.011">https://doi.org/10.1016/j.cell.2014.08.011</a>.</p>
</div>
<div>
<p>Goede, Olivia M. de, Hamid R. Razzaghian, E. Magda Price, Meaghan J. Jones, Michael S. Kobor, Wendy P. Robinson, and Pascal M. Lavoie. 2015. “Nucleated red blood cells impact DNA methylation and expression analyses of cord blood hematopoietic cells.” <em>Clinical Epigenetics</em> 7 (1): 95. <a href="https://doi.org/10.1186/s13148-015-0129-6">https://doi.org/10.1186/s13148-015-0129-6</a>.</p>
</div>
<div>
<p>Goldman, Dana. 2016. “The Economic Promise of Delayed Aging.” <em>Cold Spring Harbor Perspectives in Medicine</em> 6 (2): a025072. <a href="https://doi.org/10.1101/cshperspect.a025072">https://doi.org/10.1101/cshperspect.a025072</a>.</p>
</div>
<div>
<p>Gomez-Roman, Natividad, Carla Grandori, Robert N. Eisenman, and Robert J. White. 2003. “Direct activation of RNA polymerase III transcription by c-Myc.” <em>Nature</em> 421 (6920): 290–94. <a href="https://doi.org/10.1038/nature01327">https://doi.org/10.1038/nature01327</a>.</p>
</div>
<div>
<p>Goodenbour, Jeffrey M., and Tao Pan. 2006. “Diversity of tRNA genes in eukaryotes.” <em>Nucleic Acids Research</em> 34 (21): 6137–46. <a href="https://doi.org/10.1093/nar/gkl725">https://doi.org/10.1093/nar/gkl725</a>.</p>
</div>
<div>
<p>Gregory, T. Ryan. 2005. “Synergy between sequence and size in large-scale genomics.” <em>Nature Reviews Genetics</em> 6 (9): 699–708. <a href="https://doi.org/10.1038/nrg1674">https://doi.org/10.1038/nrg1674</a>.</p>
</div>
<div>
<p>Grossman, Robert L., Allison P. Heath, Vincent Ferretti, Harold E. Varmus, Douglas R. Lowy, Warren A. Kibbe, and Louis M. Staudt. 2016. “Toward a Shared Vision for Cancer Genomic Data.” <em>New England Journal of Medicine</em> 375 (12): 1109–12. <a href="https://doi.org/10.1056/NEJMp1607591">https://doi.org/10.1056/NEJMp1607591</a>.</p>
</div>
<div>
<p>Gu, Zuguang, Roland Eils, and Matthias Schlesner. 2016. “Complex heatmaps reveal patterns and correlations in multidimensional genomic data.” <em>Bioinformatics</em> 32 (18): 2847–9. <a href="https://doi.org/10.1093/bioinformatics/btw313">https://doi.org/10.1093/bioinformatics/btw313</a>.</p>
</div>
<div>
<p>Guo, Junjie U, Yijing Su, Joo Heon Shin, Jaehoon Shin, Hongda Li, Bin Xie, Chun Zhong, et al. 2014. “Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain.” <em>Nature Neuroscience</em> 17 (2): 215–22. <a href="https://doi.org/10.1038/nn.3607">https://doi.org/10.1038/nn.3607</a>.</p>
</div>
<div>
<p>Haerter, Jan O., Cecilia Lövkvist, Ian B. Dodd, and Kim Sneppen. 2014. “Collaboration between CpG sites is needed for stable somatic inheritance of DNA methylation states.” <em>Nucleic Acids Research</em> 42 (4): 2235–44. <a href="https://doi.org/10.1093/nar/gkt1235">https://doi.org/10.1093/nar/gkt1235</a>.</p>
</div>
<div>
<p>Hahn, Oliver, Sebastian Grönke, Thomas M. Stubbs, Gabriella Ficz, Oliver Hendrich, Felix Krueger, Simon Andrews, et al. 2017. “Dietary restriction protects from age-associated DNA methylation and induces epigenetic reprogramming of lipid metabolism.” <em>Genome Biology</em> 18 (1): 56. <a href="https://doi.org/10.1186/s13059-017-1187-1">https://doi.org/10.1186/s13059-017-1187-1</a>.</p>
</div>
<div>
<p>Hannum, Gregory, Justin Guinney, Ling Zhao, Li Zhang, Guy Hughes, SriniVas Sadda, Brandy Klotzle, et al. 2013. “Genome-wide methylation profiles reveal quantitative views of human aging rates.” <em>Molecular Cell</em> 49 (2): 359–67. <a href="https://doi.org/10.1016/j.molcel.2012.10.016">https://doi.org/10.1016/j.molcel.2012.10.016</a>.</p>
</div>
<div>
<p>Hansen, Malene, Stefan Taubert, Douglas Crawford, Nataliya Libina, Seung-Jae Lee, and Cynthia Kenyon. 2007. “Lifespan extension by conditions that inhibit translation in Caenorhabditis elegans.” <em>Aging Cell</em> 6 (1): 95–110. <a href="https://doi.org/10.1111/j.1474-9726.2006.00267.x">https://doi.org/10.1111/j.1474-9726.2006.00267.x</a>.</p>
</div>
<div>
<p>Harrison, Alan, and Anne Parle-McDermott. 2011. “DNA Methylation: A Timeline of Methods and Applications.” <em>Frontiers in Genetics</em> 2 (3): 632–49. <a href="https://doi.org/10.3389/fgene.2011.00074">https://doi.org/10.3389/fgene.2011.00074</a>.</p>
</div>
<div>
<p>Harvey, Nicholas C, Kassim Javaid, Nicholas Bishop, Stephen Kennedy, Aris T Papageorghiou, Robert Fraser, Saurabh V Gandhi, Inez Schoenmakers, Ann Prentice, and Cyrus Cooper. 2012. “MAVIDOS Maternal Vitamin D Osteoporosis Study: study protocol for a randomized controlled trial. The MAVIDOS Study Group.” <em>Trials</em> 13 (1): 13. <a href="https://doi.org/10.1186/1745-6215-13-13">https://doi.org/10.1186/1745-6215-13-13</a>.</p>
</div>
<div>
<p>Harvey, Nicholas C., Allan Sheppard, Keith M. Godfrey, Cameron McLean, Emma Garratt, Georgia Ntani, Lucy Davies, et al. 2014. “Childhood Bone Mineral Content Is Associated With Methylation Status of the RXRA Promoter at Birth.” <em>Journal of Bone and Mineral Research</em> 29 (3): 600–607. <a href="https://doi.org/10.1002/jbmr.2056">https://doi.org/10.1002/jbmr.2056</a>.</p>
</div>
<div>
<p>Hashimshony, Tamar, Jianmin Zhang, Ilana Keshet, Michael Bustin, and Howard Cedar. 2003. “The role of DNA methylation in setting up chromatin structure during development.” <em>Nature Genetics</em> 34 (2): 187–92. <a href="https://doi.org/10.1038/ng1158">https://doi.org/10.1038/ng1158</a>.</p>
</div>
<div>
<p>Hatada, I, Y Hayashizaki, S Hirotsune, H Komatsubara, and T Mukai. 1991. “A genomic scanning method for higher organisms using restriction sites as landmarks.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 88 (21): 9523–7. <a href="https://doi.org/10.1073/pnas.88.21.9523">https://doi.org/10.1073/pnas.88.21.9523</a>.</p>
</div>
<div>
<p>Hayano, Motoshi, Jae-Hyun Yang, Michael S. Bonkowski, Joao A. Amorim, Jaime M. Ross, Giuseppe Coppotelli, Patrick Griffin, et al. 2019. “DNA Break-Induced Epigenetic Drift as a Cause of Mammalian Aging.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/808659">https://doi.org/10.1101/808659</a>.</p>
</div>
<div>
<p>Hayatsu, Hikoya, Yusuke Wataya, Kazushige Kai, and Shigeru Iida. 1970. “Reaction of sodium bisulfite with uracil, cytosine, and their derivatives.” <em>Biochemistry</em> 9 (14): 2858–65. <a href="https://doi.org/10.1021/bi00816a016">https://doi.org/10.1021/bi00816a016</a>.</p>
</div>
<div>
<p>Hellman, Asaf, and Andrew Chess. 2007. “Gene body-specific methylation on the active X chromosome.” <em>Science</em> 315 (5815): 1141–3. <a href="https://doi.org/10.1126/science.1136352">https://doi.org/10.1126/science.1136352</a>.</p>
</div>
<div>
<p>Heyn, Holger, Ning Li, Humberto J Ferreira, Sebastian Moran, David G Pisano, Antonio Gomez, Javier Diez, et al. 2012. “Distinct DNA methylomes of newborns and centenarians.” <em>Proceedings of the National Academy of Sciences</em> 109 (26): 10522–7. <a href="https://doi.org/10.1073/pnas.1120658109">https://doi.org/10.1073/pnas.1120658109</a>.</p>
</div>
<div>
<p>Hoffman, Michael M, Orion J Buske, Jie Wang, Zhiping Weng, Jeff a Bilmes, and William Stafford Noble. 2012. “Unsupervised pattern discovery in human chromatin structure through genomic segmentation.” <em>Nature Methods</em> 9 (5): 473–6. <a href="https://doi.org/10.1038/nmeth.1937">https://doi.org/10.1038/nmeth.1937</a>.</p>
</div>
<div>
<p>Holland, Michelle L, Robert Lowe, Paul W Caton, Carolina Gemma, Guillermo Carbajosa, Amy F Danson, Asha A M Carpenter, Elena Loche, Susan E Ozanne, and Vardhman K Rakyan. 2016. “Early-life nutrition modulates the epigenetic state of specific rDNA genetic variants in mice.” <em>Science</em> 353 (6298): 495–98. <a href="https://doi.org/10.1126/science.aaf7040">https://doi.org/10.1126/science.aaf7040</a>.</p>
</div>
<div>
<p>Horvath, Steve. 2013. “DNA methylation age of human tissues and cell types.” <em>Genome Biology</em> 14 (10): R115. <a href="https://doi.org/10.1186/gb-2013-14-10-r115">https://doi.org/10.1186/gb-2013-14-10-r115</a>.</p>
</div>
<div>
<p>Horvath, Steve, Junko Oshima, George M. Martin, Ake T. Lu, Austin Quach, Howard Cohen, Sarah Felton, et al. 2018. “Epigenetic clock for skin and blood cells applied to Hutchinson Gilford Progeria Syndrome and ex vivo studies.” <em>Aging</em> 10 (7): 1758–75. <a href="https://doi.org/10.18632/aging.101508">https://doi.org/10.18632/aging.101508</a>.</p>
</div>
<div>
<p>Horvath, Steve, and Kenneth Raj. 2018. “DNA methylation-based biomarkers and the epigenetic clock theory of ageing.” <em>Nature Reviews Genetics</em> 19 (6): 371–84. <a href="https://doi.org/10.1038/s41576-018-0004-3">https://doi.org/10.1038/s41576-018-0004-3</a>.</p>
</div>
<div>
<p>Houseman, Eugene Andres, William P Accomando, Devin C Koestler, Brock C Christensen, Carmen J Marsit, Heather H Nelson, John K Wiencke, and Karl T Kelsey. 2012. “DNA methylation arrays as surrogate measures of cell mixture distribution.” <em>BMC Bioinformatics</em> 13 (1): 86. <a href="https://doi.org/10.1186/1471-2105-13-86">https://doi.org/10.1186/1471-2105-13-86</a>.</p>
</div>
<div>
<p>Huang, Shi-qiong, Bao Sun, Zong-ping Xiong, Yan Shu, Hong-hao Zhou, Wei Zhang, Jing Xiong, and Qing Li. 2018. “The dysregulation of tRNAs and tRNA derivatives in cancer.” <em>Journal of Experimental &amp; Clinical Cancer Research</em> 37 (1): 101. <a href="https://doi.org/10.1186/s13046-018-0745-z">https://doi.org/10.1186/s13046-018-0745-z</a>.</p>
</div>
<div>
<p>Huff, Jason T., and Daniel Zilberman. 2014. “Dnmt1-Independent CG Methylation Contributes to Nucleosome Positioning in Diverse Eukaryotes.” <em>Cell</em> 156 (6): 1286–97. <a href="https://doi.org/10.1016/j.cell.2014.01.029">https://doi.org/10.1016/j.cell.2014.01.029</a>.</p>
</div>
<div>
<p>Iben, James R., and Richard J. Maraia. 2014. “tRNA gene copy number variation in humans.” <em>Gene</em> 536 (2): 376–84. <a href="https://doi.org/10.1016/j.gene.2013.11.049">https://doi.org/10.1016/j.gene.2013.11.049</a>.</p>
</div>
<div>
<p>Illingworth, Robert S., Ulrike Gruenewald-Schneider, Shaun Webb, Alastair R W Kerr, Keith D. James, Daniel J. Turner, Colin Smith, David J. Harrison, Robert Andrews, and Adrian P. Bird. 2010. “Orphan CpG islands identify numerous conserved promoters in the mammalian genome.” Edited by Wolf Reik. <em>PLoS Genetics</em> 6 (9): e1001134. <a href="https://doi.org/10.1371/journal.pgen.1001134">https://doi.org/10.1371/journal.pgen.1001134</a>.</p>
</div>
<div>
<p>Irizarry, Rafael A, Christine Ladd-Acosta, Bo Wen, Zhijin Wu, Carolina Montano, Patrick Onyango, Hengmi Cui, et al. 2009. “The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.” <em>Nature Genetics</em> 41 (2): 178–86. <a href="https://doi.org/10.1038/ng.298">https://doi.org/10.1038/ng.298</a>.</p>
</div>
<div>
<p>Izzo, Franco, Stanley C. Lee, Asaf Poran, Ronan Chaligne, Federico Gaiti, Baptiste Gross, Rekha R. Murali, et al. 2020. “DNA methylation disruption reshapes the hematopoietic differentiation landscape.” <em>Nature Genetics</em> 52 (April). <a href="https://doi.org/10.1038/s41588-020-0595-4">https://doi.org/10.1038/s41588-020-0595-4</a>.</p>
</div>
<div>
<p>Jaffe, Andrew E, and Rafael A Irizarry. 2014. “Accounting for cellular heterogeneity is critical in epigenome-wide association studies.” <em>Genome Biology</em> 15 (2): R31. <a href="https://doi.org/10.1186/gb-2014-15-2-r31">https://doi.org/10.1186/gb-2014-15-2-r31</a>.</p>
</div>
<div>
<p>Jaiswal, Siddhartha, Pierre Fontanillas, Jason Flannick, Alisa Manning, Peter V. Grauman, Brenton G. Mar, R. Coleman Lindsley, et al. 2014. “Age-Related Clonal Hematopoiesis Associated with Adverse Outcomes.” <em>New England Journal of Medicine</em> 371 (26): 2488–98. <a href="https://doi.org/10.1056/NEJMoa1408617">https://doi.org/10.1056/NEJMoa1408617</a>.</p>
</div>
<div>
<p>Jang, Hyo Sik, Nakul M. Shah, Alan Y. Du, Zea Z. Dailey, Erica C. Pehrsson, Paula M. Godoy, David Zhang, et al. 2019. “Transposable elements drive widespread expression of oncogenes in human cancers.” <em>Nature Genetics</em> 51 (4): 611–17. <a href="https://doi.org/10.1038/s41588-019-0373-3">https://doi.org/10.1038/s41588-019-0373-3</a>.</p>
</div>
<div>
<p>Jenkinson, Garrett, Elisabet Pujadas, John Goutsias, and Andrew P Feinberg. 2017. “Potential energy landscapes identify the information-theoretic nature of the epigenome.” <em>Nature Genetics</em> 49 (5): 719–29. <a href="https://doi.org/10.1038/ng.3811">https://doi.org/10.1038/ng.3811</a>.</p>
</div>
<div>
<p>Jenuwein, T. 2001. “Translating the Histone Code.” <em>Science</em> 293 (5532): 1074–80. <a href="https://doi.org/10.1126/science.1063127">https://doi.org/10.1126/science.1063127</a>.</p>
</div>
<div>
<p>Jia, Da, Renata Z. Jurkowska, Xing Zhang, Albert Jeltsch, and Xiaodong Cheng. 2007. “Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation.” <em>Nature</em> 449 (7159): 248–51. <a href="https://doi.org/10.1038/nature06146">https://doi.org/10.1038/nature06146</a>.</p>
</div>
<div>
<p>Johnson, Nicholas D, Howard W Wiener, Alicia K Smith, Shota Nishitani, Devin M Absher, Donna K Arnett, Stella Aslibekyan, and Karen N Conneely. 2017. “Non-linear patterns in age-related DNA methylation may reflect CD4 + T cell differentiation.” <em>Epigenetics</em> 12 (6): 492–503. <a href="https://doi.org/10.1080/15592294.2017.1314419">https://doi.org/10.1080/15592294.2017.1314419</a>.</p>
</div>
<div>
<p>Johnson, Thomas E. 2006. “Recent results: Biomarkers of aging.” <em>Experimental Gerontology</em> 41 (12): 1243–6. <a href="https://doi.org/10.1016/j.exger.2006.09.006">https://doi.org/10.1016/j.exger.2006.09.006</a>.</p>
</div>
<div>
<p>Jones, Meaghan J., Sarah J. Goodman, and Michael S. Kobor. 2015. “DNA methylation and healthy human aging.” <em>Aging Cell</em> 14 (6): 924–32. <a href="https://doi.org/10.1111/acel.12349">https://doi.org/10.1111/acel.12349</a>.</p>
</div>
<div>
<p>Jordà, Mireia, Anna Díez-Villanueva, Izaskun Mallona, Berta Martín, Sergi Lois, Víctor Barrera, Manel Esteller, Tanya Vavouri, and Miguel A. Peinado. 2017. “The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells.” <em>Genome Research</em> 27 (1): 118–32. <a href="https://doi.org/10.1101/gr.207522.116">https://doi.org/10.1101/gr.207522.116</a>.</p>
</div>
<div>
<p>Joubert, Bonnie R., Janine F. Felix, Paul Yousefi, Kelly M. Bakulski, Allan C. Just, Carrie Breton, Sarah E. Reese, et al. 2016. “DNA Methylation in Newborns and Maternal Smoking in Pregnancy: Genome-wide Consortium Meta-analysis.” <em>The American Journal of Human Genetics</em> 98 (4): 680–96. <a href="https://doi.org/10.1016/j.ajhg.2016.02.019">https://doi.org/10.1016/j.ajhg.2016.02.019</a>.</p>
</div>
<div>
<p>Jung, Marc, and Gerd P Pfeifer. 2015. “Aging and DNA methylation.” <em>BMC Biology</em> 13 (1): 7. <a href="https://doi.org/10.1186/s12915-015-0118-4">https://doi.org/10.1186/s12915-015-0118-4</a>.</p>
</div>
<div>
<p>Juzenas, Simonas, Geetha Venkatesh, Matthias Hübenthal, Marc P. Hoeppner, Zhipei Gracie Du, Maren Paulsen, Philip Rosenstiel, et al. 2017. “A comprehensive, cell specific microRNA catalogue of human peripheral blood.” <em>Nucleic Acids Research</em> 45 (16): 9290–9301. <a href="https://doi.org/10.1093/nar/gkx706">https://doi.org/10.1093/nar/gkx706</a>.</p>
</div>
<div>
<p>Kaikkonen, Minna U., Michael T Y Lam, and Christopher K. Glass. 2011. “Non-coding RNAs as regulators of gene expression and epigenetics.” <em>Cardiovascular Research</em> 90 (3): 430–40. <a href="https://doi.org/10.1093/cvr/cvr097">https://doi.org/10.1093/cvr/cvr097</a>.</p>
</div>
<div>
<p>Kane, Alice E., and David A. Sinclair. 2019. “Epigenetic changes during aging and their reprogramming potential.” <em>Critical Reviews in Biochemistry and Molecular Biology</em> 54 (1): 61–83. <a href="https://doi.org/10.1080/10409238.2019.1570075">https://doi.org/10.1080/10409238.2019.1570075</a>.</p>
</div>
<div>
<p>Kazazian, Haig H., and John V. Moran. 2017. “Mobile DNA in health and disease.” <em>New England Journal of Medicine</em> 377 (4): 361–70. <a href="https://doi.org/10.1056/NEJMra1510092">https://doi.org/10.1056/NEJMra1510092</a>.</p>
</div>
<div>
<p>Keam, Simon P., Paul E. Young, Alexandra L. McCorkindale, Thurston H. Y. Dang, Jennifer L. Clancy, David T. Humphreys, Thomas Preiss, et al. 2014. “The human Piwi protein Hiwi2 associates with tRNA-derived piRNAs in somatic cells.” <em>Nucleic Acids Research</em> 42 (14): 8984–95. <a href="https://doi.org/10.1093/nar/gku620">https://doi.org/10.1093/nar/gku620</a>.</p>
</div>
<div>
<p>Kelly, Theresa K, Yaping Liu, Fides D Lay, Gangning Liang, Benjamin P Berman, and Peter a Jones. 2012. “Genome-wide mapping of nucleosome positioning and DNA methylation within individual DNA molecules.” <em>Genome Research</em> 22 (12): 2497–2506. <a href="https://doi.org/10.1101/gr.143008.112">https://doi.org/10.1101/gr.143008.112</a>.</p>
</div>
<div>
<p>Kennedy, Brian K., and Dudley W. Lamming. 2016. “The Mechanistic Target of Rapamycin: The Grand ConducTOR of Metabolism and Aging.” <em>Cell Metabolism</em> 23 (6): 990–1003. <a href="https://doi.org/10.1016/j.cmet.2016.05.009">https://doi.org/10.1016/j.cmet.2016.05.009</a>.</p>
</div>
<div>
<p>Kenyon, Cynthia, Jean Chang, Erin Gensch, Adam Rudner, and Ramon Tabtiang. 1993. “A C. elegans mutant that lives twice as long as wild type.” <em>Nature</em> 366 (6454): 461–64. <a href="https://doi.org/10.1038/366461a0">https://doi.org/10.1038/366461a0</a>.</p>
</div>
<div>
<p>Kerkel, Kristi, Alexandra Spadola, Eric Yuan, Jolanta Kosek, Le Jiang, Eldad Hod, Kerry Li, et al. 2008. “Genomic surveys by methylation-sensitive SNP analysis identify sequence-dependent allele-specific DNA methylation.” <em>Nature Genetics</em> 40 (7): 904–8. <a href="https://doi.org/10.1038/ng.174">https://doi.org/10.1038/ng.174</a>.</p>
</div>
<div>
<p>Kim, Hak Kyun, Gabriele Fuchs, Shengchun Wang, Wei Wei, Yue Zhang, Hyesuk Park, Biswajoy Roy-Chaudhuri, et al. 2017. “A transfer-RNA-derived small RNA regulates ribosome biogenesis.” <em>Nature</em> 552 (7683): 57–62. <a href="https://doi.org/10.1038/nature25005">https://doi.org/10.1038/nature25005</a>.</p>
</div>
<div>
<p>Kip, Sertac N, and Emanuel E Strehler. 2004. “Vitamin D 3 upregulates plasma membrane Ca 2+ -ATPase expression and potentiates apico-basal Ca 2+ flux in MDCK cells.” <em>American Journal of Physiology-Renal Physiology</em> 286 (2): F363–F369. <a href="https://doi.org/10.1152/ajprenal.00076.2003">https://doi.org/10.1152/ajprenal.00076.2003</a>.</p>
</div>
<div>
<p>Kirchner, Sebastian, Zhiwei Cai, Robert Rauscher, Nicolai Kastelic, Melanie Anding, Andreas Czech, Bertrand Kleizen, et al. 2017. “Alteration of protein function by a silent polymorphism linked to tRNA abundance.” Edited by Laurence Hurst. <em>PLOS Biology</em> 15 (5): e2000779. <a href="https://doi.org/10.1371/journal.pbio.2000779">https://doi.org/10.1371/journal.pbio.2000779</a>.</p>
</div>
<div>
<p>Knight, Anna K., Jeffrey M. Craig, Christiane Theda, Marie Bækvad-Hansen, Jonas Bybjerg-Grauholm, Christine S. Hansen, Mads V. Hollegaard, et al. 2016. “An epigenetic clock for gestational age at birth based on blood methylation data.” <em>Genome Biology</em> 17 (1): 206. <a href="https://doi.org/10.1186/s13059-016-1068-z">https://doi.org/10.1186/s13059-016-1068-z</a>.</p>
</div>
<div>
<p>Knudson, A G. 1971. “Mutation and cancer: statistical study of retinoblastoma.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 68 (4): 820–3. <a href="https://doi.org/10.1073/pnas.68.4.820">https://doi.org/10.1073/pnas.68.4.820</a>.</p>
</div>
<div>
<p>Koch, Carmen M., and Wolfgang Wagner. 2011. “Epigenetic-aging-signature to determine age in different tissues.” <em>Aging</em> 3 (10): 1018–27. <a href="https://doi.org/10.18632/aging.100395">https://doi.org/10.18632/aging.100395</a>.</p>
</div>
<div>
<p>Kolitz, Sarah E., and Jon R. Lorsch. 2010. “Eukaryotic initiator tRNA: Finely tuned and ready for action.” <em>FEBS Letters</em> 584 (2): 396–404. <a href="https://doi.org/10.1016/j.febslet.2009.11.047">https://doi.org/10.1016/j.febslet.2009.11.047</a>.</p>
</div>
<div>
<p>Kontis, Vasilis, James E. Bennett, Colin D. Mathers, Guangquan Li, Kyle Foreman, and Majid Ezzati. 2017. “Future life expectancy in 35 industrialised countries: projections with a Bayesian model ensemble.” <em>The Lancet</em> 389 (10076): 1323–35. <a href="https://doi.org/10.1016/S0140-6736(16)32381-9">https://doi.org/10.1016/S0140-6736(16)32381-9</a>.</p>
</div>
<div>
<p>Krishnan, Preethi, Sunita Ghosh, Bo Wang, Mieke Heyns, Dongping Li, John R. Mackey, Olga Kovalchuk, and Sambasivarao Damaraju. 2016. “Genome-wide profiling of transfer RNAs and their role as novel prognostic markers for breast cancer.” <em>Scientific Reports</em> 6 (1): 32843. <a href="https://doi.org/10.1038/srep32843">https://doi.org/10.1038/srep32843</a>.</p>
</div>
<div>
<p>Krueger, Felix. 2015. “Trim Galore.” <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim%7B\_%7Dgalore/">https://www.bioinformatics.babraham.ac.uk/projects/trim{\_}galore/</a>.</p>
</div>
<div>
<p>Krueger, Felix, and Simon R. Andrews. 2011. “Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications.” <em>Bioinformatics</em> 27 (11): 1571–2. <a href="https://doi.org/10.1093/bioinformatics/btr167">https://doi.org/10.1093/bioinformatics/btr167</a>.</p>
</div>
<div>
<p>Kunkel, Thomas A. 2004. “DNA Replication Fidelity.” <em>Journal of Biological Chemistry</em> 279 (17): 16895–8. <a href="https://doi.org/10.1074/jbc.R400006200">https://doi.org/10.1074/jbc.R400006200</a>.</p>
</div>
<div>
<p>Laird, Charles D, Nicole D Pleasant, Aaron D Clark, Jessica L Sneeden, K M Anwarul Hassan, Nathan C Manley, Jay C Vary, Todd Morgan, R Scott Hansen, and R. Stoger. 2004. “Hairpin-bisulfite PCR: Assessing epigenetic methylation patterns on complementary strands of individual DNA molecules.” <em>Proceedings of the National Academy of Sciences</em> 101 (1): 204–9. <a href="https://doi.org/10.1073/pnas.2536758100">https://doi.org/10.1073/pnas.2536758100</a>.</p>
</div>
<div>
<p>Lander, E S, L M Linton, B Birren, C Nusbaum, M C Zody, J Baldwin, K Devon, et al. 2001. “Initial sequencing and analysis of the human genome.” <em>Nature</em> 409 (6822): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.</p>
</div>
<div>
<p>Langmead, Ben, and Steven L Salzberg. 2012. “Fast gapped-read alignment with Bowtie 2.” <em>Nature Methods</em> 9 (4): 357–9. <a href="https://doi.org/10.1038/nmeth.1923">https://doi.org/10.1038/nmeth.1923</a>.</p>
</div>
<div>
<p>Lappalainen, Tuuli, and John M. Greally. 2017. “Associating cellular epigenetic models with human phenotypes.” <em>Nature Reviews Genetics</em> 18 (7): 441–51. <a href="https://doi.org/10.1038/nrg.2017.32">https://doi.org/10.1038/nrg.2017.32</a>.</p>
</div>
<div>
<p>Lee, Yong Sun, Yoshiyuki Shibata, Ankit Malhotra, and Anindya Dutta. 2009. “A novel class of small RNAs: tRNA-derived RNA fragments (tRFs).” <em>Genes &amp; Development</em> 23 (22): 2639–49. <a href="https://doi.org/10.1101/gad.1837609">https://doi.org/10.1101/gad.1837609</a>.</p>
</div>
<div>
<p>Leek, Jeffrey T., and John D. Storey. 2007. “Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis.” <em>PLoS Genetics</em> 3 (9): e161. <a href="https://doi.org/10.1371/journal.pgen.0030161">https://doi.org/10.1371/journal.pgen.0030161</a>.</p>
</div>
<div>
<p>Levine, Morgan E, Ake T Lu, Austin Quach, Brian H Chen, Themistocles L Assimes, Stefania Bandinelli, Lifang Hou, et al. 2018. “An epigenetic biomarker of aging for lifespan and healthspan.” <em>Aging</em> 10 (4): 573–91. <a href="https://doi.org/10.18632/aging.101414">https://doi.org/10.18632/aging.101414</a>.</p>
</div>
<div>
<p>Li, H., B. Handsaker, A. Wysoker, T. Fennell, J. Ruan, N. Homer, G. Marth, G. Abecasis, and R. Durbin. 2009. “The Sequence Alignment/Map format and SAMtools.” <em>Bioinformatics</em> 25 (16): 2078–9. <a href="https://doi.org/10.1093/bioinformatics/btp352">https://doi.org/10.1093/bioinformatics/btp352</a>.</p>
</div>
<div>
<p>Li, Jingqiu, Jie Yang, Ping Zhou, Yanping Le, Chengwei Zhou, Shaomin Wang, Dazhi Xu, Hui-Kuan Lin, and Zhaohui Gong. 2015. “Circular RNAs in cancer: novel insights into origins, properties, functions and implications.” <em>American Journal of Cancer Research</em> 5 (2): 472–80. <a href="http://www.ncbi.nlm.nih.gov/pubmed/25973291">http://www.ncbi.nlm.nih.gov/pubmed/25973291</a>.</p>
</div>
<div>
<p>Li, Long-Cheng, and Rajvir Dahiya. 2002. “MethPrimer: designing primers for methylation PCRs.” <em>Bioinformatics (Oxford, England)</em> 18 (11): 1427–31. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12424112">http://www.ncbi.nlm.nih.gov/pubmed/12424112</a>.</p>
</div>
<div>
<p>Li, Siqi, Xiaoliang Shi, Muxiong Chen, Ningqin Xu, Desen Sun, Rongpan Bai, Haiyan Chen, Kefeng Ding, Jinghao Sheng, and Zhengping Xu. 2019. “Angiogenin promotes colorectal cancer metastasis via tiRNA production.” <em>International Journal of Cancer</em>, March, ijc.32245. <a href="https://doi.org/10.1002/ijc.32245">https://doi.org/10.1002/ijc.32245</a>.</p>
</div>
<div>
<p>Li, Siqi, Zhengping Xu, and Jinghao Sheng. 2018. “tRNA-Derived Small RNA: A Novel Regulatory Small Non-Coding RNA.” <em>Genes</em> 9 (5): 246. <a href="https://doi.org/10.3390/genes9050246">https://doi.org/10.3390/genes9050246</a>.</p>
</div>
<div>
<p>Lienert, Florian, Christiane Wirbelauer, Indrani Som, Ann Dean, Fabio Mohn, and Dirk Schübeler. 2011. “Identification of genetic elements that autonomously determine DNA methylation states.” <em>Nature Genetics</em> 43 (11): 1091–7. <a href="https://doi.org/10.1038/ng.946">https://doi.org/10.1038/ng.946</a>.</p>
</div>
<div>
<p>Lienhard, Matthias, Christina Grimm, Markus Morkel, Ralf Herwig, and Lukas Chavez. 2014. “MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments.” <em>Bioinformatics</em> 30 (2): 284–86. <a href="https://doi.org/10.1093/bioinformatics/btt650">https://doi.org/10.1093/bioinformatics/btt650</a>.</p>
</div>
<div>
<p>Lillycrop, Karen A., Emma S. Phillips, Christopher Torrens, Mark A. Hanson, Alan A. Jackson, and Graham C. Burdge. 2008. “Feeding pregnant rats a protein-restricted diet persistently alters the methylation of specific cytosines in the hepatic PPAR<span class="math inline">\(\alpha\)</span> promoter of the offspring.” <em>British Journal of Nutrition</em> 100 (2): 278–82. <a href="https://doi.org/10.1017/S0007114507894438">https://doi.org/10.1017/S0007114507894438</a>.</p>
</div>
<div>
<p>Lin, Xinyi, Jane Yi Lin Tan, Ai Ling Teh, Ives Yubin Lim, Samantha J. Liew, Julia L. MacIsaac, Yap Seng Chong, et al. 2018. “Cell type-specific DNA methylation in neonatal cord tissue and cord blood: a 850K-reference panel and comparison of cell types.” <em>Epigenetics</em> 13 (9): 941–58. <a href="https://doi.org/10.1080/15592294.2018.1522929">https://doi.org/10.1080/15592294.2018.1522929</a>.</p>
</div>
<div>
<p>Lister, Ryan, Mattia Pelizzola, Robert H Dowen, R David Hawkins, Gary Hon, Julian Tonti-Filippini, Joseph R Nery, et al. 2009. “Human DNA methylomes at base resolution show widespread epigenomic differences.” <em>Nature</em> 462 (7271): 315–22. <a href="https://doi.org/10.1038/nature08514">https://doi.org/10.1038/nature08514</a>.</p>
</div>
<div>
<p>Lodish, Harvey, Arnold Berk, S Lawrence Zipursky, Paul Matsudaira, David Baltimore, and James Darnell. 2000. <em>Molecular Cell Biology, 4th edition</em>. 4th ed. New York: W. H. Freeman. <a href="https://www.ncbi.nlm.nih.gov/books/NBK21475/">https://www.ncbi.nlm.nih.gov/books/NBK21475/</a>.</p>
</div>
<div>
<p>Loher, Phillipe, Aristeidis G. Telonis, and Isidore Rigoutsos. 2017. “MINTmap: fast and exhaustive profiling of nuclear and mitochondrial tRNA fragments from short RNA-seq data.” <em>Scientific Reports</em> 7 (1): 41184. <a href="https://doi.org/10.1038/srep41184">https://doi.org/10.1038/srep41184</a>.</p>
</div>
<div>
<p>Lowe, Robert, Carl Barton, Christopher A. Jenkins, Christina Ernst, Oliver Forman, Denise S. Fernandez-Twinn, Christoph Bock, et al. 2018. “Ageing-associated DNA methylation dynamics are a molecular readout of lifespan variation among mammalian species.” <em>Genome Biology</em> 19 (1): 22. <a href="https://doi.org/10.1186/s13059-018-1397-1">https://doi.org/10.1186/s13059-018-1397-1</a>.</p>
</div>
<div>
<p>Lowe, Robert, Amy F Danson, Vardhman K Rakyan, Selin Yildizoglu, Frédéric Saldmann, Melanie Viltard, Gérard Friedlander, and Chris G Faulkes. 2020. “DNA methylation clocks as a predictor for ageing and age estimation in naked mole-rats , Heterocephalus glaber” 12 (March): 1–13. <a href="https://doi.org/10.18632/aging.102892">https://doi.org/10.18632/aging.102892</a>.</p>
</div>
<div>
<p>Lowe, Todd M., and Patricia P. Chan. 2016. “tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes.” <em>Nucleic Acids Research</em> 44 (W1): W54–W57. <a href="https://doi.org/10.1093/nar/gkw413">https://doi.org/10.1093/nar/gkw413</a>.</p>
</div>
<div>
<p>López-Otín, Carlos, Maria A. Blasco, Linda Partridge, Manuel Serrano, and Guido Kroemer. 2013. “The hallmarks of aging.” <em>Cell</em> 153 (6): 1194–1217. <a href="https://doi.org/10.1016/j.cell.2013.05.039">https://doi.org/10.1016/j.cell.2013.05.039</a>.</p>
</div>
<div>
<p>Lu, Ake T, Eilis Hannon, Morgan E Levine, Eileen M Crimmins, Katie Lunnon, Jonathan Mill, Daniel H Geschwind, and Steve Horvath. 2017. “Genetic architecture of epigenetic and neuronal ageing rates in human brain regions.” <em>Nature Communications</em> 8 (May): 15353. <a href="https://doi.org/10.1038/ncomms15353">https://doi.org/10.1038/ncomms15353</a>.</p>
</div>
<div>
<p>Lu, Ake T., Austin Quach, James G. Wilson, Alex P. Reiner, Abraham Aviv, Kenneth Raj, Lifang Hou, et al. 2019. “DNA methylation GrimAge strongly predicts lifespan and healthspan.” <em>Aging</em> 11 (2): 303–27. <a href="https://doi.org/10.18632/aging.101684">https://doi.org/10.18632/aging.101684</a>.</p>
</div>
<div>
<p>Luo, Yanting, Xuemei Lu, and Hehuang Xie. 2014. “Dynamic Alu Methylation during Normal Development, Aging, and Tumorigenesis.” <em>BioMed Research International</em> 2014: 1–12. <a href="https://doi.org/10.1155/2014/784706">https://doi.org/10.1155/2014/784706</a>.</p>
</div>
<div>
<p>Lyko, Frank. 2017. “The DNA methyltransferase family: a versatile toolkit for epigenetic regulation.” <em>Nature Reviews Genetics</em> 19 (2): 81–92. <a href="https://doi.org/10.1038/nrg.2017.80">https://doi.org/10.1038/nrg.2017.80</a>.</p>
</div>
<div>
<p>Mahon, Pamela, Nicholas Harvey, Sarah Crozier, Hazel Inskip, Sian Robinson, Nigel Arden, Rama Swaminathan, Cyrus Cooper, and Keith Godfrey. 2010. “Low maternal vitamin D status and fetal bone development: Cohort study.” <em>Journal of Bone and Mineral Research</em> 25 (1): 14–19. <a href="https://doi.org/10.1359/jbmr.090701">https://doi.org/10.1359/jbmr.090701</a>.</p>
</div>
<div>
<p>Mange, François, Viviane Praz, Eugenia Migliavacca, Ian M Willis, Frédéric Schütz, and Nouria Hernandez. 2017. “Diurnal regulation of RNA polymerase III transcription is under the control of both the feeding–fasting response and the circadian clock.” <em>Genome Research</em> 27 (6): 973–84. <a href="https://doi.org/10.1101/gr.217521.116">https://doi.org/10.1101/gr.217521.116</a>.</p>
</div>
<div>
<p>Marioni, Riccardo E, Sonia Shah, Allan F McRae, Brian H Chen, Elena Colicino, Sarah E Harris, Jude Gibson, et al. 2015. “DNA methylation age of blood predicts all-cause mortality in later life.” <em>Genome Biology</em> 16 (1): 25. <a href="https://doi.org/10.1186/s13059-015-0584-6">https://doi.org/10.1186/s13059-015-0584-6</a>.</p>
</div>
<div>
<p>Martin, Marcel. 2011. “Cutadapt removes adapter sequences from high-throughput sequencing reads.” <em>EMBnet.journal</em> 17 (1): 10. <a href="https://doi.org/10.14806/ej.17.1.200">https://doi.org/10.14806/ej.17.1.200</a>.</p>
</div>
<div>
<p>Martinez, German, Sarah G. Choudury, and R. Keith Slotkin. 2017. “tRNA-derived small RNAs target transposable element transcripts.” <em>Nucleic Acids Research</em> 45 (9): 5142–52. <a href="https://doi.org/10.1093/nar/gkx103">https://doi.org/10.1093/nar/gkx103</a>.</p>
</div>
<div>
<p>Maunakea, Alika K, Raman P Nagarajan, Mikhail Bilenky, Tracy J Ballinger, Cletus D’Souza, Shaun D Fouse, Brett E Johnson, et al. 2010. “Conserved role of intragenic DNA methylation in regulating alternative promoters.” <em>Nature</em> 466 (7303): 253–7. <a href="https://doi.org/10.1038/nature09165">https://doi.org/10.1038/nature09165</a>.</p>
</div>
<div>
<p>McGregor, Kevin, Sasha Bernatsky, Ines Colmegna, Marie Hudson, Tomi Pastinen, Aurélie Labbe, and Celia M. T. Greenwood. 2016. “An evaluation of methods correcting for cell-type heterogeneity in DNA methylation studies.” <em>Genome Biology</em> 17 (1): 84. <a href="https://doi.org/10.1186/s13059-016-0935-y">https://doi.org/10.1186/s13059-016-0935-y</a>.</p>
</div>
<div>
<p>McLean, Cory Y, Dave Bristor, Michael Hiller, Shoa L Clarke, Bruce T Schaar, Craig B Lowe, Aaron M Wenger, and Gill Bejerano. 2010. “GREAT improves functional interpretation of cis-regulatory regions.” <em>Nature Biotechnology</em> 28 (5): 495–501. <a href="https://doi.org/10.1038/nbt.1630">https://doi.org/10.1038/nbt.1630</a>.</p>
</div>
<div>
<p>Medvedev, Zhores A. 1990. “An Attempt at a Rational Classification of Theories of Ageing.” <em>Biological Reviews</em> 65 (3): 375–98. <a href="https://doi.org/10.1111/j.1469-185X.1990.tb01428.x">https://doi.org/10.1111/j.1469-185X.1990.tb01428.x</a>.</p>
</div>
<div>
<p>Meissner, Alexander, Andreas Gnirke, George W Bell, Bernard Ramsahoye, Eric S Lander, and Rudolf Jaenisch. 2005. “Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis.” <em>Nucleic Acids Research</em> 33 (18): 5868–77. <a href="https://doi.org/10.1093/nar/gki901">https://doi.org/10.1093/nar/gki901</a>.</p>
</div>
<div>
<p>Meuleman, Wouter. 2019. “Epilogos.” <a href="https://epilogos.altius.org/%20https://github.com/Altius/epilogos">https://epilogos.altius.org/ https://github.com/Altius/epilogos</a>.</p>
</div>
<div>
<p>Min, Josine, Gibran Hemani, George Davey Smith, Caroline L Relton, and Matthew Suderman. 2017. “Meffil: efficient normalisation and analysis of very large DNA methylation samples.” <em>bioRxiv</em> 44 (0): 125963. <a href="https://doi.org/10.1101/125963">https://doi.org/10.1101/125963</a>.</p>
</div>
<div>
<p>Moayyeri, Alireza, Christopher J. Hammond, Ana M. Valdes, and Timothy D. Spector. 2013. “Cohort Profile: TwinsUK and Healthy Ageing Twin Study.” <em>International Journal of Epidemiology</em> 42 (1): 76–85. <a href="https://doi.org/10.1093/ije/dyr207">https://doi.org/10.1093/ije/dyr207</a>.</p>
</div>
<div>
<p>Mozhui, Khyobeni, and Ashutosh K. Pandey. 2017. “Conserved effect of aging on DNA methylation and association with EZH2 polycomb protein in mice and humans.” <em>Mechanisms of Ageing and Development</em> 162 (March): 27–37. <a href="https://doi.org/10.1016/j.mad.2017.02.006">https://doi.org/10.1016/j.mad.2017.02.006</a>.</p>
</div>
<div>
<p>Murawski, M., B. Szczesniak, T. Zoladek, A. K. Hopper, N. C. Martin, and M. Boguta. 1994. “maf1 mutation alters the subcellular localization of the Mod5 protein in yeast.” <em>Acta Biochimica Polonica</em> 41 (4): 441–8. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7732762">http://www.ncbi.nlm.nih.gov/pubmed/7732762</a>.</p>
</div>
<div>
<p>Müller, Carolin A., and Conrad A. Nieduszynski. 2017. “DNA replication timing influences gene expression level.” <em>The Journal of Cell Biology</em> 216 (7): 1907–14. <a href="https://doi.org/10.1083/jcb.201701061">https://doi.org/10.1083/jcb.201701061</a>.</p>
</div>
<div>
<p>Müller, Fabian, Michael Scherer, Yassen Assenov, Pavlo Lutsik, Jörn Walter, Thomas Lengauer, and Christoph Bock. 2019. “RnBeads 2.0: comprehensive analysis of DNA methylation data.” <em>Genome Biology</em> 20 (1): 55. <a href="https://doi.org/10.1186/s13059-019-1664-9">https://doi.org/10.1186/s13059-019-1664-9</a>.</p>
</div>
<div>
<p>Nazor, Kristopher L., Gulsah Altun, Candace Lynch, Ha Tran, Julie V. Harness, Ileana Slavin, Ibon Garitaonandia, et al. 2012. “Recurrent Variations in DNA Methylation in Human Pluripotent Stem Cells and Their Differentiated Derivatives.” <em>Cell Stem Cell</em> 10 (5): 620–34. <a href="https://doi.org/10.1016/j.stem.2012.02.013">https://doi.org/10.1016/j.stem.2012.02.013</a>.</p>
</div>
<div>
<p>Niccoli, Teresa, and Linda Partridge. 2012. “Ageing as a risk factor for disease.” <em>Current Biology</em> 22 (17): R741–R752. <a href="https://doi.org/10.1016/j.cub.2012.07.024">https://doi.org/10.1016/j.cub.2012.07.024</a>.</p>
</div>
<div>
<p>Noma, Ken-ichi, Hugh P. Cam, Richard J. Maraia, and Shiv I S Grewal. 2006. “A role for TFIIIC transcription factor complex in genome organization.” <em>Cell</em> 125 (5): 859–72. <a href="https://doi.org/10.1016/j.cell.2006.04.028">https://doi.org/10.1016/j.cell.2006.04.028</a>.</p>
</div>
<div>
<p>North, B. V., D. Curtis, and P. C. Sham. 2003. “A Note on the Calculation of Empirical P Values from Monte Carlo Procedures.” <em>The American Journal of Human Genetics</em> 72 (2): 498–99. <a href="https://doi.org/10.1086/346173">https://doi.org/10.1086/346173</a>.</p>
</div>
<div>
<p>Nwanaji-Enwerem, Jamaji C., Marc G. Weisskopf, and Andrea A. Baccarelli. 2018. “Multi-tissue DNA methylation age: Molecular relationships and perspectives for advancing biomarker utility.” <em>Ageing Research Reviews</em> 45 (March): 15–23. <a href="https://doi.org/10.1016/j.arr.2018.04.005">https://doi.org/10.1016/j.arr.2018.04.005</a>.</p>
</div>
<div>
<p>Pace, D. A. 2006. “Fixed metabolic costs for highly variable rates of protein synthesis in sea urchin embryos and larvae.” <em>Journal of Experimental Biology</em> 209 (1): 158–70. <a href="https://doi.org/10.1242/jeb.01962">https://doi.org/10.1242/jeb.01962</a>.</p>
</div>
<div>
<p>Parisien, Marc, Xiaoyun Wang, and Tao Pan. 2013. “Diversity of human tRNA genes from the 1000-genomes project.” <em>RNA Biology</em> 10 (12): 1853–67. <a href="https://doi.org/10.4161/rna.27361">https://doi.org/10.4161/rna.27361</a>.</p>
</div>
<div>
<p>Partridge, Linda, Joris Deelen, and P. Eline Slagboom. 2018. “Facing up to the global challenges of ageing.” <em>Nature</em> 561 (7721): 45–56. <a href="https://doi.org/10.1038/s41586-018-0457-8">https://doi.org/10.1038/s41586-018-0457-8</a>.</p>
</div>
<div>
<p>Pavon-Eternod, Mariana, Suzana Gomes, Marsha R Rosner, and Tao Pan. 2013. “Overexpression of initiator methionine tRNA leads to global reprogramming of tRNA expression and increased proliferation in human epithelial cells.” <em>RNA</em> 19 (4): 461–66. <a href="https://doi.org/10.1261/rna.037507.112">https://doi.org/10.1261/rna.037507.112</a>.</p>
</div>
<div>
<p>Payer, Lindsay M., Jared P. Steranka, Wan Rou Yang, Maria Kryatova, Sibyl Medabalimi, Daniel Ardeljan, Chunhong Liu, Jef D. Boeke, Dimitri Avramopoulos, and Kathleen H. Burns. 2017. “Structural variants caused by Alu insertions are associated with risks for many human diseases.” <em>Proceedings of the National Academy of Sciences</em> 114 (20): E3984–E3992. <a href="https://doi.org/10.1073/pnas.1704117114">https://doi.org/10.1073/pnas.1704117114</a>.</p>
</div>
<div>
<p>Petkovich, Daniel A., Dmitriy I. Podolskiy, Alexei V. Lobanov, Sang-Goo Lee, Richard A. Miller, and Vadim N. Gladyshev. 2017. “Using DNA Methylation Profiling to Evaluate Biological Age and Longevity Interventions.” <em>Cell Metabolism</em> 25 (4): 954–960.e6. <a href="https://doi.org/10.1016/j.cmet.2017.03.016">https://doi.org/10.1016/j.cmet.2017.03.016</a>.</p>
</div>
<div>
<p>Pfeifer, Gerd P, Swati Kadam, and Seung-Gi Jin. 2013. “5-Hydroxymethylcytosine and Its Potential Roles in Development and Cancer.” <em>Epigenetics &amp; Chromatin</em> 6 (1): 10. <a href="https://doi.org/10.1186/1756-8935-6-10">https://doi.org/10.1186/1756-8935-6-10</a>.</p>
</div>
<div>
<p>Pfeifer, G P, S D Steigerwald, R S Hansen, S M Gartler, and A D Riggs. 1990. “Polymerase chain reaction-aided genomic sequencing of an X chromosome-linked CpG island: methylation patterns suggest clonal inheritance, CpG site autonomy, and an explanation of activity state stability.” <em>Proceedings of the National Academy of Sciences</em> 87 (21): 8252–6. <a href="https://doi.org/10.1073/pnas.87.21.8252">https://doi.org/10.1073/pnas.87.21.8252</a>.</p>
</div>
<div>
<p>Pidsley, Ruth, Elena Zotenko, Timothy J. Peters, Mitchell G. Lawrence, Gail P. Risbridger, Peter Molloy, Susan Van Djik, Beverly Muhlhausler, Clare Stirzaker, and Susan J. Clark. 2016. “Critical evaluation of the Illumina MethylationEPIC BeadChip microarray for whole-genome DNA methylation profiling.” <em>Genome Biology</em> 17 (1): 208. <a href="https://doi.org/10.1186/s13059-016-1066-1">https://doi.org/10.1186/s13059-016-1066-1</a>.</p>
</div>
<div>
<p>Pliatsika, Venetia, Phillipe Loher, Rogan Magee, Aristeidis G. Telonis, Eric Londin, Megumi Shigematsu, Yohei Kirino, and Isidore Rigoutsos. 2018. “MINTbase v2.0: a comprehensive database for tRNA-derived fragments that includes nuclear and mitochondrial fragments from all The Cancer Genome Atlas projects.” <em>Nucleic Acids Research</em> 46 (D1): D152–D159. <a href="https://doi.org/10.1093/nar/gkx1075">https://doi.org/10.1093/nar/gkx1075</a>.</p>
</div>
<div>
<p>Pluta, Krzysztof, Olivier Lefebvre, Nancy C Martin, Wieslaw J Smagowicz, David R Stanford, Steven R Ellis, Anita K Hopper, Andre Sentenac, and Magdalena Boguta. 2001. “Maf1p, a Negative Effector of RNA Polymerase III in Saccharomyces cerevisiae.” <em>Molecular and Cellular Biology</em> 21 (15): 5031–40. <a href="https://doi.org/10.1128/MCB.21.15.5031-5040.2001">https://doi.org/10.1128/MCB.21.15.5031-5040.2001</a>.</p>
</div>
<div>
<p>Polak, Paz, and Eytan Domany. 2006. “Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes.” <em>BMC Genomics</em> 7 (June): 133. <a href="https://doi.org/10.1186/1471-2164-7-133">https://doi.org/10.1186/1471-2164-7-133</a>.</p>
</div>
<div>
<p>Powell, J R, and E N Moriyama. 1997. “Evolution of codon usage bias in Drosophila.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 94 (15): 7784–90. <a href="https://doi.org/10.1073/pnas.0404957101">https://doi.org/10.1073/pnas.0404957101</a>.</p>
</div>
<div>
<p>Quinlan, Aaron R., and Ira M. Hall. 2010. “BEDTools: a flexible suite of utilities for comparing genomic features.” <em>Bioinformatics</em> 26 (6): 841–42. <a href="https://doi.org/10.1093/bioinformatics/btq033">https://doi.org/10.1093/bioinformatics/btq033</a>.</p>
</div>
<div>
<p>Raab, Jesse R, Jonathan Chiu, Jingchun Zhu, Sol Katzman, Sreenivasulu Kurukuti, Paul A Wade, David Haussler, and Rohinton T Kamakaka. 2012. “Human tRNA genes function as chromatin insulators.” <em>The EMBO Journal</em> 31 (2): 330–50. <a href="https://doi.org/10.1038/emboj.2011.406">https://doi.org/10.1038/emboj.2011.406</a>.</p>
</div>
<div>
<p>Raiber, Eun-ang, Guillem Portella, Sergio Martínez Cuesta, Robyn Hardisty, Pierre Murat, Zhe Li, Mario Iurlaro, et al. 2018. “5-Formylcytosine organizes nucleosomes and forms Schiff base interactions with histones in mouse embryonic stem cells.” <em>Nature Chemistry</em> 10 (12): 1258–66. <a href="https://doi.org/10.1038/s41557-018-0149-x">https://doi.org/10.1038/s41557-018-0149-x</a>.</p>
</div>
<div>
<p>Rajendiran, Smrithi, Lee D. Gibbs, Timothy Van Treuren, David L. Klinkebiel, and Jamboor K. Vishwanatha. 2016. “MIEN1 is tightly regulated by SINE Alu methylation in its promoter.” <em>Oncotarget</em> 7 (40): 65307–19. <a href="https://doi.org/10.18632/oncotarget.11675">https://doi.org/10.18632/oncotarget.11675</a>.</p>
</div>
<div>
<p>Rakyan, Vardhman K., Thomas A. Down, Siarhei Maslau, Toby Andrew, Tsun Po Yang, Huriya Beyan, Pamela Whittaker, et al. 2010. “Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains.” <em>Genome Research</em> 20 (4): 434–39. <a href="https://doi.org/10.1101/gr.103101.109">https://doi.org/10.1101/gr.103101.109</a>.</p>
</div>
<div>
<p>Regev, Aviv, Sarah A. Teichmann, Eric S. Lander, Ido Amit, Christophe Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. “The Human Cell Atlas.” <em>eLife</em> 6 (December): 1–30. <a href="https://doi.org/10.7554/eLife.27041">https://doi.org/10.7554/eLife.27041</a>.</p>
</div>
<div>
<p>Reinius, Lovisa E., Nathalie Acevedo, Maaike Joerink, Göran Pershagen, Sven-Erik Dahlén, Dario Greco, Cilla Söderhäll, Annika Scheynius, and Juha Kere. 2012. “Differential DNA Methylation in Purified Human Blood Cells: Implications for Cell Lineage and Studies on Disease Susceptibility.” Edited by Angela H. Ting. <em>PLoS ONE</em> 7 (7): e41361. <a href="https://doi.org/10.1371/journal.pone.0041361">https://doi.org/10.1371/journal.pone.0041361</a>.</p>
</div>
<div>
<p>Rhoads, Anthony, and Kin Fai Au. 2015. “PacBio Sequencing and Its Applications.” <em>Genomics, Proteomics &amp; Bioinformatics</em> 13 (5): 278–89. <a href="https://doi.org/10.1016/j.gpb.2015.08.002">https://doi.org/10.1016/j.gpb.2015.08.002</a>.</p>
</div>
<div>
<p>Rideout, E. J., L. Marshall, and S. S. Grewal. 2012. “Drosophila RNA polymerase III repressor Maf1 controls body size and developmental timing by modulating tRNAiMet synthesis and systemic insulin signaling.” <em>Proceedings of the National Academy of Sciences</em> 109 (4): 1139–44. <a href="https://doi.org/10.1073/pnas.1113311109">https://doi.org/10.1073/pnas.1113311109</a>.</p>
</div>
<div>
<p>Riggs, Arthur D, and Zhenggang Xiong. 2004. “Methylation and epigenetic fidelity.” <em>Proceedings of the National Academy of Sciences</em> 101 (1): 4–5. <a href="https://doi.org/10.1073/pnas.0307781100">https://doi.org/10.1073/pnas.0307781100</a>.</p>
</div>
<div>
<p>Rimmelé, Pauline, Carolina L. Bigarella, Raymond Liang, Brigitte Izac, Rebeca Dieguez-Gonzalez, Gaetan Barbet, Michael Donovan, et al. 2014. “Aging-like phenotype and defective lineage specification in SIRT1-deleted hematopoietic stem and progenitor cells.” <em>Stem Cell Reports</em> 3 (1): 44–59. <a href="https://doi.org/10.1016/j.stemcr.2014.04.015">https://doi.org/10.1016/j.stemcr.2014.04.015</a>.</p>
</div>
<div>
<p>Ritchie, Matthew E, Belinda Phipson, Di Wu, Yifang Hu, Charity W Law, Wei Shi, and Gordon K Smyth. 2015. “limma powers differential expression analyses for RNA-sequencing and microarray studies.” <em>Nucleic Acids Research</em> 43 (7): e47–e47. <a href="https://doi.org/10.1093/nar/gkv007">https://doi.org/10.1093/nar/gkv007</a>.</p>
</div>
<div>
<p>Romanov, G A, and B F Vanyushin. 1981. “Methylation of reiterated sequences in mammalian DNAs. Effects of the tissue type, age, malignancy and hormonal induction.” <em>Biochimica et Biophysica Acta</em> 653 (2): 204–18. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7225396">http://www.ncbi.nlm.nih.gov/pubmed/7225396</a>.</p>
</div>
<div>
<p>Rose, Nathan R., and Robert J. Klose. 2014. “Understanding the relationship between DNA methylation and histone lysine methylation.” <em>Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</em> 1839 (12): 1362–72. <a href="https://doi.org/10.1016/j.bbagrm.2014.02.007">https://doi.org/10.1016/j.bbagrm.2014.02.007</a>.</p>
</div>
<div>
<p>Rudolph, Konrad L M, Bianca M. Schmitt, Diego Villar, Robert J. White, John C. Marioni, Claudia Kutter, and Duncan T. Odom. 2016. “Codon-Driven Translational Efficiency Is Stable across Diverse Mammalian Cell States.” Edited by Nicolas Galtier. <em>PLoS Genetics</em> 12 (5): e1006024. <a href="https://doi.org/10.1371/journal.pgen.1006024">https://doi.org/10.1371/journal.pgen.1006024</a>.</p>
</div>
<div>
<p>Russo, Vincenzo, Robert Martienssen, and Arthur D Riggs. 1996. <em>Epigenetic Mechanisms of Gene Regulation</em>. Cold Spring Harbor Laboratory Press. <a href="https://cshmonographs.org/index.php/monographs/issue/view/087969490.32">https://cshmonographs.org/index.php/monographs/issue/view/087969490.32</a>.</p>
</div>
<div>
<p>Sachs, Michael C, Abigail Shoben, Gregory P Levin, Cassy Robinson-Cohen, Andrew N Hoofnagle, Nancy Swords-Jenny, Joachim H Ix, et al. 2013. “Estimating mean annual 25-hydroxyvitamin D concentrations from single measurements: the Multi-Ethnic Study of Atherosclerosis.” <em>The American Journal of Clinical Nutrition</em> 97 (6): 1243–51. <a href="https://doi.org/10.3945/ajcn.112.054502">https://doi.org/10.3945/ajcn.112.054502</a>.</p>
</div>
<div>
<p>Sagi, Dror, Roni Rak, Hila Gingold, Idan Adir, Gadi Maayan, Orna Dahan, Limor Broday, Yitzhak Pilpel, and Oded Rechavi. 2016. “Tissue- and Time-Specific Expression of Otherwise Identical tRNA Genes.” Edited by Andrew D. Chisholm. <em>PLOS Genetics</em> 12 (8): e1006264. <a href="https://doi.org/10.1371/journal.pgen.1006264">https://doi.org/10.1371/journal.pgen.1006264</a>.</p>
</div>
<div>
<p>Sandoval, Juan, Holger Heyn, Sebastian Moran, Jordi Serra-Musach, Miguel A Pujana, Marina Bibikova, and Manel Esteller. 2011a. “Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome.” <em>Epigenetics</em> 6 (6): 692–702. <a href="https://doi.org/10.2217/epi.15.114">https://doi.org/10.2217/epi.15.114</a>.</p>
</div>
<div>
<p>Sandoval, Juan, Holger Heyn, Sebastian Moran, Jordi Serra-Musach, Miguel A. Pujana, Marina Bibikova, and Manel Esteller. 2011b. “Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome.” <em>Epigenetics</em> 6 (6): 692–702. <a href="https://doi.org/10.4161/epi.6.6.16196">https://doi.org/10.4161/epi.6.6.16196</a>.</p>
</div>
<div>
<p>Saxonov, Serge, Paul Berg, and Douglas L Brutlag. 2006. “A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters.” <em>Proceedings of the National Academy of Sciences</em> 103 (5): 1412–7. <a href="https://doi.org/10.1073/pnas.0510310103">https://doi.org/10.1073/pnas.0510310103</a>.</p>
</div>
<div>
<p>Schalkwyk, Leonard C., Emma L. Meaburn, Rebecca Smith, Emma L. Dempster, Aaron R. Jeffries, Matthew N. Davies, Robert Plomin, and Jonathan Mill. 2010. “Allelic skewing of DNA methylation is widespread across the genome.” <em>American Journal of Human Genetics</em> 86 (2): 196–212. <a href="https://doi.org/10.1016/j.ajhg.2010.01.014">https://doi.org/10.1016/j.ajhg.2010.01.014</a>.</p>
</div>
<div>
<p>Schimmel, Paul. 2018. “The emerging complexity of the tRNA world: mammalian tRNAs beyond protein synthesis.” <em>Nature Reviews. Molecular Cell Biology</em> 19 (1): 45–58. <a href="https://doi.org/10.1038/nrm.2017.77">https://doi.org/10.1038/nrm.2017.77</a>.</p>
</div>
<div>
<p>Schmitt, Bianca M, Konrad L M Rudolph, Panagiota Karagianni, Nuno A Fonseca, Robert J White, Iannis Talianidis, Duncan T Odom, John C Marioni, and Claudia Kutter. 2014. “High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA-tRNA interface.” <em>Genome Research</em> 24 (11): 1797–1807. <a href="https://doi.org/10.1101/gr.176784.114">https://doi.org/10.1101/gr.176784.114</a>.</p>
</div>
<div>
<p>Schmitt, Bianca M., Konrad L. M. Rudolph, Panagiota Karagianni, Nuno A. Fonseca, Robert J. White, Iannis Talianidis, Duncan T. Odom, John C. Marioni, and Claudia Kutter. 2014. “High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA–tRNA interface.” <em>Genome Research</em> 24 (11): 1797–1807. <a href="https://doi.org/10.1101/gr.176784.114">https://doi.org/10.1101/gr.176784.114</a>.</p>
</div>
<div>
<p>Schramm, Laura. 2002. “Recruitment of RNA polymerase III to its target promoters.” <em>Genes &amp; Development</em> 16 (20): 2593–2620. <a href="https://doi.org/10.1101/gad.1018902">https://doi.org/10.1101/gad.1018902</a>.</p>
</div>
<div>
<p>Schuettengruber, Bernd, Daniel Chourrout, Michel Vervoort, Benjamin Leblanc, and Giacomo Cavalli. 2007. “Genome Regulation by Polycomb and Trithorax Proteins.” <em>Cell</em> 128 (4): 735–45. <a href="https://doi.org/10.1016/j.cell.2007.02.009">https://doi.org/10.1016/j.cell.2007.02.009</a>.</p>
</div>
<div>
<p>Schultz, Matthew D, Yupeng He, John W Whitaker, Manoj Hariharan, Eran A Mukamel, Danny Leung, Nisha Rajagopal, et al. 2015. “Human body epigenome maps reveal noncanonical DNA methylation variation.” <em>Nature</em> 523 (7559): 212–6. <a href="https://doi.org/10.1038/nature14465">https://doi.org/10.1038/nature14465</a>.</p>
</div>
<div>
<p>Sean, Davis, and Paul S. Meltzer. 2007. “GEOquery: A bridge between the Gene Expression Omnibus (GEO) and BioConductor.” <em>Bioinformatics</em> 23 (14): 1846–7. <a href="https://doi.org/10.1093/bioinformatics/btm254">https://doi.org/10.1093/bioinformatics/btm254</a>.</p>
</div>
<div>
<p>Serre, David, Byron H. Lee, and Angela H. Ting. 2010. “MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome.” <em>Nucleic Acids Research</em> 38 (2): 391–99. <a href="https://doi.org/10.1093/nar/gkp992">https://doi.org/10.1093/nar/gkp992</a>.</p>
</div>
<div>
<p>Shoemaker, Robert, Jie Deng, Wei Wang, and Kun Zhang. 2010. “Allele-specific methylation is prevalent and is contributed by CpG-SNPs in the human genome.” <em>Genome Research</em> 20 (7): 883–9. <a href="https://doi.org/10.1101/gr.104695.109">https://doi.org/10.1101/gr.104695.109</a>.</p>
</div>
<div>
<p>Shukla, Sanjeev, Ersen Kavak, Melissa Gregory, Masahiko Imashimizu, Bojan Shutinoski, Mikhail Kashlev, Philipp Oberdoerffer, Rickard Sandberg, and Shalini Oberdoerffer. 2011. “CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing.” <em>Nature</em> 479 (7371): 74–79. <a href="https://doi.org/10.1038/nature10442">https://doi.org/10.1038/nature10442</a>.</p>
</div>
<div>
<p>Simpkin, Andrew J., Gibran Hemani, Matthew Suderman, Tom R. Gaunt, Oliver Lyttleton, Wendy L. Mcardle, Susan M. Ring, et al. 2016. “Prenatal and early life influences on epigenetic age in children: a study of mother–offspring pairs from two cohort studies.” <em>Human Molecular Genetics</em> 25 (1): 191–201. <a href="https://doi.org/10.1093/hmg/ddv456">https://doi.org/10.1093/hmg/ddv456</a>.</p>
</div>
<div>
<p>Simpkin, Andrew J, Laura D Howe, Kate Tilling, Tom R Gaunt, Oliver Lyttleton, Wendy L McArdle, Susan M Ring, Steve Horvath, George Davey Smith, and Caroline L Relton. 2017. “The epigenetic clock and physical development during childhood and adolescence: longitudinal analysis from a UK birth cohort.” <em>International Journal of Epidemiology</em> 46 (2): dyw307. <a href="https://doi.org/10.1093/ije/dyw307">https://doi.org/10.1093/ije/dyw307</a>.</p>
</div>
<div>
<p>Simpkin, Andrew J., Matthew Suderman, and Laura D. Howe. 2017. “Epigenetic clocks for gestational age: statistical and study design considerations.” <em>Clinical Epigenetics</em> 9 (1): 100. <a href="https://doi.org/10.1186/s13148-017-0402-y">https://doi.org/10.1186/s13148-017-0402-y</a>.</p>
</div>
<div>
<p>Simpson, Jared T, Rachael E Workman, P C Zuzarte, Matei David, L J Dursi, and Winston Timp. 2017. “Detecting DNA cytosine methylation using nanopore sequencing.” <em>Nature Methods</em> 14 (4): 407–10. <a href="https://doi.org/10.1038/nmeth.4184">https://doi.org/10.1038/nmeth.4184</a>.</p>
</div>
<div>
<p>Sinclair, David A., and Philipp Oberdoerffer. 2009. “The ageing epigenome: Damaged beyond repair?” <em>Ageing Research Reviews</em> 8 (3): 189–98. <a href="https://doi.org/10.1016/j.arr.2009.04.004">https://doi.org/10.1016/j.arr.2009.04.004</a>.</p>
</div>
<div>
<p>Skvortsova, Ksenia, Elena Zotenko, Phuc-Loi Luu, Cathryn M. Gould, Shalima S. Nair, Susan J. Clark, and Clare Stirzaker. 2017. “Comprehensive evaluation of genome-wide 5-hydroxymethylcytosine profiling approaches in human DNA.” <em>Epigenetics &amp; Chromatin</em> 10 (1): 16. <a href="https://doi.org/10.1186/s13072-017-0123-7">https://doi.org/10.1186/s13072-017-0123-7</a>.</p>
</div>
<div>
<p>Slieker, Roderick C., Maarten van Iterson, René Luijk, Marian Beekman, Daria V Zhernakova, Matthijs H. Moed, Hailiang Mei, et al. 2016. “Age-related accrual of methylomic variability is linked to fundamental ageing mechanisms.” <em>Genome Biology</em> 17 (1): 191. <a href="https://doi.org/10.1186/s13059-016-1053-6">https://doi.org/10.1186/s13059-016-1053-6</a>.</p>
</div>
<div>
<p>Smith, Alicia K., Varun Kilaru, Mehmet Kocak, Lynn M. Almli, Kristina B. Mercer, Kerry J. Ressler, Frances A. Tylavsky, and Karen N. Conneely. 2014. “Methylation quantitative trait loci (meQTLs) are consistently detected across ancestry, developmental stage, and tissue type.” <em>BMC Genomics</em> 15 (1): 145. <a href="https://doi.org/10.1186/1471-2164-15-145">https://doi.org/10.1186/1471-2164-15-145</a>.</p>
</div>
<div>
<p>Smith, Rick W A, Cara Monroe, and Deborah A. Bolnick. 2015. “Detection of Cytosine methylation in ancient DNA from five native american populations using bisulfite sequencing.” Edited by Ludovic Orlando. <em>PloS One</em> 10 (5): e0125344. <a href="https://doi.org/10.1371/journal.pone.0125344">https://doi.org/10.1371/journal.pone.0125344</a>.</p>
</div>
<div>
<p>Stadler, Michael B., Rabih Murr, Lukas Burger, Robert Ivanek, Florian Lienert, Anne Schöler, Erik van Nimwegen, et al. 2011. “DNA-binding factors shape the mouse methylome at distal regulatory regions.” <em>Nature</em> 480 (7378): 490–5. <a href="https://doi.org/10.1038/nature10716">https://doi.org/10.1038/nature10716</a>.</p>
</div>
<div>
<p>Stubbs, Thomas M, Marc Jan Bonder, Anne-Katrien Stark, Felix Krueger, Ferdinand von Meyenn, Oliver Stegle, and Wolf Reik. 2017. “Multi-tissue DNA methylation age predictor in mouse.” <em>Genome Biology</em> 18 (1): 68. <a href="https://doi.org/10.1186/s13059-017-1203-5">https://doi.org/10.1186/s13059-017-1203-5</a>.</p>
</div>
<div>
<p>Su, Ming, Dali Han, Jerome Boyd-Kirkup, Xiaoming Yu, and Jing-Dong J. Han. 2014. “Evolution of Alu Elements toward Enhancers.” <em>Cell Reports</em> 7 (2): 376–85. <a href="https://doi.org/10.1016/j.celrep.2014.03.011">https://doi.org/10.1016/j.celrep.2014.03.011</a>.</p>
</div>
<div>
<p>Sun, Luyang, Ruofan Yu, and Weiwei Dang. 2018. “Chromatin Architectural Changes during Cellular Senescence and Aging.” <em>Genes</em> 9 (4): 211. <a href="https://doi.org/10.3390/genes9040211">https://doi.org/10.3390/genes9040211</a>.</p>
</div>
<div>
<p>Sutcliffe, Josephine E, Timothy R P Brown, Simon J Allison, Pamela H Scott, and Robert J White. 2000. “Retinoblastoma Protein Disrupts Interactions Required for RNA Polymerase III Transcription.” <em>Molecular and Cellular Biology</em> 20 (24): 9192–9202. <a href="https://doi.org/10.1128/MCB.20.24.9192-9202.2000">https://doi.org/10.1128/MCB.20.24.9192-9202.2000</a>.</p>
</div>
<div>
<p>Syddall, HE, A Aihie Sayer, EM Dennison, HJ Martin, DJP Barker, and C. Cooper. 2005. “Cohort Profile: The Hertfordshire Cohort Study.” <em>International Journal of Epidemiology</em> 34 (6): 1234–42. <a href="https://doi.org/10.1093/ije/dyi127">https://doi.org/10.1093/ije/dyi127</a>.</p>
</div>
<div>
<p>Taiwo, Oluwatosin, Gareth A. Wilson, Tiffany Morris, Stefanie Seisenberger, Wolf Reik, Daniel Pearce, Stephan Beck, and Lee M. Butcher. 2012. “Methylome analysis using MeDIP-seq with low DNA concentrations.” <em>Nature Protocols</em> 7 (4): 617–36. <a href="https://doi.org/10.1038/nprot.2012.012">https://doi.org/10.1038/nprot.2012.012</a>.</p>
</div>
<div>
<p>Tavernarakis, Nektarios. 2008. “Ageing and the regulation of protein synthesis: a balancing act?” <em>Trends in Cell Biology</em> 18 (5): 228–35. <a href="https://doi.org/10.1016/j.tcb.2008.02.004">https://doi.org/10.1016/j.tcb.2008.02.004</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E., Francesco Marabita, Matthias Lechner, Thomas Bartlett, Jesper Tegner, David Gomez-Cabrero, and Stephan Beck. 2013. “A beta-mixture quantile normalization method for correcting probe design bias in Illumina Infinium 450 k DNA methylation data.” <em>Bioinformatics</em> 29 (2): 189–96. <a href="https://doi.org/10.1093/bioinformatics/bts680">https://doi.org/10.1093/bioinformatics/bts680</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E., Usha Menon, Aleksandra Gentry-Maharaj, Susan J. Ramus, Daniel J. Weisenberger, Hui Shen, Mihaela Campan, et al. 2010. “Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.” <em>Genome Research</em> 20 (4): 440–6. <a href="https://doi.org/10.1101/gr.103606.109">https://doi.org/10.1101/gr.103606.109</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E, and Shijie C Zheng. 2017. “Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.” <em>Epigenomics</em> 9 (5): 757–68. <a href="https://doi.org/10.2217/epi-2016-0153">https://doi.org/10.2217/epi-2016-0153</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E., Joanna Zhuang, and Martin Widschwendter. 2011. “Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies.” <em>Bioinformatics</em> 27 (11): 1496–1505. <a href="https://doi.org/10.1093/bioinformatics/btr171">https://doi.org/10.1093/bioinformatics/btr171</a>.</p>
</div>
<div>
<p>Thompson, Michael J, Bridgett VonHoldt, Steve Horvath, and Matteo Pellegrini. 2017. “An epigenetic aging clock for dogs and wolves.” <em>Aging</em> 9 (3): 1055–68. <a href="https://doi.org/10.18632/aging.101211">https://doi.org/10.18632/aging.101211</a>.</p>
</div>
<div>
<p>Torres, Adrian Gabriel, Oscar Reina, Camille Stephan-Otto Attolini, and Lluís Ribas de Pouplana. 2019. “Differential expression of human tRNA genes drives the abundance of tRNA-derived fragments.” <em>Proceedings of the National Academy of Sciences</em> 116 (17): 201821120. <a href="https://doi.org/10.1073/pnas.1821120116">https://doi.org/10.1073/pnas.1821120116</a>.</p>
</div>
<div>
<p>Tran, Hong, Jacob Porter, Ming-an Sun, Hehuang Xie, and Liqing Zhang. 2014. “Objective and Comprehensive Evaluation of Bisulfite Short Read Mapping Tools.” <em>Advances in Bioinformatics</em> 2014: 1–11. <a href="https://doi.org/10.1155/2014/472045">https://doi.org/10.1155/2014/472045</a>.</p>
</div>
<div>
<p>Ullu, Elisabetta, and Christian Tschudi. 1984. “Alu sequences are processed 7SL RNA genes.” <em>Nature</em> 312 (5990): 171–72. <a href="https://doi.org/10.1038/312171a0">https://doi.org/10.1038/312171a0</a>.</p>
</div>
<div>
<p>Ushijima, Toshikazu, Naoko Watanabe, Kimiko Shimizu, Kazuaki Miyamoto, Takashi Sugimura, and Atsushi Kaneda. 2005. “Decreased fidelity in replicating CpG methylation patterns in cancer cells.” <em>Cancer Research</em> 65 (1): 11–17. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15665274">http://www.ncbi.nlm.nih.gov/pubmed/15665274</a>.</p>
</div>
<div>
<p>Van Bortle, Kevin, Douglas H. Phanstiel, and Michael P. Snyder. 2017. “Topological organization and dynamic regulation of human tRNA genes during macrophage differentiation.” <em>Genome Biology</em> 18 (1): 180. <a href="https://doi.org/10.1186/s13059-017-1310-3">https://doi.org/10.1186/s13059-017-1310-3</a>.</p>
</div>
<div>
<p>Varshney, Dhaval, Jana Vavrova-Anderson, Andrew J Oler, Victoria H Cowling, Bradley R Cairns, and Robert J White. 2015. “SINE transcription by RNA polymerase III is suppressed by histone methylation but not by DNA methylation.” <em>Nature Communications</em> 6 (1): 6569. <a href="https://doi.org/10.1038/ncomms7569">https://doi.org/10.1038/ncomms7569</a>.</p>
</div>
<div>
<p>Vertino, Paula M., Jennifer A. Sekowski, Jennifer M. Coll, Nancy Applegren, Suhua Han, Robert J. Hickey, and Linda H. Malkas. 2002. “DNMT1 is a component of a multiprotein DNA replication complex.” <em>Cell Cycle (Georgetown, Tex.)</em> 1 (6): 416–23. <a href="https://doi.org/10.4161/cc.1.6.270">https://doi.org/10.4161/cc.1.6.270</a>.</p>
</div>
<div>
<p>Viljakainen, H. T., T. Korhonen, T. Hytinantti, E. K. A. Laitinen, S. Andersson, O. Mäkitie, and C. Lamberg-Allardt. 2011. “Maternal vitamin D status affects bone growth in early childhood—a prospective cohort study.” <em>Osteoporosis International</em> 22 (3): 883–91. <a href="https://doi.org/10.1007/s00198-010-1499-4">https://doi.org/10.1007/s00198-010-1499-4</a>.</p>
</div>
<div>
<p>Viljakainen, H. T., E. Saarnio, T. Hytinantti, M. Miettinen, H. Surcel, O. Mäkitie, S. Andersson, K. Laitinen, and C. Lamberg-Allardt. 2010. “Maternal Vitamin D Status Determines Bone Variables in the Newborn.” <em>The Journal of Clinical Endocrinology &amp; Metabolism</em> 95 (4): 1749–57. <a href="https://doi.org/10.1210/jc.2009-1391">https://doi.org/10.1210/jc.2009-1391</a>.</p>
</div>
<div>
<p>Voigt, Philipp, Gary LeRoy, William J. Drury, Barry M. Zee, Jinsook Son, David B. Beck, Nicolas L. Young, Benjamin A. Garcia, and Danny Reinberg. 2012. “Asymmetrically modified nucleosomes.” <em>Cell</em> 151 (1): 181–93. <a href="https://doi.org/10.1016/j.cell.2012.09.002">https://doi.org/10.1016/j.cell.2012.09.002</a>.</p>
</div>
<div>
<p>Vorländer, Matthias K, Florence Baudin, Robyn D Moir, René Wetzel, Wim J H Hagen, Ian M Willis, and Christoph W Müller. 2020. “Structural basis for RNA polymerase III transcription repression by Maf1.” <em>Nature Structural &amp; Molecular Biology</em>, February. <a href="https://doi.org/10.1038/s41594-020-0383-y">https://doi.org/10.1038/s41594-020-0383-y</a>.</p>
</div>
<div>
<p>Waddington, C. H. 2012. “The epigenotype. 1942.” <em>International Journal of Epidemiology</em> 41 (1): 10–13. <a href="https://doi.org/10.1093/ije/dyr184">https://doi.org/10.1093/ije/dyr184</a>.</p>
</div>
<div>
<p>Wahl, Simone, Alexander Drong, Benjamin Lehne, Marie Loh, William R Scott, Sonja Kunze, Pei-Chien Tsai, et al. 2017. “Epigenome-wide association study of body mass index, and the adverse outcomes of adiposity.” <em>Nature</em> 541 (7635): 81–86. <a href="https://doi.org/10.1038/nature20784">https://doi.org/10.1038/nature20784</a>.</p>
</div>
<div>
<p>Wang, Meng, and Bernardo Lemos. 2019. “Ribosomal DNA harbors an evolutionarily conserved clock of biological aging.” <em>Genome Research</em> 29 (3): 325–33. <a href="https://doi.org/10.1101/gr.241745.118">https://doi.org/10.1101/gr.241745.118</a>.</p>
</div>
<div>
<p>Ward, Michelle C., Michael D. Wilson, Nuno L. Barbosa-Morais, Dominic Schmidt, Rory Stark, Qun Pan, Petra C. Schwalie, et al. 2013. “Latent Regulatory Potential of Human-Specific Repetitive Elements.” <em>Molecular Cell</em> 49 (2): 262–72. <a href="https://doi.org/10.1016/j.molcel.2012.11.013">https://doi.org/10.1016/j.molcel.2012.11.013</a>.</p>
</div>
<div>
<p>Weber, Christopher M, and Steven Henikoff. 2014. “Histone variants: dynamic punctuation in transcription.” <em>Genes &amp; Development</em> 28 (7): 672–82. <a href="https://doi.org/10.1101/gad.238873.114">https://doi.org/10.1101/gad.238873.114</a>.</p>
</div>
<div>
<p>Weber, Michael, Jonathan J Davies, David Wittig, Edward J Oakeley, Michael Haase, Wan L Lam, and Dirk Schübeler. 2005. “Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells.” <em>Nature Genetics</em> 37 (8): 853–62. <a href="https://doi.org/10.1038/ng1598">https://doi.org/10.1038/ng1598</a>.</p>
</div>
<div>
<p>Wei, Wei-Qi, Xiaohui Li, Qiping Feng, Michiaki Kubo, Iftikhar J. Kullo, Peggy L. Peissig, Elizabeth W. Karlson, et al. 2018. “LPA Variants Are Associated With Residual Cardiovascular Risk in Patients Receiving Statins.” <em>Circulation</em> 138 (17): 1839–49. <a href="https://doi.org/10.1161/CIRCULATIONAHA.117.031356">https://doi.org/10.1161/CIRCULATIONAHA.117.031356</a>.</p>
</div>
<div>
<p>Weidner, Carola Ingrid, Qiong Lin, Carmen Maike Koch, Lewin Eisele, Fabian Beier, Patrick Ziegler, Dirk Olaf Bauerschlag, et al. 2014. “Aging of blood can be tracked by DNA methylation changes at just three CpG sites.” <em>Genome Biology</em> 15 (2): R24. <a href="https://doi.org/10.1186/gb-2014-15-2-r24">https://doi.org/10.1186/gb-2014-15-2-r24</a>.</p>
</div>
<div>
<p>Williams, Louise, Yanxia Bei, Heidi E Church, Nan Dai, Eileen T Dimalanta, Laurence M Ettwiller, Thomas C Evans, et al. 2019. “Enzymatic Methyl-seq: The Next Generation of Methylome Analysis BISULFITE SEQUENCING ALTERNATIVE METHODS FOR DETECTING 5mC AND 5hmC.” <a href="https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis">https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis</a>.</p>
</div>
<div>
<p>Wilson, V L, and P A Jones. 1983. “DNA methylation decreases in aging but not in immortal cells.” <em>Science (New York, N.Y.)</em> 220 (4601): 1055–7. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.</p>
</div>
<div>
<p>Wilson, V. L., R. A. Smith, S. Ma, and R. G. Cutler. 1987. “Genomic 5-methyldeoxycytidine decreases with age.” <em>The Journal of Biological Chemistry</em> 262 (21): 9948–51. <a href="http://www.ncbi.nlm.nih.gov/pubmed/3611071">http://www.ncbi.nlm.nih.gov/pubmed/3611071</a>.</p>
</div>
<div>
<p>Wu, Tao P., Tao Wang, Matthew G. Seetin, Yongquan Lai, Shijia Zhu, Kaixuan Lin, Yifei Liu, et al. 2016. “DNA methylation on N(6)-adenine in mammalian embryonic stem cells.” <em>Nature</em> 532 (7599): 329–33. <a href="https://doi.org/10.1038/nature17640">https://doi.org/10.1038/nature17640</a>.</p>
</div>
<div>
<p>Wu, Xiaoji, and Yi Zhang. 2017. “TET-mediated active DNA demethylation: mechanism, function and beyond.” <em>Nature Reviews Genetics</em> 18 (9): 517–34. <a href="https://doi.org/10.1038/nrg.2017.33">https://doi.org/10.1038/nrg.2017.33</a>.</p>
</div>
<div>
<p>Xie, Mingchao, Chibo Hong, Bo Zhang, Rebecca F Lowdon, Xiaoyun Xing, Daofeng Li, Xin Zhou, et al. 2013. “DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape.” <em>Nature Genetics</em> 45 (7): 836–41. <a href="https://doi.org/10.1038/ng.2649">https://doi.org/10.1038/ng.2649</a>.</p>
</div>
<div>
<p>Xu, Wei-Lin, Ye Yang, Yi-Dan Wang, Liang-Hu Qu, and Ling-Ling Zheng. 2017. “Computational Approaches to tRNA-Derived Small RNAs.” <em>Non-Coding RNA</em> 3 (1): 2. <a href="https://doi.org/10.3390/ncrna3010002">https://doi.org/10.3390/ncrna3010002</a>.</p>
</div>
<div>
<p>Yang, Zhen, Andrew Wong, Diana Kuh, Dirk S. Paul, Vardhman K. Rakyan, R. David Leslie, Shijie C. Zheng, Martin Widschwendter, Stephan Beck, and Andrew E. Teschendorff. 2016. “Correlation of an epigenetic mitotic clock with cancer risk.” <em>Genome Biology</em> 17 (1): 205. <a href="https://doi.org/10.1186/s13059-016-1064-3">https://doi.org/10.1186/s13059-016-1064-3</a>.</p>
</div>
<div>
<p>Yin, Yimeng, Ekaterina Morgunova, Arttu Jolma, Eevi Kaasinen, Biswajyoti Sahu, Syed Khund-Sayeed, Pratyush K. Das, et al. 2017. “Impact of cytosine methylation on DNA binding specificities of human transcription factors.” <em>Science (New York, N.Y.)</em> 356 (6337): eaaj2239. <a href="https://doi.org/10.1126/science.aaj2239">https://doi.org/10.1126/science.aaj2239</a>.</p>
</div>
<div>
<p>Zaghlool, Shaza B, Mashael Al-Shafai, Wadha A Al Muftah, Pankaj Kumar, Mario Falchi, and Karsten Suhre. 2015. “Association of DNA methylation with age, gender, and smoking in an Arab population.” <em>Clinical Epigenetics</em> 7 (1): 6. <a href="https://doi.org/10.1186/s13148-014-0040-6">https://doi.org/10.1186/s13148-014-0040-6</a>.</p>
</div>
<div>
<p>Zenin, Aleksandr, Yakov Tsepilov, Sodbo Sharapov, Evgeny Getmantsev, L. I. Menshikov, Peter O. Fedichev, and Yurii Aulchenko. 2019. “Identification of 12 genetic loci associated with human healthspan.” <em>Communications Biology</em> 2 (1): 41. <a href="https://doi.org/10.1038/s42003-019-0290-0">https://doi.org/10.1038/s42003-019-0290-0</a>.</p>
</div>
<div>
<p>Zhang, Yan, Jan Hapala, Hermann Brenner, and Wolfgang Wagner. 2017. “Individual CpG sites that are associated with age and life expectancy become hypomethylated upon aging.” <em>Clinical Epigenetics</em> 9 (1): 9. <a href="https://doi.org/10.1186/s13148-017-0315-9">https://doi.org/10.1186/s13148-017-0315-9</a>.</p>
</div>
<div>
<p>Zhao, Lei, Ming-an Sun, Zejuan Li, Xue Bai, Miao Yu, Min Wang, Liji Liang, et al. 2014. “The dynamics of DNA methylation fidelity during mouse embryonic stem cell self-renewal and differentiation.” <em>Genome Research</em> 24 (8): 1296–1307. <a href="https://doi.org/10.1101/gr.163147.113">https://doi.org/10.1101/gr.163147.113</a>.</p>
</div>
<div>
<p>Zhou, Wanding, Huy Q. Dinh, Zachary Ramjan, Daniel J. Weisenberger, Charles M. Nicolet, Hui Shen, Peter W. Laird, and Benjamin P. Berman. 2018. “DNA methylation loss in late-replicating domains is linked to mitotic cell division.” <em>Nature Genetics</em> 50 (4): 591–602. <a href="https://doi.org/10.1038/s41588-018-0073-4">https://doi.org/10.1038/s41588-018-0073-4</a>.</p>
</div>
<div>
<p>Zhou, Wanding, Peter W. Laird, and Hui Shen. 2016. “Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes.” <em>Nucleic Acids Research</em> 45 (4): gkw967. <a href="https://doi.org/10.1093/nar/gkw967">https://doi.org/10.1093/nar/gkw967</a>.</p>
</div>
<div>
<p>Zhu, Kun, Andrew JO Whitehouse, Prue H Hart, Merci Kusel, Jenny Mountain, Stephen Lye, Craig Pennell, and John P Walsh. 2014. “Maternal Vitamin D Status During Pregnancy and Bone Mass in Offspring at 20 Years of Age: A Prospective Cohort Study.” <em>Journal of Bone and Mineral Research</em> 29 (5): 1088–95. <a href="https://doi.org/10.1002/jbmr.2138">https://doi.org/10.1002/jbmr.2138</a>.</p>
</div>
<div>
<p>Ziller, Michael J, Hongcang Gu, Fabian Müller, Julie Donaghey, Linus T-Y Tsai, Oliver Kohlbacher, Philip L De Jager, et al. 2013. “Charting a dynamic DNA methylation landscape of the human genome.” <em>Nature</em> 500 (7463): 477–81. <a href="https://doi.org/10.1038/nature12433">https://doi.org/10.1038/nature12433</a>.</p>
</div>
<div>
<p>Zuo, Zheng, Basab Roy, Yiming Kenny Chang, David Granas, and Gary D Stormo. 2017. “Measuring quantitative effects of methylation on transcription factor–DNA binding affinity.” <em>Science Advances</em> 3 (11): eaao1799. <a href="https://doi.org/10.1126/sciadv.aao1799">https://doi.org/10.1126/sciadv.aao1799</a>.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="discussion-3.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/07-references.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
