<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.19 and GitBook 2.6.7" />

  <meta property="og:title" content="References | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-06-20" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="discussion-3.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#illumina-dna-methylation-arrays"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a><ul>
<li class="chapter" data-level="3.1" data-path="arrays.html"><a href="arrays.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="arrays.html"><a href="arrays.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.2.1</b> Outline of EWAS</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="arrays.html"><a href="arrays.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.3.2</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.3</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.4" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.4</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.4.1</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.4.2</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.4.3</b> Southampton Women’s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>4</b> References</a></li>
<li class="chapter" data-level="5" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>5</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="5.1" data-path="tRNAs.html"><a href="tRNAs.html#abstract-2"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="tRNAs.html"><a href="tRNAs.html#introduction-2"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="tRNAs.html"><a href="tRNAs.html#Methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="tRNAs.html"><a href="tRNAs.html#participants-1"><i class="fa fa-check"></i><b>5.3.1</b> Participants</a></li>
<li class="chapter" data-level="5.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>5.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="5.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>5.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="5.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>5.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="5.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>5.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="5.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>5.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="5.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>5.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="5.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>5.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="5.3.9" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-figures"><i class="fa fa-check"></i><b>5.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="5.3.10" data-path="tRNAs.html"><a href="tRNAs.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>5.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>5.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="5.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>5.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="5.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>5.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="5.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>5.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="5.4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>5.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="tRNAs.html"><a href="tRNAs.html#discussion-1"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="tRNAs.html"><a href="tRNAs.html#data-availability"><i class="fa fa-check"></i><b>5.6</b> Data availability</a></li>
<li class="chapter" data-level="5.7" data-path="tRNAs.html"><a href="tRNAs.html#code-availablity"><i class="fa fa-check"></i><b>5.7</b> Code availablity</a></li>
<li class="chapter" data-level="5.8" data-path="tRNAs.html"><a href="tRNAs.html#acknowledgements-1"><i class="fa fa-check"></i><b>5.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="5.9" data-path="tRNAs.html"><a href="tRNAs.html#author-contributions"><i class="fa fa-check"></i><b>5.9</b> Author Contributions</a></li>
<li class="chapter" data-level="5.10" data-path="tRNAs.html"><a href="tRNAs.html#references-1"><i class="fa fa-check"></i><b>5.10</b> References</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>6</b> Results 3 -</a><ul>
<li class="chapter" data-level="6.1" data-path="Alus.html"><a href="Alus.html#abstract-3"><i class="fa fa-check"></i><b>6.1</b> Abstract</a></li>
<li class="chapter" data-level="6.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>6.2</b> Introduction</a></li>
<li class="chapter" data-level="6.3" data-path="Alus.html"><a href="Alus.html#methods-2"><i class="fa fa-check"></i><b>6.3</b> Methods</a></li>
<li class="chapter" data-level="6.4" data-path="Alus.html"><a href="Alus.html#results-1"><i class="fa fa-check"></i><b>6.4</b> Results</a><ul>
<li class="chapter" data-level="6.4.1" data-path="Alus.html"><a href="Alus.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>6.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="6.4.2" data-path="Alus.html"><a href="Alus.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>6.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="6.4.3" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>6.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="6.4.4" data-path="Alus.html"><a href="Alus.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>6.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="Alus.html"><a href="Alus.html#discussion-2"><i class="fa fa-check"></i><b>6.5</b> Discussion</a></li>
<li class="chapter" data-level="6.6" data-path="Alus.html"><a href="Alus.html#methods-3"><i class="fa fa-check"></i><b>6.6</b> Methods</a><ul>
<li class="chapter" data-level="6.6.1" data-path="Alus.html"><a href="Alus.html#participants-2"><i class="fa fa-check"></i><b>6.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="Alus.html"><a href="Alus.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>6.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="6.7.1" data-path="Alus.html"><a href="Alus.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>6.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="6.7.2" data-path="Alus.html"><a href="Alus.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>6.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="6.7.3" data-path="Alus.html"><a href="Alus.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>6.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="6.7.4" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>6.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="6.7.5" data-path="Alus.html"><a href="Alus.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>6.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>7</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references-2.html"><a href="references-2.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references-2" class="section level1 unnumbered">
<h1>References</h1>

<div id="refs" class="references">
<div>
<p>1. Kontis V, Bennett JE, Mathers CD, Li G, Foreman K, Ezzati M: <strong>Future life expectancy in 35 industrialised countries: projections with a Bayesian model ensemble</strong>. <em>The Lancet</em> 2017, <strong>389</strong>:1323–1335<a href="https://doi.org/10.1016/S0140-6736(16)32381-9">10.1016/S0140-6736(16)32381-9</a>Available: <a href="http://dx.doi.org/10.1016/S0140-6736(16)32381-9">http://dx.doi.org/10.1016/S0140-6736(16)32381-9</a>.</p>
</div>
<div>
<p>2. Niccoli T, Partridge L: <strong>Ageing as a risk factor for disease</strong>. <em>Current Biology</em> 2012, <strong>22</strong>:R741–R752<a href="https://doi.org/10.1016/j.cub.2012.07.024">10.1016/j.cub.2012.07.024</a>Available: <a href="http://dx.doi.org/10.1016/j.cub.2012.07.024">http://dx.doi.org/10.1016/j.cub.2012.07.024</a>.</p>
</div>
<div>
<p>3. Belikov AV: <strong>Age-related diseases as vicious cycles</strong>. <em>Ageing Research Reviews</em> 2019, <strong>49</strong>:11–26<a href="https://doi.org/10.1016/j.arr.2018.11.002">10.1016/j.arr.2018.11.002</a>Available: <a href="https://doi.org/10.1016/j.arr.2018.11.002">https://doi.org/10.1016/j.arr.2018.11.002</a>.</p>
</div>
<div>
<p>4. Fleming DM, Elliot AJ: <strong>The impact of influenza on the health and health care utilisation of elderly people</strong>. <em>Vaccine</em> 2005, <strong>23</strong>:S1<a href="https://doi.org/10.1016/j.vaccine.2005.04.018">10.1016/j.vaccine.2005.04.018</a>.</p>
</div>
<div>
<p>5. Barnett K, Mercer SW, Norbury M, Watt G, Wyke S, Guthrie B: <strong>Epidemiology of multimorbidity and implications for health care, research, and medical education: a cross-sectional study</strong>. <em>The Lancet</em> 2012, <strong>380</strong>:37–43<a href="https://doi.org/10.1016/S0140-6736(12)60240-2">10.1016/S0140-6736(12)60240-2</a>Available: <a href="http://dx.doi.org/10.1016/S0140-6736(12)60240-2">http://dx.doi.org/10.1016/S0140-6736(12)60240-2</a>.</p>
</div>
<div>
<p>6. Goldman D: <strong>The Economic Promise of Delayed Aging</strong>. <em>Cold Spring Harbor Perspectives in Medicine</em> 2016, <strong>6</strong>:a025072<a href="https://doi.org/10.1101/cshperspect.a025072">10.1101/cshperspect.a025072</a>Available: <a href="http://perspectivesinmedicine.cshlp.org/lookup/doi/10.1101/cshperspect.a025072">http://perspectivesinmedicine.cshlp.org/lookup/doi/10.1101/cshperspect.a025072</a>.</p>
</div>
<div>
<p>7. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R: <strong>A C. elegans mutant that lives twice as long as wild type</strong>. <em>Nature</em> 1993, <strong>366</strong>:461–464<a href="https://doi.org/10.1038/366461a0">10.1038/366461a0</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/8247153">http://www.ncbi.nlm.nih.gov/pubmed/8247153</a>.</p>
</div>
<div>
<p>8. Medvedev ZA: <strong>An Attempt at a Rational Classification of Theories of Ageing</strong>. <em>Biological Reviews</em> 1990, <strong>65</strong>:375–398<a href="https://doi.org/10.1111/j.1469-185X.1990.tb01428.x">10.1111/j.1469-185X.1990.tb01428.x</a>Available: <a href="http://doi.wiley.com/10.1111/j.1469-185X.1990.tb01428.x">http://doi.wiley.com/10.1111/j.1469-185X.1990.tb01428.x</a>.</p>
</div>
<div>
<p>9. López-Otín C, Blasco MA, Partridge L, Serrano M, Kroemer G: <strong>The hallmarks of aging.</strong> <em>Cell</em> 2013, <strong>153</strong>:1194–217<a href="https://doi.org/10.1016/j.cell.2013.05.039">10.1016/j.cell.2013.05.039</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23746838">http://www.ncbi.nlm.nih.gov/pubmed/23746838</a>.</p>
</div>
<div>
<p>10. Booth LN, Brunet A: <strong>The Aging Epigenome</strong>. <em>Molecular Cell</em> 2016, <strong>62</strong>:728–744<a href="https://doi.org/10.1016/j.molcel.2016.05.013">10.1016/j.molcel.2016.05.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2016.05.013">http://dx.doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div>
<p>11. Sinclair DA, Oberdoerffer P: <strong>The ageing epigenome: Damaged beyond repair?</strong> <em>Ageing Research Reviews</em> 2009, <strong>8</strong>:189–198<a href="https://doi.org/10.1016/j.arr.2009.04.004">10.1016/j.arr.2009.04.004</a>.</p>
</div>
<div>
<p>12. Hayano M, Yang J-H, Bonkowski MS, Amorim JA, Ross JM, Coppotelli G, Griffin P, Chew YC, Guo W, Yang X, Vera DL, Salfati EL, Das A, Thakur S, Kane AE, Mitchell SJ, Mohri Y, Nishimura EK, Schaevitz L, Garg N, Balta A-M, Rego MA, Gregory-Ksander M, Jakobs TC, Zhong L, Wakimoto H, Mostoslavsky R, Wagers AJ, Tsubota K, Bonasera SJ, et al.: <strong>DNA Break-Induced Epigenetic Drift as a Cause of Mammalian Aging</strong>. <em>bioRxiv</em> 2019, <a href="https://doi.org/10.1101/808659">10.1101/808659</a>Available: <a href="https://doi.org/10.1101/808659">https://doi.org/10.1101/808659</a>.</p>
</div>
<div>
<p>13. Kane AE, Sinclair DA: <strong>Epigenetic changes during aging and their reprogramming potential</strong>. <em>Critical Reviews in Biochemistry and Molecular Biology</em> 2019, <strong>54</strong>:61–83<a href="https://doi.org/10.1080/10409238.2019.1570075">10.1080/10409238.2019.1570075</a>Available: <a href="https://doi.org/10.1080/10409238.2019.1570075">https://doi.org/10.1080/10409238.2019.1570075</a>.</p>
</div>
<div>
<p>14. Waddington CH: <strong>The epigenotype. 1942.</strong> <em>International journal of epidemiology</em> 2012, <strong>41</strong>:10–3<a href="https://doi.org/10.1093/ije/dyr184">10.1093/ije/dyr184</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22186258">http://www.ncbi.nlm.nih.gov/pubmed/22186258</a>.</p>
</div>
<div>
<p>15. Russo V, Martienssen R, Riggs AD: <em>Epigenetic Mechanisms of Gene Regulation</em>. Cold Spring Harbor Laboratory Press; 1996 Available: <a href="https://cshmonographs.org/index.php/monographs/issue/view/087969490.32">https://cshmonographs.org/index.php/monographs/issue/view/087969490.32</a>.</p>
</div>
<div>
<p>16. Bird A: <strong>Perceptions of epigenetics.</strong> <em>Nature</em> 2007, <strong>447</strong>:396–8<a href="https://doi.org/10.1038/nature05913">10.1038/nature05913</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17522671">http://www.ncbi.nlm.nih.gov/pubmed/17522671</a>.</p>
</div>
<div>
<p>17. Bianconi E, Piovesan A, Facchin F, Beraudi A, Casadei R, Frabetti F, Vitale L, Pelleri MC, Tassani S, Piva F, Perez-Amodio S, Strippoli P, Canaider S: <strong>An estimation of the number of cells in the human body.</strong> <em>Annals of human biology</em> 2013, <strong>40</strong>:463–71<a href="https://doi.org/10.3109/03014460.2013.807878">10.3109/03014460.2013.807878</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23829164">http://www.ncbi.nlm.nih.gov/pubmed/23829164</a>.</p>
</div>
<div>
<p>18. Regev A, Teichmann SA, Lander ES, Amit I, Benoist C, Birney E, Bodenmiller B, Campbell P, Carninci P, Clatworthy M, Clevers H, Deplancke B, Dunham I, Eberwine J, Eils R, Enard W, Farmer A, Fugger L, Göttgens B, Hacohen N, Haniffa M, Hemberg M, Kim S, Klenerman P, Kriegstein A, Lein E, Linnarsson S, Lundberg E, Lundeberg J, Majumder P, et al.: <strong>The Human Cell Atlas</strong>. <em>eLife</em> 2017, <strong>6</strong>:1–30<a href="https://doi.org/10.7554/eLife.27041">10.7554/eLife.27041</a>Available: <a href="https://elifesciences.org/articles/27041">https://elifesciences.org/articles/27041</a>.</p>
</div>
<div>
<p>19. CellSystemsVoices: <strong>What Is Your Conceptual Definition of “Cell Type” in the Context of a Mature Organism?</strong> <em>Cell Systems</em> 2017, <strong>4</strong>:255–259<a href="https://doi.org/10.1016/j.cels.2017.03.006">10.1016/j.cels.2017.03.006</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S2405471217300911">https://linkinghub.elsevier.com/retrieve/pii/S2405471217300911</a>.</p>
</div>
<div>
<p>20. CellOntology: <strong>Cell Ontology</strong>. 2017, <a href="https://doi.org/10.5281/zenodo.168254">10.5281/zenodo.168254</a>Available: <a href="http://obofoundry.org/ontology/cl.html">http://obofoundry.org/ontology/cl.html</a>.Accessed 5 April 2017.</p>
</div>
<div>
<p>21. Guo JU, Su Y, Shin JH, Shin J, Li H, Xie B, Zhong C, Hu S, Le T, Fan G, Zhu H, Chang Q, Gao Y, Ming G-l, Song H: <strong>Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain.</strong> <em>Nature neuroscience</em> 2014, <strong>17</strong>:215–22<a href="https://doi.org/10.1038/nn.3607">10.1038/nn.3607</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24362762">http://www.ncbi.nlm.nih.gov/pubmed/24362762</a>.</p>
</div>
<div>
<p>22. Lister R, Pelizzola M, Dowen RH, Hawkins RD, Hon G, Tonti-Filippini J, Nery JR, Lee L, Ye Z, Ngo Q-m, Edsall L, Antosiewicz-Bourget J, Stewart R, Ruotti V, Millar AH, Thomson JA, Ren B, Ecker JR: <strong>Human DNA methylomes at base resolution show widespread epigenomic differences.</strong> <em>Nature</em> 2009, <strong>462</strong>:315–22<a href="https://doi.org/10.1038/nature08514">10.1038/nature08514</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19829295">http://www.ncbi.nlm.nih.gov/pubmed/19829295</a>.</p>
</div>
<div>
<p>23. Schultz MD, He Y, Whitaker JW, Hariharan M, Mukamel EA, Leung D, Rajagopal N, Nery JR, Urich MA, Chen H, Lin S, Lin Y, Jung I, Schmitt AD, Selvaraj S, Ren B, Sejnowski TJ, Wang W, Ecker JR: <strong>Human body epigenome maps reveal noncanonical DNA methylation variation.</strong> <em>Nature</em> 2015, <strong>523</strong>:212–6<a href="https://doi.org/10.1038/nature14465">10.1038/nature14465</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26030523">http://www.ncbi.nlm.nih.gov/pubmed/26030523</a>.</p>
</div>
<div>
<p>24. Pfeifer GP, Kadam S, Jin S-G: <strong>5-hydroxymethylcytosine and its potential roles in development and cancer.</strong> <em>Epigenetics &amp; chromatin</em> 2013, <strong>6</strong>:10<a href="https://doi.org/10.1186/1756-8935-6-10">10.1186/1756-8935-6-10</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23634848">http://www.ncbi.nlm.nih.gov/pubmed/23634848</a>.</p>
</div>
<div>
<p>25. Wu TP, Wang T, Seetin MG, Lai Y, Zhu S, Lin K, Liu Y, Byrum SD, Mackintosh SG, Zhong M, Tackett A, Wang G, Hon LS, Fang G, Swenberg Ja, Xiao AZ: <strong>DNA methylation on N(6)-adenine in mammalian embryonic stem cells.</strong> <em>Nature</em> 2016, <strong>532</strong>:329–33<a href="https://doi.org/10.1038/nature17640">10.1038/nature17640</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27027282">http://www.ncbi.nlm.nih.gov/pubmed/27027282</a>.</p>
</div>
<div>
<p>26. Bannister AJ, Kouzarides T: <strong>Regulation of chromatin by histone modifications</strong>. <em>Cell Research</em> 2011, <strong>21</strong>:381–395<a href="https://doi.org/10.1038/cr.2011.22">10.1038/cr.2011.22</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21321607">http://www.ncbi.nlm.nih.gov/pubmed/21321607</a>.</p>
</div>
<div>
<p>27. Voigt P, LeRoy G, Drury WJ, Zee BM, Son J, Beck DB, Young NL, Garcia BA, Reinberg D: <strong>Asymmetrically modified nucleosomes.</strong> <em>Cell</em> 2012, <strong>151</strong>:181–93<a href="https://doi.org/10.1016/j.cell.2012.09.002">10.1016/j.cell.2012.09.002</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23021224">http://www.ncbi.nlm.nih.gov/pubmed/23021224</a>.</p>
</div>
<div>
<p>28. Hoffman MM, Buske OJ, Wang J, Weng Z, Bilmes J a, Noble WS: <strong>Unsupervised pattern discovery in human chromatin structure through genomic segmentation.</strong> <em>Nature methods</em> 2012, <strong>9</strong>:473–6<a href="https://doi.org/10.1038/nmeth.1937">10.1038/nmeth.1937</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22426492">http://www.ncbi.nlm.nih.gov/pubmed/22426492</a>.</p>
</div>
<div>
<p>29. Carrillo-de-Santa-Pau E, Juan D, Pancaldi V, Were F, Martin-Subero I, Rico D, Valencia A: <strong>Automatic identification of informative regions with epigenomic changes associated to hematopoiesis</strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:9244–9259<a href="https://doi.org/10.1093/nar/gkx618">10.1093/nar/gkx618</a>Available: <a href="https://academic.oup.com/nar/article/45/16/9244/3976483">https://academic.oup.com/nar/article/45/16/9244/3976483</a>.</p>
</div>
<div>
<p>30. Weber CM, Henikoff S: <strong>Histone variants: dynamic punctuation in transcription.</strong> <em>Genes &amp; development</em> 2014, <strong>28</strong>:672–82<a href="https://doi.org/10.1101/gad.238873.114">10.1101/gad.238873.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24696452">http://www.ncbi.nlm.nih.gov/pubmed/24696452</a>.</p>
</div>
<div>
<p>31. Kaikkonen MU, Lam MTY, Glass CK: <strong>Non-coding RNAs as regulators of gene expression and epigenetics.</strong> <em>Cardiovascular research</em> 2011, <strong>90</strong>:430–40<a href="https://doi.org/10.1093/cvr/cvr097">10.1093/cvr/cvr097</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21558279">http://www.ncbi.nlm.nih.gov/pubmed/21558279</a>.</p>
</div>
<div>
<p>32. Cerase A, Pintacuda G, Tattermusch A, Avner P: <strong>Xist localization and function: new insights from multiple levels</strong>. <em>Genome Biology</em> 2015, <strong>16</strong>:166<a href="https://doi.org/10.1186/s13059-015-0733-y">10.1186/s13059-015-0733-y</a>Available: <a href="http://genomebiology.com/2015/16/1/166">http://genomebiology.com/2015/16/1/166</a>.</p>
</div>
<div>
<p>33. Jenuwein T: <strong>Translating the Histone Code</strong>. <em>Science</em> 2001, <strong>293</strong>:1074–1080<a href="https://doi.org/10.1126/science.1063127">10.1126/science.1063127</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20653993">http://www.ncbi.nlm.nih.gov/pubmed/20653993</a>.</p>
</div>
<div>
<p>34. Briggs AW, Stenzel U, Meyer M, Krause J, Kircher M, Pääbo S: <strong>Removal of deaminated cytosines and detection of in vivo methylation in ancient DNA.</strong> <em>Nucleic acids research</em> 2010, <strong>38</strong>:e87<a href="https://doi.org/10.1093/nar/gkp1163">10.1093/nar/gkp1163</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20028723">http://www.ncbi.nlm.nih.gov/pubmed/20028723</a>.</p>
</div>
<div>
<p>35. Smith RWA, Monroe C, Bolnick DA: <strong>Detection of Cytosine methylation in ancient DNA from five native american populations using bisulfite sequencing.</strong> <em>PloS one</em> 2015, <strong>10</strong>:e0125344<a href="https://doi.org/10.1371/journal.pone.0125344">10.1371/journal.pone.0125344</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26016479">http://www.ncbi.nlm.nih.gov/pubmed/26016479</a>.</p>
</div>
<div>
<p>36. Bauden M, Kristl T, Andersson R, Marko-Varga G, Ansari D: <strong>Characterization of histone-related chemical modifications in formalin-fixed paraffin-embedded and fresh-frozen human pancreatic cancer xenografts using LC-MS/MS</strong>. <em>Laboratory Investigation</em> 2017, <strong>97</strong>:279–288<a href="https://doi.org/10.1038/labinvest.2016.134">10.1038/labinvest.2016.134</a>Available: <a href="http://www.nature.com/doifinder/10.1038/labinvest.2016.134">http://www.nature.com/doifinder/10.1038/labinvest.2016.134</a>.</p>
</div>
<div>
<p>37. Hashimshony T, Zhang J, Keshet I, Bustin M, Cedar H: <strong>The role of DNA methylation in setting up chromatin structure during development</strong>. <em>Nature Genetics</em> 2003, <strong>34</strong>:187–192<a href="https://doi.org/10.1038/ng1158">10.1038/ng1158</a>Available: <a href="http://www.nature.com/articles/ng1158">http://www.nature.com/articles/ng1158</a>.</p>
</div>
<div>
<p>38. Estève P-O, Chin HG, Smallwood A, Feehery GR, Gangisetty O, Karpf AR, Carey MF, Pradhan S: <strong>Direct interaction between DNMT1 and G9a coordinates DNA and histone methylation during replication.</strong> <em>Genes &amp; development</em> 2006, <strong>20</strong>:3089–103<a href="https://doi.org/10.1101/gad.1463706">10.1101/gad.1463706</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17085482">http://www.ncbi.nlm.nih.gov/pubmed/17085482</a>.</p>
</div>
<div>
<p>39. Rose NR, Klose RJ: <strong>Understanding the relationship between DNA methylation and histone lysine methylation</strong>. <em>Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</em> 2014, <strong>1839</strong>:1362–1372<a href="https://doi.org/10.1016/j.bbagrm.2014.02.007">10.1016/j.bbagrm.2014.02.007</a>Available: <a href="http://dx.doi.org/10.1016/j.bbagrm.2014.02.007">http://dx.doi.org/10.1016/j.bbagrm.2014.02.007</a>.</p>
</div>
<div>
<p>40. Luo Y, Lu X, Xie H: <strong>Dynamic Alu Methylation during Normal Development, Aging, and Tumorigenesis</strong>. <em>BioMed Research International</em> 2014, <strong>2014</strong>:1–12<a href="https://doi.org/10.1155/2014/784706">10.1155/2014/784706</a>Available: <a href="http://www.hindawi.com/journals/bmri/2014/784706/">http://www.hindawi.com/journals/bmri/2014/784706/</a>.</p>
</div>
<div>
<p>41. Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, Baldwin J, Devon K, Dewar K, Doyle M, FitzHugh W, Funke R, Gage D, Harris K, Heaford A, Howland J, Kann L, Lehoczky J, LeVine R, McEwan P, McKernan K, Meldrim J, Mesirov JP, Miranda C, Morris W, Naylor J, Raymond C, Rosetti M, Santos R, Sheridan A, Sougnez C, et al.: <strong>Initial sequencing and analysis of the human genome.</strong> <em>Nature</em> 2001, <strong>409</strong>:860–921<a href="https://doi.org/10.1038/35057062">10.1038/35057062</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011">http://www.ncbi.nlm.nih.gov/pubmed/11237011</a>.</p>
</div>
<div>
<p>42. Duncan BK, Miller JH: <strong>Mutagenic deamination of cytosine residues in DNA.</strong> <em>Nature</em> 1980, <strong>287</strong>:560–1Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6999365">http://www.ncbi.nlm.nih.gov/pubmed/6999365</a>.</p>
</div>
<div>
<p>43. Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P, Cui H, Gabo K, Rongione M, Webster M, Ji H, Potash J, Sabunciyan S, Feinberg AP: <strong>The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.</strong> <em>Nature genetics</em> 2009, <strong>41</strong>:178–186<a href="https://doi.org/10.1038/ng.298">10.1038/ng.298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151715">http://www.ncbi.nlm.nih.gov/pubmed/19151715</a>.</p>
</div>
<div>
<p>44. Saxonov S, Berg P, Brutlag DL: <strong>A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters</strong>. <em>Proceedings of the National Academy of Sciences</em> 2006, <strong>103</strong>:1412–1417<a href="https://doi.org/10.1073/pnas.0510310103">10.1073/pnas.0510310103</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.0510310103">http://www.pnas.org/cgi/doi/10.1073/pnas.0510310103</a>.</p>
</div>
<div>
<p>45. Bell JSK, Vertino PM: <strong>Orphan CpG islands define a novel class of highly active enhancers.</strong> <em>Epigenetics</em> 2017, <strong>12</strong>:449–464<a href="https://doi.org/10.1080/15592294.2017.1297910">10.1080/15592294.2017.1297910</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28448736">http://www.ncbi.nlm.nih.gov/pubmed/28448736</a>.</p>
</div>
<div>
<p>46. Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr ARW, James KD, Turner DJ, Smith C, Harrison DJ, Andrews R, Bird AP: <strong>Orphan CpG islands identify numerous conserved promoters in the mammalian genome.</strong> <em>PLoS genetics</em> 2010, <strong>6</strong>:e1001134<a href="https://doi.org/10.1371/journal.pgen.1001134">10.1371/journal.pgen.1001134</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20885785">http://www.ncbi.nlm.nih.gov/pubmed/20885785</a>.</p>
</div>
<div>
<p>47. Maunakea AK, Nagarajan RP, Bilenky M, Ballinger TJ, D’Souza C, Fouse SD, Johnson BE, Hong C, Nielsen C, Zhao Y, Turecki G, Delaney A, Varhol R, Thiessen N, Shchors K, Heine VM, Rowitch DH, Xing X, Fiore C, Schillebeeckx M, Jones SJM, Haussler D, Marra MA, Hirst M, Wang T, Costello JF: <strong>Conserved role of intragenic DNA methylation in regulating alternative promoters.</strong> <em>Nature</em> 2010, <strong>466</strong>:253–7<a href="https://doi.org/10.1038/nature09165">10.1038/nature09165</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20613842">http://www.ncbi.nlm.nih.gov/pubmed/20613842</a>.</p>
</div>
<div>
<p>48. Ehrlich M, Gama-Sosa MA, Huang L-H, Midgett RM, Kuo KC, McCune RA, Gehrke C: <strong>Amount and distribution of 5-methylcytosine in human DNA from different types of tissues or cells</strong>. <em>Nucleic Acids Research</em> 1982, <strong>10</strong>:2709–2721<a href="https://doi.org/10.1093/nar/10.8.2709">10.1093/nar/10.8.2709</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/10.8.2709">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/10.8.2709</a>.</p>
</div>
<div>
<p>49. Bird A: <strong>DNA methylation patterns and epigenetic memory.</strong> <em>Genes &amp; development</em> 2002, <strong>16</strong>:6–21<a href="https://doi.org/10.1101/gad.947102">10.1101/gad.947102</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11782440">http://www.ncbi.nlm.nih.gov/pubmed/11782440</a>.</p>
</div>
<div>
<p>50. Stadler MB, Murr R, Burger L, Ivanek R, Lienert F, Schöler A, Nimwegen E van, Wirbelauer C, Oakeley EJ, Gaidatzis D, Tiwari VK, Schübeler D: <strong>DNA-binding factors shape the mouse methylome at distal regulatory regions.</strong> <em>Nature</em> 2011, <strong>480</strong>:490–5<a href="https://doi.org/10.1038/nature10716">10.1038/nature10716</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22170606">http://www.ncbi.nlm.nih.gov/pubmed/22170606</a>.</p>
</div>
<div>
<p>51. Yin Y, Morgunova E, Jolma A, Kaasinen E, Sahu B, Khund-Sayeed S, Das PK, Kivioja T, Dave K, Zhong F, Nitta KR, Taipale M, Popov A, Ginno PA, Domcke S, Yan J, Schübeler D, Vinson C, Taipale J: <strong>Impact of cytosine methylation on DNA binding specificities of human transcription factors.</strong> <em>Science (New York, N.Y.)</em> 2017, <strong>356</strong>:eaaj2239<a href="https://doi.org/10.1126/science.aaj2239">10.1126/science.aaj2239</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28473536">http://www.ncbi.nlm.nih.gov/pubmed/28473536</a>.</p>
</div>
<div>
<p>52. Zuo Z, Roy B, Chang YK, Granas D, Stormo GD: <strong>Measuring quantitative effects of methylation on transcription factor–DNA binding affinity</strong>. <em>Science Advances</em> 2017, <strong>3</strong>:eaao1799<a href="https://doi.org/10.1126/sciadv.aao1799">10.1126/sciadv.aao1799</a>Available: <a href="http://advances.sciencemag.org/lookup/doi/10.1126/sciadv.aao1799">http://advances.sciencemag.org/lookup/doi/10.1126/sciadv.aao1799</a>.</p>
</div>
<div>
<p>53. Kelly TK, Liu Y, Lay FD, Liang G, Berman BP, Jones P a: <strong>Genome-wide mapping of nucleosome positioning and DNA methylation within individual DNA molecules</strong>. <em>Genome Research</em> 2012, <strong>22</strong>:2497–2506<a href="https://doi.org/10.1101/gr.143008.112">10.1101/gr.143008.112</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22960375">http://www.ncbi.nlm.nih.gov/pubmed/22960375</a>.</p>
</div>
<div>
<p>54. Huff JT, Zilberman D: <strong>Dnmt1-Independent CG Methylation Contributes to Nucleosome Positioning in Diverse Eukaryotes</strong>. <em>Cell</em> 2014, <strong>156</strong>:1286–1297<a href="https://doi.org/10.1016/j.cell.2014.01.029">10.1016/j.cell.2014.01.029</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24630728">http://www.ncbi.nlm.nih.gov/pubmed/24630728</a>.</p>
</div>
<div>
<p>55. Raiber E-a, Portella G, Martínez Cuesta S, Hardisty R, Murat P, Li Z, Iurlaro M, Dean W, Spindel J, Beraldi D, Liu Z, Dawson MA, Reik W, Balasubramanian S: <strong>5-Formylcytosine organizes nucleosomes and forms Schiff base interactions with histones in mouse embryonic stem cells</strong>. <em>Nature Chemistry</em> 2018, <strong>10</strong>:1258–1266<a href="https://doi.org/10.1038/s41557-018-0149-x">10.1038/s41557-018-0149-x</a>Available: <a href="http://www.nature.com/articles/s41557-018-0149-x">http://www.nature.com/articles/s41557-018-0149-x</a>.</p>
</div>
<div>
<p>56. Vertino PM, Sekowski JA, Coll JM, Applegren N, Han S, Hickey RJ, Malkas LH: <strong>DNMT1 is a component of a multiprotein DNA replication complex.</strong> <em>Cell cycle (Georgetown, Tex.)</em> 2002, <strong>1</strong>:416–23<a href="https://doi.org/10.4161/cc.1.6.270">10.4161/cc.1.6.270</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12548018">http://www.ncbi.nlm.nih.gov/pubmed/12548018</a>.</p>
</div>
<div>
<p>57. Bostick M, Kim JK, Esteve P-O, Clark A, Pradhan S, Jacobsen SE: <strong>UHRF1 Plays a Role in Maintaining DNA Methylation in Mammalian Cells</strong>. <em>Science</em> 2007, <strong>317</strong>:1760–1764<a href="https://doi.org/10.1126/science.1147939">10.1126/science.1147939</a>Available: <a href="http://www.sciencemag.org/cgi/doi/10.1126/science.1147939">http://www.sciencemag.org/cgi/doi/10.1126/science.1147939</a>.</p>
</div>
<div>
<p>58. Jia D, Jurkowska RZ, Zhang X, Jeltsch A, Cheng X: <strong>Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation</strong>. <em>Nature</em> 2007, <strong>449</strong>:248–251<a href="https://doi.org/10.1038/nature06146">10.1038/nature06146</a>Available: <a href="http://www.nature.com/articles/nature06146">http://www.nature.com/articles/nature06146</a>.</p>
</div>
<div>
<p>59. Lyko F: <strong>The DNA methyltransferase family: a versatile toolkit for epigenetic regulation</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>19</strong>:81–92<a href="https://doi.org/10.1038/nrg.2017.80">10.1038/nrg.2017.80</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.80">http://www.nature.com/doifinder/10.1038/nrg.2017.80</a>.</p>
</div>
<div>
<p>60. Wu X, Zhang Y: <strong>TET-mediated active DNA demethylation: mechanism, function and beyond</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:517–534<a href="https://doi.org/10.1038/nrg.2017.33">10.1038/nrg.2017.33</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.33">http://www.nature.com/doifinder/10.1038/nrg.2017.33</a>.</p>
</div>
<div>
<p>61. Lienert F, Wirbelauer C, Som I, Dean A, Mohn F, Schübeler D: <strong>Identification of genetic elements that autonomously determine DNA methylation states.</strong> <em>Nature genetics</em> 2011, <strong>43</strong>:1091–7<a href="https://doi.org/10.1038/ng.946">10.1038/ng.946</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21964573">http://www.ncbi.nlm.nih.gov/pubmed/21964573</a>.</p>
</div>
<div>
<p>62. Ziller MJ, Gu H, Müller F, Donaghey J, Tsai LT-Y, Kohlbacher O, De Jager PL, Rosen ED, Bennett DA, Bernstein BE, Gnirke A, Meissner A: <strong>Charting a dynamic DNA methylation landscape of the human genome.</strong> <em>Nature</em> 2013, <strong>500</strong>:477–81<a href="https://doi.org/10.1038/nature12433">10.1038/nature12433</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23925113">http://www.ncbi.nlm.nih.gov/pubmed/23925113</a>.</p>
</div>
<div>
<p>63. Boks MP, Derks EM, Weisenberger DJ, Strengman E, Janson E, Sommer IE, Kahn RS, Ophoff RA: <strong>The relationship of DNA methylation with age, gender and genotype in twins and healthy controls.</strong> <em>PloS one</em> 2009, <strong>4</strong>:e6767<a href="https://doi.org/10.1371/journal.pone.0006767">10.1371/journal.pone.0006767</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19774229">http://www.ncbi.nlm.nih.gov/pubmed/19774229</a>.</p>
</div>
<div>
<p>64. Kerkel K, Spadola A, Yuan E, Kosek J, Jiang L, Hod E, Li K, Murty VV, Schupf N, Vilain E, Morris M, Haghighi F, Tycko B: <strong>Genomic surveys by methylation-sensitive SNP analysis identify sequence-dependent allele-specific DNA methylation.</strong> <em>Nature genetics</em> 2008, <strong>40</strong>:904–8<a href="https://doi.org/10.1038/ng.174">10.1038/ng.174</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18568024">http://www.ncbi.nlm.nih.gov/pubmed/18568024</a>.</p>
</div>
<div>
<p>65. Schalkwyk LC, Meaburn EL, Smith R, Dempster EL, Jeffries AR, Davies MN, Plomin R, Mill J: <strong>Allelic skewing of DNA methylation is widespread across the genome.</strong> <em>American journal of human genetics</em> 2010, <strong>86</strong>:196–212<a href="https://doi.org/10.1016/j.ajhg.2010.01.014">10.1016/j.ajhg.2010.01.014</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20159110">http://www.ncbi.nlm.nih.gov/pubmed/20159110</a>.</p>
</div>
<div>
<p>66. Shoemaker R, Deng J, Wang W, Zhang K: <strong>Allele-specific methylation is prevalent and is contributed by CpG-SNPs in the human genome.</strong> <em>Genome research</em> 2010, <strong>20</strong>:883–9<a href="https://doi.org/10.1101/gr.104695.109">10.1101/gr.104695.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20418490">http://www.ncbi.nlm.nih.gov/pubmed/20418490</a>.</p>
</div>
<div>
<p>67. Bell CG, Gao F, Yuan W, Roos L, Acton RJ, Xia Y, Bell J, Ward K, Mangino M, Hysi PG, Wang J, Spector TD: <strong>Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci</strong>. <em>Nature Communications</em> 2018, <strong>9</strong>:8<a href="https://doi.org/10.1038/s41467-017-01586-1">10.1038/s41467-017-01586-1</a>Available: <a href="http://www.nature.com/articles/s41467-017-01586-1">http://www.nature.com/articles/s41467-017-01586-1</a>.</p>
</div>
<div>
<p>68. Ciernia AV, LaSalle J: <strong>The landscape of DNA methylation amid a perfect storm of autism aetiologies</strong>. <em>Nature Reviews Neuroscience</em> 2016, <strong>17</strong>:411–423<a href="https://doi.org/10.1038/nrn.2016.41">10.1038/nrn.2016.41</a>Available: <a href="http://dx.doi.org/10.1038/nrn.2016.41">http://dx.doi.org/10.1038/nrn.2016.41</a>.</p>
</div>
<div>
<p>69. Ushijima T, Watanabe N, Shimizu K, Miyamoto K, Sugimura T, Kaneda A: <strong>Decreased fidelity in replicating CpG methylation patterns in cancer cells.</strong> <em>Cancer research</em> 2005, <strong>65</strong>:11–7Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15665274">http://www.ncbi.nlm.nih.gov/pubmed/15665274</a>.</p>
</div>
<div>
<p>70. Kunkel TA: <strong>DNA Replication Fidelity</strong>. <em>Journal of Biological Chemistry</em> 2004, <strong>279</strong>:16895–16898<a href="https://doi.org/10.1074/jbc.R400006200">10.1074/jbc.R400006200</a>Available: <a href="http://www.jbc.org/lookup/doi/10.1074/jbc.R400006200">http://www.jbc.org/lookup/doi/10.1074/jbc.R400006200</a>.</p>
</div>
<div>
<p>71. Laird CD, Pleasant ND, Clark AD, Sneeden JL, Hassan KMA, Manley NC, Vary JC, Morgan T, Hansen RS, Stoger R: <strong>Hairpin-bisulfite PCR: Assessing epigenetic methylation patterns on complementary strands of individual DNA molecules</strong>. <em>Proceedings of the National Academy of Sciences</em> 2004, <strong>101</strong>:204–209<a href="https://doi.org/10.1073/pnas.2536758100">10.1073/pnas.2536758100</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.2536758100">http://www.pnas.org/cgi/doi/10.1073/pnas.2536758100</a>.</p>
</div>
<div>
<p>72. Pfeifer GP, Steigerwald SD, Hansen RS, Gartler SM, Riggs AD: <strong>Polymerase chain reaction-aided genomic sequencing of an X chromosome-linked CpG island: methylation patterns suggest clonal inheritance, CpG site autonomy, and an explanation of activity state stability.</strong> <em>Proceedings of the National Academy of Sciences</em> 1990, <strong>87</strong>:8252–8256<a href="https://doi.org/10.1073/pnas.87.21.8252">10.1073/pnas.87.21.8252</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.87.21.8252">http://www.pnas.org/cgi/doi/10.1073/pnas.87.21.8252</a>.</p>
</div>
<div>
<p>73. Riggs AD, Xiong Z: <strong>Methylation and epigenetic fidelity</strong>. <em>Proceedings of the National Academy of Sciences</em> 2004, <strong>101</strong>:4–5<a href="https://doi.org/10.1073/pnas.0307781100">10.1073/pnas.0307781100</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/14695893">http://www.ncbi.nlm.nih.gov/pubmed/14695893</a>.</p>
</div>
<div>
<p>74. Jenkinson G, Pujadas E, Goutsias J, Feinberg AP: <strong>Potential energy landscapes identify the information-theoretic nature of the epigenome.</strong> <em>Nature genetics</em> 2017, <strong>49</strong>:719–729<a href="https://doi.org/10.1038/ng.3811">10.1038/ng.3811</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28346445">http://www.ncbi.nlm.nih.gov/pubmed/28346445</a>.</p>
</div>
<div>
<p>75. Zhao L, Sun M-a, Li Z, Bai X, Yu M, Wang M, Liang L, Shao X, Arnovitz S, Wang Q, He C, Lu X, Chen J, Xie H: <strong>The dynamics of DNA methylation fidelity during mouse embryonic stem cell self-renewal and differentiation</strong>. <em>Genome Research</em> 2014, <strong>24</strong>:1296–1307<a href="https://doi.org/10.1101/gr.163147.113">10.1101/gr.163147.113</a>Available: <a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.163147.113">http://genome.cshlp.org/lookup/doi/10.1101/gr.163147.113</a>.</p>
</div>
<div>
<p>76. Hatada I, Hayashizaki Y, Hirotsune S, Komatsubara H, Mukai T: <strong>A genomic scanning method for higher organisms using restriction sites as landmarks.</strong> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 1991, <strong>88</strong>:9523–7<a href="https://doi.org/10.1073/pnas.88.21.9523">10.1073/pnas.88.21.9523</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1946366">http://www.ncbi.nlm.nih.gov/pubmed/1946366</a>.</p>
</div>
<div>
<p>77. Weber M, Davies JJ, Wittig D, Oakeley EJ, Haase M, Lam WL, Schübeler D: <strong>Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells</strong>. <em>Nature Genetics</em> 2005, <strong>37</strong>:853–862<a href="https://doi.org/10.1038/ng1598">10.1038/ng1598</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16007088">http://www.ncbi.nlm.nih.gov/pubmed/16007088</a>.</p>
</div>
<div>
<p>78. Harrison A, Parle-McDermott A: <strong>DNA Methylation: A Timeline of Methods and Applications</strong>. <em>Frontiers in Genetics</em> 2011, <strong>2</strong>:632–649<a href="https://doi.org/10.3389/fgene.2011.00074">10.3389/fgene.2011.00074</a>Available: <a href="http://journal.frontiersin.org/article/10.3389/fgene.2011.00074/abstract">http://journal.frontiersin.org/article/10.3389/fgene.2011.00074/abstract</a>.</p>
</div>
<div>
<p>79. Down TA, Rakyan VK, Turner DJ, Flicek P, Li H, Kulesha E, Gräf S, Johnson N, Herrero J, Tomazou EM, Thorne NP, Bäckdahl L, Herberth M, Howe KL, Jackson DK, Miretti MM, Marioni JC, Birney E, Hubbard TJP, Durbin R, Tavaré S, Beck S: <strong>A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis</strong>. <em>Nature Biotechnology</em> 2008, <strong>26</strong>:779–785<a href="https://doi.org/10.1038/nbt1414">10.1038/nbt1414</a>Available: <a href="http://www.nature.com/articles/nbt1414">http://www.nature.com/articles/nbt1414</a>.</p>
</div>
<div>
<p>80. Lienhard M, Grimm C, Morkel M, Herwig R, Chavez L: <strong>MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments</strong>. <em>Bioinformatics</em> 2014, <strong>30</strong>:284–286<a href="https://doi.org/10.1093/bioinformatics/btt650">10.1093/bioinformatics/btt650</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24227674">http://www.ncbi.nlm.nih.gov/pubmed/24227674</a>.</p>
</div>
<div>
<p>81. Serre D, Lee BH, Ting AH: <strong>MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome</strong>. <em>Nucleic Acids Research</em> 2010, <strong>38</strong>:391–399<a href="https://doi.org/10.1093/nar/gkp992">10.1093/nar/gkp992</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp992">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp992</a>.</p>
</div>
<div>
<p>82. Li J, Yang J, Zhou P, Le Y, Zhou C, Wang S, Xu D, Lin H-K, Gong Z: <strong>Circular RNAs in cancer: novel insights into origins, properties, functions and implications.</strong> <em>American journal of cancer research</em> 2015, <strong>5</strong>:472–80Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25973291">http://www.ncbi.nlm.nih.gov/pubmed/25973291</a>.</p>
</div>
<div>
<p>83. Bock C, Tomazou EM, Brinkman AB, Müller F, Simmer F, Gu H, Jäger N, Gnirke A, Stunnenberg HG, Meissner A: <strong>Quantitative comparison of genome-wide DNA methylation mapping technologies</strong>. <em>Nature Biotechnology</em> 2010, <strong>28</strong>:1106–1114<a href="https://doi.org/10.1038/nbt.1681">10.1038/nbt.1681</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20852634">http://www.ncbi.nlm.nih.gov/pubmed/20852634</a>.</p>
</div>
<div>
<p>84. Hayatsu H, Wataya Y, Kai K, Iida S: <strong>Reaction of sodium bisulfite with uracil, cytosine, and their derivatives</strong>. <em>Biochemistry</em> 1970, <strong>9</strong>:2858–2865<a href="https://doi.org/10.1021/bi00816a016">10.1021/bi00816a016</a>Available: <a href="http://pubs.acs.org/doi/abs/10.1021/bi00816a016">http://pubs.acs.org/doi/abs/10.1021/bi00816a016</a>.</p>
</div>
<div>
<p>85. Adamowicz M, Maratou K, Aitman TJ: <strong>Multiplexed DNA Methylation Analysis of Target Regions Using Microfluidics (Fluidigm)</strong>. In <em>Methods in molecular biology (clifton, n.j.)</em> Vol. 1708 2018:349–363. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29224153">http://www.ncbi.nlm.nih.gov/pubmed/29224153</a>.</p>
</div>
<div>
<p>86. Skvortsova K, Zotenko E, Luu P-L, Gould CM, Nair SS, Clark SJ, Stirzaker C: <strong>Comprehensive evaluation of genome-wide 5-hydroxymethylcytosine profiling approaches in human DNA</strong>. <em>Epigenetics &amp; Chromatin</em> 2017, <strong>10</strong>:16<a href="https://doi.org/10.1186/s13072-017-0123-7">10.1186/s13072-017-0123-7</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-017-0123-7">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-017-0123-7</a>.</p>
</div>
<div>
<p>87. Williams L, Bei Y, Church HE, Dai N, Dimalanta ET, Ettwiller LM, Evans TC, Langhorst BW, Borgaro JG, Guan S, Marks K, Menin JF, Nichols NM, Chaithanya Ponnaluri VK, Saleh L, Samaranayake M, Sexton BS, Sun Z, Tamanaha E, Vaisvila R, Yigit E, Davis TB: <strong>Enzymatic Methyl-seq: The Next Generation of Methylome Analysis BISULFITE SEQUENCING ALTERNATIVE METHODS FOR DETECTING 5mC AND 5hmC</strong>. 2019,:2–5Available: <a href="https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis">https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis</a>.</p>
</div>
<div>
<p>88. Rhoads A, Au KF: <strong>PacBio Sequencing and Its Applications</strong>. <em>Genomics, Proteomics &amp; Bioinformatics</em> 2015, <strong>13</strong>:278–289<a href="https://doi.org/10.1016/j.gpb.2015.08.002">10.1016/j.gpb.2015.08.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1672022915001345">https://linkinghub.elsevier.com/retrieve/pii/S1672022915001345</a>.</p>
</div>
<div>
<p>89. Simpson JT, Workman RE, Zuzarte PC, David M, Dursi LJ, Timp W: <strong>Detecting DNA cytosine methylation using nanopore sequencing</strong>. <em>Nature Methods</em> 2017, <strong>14</strong>:407–410<a href="https://doi.org/10.1038/nmeth.4184">10.1038/nmeth.4184</a>Available: <a href="http://www.nature.com/articles/nmeth.4184">http://www.nature.com/articles/nmeth.4184</a>.</p>
</div>
<div>
<p>90. Wilson VL, Jones PA: <strong>DNA methylation decreases in aging but not in immortal cells.</strong> <em>Science (New York, N.Y.)</em> 1983, <strong>220</strong>:1055–7Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.</p>
</div>
<div>
<p>91. Romanov GA, Vanyushin BF: <strong>Methylation of reiterated sequences in mammalian DNAs. Effects of the tissue type, age, malignancy and hormonal induction.</strong> <em>Biochimica et biophysica acta</em> 1981, <strong>653</strong>:204–18Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7225396">http://www.ncbi.nlm.nih.gov/pubmed/7225396</a>.</p>
</div>
<div>
<p>92. Berdyshev GD, Korotaev GK, Boiarskikh GV, Vaniushin BF: <strong>Nucleotide composition of DNA and RNA from somatic tissues of humpback and its changes during spawning.</strong> <em>Biokhimiia (Moscow, Russia)</em> 1967, <strong>32</strong>:988–93Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5628601">http://www.ncbi.nlm.nih.gov/pubmed/5628601</a>.</p>
</div>
<div>
<p>93. Wilson VL, Smith RA, Ma S, Cutler RG: <strong>Genomic 5-methyldeoxycytidine decreases with age.</strong> <em>The Journal of biological chemistry</em> 1987, <strong>262</strong>:9948–51Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3611071">http://www.ncbi.nlm.nih.gov/pubmed/3611071</a>.</p>
</div>
<div>
<p>94. Cole JJ, Robertson NA, Rather MI, Thomson JP, McBryan T, Sproul D, Wang T, Brock C, Clark W, Ideker T, Meehan RR, Miller RA, Brown-Borg HM, Adams PD: <strong>Diverse interventions that extend mouse lifespan suppress shared age-associated epigenetic changes at critical gene regulatory regions</strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:58<a href="https://doi.org/10.1186/s13059-017-1185-3">10.1186/s13059-017-1185-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1185-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1185-3</a>.</p>
</div>
<div>
<p>95. Meissner A, Gnirke A, Bell GW, Ramsahoye B, Lander ES, Jaenisch R: <strong>Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis.</strong> <em>Nucleic acids research</em> 2005, <strong>33</strong>:5868–77<a href="https://doi.org/10.1093/nar/gki901">10.1093/nar/gki901</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16224102">http://www.ncbi.nlm.nih.gov/pubmed/16224102</a>.</p>
</div>
<div>
<p>96. Feinberg AP, Tycko B: <strong>The history of cancer epigenetics</strong>. <em>Nature Reviews Cancer</em> 2004, <strong>4</strong>:143–153<a href="https://doi.org/10.1038/nrc1279">10.1038/nrc1279</a>Available: <a href="http://www.nature.com/articles/nrc1279">http://www.nature.com/articles/nrc1279</a>.</p>
</div>
<div>
<p>97. Knudson AG: <strong>Mutation and cancer: statistical study of retinoblastoma.</strong> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 1971, <strong>68</strong>:820–3<a href="https://doi.org/10.1073/pnas.68.4.820">10.1073/pnas.68.4.820</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5279523">http://www.ncbi.nlm.nih.gov/pubmed/5279523</a>.</p>
</div>
<div>
<p>98. Esteller M, Silva JM, Dominguez G, Bonilla F, Matias-Guiu X, Lerma E, Bussaglia E, Prat J, Harkes IC, Repasky EA, Gabrielson E, Schutte M, Baylin SB, Herman JG: <strong>Promoter hypermethylation and BRCA1 inactivation in sporadic breast and ovarian tumors.</strong> <em>Journal of the National Cancer Institute</em> 2000, <strong>92</strong>:564–9<a href="https://doi.org/10.1093/jnci/92.7.564">10.1093/jnci/92.7.564</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10749912">http://www.ncbi.nlm.nih.gov/pubmed/10749912</a>.</p>
</div>
<div>
<p>99. Feinberg AP: <strong>The Key Role of Epigenetics in Human Disease Prevention and Mitigation</strong>. <em>New England Journal of Medicine</em> 2018, <strong>378</strong>:1323–1334<a href="https://doi.org/10.1056/NEJMra1402513">10.1056/NEJMra1402513</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29617578">http://www.ncbi.nlm.nih.gov/pubmed/29617578</a>.</p>
</div>
<div>
<p>100. Fraga MF, Ballestar E, Paz MF, Ropero S, Setien F, Ballestar ML, Heine-Suner D, Cigudosa JC, Urioste M, Benitez J, Boix-Chornet M, Sanchez-Aguilera A, Ling C, Carlsson E, Poulsen P, Vaag A, Stephan Z, Spector TD, Wu Y-Z, Plass C, Esteller M: <strong>From The Cover: Epigenetic differences arise during the lifetime of monozygotic twins</strong>. <em>Proceedings of the National Academy of Sciences</em> 2005, <strong>102</strong>:10604–10609<a href="https://doi.org/10.1073/pnas.0500398102">10.1073/pnas.0500398102</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16009939">http://www.ncbi.nlm.nih.gov/pubmed/16009939</a>.</p>
</div>
<div>
<p>101. Slieker RC, Iterson M van, Luijk R, Beekman M, Zhernakova DV, Moed MH, Mei H, Galen M van, Deelen P, Bonder MJ, Zhernakova A, Uitterlinden AG, Tigchelaar EF, Stehouwer CDA, Schalkwijk CG, Kallen CJH van der, Hofman A, Heemst D van, Geus EJ de, Dongen J van, Deelen J, Berg LH van den, Meurs J van, Jansen R, ’t Hoen PAC, Franke L, Wijmenga C, Veldink JH, Swertz MA, Greevenbroek MMJ van, et al.: <strong>Age-related accrual of methylomic variability is linked to fundamental ageing mechanisms.</strong> <em>Genome biology</em> 2016, <strong>17</strong>:191<a href="https://doi.org/10.1186/s13059-016-1053-6">10.1186/s13059-016-1053-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27654999">http://www.ncbi.nlm.nih.gov/pubmed/27654999</a>.</p>
</div>
<div>
<p>102. Bibikova M: <strong>High-throughput DNA methylation profiling using universal bead arrays</strong>. <em>Genome Research</em> 2006, <strong>16</strong>:383–393<a href="https://doi.org/10.1101/gr.4410706">10.1101/gr.4410706</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.4410706">http://www.genome.org/cgi/doi/10.1101/gr.4410706</a>.</p>
</div>
<div>
<p>103. Eckhardt F, Lewin J, Cortese R, Rakyan VK, Attwood J, Burger M, Burton J, Cox TV, Davies R, Down TA, Haefliger C, Horton R, Howe K, Jackson DK, Kunde J, Koenig C, Liddle J, Niblett D, Otto T, Pettett R, Seemann S, Thompson C, West T, Rogers J, Olek A, Berlin K, Beck S: <strong>DNA methylation profiling of human chromosomes 6, 20 and 22</strong>. <em>Nature Genetics</em> 2006, <strong>38</strong>:1378–1385<a href="https://doi.org/10.1038/ng1909">10.1038/ng1909</a>Available: <a href="http://www.nature.com/doifinder/10.1038/ng1909">http://www.nature.com/doifinder/10.1038/ng1909</a>.</p>
</div>
<div>
<p>104. Bjornsson HT: <strong>Intra-individual Change Over Time in DNA Methylation With Familial Clustering</strong>. <em>JAMA</em> 2008, <strong>299</strong>:2877<a href="https://doi.org/10.1001/jama.299.24.2877">10.1001/jama.299.24.2877</a>Available: <a href="http://jama.jamanetwork.com/article.aspx?doi=10.1001/jama.299.24.2877">http://jama.jamanetwork.com/article.aspx?doi=10.1001/jama.299.24.2877</a>.</p>
</div>
<div>
<p>105. Christensen BC, Houseman EA, Marsit CJ, Zheng S, Wrensch MR, Wiemels JL, Nelson HH, Karagas MR, Padbury JF, Bueno R, Sugarbaker DJ, Yeh R-F, Wiencke JK, Kelsey KT: <strong>Aging and Environmental Exposures Alter Tissue-Specific DNA Methylation Dependent upon CpG Island Context</strong>. <em>PLoS Genetics</em> 2009, <strong>5</strong>:e1000602<a href="https://doi.org/10.1371/journal.pgen.1000602">10.1371/journal.pgen.1000602</a>Available: <a href="https://dx.plos.org/10.1371/journal.pgen.1000602">https://dx.plos.org/10.1371/journal.pgen.1000602</a>.</p>
</div>
<div>
<p>106. Rakyan VK, Down TA, Maslau S, Andrew T, Yang TP, Beyan H, Whittaker P, McCann OT, Finer S, Valdes AM, Leslie RD, Deloukas P, Spector TD: <strong>Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains</strong>. <em>Genome Research</em> 2010, <strong>20</strong>:434–439<a href="https://doi.org/10.1101/gr.103101.109">10.1101/gr.103101.109</a>Available: <a href="http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109">http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109</a>.</p>
</div>
<div>
<p>107. Bibikova M, Le J, Barnes B, Saedinia-Melnyk S, Zhou L, Shen R, Gunderson KL: <strong>Genome-wide DNA methylation profiling using Infinium assay</strong>. <em>Epigenomics</em> 2009, <strong>1</strong>:177–200<a href="https://doi.org/10.2217/epi.09.14">10.2217/epi.09.14</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi.09.14">https://www.futuremedicine.com/doi/10.2217/epi.09.14</a>.</p>
</div>
<div>
<p>108. Teschendorff AE, Menon U, Gentry-Maharaj A, Ramus SJ, Weisenberger DJ, Shen H, Campan M, Noushmehr H, Bell CG, Maxwell AP, Savage DA, Mueller-Holzner E, Marth C, Kocjan G, Gayther SA, Jones A, Beck S, Wagner W, Laird PW, Jacobs IJ, Widschwendter M: <strong>Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.</strong> <em>Genome research</em> 2010, <strong>20</strong>:440–6<a href="https://doi.org/10.1101/gr.103606.109">10.1101/gr.103606.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20219944">http://www.ncbi.nlm.nih.gov/pubmed/20219944</a>.</p>
</div>
<div>
<p>109. Jung M, Pfeifer GP: <strong>Aging and DNA methylation</strong>. <em>BMC Biology</em> 2015, <strong>13</strong>:7<a href="https://doi.org/10.1186/s12915-015-0118-4">10.1186/s12915-015-0118-4</a>Available: <a href="http://www.biomedcentral.com/1741-7007/13/7">http://www.biomedcentral.com/1741-7007/13/7</a>.</p>
</div>
<div>
<p>110. Bocklandt S, Lin W, Sehl ME, Sánchez FJ, Sinsheimer JS, Horvath S, Vilain E: <strong>Epigenetic Predictor of Age</strong>. <em>PLoS ONE</em> 2011, <strong>6</strong>:e14821<a href="https://doi.org/10.1371/journal.pone.0014821">10.1371/journal.pone.0014821</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0014821">https://dx.plos.org/10.1371/journal.pone.0014821</a>.</p>
</div>
<div>
<p>111. Koch CM, Wagner W: <strong>Epigenetic-aging-signature to determine age in different tissues.</strong> <em>Aging</em> 2011, <strong>3</strong>:1018–27<a href="https://doi.org/10.18632/aging.100395">10.18632/aging.100395</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22067257">http://www.ncbi.nlm.nih.gov/pubmed/22067257</a>.</p>
</div>
<div>
<p>112. Bell JT, Tsai P-C, Yang T-P, Pidsley R, Nisbet J, Glass D, Mangino M, Zhai G, Zhang F, Valdes A, Shin S-Y, Dempster EL, Murray RM, Grundberg E, Hedman AK, Nica A, Small KS, MuTHER Consortium, Dermitzakis ET, McCarthy MI, Mill J, Spector TD, Deloukas P: <strong>Epigenome-wide scans identify differentially methylated regions for age and age-related phenotypes in a healthy ageing population.</strong> <em>PLoS genetics</em> 2012, <strong>8</strong>:e1002629<a href="https://doi.org/10.1371/journal.pgen.1002629">10.1371/journal.pgen.1002629</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22532803">http://www.ncbi.nlm.nih.gov/pubmed/22532803</a>.</p>
</div>
<div>
<p>113. Heyn H, Li N, Ferreira HJ, Moran S, Pisano DG, Gomez A, Diez J, Sanchez-Mut JV, Setien F, Carmona FJ, Puca AA, Sayols S, Pujana MA, Serra-Musach J, Iglesias-Platas I, Formiga F, Fernandez AF, Fraga MF, Heath SC, Valencia A, Gut IG, Wang J, Esteller M: <strong>Distinct DNA methylomes of newborns and centenarians</strong>. <em>Proceedings of the National Academy of Sciences</em> 2012, <strong>109</strong>:10522–10527<a href="https://doi.org/10.1073/pnas.1120658109">10.1073/pnas.1120658109</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1120658109">http://www.pnas.org/cgi/doi/10.1073/pnas.1120658109</a>.</p>
</div>
<div>
<p>114. Bibikova M, Barnes B, Tsan C, Ho V, Klotzle B, Le JM, Delano D, Zhang L, Schroth GP, Gunderson KL, Fan J-B, Shen R: <strong>High density DNA methylation array with single CpG site resolution.</strong> <em>Genomics</em> 2011, <strong>98</strong>:288–95<a href="https://doi.org/10.1016/j.ygeno.2011.07.007">10.1016/j.ygeno.2011.07.007</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21839163">http://www.ncbi.nlm.nih.gov/pubmed/21839163</a>.</p>
</div>
<div>
<p>115. Affinito O, Scala G, Palumbo D, Florio E, Monticelli A, Miele G, Avvedimento VE, Usiello A, Chiariotti L, Cocozza S: <strong>Modeling DNA methylation by analyzing the individual configurations of single molecules.</strong> <em>Epigenetics</em> 2016, <strong>11</strong>:881–888<a href="https://doi.org/10.1080/15592294.2016.1246108">10.1080/15592294.2016.1246108</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27748645">http://www.ncbi.nlm.nih.gov/pubmed/27748645</a>.</p>
</div>
<div>
<p>116. Haerter JO, Lövkvist C, Dodd IB, Sneppen K: <strong>Collaboration between CpG sites is needed for stable somatic inheritance of DNA methylation states</strong>. <em>Nucleic Acids Research</em> 2014, <strong>42</strong>:2235–2244<a href="https://doi.org/10.1093/nar/gkt1235">10.1093/nar/gkt1235</a>Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3936770">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3936770</a>.</p>
</div>
<div>
<p>117. Garagnani P, Bacalini MG, Pirazzini C, Gori D, Giuliani C, Mari D, Di Blasio AM, Gentilini D, Vitale G, Collino S, Rezzi S, Castellani G, Capri M, Salvioli S, Franceschi C: <strong>Methylation of ELOVL2 gene as a new epigenetic marker of age.</strong> <em>Aging cell</em> 2012, <strong>11</strong>:1132–4<a href="https://doi.org/10.1111/acel.12005">10.1111/acel.12005</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23061750">http://www.ncbi.nlm.nih.gov/pubmed/23061750</a>.</p>
</div>
<div>
<p>118. Hannum G, Guinney J, Zhao L, Zhang L, Hughes G, Sadda S, Klotzle B, Bibikova M, Fan J-B, Gao Y, Deconde R, Chen M, Rajapakse I, Friend S, Ideker T, Zhang K: <strong>Genome-wide methylation profiles reveal quantitative views of human aging rates.</strong> <em>Molecular cell</em> 2013, <strong>49</strong>:359–367<a href="https://doi.org/10.1016/j.molcel.2012.10.016">10.1016/j.molcel.2012.10.016</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23177740">http://www.ncbi.nlm.nih.gov/pubmed/23177740</a>.</p>
</div>
<div>
<p>119. Horvath S: <strong>DNA methylation age of human tissues and cell types.</strong> <em>Genome biology</em> 2013, <strong>14</strong>:R115<a href="https://doi.org/10.1186/gb-2013-14-10-r115">10.1186/gb-2013-14-10-r115</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24138928">http://www.ncbi.nlm.nih.gov/pubmed/24138928</a>.</p>
</div>
<div>
<p>120. Friedman J, Hastie T, Tibshirani R: <strong>Regularization Paths for Generalized Linear Models via Coordinate Descent</strong>. <em>Journal of Statistical Software</em> 2010, <strong>33</strong>:7–10<a href="https://doi.org/10.18637/jss.v033.i01">10.18637/jss.v033.i01</a>Available: <a href="http://www.jstatsoft.org/v33/i01/">http://www.jstatsoft.org/v33/i01/</a>.</p>
</div>
<div>
<p>121. Florath I, Butterbach K, Muller H, Bewerunge-Hudler M, Brenner H: <strong>Cross-sectional and longitudinal changes in DNA methylation with age: an epigenome-wide analysis revealing over 60 novel age-associated CpG sites</strong>. <em>Human Molecular Genetics</em> 2014, <strong>23</strong>:1186–1201<a href="https://doi.org/10.1093/hmg/ddt531">10.1093/hmg/ddt531</a>Available: <a href="https://academic.oup.com/hmg/article-lookup/doi/10.1093/hmg/ddt531">https://academic.oup.com/hmg/article-lookup/doi/10.1093/hmg/ddt531</a>.</p>
</div>
<div>
<p>122. Bacalini MG, Boattini A, Gentilini D, Giampieri E, Pirazzini C, Giuliani C, Fontanesi E, Remondini D, Capri M, Del Rio A, Luiselli D, Vitale G, Mari D, Castellani G, Di Blasio AM, Salvioli S, Franceschi C, Garagnani P: <strong>A meta-analysis on age-associated changes in blood DNA methylation: results from an original analysis pipeline for Infinium 450k data</strong>. <em>Aging</em> 2015, <strong>7</strong>:97–109<a href="https://doi.org/10.18632/aging.100718">10.18632/aging.100718</a>Available: <a href="http://www.aging-us.com/article/100718">http://www.aging-us.com/article/100718</a>.</p>
</div>
<div>
<p>123. Zaghlool SB, Al-Shafai M, Al Muftah WA, Kumar P, Falchi M, Suhre K: <strong>Association of DNA methylation with age, gender, and smoking in an Arab population</strong>. <em>Clinical Epigenetics</em> 2015, <strong>7</strong>:6<a href="https://doi.org/10.1186/s13148-014-0040-6">10.1186/s13148-014-0040-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25663950">http://www.ncbi.nlm.nih.gov/pubmed/25663950</a>.</p>
</div>
<div>
<p>124. Benton MC, Sutherland HG, Macartney-Coxson D, Haupt LM, Lea RA, Griffiths LR: <strong>Methylome-wide association study of whole blood DNA in the Norfolk Island isolate identifies robust loci associated with age</strong>. <em>Aging</em> 2017, <strong>9</strong>:753–768<a href="https://doi.org/10.18632/aging.101187">10.18632/aging.101187</a>Available: <a href="http://www.aging-us.com/article/101187/text">http://www.aging-us.com/article/101187/text</a>.</p>
</div>
<div>
<p>125. Johnson ND, Wiener HW, Smith AK, Nishitani S, Absher DM, Arnett DK, Aslibekyan S, Conneely KN: <strong>Non-linear patterns in age-related DNA methylation may reflect CD4 + T cell differentiation</strong>. <em>Epigenetics</em> 2017, <strong>12</strong>:492–503<a href="https://doi.org/10.1080/15592294.2017.1314419">10.1080/15592294.2017.1314419</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28387568">http://www.ncbi.nlm.nih.gov/pubmed/28387568</a>.</p>
</div>
<div>
<p>126. Smith AK, Kilaru V, Kocak M, Almli LM, Mercer KB, Ressler KJ, Tylavsky FA, Conneely KN: <strong>Methylation quantitative trait loci (meQTLs) are consistently detected across ancestry, developmental stage, and tissue type</strong>. <em>BMC Genomics</em> 2014, <strong>15</strong>:145<a href="https://doi.org/10.1186/1471-2164-15-145">10.1186/1471-2164-15-145</a>Available: <a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-145">http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-145</a>.</p>
</div>
<div>
<p>127. Jaiswal S, Fontanillas P, Flannick J, Manning A, Grauman PV, Mar BG, Lindsley RC, Mermel CH, Burtt N, Chavez A, Higgins JM, Moltchanov V, Kuo FC, Kluk MJ, Henderson B, Kinnunen L, Koistinen HA, Ladenvall C, Getz G, Correa A, Banahan BF, Gabriel S, Kathiresan S, Stringham HM, McCarthy MI, Boehnke M, Tuomilehto J, Haiman C, Groop L, Atzmon G, et al.: <strong>Age-Related Clonal Hematopoiesis Associated with Adverse Outcomes</strong>. <em>New England Journal of Medicine</em> 2014, <strong>371</strong>:2488–2498<a href="https://doi.org/10.1056/NEJMoa1408617">10.1056/NEJMoa1408617</a>Available: <a href="http://www.nejm.org/doi/10.1056/NEJMoa1408617">http://www.nejm.org/doi/10.1056/NEJMoa1408617</a>.</p>
</div>
<div>
<p>128. Weidner CI, Lin Q, Koch CM, Eisele L, Beier F, Ziegler P, Bauerschlag DO, Jöckel K-H, Erbel R, Mühleisen TW, Zenke M, Brümmendorf TH, Wagner W: <strong>Aging of blood can be tracked by DNA methylation changes at just three CpG sites.</strong> <em>Genome biology</em> 2014, <strong>15</strong>:R24<a href="https://doi.org/10.1186/gb-2014-15-2-r24">10.1186/gb-2014-15-2-r24</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24490752">http://www.ncbi.nlm.nih.gov/pubmed/24490752</a>.</p>
</div>
<div>
<p>129. Johnson TE: <strong>Recent results: Biomarkers of aging</strong>. <em>Experimental Gerontology</em> 2006, <strong>41</strong>:1243–1246<a href="https://doi.org/10.1016/j.exger.2006.09.006">10.1016/j.exger.2006.09.006</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0531556506002865">https://linkinghub.elsevier.com/retrieve/pii/S0531556506002865</a>.</p>
</div>
<div>
<p>130. Thompson MJ, VonHoldt B, Horvath S, Pellegrini M: <strong>An epigenetic aging clock for dogs and wolves.</strong> <em>Aging</em> 2017, <strong>9</strong>:1055–1068<a href="https://doi.org/10.18632/aging.101211">10.18632/aging.101211</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28373601">http://www.ncbi.nlm.nih.gov/pubmed/28373601</a>.</p>
</div>
<div>
<p>131. Stubbs TM, Bonder MJ, Stark A-K, Krueger F, Meyenn F von, Stegle O, Reik W: <strong>Multi-tissue DNA methylation age predictor in mouse</strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:68<a href="https://doi.org/10.1186/s13059-017-1203-5">10.1186/s13059-017-1203-5</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28399939">http://www.ncbi.nlm.nih.gov/pubmed/28399939</a>.</p>
</div>
<div>
<p>132. Lowe R, Danson AF, Rakyan VK, Yildizoglu S, Saldmann F, Viltard M, Friedlander G, Faulkes CG: <strong>DNA methylation clocks as a predictor for ageing and age estimation in naked mole-rats , Heterocephalus glaber</strong>. 2020, <strong>12</strong>:1–13<a href="https://doi.org/10.18632/aging.102892">10.18632/aging.102892</a>.</p>
</div>
<div>
<p>133. Lowe R, Barton C, Jenkins CA, Ernst C, Forman O, Fernandez-Twinn DS, Bock C, Rossiter SJ, Faulkes CG, Ozanne SE, Walter L, Odom DT, Mellersh C, Rakyan VK: <strong>Ageing-associated DNA methylation dynamics are a molecular readout of lifespan variation among mammalian species</strong>. <em>Genome Biology</em> 2018, <strong>19</strong>:22<a href="https://doi.org/10.1186/s13059-018-1397-1">10.1186/s13059-018-1397-1</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29452591">http://www.ncbi.nlm.nih.gov/pubmed/29452591</a>.</p>
</div>
<div>
<p>134. Wang M, Lemos B: <strong>Ribosomal DNA harbors an evolutionarily conserved clock of biological aging.</strong> <em>Genome research</em> 2019, <strong>29</strong>:325–333<a href="https://doi.org/10.1101/gr.241745.118">10.1101/gr.241745.118</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30765617">http://www.ncbi.nlm.nih.gov/pubmed/30765617</a>.</p>
</div>
<div>
<p>135. Marioni RE, Shah S, McRae AF, Chen BH, Colicino E, Harris SE, Gibson J, Henders AK, Redmond P, Cox SR, Pattie A, Corley J, Murphy L, Martin NG, Montgomery GW, Feinberg AP, Fallin MD, Multhaup ML, Jaffe AE, Joehanes R, Schwartz J, Just AC, Lunetta KL, Murabito JM, Starr JM, Horvath S, Baccarelli AA, Levy D, Visscher PM, Wray NR, et al.: <strong>DNA methylation age of blood predicts all-cause mortality in later life.</strong> <em>Genome biology</em> 2015, <strong>16</strong>:25<a href="https://doi.org/10.1186/s13059-015-0584-6">10.1186/s13059-015-0584-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25633388">http://www.ncbi.nlm.nih.gov/pubmed/25633388</a>.</p>
</div>
<div>
<p>136. Christiansen L, Lenart A, Tan Q, Vaupel JW, Aviv A, McGue M, Christensen K: <strong>DNA methylation age is associated with mortality in a longitudinal Danish twin study</strong>. <em>Aging Cell</em> 2016, <strong>15</strong>:149–154<a href="https://doi.org/10.1111/acel.12421">10.1111/acel.12421</a>Available: <a href="http://doi.wiley.com/10.1111/acel.12421">http://doi.wiley.com/10.1111/acel.12421</a>.</p>
</div>
<div>
<p>137. Chen BH, Marioni RE, Colicino E, Peters MJ, Ward-Caviness CK, Tsai P-C, Roetker NS, Just AC, Demerath EW, Guan W, Bressler J, Fornage M, Studenski S, Vandiver AR, Moore AZ, Tanaka T, Kiel DP, Liang L, Vokonas P, Schwartz J, Lunetta KL, Murabito JM, Bandinelli S, Hernandez DG, Melzer D, Nalls M, Pilling LC, Price TR, Singleton AB, Gieger C, et al.: <strong>DNA methylation-based measures of biological age: meta-analysis predicting time to death.</strong> <em>Aging</em> 2016, <strong>8</strong>:1844–1865<a href="https://doi.org/10.18632/aging.101020">10.18632/aging.101020</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27690265">http://www.ncbi.nlm.nih.gov/pubmed/27690265</a>.</p>
</div>
<div>
<p>138. Levine ME, Lu AT, Quach A, Chen BH, Assimes TL, Bandinelli S, Hou L, Baccarelli AA, Stewart JD, Li Y, Whitsel EA, Wilson JG, Reiner AP, Aviv A, Lohman K, Liu Y, Ferrucci L, Horvath S: <strong>An epigenetic biomarker of aging for lifespan and healthspan</strong>. <em>Aging</em> 2018, <strong>10</strong>:573–591<a href="https://doi.org/10.18632/aging.101414">10.18632/aging.101414</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29676998">http://www.ncbi.nlm.nih.gov/pubmed/29676998</a>.</p>
</div>
<div>
<p>139. Lu AT, Quach A, Wilson JG, Reiner AP, Aviv A, Raj K, Hou L, Baccarelli AA, Li Y, Stewart JD, Whitsel EA, Assimes TL, Ferrucci L, Horvath S: <strong>DNA methylation GrimAge strongly predicts lifespan and healthspan</strong>. <em>Aging</em> 2019, <strong>11</strong>:303–327<a href="https://doi.org/10.18632/aging.101684">10.18632/aging.101684</a>Available: <a href="http://www.aging-us.com/article/101684/text">http://www.aging-us.com/article/101684/text</a>.</p>
</div>
<div>
<p>140. Zhang Y, Hapala J, Brenner H, Wagner W: <strong>Individual CpG sites that are associated with age and life expectancy become hypomethylated upon aging</strong>. <em>Clinical Epigenetics</em> 2017, <strong>9</strong>:9<a href="https://doi.org/10.1186/s13148-017-0315-9">10.1186/s13148-017-0315-9</a>Available: <a href="http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9">http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9</a>.</p>
</div>
<div>
<p>141. Jones MJ, Goodman SJ, Kobor MS: <strong>DNA methylation and healthy human aging</strong>. <em>Aging Cell</em> 2015, <strong>14</strong>:924–932<a href="https://doi.org/10.1111/acel.12349">10.1111/acel.12349</a>Available: <a href="http://doi.wiley.com/10.1111/acel.12349">http://doi.wiley.com/10.1111/acel.12349</a>.</p>
</div>
<div>
<p>142. Rimmelé P, Bigarella CL, Liang R, Izac B, Dieguez-Gonzalez R, Barbet G, Donovan M, Brugnara C, Blander JM, Sinclair DA, Ghaffari S: <strong>Aging-like phenotype and defective lineage specification in SIRT1-deleted hematopoietic stem and progenitor cells.</strong> <em>Stem cell reports</em> 2014, <strong>3</strong>:44–59<a href="https://doi.org/10.1016/j.stemcr.2014.04.015">10.1016/j.stemcr.2014.04.015</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25068121">http://www.ncbi.nlm.nih.gov/pubmed/25068121</a>.</p>
</div>
<div>
<p>143. Izzo F, Lee SC, Poran A, Chaligne R, Gaiti F, Gross B, Murali RR, Deochand SD, Ang C, Jones PW, Nam AS, Kim K-T, Kothen-Hill S, Schulman RC, Ki M, Lhoumaud P, Skok JA, Viny AD, Levine RL, Kenigsberg E, Abdel-Wahab O, Landau DA: <strong>DNA methylation disruption reshapes the hematopoietic differentiation landscape</strong>. <em>Nature Genetics</em> 2020, <strong>52</strong><a href="https://doi.org/10.1038/s41588-020-0595-4">10.1038/s41588-020-0595-4</a>Available: <a href="http://dx.doi.org/10.1038/s41588-020-0595-4">http://dx.doi.org/10.1038/s41588-020-0595-4</a>.</p>
</div>
<div>
<p>144. Geiger H, Haan G de, Florian MC: <strong>The ageing haematopoietic stem cell compartment</strong>. <em>Nature Reviews Immunology</em> 2013, <strong>13</strong>:376–389<a href="https://doi.org/10.1038/nri3433">10.1038/nri3433</a>Available: <a href="http://dx.doi.org/10.1038/nri3433">http://dx.doi.org/10.1038/nri3433</a>.</p>
</div>
<div>
<p>145. Jaffe AE, Irizarry RA: <strong>Accounting for cellular heterogeneity is critical in epigenome-wide association studies.</strong> <em>Genome biology</em> 2014, <strong>15</strong>:R31<a href="https://doi.org/10.1186/gb-2014-15-2-r31">10.1186/gb-2014-15-2-r31</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24495553">http://www.ncbi.nlm.nih.gov/pubmed/24495553</a>.</p>
</div>
<div>
<p>146. Houseman EA, Accomando WP, Koestler DC, Christensen BC, Marsit CJ, Nelson HH, Wiencke JK, Kelsey KT: <strong>DNA methylation arrays as surrogate measures of cell mixture distribution</strong>. <em>BMC Bioinformatics</em> 2012, <strong>13</strong>:86<a href="https://doi.org/10.1186/1471-2105-13-86">10.1186/1471-2105-13-86</a>Available: <a href="http://www.biomedcentral.com/1471-2105/13/86">http://www.biomedcentral.com/1471-2105/13/86</a>.</p>
</div>
<div>
<p>147. Schuettengruber B, Chourrout D, Vervoort M, Leblanc B, Cavalli G: <strong>Genome Regulation by Polycomb and Trithorax Proteins</strong>. <em>Cell</em> 2007, <strong>128</strong>:735–745<a href="https://doi.org/10.1016/j.cell.2007.02.009">10.1016/j.cell.2007.02.009</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17320510">http://www.ncbi.nlm.nih.gov/pubmed/17320510</a>.</p>
</div>
<div>
<p>148. Hahn O, Grönke S, Stubbs TM, Ficz G, Hendrich O, Krueger F, Andrews S, Zhang Q, Wakelam MJ, Beyer A, Reik W, Partridge L: <strong>Dietary restriction protects from age-associated DNA methylation and induces epigenetic reprogramming of lipid metabolism</strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:56<a href="https://doi.org/10.1186/s13059-017-1187-1">10.1186/s13059-017-1187-1</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1187-1">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1187-1</a>.</p>
</div>
<div>
<p>149. Horvath S, Raj K: <strong>DNA methylation-based biomarkers and the epigenetic clock theory of ageing</strong>. <em>Nature Reviews Genetics</em> 2018, <strong>19</strong>:371–384<a href="https://doi.org/10.1038/s41576-018-0004-3">10.1038/s41576-018-0004-3</a>Available: <a href="http://www.nature.com/articles/s41576-018-0004-3">http://www.nature.com/articles/s41576-018-0004-3</a>.</p>
</div>
<div>
<p>150. Bell CG, Lowe R, Adams PD, Baccarelli AA, Beck S, Bell JT, Christensen BC, Gladyshev VN, Heijmans BT, Horvath S, Ideker T, Issa J-PJ, Kelsey KT, Marioni RE, Reik W, Relton CL, Schalkwyk LC, Teschendorff AE, Wagner W, Zhang K, Rakyan VK: <strong>DNA methylation aging clocks: challenges and recommendations.</strong> <em>Genome biology</em> 2019, <strong>20</strong>:249<a href="https://doi.org/10.1186/s13059-019-1824-y">10.1186/s13059-019-1824-y</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31767039">http://www.ncbi.nlm.nih.gov/pubmed/31767039</a>.</p>
</div>
<div>
<p>151. Bell CG, Xia Y, Yuan W, Gao F, Ward K, Roos L, Mangino M, Hysi PG, Bell J, Wang J, Spector TD: <strong>Novel regional age-associated DNA methylation changes within human common disease-associated loci</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:193<a href="https://doi.org/10.1186/s13059-016-1051-8">10.1186/s13059-016-1051-8</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27663977">http://www.ncbi.nlm.nih.gov/pubmed/27663977</a>.</p>
</div>
<div>
<p>152. Harvey NC, Javaid K, Bishop N, Kennedy S, Papageorghiou AT, Fraser R, Gandhi SV, Schoenmakers I, Prentice A, Cooper C: <strong>MAVIDOS Maternal Vitamin D Osteoporosis Study: study protocol for a randomized controlled trial. The MAVIDOS Study Group</strong>. <em>Trials</em> 2012, <strong>13</strong>:13<a href="https://doi.org/10.1186/1745-6215-13-13">10.1186/1745-6215-13-13</a>Available: <a href="http://trialsjournal.biomedcentral.com/articles/10.1186/1745-6215-13-13">http://trialsjournal.biomedcentral.com/articles/10.1186/1745-6215-13-13</a>.</p>
</div>
<div>
<p>153. Cooper C, Harvey NC, Bishop NJ, Kennedy S, Papageorghiou AT, Schoenmakers I, Fraser R, Gandhi SV, Carr A, D’Angelo S, Crozier SR, Moon RJ, Arden NK, Dennison EM, Godfrey KM, Inskip HM, Prentice A, Mughal MZ, Eastell R, Reid DM, Javaid MK: <strong>Maternal gestational vitamin D supplementation and offspring bone health (MAVIDOS): a multicentre, double-blind, randomised placebo-controlled trial</strong>. <em>The Lancet Diabetes &amp; Endocrinology</em> 2016, <strong>4</strong>:393–402<a href="https://doi.org/10.1016/S2213-8587(16)00044-9">10.1016/S2213-8587(16)00044-9</a>Available: <a href="http://dx.doi.org/10.1016/S2213-8587(16)00044-9">http://dx.doi.org/10.1016/S2213-8587(16)00044-9</a>.</p>
</div>
<div>
<p>154. Moran S, Arribas C, Esteller M: <strong>Validation of a DNA methylation microarray for 850,000 CpG sites of the human genome enriched in enhancer sequences.</strong> <em>Epigenomics</em> 2016, <strong>8</strong>:389–99<a href="https://doi.org/10.2217/epi.15.114">10.2217/epi.15.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26673039">http://www.ncbi.nlm.nih.gov/pubmed/26673039</a>.</p>
</div>
<div>
<p>155. Gunderson KL: <strong>Decoding Randomly Ordered DNA Arrays</strong>. <em>Genome Research</em> 2004, <strong>14</strong>:870–877<a href="https://doi.org/10.1101/gr.2255804">10.1101/gr.2255804</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.2255804">http://www.genome.org/cgi/doi/10.1101/gr.2255804</a>.</p>
</div>
<div>
<p>156. Michael KL, Taylor LC, Schultz SL, Walt DR: <strong>Randomly Ordered Addressable High-Density Optical Sensor Arrays</strong>. <em>Analytical Chemistry</em> 1998, <strong>70</strong>:1242–1248<a href="https://doi.org/10.1021/ac971343r">10.1021/ac971343r</a>Available: <a href="https://pubs.acs.org/doi/10.1021/ac971343r">https://pubs.acs.org/doi/10.1021/ac971343r</a>.</p>
</div>
<div>
<p>157. Clark C, Palta P, Joyce CJ, Scott C, Grundberg E, Deloukas P, Palotie A, Coffey AJ: <strong>A Comparison of the Whole Genome Approach of MeDIP-Seq to the Targeted Approach of the Infinium HumanMethylation450 BeadChip for Methylome Profiling</strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e50233<a href="https://doi.org/10.1371/journal.pone.0050233">10.1371/journal.pone.0050233</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0050233">https://dx.plos.org/10.1371/journal.pone.0050233</a>.</p>
</div>
<div>
<p>158. Smith ML, Baggerly KA, Bengtsson H, Ritchie ME, Hansen KD: <strong>illuminaio: An open source IDAT parsing tool for Illumina microarrays</strong>. <em>F1000Research</em> 2013, <strong>2</strong>:1–8<a href="https://doi.org/10.12688/f1000research.2-264.v1">10.12688/f1000research.2-264.v1</a>.</p>
</div>
<div>
<p>159. Dedeurwaerder S, Defrance M, Calonne E, Denis H, Sotiriou C, Fuks F: <strong>Evaluation of the Infinium Methylation 450K technology</strong>. <em>Epigenomics</em> 2011, <strong>3</strong>:771–784<a href="https://doi.org/10.2217/epi.11.105">10.2217/epi.11.105</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi.11.105">https://www.futuremedicine.com/doi/10.2217/epi.11.105</a>.</p>
</div>
<div>
<p>160. Fortin J-P, Triche TJ, Hansen KD: <strong>Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi</strong>. <em>Bioinformatics</em> 2016,:btw691<a href="https://doi.org/10.1093/bioinformatics/btw691">10.1093/bioinformatics/btw691</a>Available: <a href="http://biorxiv.org/lookup/doi/10.1101/065490">http://biorxiv.org/lookup/doi/10.1101/065490</a>.</p>
</div>
<div>
<p>161. Teschendorff AE, Marabita F, Lechner M, Bartlett T, Tegner J, Gomez-Cabrero D, Beck S: <strong>A beta-mixture quantile normalization method for correcting probe design bias in Illumina Infinium 450 k DNA methylation data</strong>. <em>Bioinformatics</em> 2013, <strong>29</strong>:189–196<a href="https://doi.org/10.1093/bioinformatics/bts680">10.1093/bioinformatics/bts680</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bts680">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bts680</a>.</p>
</div>
<div>
<p>162. Leek JT, Scharpf RB, Bravo HC, Simcha D, Langmead B, Johnson WE, Geman D, Baggerly K, Irizarry RA: <strong>Tackling the widespread and critical impact of batch effects in high-throughput data</strong>. <em>Nature Reviews Genetics</em> 2010, <strong>11</strong>:733–739<a href="https://doi.org/10.1038/nrg2825">10.1038/nrg2825</a>Available: <a href="http://www.nature.com/articles/nrg2825">http://www.nature.com/articles/nrg2825</a>.</p>
</div>
<div>
<p>163. Fortin J-P, Labbe A, Lemire M, Zanke BW, Hudson TJ, Fertig EJ, Greenwood CMT, Hansen KD: <strong>Functional normalization of 450k methylation array data improves replication in large cancer studies</strong>. <em>Genome Biology</em> 2014, <strong>15</strong>:503<a href="https://doi.org/10.1186/s13059-014-0503-2">10.1186/s13059-014-0503-2</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0503-2">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0503-2</a>.</p>
</div>
<div>
<p>164. Pidsley R, Zotenko E, Peters TJ, Lawrence MG, Risbridger GP, Molloy P, Van Djik S, Muhlhausler B, Stirzaker C, Clark SJ: <strong>Critical evaluation of the Illumina MethylationEPIC BeadChip microarray for whole-genome DNA methylation profiling.</strong> <em>Genome biology</em> 2016, <strong>17</strong>:208<a href="https://doi.org/10.1186/s13059-016-1066-1">10.1186/s13059-016-1066-1</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27717381">http://www.ncbi.nlm.nih.gov/pubmed/27717381</a>.</p>
</div>
<div>
<p>165. Zhou W, Laird PW, Shen H: <strong>Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes</strong>. <em>Nucleic Acids Research</em> 2016, <strong>45</strong>:gkw967<a href="https://doi.org/10.1093/nar/gkw967">10.1093/nar/gkw967</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27924034">http://www.ncbi.nlm.nih.gov/pubmed/27924034</a>.</p>
</div>
<div>
<p>166. Andrews SV, Ladd-Acosta C, Feinberg AP, Hansen KD, Fallin MD: <strong>“Gap hunting” to characterize clustered probe signals in Illumina methylation array data</strong>. <em>Epigenetics &amp; Chromatin</em> 2016, <strong>9</strong>:56<a href="https://doi.org/10.1186/s13072-016-0107-z">10.1186/s13072-016-0107-z</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z</a>.</p>
</div>
<div>
<p>167. Taiwo O, Wilson GA, Morris T, Seisenberger S, Reik W, Pearce D, Beck S, Butcher LM: <strong>Methylome analysis using MeDIP-seq with low DNA concentrations</strong>. <em>Nature Protocols</em> 2012, <strong>7</strong>:617–636<a href="https://doi.org/10.1038/nprot.2012.012">10.1038/nprot.2012.012</a>Available: <a href="http://www.nature.com/articles/nprot.2012.012">http://www.nature.com/articles/nprot.2012.012</a>.</p>
</div>
<div>
<p>168. Chavez L, Jozefczuk J, Grimm C, Dietrich J, Timmermann B, Lehrach H, Herwig R, Adjaye J: <strong>Computational analysis of genome-wide DNA methylation during the differentiation of human embryonic stem cells along the endodermal lineage</strong>. <em>Genome Research</em> 2010, <strong>20</strong>:1441–1450<a href="https://doi.org/10.1101/gr.110114.110">10.1101/gr.110114.110</a>Available: <a href="http://genome.cshlp.org/cgi/doi/10.1101/gr.110114.110">http://genome.cshlp.org/cgi/doi/10.1101/gr.110114.110</a>.</p>
</div>
<div>
<p>169. Andrews S: <strong>FastQC</strong>. 2010, Available: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</p>
</div>
<div>
<p>170. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R: <strong>The Sequence Alignment/Map format and SAMtools</strong>. <em>Bioinformatics</em> 2009, <strong>25</strong>:2078–2079<a href="https://doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btp352">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btp352</a>.</p>
</div>
<div>
<p>171. Quinlan AR, Hall IM: <strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong>. <em>Bioinformatics</em> 2010, <strong>26</strong>:841–842<a href="https://doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a>Available: <a href="http://bedtools.readthedocs.io/en/latest/index.html">http://bedtools.readthedocs.io/en/latest/index.html</a>.</p>
</div>
<div>
<p>172. Moayyeri A, Hammond CJ, Valdes AM, Spector TD: <strong>Cohort Profile: TwinsUK and Healthy Ageing Twin Study</strong>. <em>International Journal of Epidemiology</em> 2013, <strong>42</strong>:76–85<a href="https://doi.org/10.1093/ije/dyr207">10.1093/ije/dyr207</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22253318">http://www.ncbi.nlm.nih.gov/pubmed/22253318</a>.</p>
</div>
<div>
<p>173. Korbie D, Lin E, Wall D, Nair SS, Stirzaker C, Clark SJ, Trau M: <strong>Multiplex bisulfite PCR resequencing of clinical FFPE DNA</strong>. <em>Clinical Epigenetics</em> 2015, <strong>7</strong>:28<a href="https://doi.org/10.1186/s13148-015-0067-3">10.1186/s13148-015-0067-3</a>Available: <a href="https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-015-0067-3">https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-015-0067-3</a>.</p>
</div>
<div>
<p>174. Tran H, Porter J, Sun M-a, Xie H, Zhang L: <strong>Objective and Comprehensive Evaluation of Bisulfite Short Read Mapping Tools</strong>. <em>Advances in Bioinformatics</em> 2014, <strong>2014</strong>:1–11<a href="https://doi.org/10.1155/2014/472045">10.1155/2014/472045</a>Available: <a href="http://www.hindawi.com/journals/abi/2014/472045/">http://www.hindawi.com/journals/abi/2014/472045/</a>.</p>
</div>
<div>
<p>175. Böttcher R, Amberg R, Ruzius FP, Guryev V, Verhaegh WFJ, Beyerlein P, Zaag PJ van der: <strong>Using a priori knowledge to align sequencing reads to their exact genomic position</strong>. <em>Nucleic Acids Research</em> 2012, <strong>40</strong>:e125–e125<a href="https://doi.org/10.1093/nar/gks393">10.1093/nar/gks393</a>Available: <a href="https://academic.oup.com/nar/article/40/16/e125/1026881">https://academic.oup.com/nar/article/40/16/e125/1026881</a>.</p>
</div>
<div>
<p>176. Krueger F, Andrews SR: <strong>Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications</strong>. <em>Bioinformatics</em> 2011, <strong>27</strong>:1571–1572<a href="https://doi.org/10.1093/bioinformatics/btr167">10.1093/bioinformatics/btr167</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr167">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr167</a>.</p>
</div>
<div>
<p>177. Li L-C, Dahiya R: <strong>MethPrimer: designing primers for methylation PCRs.</strong> <em>Bioinformatics (Oxford, England)</em> 2002, <strong>18</strong>:1427–31Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12424112">http://www.ncbi.nlm.nih.gov/pubmed/12424112</a>.</p>
</div>
<div>
<p>178. Staa TP van, Dennison EM, Leufkens HGM, Cooper C: <strong>Epidemiology of fractures in England and Wales</strong>. <em>Bone</em> 2001, <strong>29</strong>:517–522<a href="https://doi.org/10.1016/S8756-3282(01)00614-7">10.1016/S8756-3282(01)00614-7</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S8756328201006147">https://linkinghub.elsevier.com/retrieve/pii/S8756328201006147</a>.</p>
</div>
<div>
<p>179. Streubel PN, Ricci WM, Wong A, Gardner MJ: <strong>Mortality After Distal Femur Fractures in Elderly Patients</strong>. <em>Clinical Orthopaedics and Related Research</em> 2011, <strong>469</strong>:1188–1196<a href="https://doi.org/10.1007/s11999-010-1530-2">10.1007/s11999-010-1530-2</a>Available: <a href="http://link.springer.com/10.1007/s11999-010-1530-2">http://link.springer.com/10.1007/s11999-010-1530-2</a>.</p>
</div>
<div>
<p>180. Thompson E, Greenspan S: <em>National Osteoporosis Foundation Annual Report (2018)</em>. National Osteoporosis Foundation; 2018 Available: <a href="https://cdn.nof.org/wp-content/uploads/2018%7B\_%7DNOF%7B\_%7DAnnual%7B\_%7Dreport%7B\_%7DFINAL.pdf">https://cdn.nof.org/wp-content/uploads/2018{\_}NOF{\_}Annual{\_}report{\_}FINAL.pdf</a>.</p>
</div>
<div>
<p>181. Hernandez CJ, Beaupré GS, Carter DR: <strong>A theoretical analysis of the relative influences of peak BMD, age-related bone loss and menopause on the development of osteoporosis</strong>. <em>Osteoporosis International</em> 2003, <strong>14</strong>:843–847<a href="https://doi.org/10.1007/s00198-003-1454-8">10.1007/s00198-003-1454-8</a>.</p>
</div>
<div>
<p>182. Nguyen TV, Center JR, Eisman JA: <strong>Femoral Neck Bone Loss Predicts Fracture Risk Independent of Baseline BMD</strong>. <em>Journal of Bone and Mineral Research</em> 2005, <strong>20</strong>:1195–1201<a href="https://doi.org/10.1359/JBMR.050215">10.1359/JBMR.050215</a>Available: <a href="http://doi.wiley.com/10.1359/JBMR.050215">http://doi.wiley.com/10.1359/JBMR.050215</a>.</p>
</div>
<div>
<p>183. Finkelstein JS, Brockwell SE, Mehta V, Greendale GA, Sowers MR, Ettinger B, Lo JC, Johnston JM, Cauley JA, Danielson ME, Neer RM: <strong>Bone Mineral Density Changes during the Menopause Transition in a Multiethnic Cohort of Women</strong>. <em>The Journal of Clinical Endocrinology &amp; Metabolism</em> 2008, <strong>93</strong>:861–868<a href="https://doi.org/10.1210/jc.2007-1876">10.1210/jc.2007-1876</a>Available: <a href="https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2007-1876">https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2007-1876</a>.</p>
</div>
<div>
<p>184. Hui SL, Slemenda CW, Johnston CC: <strong>The contribution of bone loss to postmenopausal osteoporosis</strong>. <em>Osteoporosis International</em> 1990, <strong>1</strong>:30–34<a href="https://doi.org/10.1007/BF01880413">10.1007/BF01880413</a>.</p>
</div>
<div>
<p>185. Harvey N, Dennison E, Cooper C: <strong>Osteoporosis: A Lifecourse Approach</strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:1917–1925<a href="https://doi.org/10.1002/jbmr.2286">10.1002/jbmr.2286</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2286">http://doi.wiley.com/10.1002/jbmr.2286</a>.</p>
</div>
<div>
<p>186. Bikle DD: <strong>Vitamin D and Bone</strong>. <em>Current Osteoporosis Reports</em> 2012, <strong>10</strong>:151–159<a href="https://doi.org/10.1007/s11914-012-0098-z">10.1007/s11914-012-0098-z</a>Available: <a href="http://link.springer.com/10.1007/s11914-012-0098-z">http://link.springer.com/10.1007/s11914-012-0098-z</a>.</p>
</div>
<div>
<p>187. Christakos S, Dhawan P, Porta A, Mady LJ, Seth T: <strong>Vitamin D and intestinal calcium absorption</strong>. <em>Molecular and Cellular Endocrinology</em> 2011, <strong>347</strong>:25–29<a href="https://doi.org/10.1016/j.mce.2011.05.038">10.1016/j.mce.2011.05.038</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0303720711002930">https://linkinghub.elsevier.com/retrieve/pii/S0303720711002930</a>.</p>
</div>
<div>
<p>188. Fukumoto S: <strong>Phosphate metabolism and vitamin D</strong>. <em>BoneKEy Reports</em> 2014, <strong>3</strong>:1–5<a href="https://doi.org/10.1038/bonekey.2013.231">10.1038/bonekey.2013.231</a>Available: <a href="http://dx.doi.org/10.1038/bonekey.2013.231">http://dx.doi.org/10.1038/bonekey.2013.231</a>.</p>
</div>
<div>
<p>189. Baird J, Kurshid MA, Kim M, Harvey N, Dennison E, Cooper C: <strong>Does birthweight predict bone mass in adulthood? A systematic review and meta-analysis</strong>. <em>Osteoporosis International</em> 2011, <strong>22</strong>:1323–1334<a href="https://doi.org/10.1007/s00198-010-1344-9">10.1007/s00198-010-1344-9</a>.</p>
</div>
<div>
<p>190. Cooper C, Eriksson JG, Forsén T, Osmond C, Tuomilehto J, Barker DJP: <strong>Maternal height, childhood growth and risk of hip fracture in later life: A longitudinal study</strong>. <em>Osteoporosis International</em> 2001, <strong>12</strong>:623–629<a href="https://doi.org/10.1007/s001980170061">10.1007/s001980170061</a>.</p>
</div>
<div>
<p>191. Inskip HM, Godfrey KM, Robinson SM, Law CM, Jp D: <strong>Europe PMC Funders Group Cohort Profile : The Southampton Women ’ s Survey</strong>. <em>International Journal of Epidemiology</em> 2006, <strong>35</strong>:42–48<a href="https://doi.org/10.1093/ije/dyi202">10.1093/ije/dyi202</a>.</p>
</div>
<div>
<p>192. Harvey NC, Mahon PA, Robinson SM, Nisbet CE, Javaid MK, Crozier SR, Inskip HM, Godfrey KM, Arden NK, Dennison EM, Cooper C, Taylor P, Greenaway LJ, Hanson M, Barker DJP, Law CM: <strong>Different indices of fetal growth predict bone size and volumetric density at 4 years of age</strong>. <em>Journal of Bone and Mineral Research</em> 2010, <strong>25</strong>:920–927<a href="https://doi.org/10.1359/jbmr.091022">10.1359/jbmr.091022</a>.</p>
</div>
<div>
<p>193. Harvey NC, Cole ZA, Crozier SR, Ntani G, Mahon PA, Robinson SM, Inskip HM, Godfrey KM, Dennison EM, Cooper C: <strong>Fetal and infant growth predict hip geometry at 6 y old: Findings from the Southampton Women’s Survey</strong>. <em>Pediatric Research</em> 2013, <strong>74</strong>:450–456<a href="https://doi.org/10.1038/pr.2013.119">10.1038/pr.2013.119</a>.</p>
</div>
<div>
<p>194. Mahon P, Harvey N, Crozier S, Inskip H, Robinson S, Arden N, Swaminathan R, Cooper C, Godfrey K: <strong>Low maternal vitamin D status and fetal bone development: Cohort study</strong>. <em>Journal of Bone and Mineral Research</em> 2010, <strong>25</strong>:14–19<a href="https://doi.org/10.1359/jbmr.090701">10.1359/jbmr.090701</a>Available: <a href="http://doi.wiley.com/10.1359/jbmr.090701">http://doi.wiley.com/10.1359/jbmr.090701</a>.</p>
</div>
<div>
<p>195. Viljakainen HT, Saarnio E, Hytinantti T, Miettinen M, Surcel H, Mäkitie O, Andersson S, Laitinen K, Lamberg-Allardt C: <strong>Maternal Vitamin D Status Determines Bone Variables in the Newborn</strong>. <em>The Journal of Clinical Endocrinology &amp; Metabolism</em> 2010, <strong>95</strong>:1749–1757<a href="https://doi.org/10.1210/jc.2009-1391">10.1210/jc.2009-1391</a>Available: <a href="https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2009-1391">https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2009-1391</a>.</p>
</div>
<div>
<p>196. Viljakainen HT, Korhonen T, Hytinantti T, Laitinen EKA, Andersson S, Mäkitie O, Lamberg-Allardt C: <strong>Maternal vitamin D status affects bone growth in early childhood—a prospective cohort study</strong>. <em>Osteoporosis International</em> 2011, <strong>22</strong>:883–891<a href="https://doi.org/10.1007/s00198-010-1499-4">10.1007/s00198-010-1499-4</a>Available: <a href="http://link.springer.com/10.1007/s00198-010-1499-4">http://link.springer.com/10.1007/s00198-010-1499-4</a>.</p>
</div>
<div>
<p>197. Zhu K, Whitehouse AJ, Hart PH, Kusel M, Mountain J, Lye S, Pennell C, Walsh JP: <strong>Maternal Vitamin D Status During Pregnancy and Bone Mass in Offspring at 20 Years of Age: A Prospective Cohort Study</strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:1088–1095<a href="https://doi.org/10.1002/jbmr.2138">10.1002/jbmr.2138</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2138">http://doi.wiley.com/10.1002/jbmr.2138</a>.</p>
</div>
<div>
<p>198. Kip SN, Strehler EE: <strong>Vitamin D 3 upregulates plasma membrane Ca 2+ -ATPase expression and potentiates apico-basal Ca 2+ flux in MDCK cells</strong>. <em>American Journal of Physiology-Renal Physiology</em> 2004, <strong>286</strong>:F363–F369<a href="https://doi.org/10.1152/ajprenal.00076.2003">10.1152/ajprenal.00076.2003</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/14583431">http://www.ncbi.nlm.nih.gov/pubmed/14583431</a>.</p>
</div>
<div>
<p>199. Burdge GC, Slater-Jefferies J, Torrens C, Phillips ES, Hanson MA, Lillycrop KA: <strong>Dietary protein restriction of pregnant rats in the F 0 generation induces altered methylation of hepatic gene promoters in the adult male offspring in the F 1 and F 2 generations</strong>. <em>British Journal of Nutrition</em> 2007, <strong>97</strong>:435–439<a href="https://doi.org/10.1017/S0007114507352392">10.1017/S0007114507352392</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507352392/type/journal%7B\_%7Darticle">https://www.cambridge.org/core/product/identifier/S0007114507352392/type/journal{\_}article</a>.</p>
</div>
<div>
<p>200. Lillycrop KA, Phillips ES, Torrens C, Hanson MA, Jackson AA, Burdge GC: <strong>Feeding pregnant rats a protein-restricted diet persistently alters the methylation of specific cytosines in the hepatic PPAR<span class="math inline">\(\alpha\)</span> promoter of the offspring</strong>. <em>British Journal of Nutrition</em> 2008, <strong>100</strong>:278–282<a href="https://doi.org/10.1017/S0007114507894438">10.1017/S0007114507894438</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507894438/type/journal%7B\_%7Darticle">https://www.cambridge.org/core/product/identifier/S0007114507894438/type/journal{\_}article</a>.</p>
</div>
<div>
<p>201. Burdge GC, Hanson MA, Slater-Jefferies JL, Lillycrop KA: <strong>Epigenetic regulation of transcription: a mechanism for inducing variations in phenotype (fetal programming) by differences in nutrition during early life?</strong> <em>British Journal of Nutrition</em> 2007, <strong>97</strong>:1036–1046<a href="https://doi.org/10.1017/S0007114507682920">10.1017/S0007114507682920</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507682920/type/journal%7B\_%7Darticle">https://www.cambridge.org/core/product/identifier/S0007114507682920/type/journal{\_}article</a>.</p>
</div>
<div>
<p>202. Holroyd C, Harvey N, Dennison E, Cooper C: <strong>Epigenetic influences in the developmental origins of osteoporosis</strong>. <em>Osteoporosis International</em> 2012, <strong>23</strong>:401–410<a href="https://doi.org/10.1007/s00198-011-1671-5">10.1007/s00198-011-1671-5</a>.</p>
</div>
<div>
<p>203. Harvey NC, Sheppard A, Godfrey KM, McLean C, Garratt E, Ntani G, Davies L, Murray R, Inskip HM, Gluckman PD, Hanson MA, Lillycrop KA, Cooper C: <strong>Childhood Bone Mineral Content Is Associated With Methylation Status of the RXRA Promoter at Birth</strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:600–607<a href="https://doi.org/10.1002/jbmr.2056">10.1002/jbmr.2056</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2056">http://doi.wiley.com/10.1002/jbmr.2056</a>.</p>
</div>
<div>
<p>204. Curtis EM, Murray R, Titcombe P, Cook E, Clarke-Harris R, Costello P, Garratt E, Holbrook JD, Barton S, Inskip H, Godfrey KM, Bell CG, Cooper C, Lillycrop KA, Harvey NC: <strong>Perinatal DNA Methylation at CDKN2A Is Associated With Offspring Bone Mass: Findings From the Southampton Women’s Survey</strong>. <em>Journal of Bone and Mineral Research</em> 2017, <strong>32</strong>:2030–2040<a href="https://doi.org/10.1002/jbmr.3153">10.1002/jbmr.3153</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.3153">http://doi.wiley.com/10.1002/jbmr.3153</a>.</p>
</div>
<div>
<p>205. Holland ML, Lowe R, Caton PW, Gemma C, Carbajosa G, Danson AF, Carpenter AAM, Loche E, Ozanne SE, Rakyan VK: <strong>Early-life nutrition modulates the epigenetic state of specific rDNA genetic variants in mice</strong>. <em>Science</em> 2016, <strong>353</strong>:495–498<a href="https://doi.org/10.1126/science.aaf7040">10.1126/science.aaf7040</a>Available: <a href="http://www.sciencemag.org/lookup/doi/10.1126/science.aaf7040">http://www.sciencemag.org/lookup/doi/10.1126/science.aaf7040</a>.</p>
</div>
<div>
<p>206. Dogan MV, Beach SRH, Philibert RA: <strong>Genetically contextual effects of smoking on genome wide DNA methylation</strong>. <em>American Journal of Medical Genetics Part B: Neuropsychiatric Genetics</em> 2017, <strong>174</strong>:595–607<a href="https://doi.org/10.1002/ajmg.b.32565">10.1002/ajmg.b.32565</a>Available: <a href="http://doi.wiley.com/10.1002/ajmg.b.32565">http://doi.wiley.com/10.1002/ajmg.b.32565</a>.</p>
</div>
<div>
<p>207. Dai L, Mehta A, Mordukhovich I, Just AC, Shen J, Hou L, Koutrakis P, Sparrow D, Vokonas PS, Baccarelli AA, Schwartz JD: <strong>Differential DNA methylation and PM 2.5 species in a 450K epigenome-wide association study</strong>. <em>Epigenetics</em> 2017, <strong>12</strong>:139–148<a href="https://doi.org/10.1080/15592294.2016.1271853">10.1080/15592294.2016.1271853</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2016.1271853">https://www.tandfonline.com/doi/full/10.1080/15592294.2016.1271853</a>.</p>
</div>
<div>
<p>208. Wahl S, Drong A, Lehne B, Loh M, Scott WR, Kunze S, Tsai P-C, Ried JS, Zhang W, Yang Y, Tan S, Fiorito G, Franke L, Guarrera S, Kasela S, Kriebel J, Richmond RC, Adamo M, Afzal U, Ala-Korpela M, Albetti B, Ammerpohl O, Apperley JF, Beekman M, Bertazzi PA, Black SL, Blancher C, Bonder M-J, Brosch M, Carstensen-Kirberg M, et al.: <strong>Epigenome-wide association study of body mass index, and the adverse outcomes of adiposity.</strong> <em>Nature</em> 2017, <strong>541</strong>:81–86<a href="https://doi.org/10.1038/nature20784">10.1038/nature20784</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28002404">http://www.ncbi.nlm.nih.gov/pubmed/28002404</a>.</p>
</div>
<div>
<p>209. Min J, Hemani G, Smith GD, Relton CL, Suderman M: <strong>Meffil: efficient normalisation and analysis of very large DNA methylation samples</strong>. <em>bioRxiv</em> 2017, <strong>44</strong>:125963<a href="https://doi.org/10.1101/125963">10.1101/125963</a>Available: <a href="https://www.biorxiv.org/content/early/2017/04/27/125963">https://www.biorxiv.org/content/early/2017/04/27/125963</a>.</p>
</div>
<div>
<p>210. Cardenas A, Allard C, Doyon M, Houseman EA, Bakulski KM, Perron P, Bouchard L, Hivert M-F: <strong>Validation of a DNA methylation reference panel for the estimation of nucleated cells types in cord blood.</strong> <em>Epigenetics</em> 2016, <strong>11</strong>:773–779<a href="https://doi.org/10.1080/15592294.2016.1233091">10.1080/15592294.2016.1233091</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27668573">http://www.ncbi.nlm.nih.gov/pubmed/27668573</a>.</p>
</div>
<div>
<p>211. Goede OM de, Razzaghian HR, Price EM, Jones MJ, Kobor MS, Robinson WP, Lavoie PM: <strong>Nucleated red blood cells impact DNA methylation and expression analyses of cord blood hematopoietic cells</strong>. <em>Clinical Epigenetics</em> 2015, <strong>7</strong>:95<a href="https://doi.org/10.1186/s13148-015-0129-6">10.1186/s13148-015-0129-6</a>Available: <a href="http://dx.doi.org/10.1186/s13148-015-0129-6">http://dx.doi.org/10.1186/s13148-015-0129-6</a>.</p>
</div>
<div>
<p>212. Bakulski KM, Feinberg JI, Andrews SV, Yang J, Brown S, L. McKenney S, Witter F, Walston J, Feinberg AP, Fallin MD: <strong>DNA methylation of cord blood cell types: Applications for mixed cell birth studies</strong>. <em>Epigenetics</em> 2016, <strong>11</strong>:354–362<a href="https://doi.org/10.1080/15592294.2016.1161875">10.1080/15592294.2016.1161875</a>Available: <a href="http://dx.doi.org/10.1080/15592294.2016.1161875">http://dx.doi.org/10.1080/15592294.2016.1161875</a>.</p>
</div>
<div>
<p>213. Gervin K, Page CM, Aass HCD, Jansen MA, Fjeldstad HE, Andreassen BK, Duijts L, Meurs JB van, Zelm MC van, Jaddoe VW, Nordeng H, Knudsen GP, Magnus P, Nystad W, Staff AC, Felix JF, Lyle R: <strong>Cell type specific DNA methylation in cord blood: A 450K-reference data set and cell count-based validation of estimated cell type composition</strong>. <em>Epigenetics</em> 2016, <strong>11</strong>:690–698<a href="https://doi.org/10.1080/15592294.2016.1214782">10.1080/15592294.2016.1214782</a>Available: <a href="http://dx.doi.org/10.1080/15592294.2016.1214782">http://dx.doi.org/10.1080/15592294.2016.1214782</a>.</p>
</div>
<div>
<p>214. Leek JT, Storey JD: <strong>Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis</strong>. <em>PLoS Genetics</em> 2007, <strong>3</strong>:e161<a href="https://doi.org/10.1371/journal.pgen.0030161">10.1371/journal.pgen.0030161</a>Available: <a href="https://dx.plos.org/10.1371/journal.pgen.0030161">https://dx.plos.org/10.1371/journal.pgen.0030161</a>.</p>
</div>
<div>
<p>215. Teschendorff AE, Zhuang J, Widschwendter M: <strong>Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies</strong>. <em>Bioinformatics</em> 2011, <strong>27</strong>:1496–1505<a href="https://doi.org/10.1093/bioinformatics/btr171">10.1093/bioinformatics/btr171</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr171">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr171</a>.</p>
</div>
<div>
<p>216. McGregor K, Bernatsky S, Colmegna I, Hudson M, Pastinen T, Labbe A, Greenwood CMT: <strong>An evaluation of methods correcting for cell-type heterogeneity in DNA methylation studies</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:84<a href="https://doi.org/10.1186/s13059-016-0935-y">10.1186/s13059-016-0935-y</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0935-y">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0935-y</a>.</p>
</div>
<div>
<p>217. Teschendorff AE, Zheng SC: <strong>Cell-type deconvolution in epigenome-wide association studies: a review and recommendations</strong>. <em>Epigenomics</em> 2017, <strong>9</strong>:757–768<a href="https://doi.org/10.2217/epi-2016-0153">10.2217/epi-2016-0153</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi-2016-0153">https://www.futuremedicine.com/doi/10.2217/epi-2016-0153</a>.</p>
</div>
<div>
<p>218. Lin X, Tan JYL, Teh AL, Lim IY, Liew SJ, MacIsaac JL, Chong YS, Gluckman PD, Kobor MS, Cheong CY, Karnani N: <strong>Cell type-specific DNA methylation in neonatal cord tissue and cord blood: a 850K-reference panel and comparison of cell types</strong>. <em>Epigenetics</em> 2018, <strong>13</strong>:941–958<a href="https://doi.org/10.1080/15592294.2018.1522929">10.1080/15592294.2018.1522929</a>Available: <a href="https://doi.org/10.1080/15592294.2018.1522929">https://doi.org/10.1080/15592294.2018.1522929</a>.</p>
</div>
<div>
<p>219. Wang S: <strong>Method to detect differentially methylated loci with case-control designs using Illumina arrays</strong>. <em>Genetic Epidemiology</em> 2011, <strong>35</strong>:686–694<a href="https://doi.org/10.1002/gepi.20619">10.1002/gepi.20619</a>Available: <a href="http://doi.wiley.com/10.1002/gepi.20619">http://doi.wiley.com/10.1002/gepi.20619</a>.</p>
</div>
<div>
<p>220. Tsai PC, Bell JT: <strong>Power and sample size estimation for epigenome-wide association scans to detect differential DNA methylation</strong>. <em>International Journal of Epidemiology</em> 2015, <strong>44</strong>:1429–1441<a href="https://doi.org/10.1093/ije/dyv041">10.1093/ije/dyv041</a>.</p>
</div>
<div>
<p>221. Graw S, Henn R, Thompson JA, Koestler DC: <strong>PwrEWAS: A user-friendly tool for comprehensive power estimation for epigenome wide association studies (EWAS)</strong>. <em>BMC Bioinformatics</em> 2019, <strong>20</strong>:1–11<a href="https://doi.org/10.1186/s12859-019-2804-7">10.1186/s12859-019-2804-7</a>.</p>
</div>
<div>
<p>222. Cohen J: <em>Statisticsl Power Analysis for the Behavioural Sciences</em>. 2nd ed. New York: Lawrence Erlbaum Associates; 1988.</p>
</div>
<div>
<p>223. Joubert BR, Felix JF, Yousefi P, Bakulski KM, Just AC, Breton C, Reese SE, Markunas CA, Richmond RC, Xu C-J, Küpers LK, Oh SS, Hoyo C, Gruzieva O, Söderhäll C, Salas LA, Baïz N, Zhang H, Lepeule J, Ruiz C, Ligthart S, Wang T, Taylor JA, Duijts L, Sharp GC, Jankipersadsing SA, Nilsen RM, Vaez A, Fallin MD, Hu D, et al.: <strong>DNA Methylation in Newborns and Maternal Smoking in Pregnancy: Genome-wide Consortium Meta-analysis</strong>. <em>The American Journal of Human Genetics</em> 2016, <strong>98</strong>:680–696<a href="https://doi.org/10.1016/j.ajhg.2016.02.019">10.1016/j.ajhg.2016.02.019</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27040690">http://www.ncbi.nlm.nih.gov/pubmed/27040690</a>.</p>
</div>
<div>
<p>224. Breton CV, Marsit CJ, Faustman E, Nadeau K, Goodrich JM, Dolinoy DC, Herbstman J, Holland N, LaSalle JM, Schmidt R, Yousefi P, Perera F, Joubert BR, Wiemels J, Taylor M, Yang IV, Chen R, Hew KM, Freeland DMH, Miller R, Murphy SK: <strong>Small-Magnitude Effect Sizes in Epigenetic End Points are Important in Children’s Environmental Health Studies: The Children’s Environmental Health and Disease Prevention Research Center’s Epigenetics Working Group</strong>. <em>Environmental Health Perspectives</em> 2017, <strong>125</strong>:511–526<a href="https://doi.org/10.1289/EHP595">10.1289/EHP595</a>Available: <a href="https://ehp.niehs.nih.gov/doi/10.1289/EHP595">https://ehp.niehs.nih.gov/doi/10.1289/EHP595</a>.</p>
</div>
<div>
<p>225. Acton RJ, Bourne E, Bell J, Lillycrop K, Wang J, Dennison E, Harvey N, Spector TD, Cooper C, Bell CG: <strong>The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</strong>. <em>bioRxiv</em> 2019,:1–48<a href="https://doi.org/10.1101/870352">10.1101/870352</a>Available: <a href="https://www.biorxiv.org/content/10.1101/870352v1">https://www.biorxiv.org/content/10.1101/870352v1</a>.</p>
</div>
<div>
<p>226. Vorländer MK, Baudin F, Moir RD, Wetzel R, Hagen WJH, Willis IM, Müller CW: <strong>Structural basis for RNA polymerase III transcription repression by Maf1</strong>. <em>Nature Structural &amp; Molecular Biology</em> 2020, <a href="https://doi.org/10.1038/s41594-020-0383-y">10.1038/s41594-020-0383-y</a>Available: <a href="http://www.nature.com/articles/s41594-020-0383-y">http://www.nature.com/articles/s41594-020-0383-y</a>.</p>
</div>
<div>
<p>227. Partridge L, Deelen J, Slagboom PE: <strong>Facing up to the global challenges of ageing</strong>. <em>Nature</em> 2018, <strong>561</strong>:45–56<a href="https://doi.org/10.1038/s41586-018-0457-8">10.1038/s41586-018-0457-8</a>Available: <a href="http://www.nature.com/articles/s41586-018-0457-8">http://www.nature.com/articles/s41586-018-0457-8</a>.</p>
</div>
<div>
<p>228. Campisi J, Kapahi P, Lithgow GJ, Melov S, Newman JC, Verdin E: <strong>From discoveries in ageing research to therapeutics for healthy ageing</strong>. <em>Nature</em> 2019, <strong>571</strong>:183–192<a href="https://doi.org/10.1038/s41586-019-1365-2">10.1038/s41586-019-1365-2</a>Available: <a href="http://www.nature.com/articles/s41586-019-1365-2">http://www.nature.com/articles/s41586-019-1365-2</a>.</p>
</div>
<div>
<p>229. Chuong EB, Elde NC, Feschotte C: <strong>Regulatory activities of transposable elements: from conflicts to benefits</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:71–86<a href="https://doi.org/10.1038/nrg.2016.139">10.1038/nrg.2016.139</a>Available: <a href="http://www.nature.com/articles/nrg.2016.139">http://www.nature.com/articles/nrg.2016.139</a>.</p>
</div>
<div>
<p>230. Sandoval J, Heyn H, Moran S, Serra-Musach J, Pujana MA, Bibikova M, Esteller M: <strong>Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome</strong>. <em>Epigenetics</em> 2011, <strong>6</strong>:692–702<a href="https://doi.org/10.4161/epi.6.6.16196">10.4161/epi.6.6.16196</a>Available: <a href="http://www.tandfonline.com/doi/abs/10.4161/epi.6.6.16196">http://www.tandfonline.com/doi/abs/10.4161/epi.6.6.16196</a>.</p>
</div>
<div>
<p>231. Horvath S, Oshima J, Martin GM, Lu AT, Quach A, Cohen H, Felton S, Matsuyama M, Lowe D, Kabacik S, Wilson JG, Reiner AP, Maierhofer A, Flunkert J, Aviv A, Hou L, Baccarelli AA, Li Y, Stewart JD, Whitsel EA, Ferrucci L, Matsuyama S, Raj K: <strong>Epigenetic clock for skin and blood cells applied to Hutchinson Gilford Progeria Syndrome and ex vivo studies</strong>. <em>Aging</em> 2018, <strong>10</strong>:1758–1775<a href="https://doi.org/10.18632/aging.101508">10.18632/aging.101508</a>Available: <a href="http://www.aging-us.com/article/101508/text">http://www.aging-us.com/article/101508/text</a>.</p>
</div>
<div>
<p>232. Field AE, Robertson NA, Wang T, Havas A, Ideker T, Adams PD: <strong>DNA Methylation Clocks in Aging: Categories, Causes, and Consequences</strong>. <em>Molecular Cell</em> 2018, <strong>71</strong>:882–895<a href="https://doi.org/10.1016/j.molcel.2018.08.008">10.1016/j.molcel.2018.08.008</a>Available: <a href="https://doi.org/10.1016/j.molcel.2018.08.008">https://doi.org/10.1016/j.molcel.2018.08.008</a>.</p>
</div>
<div>
<p>233. Rideout EJ, Marshall L, Grewal SS: <strong>Drosophila RNA polymerase III repressor Maf1 controls body size and developmental timing by modulating tRNAiMet synthesis and systemic insulin signaling</strong>. <em>Proceedings of the National Academy of Sciences</em> 2012, <strong>109</strong>:1139–1144<a href="https://doi.org/10.1073/pnas.1113311109">10.1073/pnas.1113311109</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1113311109">http://www.pnas.org/cgi/doi/10.1073/pnas.1113311109</a>.</p>
</div>
<div>
<p>234. Kolitz SE, Lorsch JR: <strong>Eukaryotic initiator tRNA: Finely tuned and ready for action</strong>. <em>FEBS Letters</em> 2010, <strong>584</strong>:396–404<a href="https://doi.org/10.1016/j.febslet.2009.11.047">10.1016/j.febslet.2009.11.047</a>Available: <a href="http://doi.wiley.com/10.1016/j.febslet.2009.11.047">http://doi.wiley.com/10.1016/j.febslet.2009.11.047</a>.</p>
</div>
<div>
<p>235. Pavon-Eternod M, Gomes S, Rosner MR, Pan T: <strong>Overexpression of initiator methionine tRNA leads to global reprogramming of tRNA expression and increased proliferation in human epithelial cells</strong>. <em>RNA</em> 2013, <strong>19</strong>:461–466<a href="https://doi.org/10.1261/rna.037507.112">10.1261/rna.037507.112</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23431330">http://www.ncbi.nlm.nih.gov/pubmed/23431330</a>.</p>
</div>
<div>
<p>236. Eigen M, Lindemann B, Tietze M, Winkler-Oswatitsch R, Dress A, Haeseler A von: <strong>How old is the genetic code? Statistical geometry of tRNA provides an answer</strong>. <em>Science</em> 1989, <strong>244</strong>:673–679<a href="https://doi.org/10.1126/science.2497522">10.1126/science.2497522</a>Available: <a href="http://www.sciencemag.org/cgi/doi/10.1126/science.2497522">http://www.sciencemag.org/cgi/doi/10.1126/science.2497522</a>.</p>
</div>
<div>
<p>237. Tavernarakis N: <strong>Ageing and the regulation of protein synthesis: a balancing act?</strong> <em>Trends in Cell Biology</em> 2008, <strong>18</strong>:228–235<a href="https://doi.org/10.1016/j.tcb.2008.02.004">10.1016/j.tcb.2008.02.004</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0962892408000792">https://linkinghub.elsevier.com/retrieve/pii/S0962892408000792</a>.</p>
</div>
<div>
<p>238. Pliatsika V, Loher P, Magee R, Telonis AG, Londin E, Shigematsu M, Kirino Y, Rigoutsos I: <strong>MINTbase v2.0: a comprehensive database for tRNA-derived fragments that includes nuclear and mitochondrial fragments from all The Cancer Genome Atlas projects</strong>. <em>Nucleic Acids Research</em> 2018, <strong>46</strong>:D152–D159<a href="https://doi.org/10.1093/nar/gkx1075">10.1093/nar/gkx1075</a>Available: <a href="http://academic.oup.com/nar/article/46/D1/D152/4653530">http://academic.oup.com/nar/article/46/D1/D152/4653530</a>.</p>
</div>
<div>
<p>239. Schimmel P: <strong>The emerging complexity of the tRNA world: mammalian tRNAs beyond protein synthesis.</strong> <em>Nature reviews. Molecular cell biology</em> 2018, <strong>19</strong>:45–58<a href="https://doi.org/10.1038/nrm.2017.77">10.1038/nrm.2017.77</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28875994">http://www.ncbi.nlm.nih.gov/pubmed/28875994</a>.</p>
</div>
<div>
<p>240. Lee YS, Shibata Y, Malhotra A, Dutta A: <strong>A novel class of small RNAs: tRNA-derived RNA fragments (tRFs).</strong> <em>Genes &amp; development</em> 2009, <strong>23</strong>:2639–49<a href="https://doi.org/10.1101/gad.1837609">10.1101/gad.1837609</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19933153">http://www.ncbi.nlm.nih.gov/pubmed/19933153</a>.</p>
</div>
<div>
<p>241. Torres AG, Reina O, Stephan-Otto Attolini C, Ribas de Pouplana L: <strong>Differential expression of human tRNA genes drives the abundance of tRNA-derived fragments</strong>. <em>Proceedings of the National Academy of Sciences</em> 2019, <strong>116</strong>:201821120<a href="https://doi.org/10.1073/pnas.1821120116">10.1073/pnas.1821120116</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1821120116">http://www.pnas.org/lookup/doi/10.1073/pnas.1821120116</a>.</p>
</div>
<div>
<p>242. Li S, Xu Z, Sheng J: <strong>tRNA-Derived Small RNA: A Novel Regulatory Small Non-Coding RNA</strong>. <em>Genes</em> 2018, <strong>9</strong>:246<a href="https://doi.org/10.3390/genes9050246">10.3390/genes9050246</a>Available: <a href="http://www.mdpi.com/2073-4425/9/5/246">http://www.mdpi.com/2073-4425/9/5/246</a>.</p>
</div>
<div>
<p>243. Xu W-L, Yang Y, Wang Y-D, Qu L-H, Zheng L-L: <strong>Computational Approaches to tRNA-Derived Small RNAs</strong>. <em>Non-Coding RNA</em> 2017, <strong>3</strong>:2<a href="https://doi.org/10.3390/ncrna3010002">10.3390/ncrna3010002</a>Available: <a href="http://www.mdpi.com/2311-553X/3/1/2">http://www.mdpi.com/2311-553X/3/1/2</a>.</p>
</div>
<div>
<p>244. Parisien M, Wang X, Pan T: <strong>Diversity of human tRNA genes from the 1000-genomes project</strong>. <em>RNA Biology</em> 2013, <strong>10</strong>:1853–1867<a href="https://doi.org/10.4161/rna.27361">10.4161/rna.27361</a>Available: <a href="http://www.tandfonline.com/doi/full/10.4161/rna.27361">http://www.tandfonline.com/doi/full/10.4161/rna.27361</a>.</p>
</div>
<div>
<p>245. Chan PP, Lowe TM: <strong>GtRNAdb: a database of transfer RNA genes detected in genomic sequence.</strong> <em>Nucleic acids research</em> 2009, <strong>37</strong>:D93–7<a href="https://doi.org/10.1093/nar/gkn787">10.1093/nar/gkn787</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18984615">http://www.ncbi.nlm.nih.gov/pubmed/18984615</a>.</p>
</div>
<div>
<p>246. Lodish H, Berk A, Zipursky SL, Matsudaira P, Baltimore D, Darnell J: <em>Molecular Cell Biology, 4th edition</em>. 4th ed. New York: W. H. Freeman; 2000 Available: <a href="https://www.ncbi.nlm.nih.gov/books/NBK21475/">https://www.ncbi.nlm.nih.gov/books/NBK21475/</a>.</p>
</div>
<div>
<p>247. Schramm L: <strong>Recruitment of RNA polymerase III to its target promoters</strong>. <em>Genes &amp; Development</em> 2002, <strong>16</strong>:2593–2620<a href="https://doi.org/10.1101/gad.1018902">10.1101/gad.1018902</a>Available: <a href="http://www.genesdev.org/cgi/doi/10.1101/gad.1018902">http://www.genesdev.org/cgi/doi/10.1101/gad.1018902</a>.</p>
</div>
<div>
<p>248. Canella D, Praz V, Reina JH, Cousin P, Hernandez N: <strong>Defining the RNA polymerase III transcriptome: Genome-wide localization of the RNA polymerase III transcription machinery in human cells.</strong> <em>Genome research</em> 2010, <strong>20</strong>:710–21<a href="https://doi.org/10.1101/gr.101337.109">10.1101/gr.101337.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20413673">http://www.ncbi.nlm.nih.gov/pubmed/20413673</a>.</p>
</div>
<div>
<p>249. Besser D, Götz F, Schulze-Forster K, Wagner H, Kröger H, Simon D: <strong>DNA methylation inhibits transcription by RNA polymerase III of a tRNA gene, but not of a 5S rRNA gene.</strong> <em>FEBS letters</em> 1990, <strong>269</strong>:358–62<a href="https://doi.org/10.1016/0014-5793(90)81193-R">10.1016/0014-5793(90)81193-R</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2401361">http://www.ncbi.nlm.nih.gov/pubmed/2401361</a>.</p>
</div>
<div>
<p>250. Varshney D, Vavrova-Anderson J, Oler AJ, Cowling VH, Cairns BR, White RJ: <strong>SINE transcription by RNA polymerase III is suppressed by histone methylation but not by DNA methylation.</strong> <em>Nature communications</em> 2015, <strong>6</strong>:6569<a href="https://doi.org/10.1038/ncomms7569">10.1038/ncomms7569</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25798578">http://www.ncbi.nlm.nih.gov/pubmed/25798578</a>.</p>
</div>
<div>
<p>251. Murawski M, Szczesniak B, Zoladek T, Hopper AK, Martin NC, Boguta M: <strong>maf1 mutation alters the subcellular localization of the Mod5 protein in yeast.</strong> <em>Acta biochimica Polonica</em> 1994, <strong>41</strong>:441–8Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7732762">http://www.ncbi.nlm.nih.gov/pubmed/7732762</a>.</p>
</div>
<div>
<p>252. Pluta K, Lefebvre O, Martin NC, Smagowicz WJ, Stanford DR, Ellis SR, Hopper AK, Sentenac A, Boguta M: <strong>Maf1p, a Negative Effector of RNA Polymerase III in Saccharomyces cerevisiae</strong>. <em>Molecular and Cellular Biology</em> 2001, <strong>21</strong>:5031–5040<a href="https://doi.org/10.1128/MCB.21.15.5031-5040.2001">10.1128/MCB.21.15.5031-5040.2001</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11438659">http://www.ncbi.nlm.nih.gov/pubmed/11438659</a>.</p>
</div>
<div>
<p>253. Mange F, Praz V, Migliavacca E, Willis IM, Schütz F, Hernandez N: <strong>Diurnal regulation of RNA polymerase III transcription is under the control of both the feeding–fasting response and the circadian clock</strong>. <em>Genome Research</em> 2017, <strong>27</strong>:973–984<a href="https://doi.org/10.1101/gr.217521.116">10.1101/gr.217521.116</a>Available: <a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.217521.116">http://genome.cshlp.org/lookup/doi/10.1101/gr.217521.116</a>.</p>
</div>
<div>
<p>254. Kennedy BK, Lamming DW: <strong>The Mechanistic Target of Rapamycin: The Grand ConducTOR of Metabolism and Aging</strong>. <em>Cell Metabolism</em> 2016, <strong>23</strong>:990–1003<a href="https://doi.org/10.1016/j.cmet.2016.05.009">10.1016/j.cmet.2016.05.009</a>Available: <a href="http://dx.doi.org/10.1016/j.cmet.2016.05.009">http://dx.doi.org/10.1016/j.cmet.2016.05.009</a>.</p>
</div>
<div>
<p>255. Schmitt BM, Rudolph KLM, Karagianni P, Fonseca NA, White RJ, Talianidis I, Odom DT, Marioni JC, Kutter C: <strong>High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA-tRNA interface.</strong> <em>Genome research</em> 2014, <strong>24</strong>:1797–807<a href="https://doi.org/10.1101/gr.176784.114">10.1101/gr.176784.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25122613">http://www.ncbi.nlm.nih.gov/pubmed/25122613</a>.</p>
</div>
<div>
<p>256. Powell JR, Moriyama EN: <strong>Evolution of codon usage bias in Drosophila.</strong> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 1997, <strong>94</strong>:7784–90<a href="https://doi.org/10.1073/pnas.0404957101">10.1073/pnas.0404957101</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/9223264">http://www.ncbi.nlm.nih.gov/pubmed/9223264</a>.</p>
</div>
<div>
<p>257. Goodenbour JM, Pan T: <strong>Diversity of tRNA genes in eukaryotes.</strong> <em>Nucleic acids research</em> 2006, <strong>34</strong>:6137–46<a href="https://doi.org/10.1093/nar/gkl725">10.1093/nar/gkl725</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17088292">http://www.ncbi.nlm.nih.gov/pubmed/17088292</a>.</p>
</div>
<div>
<p>258. Geslain R, Pan T: <strong>Functional Analysis of Human tRNA Isodecoders</strong>. <em>Journal of Molecular Biology</em> 2010, <strong>396</strong>:821–831<a href="https://doi.org/10.1016/j.jmb.2009.12.018">10.1016/j.jmb.2009.12.018</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S002228360901523X">https://linkinghub.elsevier.com/retrieve/pii/S002228360901523X</a>.</p>
</div>
<div>
<p>259. Kirchner S, Cai Z, Rauscher R, Kastelic N, Anding M, Czech A, Kleizen B, Ostedgaard LS, Braakman I, Sheppard DN, Ignatova Z: <strong>Alteration of protein function by a silent polymorphism linked to tRNA abundance</strong>. <em>PLOS Biology</em> 2017, <strong>15</strong>:e2000779<a href="https://doi.org/10.1371/journal.pbio.2000779">10.1371/journal.pbio.2000779</a>Available: <a href="http://dx.plos.org/10.1371/journal.pbio.2000779">http://dx.plos.org/10.1371/journal.pbio.2000779</a>.</p>
</div>
<div>
<p>260. Pace DA: <strong>Fixed metabolic costs for highly variable rates of protein synthesis in sea urchin embryos and larvae</strong>. <em>Journal of Experimental Biology</em> 2006, <strong>209</strong>:158–170<a href="https://doi.org/10.1242/jeb.01962">10.1242/jeb.01962</a>Available: <a href="http://jeb.biologists.org/cgi/doi/10.1242/jeb.01962">http://jeb.biologists.org/cgi/doi/10.1242/jeb.01962</a>.</p>
</div>
<div>
<p>261. Nwanaji-Enwerem JC, Weisskopf MG, Baccarelli AA: <strong>Multi-tissue DNA methylation age: Molecular relationships and perspectives for advancing biomarker utility</strong>. <em>Ageing Research Reviews</em> 2018, <strong>45</strong>:15–23<a href="https://doi.org/10.1016/j.arr.2018.04.005">10.1016/j.arr.2018.04.005</a>Available: <a href="https://doi.org/10.1016/j.arr.2018.04.005">https://doi.org/10.1016/j.arr.2018.04.005</a>.</p>
</div>
<div>
<p>262. Hansen M, Taubert S, Crawford D, Libina N, Lee S-J, Kenyon C: <strong>Lifespan extension by conditions that inhibit translation in Caenorhabditis elegans</strong>. <em>Aging Cell</em> 2007, <strong>6</strong>:95–110<a href="https://doi.org/10.1111/j.1474-9726.2006.00267.x">10.1111/j.1474-9726.2006.00267.x</a>Available: <a href="http://doi.wiley.com/10.1111/j.1474-9726.2006.00267.x">http://doi.wiley.com/10.1111/j.1474-9726.2006.00267.x</a>.</p>
</div>
<div>
<p>263. Filer D, Thompson MA, Takhaveev V, Dobson AJ, Kotronaki I, Green JWM, Heinemann M, Tullet JMA, Alic N: <strong>RNA polymerase III limits longevity downstream of TORC1</strong>. <em>Nature</em> 2017, <strong>552</strong>:263–267<a href="https://doi.org/10.1038/nature25007">10.1038/nature25007</a>Available: <a href="http://www.nature.com/articles/nature25007">http://www.nature.com/articles/nature25007</a>.</p>
</div>
<div>
<p>264. Dhahbi JM, Spindler SR, Atamna H, Yamakawa A, Boffelli D, Mote P, Martin DI: <strong>5’ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction</strong>. <em>BMC Genomics</em> 2013, <strong>14</strong>:298<a href="https://doi.org/10.1186/1471-2164-14-298">10.1186/1471-2164-14-298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23638709">http://www.ncbi.nlm.nih.gov/pubmed/23638709</a>.</p>
</div>
<div>
<p>265. Van Bortle K, Phanstiel DH, Snyder MP: <strong>Topological organization and dynamic regulation of human tRNA genes during macrophage differentiation</strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:180<a href="https://doi.org/10.1186/s13059-017-1310-3">10.1186/s13059-017-1310-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1310-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1310-3</a>.</p>
</div>
<div>
<p>266. Syddall H, Aihie Sayer A, Dennison E, Martin H, Barker D, Cooper C: <strong>Cohort Profile: The Hertfordshire Cohort Study</strong>. <em>International Journal of Epidemiology</em> 2005, <strong>34</strong>:1234–1242<a href="https://doi.org/10.1093/ije/dyi127">10.1093/ije/dyi127</a>Available: <a href="http://academic.oup.com/ije/article/34/6/1234/707357/Cohort-Profile-The-Hertfordshire-Cohort-Study">http://academic.oup.com/ije/article/34/6/1234/707357/Cohort-Profile-The-Hertfordshire-Cohort-Study</a>.</p>
</div>
<div>
<p>267. Lowe TM, Chan PP: <strong>tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes</strong>. <em>Nucleic Acids Research</em> 2016, <strong>44</strong>:W54–W57<a href="https://doi.org/10.1093/nar/gkw413">10.1093/nar/gkw413</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw413">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw413</a>.</p>
</div>
<div>
<p>268. Amemiya HM, Kundaje A, Boyle AP: <strong>The ENCODE Blacklist: Identification of Problematic Regions of the Genome</strong>. <em>Scientific Reports</em> 2019, <strong>9</strong>:9354<a href="https://doi.org/10.1038/s41598-019-45839-z">10.1038/s41598-019-45839-z</a>Available: <a href="http://www.nature.com/articles/s41598-019-45839-z">http://www.nature.com/articles/s41598-019-45839-z</a>.</p>
</div>
<div>
<p>269. Ernst J, Kheradpour P, Mikkelsen TS, Shoresh N, Ward LD, Epstein CB, Zhang X, Wang L, Issner R, Coyne M, Ku M, Durham T, Kellis M, Bernstein BE: <strong>Mapping and analysis of chromatin state dynamics in nine human cell types</strong>. <em>Nature</em> 2011, <strong>473</strong>:43–49<a href="https://doi.org/10.1038/nature09906">10.1038/nature09906</a>Available: <a href="http://www.nature.com/articles/nature09906">http://www.nature.com/articles/nature09906</a>.</p>
</div>
<div>
<p>270. Bell CG, Wilson GA, Butcher LM, Roos C, Walter L, Beck S: <strong>Human-specific CpG "beacons" identify loci associated with human-specific traits and disease.</strong> <em>Epigenetics</em> 2012, <strong>7</strong>:1188–99<a href="https://doi.org/10.4161/epi.22127">10.4161/epi.22127</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22968434">http://www.ncbi.nlm.nih.gov/pubmed/22968434</a>.</p>
</div>
<div>
<p>271. North BV, Curtis D, Sham PC: <strong>A Note on the Calculation of Empirical P Values from Monte Carlo Procedures</strong>. <em>The American Journal of Human Genetics</em> 2003, <strong>72</strong>:498–499<a href="https://doi.org/10.1086/346173">10.1086/346173</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002929707605618">https://linkinghub.elsevier.com/retrieve/pii/S0002929707605618</a>.</p>
</div>
<div>
<p>272. Ewels P, Magnusson M, Lundin S, Käller M: <strong>MultiQC: summarize analysis results for multiple tools and samples in a single report.</strong> <em>Bioinformatics (Oxford, England)</em> 2016, <strong>32</strong>:3047–8<a href="https://doi.org/10.1093/bioinformatics/btw354">10.1093/bioinformatics/btw354</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27312411">http://www.ncbi.nlm.nih.gov/pubmed/27312411</a>.</p>
</div>
<div>
<p>273. Krueger F: <strong>Trim Galore</strong>. 2015, Available: <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim%7B\_%7Dgalore/">https://www.bioinformatics.babraham.ac.uk/projects/trim{\_}galore/</a>.</p>
</div>
<div>
<p>274. Martin M: <strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong>. <em>EMBnet.journal</em> 2011, <strong>17</strong>:10<a href="https://doi.org/10.14806/ej.17.1.200">10.14806/ej.17.1.200</a>Available: <a href="http://journal.embnet.org/index.php/embnetjournal/article/view/200">http://journal.embnet.org/index.php/embnetjournal/article/view/200</a>.</p>
</div>
<div>
<p>275. Langmead B, Salzberg SL: <strong>Fast gapped-read alignment with Bowtie 2.</strong> <em>Nature methods</em> 2012, <strong>9</strong>:357–9<a href="https://doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">http://www.ncbi.nlm.nih.gov/pubmed/22388286</a>.</p>
</div>
<div>
<p>276. Müller F, Scherer M, Assenov Y, Lutsik P, Walter J, Lengauer T, Bock C: <strong>RnBeads 2.0: comprehensive analysis of DNA methylation data</strong>. <em>Genome Biology</em> 2019, <strong>20</strong>:55<a href="https://doi.org/10.1186/s13059-019-1664-9">10.1186/s13059-019-1664-9</a>Available: <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1664-9">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1664-9</a>.</p>
</div>
<div>
<p>277. Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK: <strong>limma powers differential expression analyses for RNA-sequencing and microarray studies</strong>. <em>Nucleic Acids Research</em> 2015, <strong>43</strong>:e47–e47<a href="https://doi.org/10.1093/nar/gkv007">10.1093/nar/gkv007</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25605792">http://www.ncbi.nlm.nih.gov/pubmed/25605792</a>.</p>
</div>
<div>
<p>278. Meuleman W: <strong>Epilogos</strong>. 2019, Available: <a href="https://epilogos.altius.org/%20https://github.com/Altius/epilogos">https://epilogos.altius.org/ https://github.com/Altius/epilogos</a>.</p>
</div>
<div>
<p>279. Reinius LE, Acevedo N, Joerink M, Pershagen G, Dahlén S-E, Greco D, Söderhäll C, Scheynius A, Kere J: <strong>Differential DNA Methylation in Purified Human Blood Cells: Implications for Cell Lineage and Studies on Disease Susceptibility</strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e41361<a href="https://doi.org/10.1371/journal.pone.0041361">10.1371/journal.pone.0041361</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0041361">https://dx.plos.org/10.1371/journal.pone.0041361</a>.</p>
</div>
<div>
<p>280. Sean D, Meltzer PS: <strong>GEOquery: A bridge between the Gene Expression Omnibus (GEO) and BioConductor</strong>. <em>Bioinformatics</em> 2007, <strong>23</strong>:1846–1847<a href="https://doi.org/10.1093/bioinformatics/btm254">10.1093/bioinformatics/btm254</a>.</p>
</div>
<div>
<p>281. Grossman RL, Heath AP, Ferretti V, Varmus HE, Lowy DR, Kibbe WA, Staudt LM: <strong>Toward a Shared Vision for Cancer Genomic Data</strong>. <em>New England Journal of Medicine</em> 2016, <strong>375</strong>:1109–1112<a href="https://doi.org/10.1056/NEJMp1607591">10.1056/NEJMp1607591</a>Available: <a href="http://www.nejm.org/doi/10.1056/NEJMp1607591">http://www.nejm.org/doi/10.1056/NEJMp1607591</a>.</p>
</div>
<div>
<p>282. Yang Z, Wong A, Kuh D, Paul DS, Rakyan VK, Leslie RD, Zheng SC, Widschwendter M, Beck S, Teschendorff AE: <strong>Correlation of an epigenetic mitotic clock with cancer risk</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:205<a href="https://doi.org/10.1186/s13059-016-1064-3">10.1186/s13059-016-1064-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3</a>.</p>
</div>
<div>
<p>283. Nazor KL, Altun G, Lynch C, Tran H, Harness JV, Slavin I, Garitaonandia I, Müller F-J, Wang Y-C, Boscolo FS, Fakunle E, Dumevska B, Lee S, Park HS, Olee T, D’Lima DD, Semechkin R, Parast MM, Galat V, Laslett AL, Schmidt U, Keirstead HS, Loring JF, Laurent LC: <strong>Recurrent Variations in DNA Methylation in Human Pluripotent Stem Cells and Their Differentiated Derivatives</strong>. <em>Cell Stem Cell</em> 2012, <strong>10</strong>:620–634<a href="https://doi.org/10.1016/j.stem.2012.02.013">10.1016/j.stem.2012.02.013</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10449618">http://www.ncbi.nlm.nih.gov/pubmed/10449618</a>.</p>
</div>
<div>
<p>284. Juzenas S, Venkatesh G, Hübenthal M, Hoeppner MP, Du ZG, Paulsen M, Rosenstiel P, Senger P, Hofmann-Apitius M, Keller A, Kupcinskas L, Franke A, Hemmrich-Stanisak G: <strong>A comprehensive, cell specific microRNA catalogue of human peripheral blood</strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:9290–9301<a href="https://doi.org/10.1093/nar/gkx706">10.1093/nar/gkx706</a>Available: <a href="http://academic.oup.com/nar/article/45/16/9290/4080663">http://academic.oup.com/nar/article/45/16/9290/4080663</a>.</p>
</div>
<div>
<p>285. Loher P, Telonis AG, Rigoutsos I: <strong>MINTmap: fast and exhaustive profiling of nuclear and mitochondrial tRNA fragments from short RNA-seq data.</strong> <em>Scientific reports</em> 2017, <strong>7</strong>:41184<a href="https://doi.org/10.1038/srep41184">10.1038/srep41184</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28220888">http://www.ncbi.nlm.nih.gov/pubmed/28220888</a>.</p>
</div>
<div>
<p>286. Petkovich DA, Podolskiy DI, Lobanov AV, Lee S-G, Miller RA, Gladyshev VN: <strong>Using DNA Methylation Profiling to Evaluate Biological Age and Longevity Interventions</strong>. <em>Cell Metabolism</em> 2017, <strong>25</strong>:954–960.e6<a href="https://doi.org/10.1016/j.cmet.2017.03.016">10.1016/j.cmet.2017.03.016</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1550413117301687">https://linkinghub.elsevier.com/retrieve/pii/S1550413117301687</a>.</p>
</div>
<div>
<p>287. Derrien T, Estellé J, Marco Sola S, Knowles DG, Raineri E, Guigó R, Ribeca P: <strong>Fast Computation and Applications of Genome Mappability</strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e30377<a href="https://doi.org/10.1371/journal.pone.0030377">10.1371/journal.pone.0030377</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0030377">https://dx.plos.org/10.1371/journal.pone.0030377</a>.</p>
</div>
<div>
<p>288. Gu Z, Eils R, Schlesner M: <strong>Complex heatmaps reveal patterns and correlations in multidimensional genomic data</strong>. <em>Bioinformatics</em> 2016, <strong>32</strong>:2847–2849<a href="https://doi.org/10.1093/bioinformatics/btw313">10.1093/bioinformatics/btw313</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btw313">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btw313</a>.</p>
</div>
<div>
<p>289. Schmitt BM, Rudolph KLM, Karagianni P, Fonseca NA, White RJ, Talianidis I, Odom DT, Marioni JC, Kutter C: <strong>High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA–tRNA interface</strong>. <em>Genome Research</em> 2014, <strong>24</strong>:1797–1807<a href="https://doi.org/10.1101/gr.176784.114">10.1101/gr.176784.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25122613">http://www.ncbi.nlm.nih.gov/pubmed/25122613</a>.</p>
</div>
<div>
<p>290. Dittmar KA, Goodenbour JM, Pan T: <strong>Tissue-specific differences in human transfer RNA expression.</strong> <em>PLoS genetics</em> 2006, <strong>2</strong>:e221<a href="https://doi.org/10.1371/journal.pgen.0020221">10.1371/journal.pgen.0020221</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17194224">http://www.ncbi.nlm.nih.gov/pubmed/17194224</a>.</p>
</div>
<div>
<p>291. Sagi D, Rak R, Gingold H, Adir I, Maayan G, Dahan O, Broday L, Pilpel Y, Rechavi O: <strong>Tissue- and Time-Specific Expression of Otherwise Identical tRNA Genes</strong>. <em>PLOS Genetics</em> 2016, <strong>12</strong>:e1006264<a href="https://doi.org/10.1371/journal.pgen.1006264">10.1371/journal.pgen.1006264</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27560950">http://www.ncbi.nlm.nih.gov/pubmed/27560950</a>.</p>
</div>
<div>
<p>292. Ehrlich M: <strong>DNA hypermethylation in disease: mechanisms and clinical relevance.</strong> <em>Epigenetics</em> 2019, <strong>14</strong>:1141–1163<a href="https://doi.org/10.1080/15592294.2019.1638701">10.1080/15592294.2019.1638701</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31284823">http://www.ncbi.nlm.nih.gov/pubmed/31284823</a>.</p>
</div>
<div>
<p>293. Lappalainen T, Greally JM: <strong>Associating cellular epigenetic models with human phenotypes</strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:441–451<a href="https://doi.org/10.1038/nrg.2017.32">10.1038/nrg.2017.32</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.32">http://www.nature.com/doifinder/10.1038/nrg.2017.32</a>.</p>
</div>
<div>
<p>294. Keam SP, Young PE, McCorkindale AL, Dang THY, Clancy JL, Humphreys DT, Preiss T, Hutvagner G, Martin DIK, Cropley JE, Suter CM: <strong>The human Piwi protein Hiwi2 associates with tRNA-derived piRNAs in somatic cells</strong>. <em>Nucleic Acids Research</em> 2014, <strong>42</strong>:8984–8995<a href="https://doi.org/10.1093/nar/gku620">10.1093/nar/gku620</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gku620">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gku620</a>.</p>
</div>
<div>
<p>295. Li S, Shi X, Chen M, Xu N, Sun D, Bai R, Chen H, Ding K, Sheng J, Xu Z: <strong>Angiogenin promotes colorectal cancer metastasis via tiRNA production</strong>. <em>International Journal of Cancer</em> 2019,:ijc.32245<a href="https://doi.org/10.1002/ijc.32245">10.1002/ijc.32245</a>Available: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.32245">https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.32245</a>.</p>
</div>
<div>
<p>296. Kim HK, Fuchs G, Wang S, Wei W, Zhang Y, Park H, Roy-Chaudhuri B, Li P, Xu J, Chu K, Zhang F, Chua M-S, So S, Zhang QC, Sarnow P, Kay MA: <strong>A transfer-RNA-derived small RNA regulates ribosome biogenesis</strong>. <em>Nature</em> 2017, <strong>552</strong>:57–62<a href="https://doi.org/10.1038/nature25005">10.1038/nature25005</a>Available: <a href="http://www.nature.com/articles/nature25005">http://www.nature.com/articles/nature25005</a>.</p>
</div>
<div>
<p>297. Martinez G, Choudury SG, Slotkin RK: <strong>tRNA-derived small RNAs target transposable element transcripts</strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:5142–5152<a href="https://doi.org/10.1093/nar/gkx103">10.1093/nar/gkx103</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx103">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx103</a>.</p>
</div>
<div>
<p>298. Gomez-Roman N, Grandori C, Eisenman RN, White RJ: <strong>Direct activation of RNA polymerase III transcription by c-Myc</strong>. <em>Nature</em> 2003, <strong>421</strong>:290–294<a href="https://doi.org/10.1038/nature01327">10.1038/nature01327</a>Available: <a href="http://www.nature.com/articles/nature01327">http://www.nature.com/articles/nature01327</a>.</p>
</div>
<div>
<p>299. Crighton D: <strong>p53 represses RNA polymerase III transcription by targeting TBP and inhibiting promoter occupancy by TFIIIB</strong>. <em>The EMBO Journal</em> 2003, <strong>22</strong>:2810–2820<a href="https://doi.org/10.1093/emboj/cdg265">10.1093/emboj/cdg265</a>Available: <a href="http://emboj.embopress.org/cgi/doi/10.1093/emboj/cdg265">http://emboj.embopress.org/cgi/doi/10.1093/emboj/cdg265</a>.</p>
</div>
<div>
<p>300. Sutcliffe JE, Brown TRP, Allison SJ, Scott PH, White RJ: <strong>Retinoblastoma Protein Disrupts Interactions Required for RNA Polymerase III Transcription</strong>. <em>Molecular and Cellular Biology</em> 2000, <strong>20</strong>:9192–9202<a href="https://doi.org/10.1128/MCB.20.24.9192-9202.2000">10.1128/MCB.20.24.9192-9202.2000</a>Available: <a href="http://mcb.asm.org/cgi/doi/10.1128/MCB.20.24.9192-9202.2000">http://mcb.asm.org/cgi/doi/10.1128/MCB.20.24.9192-9202.2000</a>.</p>
</div>
<div>
<p>301. Gingold H, Tehler D, Christoffersen NR, Nielsen MM, Asmar F, Kooistra SM, Christophersen NS, Christensen LL, Borre M, Sørensen KD, Andersen LD, Andersen CL, Hulleman E, Wurdinger T, Ralfkiær E, Helin K, Grønbæk K, Ørntoft T, Waszak SM, Dahan O, Pedersen JS, Lund AH, Pilpel Y: <strong>A Dual Program for Translation Regulation in Cellular Proliferation and Differentiation</strong>. <em>Cell</em> 2014, <strong>158</strong>:1281–1292<a href="https://doi.org/10.1016/j.cell.2014.08.011">10.1016/j.cell.2014.08.011</a>Available: <a href="http://www.sciencedirect.com/science/article/pii/S0092867414010423">http://www.sciencedirect.com/science/article/pii/S0092867414010423</a>.</p>
</div>
<div>
<p>302. Rudolph KLM, Schmitt BM, Villar D, White RJ, Marioni JC, Kutter C, Odom DT: <strong>Codon-Driven Translational Efficiency Is Stable across Diverse Mammalian Cell States.</strong> <em>PLoS genetics</em> 2016, <strong>12</strong>:e1006024<a href="https://doi.org/10.1371/journal.pgen.1006024">10.1371/journal.pgen.1006024</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27166679">http://www.ncbi.nlm.nih.gov/pubmed/27166679</a>.</p>
</div>
<div>
<p>303. Gingold H, Dahan O, Pilpel Y: <strong>Dynamic changes in translational efficiency are deduced from codon usage of the transcriptome.</strong> <em>Nucleic acids research</em> 2012, <strong>40</strong>:10053–63<a href="https://doi.org/10.1093/nar/gks772">10.1093/nar/gks772</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22941644">http://www.ncbi.nlm.nih.gov/pubmed/22941644</a>.</p>
</div>
<div>
<p>304. Iben JR, Maraia RJ: <strong>tRNA gene copy number variation in humans</strong>. <em>Gene</em> 2014, <strong>536</strong>:376–384<a href="https://doi.org/10.1016/j.gene.2013.11.049">10.1016/j.gene.2013.11.049</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0378111913015758">https://linkinghub.elsevier.com/retrieve/pii/S0378111913015758</a>.</p>
</div>
<div>
<p>305. Darrow EM, Chadwick BP: <strong>A novel tRNA variable number tandem repeat at human chromosome 1q23.3 is implicated as a boundary element based on conservation of a CTCF motif in mouse</strong>. <em>Nucleic Acids Research</em> 2014, <strong>42</strong>:6421–6435<a href="https://doi.org/10.1093/nar/gku280">10.1093/nar/gku280</a>Available: <a href="http://academic.oup.com/nar/article/42/10/6421/2435926/A-novel-tRNA-variable-number-tandem-repeat-at">http://academic.oup.com/nar/article/42/10/6421/2435926/A-novel-tRNA-variable-number-tandem-repeat-at</a>.</p>
</div>
<div>
<p>306. Huang S-q, Sun B, Xiong Z-p, Shu Y, Zhou H-h, Zhang W, Xiong J, Li Q: <strong>The dysregulation of tRNAs and tRNA derivatives in cancer</strong>. <em>Journal of Experimental &amp; Clinical Cancer Research</em> 2018, <strong>37</strong>:101<a href="https://doi.org/10.1186/s13046-018-0745-z">10.1186/s13046-018-0745-z</a>Available: <a href="https://jeccr.biomedcentral.com/articles/10.1186/s13046-018-0745-z">https://jeccr.biomedcentral.com/articles/10.1186/s13046-018-0745-z</a>.</p>
</div>
<div>
<p>307. Krishnan P, Ghosh S, Wang B, Heyns M, Li D, Mackey JR, Kovalchuk O, Damaraju S: <strong>Genome-wide profiling of transfer RNAs and their role as novel prognostic markers for breast cancer</strong>. <em>Scientific Reports</em> 2016, <strong>6</strong>:32843<a href="https://doi.org/10.1038/srep32843">10.1038/srep32843</a>Available: <a href="http://dx.doi.org/10.1038/srep32843">http://dx.doi.org/10.1038/srep32843</a>.</p>
</div>
<div>
<p>308. Müller CA, Nieduszynski CA: <strong>DNA replication timing influences gene expression level</strong>. <em>The Journal of Cell Biology</em> 2017, <strong>216</strong>:1907–1914<a href="https://doi.org/10.1083/jcb.201701061">10.1083/jcb.201701061</a>Available: <a href="http://www.jcb.org/lookup/doi/10.1083/jcb.201701061">http://www.jcb.org/lookup/doi/10.1083/jcb.201701061</a>.</p>
</div>
<div>
<p>309. Du Q, Bert SA, Armstrong NJ, Caldon CE, Song JZ, Nair SS, Gould CM, Luu P-L, Peters T, Khoury A, Qu W, Zotenko E, Stirzaker C, Clark SJ: <strong>Replication timing and epigenome remodelling are associated with the nature of chromosomal rearrangements in cancer</strong>. <em>Nature Communications</em> 2019, <strong>10</strong>:416<a href="https://doi.org/10.1038/s41467-019-08302-1">10.1038/s41467-019-08302-1</a>Available: <a href="http://dx.doi.org/10.1038/s41467-019-08302-1">http://dx.doi.org/10.1038/s41467-019-08302-1</a>.</p>
</div>
<div>
<p>310. Cruickshanks HA, McBryan T, Nelson DM, VanderKraats ND, Shah PP, Tuyn J van, Singh Rai T, Brock C, Donahue G, Dunican DS, Drotar ME, Meehan RR, Edwards JR, Berger SL, Adams PD: <strong>Senescent cells harbour features of the cancer epigenome</strong>. <em>Nature Cell Biology</em> 2013, <strong>15</strong>:1495–1506<a href="https://doi.org/10.1038/ncb2879">10.1038/ncb2879</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24270890">http://www.ncbi.nlm.nih.gov/pubmed/24270890</a>.</p>
</div>
<div>
<p>311. Raab JR, Chiu J, Zhu J, Katzman S, Kurukuti S, Wade PA, Haussler D, Kamakaka RT: <strong>Human tRNA genes function as chromatin insulators.</strong> <em>The EMBO journal</em> 2012, <strong>31</strong>:330–50<a href="https://doi.org/10.1038/emboj.2011.406">10.1038/emboj.2011.406</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22085927">http://www.ncbi.nlm.nih.gov/pubmed/22085927</a>.</p>
</div>
<div>
<p>312. Noma K-i, Cam HP, Maraia RJ, Grewal SIS: <strong>A role for TFIIIC transcription factor complex in genome organization.</strong> <em>Cell</em> 2006, <strong>125</strong>:859–72<a href="https://doi.org/10.1016/j.cell.2006.04.028">10.1016/j.cell.2006.04.028</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16751097">http://www.ncbi.nlm.nih.gov/pubmed/16751097</a>.</p>
</div>
<div>
<p>313. Sun L, Yu R, Dang W: <strong>Chromatin Architectural Changes during Cellular Senescence and Aging</strong>. <em>Genes</em> 2018, <strong>9</strong>:211<a href="https://doi.org/10.3390/genes9040211">10.3390/genes9040211</a>Available: <a href="http://www.mdpi.com/2073-4425/9/4/211">http://www.mdpi.com/2073-4425/9/4/211</a>.</p>
</div>
<div>
<p>314. Gregory TR: <strong>Synergy between sequence and size in large-scale genomics</strong>. <em>Nature Reviews Genetics</em> 2005, <strong>6</strong>:699–708<a href="https://doi.org/10.1038/nrg1674">10.1038/nrg1674</a>.</p>
</div>
<div>
<p>315. Deniz Ö, Frost JM, Branco MR: <strong>Regulation of transposable elements by DNA modifications</strong>. <em>Nature Reviews Genetics</em> 2019, <strong>20</strong>:417–431<a href="https://doi.org/10.1038/s41576-019-0106-6">10.1038/s41576-019-0106-6</a>.</p>
</div>
<div>
<p>316. Kazazian HH, Moran JV: <strong>Mobile DNA in health and disease</strong>. <em>New England Journal of Medicine</em> 2017, <strong>377</strong>:361–370<a href="https://doi.org/10.1056/NEJMra1510092">10.1056/NEJMra1510092</a>.</p>
</div>
<div>
<p>317. Deaton AM, Bird A: <strong>CpG islands and the regulation of transcription</strong>. <em>Genes &amp; development</em> 2011, <strong>25</strong>:1010–22<a href="https://doi.org/10.1101/gad.2037511">10.1101/gad.2037511</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21576262">http://www.ncbi.nlm.nih.gov/pubmed/21576262</a>.</p>
</div>
<div>
<p>318. Hellman A, Chess A: <strong>Gene body-specific methylation on the active X chromosome</strong>. <em>Science</em> 2007, <strong>315</strong>:1141–1143<a href="https://doi.org/10.1126/science.1136352">10.1126/science.1136352</a>.</p>
</div>
<div>
<p>319. Shukla S, Kavak E, Gregory M, Imashimizu M, Shutinoski B, Kashlev M, Oberdoerffer P, Sandberg R, Oberdoerffer S: <strong>CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing</strong>. <em>Nature</em> 2011, <strong>479</strong>:74–79<a href="https://doi.org/10.1038/nature10442">10.1038/nature10442</a>Available: <a href="http://dx.doi.org/10.1038/nature10442">http://dx.doi.org/10.1038/nature10442</a>.</p>
</div>
<div>
<p>320. Blattler A, Yao L, Witt H, Guo Y, Nicolet CM, Berman BP, Farnham PJ: <strong>Global loss of DNA methylation uncovers intronic enhancers in genes showing expression changes</strong>. <em>Genome Biology</em> 2014, <strong>15</strong>:469<a href="https://doi.org/10.1186/s13059-014-0469-0">10.1186/s13059-014-0469-0</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0</a>.</p>
</div>
<div>
<p>321. Gama-Sosa MA, Slagel VA, Trewyn RW, Oxenhandler R, Kuo KC, Gehrke CW, Ehrlich M: <strong>The 5-methylcytosine content of DNA from human tumors</strong>. <em>Nucleic Acids Research</em> 1983, <strong>11</strong>:6883–6894<a href="https://doi.org/10.1093/nar/11.19.6883">10.1093/nar/11.19.6883</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/11.19.6883">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/11.19.6883</a>.</p>
</div>
<div>
<p>322. Bollati V, Schwartz J, Wright R, Litonjua A, Tarantini L, Suh H, Sparrow D, Vokonas P, Baccarelli A: <strong>Decline in genomic DNA methylation through aging in a cohort of elderly subjects</strong>. <em>Mechanisms of Ageing and Development</em> 2009, <strong>130</strong>:234–239<a href="https://doi.org/10.1016/j.mad.2008.12.003">10.1016/j.mad.2008.12.003</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820">https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820</a>.</p>
</div>
<div>
<p>323. Dewannieux M, Esnault C, Heidmann T: <strong>LINE-mediated retrotransposition of marked Alu sequences</strong>. <em>Nature Genetics</em> 2003, <strong>35</strong>:41–48<a href="https://doi.org/10.1038/ng1223">10.1038/ng1223</a>Available: <a href="http://www.nature.com/articles/ng1223">http://www.nature.com/articles/ng1223</a>.</p>
</div>
<div>
<p>324. Deininger P: <strong>Alu elements: know the SINEs</strong>. <em>Genome Biology</em> 2011, <strong>12</strong>:236<a href="https://doi.org/10.1186/gb-2011-12-12-236">10.1186/gb-2011-12-12-236</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236">http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236</a>.</p>
</div>
<div>
<p>325. Ullu E, Tschudi C: <strong>Alu sequences are processed 7SL RNA genes</strong>. <em>Nature</em> 1984, <strong>312</strong>:171–172<a href="https://doi.org/10.1038/312171a0">10.1038/312171a0</a>Available: <a href="http://www.nature.com/articles/312171a0">http://www.nature.com/articles/312171a0</a>.</p>
</div>
<div>
<p>326. Ade C, Roy-Engel AM, Deininger PL: <strong>Alu elements: an intrinsic source of human genome instability</strong>. <em>Current Opinion in Virology</em> 2013, <strong>3</strong>:639–645<a href="https://doi.org/10.1016/j.coviro.2013.09.002">10.1016/j.coviro.2013.09.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533">https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533</a>.</p>
</div>
<div>
<p>327. Cordaux R, Batzer MA: <strong>The impact of retrotransposons on human genome evolution</strong>. <em>Nature Reviews Genetics</em> 2009, <strong>10</strong>:691–703<a href="https://doi.org/10.1038/nrg2640">10.1038/nrg2640</a>Available: <a href="http://www.nature.com/articles/nrg2640">http://www.nature.com/articles/nrg2640</a>.</p>
</div>
<div>
<p>328. Belancio VP, Hedges DJ, Deininger P: <strong>Mammalian non-LTR retrotransposons: For better or worse, in sickness and in health</strong>. <em>Genome Research</em> 2008, <strong>18</strong>:343–358<a href="https://doi.org/10.1101/gr.5558208">10.1101/gr.5558208</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.5558208">http://www.genome.org/cgi/doi/10.1101/gr.5558208</a>.</p>
</div>
<div>
<p>329. Jordà M, Díez-Villanueva A, Mallona I, Martín B, Lois S, Barrera V, Esteller M, Vavouri T, Peinado MA: <strong>The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells</strong>. <em>Genome Research</em> 2017, <strong>27</strong>:118–132<a href="https://doi.org/10.1101/gr.207522.116">10.1101/gr.207522.116</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27999094">http://www.ncbi.nlm.nih.gov/pubmed/27999094</a>.</p>
</div>
<div>
<p>330. Polak P, Domany E: <strong>Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes.</strong> <em>BMC genomics</em> 2006, <strong>7</strong>:133<a href="https://doi.org/10.1186/1471-2164-7-133">10.1186/1471-2164-7-133</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16740159">http://www.ncbi.nlm.nih.gov/pubmed/16740159</a>.</p>
</div>
<div>
<p>331. Chen L-L, Yang L: <strong>ALU ternative Regulation for Gene Expression</strong>. <em>Trends in Cell Biology</em> 2017, <strong>27</strong>:480–490<a href="https://doi.org/10.1016/j.tcb.2017.01.002">10.1016/j.tcb.2017.01.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028">https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028</a>.</p>
</div>
<div>
<p>332. Ferrari R, de Llobet Cucalon LI, Di Vona C, Le Dilly F, Vidal E, Lioutas A, Oliete JQ, Jochem L, Cutts E, Dieci G, Vannini A, Teichmann M, Luna S de la, Beato M: <strong>TFIIIC Binding to Alu Elements Controls Gene Expression via Chromatin Looping and Histone Acetylation</strong>. <em>Molecular Cell</em> 2019,:1–13<a href="https://doi.org/10.1016/j.molcel.2019.10.020">10.1016/j.molcel.2019.10.020</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981">https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981</a>.</p>
</div>
<div>
<p>333. Su M, Han D, Boyd-Kirkup J, Yu X, Han J-DJ: <strong>Evolution of Alu Elements toward Enhancers</strong>. <em>Cell Reports</em> 2014, <strong>7</strong>:376–385<a href="https://doi.org/10.1016/j.celrep.2014.03.011">10.1016/j.celrep.2014.03.011</a>Available: <a href="http://dx.doi.org/10.1016/j.celrep.2014.03.011">http://dx.doi.org/10.1016/j.celrep.2014.03.011</a>.</p>
</div>
<div>
<p>334. Ward MC, Wilson MD, Barbosa-Morais NL, Schmidt D, Stark R, Pan Q, Schwalie PC, Menon S, Lukk M, Watt S, Thybert D, Kutter C, Kirschner K, Flicek P, Blencowe BJ, Odom DT: <strong>Latent Regulatory Potential of Human-Specific Repetitive Elements</strong>. <em>Molecular Cell</em> 2013, <strong>49</strong>:262–272<a href="https://doi.org/10.1016/j.molcel.2012.11.013">10.1016/j.molcel.2012.11.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2012.11.013">http://dx.doi.org/10.1016/j.molcel.2012.11.013</a>.</p>
</div>
<div>
<p>335. Xie M, Hong C, Zhang B, Lowdon RF, Xing X, Li D, Zhou X, Lee HJ, Maire CL, Ligon KL, Gascard P, Sigaroudinia M, Tlsty TD, Kadlecek T, Weiss A, O’Geen H, Farnham PJ, Madden PAF, Mungall AJ, Tam A, Kamoh B, Cho S, Moore R, Hirst M, Marra MA, Costello JF, Wang T: <strong>DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape</strong>. <em>Nature Genetics</em> 2013, <strong>45</strong>:836–841<a href="https://doi.org/10.1038/ng.2649">10.1038/ng.2649</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23708189">http://www.ncbi.nlm.nih.gov/pubmed/23708189</a>.</p>
</div>
<div>
<p>336. Jang HS, Shah NM, Du AY, Dailey ZZ, Pehrsson EC, Godoy PM, Zhang D, Li D, Xing X, Kim S, O’Donnell D, Gordon JI, Wang T: <strong>Transposable elements drive widespread expression of oncogenes in human cancers</strong>. <em>Nature Genetics</em> 2019, <strong>51</strong>:611–617<a href="https://doi.org/10.1038/s41588-019-0373-3">10.1038/s41588-019-0373-3</a>Available: <a href="http://dx.doi.org/10.1038/s41588-019-0373-3">http://dx.doi.org/10.1038/s41588-019-0373-3</a>.</p>
</div>
<div>
<p>337. Rajendiran S, Gibbs LD, Van Treuren T, Klinkebiel DL, Vishwanatha JK: <strong>MIEN1 is tightly regulated by SINE Alu methylation in its promoter</strong>. <em>Oncotarget</em> 2016, <strong>7</strong>:65307–65319<a href="https://doi.org/10.18632/oncotarget.11675">10.18632/oncotarget.11675</a>Available: <a href="http://www.oncotarget.com/fulltext/11675">http://www.oncotarget.com/fulltext/11675</a>.</p>
</div>
<div>
<p>338. Payer LM, Steranka JP, Yang WR, Kryatova M, Medabalimi S, Ardeljan D, Liu C, Boeke JD, Avramopoulos D, Burns KH: <strong>Structural variants caused by Alu insertions are associated with risks for many human diseases</strong>. <em>Proceedings of the National Academy of Sciences</em> 2017, <strong>114</strong>:E3984–E3992<a href="https://doi.org/10.1073/pnas.1704117114">10.1073/pnas.1704117114</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114">http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114</a>.</p>
</div>
<div>
<p>339. Mozhui K, Pandey AK: <strong>Conserved effect of aging on DNA methylation and association with EZH2 polycomb protein in mice and humans</strong>. <em>Mechanisms of Ageing and Development</em> 2017, <strong>162</strong>:27–37<a href="https://doi.org/10.1016/j.mad.2017.02.006">10.1016/j.mad.2017.02.006</a>Available: <a href="http://dx.doi.org/10.1016/j.mad.2017.02.006">http://dx.doi.org/10.1016/j.mad.2017.02.006</a>.</p>
</div>
<div>
<p>340. Lu AT, Hannon E, Levine ME, Crimmins EM, Lunnon K, Mill J, Geschwind DH, Horvath S: <strong>Genetic architecture of epigenetic and neuronal ageing rates in human brain regions</strong>. <em>Nature Communications</em> 2017, <strong>8</strong>:15353<a href="https://doi.org/10.1038/ncomms15353">10.1038/ncomms15353</a>Available: <a href="http://dx.doi.org/10.1038/ncomms15353">http://dx.doi.org/10.1038/ncomms15353</a>.</p>
</div>
<div>
<p>341. Gibson J, Russ TC, Clarke T-K, Howard DM, Hillary RF, Evans KL, Walker RM, Bermingham ML, Morris SW, Campbell A, Hayward C, Murray AD, Porteous DJ, Horvath S, Lu AT, McIntosh AM, Whalley HC, Marioni RE: <strong>A meta-analysis of genome-wide association studies of epigenetic age acceleration</strong>. <em>PLOS Genetics</em> 2019, <strong>15</strong>:e1008104<a href="https://doi.org/10.1371/journal.pgen.1008104">10.1371/journal.pgen.1008104</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31738745">http://www.ncbi.nlm.nih.gov/pubmed/31738745</a>.</p>
</div>
<div>
<p>342. Bycroft C, Freeman C, Petkova D, Band G, Elliott LT, Sharp K, Motyer A, Vukcevic D, Delaneau O, O’Connell J, Cortes A, Welsh S, McVean G, Leslie S, Donnelly P, Marchini J: <strong>Genome-wide genetic data on ~500,000 UK Biobank participants</strong>. <em>bioRxiv</em> 2017,:166298<a href="https://doi.org/10.1101/166298">10.1101/166298</a>Available: <a href="https://www.biorxiv.org/content/10.1101/166298v1">https://www.biorxiv.org/content/10.1101/166298v1</a>.</p>
</div>
<div>
<p>343. McLean CY, Bristor D, Hiller M, Clarke SL, Schaar BT, Lowe CB, Wenger AM, Bejerano G: <strong>GREAT improves functional interpretation of cis-regulatory regions</strong>. <em>Nature Biotechnology</em> 2010, <strong>28</strong>:495–501<a href="https://doi.org/10.1038/nbt.1630">10.1038/nbt.1630</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20436461">http://www.ncbi.nlm.nih.gov/pubmed/20436461</a>.</p>
</div>
<div>
<p>344. Engebretsen S, Bohlin J: <strong>Statistical predictions with glmnet</strong>. <em>Clinical Epigenetics</em> 2019, <strong>11</strong>:123<a href="https://doi.org/10.1186/s13148-019-0730-1">10.1186/s13148-019-0730-1</a>Available: <a href="https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-019-0730-1">https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-019-0730-1</a>.</p>
</div>
<div>
<p>345. Zhou W, Dinh HQ, Ramjan Z, Weisenberger DJ, Nicolet CM, Shen H, Laird PW, Berman BP: <strong>DNA methylation loss in late-replicating domains is linked to mitotic cell division</strong>. <em>Nature Genetics</em> 2018, <strong>50</strong>:591–602<a href="https://doi.org/10.1038/s41588-018-0073-4">10.1038/s41588-018-0073-4</a>Available: <a href="http://dx.doi.org/10.1038/s41588-018-0073-4">http://dx.doi.org/10.1038/s41588-018-0073-4</a>.</p>
</div>
<div>
<p>346. Zenin A, Tsepilov Y, Sharapov S, Getmantsev E, Menshikov LI, Fedichev PO, Aulchenko Y: <strong>Identification of 12 genetic loci associated with human healthspan</strong>. <em>Communications Biology</em> 2019, <strong>2</strong>:41<a href="https://doi.org/10.1038/s42003-019-0290-0">10.1038/s42003-019-0290-0</a>Available: <a href="http://dx.doi.org/10.1038/s42003-019-0290-0">http://dx.doi.org/10.1038/s42003-019-0290-0</a>.</p>
</div>
<div>
<p>347. Denny JC, Bastarache L, Ritchie MD, Carroll RJ, Zink R, Mosley JD, Field JR, Pulley JM, Ramirez AH, Bowton E, Basford M a, Carrell DS, Peissig PL, Kho AN, Pacheco J a, Rasmussen LV, Crosslin DR, Crane PK, Pathak J, Bielinski SJ, Pendergrass SA, Xu H, Hindorff LA, Li R, Manolio TA, Chute CG, Chisholm RL, Larson EB, Jarvik GP, Brilliant MH, et al.: <strong>Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data</strong>. <em>Nature Biotechnology</em> 2013, <strong>31</strong>:1102–1111<a href="https://doi.org/10.1038/nbt.2749">10.1038/nbt.2749</a>Available: <a href="http://www.nature.com/articles/nbt.2749">http://www.nature.com/articles/nbt.2749</a>.</p>
</div>
<div>
<p>348. Bush WS, Oetjens MT, Crawford DC: <strong>Unravelling the human genome-phenome relationship using phenome-wide association studies</strong>. <em>Nature Reviews Genetics</em> 2016, <strong>17</strong>:129–145<a href="https://doi.org/10.1038/nrg.2015.36">10.1038/nrg.2015.36</a>Available: <a href="http://dx.doi.org/10.1038/nrg.2015.36">http://dx.doi.org/10.1038/nrg.2015.36</a>.</p>
</div>
<div>
<p>349. Wei W-Q, Li X, Feng Q, Kubo M, Kullo IJ, Peissig PL, Karlson EW, Jarvik GP, Lee MTM, Shang N, Larson EA, Edwards T, Shaffer CM, Mosley JD, Maeda S, Horikoshi M, Ritchie M, Williams MS, Larson EB, Crosslin DR, Bland ST, Pacheco JA, Rasmussen-Torvik LJ, Cronkite D, Hripcsak G, Cox NJ, Wilke RA, Stein CM, Rotter JI, Momozawa Y, et al.: <strong>LPA Variants Are Associated With Residual Cardiovascular Risk in Patients Receiving Statins</strong>. <em>Circulation</em> 2018, <strong>138</strong>:1839–1849<a href="https://doi.org/10.1161/CIRCULATIONAHA.117.031356">10.1161/CIRCULATIONAHA.117.031356</a>Available: <a href="https://www.ahajournals.org/doi/10.1161/CIRCULATIONAHA.117.031356">https://www.ahajournals.org/doi/10.1161/CIRCULATIONAHA.117.031356</a>.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="discussion-3.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/07-references.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
