<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Results | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Results | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Results | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-08-14" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="Alus.html"/>
<link rel="next" href="discussion.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#illumina-dna-methylation-arrays"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a><ul>
<li class="chapter" data-level="3.1" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.2.1</b> Outline of EWAS</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.3.2</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.3</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.4" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.4</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.4.1</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.4.2</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.4.3</b> Southampton Women’s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-abstract"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Introduction"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#tRNAmethods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Participants"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-methylation-in-cancer"><i class="fa fa-check"></i><b>4.5</b> tRNA Gene Methylation in Cancer</a></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Discussion"><i class="fa fa-check"></i><b>4.6</b> Discussion</a></li>
<li class="chapter" data-level="4.7" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-data-avail"><i class="fa fa-check"></i><b>4.7</b> Data availability</a></li>
<li class="chapter" data-level="4.8" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-code-avail"><i class="fa fa-check"></i><b>4.8</b> Code availablity</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>5.3.1</b> DNA methylome data</a></li>
<li class="chapter" data-level="5.3.2" data-path="Alus.html"><a href="Alus.html#alu-element-annotation-and-pre-filtering"><i class="fa fa-check"></i><b>5.3.2</b> Alu element annotation and pre-filtering</a></li>
<li class="chapter" data-level="5.3.3" data-path="Alus.html"><a href="Alus.html#sample-selection"><i class="fa fa-check"></i><b>5.3.3</b> Sample selection</a></li>
<li class="chapter" data-level="5.3.4" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.3.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.3.5" data-path="Alus.html"><a href="Alus.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.3.5</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.3.6" data-path="Alus.html"><a href="Alus.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.3.6</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>6</b> Results</a><ul>
<li class="chapter" data-level="6.1" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>6.1</b> Simple Clock Model</a></li>
<li class="chapter" data-level="6.2" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>6.2</b> Results</a><ul>
<li class="chapter" data-level="6.2.1" data-path="results.html"><a href="results.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>6.2.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="6.2.2" data-path="results.html"><a href="results.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>6.2.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="6.2.3" data-path="results.html"><a href="results.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>6.2.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="6.2.4" data-path="results.html"><a href="results.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>6.2.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>7</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="results" class="section level1">
<h1><span class="header-section-number">6</span> Results</h1>
<div id="simple-clock-model" class="section level2">
<h2><span class="header-section-number">6.1</span> Simple Clock Model</h2>
<p>A variety of different initial filtering criteria were applied to the Alu elements included in the models, as well as a number of data transformations.
Binarisation of gene expression data yielded increased predictive accuracy in gene expression based ageing clocks <span class="citation">[<a href="#ref-Meyer2020" role="doc-biblioref">377</a>]</span>.
The reasoning behind this approach is that the quantitation of changes in gene expression is noisy and binarising the data may reduce the noise whilst retaining the information important for age prediction.</p>
<p>Binarisation of DNA methylation data was achieved by a simple procedure: assigning windows in a given sample a DNA methylation value of 1 if their DNA methylation value was greater than the median for this window across samples and a 0 if it was less.</p>
<p>Whilst Human epigenetic age is logarithmic with time across the lifespan the effects of this are more relevant at younger ages <span class="citation">[<a href="#ref-Snir2019" role="doc-biblioref">378</a>]</span>.
We fitted Alu Clock models on the natural logarithm of age in years this did not improve the quality of predicted to actual age values, likely due to the older age distribution of our data.</p>
<p>An Alu element length filter of at least 180bp or approximately two-thirds the length of a full length Alu element (~280bp) was imposed.
This minimises the number of potentially misaligned reads which, in this context, could lead to reduced accuracy of Alu DNA methylation estimates <span class="citation">[<a href="#ref-Hellen2013" role="doc-biblioref">379</a>]</span>.</p>
<p>CpG density filters were employed to prioritise regions likely to have dynamic and functionally interesting differences in DNA methylation. Filters were used for CpG densities above the genomic background of 2%, as well as for &gt;5% the approximate threshold for CpG island shores and low methylation regions <span class="citation">[<a href="#ref-Irizarry2009" role="doc-biblioref">43</a>,<a href="#ref-Stadler2011" role="doc-biblioref">50</a>]</span>.</p>
<p>To limit the selection of predictive sites to those likely to have consistent directions of change across samples a clock was constructed with only windows which show nominally significant changes in both members of an MZ twin pair in a consistent direction.
Linear models to predict age based on DNA methylation were fitted for 687 pairs of monozygotic (MZ) twins.
Batch information and blood cell counts (eosinophils, lymphocytes, monocytes, neutrophils) were included as covariates.</p>
<p>To produce a model that specifically capture age as a function of the de-repression of Alu elements clocks were also constructed with only windows which exhibited nominally significant (p &lt; 0.05) decreases in DNAm in both MZ twins.
This filtering was based on the model used in selecting windows with a consistent direction of change.</p>
<p>AluY</p>
</div>
<div id="results-1" class="section level2">
<h2><span class="header-section-number">6.2</span> Results</h2>
<p>Age-related DNA methylation changes in Alu Elements</p>
<p>Study Design - Figure 1</p>
<div class="figure" style="text-align: center"><span id="fig:modelSummaryTabsMain"></span>
<img src="figs/model_summary_tabs_main.png" alt="Alu Clock Model Summaries , , , , , " width="100%" />
<p class="caption">
Figure 6.1: <strong>Alu Clock Model Summaries</strong> , , , , , 
</p>
</div>

<ul>
<li>This sequencing-based dataset – MeDIP-seq fragments 200-500bp – size Alu elements 300bp</li>
<li>Remove strongly genetically driven loci
<ul>
<li>Removed known polymorphic Alu in the human population <span class="citation">[<a href="#ref-Payer2017" role="doc-biblioref">368</a>]</span></li>
<li>?Adapted Gap Hunting algorithm (see Methods)</li>
</ul></li>
<li>Genome-wide significant/ Study-wide Significant windows</li>
<li>?Validated Alu methylation with target BiS-seq</li>
<li>Enrichment Analysis - GREAT <span class="citation">[<a href="#ref-McLean2010" role="doc-biblioref">380</a>]</span></li>
<li>?Association with GWAS disease loci</li>
</ul>
<div id="alu-element-specific-dna-methylation-clock" class="section level3">
<h3><span class="header-section-number">6.2.1</span> Alu-element specific DNA methylation Clock</h3>
<ul>
<li>Elastic net regression with R package glmnet <span class="citation">[<a href="#ref-Friedman2010" role="doc-biblioref">120</a>]</span>, <span class="citation">[<a href="#ref-Engebretsen2019" role="doc-biblioref">381</a>]</span></li>
</ul>
</div>
<div id="comparison-of-alu-clock-with-array-derived-dna-methylation-clocks" class="section level3">
<h3><span class="header-section-number">6.2.2</span> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</h3>
<ul>
<li>Compared with
<ul>
<li>Horvath <span class="citation">[<a href="#ref-Horvath2013" role="doc-biblioref">119</a>]</span> ,</li>
<li>Hannum et al. <span class="citation">[<a href="#ref-Hannum2013" role="doc-biblioref">118</a>]</span></li>
<li>PhenoAge <span class="citation">[<a href="#ref-Levine2018" role="doc-biblioref">138</a>]</span></li>
<li>GrimAge <span class="citation">[<a href="#ref-Lu2019" role="doc-biblioref">139</a>]</span></li>
<li>EpiTOC <span class="citation">[<a href="#ref-Yang2016" role="doc-biblioref">310</a>]</span></li>
<li>?Remethylation Window <span class="citation">[<a href="#ref-Zhou2018" role="doc-biblioref">382</a>]</span></li>
</ul></li>
</ul>
</div>
<div id="genetic-association-for-alu-clock-calculated-age-acceleration" class="section level3">
<h3><span class="header-section-number">6.2.3</span> Genetic Association for Alu-Clock calculated Age Acceleration</h3>
<ul>
<li>We performed a GWAS on Age-acceleration similar to Lu et al. 42 calculated in a separate dataset used to construct the Alu-clock</li>
<li>Genome-wide significant &amp; nominally significant results</li>
</ul>
</div>
<div id="disease-association-with-these-variants-enrichment-for-age-related-diseases" class="section level3">
<h3><span class="header-section-number">6.2.4</span> Disease Association with these Variants / Enrichment for Age-Related Diseases</h3>
<ul>
<li>Ageing and genetics shared risk <span class="citation">[<a href="#ref-Zenin2019" role="doc-biblioref">383</a>]</span></li>
<li><a href="https://www.eurekalert.org/pub_releases/2019-01/g-aac012919.php" class="uri">https://www.eurekalert.org/pub_releases/2019-01/g-aac012919.php</a></li>
<li>GWAS results from UKBioBank</li>
<li><a href="http://www.nealelab.is/uk-biobank" class="uri">http://www.nealelab.is/uk-biobank</a></li>
<li>Explore via PheWAS of these Alu-related variants
<ul>
<li>Phewascatelog.org</li>
<li>Denny et al Nat Biotech 2013 <span class="citation">[<a href="#ref-Denny2014" role="doc-biblioref">384</a>]</span></li>
<li>Bush NRG <span class="citation">[<a href="#ref-Bush2016" role="doc-biblioref">385</a>]</span></li>
<li>Wei et al. Circulation 2018 <span class="citation">[<a href="#ref-Wei2018" role="doc-biblioref">386</a>]</span></li>
<li>Other papers (Zhao et al. Plos One 2019; We et al. BioRxiv 462077; Nielsen et al Nat Gen 2018; Giri et al. Nat Gen 2019; Bastarache et al Science (2018))</li>
<li><a href="http://pheweb.sph.umich.edu/" class="uri">http://pheweb.sph.umich.edu/</a></li>
</ul></li>
<li>Enrichment of nominal significant results with Age-related disease cf. Random</li>
</ul>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion" class="section level2">
<h2><span class="header-section-number">6.3</span> Discussion</h2>
<div style="page-break-after: always;"></div>
<p>To identify strongly genetically we ?utilised the Gap Hunting algorithm of Andrews et al. <span class="citation">[<a href="#ref-Andrews2016" role="doc-biblioref">166</a>]</span> adapted
We excluded all known
Excluded polymorphic Alus <span class="citation">[<a href="#ref-Payer2017" role="doc-biblioref">368</a>]</span>
?Exclude any methylation overlapping STRs</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Irizarry2009">
<p>43. Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P, Cui H, Gabo K, Rongione M, Webster M, Ji H, Potash J, Sabunciyan S, Feinberg AP: <strong>The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.</strong> <em>Nature genetics</em> 2009, <strong>41</strong>:178–186<a href="https://doi.org/10.1038/ng.298">10.1038/ng.298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151715">http://www.ncbi.nlm.nih.gov/pubmed/19151715</a>.</p>
</div>
<div id="ref-Stadler2011">
<p>50. Stadler MB, Murr R, Burger L, Ivanek R, Lienert F, Schöler A, Nimwegen E van, Wirbelauer C, Oakeley EJ, Gaidatzis D, Tiwari VK, Schübeler D: <strong>DNA-binding factors shape the mouse methylome at distal regulatory regions.</strong> <em>Nature</em> 2011, <strong>480</strong>:490–5<a href="https://doi.org/10.1038/nature10716">10.1038/nature10716</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22170606">http://www.ncbi.nlm.nih.gov/pubmed/22170606</a>.</p>
</div>
<div id="ref-Hannum2013">
<p>118. Hannum G, Guinney J, Zhao L, Zhang L, Hughes G, Sadda S, Klotzle B, Bibikova M, Fan J-B, Gao Y, Deconde R, Chen M, Rajapakse I, Friend S, Ideker T, Zhang K: <strong>Genome-wide methylation profiles reveal quantitative views of human aging rates.</strong> <em>Molecular cell</em> 2013, <strong>49</strong>:359–367<a href="https://doi.org/10.1016/j.molcel.2012.10.016">10.1016/j.molcel.2012.10.016</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23177740">http://www.ncbi.nlm.nih.gov/pubmed/23177740</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>119. Horvath S: <strong>DNA methylation age of human tissues and cell types.</strong> <em>Genome biology</em> 2013, <strong>14</strong>:R115<a href="https://doi.org/10.1186/gb-2013-14-10-r115">10.1186/gb-2013-14-10-r115</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24138928">http://www.ncbi.nlm.nih.gov/pubmed/24138928</a>.</p>
</div>
<div id="ref-Friedman2010">
<p>120. Friedman J, Hastie T, Tibshirani R: <strong>Regularization Paths for Generalized Linear Models via Coordinate Descent</strong>. <em>Journal of Statistical Software</em> 2010, <strong>33</strong>:7–10<a href="https://doi.org/10.18637/jss.v033.i01">10.18637/jss.v033.i01</a>Available: <a href="http://www.jstatsoft.org/v33/i01/">http://www.jstatsoft.org/v33/i01/</a>.</p>
</div>
<div id="ref-Levine2018">
<p>138. Levine ME, Lu AT, Quach A, Chen BH, Assimes TL, Bandinelli S, Hou L, Baccarelli AA, Stewart JD, Li Y, Whitsel EA, Wilson JG, Reiner AP, Aviv A, Lohman K, Liu Y, Ferrucci L, Horvath S: <strong>An epigenetic biomarker of aging for lifespan and healthspan</strong>. <em>Aging</em> 2018, <strong>10</strong>:573–591<a href="https://doi.org/10.18632/aging.101414">10.18632/aging.101414</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29676998">http://www.ncbi.nlm.nih.gov/pubmed/29676998</a>.</p>
</div>
<div id="ref-Lu2019">
<p>139. Lu AT, Quach A, Wilson JG, Reiner AP, Aviv A, Raj K, Hou L, Baccarelli AA, Li Y, Stewart JD, Whitsel EA, Assimes TL, Ferrucci L, Horvath S: <strong>DNA methylation GrimAge strongly predicts lifespan and healthspan</strong>. <em>Aging</em> 2019, <strong>11</strong>:303–327<a href="https://doi.org/10.18632/aging.101684">10.18632/aging.101684</a>Available: <a href="http://www.aging-us.com/article/101684/text">http://www.aging-us.com/article/101684/text</a>.</p>
</div>
<div id="ref-Andrews2016">
<p>166. Andrews SV, Ladd-Acosta C, Feinberg AP, Hansen KD, Fallin MD: <strong>“Gap hunting” to characterize clustered probe signals in Illumina methylation array data</strong>. <em>Epigenetics &amp; Chromatin</em> 2016, <strong>9</strong>:56<a href="https://doi.org/10.1186/s13072-016-0107-z">10.1186/s13072-016-0107-z</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z</a>.</p>
</div>
<div id="ref-Yang2016">
<p>310. Yang Z, Wong A, Kuh D, Paul DS, Rakyan VK, Leslie RD, Zheng SC, Widschwendter M, Beck S, Teschendorff AE: <strong>Correlation of an epigenetic mitotic clock with cancer risk</strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:205<a href="https://doi.org/10.1186/s13059-016-1064-3">10.1186/s13059-016-1064-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3</a>.</p>
</div>
<div id="ref-Payer2017">
<p>368. Payer LM, Steranka JP, Yang WR, Kryatova M, Medabalimi S, Ardeljan D, Liu C, Boeke JD, Avramopoulos D, Burns KH: <strong>Structural variants caused by Alu insertions are associated with risks for many human diseases</strong>. <em>Proceedings of the National Academy of Sciences</em> 2017, <strong>114</strong>:E3984–E3992<a href="https://doi.org/10.1073/pnas.1704117114">10.1073/pnas.1704117114</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114">http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114</a>.</p>
</div>
<div id="ref-Meyer2020">
<p>377. Meyer DH: <strong>A transcriptome based aging clock near the theoretical limit of accuracy</strong>. 2020,.</p>
</div>
<div id="ref-Snir2019">
<p>378. Snir S, Farrell C, Pellegrini M: <strong>Human epigenetic ageing is logarithmic with time across the entire lifespan</strong>. <em>Epigenetics</em> 2019, <strong>0</strong>:1–15<a href="https://doi.org/10.1080/15592294.2019.1623634">10.1080/15592294.2019.1623634</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634">https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634</a>.</p>
</div>
<div id="ref-Hellen2013">
<p>379. Hellen EH, Brookfield JF: <strong>Alu elements in primates are preferentially lost from areas of high GC content</strong>. <em>PeerJ</em> 2013, <strong>1</strong>:e78<a href="https://doi.org/10.7717/peerj.78">10.7717/peerj.78</a>Available: <a href="https://peerj.com/articles/78">https://peerj.com/articles/78</a>.</p>
</div>
<div id="ref-McLean2010">
<p>380. McLean CY, Bristor D, Hiller M, Clarke SL, Schaar BT, Lowe CB, Wenger AM, Bejerano G: <strong>GREAT improves functional interpretation of cis-regulatory regions</strong>. <em>Nature Biotechnology</em> 2010, <strong>28</strong>:495–501<a href="https://doi.org/10.1038/nbt.1630">10.1038/nbt.1630</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20436461">http://www.ncbi.nlm.nih.gov/pubmed/20436461</a>.</p>
</div>
<div id="ref-Engebretsen2019">
<p>381. Engebretsen S, Bohlin J: <strong>Statistical predictions with glmnet</strong>. <em>Clinical Epigenetics</em> 2019, <strong>11</strong>:123<a href="https://doi.org/10.1186/s13148-019-0730-1">10.1186/s13148-019-0730-1</a>Available: <a href="https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-019-0730-1">https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-019-0730-1</a>.</p>
</div>
<div id="ref-Zhou2018">
<p>382. Zhou W, Dinh HQ, Ramjan Z, Weisenberger DJ, Nicolet CM, Shen H, Laird PW, Berman BP: <strong>DNA methylation loss in late-replicating domains is linked to mitotic cell division</strong>. <em>Nature Genetics</em> 2018, <strong>50</strong>:591–602<a href="https://doi.org/10.1038/s41588-018-0073-4">10.1038/s41588-018-0073-4</a>Available: <a href="http://dx.doi.org/10.1038/s41588-018-0073-4">http://dx.doi.org/10.1038/s41588-018-0073-4</a>.</p>
</div>
<div id="ref-Zenin2019">
<p>383. Zenin A, Tsepilov Y, Sharapov S, Getmantsev E, Menshikov LI, Fedichev PO, Aulchenko Y: <strong>Identification of 12 genetic loci associated with human healthspan</strong>. <em>Communications Biology</em> 2019, <strong>2</strong>:41<a href="https://doi.org/10.1038/s42003-019-0290-0">10.1038/s42003-019-0290-0</a>Available: <a href="http://dx.doi.org/10.1038/s42003-019-0290-0">http://dx.doi.org/10.1038/s42003-019-0290-0</a>.</p>
</div>
<div id="ref-Denny2014">
<p>384. Denny JC, Bastarache L, Ritchie MD, Carroll RJ, Zink R, Mosley JD, Field JR, Pulley JM, Ramirez AH, Bowton E, Basford M a, Carrell DS, Peissig PL, Kho AN, Pacheco J a, Rasmussen LV, Crosslin DR, Crane PK, Pathak J, Bielinski SJ, Pendergrass SA, Xu H, Hindorff LA, Li R, Manolio TA, Chute CG, Chisholm RL, Larson EB, Jarvik GP, Brilliant MH, et al.: <strong>Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data</strong>. <em>Nature Biotechnology</em> 2013, <strong>31</strong>:1102–1111<a href="https://doi.org/10.1038/nbt.2749">10.1038/nbt.2749</a>Available: <a href="http://www.nature.com/articles/nbt.2749">http://www.nature.com/articles/nbt.2749</a>.</p>
</div>
<div id="ref-Bush2016">
<p>385. Bush WS, Oetjens MT, Crawford DC: <strong>Unravelling the human genome-phenome relationship using phenome-wide association studies</strong>. <em>Nature Reviews Genetics</em> 2016, <strong>17</strong>:129–145<a href="https://doi.org/10.1038/nrg.2015.36">10.1038/nrg.2015.36</a>Available: <a href="http://dx.doi.org/10.1038/nrg.2015.36">http://dx.doi.org/10.1038/nrg.2015.36</a>.</p>
</div>
<div id="ref-Wei2018">
<p>386. Wei W-Q, Li X, Feng Q, Kubo M, Kullo IJ, Peissig PL, Karlson EW, Jarvik GP, Lee MTM, Shang N, Larson EA, Edwards T, Shaffer CM, Mosley JD, Maeda S, Horikoshi M, Ritchie M, Williams MS, Larson EB, Crosslin DR, Bland ST, Pacheco JA, Rasmussen-Torvik LJ, Cronkite D, Hripcsak G, Cox NJ, Wilke RA, Stein CM, Rotter JI, Momozawa Y, et al.: <strong>LPA Variants Are Associated With Residual Cardiovascular Risk in Patients Receiving Statins</strong>. <em>Circulation</em> 2018, <strong>138</strong>:1839–1849<a href="https://doi.org/10.1161/CIRCULATIONAHA.117.031356">10.1161/CIRCULATIONAHA.117.031356</a>Available: <a href="https://www.ahajournals.org/doi/10.1161/CIRCULATIONAHA.117.031356">https://www.ahajournals.org/doi/10.1161/CIRCULATIONAHA.117.031356</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="Alus.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/05-Alu.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
