<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Results 3 - | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Results 3 - | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Results 3 - | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="tRNAs.html"/>
<link rel="next" href="discussion-3.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#illumina-methylation-arrays"><i class="fa fa-check"></i><b>2.1</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="3.1" data-path="arrays.html"><a href="arrays.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="arrays.html"><a href="arrays.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#background"><i class="fa fa-check"></i><b>3.2.1</b> Background</a></li>
<li class="chapter" data-level="3.2.2" data-path="arrays.html"><a href="arrays.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>3.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="arrays.html"><a href="arrays.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#summary-of-results"><i class="fa fa-check"></i><b>3.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#quality-control-qc"><i class="fa fa-check"></i><b>3.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#functional-normalisation-1"><i class="fa fa-check"></i><b>3.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.4.4" data-path="arrays.html"><a href="arrays.html#ewass"><i class="fa fa-check"></i><b>3.4.4</b> EWASs</a></li>
<li class="chapter" data-level="3.4.5" data-path="arrays.html"><a href="arrays.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>3.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>3.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="3.5.1" data-path="arrays.html"><a href="arrays.html#section"><i class="fa fa-check"></i><b>3.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.6</b> Discussion</a><ul>
<li class="chapter" data-level="3.6.1" data-path="arrays.html"><a href="arrays.html#future-work"><i class="fa fa-check"></i><b>3.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#Methods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#participants-1"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="4.3.9" data-path="tRNAs.html"><a href="tRNAs.html#supplementary-figures"><i class="fa fa-check"></i><b>4.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.3.10" data-path="tRNAs.html"><a href="tRNAs.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>4.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#discussion-1"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#data-availability"><i class="fa fa-check"></i><b>4.6</b> Data availability</a></li>
<li class="chapter" data-level="4.7" data-path="tRNAs.html"><a href="tRNAs.html#code-availablity"><i class="fa fa-check"></i><b>4.7</b> Code availablity</a></li>
<li class="chapter" data-level="4.8" data-path="tRNAs.html"><a href="tRNAs.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.9" data-path="tRNAs.html"><a href="tRNAs.html#author-contributions"><i class="fa fa-check"></i><b>4.9</b> Author Contributions</a></li>
<li class="chapter" data-level="4.10" data-path="tRNAs.html"><a href="tRNAs.html#references"><i class="fa fa-check"></i><b>4.10</b> References</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results-3-.html"><a href="results-3-.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="results-3-.html"><a href="results-3-.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="results-3-.html"><a href="results-3-.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="results-3-.html"><a href="results-3-.html#methods-2"><i class="fa fa-check"></i><b>5.3</b> Methods</a></li>
<li class="chapter" data-level="5.4" data-path="results-3-.html"><a href="results-3-.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>5.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.4.2" data-path="results-3-.html"><a href="results-3-.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="5.4.3" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>5.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="5.4.4" data-path="results-3-.html"><a href="results-3-.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>5.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="results-3-.html"><a href="results-3-.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="results-3-.html"><a href="results-3-.html#methods-3"><i class="fa fa-check"></i><b>5.6</b> Methods</a><ul>
<li class="chapter" data-level="5.6.1" data-path="results-3-.html"><a href="results-3-.html#participants-2"><i class="fa fa-check"></i><b>5.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="results-3-.html"><a href="results-3-.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>5.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="5.7.1" data-path="results-3-.html"><a href="results-3-.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>5.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="5.7.2" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>5.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.7.3" data-path="results-3-.html"><a href="results-3-.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.7.4" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.7.5" data-path="results-3-.html"><a href="results-3-.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion-3.html"><a href="discussion-3.html"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="results-3--" class="section level1">
<h1><span class="header-section-number">5</span> Results 3 -</h1>
<div id="abstract-3" class="section level2">
<h2><span class="header-section-number">5.1</span> Abstract</h2>
</div>
<div id="AluIntro" class="section level2">
<h2><span class="header-section-number">5.2</span> Introduction</h2>
<p>Age is a significant risk factor across the breadth of chronic diseases, including cardiovascular, obesity-related, ophthalmic, musculoskeletal and neurological disorders <span class="citation">(Partridge, Deelen, and Slagboom <a href="#ref-Partridge2018" role="doc-biblioref">2018</a>)</span>.
The molecular changes that occur with age include alterations to the epigenome <span class="citation">(Booth and Brunet <a href="#ref-Booth2016" role="doc-biblioref">2016</a>)</span> – the chemical modifications and packaging of the genome that indicate or facilitate cell-specific activity <span class="citation">(Bird <a href="#ref-Bird2007" role="doc-biblioref">2007</a>)</span>.
This includes age-related changes to DNA methylation, the most common modification of DNA <span class="citation">(López-Otín et al. <a href="#ref-Lopez-Otin2013" role="doc-biblioref">2013</a>)</span>.
This epigenetic mechanism is crucial in repressing the large proportion of the human genome (~45% <span class="citation">(Gregory <a href="#ref-Gregory2005" role="doc-biblioref">2005</a>)</span>) that is comprised of repetitive or transposable elements <span class="citation">(Deniz, Frost, and Branco <a href="#ref-Deniz2019" role="doc-biblioref">2019</a>)</span>, <span class="citation">(Kazazian and Moran <a href="#ref-Kazazian2017" role="doc-biblioref">2017</a>)</span>.
It also classically leads to robust repression of gene expression through promoter DNA methylation <span class="citation">(Deaton and Bird <a href="#ref-Deaton2011" role="doc-biblioref">2011</a>)</span>, and has more nuanced effects on expression within the gene body <span class="citation">(Hellman and Chess <a href="#ref-Hellman2007" role="doc-biblioref">2007</a>)</span>, as well as influencing splicing <span class="citation">(Shukla et al. <a href="#ref-Shukla2011" role="doc-biblioref">2011</a>)</span> and transcription factor binding-sites (TFBSs) <span class="citation">(Blattler et al. <a href="#ref-Blattler2014" role="doc-biblioref">2014</a>)</span>, <span class="citation">(Yin et al. <a href="#ref-Yin2017" role="doc-biblioref">2017</a>)</span>.</p>
<p>A significant decrease in DNA methylation with age was first observed with pioneering global measures <span class="citation">(Wilson and Jones <a href="#ref-Wilson1983" role="doc-biblioref">1983</a>)</span>, in similar fashion to the reduced methylation seen in cancer tissue <span class="citation">(Gama-Sosa et al. <a href="#ref-Gama-Sosa1983" role="doc-biblioref">1983</a>)</span>.
This DNA methylation loss was seen to be strongly contributed to by the demethylation of repetitive elements <span class="citation">(Bollati et al. <a href="#ref-Bollati2010" role="doc-biblioref">2009</a>)</span>.
Random drift in the epigenetic state with age was also observed <span class="citation">(Bollati et al. <a href="#ref-Bollati2010" role="doc-biblioref">2009</a>)</span>, and more recent studies have identified targeted hypermethylation of specific functional regions, such as the promoters of polycomb target genes <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span>, bivalent chromatin domains <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>)</span> and the genomic loci of specific tRNA genes <span class="citation">(Acton et al. <a href="#ref-Acton2020" role="doc-biblioref">2019</a>)</span></p>
<p>Alu elements are the most successful transposon in terms of numbers within the human genome, totalling ~1.19 million copies and comprising ~10.7% of the genomic sequence <span class="citation">(Dewannieux, Esnault, and Heidmann <a href="#ref-Dewannieux2003" role="doc-biblioref">2003</a>)</span>, <span class="citation">(Deininger <a href="#ref-Deininger2011" role="doc-biblioref">2011</a>)</span>.
This ~300 bp primate-specific retrotransposon is a member of the short interspersed nuclear elements (SINEs) and is derived from the 7SL RNA gene <span class="citation">(Ullu and Tschudi <a href="#ref-Ullu1984" role="doc-biblioref">1984</a>)</span>.
Alu elements arose ~65 million years ago (MYA), with a peak in amplification around ~40 MYA <span class="citation">(Ade, Roy-Engel, and Deininger <a href="#ref-Ade2013" role="doc-biblioref">2013</a>)</span>.
Non-LTR retrotransposons, which include Alus as well as L1 and SVA elements, are active in human.
However, Alu elements themselves are non-autonomous and need to commandeer a nearby L1 element’s retrotransposition molecular machinery <span class="citation">(Cordaux and Batzer <a href="#ref-Cordaux2009" role="doc-biblioref">2009</a>)</span>.
Alu insertion mutagenesis as well as Alu element mediated recombination account for 0.4% of all human genetic disorders <span class="citation">(Kazazian and Moran <a href="#ref-Kazazian2017" role="doc-biblioref">2017</a>)</span>, <span class="citation">(Belancio, Hedges, and Deininger <a href="#ref-Belancio2008" role="doc-biblioref">2008</a>)</span>.</p>
<p>DNA methylation represses Alu elements and due to their sequences being rich in CpGs they in fact account for ~25% of all the CpGs in human genome (~7.5 million) <span class="citation">(Deininger <a href="#ref-Deininger2011" role="doc-biblioref">2011</a>)</span>.
Alu repeats are predominately methylated across all normal tissues, and even in cancer cells they are the more resistant to hypomethylation than other repetitive elements, implying a strong pressure to maintain their epigenetic state <span class="citation">(Jordà et al. <a href="#ref-Jorda2017" role="doc-biblioref">2017</a>)</span>.
These elements are an abundant genetic resource of regulatory sequences with
numerous TFBSs residing within its canonical sequence <span class="citation">(Polak and Domany <a href="#ref-Polak2006" role="doc-biblioref">2006</a>)</span>.
They can influence gene expression <span class="citation">(Chen and Yang <a href="#ref-Chen2017b" role="doc-biblioref">2017</a>)</span>, <span class="citation">(Ferrari et al. <a href="#ref-RobertoFerrari2019" role="doc-biblioref">2019</a>)</span> and there is evidence that some may possess enhancer activity <span class="citation">(Su et al. <a href="#ref-Su2014" role="doc-biblioref">2014</a>)</span>.
The deterioration of epigenomic repression can unmask latent regulatory function within repetitive elements <span class="citation">(Ward et al. <a href="#ref-Ward2013" role="doc-biblioref">2013</a>)</span>, <span class="citation">(Chuong, Elde, and Feschotte <a href="#ref-Chuong2017" role="doc-biblioref">2017</a>)</span>, <span class="citation">(Xie et al. <a href="#ref-Xie2013" role="doc-biblioref">2013</a>)</span>.
This has been clearly displayed as a pathogenic mechanism in cancer, with epigenetic reactivation of cryptic cis-regulatory elements (CREs) within transposons acting as oncogenic drivers <span class="citation">(Jang et al. <a href="#ref-Jang2019" role="doc-biblioref">2019</a>)</span> , including Alu elements <span class="citation">(Rajendiran et al. <a href="#ref-Rajendiran2016" role="doc-biblioref">2016</a>)</span>.
There is the potential for these changes to also influence common non-malignant age-related diseases <span class="citation">(Chuong, Elde, and Feschotte <a href="#ref-Chuong2017" role="doc-biblioref">2017</a>)</span>, <span class="citation">(Payer et al. <a href="#ref-Payer2017" role="doc-biblioref">2017</a>)</span>.</p>
<p>The identification of specific age-related DNA methylation changes has enabled the construction of ‘DNA methylation clocks‘ <span class="citation">(Horvath and Raj <a href="#ref-Horvath2018" role="doc-biblioref">2018</a>)</span> that can predict an individual’s chronological age, but also capture aspects of biological ageing, such as all-cause mortality <span class="citation">(Marioni et al. <a href="#ref-Marioni2015" role="doc-biblioref">2015</a>)</span>.
Hypermethylated and hypomethylated cytosines approximately equally drive these DNA methylation clocks, although these age-related increases or decreases in methylation are driven by distinct biological mechanisms <span class="citation">(Mozhui and Pandey <a href="#ref-Mozhui2017" role="doc-biblioref">2017</a>)</span>, <span class="citation">(Wu and Zhang <a href="#ref-Wu2017" role="doc-biblioref">2017</a>)</span>.
Interestingly, hypomethylating CpGs have been identified to be the most indicative of biological ageing-related all-cause mortality effects in blood <span class="citation">(Zhang et al. <a href="#ref-Zhang2017" role="doc-biblioref">2017</a>)</span>.
However, due to poor coverage of the repetitive space because of technical issues <span class="citation">(Bell et al. <a href="#ref-Bell2019" role="doc-biblioref">2019</a>)</span>, much of the genomic hypomethylation is not well captured by array analysis.</p>
<p>In this study we set about to identify and explore age-related DNA methylation changes in Alu elements in a large (&gt;3,000) immunoprecipitation-derived sequencing-based dataset (MeDIP-seq) from peripheral blood DNA. We constructed an Alu-specific DNA methylation ageing clock and compared how this contrasted with other clocks’ measures. We also investigated genetic influences on age- acceleration measures from this Alu-clock via GWAS, as has been with shown previously with an association in the TERT gene with the Horvath clock <span class="citation">(Lu et al. <a href="#ref-Lu2017" role="doc-biblioref">2017</a>)</span>, <span class="citation">(Gibson et al. <a href="#ref-Gibson2019" role="doc-biblioref">2019</a>)</span>.
These results may reveal mechanisms involved in controlling the epigenetic state of this large portion of the genome.
Finally, we interrogated these genetic associations for any potential connection with common age-related diseases through PheWAS in the UKBiobank dataset <span class="citation">(Bycroft et al. <a href="#ref-Bycroft2017" role="doc-biblioref">2017</a>)</span>.</p>
</div>
<div id="methods-2" class="section level2">
<h2><span class="header-section-number">5.3</span> Methods</h2>
</div>
<div id="results-1" class="section level2">
<h2><span class="header-section-number">5.4</span> Results</h2>
<p>Age-related DNA methylation changes in Alu Elements</p>
<p>Study Design - Figure 1</p>
<p>(ref:studyDesignCap)</p>
<ul>
<li>This sequencing-based dataset – MeDIP-seq fragments 200-500bp – size Alu elements 300bp</li>
<li>Remove strongly genetically driven loci
<ul>
<li>Removed known polymorphic Alu in the human population <span class="citation">(Payer et al. <a href="#ref-Payer2017" role="doc-biblioref">2017</a>)</span></li>
<li>?Adapted Gap Hunting algorithm (see Methods)</li>
</ul></li>
<li>Genome-wide significant/ Study-wide Significant windows</li>
<li>?Validated Alu methylation with target BiS-seq</li>
<li>Enrichment Analysis - GREAT <span class="citation">(McLean et al. <a href="#ref-McLean2010" role="doc-biblioref">2010</a>)</span></li>
<li>?Association with GWAS disease loci</li>
</ul>
<div id="alu-element-specific-dna-methylation-clock" class="section level3">
<h3><span class="header-section-number">5.4.1</span> Alu-element specific DNA methylation Clock</h3>
<ul>
<li>Elastic net regression with R package glmnet <span class="citation">(Friedman, Hastie, and Tibshirani <a href="#ref-Friedman2010" role="doc-biblioref">2010</a>)</span>, <span class="citation">(Engebretsen and Bohlin <a href="#ref-Engebretsen2019" role="doc-biblioref">2019</a>)</span></li>
</ul>
</div>
<div id="comparison-of-alu-clock-with-array-derived-dna-methylation-clocks" class="section level3">
<h3><span class="header-section-number">5.4.2</span> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</h3>
<ul>
<li>Compared with
<ul>
<li>Horvath <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span> ,</li>
<li>Hannum et al. <span class="citation">(Hannum et al. <a href="#ref-Hannum2013" role="doc-biblioref">2013</a>)</span></li>
<li>PhenoAge <span class="citation">(Levine et al. <a href="#ref-Levine2018" role="doc-biblioref">2018</a>)</span></li>
<li>GrimAge <span class="citation">(Lu et al. <a href="#ref-Lu2019" role="doc-biblioref">2019</a>)</span></li>
<li>EpiTOC <span class="citation">(Yang et al. <a href="#ref-Yang2016" role="doc-biblioref">2016</a>)</span></li>
<li>?Remethylation Window <span class="citation">(Zhou et al. <a href="#ref-Zhou2018" role="doc-biblioref">2018</a>)</span></li>
</ul></li>
</ul>
</div>
<div id="genetic-association-for-alu-clock-calculated-age-acceleration" class="section level3">
<h3><span class="header-section-number">5.4.3</span> Genetic Association for Alu-Clock calculated Age Acceleration</h3>
<ul>
<li>We performed a GWAS on Age-acceleration similar to Lu et al. 42 calculated in a separate dataset used to construct the Alu-clock</li>
<li>Genome-wide significant &amp; nominally significant results</li>
</ul>
</div>
<div id="disease-association-with-these-variants-enrichment-for-age-related-diseases" class="section level3">
<h3><span class="header-section-number">5.4.4</span> Disease Association with these Variants / Enrichment for Age-Related Diseases</h3>
<ul>
<li>Ageing and genetics shared risk <span class="citation">(Zenin et al. <a href="#ref-Zenin2019" role="doc-biblioref">2019</a>)</span></li>
<li><a href="https://www.eurekalert.org/pub_releases/2019-01/g-aac012919.php" class="uri">https://www.eurekalert.org/pub_releases/2019-01/g-aac012919.php</a></li>
<li>GWAS results from UKBioBank</li>
<li><a href="http://www.nealelab.is/uk-biobank" class="uri">http://www.nealelab.is/uk-biobank</a></li>
<li>Explore via PheWAS of these Alu-related variants
<ul>
<li>Phewascatelog.org</li>
<li>Denny et al Nat Biotech 2013 <span class="citation">(Denny et al. <a href="#ref-Denny2014" role="doc-biblioref">2013</a>)</span></li>
<li>Bush NRG <span class="citation">(Bush, Oetjens, and Crawford <a href="#ref-Bush2016" role="doc-biblioref">2016</a>)</span></li>
<li>Wei et al. Circulation 2018 <span class="citation">(Wei et al. <a href="#ref-Wei2018" role="doc-biblioref">2018</a>)</span></li>
<li>Other papers (Zhao et al. Plos One 2019; We et al. BioRxiv 462077; Nielsen et al Nat Gen 2018; Giri et al. Nat Gen 2019; Bastarache et al Science (2018))</li>
<li><a href="http://pheweb.sph.umich.edu/" class="uri">http://pheweb.sph.umich.edu/</a></li>
</ul></li>
<li>Enrichment of nominal significant results with Age-related disease cf. Random</li>
</ul>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion-2" class="section level2">
<h2><span class="header-section-number">5.5</span> Discussion</h2>
<div style="page-break-after: always;"></div>
</div>
<div id="methods-3" class="section level2">
<h2><span class="header-section-number">5.6</span> Methods</h2>
<div id="participants-2" class="section level3">
<h3><span class="header-section-number">5.6.1</span> Participants</h3>
<p>Human participants were drawn from the TwinsUK Bioresource.
These deeply phenotyped individuals DNA methylomes</p>
<p>MeDIP-seq data as described in <span class="citation">(Bell et al. <a href="#ref-Bell2017a" role="doc-biblioref">2018</a>)</span> – Briefly All data is available at</p>
</div>
</div>
<div id="age-related-dna-methylation-changes" class="section level2">
<h2><span class="header-section-number">5.7</span> Age-related DNA methylation changes</h2>
<p>To identify strongly genetically we ?utilised the Gap Hunting algorithm of Andrews et al. <span class="citation">(Andrews et al. <a href="#ref-Andrews2016" role="doc-biblioref">2016</a>)</span> adapted
We excluded all known
Excluded polymorphic Alus <span class="citation">(Payer et al. <a href="#ref-Payer2017" role="doc-biblioref">2017</a>)</span>
?Exclude any methylation overlapping STRs</p>
<div id="validation-with-target-bis-seq" class="section level3">
<h3><span class="header-section-number">5.7.1</span> ?Validation with target BiS-seq</h3>
</div>
<div id="alu-element-specific-dna-methylation-clock-1" class="section level3">
<h3><span class="header-section-number">5.7.2</span> Alu element specific DNA methylation Clock</h3>
</div>
<div id="comparison-with-array-derived-clocks" class="section level3">
<h3><span class="header-section-number">5.7.3</span> Comparison with Array-derived Clocks</h3>
</div>
<div id="genetic-association-for-alu-clock" class="section level3">
<h3><span class="header-section-number">5.7.4</span> Genetic Association for Alu-Clock</h3>
</div>
<div id="association-of-these-variants-with-age-related-disease" class="section level3">
<h3><span class="header-section-number">5.7.5</span> Association of these variants with Age-related Disease</h3>
<div id="data-availability-1" class="section level4">
<h4><span class="header-section-number">5.7.5.1</span> Data availability</h4>
<p>The MeDIP-seq data supporting the results of this article are available in the EMBL-EBI European Genome-phenome Archive (EGA) under Data set Accession number EGAD00010000983 (<a href="https://www.ebi.ac.uk/ega/datasets/" class="uri">https://www.ebi.ac.uk/ega/datasets/</a>
EGAD00010000983).</p>
</div>
<div id="code-availablity-1" class="section level4">
<h4><span class="header-section-number">5.7.5.2</span> Code availablity</h4>
<p>Available at <a href="https://github.com/richardjacton">GitHub</a></p>
<div style="page-break-after: always;"></div>
</div>
<div id="acknowledgements-2" class="section level4">
<h4><span class="header-section-number">5.7.5.3</span> Acknowledgements</h4>
</div>
<div id="author-contributions-1" class="section level4">
<h4><span class="header-section-number">5.7.5.4</span> Author Contributions</h4>
<div style="page-break-after: always;"></div>

</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Acton2020">
<p>Acton, Richard J, Emma Bourne, Jordana Bell, Karen Lillycrop, Jun Wang, Elaine Dennison, Nicholas Harvey, Tim D Spector, Cyrus Cooper, and Christopher G Bell. 2019. “The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation.” <em>bioRxiv</em>, 1–48. <a href="https://doi.org/10.1101/870352">https://doi.org/10.1101/870352</a>.</p>
</div>
<div id="ref-Ade2013">
<p>Ade, Catherine, Astrid M. Roy-Engel, and Prescott L. Deininger. 2013. “Alu elements: an intrinsic source of human genome instability.” <em>Current Opinion in Virology</em> 3 (6): 639–45. <a href="https://doi.org/10.1016/j.coviro.2013.09.002">https://doi.org/10.1016/j.coviro.2013.09.002</a>.</p>
</div>
<div id="ref-Andrews2016">
<p>Andrews, Shan V, Christine Ladd-Acosta, Andrew P Feinberg, Kasper D Hansen, and M Daniele Fallin. 2016. “‘Gap hunting’ to characterize clustered probe signals in Illumina methylation array data.” <em>Epigenetics &amp; Chromatin</em> 9 (1): 56. <a href="https://doi.org/10.1186/s13072-016-0107-z">https://doi.org/10.1186/s13072-016-0107-z</a>.</p>
</div>
<div id="ref-Belancio2008">
<p>Belancio, Victoria P., Dale J. Hedges, and Prescott Deininger. 2008. “Mammalian non-LTR retrotransposons: For better or worse, in sickness and in health.” <em>Genome Research</em> 18 (3): 343–58. <a href="https://doi.org/10.1101/gr.5558208">https://doi.org/10.1101/gr.5558208</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>Bell, Christopher G., Fei Gao, Wei Yuan, Leonie Roos, Richard J. Acton, Yudong Xia, Jordana Bell, et al. 2018. “Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci.” <em>Nature Communications</em> 9 (1): 8. <a href="https://doi.org/10.1038/s41467-017-01586-1">https://doi.org/10.1038/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Bell2019">
<p>Bell, Christopher G, Robert Lowe, Peter D Adams, Andrea A Baccarelli, Stephan Beck, Jordana T Bell, Brock C. Christensen, et al. 2019. “DNA methylation aging clocks: challenges and recommendations.” <em>Genome Biology</em> 20 (1): 249. <a href="https://doi.org/10.1186/s13059-019-1824-y">https://doi.org/10.1186/s13059-019-1824-y</a>.</p>
</div>
<div id="ref-Bird2007">
<p>Bird, Adrian. 2007. “Perceptions of epigenetics.” <em>Nature</em> 447 (7143): 396–8. <a href="https://doi.org/10.1038/nature05913">https://doi.org/10.1038/nature05913</a>.</p>
</div>
<div id="ref-Blattler2014">
<p>Blattler, Adam, Lijing Yao, Heather Witt, Yu Guo, Charles M Nicolet, Benjamin P Berman, and Peggy J Farnham. 2014. “Global loss of DNA methylation uncovers intronic enhancers in genes showing expression changes.” <em>Genome Biology</em> 15 (9): 469. <a href="https://doi.org/10.1186/s13059-014-0469-0">https://doi.org/10.1186/s13059-014-0469-0</a>.</p>
</div>
<div id="ref-Bollati2010">
<p>Bollati, Valentina, Joel Schwartz, Robert Wright, Augusto Litonjua, Letizia Tarantini, Helen Suh, David Sparrow, Pantel Vokonas, and Andrea Baccarelli. 2009. “Decline in genomic DNA methylation through aging in a cohort of elderly subjects.” <em>Mechanisms of Ageing and Development</em> 130 (4): 234–39. <a href="https://doi.org/10.1016/j.mad.2008.12.003">https://doi.org/10.1016/j.mad.2008.12.003</a>.</p>
</div>
<div id="ref-Booth2016">
<p>Booth, Lauren N., and Anne Brunet. 2016. “The Aging Epigenome.” <em>Molecular Cell</em> 62 (5): 728–44. <a href="https://doi.org/10.1016/j.molcel.2016.05.013">https://doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div id="ref-Bush2016">
<p>Bush, William S., Matthew T. Oetjens, and Dana C. Crawford. 2016. “Unravelling the human genome-phenome relationship using phenome-wide association studies.” <em>Nature Reviews Genetics</em> 17 (3): 129–45. <a href="https://doi.org/10.1038/nrg.2015.36">https://doi.org/10.1038/nrg.2015.36</a>.</p>
</div>
<div id="ref-Bycroft2017">
<p>Bycroft, Clare, Colin Freeman, Desislava Petkova, Gavin Band, Lloyd T. Elliott, Kevin Sharp, Allan Motyer, et al. 2017. “Genome-wide genetic data on ~500,000 UK Biobank participants.” <em>bioRxiv</em>, 166298. <a href="https://doi.org/10.1101/166298">https://doi.org/10.1101/166298</a>.</p>
</div>
<div id="ref-Chen2017b">
<p>Chen, Ling-Ling, and Li Yang. 2017. “ALU ternative Regulation for Gene Expression.” <em>Trends in Cell Biology</em> 27 (7): 480–90. <a href="https://doi.org/10.1016/j.tcb.2017.01.002">https://doi.org/10.1016/j.tcb.2017.01.002</a>.</p>
</div>
<div id="ref-Chuong2017">
<p>Chuong, Edward B., Nels C. Elde, and Cédric Feschotte. 2017. “Regulatory activities of transposable elements: from conflicts to benefits.” <em>Nature Reviews Genetics</em> 18 (2): 71–86. <a href="https://doi.org/10.1038/nrg.2016.139">https://doi.org/10.1038/nrg.2016.139</a>.</p>
</div>
<div id="ref-Cordaux2009">
<p>Cordaux, Richard, and Mark A. Batzer. 2009. “The impact of retrotransposons on human genome evolution.” <em>Nature Reviews Genetics</em> 10 (10): 691–703. <a href="https://doi.org/10.1038/nrg2640">https://doi.org/10.1038/nrg2640</a>.</p>
</div>
<div id="ref-Deaton2011">
<p>Deaton, Aimée M., and Adrian Bird. 2011. “CpG islands and the regulation of transcription.” <em>Genes &amp; Development</em> 25 (10): 1010–22. <a href="https://doi.org/10.1101/gad.2037511">https://doi.org/10.1101/gad.2037511</a>.</p>
</div>
<div id="ref-Deininger2011">
<p>Deininger, Prescott. 2011. “Alu elements: know the SINEs.” <em>Genome Biology</em> 12 (12): 236. <a href="https://doi.org/10.1186/gb-2011-12-12-236">https://doi.org/10.1186/gb-2011-12-12-236</a>.</p>
</div>
<div id="ref-Deniz2019">
<p>Deniz, Özgen, Jennifer M. Frost, and Miguel R. Branco. 2019. “Regulation of transposable elements by DNA modifications.” <em>Nature Reviews Genetics</em> 20 (7): 417–31. <a href="https://doi.org/10.1038/s41576-019-0106-6">https://doi.org/10.1038/s41576-019-0106-6</a>.</p>
</div>
<div id="ref-Denny2014">
<p>Denny, Joshua C, Lisa Bastarache, Marylyn D Ritchie, Robert J Carroll, Raquel Zink, Jonathan D Mosley, Julie R Field, et al. 2013. “Systematic comparison of phenome-wide association study of electronic medical record data and genome-wide association study data.” <em>Nature Biotechnology</em> 31 (12): 1102–11. <a href="https://doi.org/10.1038/nbt.2749">https://doi.org/10.1038/nbt.2749</a>.</p>
</div>
<div id="ref-Dewannieux2003">
<p>Dewannieux, Marie, Cécile Esnault, and Thierry Heidmann. 2003. “LINE-mediated retrotransposition of marked Alu sequences.” <em>Nature Genetics</em> 35 (1): 41–48. <a href="https://doi.org/10.1038/ng1223">https://doi.org/10.1038/ng1223</a>.</p>
</div>
<div id="ref-Engebretsen2019">
<p>Engebretsen, Solveig, and Jon Bohlin. 2019. “Statistical predictions with glmnet.” <em>Clinical Epigenetics</em> 11 (1): 123. <a href="https://doi.org/10.1186/s13148-019-0730-1">https://doi.org/10.1186/s13148-019-0730-1</a>.</p>
</div>
<div id="ref-RobertoFerrari2019">
<p>Ferrari, Roberto, Lara Isabel de Llobet Cucalon, Chiara Di Vona, François Le Dilly, Enrique Vidal, Antonios Lioutas, Javier Quilez Oliete, et al. 2019. “TFIIIC Binding to Alu Elements Controls Gene Expression via Chromatin Looping and Histone Acetylation.” <em>Molecular Cell</em>, November, 1–13. <a href="https://doi.org/10.1016/j.molcel.2019.10.020">https://doi.org/10.1016/j.molcel.2019.10.020</a>.</p>
</div>
<div id="ref-Friedman2010">
<p>Friedman, Jerome, Trevor Hastie, and Robert Tibshirani. 2010. “Regularization Paths for Generalized Linear Models via Coordinate Descent.” <em>Journal of Statistical Software</em> 33 (1): 7–10. <a href="https://doi.org/10.18637/jss.v033.i01">https://doi.org/10.18637/jss.v033.i01</a>.</p>
</div>
<div id="ref-Gama-Sosa1983">
<p>Gama-Sosa, Miguel A., Valerie A. Slagel, Ronald W. Trewyn, Ronald Oxenhandler, Kenneth C. Kuo, Charles W. Gehrke, and Melanie Ehrlich. 1983. “The 5-methylcytosine content of DNA from human tumors.” <em>Nucleic Acids Research</em> 11 (19): 6883–94. <a href="https://doi.org/10.1093/nar/11.19.6883">https://doi.org/10.1093/nar/11.19.6883</a>.</p>
</div>
<div id="ref-Gibson2019">
<p>Gibson, Jude, Tom C Russ, Toni-Kim Clarke, David M Howard, Robert F Hillary, Kathryn L Evans, Rosie M Walker, et al. 2019. “A meta-analysis of genome-wide association studies of epigenetic age acceleration.” Edited by John M. Greally. <em>PLOS Genetics</em> 15 (11): e1008104. <a href="https://doi.org/10.1371/journal.pgen.1008104">https://doi.org/10.1371/journal.pgen.1008104</a>.</p>
</div>
<div id="ref-Gregory2005">
<p>Gregory, T. Ryan. 2005. “Synergy between sequence and size in large-scale genomics.” <em>Nature Reviews Genetics</em> 6 (9): 699–708. <a href="https://doi.org/10.1038/nrg1674">https://doi.org/10.1038/nrg1674</a>.</p>
</div>
<div id="ref-Hannum2013">
<p>Hannum, Gregory, Justin Guinney, Ling Zhao, Li Zhang, Guy Hughes, SriniVas Sadda, Brandy Klotzle, et al. 2013. “Genome-wide methylation profiles reveal quantitative views of human aging rates.” <em>Molecular Cell</em> 49 (2): 359–67. <a href="https://doi.org/10.1016/j.molcel.2012.10.016">https://doi.org/10.1016/j.molcel.2012.10.016</a>.</p>
</div>
<div id="ref-Hellman2007">
<p>Hellman, Asaf, and Andrew Chess. 2007. “Gene body-specific methylation on the active X chromosome.” <em>Science</em> 315 (5815): 1141–3. <a href="https://doi.org/10.1126/science.1136352">https://doi.org/10.1126/science.1136352</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>Horvath, Steve. 2013. “DNA methylation age of human tissues and cell types.” <em>Genome Biology</em> 14 (10): R115. <a href="https://doi.org/10.1186/gb-2013-14-10-r115">https://doi.org/10.1186/gb-2013-14-10-r115</a>.</p>
</div>
<div id="ref-Horvath2018">
<p>Horvath, Steve, and Kenneth Raj. 2018. “DNA methylation-based biomarkers and the epigenetic clock theory of ageing.” <em>Nature Reviews Genetics</em> 19 (6): 371–84. <a href="https://doi.org/10.1038/s41576-018-0004-3">https://doi.org/10.1038/s41576-018-0004-3</a>.</p>
</div>
<div id="ref-Jang2019">
<p>Jang, Hyo Sik, Nakul M. Shah, Alan Y. Du, Zea Z. Dailey, Erica C. Pehrsson, Paula M. Godoy, David Zhang, et al. 2019. “Transposable elements drive widespread expression of oncogenes in human cancers.” <em>Nature Genetics</em> 51 (4): 611–17. <a href="https://doi.org/10.1038/s41588-019-0373-3">https://doi.org/10.1038/s41588-019-0373-3</a>.</p>
</div>
<div id="ref-Jorda2017">
<p>Jordà, Mireia, Anna Díez-Villanueva, Izaskun Mallona, Berta Martín, Sergi Lois, Víctor Barrera, Manel Esteller, Tanya Vavouri, and Miguel A. Peinado. 2017. “The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells.” <em>Genome Research</em> 27 (1): 118–32. <a href="https://doi.org/10.1101/gr.207522.116">https://doi.org/10.1101/gr.207522.116</a>.</p>
</div>
<div id="ref-Kazazian2017">
<p>Kazazian, Haig H., and John V. Moran. 2017. “Mobile DNA in health and disease.” <em>New England Journal of Medicine</em> 377 (4): 361–70. <a href="https://doi.org/10.1056/NEJMra1510092">https://doi.org/10.1056/NEJMra1510092</a>.</p>
</div>
<div id="ref-Levine2018">
<p>Levine, Morgan E, Ake T Lu, Austin Quach, Brian H Chen, Themistocles L Assimes, Stefania Bandinelli, Lifang Hou, et al. 2018. “An epigenetic biomarker of aging for lifespan and healthspan.” <em>Aging</em> 10 (4): 573–91. <a href="https://doi.org/10.18632/aging.101414">https://doi.org/10.18632/aging.101414</a>.</p>
</div>
<div id="ref-Lopez-Otin2013">
<p>López-Otín, Carlos, Maria A. Blasco, Linda Partridge, Manuel Serrano, and Guido Kroemer. 2013. “The hallmarks of aging.” <em>Cell</em> 153 (6): 1194–1217. <a href="https://doi.org/10.1016/j.cell.2013.05.039">https://doi.org/10.1016/j.cell.2013.05.039</a>.</p>
</div>
<div id="ref-Lu2017">
<p>Lu, Ake T, Eilis Hannon, Morgan E Levine, Eileen M Crimmins, Katie Lunnon, Jonathan Mill, Daniel H Geschwind, and Steve Horvath. 2017. “Genetic architecture of epigenetic and neuronal ageing rates in human brain regions.” <em>Nature Communications</em> 8 (May): 15353. <a href="https://doi.org/10.1038/ncomms15353">https://doi.org/10.1038/ncomms15353</a>.</p>
</div>
<div id="ref-Lu2019">
<p>Lu, Ake T., Austin Quach, James G. Wilson, Alex P. Reiner, Abraham Aviv, Kenneth Raj, Lifang Hou, et al. 2019. “DNA methylation GrimAge strongly predicts lifespan and healthspan.” <em>Aging</em> 11 (2): 303–27. <a href="https://doi.org/10.18632/aging.101684">https://doi.org/10.18632/aging.101684</a>.</p>
</div>
<div id="ref-Marioni2015">
<p>Marioni, Riccardo E, Sonia Shah, Allan F McRae, Brian H Chen, Elena Colicino, Sarah E Harris, Jude Gibson, et al. 2015. “DNA methylation age of blood predicts all-cause mortality in later life.” <em>Genome Biology</em> 16 (1): 25. <a href="https://doi.org/10.1186/s13059-015-0584-6">https://doi.org/10.1186/s13059-015-0584-6</a>.</p>
</div>
<div id="ref-McLean2010">
<p>McLean, Cory Y, Dave Bristor, Michael Hiller, Shoa L Clarke, Bruce T Schaar, Craig B Lowe, Aaron M Wenger, and Gill Bejerano. 2010. “GREAT improves functional interpretation of cis-regulatory regions.” <em>Nature Biotechnology</em> 28 (5): 495–501. <a href="https://doi.org/10.1038/nbt.1630">https://doi.org/10.1038/nbt.1630</a>.</p>
</div>
<div id="ref-Mozhui2017">
<p>Mozhui, Khyobeni, and Ashutosh K. Pandey. 2017. “Conserved effect of aging on DNA methylation and association with EZH2 polycomb protein in mice and humans.” <em>Mechanisms of Ageing and Development</em> 162 (March): 27–37. <a href="https://doi.org/10.1016/j.mad.2017.02.006">https://doi.org/10.1016/j.mad.2017.02.006</a>.</p>
</div>
<div id="ref-Partridge2018">
<p>Partridge, Linda, Joris Deelen, and P. Eline Slagboom. 2018. “Facing up to the global challenges of ageing.” <em>Nature</em> 561 (7721): 45–56. <a href="https://doi.org/10.1038/s41586-018-0457-8">https://doi.org/10.1038/s41586-018-0457-8</a>.</p>
</div>
<div id="ref-Payer2017">
<p>Payer, Lindsay M., Jared P. Steranka, Wan Rou Yang, Maria Kryatova, Sibyl Medabalimi, Daniel Ardeljan, Chunhong Liu, Jef D. Boeke, Dimitri Avramopoulos, and Kathleen H. Burns. 2017. “Structural variants caused by Alu insertions are associated with risks for many human diseases.” <em>Proceedings of the National Academy of Sciences</em> 114 (20): E3984–E3992. <a href="https://doi.org/10.1073/pnas.1704117114">https://doi.org/10.1073/pnas.1704117114</a>.</p>
</div>
<div id="ref-Polak2006">
<p>Polak, Paz, and Eytan Domany. 2006. “Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes.” <em>BMC Genomics</em> 7 (June): 133. <a href="https://doi.org/10.1186/1471-2164-7-133">https://doi.org/10.1186/1471-2164-7-133</a>.</p>
</div>
<div id="ref-Rajendiran2016">
<p>Rajendiran, Smrithi, Lee D. Gibbs, Timothy Van Treuren, David L. Klinkebiel, and Jamboor K. Vishwanatha. 2016. “MIEN1 is tightly regulated by SINE Alu methylation in its promoter.” <em>Oncotarget</em> 7 (40): 65307–19. <a href="https://doi.org/10.18632/oncotarget.11675">https://doi.org/10.18632/oncotarget.11675</a>.</p>
</div>
<div id="ref-Rakyan2010">
<p>Rakyan, Vardhman K., Thomas A. Down, Siarhei Maslau, Toby Andrew, Tsun Po Yang, Huriya Beyan, Pamela Whittaker, et al. 2010. “Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains.” <em>Genome Research</em> 20 (4): 434–39. <a href="https://doi.org/10.1101/gr.103101.109">https://doi.org/10.1101/gr.103101.109</a>.</p>
</div>
<div id="ref-Shukla2011">
<p>Shukla, Sanjeev, Ersen Kavak, Melissa Gregory, Masahiko Imashimizu, Bojan Shutinoski, Mikhail Kashlev, Philipp Oberdoerffer, Rickard Sandberg, and Shalini Oberdoerffer. 2011. “CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing.” <em>Nature</em> 479 (7371): 74–79. <a href="https://doi.org/10.1038/nature10442">https://doi.org/10.1038/nature10442</a>.</p>
</div>
<div id="ref-Su2014">
<p>Su, Ming, Dali Han, Jerome Boyd-Kirkup, Xiaoming Yu, and Jing-Dong J. Han. 2014. “Evolution of Alu Elements toward Enhancers.” <em>Cell Reports</em> 7 (2): 376–85. <a href="https://doi.org/10.1016/j.celrep.2014.03.011">https://doi.org/10.1016/j.celrep.2014.03.011</a>.</p>
</div>
<div id="ref-Teschendorff2010">
<p>Teschendorff, Andrew E., Usha Menon, Aleksandra Gentry-Maharaj, Susan J. Ramus, Daniel J. Weisenberger, Hui Shen, Mihaela Campan, et al. 2010. “Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.” <em>Genome Research</em> 20 (4): 440–6. <a href="https://doi.org/10.1101/gr.103606.109">https://doi.org/10.1101/gr.103606.109</a>.</p>
</div>
<div id="ref-Ullu1984">
<p>Ullu, Elisabetta, and Christian Tschudi. 1984. “Alu sequences are processed 7SL RNA genes.” <em>Nature</em> 312 (5990): 171–72. <a href="https://doi.org/10.1038/312171a0">https://doi.org/10.1038/312171a0</a>.</p>
</div>
<div id="ref-Ward2013">
<p>Ward, Michelle C., Michael D. Wilson, Nuno L. Barbosa-Morais, Dominic Schmidt, Rory Stark, Qun Pan, Petra C. Schwalie, et al. 2013. “Latent Regulatory Potential of Human-Specific Repetitive Elements.” <em>Molecular Cell</em> 49 (2): 262–72. <a href="https://doi.org/10.1016/j.molcel.2012.11.013">https://doi.org/10.1016/j.molcel.2012.11.013</a>.</p>
</div>
<div id="ref-Wei2018">
<p>Wei, Wei-Qi, Xiaohui Li, Qiping Feng, Michiaki Kubo, Iftikhar J. Kullo, Peggy L. Peissig, Elizabeth W. Karlson, et al. 2018. “LPA Variants Are Associated With Residual Cardiovascular Risk in Patients Receiving Statins.” <em>Circulation</em> 138 (17): 1839–49. <a href="https://doi.org/10.1161/CIRCULATIONAHA.117.031356">https://doi.org/10.1161/CIRCULATIONAHA.117.031356</a>.</p>
</div>
<div id="ref-Wilson1983">
<p>Wilson, V L, and P A Jones. 1983. “DNA methylation decreases in aging but not in immortal cells.” <em>Science (New York, N.Y.)</em> 220 (4601): 1055–7. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.</p>
</div>
<div id="ref-Wu2017">
<p>Wu, Xiaoji, and Yi Zhang. 2017. “TET-mediated active DNA demethylation: mechanism, function and beyond.” <em>Nature Reviews Genetics</em> 18 (9): 517–34. <a href="https://doi.org/10.1038/nrg.2017.33">https://doi.org/10.1038/nrg.2017.33</a>.</p>
</div>
<div id="ref-Xie2013">
<p>Xie, Mingchao, Chibo Hong, Bo Zhang, Rebecca F Lowdon, Xiaoyun Xing, Daofeng Li, Xin Zhou, et al. 2013. “DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape.” <em>Nature Genetics</em> 45 (7): 836–41. <a href="https://doi.org/10.1038/ng.2649">https://doi.org/10.1038/ng.2649</a>.</p>
</div>
<div id="ref-Yang2016">
<p>Yang, Zhen, Andrew Wong, Diana Kuh, Dirk S. Paul, Vardhman K. Rakyan, R. David Leslie, Shijie C. Zheng, Martin Widschwendter, Stephan Beck, and Andrew E. Teschendorff. 2016. “Correlation of an epigenetic mitotic clock with cancer risk.” <em>Genome Biology</em> 17 (1): 205. <a href="https://doi.org/10.1186/s13059-016-1064-3">https://doi.org/10.1186/s13059-016-1064-3</a>.</p>
</div>
<div id="ref-Yin2017">
<p>Yin, Yimeng, Ekaterina Morgunova, Arttu Jolma, Eevi Kaasinen, Biswajyoti Sahu, Syed Khund-Sayeed, Pratyush K. Das, et al. 2017. “Impact of cytosine methylation on DNA binding specificities of human transcription factors.” <em>Science (New York, N.Y.)</em> 356 (6337): eaaj2239. <a href="https://doi.org/10.1126/science.aaj2239">https://doi.org/10.1126/science.aaj2239</a>.</p>
</div>
<div id="ref-Zenin2019">
<p>Zenin, Aleksandr, Yakov Tsepilov, Sodbo Sharapov, Evgeny Getmantsev, L. I. Menshikov, Peter O. Fedichev, and Yurii Aulchenko. 2019. “Identification of 12 genetic loci associated with human healthspan.” <em>Communications Biology</em> 2 (1): 41. <a href="https://doi.org/10.1038/s42003-019-0290-0">https://doi.org/10.1038/s42003-019-0290-0</a>.</p>
</div>
<div id="ref-Zhang2017">
<p>Zhang, Yan, Jan Hapala, Hermann Brenner, and Wolfgang Wagner. 2017. “Individual CpG sites that are associated with age and life expectancy become hypomethylated upon aging.” <em>Clinical Epigenetics</em> 9 (1): 9. <a href="https://doi.org/10.1186/s13148-017-0315-9">https://doi.org/10.1186/s13148-017-0315-9</a>.</p>
</div>
<div id="ref-Zhou2018">
<p>Zhou, Wanding, Huy Q. Dinh, Zachary Ramjan, Daniel J. Weisenberger, Charles M. Nicolet, Hui Shen, Peter W. Laird, and Benjamin P. Berman. 2018. “DNA methylation loss in late-replicating domains is linked to mitotic cell division.” <em>Nature Genetics</em> 50 (4): 591–602. <a href="https://doi.org/10.1038/s41588-018-0073-4">https://doi.org/10.1038/s41588-018-0073-4</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="tRNAs.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="discussion-3.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/05-Alu.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
