<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4 Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="4 Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4 Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />


<meta name="date" content="2020-05-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="MAVIDOS.html"/>
<link rel="next" href="results-3-.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">The Epigenomics of Human Ageing</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.1</b> MeDIP-seq</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#illumina-methylation-arrays"><i class="fa fa-check"></i><b>2.2</b> Illumina Methylation arrays</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#whole-genome"><i class="fa fa-check"></i><b>2.3</b> Whole Genome</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.4</b> Targeted Bisulfite sequencing</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#discussion"><i class="fa fa-check"></i><b>2.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="MAVIDOS.html"><a href="MAVIDOS.html"><i class="fa fa-check"></i><b>3</b> Results 1 - Assessment of Umbilical Cord Tissue DNA Methylation with Respect to Bone Health, Environmental Factors and Ageing</a><ul>
<li class="chapter" data-level="3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#background"><i class="fa fa-check"></i><b>3.2.1</b> Background</a></li>
<li class="chapter" data-level="3.2.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-for-maternal-vitamin-d-status-and-offspring-bone-outcomes"><i class="fa fa-check"></i><b>3.2.2</b> EWAS for Maternal Vitamin D Status and Offspring Bone Outcomes</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewas-models"><i class="fa fa-check"></i><b>3.3.2</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.3</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#summary-of-results"><i class="fa fa-check"></i><b>3.4.1</b> Summary of Results</a></li>
<li class="chapter" data-level="3.4.2" data-path="MAVIDOS.html"><a href="MAVIDOS.html#quality-control-qc"><i class="fa fa-check"></i><b>3.4.2</b> Quality Control (QC)</a></li>
<li class="chapter" data-level="3.4.3" data-path="MAVIDOS.html"><a href="MAVIDOS.html#functional-normalisation-1"><i class="fa fa-check"></i><b>3.4.3</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.4.4" data-path="MAVIDOS.html"><a href="MAVIDOS.html#ewass"><i class="fa fa-check"></i><b>3.4.4</b> EWASs</a></li>
<li class="chapter" data-level="3.4.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#keep-concordance-of-epic-and-850k-ewas-results"><i class="fa fa-check"></i><b>3.4.5</b> !!! keep? Concordance of EPIC and 850k EWAS results !!!</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="MAVIDOS.html"><a href="MAVIDOS.html#swsnew-mavidos-ewas"><i class="fa fa-check"></i><b>3.5</b> !!SWS/New MAVIDOS EWAS!!</a><ul>
<li class="chapter" data-level="3.5.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#section"><i class="fa fa-check"></i><b>3.5.1</b> </a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="MAVIDOS.html"><a href="MAVIDOS.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a><ul>
<li class="chapter" data-level="3.6.1" data-path="MAVIDOS.html"><a href="MAVIDOS.html#future-work"><i class="fa fa-check"></i><b>3.6.1</b> Future Work</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNApaper.html"><a href="tRNApaper.html"><i class="fa fa-check"></i><b>4</b> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#introduction-2"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#Methods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNApaper.html"><a href="tRNApaper.html#participants"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNApaper.html"><a href="tRNApaper.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNApaper.html"><a href="tRNApaper.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNApaper.html"><a href="tRNApaper.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNApaper.html"><a href="tRNApaper.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNApaper.html"><a href="tRNApaper.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-materials"><i class="fa fa-check"></i>Supplementary materials</a><ul>
<li class="chapter" data-level="4.3.9" data-path="tRNApaper.html"><a href="tRNApaper.html#supplementary-figures"><i class="fa fa-check"></i><b>4.3.9</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.3.10" data-path="tRNApaper.html"><a href="tRNApaper.html#mintmap-reference-fragment-distribution"><i class="fa fa-check"></i><b>4.3.10</b> MINTmap reference Fragment distribution</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNApaper.html"><a href="tRNApaper.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNApaper.html"><a href="tRNApaper.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNApaper.html"><a href="tRNApaper.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.4.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNApaper.html"><a href="tRNApaper.html#discussion-2"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="tRNApaper.html"><a href="tRNApaper.html#data-availability"><i class="fa fa-check"></i><b>4.6</b> Data availability</a></li>
<li class="chapter" data-level="4.7" data-path="tRNApaper.html"><a href="tRNApaper.html#code-availablity"><i class="fa fa-check"></i><b>4.7</b> Code availablity</a></li>
<li class="chapter" data-level="4.8" data-path="tRNApaper.html"><a href="tRNApaper.html#acknowledgements-1"><i class="fa fa-check"></i><b>4.8</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.9" data-path="tRNApaper.html"><a href="tRNApaper.html#author-contributions"><i class="fa fa-check"></i><b>4.9</b> Author Contributions</a></li>
<li class="chapter" data-level="4.10" data-path="tRNApaper.html"><a href="tRNApaper.html#references"><i class="fa fa-check"></i><b>4.10</b> References</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results-3-.html"><a href="results-3-.html"><i class="fa fa-check"></i><b>5</b> Results 3 -</a><ul>
<li class="chapter" data-level="5.1" data-path="results-3-.html"><a href="results-3-.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="results-3-.html"><a href="results-3-.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="results-3-.html"><a href="results-3-.html#methods-2"><i class="fa fa-check"></i><b>5.3</b> Methods</a></li>
<li class="chapter" data-level="5.4" data-path="results-3-.html"><a href="results-3-.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock"><i class="fa fa-check"></i><b>5.4.1</b> Alu-element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.4.2" data-path="results-3-.html"><a href="results-3-.html#comparison-of-alu-clock-with-array-derived-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.2</b> Comparison of Alu-Clock with Array-derived DNA methylation Clocks</a></li>
<li class="chapter" data-level="5.4.3" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock-calculated-age-acceleration"><i class="fa fa-check"></i><b>5.4.3</b> Genetic Association for Alu-Clock calculated Age Acceleration</a></li>
<li class="chapter" data-level="5.4.4" data-path="results-3-.html"><a href="results-3-.html#disease-association-with-these-variants-enrichment-for-age-related-diseases"><i class="fa fa-check"></i><b>5.4.4</b> Disease Association with these Variants / Enrichment for Age-Related Diseases</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="results-3-.html"><a href="results-3-.html#discussion-3"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
<li class="chapter" data-level="5.6" data-path="results-3-.html"><a href="results-3-.html#methods-3"><i class="fa fa-check"></i><b>5.6</b> Methods</a><ul>
<li class="chapter" data-level="5.6.1" data-path="results-3-.html"><a href="results-3-.html#participants-1"><i class="fa fa-check"></i><b>5.6.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="results-3-.html"><a href="results-3-.html#age-related-dna-methylation-changes"><i class="fa fa-check"></i><b>5.7</b> Age-related DNA methylation changes</a><ul>
<li class="chapter" data-level="5.7.1" data-path="results-3-.html"><a href="results-3-.html#validation-with-target-bis-seq"><i class="fa fa-check"></i><b>5.7.1</b> ?Validation with target BiS-seq</a></li>
<li class="chapter" data-level="5.7.2" data-path="results-3-.html"><a href="results-3-.html#alu-element-specific-dna-methylation-clock-1"><i class="fa fa-check"></i><b>5.7.2</b> Alu element specific DNA methylation Clock</a></li>
<li class="chapter" data-level="5.7.3" data-path="results-3-.html"><a href="results-3-.html#comparison-with-array-derived-clocks"><i class="fa fa-check"></i><b>5.7.3</b> Comparison with Array-derived Clocks</a></li>
<li class="chapter" data-level="5.7.4" data-path="results-3-.html"><a href="results-3-.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.7.4</b> Genetic Association for Alu-Clock</a></li>
<li class="chapter" data-level="5.7.5" data-path="results-3-.html"><a href="results-3-.html#association-of-these-variants-with-age-related-disease"><i class="fa fa-check"></i><b>5.7.5</b> Association of these variants with Age-related Disease</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion-4.html"><a href="discussion-4.html"><i class="fa fa-check"></i><b>6</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="tRNApaper" class="section level1">
<h1><span class="header-section-number">4</span> Results 2 - The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</h1>
<p><span class="math inline">\(Richard J. Acton ^{1,2,3}\)</span>, <span class="math inline">\(Wei Yuan ^{4,5}\)</span>, <span class="math inline">\(Fei Gao ^{6}\)</span>, <span class="math inline">\(Yudong Xia ^{6}\)</span>, <span class="math inline">\(Emma Bourne ^{7}\)</span>, <span class="math inline">\(Eva Wozniak ^{7}\)</span>, <span class="math inline">\(Jordana Bell ^{4}\)</span>, <span class="math inline">\(Karen Lillycrop ^{3}\)</span>, <span class="math inline">\(Jun Wang ^{6}\)</span>, <span class="math inline">\(Elaine Dennison ^{2}\)</span>, <span class="math inline">\(Nicholas Harvey ^{2}\)</span>, <span class="math inline">\(Charles A. Mein ^{7}\)</span>, <span class="math inline">\(Tim D. Spector ^{4}\)</span>, <span class="math inline">\(Pirro G. Hysi ^{4}\)</span>, <span class="math inline">\(Cyrus Cooper ^{2}\)</span>, <span class="math inline">\(Christopher G. Bell ^{1}\)</span></p>
<ol style="list-style-type: decimal">
<li>William Harvey Research Institute, Barts &amp; The London School of Medicine and Dentistry, Charterhouse Square, Queen Mary University of London, London, U.K.</li>
<li>MRC Lifecourse Epidemiology Unit, University of Southampton, Southampton, U.K.</li>
<li>Human Development and Health, Institute of Developmental Sciences, University of Southampton, Southampton, U.K.<br />
</li>
<li>Department of Twin Research &amp; Genetic Epidemiology, St Thomas Hospital, King’s College London, London, U.K.</li>
<li>Institute of Cancer Research, Sutton, U.K.</li>
<li>BGI-Shenzhen, Shenzhen, China</li>
<li>Barts &amp; The London Genome Centre, Blizard Institute, Barts &amp; The London School of Medicine and Dentistry, Queen Mary University of London, London, U.K.</li>
</ol>
<p>2020-05-06</p>
<div class="figure">
<img src="figs/biorxiv_screenshot.png" alt="" />
<p class="caption"><a href="https://doi.org/10.1101/870352" class="uri">https://doi.org/10.1101/870352</a></p>
</div>
<p><a href="https://doi.org/10.1101/870352"><span class="math inline">\(bioR \chi v\)</span></a> <span class="citation">(Acton et al. <a href="#ref-Acton2020" role="doc-biblioref">2019</a>)</span></p>
<p><a href="https://nature-research-under-consideration.nature.com/users/37265-nature-communications/posts/57741-the-genomic-loci-of-specific-human-trna-genes-exhibit-ageing-related-dna-hypermethylation">Under Consideration at Nature Communications</a></p>
<ul>
<li><span class="citation">Vorländer et al. (<a href="#ref-Vorlander2020" role="doc-biblioref">2020</a>)</span> - MAF1 structural</li>
</ul>
<div style="page-break-after: always;"></div>
<div id="abstract-2" class="section level2">
<h2><span class="header-section-number">4.1</span> Abstract</h2>
<p>Understanding how the epigenome deteriorates with age and subsequently impacts on biological function may bring unique insights to ageing-related disease mechanisms. As a central cellular apparatus, tRNAs are fundamental to the information flow from DNA to proteins.
Whilst only being transcribed from ~46kb (&lt;0.002%) of the human genome, their transcripts are the second most abundant in the cell.
Furthermore, it is now increasingly recognised that tRNAs and their fragments also have complex regulatory functions.
In both their core translational and additional regulatory roles, tRNAs are intimately involved in the control of metabolic processes known to affect ageing.
Experimentally DNA methylation can alter tRNA expression, but little is known about the genomic DNA methylation state of tRNAs.</p>
<p>Here, we find that the human genomic tRNA loci (610 tRNA genes termed the tRNAome) are enriched for ageing-related DNA hypermethylation.
We initially identified DNA hypermethylation of 44 and 21 specific tRNA genes, at study-wide (p &lt; <span class="math inline">\(4.34\times10^{-9}\)</span>) and genome-wide (<span class="math inline">\(p &lt; 4.34\times10^{-9}\)</span>) significance, respectively, in 4,350 MeDIP-seq peripheral blood DNA methylomes (16 - 82 years).
This starkly contrasted with 0 hypomethylated at both these significance levels.
Further analysing the 21 genome-wide results, we found 3 of these tRNAs to be independent of major changes in cell-type composition (tRNA-iMet-CAT-1-4, tRNA-Ser-AGA-2-6, tRNA-Ile-AAT-4-1).
We also excluded the ageing-related changes being due to the inherent CpG density of the tRNAome by permutation analysis (1,000x, Empirical p-value &lt; <span class="math inline">\(1\times10^{-3}\)</span>).
We additionally explored 79 tRNA loci in an independent cohort using Fluidigm deep targeted bisulfite-sequencing of pooled DNA (n=190) across a range of 4 timepoints (aged ~4, ~28, ~63, ~78 years).
This revealed these ageing changes to be specific to particular isodecoder copies of these tRNA (tRNAs coding for the same amino acid but with sequence body differences) and included replication of 2 of the 3 genome-wide tRNAs.
Additionally, this isodecoder-specificity may indicate the potential for regulatory fragment changes with age.</p>
<p>In this study we provide the first comprehensive evaluation at the genomic DNA methylation state of the human tRNAome, revealing a discreet and strongly directional hypermethylation with advancing age.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="introduction-2" class="section level2">
<h2><span class="header-section-number">4.2</span> Introduction</h2>
<p>Ageing is implicated as a risk factor in multiple chronic diseases <span class="citation">(Partridge, Deelen, and Slagboom <a href="#ref-Partridge2018" role="doc-biblioref">2018</a>)</span>.
Understanding how the ageing process leads to deteriorating biological function is now a major research focus, with hopes to increase the human ‘healthspan’ and ameliorate the extensive physical, social and economic costs of these ageing-related disorders <span class="citation">(Campisi et al. <a href="#ref-Campisi2019" role="doc-biblioref">2019</a>)</span>.
Epigenetic processes, which influence or inform cell-type specific gene expression, are altered with age and are a fundamental hallmark of this progression, indeed they are arguably a hub mediating other hallmarks including stem cell exhaustion, cellular senescence, and mitochondrial dysfunction <span class="citation">(López-Otín et al. <a href="#ref-Lopez-Otin2013" role="doc-biblioref">2013</a>; Booth and Brunet <a href="#ref-Booth2016" role="doc-biblioref">2016</a>)</span>.</p>
<p>DNA methylation (DNAm) is the most common epigenetic modification of DNA and age-associated changes in this mark were recognised in mammalian tissues as early as 1983 <span class="citation">(Wilson and Jones <a href="#ref-Wilson1983" role="doc-biblioref">1983</a>)</span>.
Changes in DNAm with age are extensive with thousands of loci affected.
Many of these changes represent ‘drift’ <span class="citation">(Fraga et al. <a href="#ref-Fraga2005" role="doc-biblioref">2005</a>)</span> arising from the imperfect maintenance of methylation state.
Specific genomic regions show distinct directional changes, with loss of DNA methylation in repetitive or transposable elements <span class="citation">(Chuong, Elde, and Feschotte <a href="#ref-Chuong2017" role="doc-biblioref">2017</a>)</span>, and gains in certain promoters, including the targets of polycomb repressor complex <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> as well as bivalent domains <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>)</span>.
The advent of high-throughput DNAm arrays <span class="citation">(Sandoval, Heyn, Moran, Serra-Musach, Pujana, et al. <a href="#ref-Sandoval2011" role="doc-biblioref">2011</a><a href="#ref-Sandoval2011" role="doc-biblioref">b</a>; Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>; Hannum et al. <a href="#ref-Hannum2013" role="doc-biblioref">2013</a>)</span> has elucidated more detailed patterns of ageing related changes in DNAm.
The identification of precise individual CpG sites that exhibit consistent changes with age enabled the construction of predictors of chronological age known as epigenetic or DNAm ‘clocks’ <span class="citation">(Hannum et al. <a href="#ref-Hannum2013" role="doc-biblioref">2013</a>; Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>; Levine et al. <a href="#ref-Levine2018" role="doc-biblioref">2018</a>; Horvath et al. <a href="#ref-Horvath2018a" role="doc-biblioref">2018</a>; Lu et al. <a href="#ref-Lu2019" role="doc-biblioref">2019</a>)</span>.
Furthermore, it was observed that ‘acceleration’ of this DNAm-derived measure is a biomarker of ‘biological’ ageing due to associations with morbidity and mortality (Reviewed in <span class="citation">(Horvath and Raj <a href="#ref-Horvath2018" role="doc-biblioref">2018</a>)</span> &amp; <span class="citation">(Field et al. <a href="#ref-Field2018" role="doc-biblioref">2018</a>)</span>).
In a previous investigation of ageing-related DNAm changes within common disease-associated GWAS regions, we identified hypermethylation of the specific transfer RNA gene, tRNA-iMet-CAT-1-4 <span class="citation">(Bell et al. <a href="#ref-Bell2016" role="doc-biblioref">2016</a>)</span>.
The initiator methionine tRNA possesses certain unique properties, including its capacity to be rate limiting for translation <span class="citation">(Rideout, Marshall, and Grewal <a href="#ref-Rideout2012" role="doc-biblioref">2012</a>)</span>, association with the translation initiation factor eIF2 <span class="citation">(Kolitz and Lorsch <a href="#ref-Kolitz2010" role="doc-biblioref">2010</a>)</span>, and ability to impact the expression of other tRNA genes <span class="citation">(Pavon-Eternod et al. <a href="#ref-Pavon-Eternod2013" role="doc-biblioref">2013</a>)</span>.</p>
<p>tRNAs are fundamental in the translation process for all domains of life and are thus evolutionarily ancient <span class="citation">(Eigen et al. <a href="#ref-Eigen1989" role="doc-biblioref">1989</a>)</span>.
This translation machinery and the regulation of protein synthesis are controlled by conserved signalling pathways shown to be modifiable in longevity and ageing interventions <span class="citation">(Tavernarakis <a href="#ref-Tavernarakis2008" role="doc-biblioref">2008</a>)</span>.
Additionally, beyond their core role in the information flow from DNA to protein sequence, tRNAs can fragment into numerous tRNA-derived small RNAs (tsRNAs) <span class="citation">(Pliatsika et al. <a href="#ref-Pliatsika2018" role="doc-biblioref">2018</a>)</span> with signalling and regulatory functions <span class="citation">(Schimmel <a href="#ref-Schimmel2017" role="doc-biblioref">2018</a>; Lee et al. <a href="#ref-Lee2009" role="doc-biblioref">2009</a>)</span>.
Four main types of tsRNAs have been identified, 5’ tRNA halves, 3’ tRNA halves, and the tRNA derived fragments (tRFs) of ~18 and ~22bp sizes <span class="citation">(Torres et al. <a href="#ref-Torres2019" role="doc-biblioref">2019</a>; Li, Xu, and Sheng <a href="#ref-Li2018a" role="doc-biblioref">2018</a>; Xu et al. <a href="#ref-Xu2017a" role="doc-biblioref">2017</a>)</span>.</p>
<p>The human tRNAome, comprising all of the genomic locations of tRNA genes, represents an extremely small portion of the genome <span class="citation">(Parisien, Wang, and Pan <a href="#ref-Parisien2013" role="doc-biblioref">2013</a>)</span>.
The 610 annotated tRNA genes (gtRNAdb <span class="citation">(Chan and Lowe <a href="#ref-Chan2009" role="doc-biblioref">2009</a>)</span>) cover &lt;46 kb (including introns) and represents &lt;0.002% of the human genome.
Yet these genes produce the second most abundant RNA species next to ribosomal RNA <span class="citation">(Lodish et al. <a href="#ref-Lodish2000" role="doc-biblioref">2000</a>)</span> and are required for the production of all proteins.
tRNA genes are transcribed by RNA polymerase III (polIII) <span class="citation">(Schramm <a href="#ref-Schramm2002" role="doc-biblioref">2002</a>)</span> and have type II polIII promoters which contain A and B-box internal promoter elements bound by the complex TFIIIC, followed by TFIIIB, and polIII <span class="citation">(Canella et al. <a href="#ref-Canella2010" role="doc-biblioref">2010</a>)</span>.
DNAm is able to repress the expression of tRNA genes experimentally <span class="citation">(Besser et al. <a href="#ref-Besser1990" role="doc-biblioref">1990</a>)</span> but may also represent co-ordination with the local repressive chromatin state <span class="citation">(Varshney et al. <a href="#ref-Varshney2015" role="doc-biblioref">2015</a>)</span>.
Transcription is also repressed by the highly conserved polIII specific transcription factor Maf1 <span class="citation">(Murawski et al. <a href="#ref-Murawski1994" role="doc-biblioref">1994</a>; Pluta et al. <a href="#ref-Pluta2001" role="doc-biblioref">2001</a>)</span>, who’s activity is modulated by the Target of Rapamycin Kinase Complex 1 (TORC1) <span class="citation">(Mange et al. <a href="#ref-Mange2017" role="doc-biblioref">2017</a>)</span>, a highly conserved hub for signals that modulate ageing <span class="citation">(Kennedy and Lamming <a href="#ref-Kennedy2016" role="doc-biblioref">2016</a>)</span>.</p>
<p>Assuming similar rates of transcription one would expect that the more frequently an amino acid is used in the exome, the more copies of that tRNA gene there would be in the genome.
Indeed, tRNA gene dosage is quite closely matched to amino acid usage frequency in the human exome, though the correlation is less strong for codon usage.
Codon usage frequencies have been claimed to be mostly invariant in the transcriptomes of a wide range of tissues, as well as across developmental time <span class="citation">(Bianca M Schmitt et al. <a href="#ref-Schmitt2014" role="doc-biblioref">2014</a>)</span>.
However, differences in the codon usage of genes with highly tissue-specifically expressed genes have been observed <span class="citation">(Powell and Moriyama <a href="#ref-Powell1997" role="doc-biblioref">1997</a>)</span>.
The number of isodecoders (tRNAs with the same anticodon but which differ in sequence elsewhere) in the genome expands with organismal complexity <span class="citation">(Goodenbour and Pan <a href="#ref-Goodenbour2006" role="doc-biblioref">2006</a>)</span>.
Isodecoders are distinct in translational efficiency <span class="citation">(Geslain and Pan <a href="#ref-Geslain2010" role="doc-biblioref">2010</a>)</span>, which can have consequences in human disease <span class="citation">(Kirchner et al. <a href="#ref-Kirchner2017" role="doc-biblioref">2017</a>)</span>.
However, such differences seem likely to explain only some of the tissue specificity of tRNA gene regulation.</p>
<p>Ageing is linked to core aspects of metabolic regulation, with nutrient sensing and stress response acting as major modulators of ageing.
tRNAs as well as tsRNAs are integral to the regulation of protein synthesis and stress response.
protein synthesis represents a substantial proportion of total cellular energy expenditure and this fraction can vary considerably with nutrient availability <span class="citation">(Pace <a href="#ref-Pace2006" role="doc-biblioref">2006</a>)</span>.
Metabolic processes are also recognised to modulate the age estimates of DNAm clocks <span class="citation">(Nwanaji-Enwerem, Weisskopf, and Baccarelli <a href="#ref-Nwanaji-Enwerem2018" role="doc-biblioref">2018</a>)</span>.
Partial inhibition of translation increases lifespan in multiple model organisms <span class="citation">(Hansen et al. <a href="#ref-Hansen2007" role="doc-biblioref">2007</a>)</span> and PolIII inhibition increases longevity acting downstream of TORC1 <span class="citation">(Filer et al. <a href="#ref-Filer2017" role="doc-biblioref">2017</a>)</span>.
Furthermore, 5’ tRNA halves circulating in serum are modulated by ageing and caloric restriction <span class="citation">(Dhahbi et al. <a href="#ref-Dhahbi2013" role="doc-biblioref">2013</a>)</span>.</p>
<p>We directly investigated ageing-related changes in the epigenetic DNA methylation state of the entire tRNAome, facilitated by the availability of a large-scale MeDIP-seq dataset.
Arrays poorly cover this portion of genome.
The 450k and EPIC arrays have 110 robust probes covering 84 tRNAs and 129 robust probes covering 89 tRNAs respectively <span class="citation">(Zhou, Laird, and Shen <a href="#ref-Zhou2017" role="doc-biblioref">2016</a>)</span>.
tRNA genes sit at the heart not only of the core biological process of translation but at a nexus of signalling networks operating in several different paradigms, from small RNA signalling to large scale chromatin organisation <span class="citation">(Van Bortle, Phanstiel, and Snyder <a href="#ref-VanBortle2017" role="doc-biblioref">2017</a>)</span>.
In summation tRNA biology, protein synthesis, nutrient sensing, stress response and ageing are intimately interlinked.
We have identified tRNAome DNA hypermethylation and independently replicated this newly described ageing-related observation.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="Methods" class="section level2">
<h2><span class="header-section-number">4.3</span> Methods</h2>
<div id="participants" class="section level3">
<h3><span class="header-section-number">4.3.1</span> Participants</h3>
<p>Participants in the ‘EpiTwins’ study are adult volunteers from the TwinsUK Register.
The participants were aged between 16 and 82 years, with a median of ~55 years (cohort profile <span class="citation">(Moayyeri et al. <a href="#ref-Moayyeri2013" role="doc-biblioref">2013</a>)</span>).
Ethics for the collection of these data were approved by Guy’s &amp; St Thomas’ NHS Foundation Trust Ethics Committee (EC04/015—15-Mar-04) and written informed consent was obtained from all participants.</p>
<p>Participants for our targeted bisulfite sequencing of select tRNA loci were drawn from two studies.
Samples from participants aged 4 and 28 years are from the MAVIDOS <span class="citation">(Harvey et al. <a href="#ref-Harvey2012a" role="doc-biblioref">2012</a>)</span> study and participants aged 63 and 78 years are from the Hertfordshire cohort study <span class="citation">(Syddall et al. <a href="#ref-Syddall2005" role="doc-biblioref">2005</a>)</span>.
Due to a limited number of available samples, the two 4 year old pools contained DNA from 20 individuals each, with all other pools having 25 contributing individuals.
Pool 1, the first 4 year old pool used DNA from all male samples, with all other pools using all female samples.
Thus, the total number of participants was 190 (see Table ).
Samples from the 28 year old time point are all from pregnant women at ~11 weeks gestation.</p>
</div>
<div id="tRNAanno" class="section level3">
<h3><span class="header-section-number">4.3.2</span> tRNA annotation information</h3>
<p>Genomic coordinates of the tRNA genes were downloaded from GtRNAdb <span class="citation">(Chan and Lowe <a href="#ref-Chan2009" role="doc-biblioref">2009</a>)</span>.
The 2 tRNAs located in chr1_gl000192_random are tRNA-Gly-CCC-8-1 &amp; tRNA-Asn-ATT-1-2 (<a href="./Supplementary_Files/hg19-tRNAs.bed">Supplementary File S1</a>).
Stem/loop structure annotations were inferred from output of <code>tRNAscan</code> <span class="citation">(Lowe and Chan <a href="#ref-Lowe2016" role="doc-biblioref">2016</a>)</span> with a custom <code>perl6</code> script.
The 213 probes overlapping tRNA genes were derived from intersecting the tRNA gene annotation data from gtRNAdb with the Illumina 450k array manifest annotation for the hg19 genome build using bedtools v2.17.0 <span class="citation">(Quinlan and Hall <a href="#ref-Quinlan2010" role="doc-biblioref">2010</a>)</span>.
We excluded 107 tRNAs from blacklisted regions of hg19 <span class="citation">(Amemiya, Kundaje, and Boyle <a href="#ref-Amemiya2019" role="doc-biblioref">2019</a>)</span>.</p>
</div>
<div id="dna-methylome-data" class="section level3">
<h3><span class="header-section-number">4.3.3</span> DNA methylome data</h3>
<div id="EpiTwinsMethods" class="section level4">
<h4><span class="header-section-number">4.3.3.1</span> TwinsUK MeDIP-seq methylomes</h4>
<p>The Methylated DNA Immunoprecipitation sequencing (MeDIP-seq) data was processed as previously described <span class="citation">(Bell et al. <a href="#ref-Bell2016" role="doc-biblioref">2016</a>, <a href="#ref-Bell2017a" role="doc-biblioref">2018</a>)</span>.
These processed data are available from the European Genome-phenome Archive (EGA) (<a href="https://www.ebi.ac.uk/ega" class="uri">https://www.ebi.ac.uk/ega</a>) under study number EGAS00001001910 and dataset EGAD00010000983.
The dataset used in this work consists of 4350 whole blood methylomes with age data.
4054 are female and 270 male.
3001 have full blood counts.
There are 3652 individuals in this data set.
These individuals originate from 1933 unique families.
There are 1234 monozygotic (MZ) twin pairs (2468 individuals), and 458 dizygotic (DZ) twin pairs (916 individuals).</p>
<p>MeDIP-seq used a monoclonal anti-5mC antibody to bind denatured fragmented genomic DNA at methylated CpG sites.
This antibody-bound fraction of DNA was isolated and sequenced <span class="citation">(Down et al. <a href="#ref-Down2009" role="doc-biblioref">2008</a>)</span>.
MeDIP-seq 50-bp single-end sequencing reads were aligned to the hg19/GRCh37 assembly of the human genome and duplicates were removed.
<code>MEDIPS</code> (v1.0) was used for the MeDIP-seq specific analysis <span class="citation">(Lienhard et al. <a href="#ref-Lienhard2014" role="doc-biblioref">2014</a>)</span>.
This produced reads per million base pairs (RPM) values binned into 500bp windows with a 250bp slide in the BED format, resulting in ~12.8 million windows on the genome.
MeDIP-seq data from regions of interest was extracted using Bedtools v2.17.0 <span class="citation">(Quinlan and Hall <a href="#ref-Quinlan2010" role="doc-biblioref">2010</a>)</span>.</p>
</div>
<div id="medipmodels" class="section level4">
<h4><span class="header-section-number">4.3.3.2</span> Analysis of DNA methylome data for Significant Ageing-related changes</h4>
<p>All analysis was performed in R/3.5.2.
Linear models were fitted to age using the MeDIP-seq DNA methylome data, as quantile normalised RPM scores at each 500bp window.
Models were fitted with: 1. No covariates; 2. Batch information as a fixed effect; 3. Blood cell-type counts for neutrophils, monocytes, eosinophils, and lymphocytes as fixed effects; and 4. Batch and Blood Cell counts as fixed effects.
Model 1 &amp; 2 were fitted on the full set of 4350 as batch information was available for all samples but blood cell count data was only available for a subset of 3001 methylomes.
Models 1 &amp; 2 fitted in the n=3001 subset were similar to those fitted in the complete set of 4350.
Models 3 &amp; 4 were fitted in the n=3001 subset with full covariate information and sets of significant tRNAs identified at study-wide and genone wide levels in model 4 were used in subsequent analyses.
Models were also fitted for two unrelated subsets created by selecting one twin from each pair (Monozygotic or Dizygotic), yeilding sets with n = 1198 &amp; 1206 DNA methylomes.
One additional model was fitted for longitudinal analysis, samples were selected by identifying individuals with a DNA methylome at more than one time point and filtering for only those with a minimum of 5 years between samples.
This yielded 658 methylomes from 329 individuals with age differences of 5-16.1 yrs, median 7.6 yrs.
Models for this set included participant identifier as a fixed effect in addition to blood cell counts and batch information.</p>
</div>
<div id="AgeErichmentPermutation" class="section level4">
<h4><span class="header-section-number">4.3.3.3</span> Permutation Analysis for Enrichment with Age-related Changes</h4>
<p>We performed a permutation analysis to determine whether the CpG distribution of sets of the tRNAome was the principle driver of the ageing-related changes observed.
Windows overlapping tRNAs have a higher proportion of windows with a greater CpG density than their surrounding sequences (see supplementary Figure ).
CpGs residing within moderate CpG density loci are the most dynamic in the genome <span class="citation">(Ziller et al. <a href="#ref-Ziller2013" role="doc-biblioref">2013</a>)</span> and CpG dense CpG island regions include specific ageing-related changes <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>; Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>; Bell et al. <a href="#ref-Bell2016" role="doc-biblioref">2016</a>)</span>.
For comparison we also performed the permuation in the CGI regions from the Polycomb group protein target promoters in Teschendorff <em>et al.</em> <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> and bivalent loci from ENCODE ChromHmm ‘Poised Promoter’ classification in the GM12878 cell-line <span class="citation">(Ernst et al. <a href="#ref-Ernst2011" role="doc-biblioref">2011</a>)</span>.
A random set of 500bp windows representing an equivalent CpG density distribution of the feature set in question were selected from the genome-wide data.
Above a certain CpG density there are insufficient windows to sample without replacement within a permutation.
Furthermore, above ~<span class="math inline">\(\ge18\%\)</span> CpG density CpG Islands become consistently hypomethylated <span class="citation">(Christopher G. Bell et al. <a href="#ref-Bell2012a" role="doc-biblioref">2012</a>)</span>.
Therefore, all windows with a CpG density of <span class="math inline">\(\ge18\%\)</span> (45 CpGs per 500bp) were grouped and sampled from the same pool.
i.e. a window overlapping a tRNA gene which had a 20% density could be represented in permutation by one with any density <span class="math inline">\(\ge18\%\)</span>.
This permutation was performed 1,000 times to determine an Empirical p value by calculating the number of times the permutation result exceeded the observed number of significant windows in the feature set.
<span class="math inline">\(Empirical~p-value = \frac{r+1}{N+1}\)</span>, where r is the sum of significant windows in all permutations and N is number of permutations <span class="citation">(North, Curtis, and Sham <a href="#ref-North2003" role="doc-biblioref">2003</a>)</span>.</p>
</div>
<div id="neonate-and-centenarian-whole-genome-bisulfite-sequencing" class="section level4">
<h4><span class="header-section-number">4.3.3.4</span> Neonate and Centenarian Whole Genome Bisulfite Sequencing</h4>
<p>DNA methylation calls were downloaded from <a href="GEO:%5BGSE31263%5D(https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31263" class="uri">GEO:[GSE31263](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31263</a>) and intersected with tRNA genes using bedtools v2.17.0 <span class="citation">(Quinlan and Hall <a href="#ref-Quinlan2010" role="doc-biblioref">2010</a>)</span>.</p>
</div>
<div id="sample-pooling-and-epic-array" class="section level4">
<h4><span class="header-section-number">4.3.3.5</span> Sample pooling and EPIC array</h4>
<p>We performed an Illumina Infinium DNA methylation EPIC array ((C) Illumina) and targeted bisulfite sequencing of select tRNA gene loci.
Here we used DNA extracted from whole blood and pooled into 8 samples from unrelated individuals at 4 time-points with 2 pools at each time-point.
The timepoints were 4, 28, 63, and 78 years.
Using the EPIC array we were able to infer the DNAm age using the Horvath DNAm clock <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span> and blood cell-type composition of our samples using the Houseman algorithm <span class="citation">(Houseman et al. <a href="#ref-Houseman2012" role="doc-biblioref">2012</a>)</span>.</p>
</div>
<div id="targetedBiSseq" class="section level4">
<h4><span class="header-section-number">4.3.3.6</span> Targeted Bisulfite Sequencing</h4>
<p>We selected tRNA loci for targeted sequencing in which have had observed changes and DNAm with age and closely related tRNAs in which changes were not observed.
Primer design was performed using ‘methPrimer’ <span class="citation">(Li and Dahiya <a href="#ref-Li2002" role="doc-biblioref">2002</a>)</span> (<a href="./Supplementary_Files/Targeted_BiS-seq_Primers.txt">Supplementary File S2</a>).
A total of 84 tRNA loci were targeted and 79 subsequently generated reliable results post-QC.
The targeted tRNAs covered a total of 723 CpGs with a median of 8 CpGs per tRNA (range 1-13), data passing QC was generated for 458 CpGs, median 6 (range 1-9) per tRNA.</p>
<p>Quality was assessed before and after read trimming using <code>fastqc</code> <span class="citation">(Andrews <a href="#ref-Andrews2010" role="doc-biblioref">2010</a>)</span> and <code>multiqc</code> <span class="citation">(Ewels et al. <a href="#ref-Ewels2016" role="doc-biblioref">2016</a>)</span> to visualise the results.
Targeting primers were trimmed with <code>cutadapt</code> <span class="citation">(Martin <a href="#ref-Martin2011" role="doc-biblioref">2011</a>)</span> and a custom <code>perl5</code> script.
Quality trimming was performed with <code>trim_galore</code> <span class="citation">(Krueger <a href="#ref-Krueger2015" role="doc-biblioref">2015</a>)</span>.
Alignment and methylation calling was performed with <code>Bismark</code> (v0.20.0) <span class="citation">(Krueger and Andrews <a href="#ref-Krueger2011" role="doc-biblioref">2011</a>)</span> making use of <code>bowtie2</code> <span class="citation">(Langmead and Salzberg <a href="#ref-Langmead2012" role="doc-biblioref">2012</a>)</span>.
The alignment was performed against both the whole hg19 genome and just the tRNAome +/- 100bp to assess the possible impact of off-target mapping. Mapping to the whole genome did produce purported methylation calls at a larger number of loci than mapping just to the tRNAome (683,783 vs 45,861 respectively).
Introducing a minimum coverage threshold of 25 reads dramatically reduced this and brought the number of sites into line with that in the tRNAome set (36,065 vs 33,664 respectively) suggesting a small number of ambiguously mapping reads. All subsequent analysis was performed using the alignment to just the tRNAome with a minimum coverage of 25 reads.</p>
<p>We performed pairwise differential methylation analysis of the tRNA genes at the different time points using <code>RnBeads</code> <span class="citation">(Müller et al. <a href="#ref-Muller2019" role="doc-biblioref">2019</a>)</span> with <code>limma</code> <span class="citation">(Ritchie et al. <a href="#ref-Ritchie2015" role="doc-biblioref">2015</a>)</span> and a minimum coverage of 25 reads.
We also performed linear regression predicting age from DNA methylation at the targeted tRNA sites, permitting us to compare rates of increase with age.
For the linear regression, we used only CpG sites with more than 25 reads mapped to the regions of the genome targeted for amplification.</p>
</div>
<div id="twinsuk-illumina-450k-array-methylomes" class="section level4">
<h4><span class="header-section-number">4.3.3.7</span> TwinsUK Illumina 450k array methylomes</h4>
<p>Illumina Infinium DNA methylation 450k arrays ((C) Illumina) were also performed on TwinsUK participants, in 770 Blood-derived DNA samples which had matched MeDIP-seq data.
These data were preprocessed in the form of methylation ‘beta’ values pre-processed as previously described <span class="citation">(Bell et al. <a href="#ref-Bell2016" role="doc-biblioref">2016</a>, <a href="#ref-Bell2017a" role="doc-biblioref">2018</a>)</span>.
Cell-type correction was performed using cell-count data and the following model: <code>lm(age ~ beta + eosinophils + lymphocytes + monocytes + neutrophils)</code>.</p>
</div>
</div>
<div id="chromatin-segmentation-data" class="section level3">
<h3><span class="header-section-number">4.3.4</span> Chromatin Segmentation Data</h3>
<p>Epilogos chromatin segmentation data <span class="citation">(Meuleman <a href="#ref-Meuleman2019" role="doc-biblioref">2019</a>)</span> was downloaded for the tRNA gene regions +/- 200bp from <a href="https://explore.altius.org/tabix/epilogos/hg19.15.Blood_T-cell.KL.gz" class="uri">https://explore.altius.org/tabix/epilogos/hg19.15.Blood_T-cell.KL.gz</a> using the <code>tabix</code> utility.
The data used was the ‘Blood &amp; T-cell’ 15 State model based on segmentation of 14 cell-types.
This data was manipulated and visualised with <code>R</code> and <code>ggplot2</code>.</p>
</div>
<div id="sortedbloodmethods" class="section level3">
<h3><span class="header-section-number">4.3.5</span> Isolated Blood Cell Type Specific Data</h3>
<p>Data from 7 cell-type fractions from 6 Male individuals was downloaded from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069">GSE35069</a> <span class="citation">(Reinius et al. <a href="#ref-Reinius2012" role="doc-biblioref">2012</a>)</span> using <code>GEOquery</code> <span class="citation">(Sean and Meltzer <a href="#ref-Davis2007p" role="doc-biblioref">2007</a>)</span>.
Five of the 6 top age hypermethylating tRNAs are covered by this array dataset.</p>
</div>
<div id="TCGAmethods" class="section level3">
<h3><span class="header-section-number">4.3.6</span> Cancer and Tissue Specific Methylation Data</h3>
<p>Data was downloaded from the TCGA (The Cancer Genome Atlas) via the GDC (genomic data commons) data portal <span class="citation">(Grossman et al. <a href="#ref-Grossman2016" role="doc-biblioref">2016</a>)</span> using the <code>GenomicDataCommons</code> <code>R</code> package.
Data from foetal tissue <span class="citation">(Yang et al. <a href="#ref-Yang2016" role="doc-biblioref">2016</a>; Nazor et al. <a href="#ref-Nazor2012" role="doc-biblioref">2012</a>)</span> was downloaded from GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72867">GSE72867</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30654">GSE30654</a>).
From the TCGA, we selected samples for which DNAm data was available from both the primary site and normal solid tissue, and for which we could infer an approximate age (within one year).
We selected those probes overlapping tRNA genes yielding 73,403 data points across 19 tissues with an age range of 15-90yrs (median 63.4) (<a href="./Supplementary_Files/TCGA_samples_used.tsv">Supplementary File S3</a>)</p>
</div>
<div id="mintmapmethods" class="section level3">
<h3><span class="header-section-number">4.3.7</span> Assaying tRNA expression in blood with MINTmap</h3>
<p>We used small RNA-seq data from sorted blood cell fractions <span class="citation">(Juzenas et al. <a href="#ref-Juzenas2017" role="doc-biblioref">2017</a>)</span> (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE100467">GSE100467</a>) and the MINTmap <span class="citation">(Loher, Telonis, and Rigoutsos <a href="#ref-Loher2017" role="doc-biblioref">2017</a>)</span> tRNA fragment alignment tool.
This dataset covered 42 individuals aged 21-63.
We also created a customised MINTmap reference designed to include only fragments which unambiguously map to a single tRNA gene locus and which overlap the 5’ or 3’ end of the genomic tRNA sequence by at least one base with no mismatches.
This reference is intended to capture pre-tRNAs prior to processing and CCA addition operating under the assumption that the levels of pre-tRNAs will be informative about the amount of transcription taking place at the tRNA loci.
This approach provides at most a many to one mapping of tRNA fragment to a tRNA gene.</p>
<p>Assaying the expression of tRNA genes presents numerous difficulties <span class="citation">(Torres et al. <a href="#ref-Torres2019" role="doc-biblioref">2019</a>)</span>, and usually requires variants on standard RNA-seq protocols.
Our custom MINTmap reference build yielded 383 fragments mapping to 92 distinct tRNA loci in this data.
To control quality only fragments with more than 20 total instances in the dataset and present in more than 20 individuals were considered.</p>
<p>The maximum length of a fragment was limited to 50nt, due to the read length of the small RNA-seq data.</p>
</div>
<div id="mousemethods" class="section level3">
<h3><span class="header-section-number">4.3.8</span> Mouse RRBS Analysis</h3>
<p>We downloaded methylation calls and coverage information resulting from RRBS performed by Petkovich <em>et al.</em> <span class="citation">(Petkovich et al. <a href="#ref-Petkovich2017" role="doc-biblioref">2017</a>)</span> from GEO using <code>GEOquery</code> <span class="citation">(Sean and Meltzer <a href="#ref-Davis2007p" role="doc-biblioref">2007</a>)</span> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE80672">GSE80672</a>.
These data from 152 mice covered 68 tRNA and 436 CpGs after QC requiring &gt;50 reads per CpG and &gt;10 data points per tRNA.
We excluded 5 tRNAs from blacklisted regions of mm10 <span class="citation">(Amemiya, Kundaje, and Boyle <a href="#ref-Amemiya2019" role="doc-biblioref">2019</a>)</span>.
After QC there were 58 tRNA genes and 385 CpGs.
We performed simple linear modeling to predict age from methylation level at each tRNA and each CpG.</p>
<div style="page-break-after: always;"></div>
</div>
</div>
<div id="supplementary-materials" class="section level2 unnumbered">
<h2>Supplementary materials</h2>
<div id="supplementary-figures" class="section level3">
<h3><span class="header-section-number">4.3.9</span> Supplementary Figures</h3>
<div class="figure" style="text-align: center"><span id="fig:mappabilityiMet1"></span>
<img src="figs/mappability-iMet1.png" alt="Example of mappability data from the encode mappability tracks (Derrien et al. 2012) for the initiator methionine tRNA genes." width="100%" />
<p class="caption">
Figure 4.1: Example of mappability data from the encode mappability tracks <span class="citation">(Derrien et al. <a href="#ref-Derrien2012" role="doc-biblioref">2012</a>)</span> for the initiator methionine tRNA genes.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:mappabilityScoreDensity"></span>
<img src="figs/mappabilityScoreDensity_tRNAVsFlank500.png" alt="Mappability score density of the tRNAome increases with read length and is greater when flanking regions (\(\pm500bp\)) are included. Mappability score density is computed as the area under the encode mappability tracks (Derrien et al. 2012) over the length of the region." width="100%" />
<p class="caption">
Figure 4.2: Mappability score density of the tRNAome increases with read length and is greater when flanking regions (<span class="math inline">\(\pm500bp\)</span>) are included. Mappability score density is computed as the area under the encode mappability tracks <span class="citation">(Derrien et al. <a href="#ref-Derrien2012" role="doc-biblioref">2012</a>)</span> over the length of the region.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:CpGcountBGvstRNAPlot"></span>
<img src="figs/CpGcountBGvstRNAPlot.png" alt="CpG Density in windows overlapping tRNA genes compared to that of non-tRNA overlapping windows in flanking sequences (+/-5kb)" width="100%" />
<p class="caption">
Figure 4.3: CpG Density in windows overlapping tRNA genes compared to that of non-tRNA overlapping windows in flanking sequences (+/-5kb)
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:meanMethbytRNAMatHeatmapLow"></span>
<img src="figs/meanMethbytRNAMatHeatmapLow_bl.png" alt="Heatmap Mean Methylation of probes covering each tRNA in 7 cell-type fractions from 6 Male individuals. Showing all 150 tRNAs covered by 213 probes on the Illumina 450k array. Data from GSE35069 (Reinius et al. 2012) downloaded using GEOquery (Sean and Meltzer 2007). Generated with the ComplexHeatmap R package (Gu, Eils, and Schlesner 2016)." width="90%" />
<p class="caption">
Figure 4.4: Heatmap Mean Methylation of probes covering each tRNA in 7 cell-type fractions from 6 Male individuals. Showing all 150 tRNAs covered by 213 probes on the Illumina 450k array. Data from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069">GSE35069</a> <span class="citation">(Reinius et al. <a href="#ref-Reinius2012" role="doc-biblioref">2012</a>)</span> downloaded using <code>GEOquery</code> <span class="citation">(Sean and Meltzer <a href="#ref-Davis2007p" role="doc-biblioref">2007</a>)</span>. Generated with the <code>ComplexHeatmap</code> <code>R</code> package <span class="citation">(Gu, Eils, and Schlesner <a href="#ref-Gu2016" role="doc-biblioref">2016</a>)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:cancerVsNormalCombinedPlots"></span>
<img src="figs/cancerVsNormalCombinedPlots.png" alt="Global properties of tRNA methylation data for 45 tRNA genes across 19 tissues with matched normal and tumour samples from 733 cases in TCGA (Yang et al. 2016; Nazor et al. 2012)." width="100%" />
<p class="caption">
Figure 4.5: Global properties of tRNA methylation data for 45 tRNA genes across 19 tissues with matched normal and tumour samples from 733 cases in TCGA <span class="citation">(Yang et al. <a href="#ref-Yang2016" role="doc-biblioref">2016</a>; Nazor et al. <a href="#ref-Nazor2012" role="doc-biblioref">2012</a>)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:meanBetasByTissueTNMHeatmap"></span>
<img src="figs/meanBetasByTissueTNMpseudoSplitHeatmap.png" alt="Mean Methylation of 43 tRNAs in 19 tissues. Possible pseudogene (tRNA-Asn-ATT-1-1) is shown in a separate cluster beneath the main heatmap (Gu, Eils, and Schlesner 2016)." width="90%" />
<p class="caption">
Figure 4.6: Mean Methylation of 43 tRNAs in 19 tissues. Possible pseudogene (tRNA-Asn-ATT-1-1) is shown in a separate cluster beneath the main heatmap <span class="citation">(Gu, Eils, and Schlesner <a href="#ref-Gu2016" role="doc-biblioref">2016</a>)</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:meanMethBytRNAByFetalTissueHeatmap"></span>
<img src="figs/meanMethBytRNAByFetalTissueHeatmapPseudosplit.png" alt="Mean Methylation of 115 tRNAs in 11 tissues. Possible pseudogenes are shown in a separate cluster beneath the main heatmap (Gu, Eils, and Schlesner 2016)." width="60%" />
<p class="caption">
Figure 4.7: Mean Methylation of 115 tRNAs in 11 tissues. Possible pseudogenes are shown in a separate cluster beneath the main heatmap <span class="citation">(Gu, Eils, and Schlesner <a href="#ref-Gu2016" role="doc-biblioref">2016</a>)</span>.
</p>
</div>

</div>
<div id="mintmap-reference-fragment-distribution" class="section level3">
<h3><span class="header-section-number">4.3.10</span> MINTmap reference Fragment distribution</h3>
<p>In the original MINTmap reference (Figure b) there are peaks at around 18nt, 22nt and 32nt.
This is consistent with the expected tRNA fragment size distributions with ‘tRNA halves’ at 30-33nt and other tRFs at 18nt and 22nt.
In our custom reference (Figure a) whilst there is still a peak at ~18nt, with suggestions of peaks near 22nt and 32nt the tRNA fragment length distribution is somewhat different from that of the standard MINTmap reference.
There are larger peaks at ~28 and ~40nt consistent with the longer fragments expected given that this reference aimed to target fragments derived from pre-tRNAs not tRFs derived from mature tRNAs.</p>
<div class="figure" style="text-align: center"><span id="fig:combinedFragSizeDists"></span>
<img src="figs/combinedFragSizeDists.png" alt="Comparison of the fragment size distributions between our custom reference (A) and the original the MINTmap reference (B)." width="100%" />
<p class="caption">
Figure 4.8: Comparison of the fragment size distributions between our custom reference (A) and the original the MINTmap reference (B).
</p>
</div>

<div style="page-break-after: always;"></div>
</div>
</div>
<div id="res" class="section level2">
<h2><span class="header-section-number">4.4</span> Results</h2>
<div id="dna-methylation-of-specific-trna-gene-loci-changes-with-age" class="section level3">
<h3><span class="header-section-number">4.4.1</span> DNA Methylation of Specific tRNA Gene Loci Changes with Age</h3>
<p>Due to tRNAs critical role in translation and evidence of their modulation in ageing and longevity-related pathways, we interrogated these genes for evidence of ageing-related epigenetic changes.
Our discovery set was a large-scale peripheral blood-derived DNA methylome dataset comprising of 4350 samples (see Figure ).
This sequencing-based dataset had been generated by Methylated DNA Immunoprecipitation (MeDIP-seq) <span class="citation">(Down et al. <a href="#ref-Down2009" role="doc-biblioref">2008</a>)</span>, which relies on the enrichment of methylated fragments of 200-500 bp to give a regional DNAm assessment (500 bp semi-overlapping windows, see <a href="tRNApaper.html#EpiTwinsMethods">Methods</a>).
In total the human tRNAome is comprised of 610 tRNAs (gtRNAdb)(see Figure ), though only 492 are autosomal and do not reside in blacklisted regions of the genome <span class="citation">(Amemiya, Kundaje, and Boyle <a href="#ref-Amemiya2019" role="doc-biblioref">2019</a>)</span>.
Due to the small size of these tRNAs (60-86bp, median 73bp, excluding introns which are present in ~30 tRNAs with sizes from 10-99bp, median 19bp), this fragment-based method enabled a robust examination of the epigenetic state of these highly similar sequences.
This was supported by a mappability assessment.
The median mappability score density for the tRNAome was 0.90 for 50mers when considering tRNA genes <span class="math inline">\(\pm500bp\)</span> reflecting the regional nature of the MeDIP-seq assay.
In contrast the 50mer mappability density is 0.68 for tRNA genes alone representative of the mappability of reads generated using a technique such as whole-genome bisulfite sequencing (see Figures  &amp; ).</p>
<p>We identified 21 genome-wide significant and 44 study-wide significant results (p &lt; <span class="math inline">\(4.34\times10^{-9}\)</span> and <span class="math inline">\(8.36\times10^{-5}\)</span>, respectively, via linear regression see <a href="tRNApaper.html#medipmodels">Methods</a> (batch corrected n=4350).
Study-wide significance was calculated conservatively for all 598 autosomal tRNAs.
There was a strong directional trend with all results at both significance levels being due to increases in DNA methylation.
Age-related changes in cell type proportion are strong in heterogeneous peripheral blood, and include a myeloid skew, loss of naive T cells and increases in senescent cells <span class="citation">(Geiger, Haan, and Florian <a href="#ref-Geiger2013" role="doc-biblioref">2013</a>)</span>.
A subset of 3 genome-wide and 16 study-wide significant hypermethylation results remained significant even after correcting for potential cell-type changes by including lymphocytes, monocytes, neutrophils and eosinophil cell count data (n=3001, Listed in Table , Red in Figure ).
tRNA-iMet-CAT-1-4 is located on chromosome 6.
tRNA-Ile-AAT-4-1 and tRNA-Ser-AGA-2-6 are neighbours and are located on chromosome 17 within the 3’ UTR of <em>CTC1</em> (CST Telomere Replication Complex Component 1).
Going forward we refer to these most robustly corrected sets of 3 and 16 tRNA genes as the genome-wide and study-wide significant tRNA genes respectively.</p>
<p>Due to the related nature of these twin samples, we also analysed these data in two subsets of n=1198 &amp; 1206 by selecting one twin from each pair into the seperate sets.
This analysis also included correction for Batch and Blood Cell counts.
Whilst in these smaller datasets no tRNAs were genome-wide significant, 5 and 7 tRNA genes, respectively, reached study-wide significance.
In these sets 5/5 and 6/7 of these were present in 16 study-wide significant tRNA genes.</p>
<p>Furthermore, we examined a subset of samples with longitudinal data (n=658 methylomes from 329 individuals, median age difference 7.6 yrs).
At the nominal significance threshold (p &lt; 0.05) this yielded a split of 41 hypermethylating tRNA genes and 22 hypomethylating tRNA genes.
Of these hypermethylated tRNAs, 2 are in the previously identified genome-wide significant set of 3 (with tRNA-iMet-CAT-1-4 ranked 3rd by p-value) and 9 are in the study-wide significant set of 16.</p>
<div class="figure" style="text-align: center"><span id="fig:OverviewOfAnalyses"></span>
<img src="figs/OverviewOfAnalysesFigure_tRNA_3.png" alt="Study Structure. tRNAs differentially methylated with age initially identified in MeDIP-seq, validated (where covered in 450k array) and replicated in targeted bisulfite sequencing of pooled samples. Tissue specificity of these effects was explored in TCGA and foetal tissue data." width="100%" />
<p class="caption">
Figure 4.9: Study Structure. tRNAs differentially methylated with age initially identified in MeDIP-seq, validated (where covered in 450k array) and replicated in targeted bisulfite sequencing of pooled samples. Tissue specificity of these effects was explored in TCGA and foetal tissue data.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:kutteresq"></span>
<img src="figs/kutteresq_BLcount.png" alt="The genetic code as represented in the human tRNAome. The triplet genetic code leads to the incorporation of specific amino acids into an elongating protein via corresponding tRNAs. n is the number of tRNA genes which encode a given amino acid, the number in parentheses is how many of those may be pseudogenes based on their tRNAscan score (Lowe and Chan 2016), and the number in square braces is the number in blacklisted regions (Amemiya, Kundaje, and Boyle 2019). There are a total of 610 tRNAs in GtRNAdb (Chan and Lowe 2009), 116 of which are potential pseudogenes, and 107 are in blacklisted regions (Amemiya, Kundaje, and Boyle 2019). Notably 7 of the 61 non-STOP codons are missing from the human tRNAome therefore these codons are handled by wobble base matching (e.g. GCG Arg, ACC Gly). Also of note are the suppressor and selenocysteine tRNAs. The 20 methionine tRNAs are split equally between initiator methionine and internally incorporating methionine tRNAs, which are structurally distinct. There are also 23 nuclear encoded mitochondrial tRNAs." width="100%" />
<p class="caption">
Figure 4.10: The genetic code as represented in the human tRNAome. The triplet genetic code leads to the incorporation of specific amino acids into an elongating protein via corresponding tRNAs. n is the number of tRNA genes which encode a given amino acid, the number in parentheses is how many of those may be pseudogenes based on their tRNAscan score <span class="citation">(Lowe and Chan <a href="#ref-Lowe2016" role="doc-biblioref">2016</a>)</span>, and the number in square braces is the number in blacklisted regions <span class="citation">(Amemiya, Kundaje, and Boyle <a href="#ref-Amemiya2019" role="doc-biblioref">2019</a>)</span>. There are a total of 610 tRNAs in GtRNAdb <span class="citation">(Chan and Lowe <a href="#ref-Chan2009" role="doc-biblioref">2009</a>)</span>, 116 of which are potential pseudogenes, and 107 are in blacklisted regions <span class="citation">(Amemiya, Kundaje, and Boyle <a href="#ref-Amemiya2019" role="doc-biblioref">2019</a>)</span>. Notably 7 of the 61 non-STOP codons are missing from the human tRNAome therefore these codons are handled by wobble base matching (<em>e.g.</em> GCG Arg, ACC Gly). Also of note are the suppressor and selenocysteine tRNAs. The 20 methionine tRNAs are split equally between initiator methionine and internally incorporating methionine tRNAs, which are structurally distinct. There are also 23 nuclear encoded mitochondrial tRNAs.
</p>
</div>

</div>
<div id="trna-genes-are-enriched-for-age-related-dna-hypermethylation" class="section level3">
<h3><span class="header-section-number">4.4.2</span> tRNA Genes are Enriched for Age Related DNA Hypermethylation</h3>
<p>Whilst ageing changes are pervasive throughout the DNA methylome, we identified a strong enrichment for this to occur within the discrete tRNAome (Fisher’s Exact Test p = <span class="math inline">\(1.05\times10^{-27}\)</span>) (see Figure ).
This is still significant if the 6 of the study-wide significant 16 tRNAs that overlap polycomb or bivalent regions are excluded (p = <span class="math inline">\(4.66\times10^{-15}\)</span>)</p>
<div class="figure" style="text-align: center"><span id="fig:sizePropEnrich"></span>
<img src="figs/nSigWinProps_WGfishers_BB_sws_hyper_2_BL_noPolyBivExcl_BarTxtast_sr.png" alt="Age-related DNA hypermethylation Genomic Region Enrichment. (n = 3001, tRNAs are enriched compared to the genomic background, Fisher’s Exact Test p &lt; \(1.05\times10^{-27}\), Blood cell-type and batch corrected)." width="90%" />
<p class="caption">
Figure 4.11: Age-related DNA hypermethylation Genomic Region Enrichment. (n = 3001, tRNAs are enriched compared to the genomic background, Fisher’s Exact Test p &lt; <span class="math inline">\(1.05\times10^{-27}\)</span>, Blood cell-type and batch corrected).
</p>
</div>

<p>CpG density itself is known to have a clear impact on the potential for variability of the DNA methylome as well as ageing-related changes <span class="citation">(Ziller et al. <a href="#ref-Ziller2013" role="doc-biblioref">2013</a>; Christensen et al. <a href="#ref-Christensen2009" role="doc-biblioref">2009</a>)</span>.
To assess whether this hypermethylation finding was being merely driven by the inherent CpG density of the tRNAome, we performed a CpG density matched permutation analysis (1,000X, see <a href="tRNApaper.html#AgeErichmentPermutation">Methods</a>).
This supported the specific nature of these age-related DNAm changes within the functional tRNAome (Empirical p-value &lt; <span class="math inline">\(1.0\times10^{-3}\)</span>, Figure ).
As a point of comparison for this genomic functional unit, we also performed the same permutations for the known age-related changes in the promoters of genes that are polycomb group targets <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2010" role="doc-biblioref">2010</a>)</span> and those with a bivalent chromatin state <span class="citation">(Rakyan et al. <a href="#ref-Rakyan2010" role="doc-biblioref">2010</a>)</span>.
We were able to reproduce the enrichment of the polycomb group targets and bivalent regions (Empirical p-value &lt; <span class="math inline">\(1.0\times10^{-3}\)</span>) in our dataset.</p>
<div class="figure" style="text-align: center"><span id="fig:permHists"></span>
<img src="figs/tRNA-poly-bivP_n1000_combined_perm_hist.png" alt="CpG Density Genome-wide Permutation Analysis. Each permutation represented a random set of windows matching the CpG density of the functional unit (bivalent domains, polycomb group target promoters &amp; the tRNAome). These are subsequently assessed for signficant age-related DNAm changes (see Methods). The red line is the observed number of significant loci." width="100%" />
<p class="caption">
Figure 4.12: CpG Density Genome-wide Permutation Analysis. Each permutation represented a random set of windows matching the CpG density of the functional unit (bivalent domains, polycomb group target promoters &amp; the tRNAome). These are subsequently assessed for signficant age-related DNAm changes (see <a href="tRNApaper.html#AgeErichmentPermutation">Methods</a>). The red line is the observed number of significant loci.
</p>
</div>



<div class="figure" style="text-align: center"><span id="fig:circos"></span>
<img src="figs/tRNA_circos_test_inkscape_4.png" alt="Human tRNAome overview. From the outside in: Chromosome ideograms scaled by the number of tRNA genes (total = 598), as excludes chromosome X (10), Y (0) and contig chr1_gl000192_random (2; see Methods). tRNA genes within 20kbp of one another are grouped with breaks inserted between these clusters. Radial grey lines represent the location of tRNA genes in the genome. \(-log_{10}(p-value)\) for the blood cell-type and batch corrected age model are shown for each window overlapping a tRNA gene in green. Mean methylation across all samples (n=3001) in RPM (reads per million base pairs) is shown in blue. Genome-wide significant cell-type &amp; batch corrected (\(p &lt; 4.34\times10^{-9}\)) tRNAs show in red. The 158 Loci covered by 213 probes on the 450k array which directly overlap a tRNA gene are shown with green triangles. The 84 loci targeted for bisulfite sequencing in this study are indicated in magenta. Mappability score density is computed as the area under the encode mappability tracks (Derrien et al. 2012) over the length of the region." width="100%" />
<p class="caption">
Figure 4.13: Human tRNAome overview. From the outside in: Chromosome ideograms scaled by the number of tRNA genes (total = 598), as excludes chromosome X (10), Y (0) and contig chr1_gl000192_random (2; see <a href="tRNApaper.html#tRNAanno">Methods</a>). tRNA genes within 20kbp of one another are grouped with breaks inserted between these clusters. Radial grey lines represent the location of tRNA genes in the genome. <span class="math inline">\(-log_{10}(p-value)\)</span> for the blood cell-type and batch corrected age model are shown for each window overlapping a tRNA gene in green. Mean methylation across all samples (n=3001) in RPM (reads per million base pairs) is shown in blue. Genome-wide significant cell-type &amp; batch corrected (<span class="math inline">\(p &lt; 4.34\times10^{-9}\)</span>) tRNAs show in red. The 158 Loci covered by 213 probes on the 450k array which directly overlap a tRNA gene are shown with green triangles. The 84 loci targeted for bisulfite sequencing in this study are indicated in magenta. Mappability score density is computed as the area under the encode mappability tracks <span class="citation">(Derrien et al. <a href="#ref-Derrien2012" role="doc-biblioref">2012</a>)</span> over the length of the region.
</p>
</div>

<div style="page-break-after: always;"></div>
<div id="age-related-trnaome-dna-hypermethylation-is-even-observed-in-one-newborn-versus-one-centenarian" class="section level4">
<h4><span class="header-section-number">4.4.2.1</span> Age-related tRNAome DNA Hypermethylation is even observed in one Newborn versus one Centenarian</h4>
<p>We examined an available Whole Genome Bisulphite sequencing (WGBS) dataset from Heyn <em>et al.</em> <span class="citation">(Heyn et al. <a href="#ref-Heyn2012" role="doc-biblioref">2012</a>)</span> (see <a href="tRNApaper.html#Methods">Methods</a>)
These data consisted of blood-derived DNA WGBS in one newborn child and one 26 year old, and centenarian (103 years).
In their analysis, the centenarian was found to have more hypomethylated CpGs than the neonate across all genomic compartments, including promoters, exonic, intronic, and intergenic regions.
However, even in this examination of 3 individuals of 3 different age in the 55% of tRNA that possessed coverage, we observed DNA hypermethylation with age among the study-wide significant tRNA hypermethylators. The centenarian was significantly more methylated in this set of tRNAs than the neonate (Wilcoxon rank sum test, 6.14% increase (95% CI -Inf - 4.31), W = 717, p = <span class="math inline">\(6.14\times10^{-4}\)</span>, see Figure ).</p>
<div class="figure" style="text-align: center"><span id="fig:heyn"></span>
<img src="figs/heynViolinBBswsOnlyLabelled_BL.png" alt="Whole Genome Bisulfite Sequencing Data in a newborn, as adult and a centenarian. Each point represents the methylation level at an individual CpG within a tRNA gene. Numbers in parentheses indicate the number of tRNA genes for which methylation data is shown. SWS: Available study-wide significant tRNA genes." width="60%" />
<p class="caption">
Figure 4.14: Whole Genome Bisulfite Sequencing Data in a newborn, as adult and a centenarian. Each point represents the methylation level at an individual CpG within a tRNA gene. Numbers in parentheses indicate the number of tRNA genes for which methylation data is shown. SWS: Available study-wide significant tRNA genes.
</p>
</div>

</div>
<div id="age-related-changes-independently-replicated-with-targeted-bisulfite-sequencing" class="section level4">
<h4><span class="header-section-number">4.4.2.2</span> Age-related Changes Independently Replicated with Targeted Bisulfite Sequencing</h4>
<p>In order to further robustly support these-ageing related changes, we attempted to replicate these findings ourselves in an independent ageing dataset.
Furthermore, we employed a different technology Targeted bisulfite (BiS) sequencing to also further validate the MeDIP-seq-derived results.
These data provide individual CpG resolution to identify what may be driving the regional DNAm changes observed.</p>
<p>We performed this targeted BiS-seq in blood-derived DNA from 8 pools of age-matched individuals at 4 time-points (~4, ~28, ~63, ~78 years) from a total of 190 individuals, as detailed in Table .
A total of 79 tRNA loci generated reliable results post-QC (see <a href="tRNApaper.html#targetedBiSseq">Methods</a>).
These tRNAs covered a total of 458 CpGs with a median of 6 CpGs per tRNA (range 1-9).
Median Coverage per site across pools, technical replicates and batches was 679 reads (mean 5902).</p>
<p>Firstly, we ran the 8 Pooled samples on the Illumina EPIC (850k) array to confirm that our pooling approach was applicable for DNAm ageing-related evaluation.
This showed an <span class="math inline">\(R^2 = 0.98\)</span> between pool mean chronological age and Horvath clock DNAm predicted age <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span>(see Figure ).
Therefore, this confirmed the utility of this novel pooling approach.
We also used these array derived data to estimate the major blood cell proportions for each of these pools with the Houseman algorithm <span class="citation">(Houseman et al. <a href="#ref-Houseman2012" role="doc-biblioref">2012</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:DNAmAgeVsChronAge"></span>
<img src="figs/DNAmAgeVsChronAge.png" alt="Chronological Age compared with DNAm Age estimated with Horvath’s 2013 multi-tissue clock (Horvath 2013) based on EPIC array data for the 8 pooled samples used in the Targeted Bisulfite sequencing (See Table for pool details)." width="60%" />
<p class="caption">
Figure 4.15: Chronological Age compared with DNAm Age estimated with Horvath’s 2013 multi-tissue clock <span class="citation">(Horvath <a href="#ref-Horvath2013" role="doc-biblioref">2013</a>)</span> based on EPIC array data for the 8 pooled samples used in the Targeted Bisulfite sequencing (See Table  for pool details).
</p>
</div>

<p>We noted that individual tRNA loci exhibiting age-related changes in DNAm had duplicate or isodecoder (same anticodon but body sequence variation) sequences in the genome, which despite exact or near sequence identity did not show similar changes.
tRNA-iMet-CAT-1-4 for instance is 1 of 8 identical copies in the genome and was the only locus that showed significant changes.
The results of pairwise differential methylation tests between age groups for the 6 top tRNAs from the MeDIP-seq models are listed in Table .</p>
<p>Of the 3 top hits in MeDIP-seq, tRNA-iMet-CAT-1-4 (Figure c) and tRNA-Ser-AGA-2-6 (Figure i) exceeded nominal significance (p-values = <span class="math inline">\(9.35\times10^{-4}\)</span> &amp; <span class="math inline">\(4.28\times10^{-2}\)</span> , respectively).
However, tRNA-Ile-AAT-4-1 (Figure n) showed a nominal decrease in DNAm with age.
tRNA tRNA-Leu-TAG-2-1 from the study-wide significant set also showed nominally significant hypermethylation with age (Figure u).
Also, four of the individual CpGs in tRNA-iMet-CAT-1-4 exhibited nominally significant increases in DNAm with Age (Figure ).</p>
<div class="figure" style="text-align: center"><span id="fig:tRNAiMetCAT1x4nomSigCpG"></span>
<img src="figs/tRNA-iMet-CAT-1-4_nomSig_CpG.png" alt="Individual CpG methylation increases (nominally significant p &lt; 0.05) in tRNA-iMet-CAT-1-4." width="80%" />
<p class="caption">
Figure 4.16: Individual CpG methylation increases (nominally significant p &lt; 0.05) in tRNA-iMet-CAT-1-4.
</p>
</div>



<div style="page-break-after: always;"></div>


<div id="select-duplicates-isodecoders-of-hypermethylating-trna-loci-remain-unchanged" class="section level5">
<h5><span class="header-section-number">4.4.2.2.1</span> Select Duplicates &amp; Isodecoders of Hypermethylating tRNA loci remain unchanged</h5>
<p>We targeted a selection of these duplicate and isodecoder loci for bisulfite sequencing in order to confirm that the identified DNAm changes are specific to a given locus and not general to related tRNAs.
Examining the tRNA-iMet-CAT-1 family, only the previously identified 1-4 version confirmed significant hypermethylation (not 1-2, 1-3 or 1-5)(Figure a-e). Likewise the tRNA-Ser-AGA-2-6 version was supported compared to 2-1,2-4 and 2-5(Figure f-j)).</p>
<div class="figure" style="text-align: center"><span id="fig:combinedKaryoplots"></span>
<img src="figs/combinedKaryoplots_sig_bl.png" alt="(A-Q) Mean methylation across replicates at each CpG in each pool for select tRNAs. Ideograms show the locations of all tRNAs with the same anticodon, with those featured in the scatter plots highlighted in red and placed above the ideogram. (R-V) Except for leucine isoacceptors tRNAs are not all of the same codon and only tRNAs on the same chromosome as those plotted are labelled on the ideograms. (experiment-wide Bonferroni p = \(6.41\times10^{-4}\))." width="100%" />
<p class="caption">
Figure 4.17: (A-Q) Mean methylation across replicates at each CpG in each pool for select tRNAs. Ideograms show the locations of all tRNAs with the same anticodon, with those featured in the scatter plots highlighted in red and placed above the ideogram. (R-V) Except for leucine isoacceptors tRNAs are not all of the same codon and only tRNAs on the same chromosome as those plotted are labelled on the ideograms. (experiment-wide Bonferroni p = <span class="math inline">\(6.41\times10^{-4}\)</span>).
</p>
</div>

</div>
</div>
<div id="dna-methylation-450k-array-data-validates-the-medip-seq-results" class="section level4">
<h4><span class="header-section-number">4.4.2.3</span> DNA methylation 450k Array Data Validates the MeDIP-seq Results</h4>
<p>Although DNA methylation array poorly cover the tRNAome, we wished to attempt to see if this BiS-based but differing and well-established technology was supportive at all of our DNA hypermethylation findings.
TwinsUK had available 450k array on 587 individuals, and this platform includes 143 probes, covering 103 tRNAs.
All the 3 top tRNAs in the MeDIP-seq results were covered by this data set, and 7 of the 16 study-wide significant set.
9 tRNAs show significant (p &lt; <span class="math inline">\(4.58\times10^{-4}\)</span>) increases in DNA methylation with age in models corrected for blood cell counts including all 3 of the 3 tRNAs identified in the MeDIP-seq as genome-wide significant and 5 of the 7 study-wide significant set present on the array (Figure ).
Although it should be noted that 56 of these 143 probes are within the non-robust set of Zhou et al. <span class="citation">(Zhou, Laird, and Shen <a href="#ref-Zhou2017" role="doc-biblioref">2016</a>)</span>, including 1 of the genome-wide, and 1 of the study-wide results (covering tRNA-Ile-AAT-4-1 &amp; tRNA-Val-AAC-4-1), respectively.</p>
<div class="figure" style="text-align: center"><span id="fig:bloodEpitwins450kCounts"></span>
<img src="figs/blood_epitwins450k_Counts_sws_bl_zhou.png" alt="Volcano-like plot. tRNAs are labelled if they are significant here or were in the MeDIP-seq data (Red). Model slope: the model coefficient for the methylation values. Unfilled circles indicate those probes in the general mask generated by Zhou et al. (Zhou, Laird, and Shen 2016). Significance threshold: \(0.05/103 \approx 4.58\times10^{-4}\) (the number of tRNA genes examined)." width="100%" />
<p class="caption">
Figure 4.18: Volcano-like plot. tRNAs are labelled if they are significant here or were in the MeDIP-seq data (Red). Model slope: the model coefficient for the methylation values. Unfilled circles indicate those probes in the general mask generated by Zhou et al. <span class="citation">(Zhou, Laird, and Shen <a href="#ref-Zhou2017" role="doc-biblioref">2016</a>)</span>. Significance threshold: <span class="math inline">\(0.05/103 \approx 4.58\times10^{-4}\)</span> (the number of tRNA genes examined).
</p>
</div>

</div>
<div id="ageing-related-trna-loci-show-increased-enhancer-related-chromatin-signatures" class="section level4">
<h4><span class="header-section-number">4.4.2.4</span> Ageing-Related tRNA Loci show increased Enhancer-Related Chromatin Signatures</h4>
<p>We further explored the activity of the tRNAome in public Chromatin segmentation data in blood (Epilogos Blood &amp; T-cells set) <span class="citation">(Meuleman <a href="#ref-Meuleman2019" role="doc-biblioref">2019</a>)</span>.
This shows proportionally more Enhancer-related (Enh, EnhBiv &amp; EnhG) chromatin states at tRNA genes hypermethylating with age than the stronger Promoter-related (TSS) in other tRNAs. (Figure ).
Whereas these characteristics are less frequently predominant in the rest of the tRNAs (Figure ).
Age-hypermethylating tRNA are enriched for enhancer chromatin states compared to the rest of the tRNAome (Fisher’s Exact test p = 0.01).</p>
<div class="figure" style="text-align: center"><span id="fig:combinedEpilogosPlot"></span>
<img src="figs/maxScoreBytRNAtRNAge6SplitPlot_BL.png" alt="Chromatin segmentation data from the Epilogos (Meuleman 2019) ‘Blood &amp; T-cell’ 15 State model (tRNA genes +/- 200bp). Frequency with which a model state was the predominant state at a given tRNA. Proportions of predominant tRNA state for the 14 study-wide significant age-hypermethylating tRNAs covered compared to other 371 available tRNAs." width="90%" />
<p class="caption">
Figure 4.19: Chromatin segmentation data from the Epilogos <span class="citation">(Meuleman <a href="#ref-Meuleman2019" role="doc-biblioref">2019</a>)</span> ‘Blood &amp; T-cell’ 15 State model (tRNA genes +/- 200bp). Frequency with which a model state was the predominant state at a given tRNA. Proportions of predominant tRNA state for the 14 study-wide significant age-hypermethylating tRNAs covered compared to other 371 available tRNAs.
</p>
</div>

</div>
<div id="age-hypermethylating-trnas-are-more-methylated-in-lymphoid-than-myeloid-cells" class="section level4">
<h4><span class="header-section-number">4.4.2.5</span> Age Hypermethylating tRNAs are more methylated in Lymphoid than Myeloid cells</h4>
<p>Three tRNA genes remained genome-wide significant and 16 study-wide significant following correction for major cell-type fraction.
This is suggestive of either cell-type independent change or, presumably less likely, a very large effect in a minor cell-type fraction.
tRNAs have exhibited tissue-specific expression <span class="citation">(Bianca M. Schmitt et al. <a href="#ref-Schmitt2014b" role="doc-biblioref">2014</a>; Dittmar, Goodenbour, and Pan <a href="#ref-Dittmar2006" role="doc-biblioref">2006</a>; Sagi et al. <a href="#ref-Sagi2016" role="doc-biblioref">2016</a>)</span> and blood cell-type populations change with age.
Specifically, there is shift to favour the production of cells in the myeloid lineage <span class="citation">(Geiger, Haan, and Florian <a href="#ref-Geiger2013" role="doc-biblioref">2013</a>)</span>.
These points lead us to examine tRNA gene DNAm in sorted cell populations.
We used a publically available 450k array dataset <span class="citation">(Reinius et al. <a href="#ref-Reinius2012" role="doc-biblioref">2012</a>)</span>) that has been used in the construction of cell-type specific DNAm references for cell-type fraction prediction using the Houseman algorithm <span class="citation">(Houseman et al. <a href="#ref-Houseman2012" role="doc-biblioref">2012</a>)</span> (see <a href="tRNApaper.html#sortedbloodmethods">Methods</a>).
This consists of data from 6 individuals (aged <span class="math inline">\(38 \pm 13.6\)</span>/yrs) from seven isolated cell populations (CD4+ T cells, CD8+ T cells, CD56+ NK cells, CD19+ B cells, CD14+ monocytes, neutrophils, and eosinophils).
We found that tRNA gene DNAm could separate myeloid from lymphoid lineages (Figures  &amp; ).</p>
<p>Of the eight study-wide significant tRNAs with array coverage, we identified that collectively these eight are significantly more methylated in the lymphoid than the myeloid lineage (1.1% difference, Wilcoxon rank sum test p = <span class="math inline">\(1.50\times10^{-6}\)</span> 95% CI 0.7%-<span class="math inline">\(\infty\)</span>).
Thus, any age related increases in myeloid cell proportion would be expected to dampen rather than exaggerate the age-related hypermethylation signal that we observed.
In addition tRNA-Ile-AAT-4-1 and tRNA-Ser-AGA-2-6 have the highest variance in their DNAm of all 129 tRNAs covered in this dataset.
This could represent ageing-related changes as these samples range across almost 3 decades.
Another possibly may be that these loci as well as hypermethylating are also increasing their variability with age in a similar fashion to those identified by Slieker <em>et al.</em> <span class="citation">(Slieker et al. <a href="#ref-Slieker2016" role="doc-biblioref">2016</a>)</span>.
In that study they identified that those loci accruing methylomic variability were associated with fundamental ageing mechanisms.</p>
<div class="figure" style="text-align: center"><span id="fig:meanMethbytRNAMatLowBB6"></span>
<img src="figs/meanMethbytRNAMatLowBB6_bl.png" alt="Heatmap (Gu, Eils, and Schlesner 2016) of mean methylation of probes covering each tRNA in 7 cell-type fractions from 6 Male individuals. Data from GSE35069 (Reinius et al. 2012). Of the 16 study-wide significant hypermethylating tRNAs, 8 are covered by this dataset." width="100%" />
<p class="caption">
Figure 4.20: Heatmap <span class="citation">(Gu, Eils, and Schlesner <a href="#ref-Gu2016" role="doc-biblioref">2016</a>)</span> of mean methylation of probes covering each tRNA in 7 cell-type fractions from 6 Male individuals. Data from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069">GSE35069</a> <span class="citation">(Reinius et al. <a href="#ref-Reinius2012" role="doc-biblioref">2012</a>)</span>. Of the 16 study-wide significant hypermethylating tRNAs, 8 are covered by this dataset.
</p>
</div>

</div>
</div>
<div id="trna-gene-dna-methylation-in-other-tissues" class="section level3">
<h3><span class="header-section-number">4.4.3</span> tRNA Gene DNA Methylation in Other Tissues</h3>
<p>Some tRNA gene expression has been shown to be highly tissue specific <span class="citation">(Bianca M. Schmitt et al. <a href="#ref-Schmitt2014b" role="doc-biblioref">2014</a>; Dittmar, Goodenbour, and Pan <a href="#ref-Dittmar2006" role="doc-biblioref">2006</a>; Sagi et al. <a href="#ref-Sagi2016" role="doc-biblioref">2016</a>)</span>.
It follows that our observations of changes in DNAm with age in blood might be specific to that tissue.
We used a mix of 450k and 27k array data from ‘solid tissue normal’ samples made available by TCGA (The Cancer Genome Atlas) and data from foetal tissue <span class="citation">(Yang et al. <a href="#ref-Yang2016" role="doc-biblioref">2016</a>; Nazor et al. <a href="#ref-Nazor2012" role="doc-biblioref">2012</a>)</span> downloaded from GEO (see <a href="tRNApaper.html#TCGAmethods">Methods</a>).
The samples from TCGA range in age from 15-90 (n = 733).
Only 43 tRNA genes had adequate data to compare across tissues in this dataset and 115 in the foetal tissue data.</p>
<div id="trna-genes-also-hypermethylate-with-age-in-solid-tissue" class="section level4">
<h4><span class="header-section-number">4.4.3.1</span> tRNA Genes also Hypermethylate with Age in Solid Tissue</h4>
<p>Only 2 of the 3 tRNA genes we identified as genome-wide significant and a further 1 of the study-wide significant tRNA genes are present in the set of 45 tRNA genes in the TCGA data, thus limiting our ability to draw conclusions about the tissue specificity of our results.
Solid tissue samples have a strong preponderance for low levels of methylation consistent with the active transcription of many tRNA genes and show slight increasing methylation with age but age accounts for very little of the variance (linear regression slope estimate = <span class="math inline">\(1.52\)</span>; <span class="math inline">\(R^2= 0.0002\)</span>; p-value <span class="math inline">\(1.34\times10^{-3}\)</span> (Figure d).
In a pan-tissue analysis we found that 10 tRNA genes showed changes in DNAm with age, 9 of which were hypermethylation (p-value <span class="math inline">\(&lt;1.1\times10^{-3}\)</span>).
One of these tRNA genes, tRNA-Ser-TGA-2-1 was also present in study-wide significant set of tRNA genes.
Figures  &amp;  illustrate minimal tissue specific differences.
Interestingly, however, tRNA-iMet-CAT-1-4 and tRNA-Ser-AGA-2-6 appeared more variable in methylation state than many other tRNAs in the TCGA normal tissue samples (Figure ) and indeed have the highest variance in DNA methylation across tissues (Figure c).</p>
</div>
</div>
<div id="expression-of-trnas-in-blood-with-age" class="section level3">
<h3><span class="header-section-number">4.4.4</span> Expression of tRNAs in Blood with Age</h3>
<p>Having observed specific tRNA gene isodecoders hypermethylating with age we explored the expression of tsRNA in blood cell-types.
We devised a bioinformatic approach to attempt to assay tRNA transcription in order to use standard publicly available small RNA-seq datasets.
We created customised MINTmap <span class="citation">(Loher, Telonis, and Rigoutsos <a href="#ref-Loher2017" role="doc-biblioref">2017</a>)</span> reference designed to include only fragments which unambiguously map to a single tRNA gene locus and which overlap the 5’ or 3’ end of the genomic tRNA sequence by at least one base with no mismatches.
This reference is intended to capture pre-tRNAs prior to processing and CCA addition operating under the assumption that the levels of pre-tRNAs will be informative about the amount of transcription taking place at the tRNA loci (see <a href="tRNApaper.html#mintmapmethods">Methods</a>).
Our custom MINTmap reference build yielded 383 fragments mapping to 92 distinct tRNA loci in this data.
The lack of coverage of age hypermethylating tRNAs by uniquely attributable RNA-seq reads prevented us from drawing any strong conclusions about the relationship between DNAm changes and changes in tRNA transcription.</p>
<p>Using the original MINTmap reference optimised to detect tRNA fragments derived from mature tRNAs there were 5384 unique fragments derived from as many as 417 tRNA loci.
However, the mapping between fragments and loci in this reference is many to many, with each tRNA gene able to give rise to many fragments and each fragment attributable to at least 1 and usually many tRNA genes.
We limited our examination of these fragments to those with a length of greater than or equal to 40nt to capture reads more likely to be derived from mature tRNAs rather than tRFs or tRNA halves (Figure ).
We identified 48 tsRNAs with nominally significant expression changes (p &lt; 0.05), 8 increased and 40 decreased in abundance with age.
For example 5 of 6 fragments showing significant age-related expression changes derived from the Gln-CTG family of tRNAs are decreasing with age (Figure ).
This is suggestive that expression of some tRNA genes may decline with age but this possibility is in need of additional tRNA expression data before it can be asserted with confidence.</p>
<div class="figure" style="text-align: center"><span id="fig:mintmapres"></span>
<img src="figs/nonSigIsoMapbbswsPlotsGln-CTG.png" alt="tRNA fragments derived from the Gln-CTG family of tRNAs, selected as tRNA-Gln-CTG-7-1 is one of the 16 study-wide significant age-hypermethylating tRNA genes. Pane titles contain the MINTbase Plates, unique identifiers of the tRNA fragments (Pliatsika et al. 2018)." width="100%" />
<p class="caption">
Figure 4.21: tRNA fragments derived from the Gln-CTG family of tRNAs, selected as tRNA-Gln-CTG-7-1 is one of the 16 study-wide significant age-hypermethylating tRNA genes. Pane titles contain the MINTbase Plates, unique identifiers of the tRNA fragments <span class="citation">(Pliatsika et al. <a href="#ref-Pliatsika2018" role="doc-biblioref">2018</a>)</span>.
</p>
</div>

</div>
<div id="mice-also-show-age-related-trna-gene-dna-hypermethylation" class="section level3">
<h3><span class="header-section-number">4.4.5</span> Mice also show age-related tRNA gene DNA hypermethylation</h3>
<p>We examined the DNA methylation of the mouse tRNAome in using data from a reduced representation bisulfite sequencing (RRBS) experiment performed by Petkovich et al. <span class="citation">(Petkovich et al. <a href="#ref-Petkovich2017" role="doc-biblioref">2017</a>)</span>.
These data from 152 mice covered 51 tRNA genes and 385 CpGs after QC (see <a href="tRNApaper.html#mousemethods">Methods</a>), representing ~11% of the mouse tRNAome.
The mice ranged in age from 0.67-35 months.</p>
<p>Three of the 51 tRNAs showed Bonferroni significant DNA methylation changes with age (p-value &lt; <span class="math inline">\(1.08 \times 10^{-4}\)</span>) and all were in the hypermethylation direction.
These three are tRNA-Asp-GTC-1-12, tRNA-Ile-AAT-1-4, tRNA-Glu-TTC-1-3 (Figure ).</p>
<div class="figure" style="text-align: center"><span id="fig:mouse"></span>
<img src="figs/CpGPlotComb.png" alt="DNA methylation of CpGs in 3 tRNA which significantly hypermethylate with age in mice. 6 CpGs reach bonferroni significance and 7 show nominally significant increases." width="100%" />
<p class="caption">
Figure 4.22: DNA methylation of CpGs in 3 tRNA which significantly hypermethylate with age in mice. 6 CpGs reach bonferroni significance and 7 show nominally significant increases.
</p>
</div>

<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion-2" class="section level2">
<h2><span class="header-section-number">4.5</span> Discussion</h2>
<p>Our work has identified a previously unknown enrichment for age-related epigenetic changes within the tRNA genes of the human genome.
This observation was strongly directional with increasing DNA methylation with age <span class="citation">(Ehrlich <a href="#ref-Ehrlich2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>The MeDIP-seq dataset employed brought advantages in exploring this undefined terrain of the tRNAome.
Firstly, being genome-wide it provides much increased access, as these regions are poorly covered by current arrays.
Secondly, being a fragment-based regional assessment of DNA methylation, the individual but highly similar small tRNA genes can be surrounded by unique sequence.</p>
<p>We determined by genome-wide permutation that this strong hypermethylation signal was specific to the tRNAome, and not merely driven by the underlying CpG density of these loci.
A targeted BiS-seq experiment validated the defined nature of the tRNA change in an independent dataset, with a successful pooling approach, which may also be useful for other ageing-related targeted DNA methylome evaluations.
Additionally, we gained support for our results from limited DNA methylation array data.</p>
<p>We subsequently explored further what was driving this age-related phenomenon and its possible biological implications.
As this result was observed in peripheral blood, we were well aware that we were examining DNA derived from a heterogeneous cell type population <span class="citation">(Lappalainen and Greally <a href="#ref-Lappalainen2017" role="doc-biblioref">2017</a>)</span>.
Moreover, that there are well known age-related proportional changes in peripheral blood cell composition <span class="citation">(Geiger, Haan, and Florian <a href="#ref-Geiger2013" role="doc-biblioref">2013</a>)</span>.
The TwinsUK MeDIP-seq and 450k array DNA methylation data included measured haematological values.
Therefore, we adjusted for major cell type effects, such as a myeloid skew, and distinct tRNAs were still significant.
Although, a caveat to our study is that this can not exclude changes in minor specific sub-cell fractions types.
However, that these age-related effects were strong enough to be observed in both a regional MEDIP-seq assessment and a pooled sequencing approach, implies that they not extremely subtle.
We examined age-related tRNA gene DNA methylation changes in the limited subset of mouse tRNA genes covered in publicly available RRBS data (~13%) and were able to identify tRNAs exhibiting DNA hypermethylation with age in this set.
This suggests that age-related tRNA gene hypermethylation may not be unique to humans, but at least observed across mammals.</p>
<p>Due to the high number of hypermethylating tRNA prior to cell-type correction, we were also curious whether the epigenetic state of this small tRNAome fraction of genome could capture and in fact be a defined fingerprint of cell type.
We found that tRNA gene DNA methylation could separate myeloid from lymphoid lineages.
There also was some suggestion of more fine-grained blood cell-type signatures in tRNA DNAm, such as the separation of CD19+ B cells from CD4/8+ T cells.
Ageing is also known to lead to an increase in senescent cells (<em>e.g.</em> CD8+ CD28- cells).
Whether these epigenetic changes in the tRNAome uniquely represent these cell-types will require technical advances to enable future single cell DNA methylome analysis to accurately assess these regions.
If further supported, the epigenetic state of these loci may aid the taxonomy of cell-type definition.</p>
<p>This signal within the tRNA families was observed to occur at specific Isodecoders.
After correcting for major cell types, we identified 2 tRNA genes tRNA-iMet-CAT-1-4 and tRNA-Ser-AGA-2-6 which had the most consistent hypermethylation across 3 different assays.
Isodecoders expand in number with organismal complexity and the high prevalence in mammals has been suggested due to their additional regulatory functionality <span class="citation">(Goodenbour and Pan <a href="#ref-Goodenbour2006" role="doc-biblioref">2006</a>; Keam et al. <a href="#ref-Keam2014" role="doc-biblioref">2014</a>)</span>.
They also have distinct translational efficiency <span class="citation">(Geslain and Pan <a href="#ref-Geslain2010" role="doc-biblioref">2010</a>)</span>, which can also have consequences in human disease <span class="citation">(Kirchner et al. <a href="#ref-Kirchner2017" role="doc-biblioref">2017</a>)</span>.
Furthermore, there is great complexity to the fragmentation of tRNA <span class="citation">(Schimmel <a href="#ref-Schimmel2017" role="doc-biblioref">2018</a>)</span>, with physiological processes such as stress shown to induce fragment production <span class="citation">(Li et al. <a href="#ref-Li2019" role="doc-biblioref">2019</a>)</span>.
These resultant tsRNAs can feedback on protein synthesis by regulating ribosome biogenesis <span class="citation">(Kim et al. <a href="#ref-Kim2017" role="doc-biblioref">2017</a>)</span> and others have diverse regulatory functions such as targeting transposable element transcripts <span class="citation">(Martinez, Choudury, and Slotkin <a href="#ref-Martinez2017" role="doc-biblioref">2017</a>)</span>.
They are also observed to circulate in the blood in a cell-free fashion, and fragment levels can be modulated by ageing and calorie restriction <span class="citation">(Dhahbi et al. <a href="#ref-Dhahbi2013" role="doc-biblioref">2013</a>)</span>.
The isodecoder specific nature of our findings frame a possible hypothesis for regulatory change with age and future work will be required to unravel this potential.</p>
<p>Whilst, the expression of the tRNA genes has long been simplified as ‘constitutive’, some observations have indicated that many tRNA genes are expressed in a tissue-specific fashion in diverse organisms <span class="citation">(Dittmar, Goodenbour, and Pan <a href="#ref-Dittmar2006" role="doc-biblioref">2006</a>; Sagi et al. <a href="#ref-Sagi2016" role="doc-biblioref">2016</a>)</span>.
Although others have found the majority of isodecoders are transcribed in different cell types <span class="citation">(Parisien, Wang, and Pan <a href="#ref-Parisien2013" role="doc-biblioref">2013</a>)</span>.
Several transcription factors acting via TFIIIB <span class="citation">(Gomez-Roman et al. <a href="#ref-Gomez-Roman2003" role="doc-biblioref">2003</a>)</span> have a negative (the tumour suppressors p53 <span class="citation">(Crighton <a href="#ref-Crighton2003" role="doc-biblioref">2003</a>)</span> and Rb <span class="citation">(Sutcliffe et al. <a href="#ref-Sutcliffe2000" role="doc-biblioref">2000</a>)</span>) or positive (the proto-oncogene c-Myc) influence <span class="citation">(Gomez-Roman et al. <a href="#ref-Gomez-Roman2003" role="doc-biblioref">2003</a>)</span>.
Regulatory sequence in the flanking or the internal regions of tRNA genes do not explain tRNA expression variation <span class="citation">(Bianca M Schmitt et al. <a href="#ref-Schmitt2014" role="doc-biblioref">2014</a>)</span>.
Whilst DNAm is able to repress the expression of tRNA genes <span class="citation">(Besser et al. <a href="#ref-Besser1990" role="doc-biblioref">1990</a>)</span>, the broader chromatin environment also affects tRNA transcription.
Due to the co-ordinated nature of epigenomic modifications, it may also be revealing to evaluate ageing-related histone modification in these tRNA loci.</p>
<p>Changes in the epigenetic state of specific tRNA could be modulating transcription efficiency or even codon availability in the ageing cell.
tRNA gene dosage is quite closely matched to amino acid usage frequency in the human exome.
However, the transcriptome codon usage frequency and tRNA gene expression have been claimed to vary with the replicative state of cells, separating differentiated from replicating cells <span class="citation">(Gingold et al. <a href="#ref-Gingold2014" role="doc-biblioref">2014</a>)</span>.
Others have argued that these differences are substantially explained by variation in GC content <span class="citation">(Rudolph et al. <a href="#ref-Rudolph2016" role="doc-biblioref">2016</a>)</span> and that codon usage frequencies are observed to be mostly invariant in the transcriptomes of a wide range of tissues, as well as across developmental time <span class="citation">(Bianca M Schmitt et al. <a href="#ref-Schmitt2014" role="doc-biblioref">2014</a>)</span>.
Although, experimental stress-related states have revealed changes with an over-representation of codons that are translated by rare tRNAs <span class="citation">(Gingold, Dahan, and Pilpel <a href="#ref-Gingold2012" role="doc-biblioref">2012</a>)</span>.</p>
<p>tRNA sequences themselves are under strong structural (both secondary and tertiary) <span class="citation">(Goodenbour and Pan <a href="#ref-Goodenbour2006" role="doc-biblioref">2006</a>)</span> as well as functional constraint, which leads to an order of magnitude reduction in variation compared the background genomic mutation rate <span class="citation">(Parisien, Wang, and Pan <a href="#ref-Parisien2013" role="doc-biblioref">2013</a>)</span>.
However, polymorphic tRNA could be another potential caveat to our work.
Although, there is no significant population variation in, for example, tRNA iMet sequences in 1,000 Genomes data.
Indeed, there are only 11 new isodecoder sequences with high confidence (tRNAscan scores <span class="math inline">\(\ge50\)</span>) at &gt;1% population frequency <span class="citation">(Parisien, Wang, and Pan <a href="#ref-Parisien2013" role="doc-biblioref">2013</a>)</span>.
There is also some evidence for tRNA copy number variation at specific loci, although this remains under-characterised <span class="citation">(Iben and Maraia <a href="#ref-Iben2015" role="doc-biblioref">2014</a>; Darrow and Chadwick <a href="#ref-Darrow2014" role="doc-biblioref">2014</a>)</span>.
Another potential cause we considered was whether age-related somatic copy number increases could be occurring in these loci.
Population or somatic copy number expansions could lead to increased methylated reads in MeDIP-seq without any epigenetic state change.
However, this would not be consistent with the targeted and array BiS conversion methodologies, where the proportion of methylated to unmethylated reads would still be constant.</p>
<p>It is worth noting the parallels with known cancer and ageing epigenetic changes, and that tRNAs are also dysregulated in cancer <span class="citation">(Huang et al. <a href="#ref-Huang2018" role="doc-biblioref">2018</a>)</span>, with proposed utility as prognostic markers <span class="citation">(Krishnan et al. <a href="#ref-Krishnan2016" role="doc-biblioref">2016</a>)</span>.
Furthermore, the early replicating state of tRNA loci, potentially associated with high expression <span class="citation">(Müller and Nieduszynski <a href="#ref-Muller2017" role="doc-biblioref">2017</a>)</span>, may make them prone to hypermethylate, as is observed in early replicating loci in both cancer <span class="citation">(Du et al. <a href="#ref-Du2019" role="doc-biblioref">2019</a>)</span> and senescent cells <span class="citation">(Cruickshanks et al. <a href="#ref-Cruickshanks2013" role="doc-biblioref">2013</a>)</span>.
Interestingly, tRNA gene loci may also play a role in local as well as large scale genome organisation <span class="citation">(Hamdani2019; Van Bortle, Phanstiel, and Snyder <a href="#ref-VanBortle2017" role="doc-biblioref">2017</a>)</span>.
tRNA gene clusters act as insulators <span class="citation">(Raab et al. <a href="#ref-Raab2011" role="doc-biblioref">2012</a>)</span> and have extensive long-range chromatin interactions with other tRNA gene loci <span class="citation">(Van Bortle, Phanstiel, and Snyder <a href="#ref-VanBortle2017" role="doc-biblioref">2017</a>)</span>.
The coordinated transcription of tRNAs at subnuclear foci and the B-box sequence elements bound by TFIIIC and not PolIII may represent an organising principle for 3D-chromatin by providing spatial constraints <span class="citation">(Noma et al. <a href="#ref-Noma2006" role="doc-biblioref">2006</a>)</span>.
Therefore, these tRNA epigenetic changes could contribute to the structural changes that are also observed in ageing <span class="citation">(Sun, Yu, and Dang <a href="#ref-Sun2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>In conclusion, due to the unique challenges that make the tRNAome difficult to examine it has remained epigenetically under-characterised despite its critical importance for cell function.
We directly interrogated the epigenetic DNA methylation state of the functionally important tRNAome, across the age spectrum in a range of datasets as well as methodologies and identified an enrichment for age-related DNA hypermethylation in the human tRNA genes.</p>
<div style="page-break-after: always;"></div>
</div>
<div id="data-availability" class="section level2">
<h2><span class="header-section-number">4.6</span> Data availability</h2>
<p>The MeDIP-seq data supporting the results of this article are available in the EMBL-EBI European Genome-phenome Archive (EGA) under Data set Accession number EGAD00010000983 (<a href="https://www.ebi.ac.uk/ega/datasets/" class="uri">https://www.ebi.ac.uk/ega/datasets/</a>
EGAD00010000983).
The targeted BiS-sequencing data will be available on publication.</p>
</div>
<div id="code-availablity" class="section level2">
<h2><span class="header-section-number">4.7</span> Code availablity</h2>
<p>Available at <a href="https://github.com/RichardJActon/tRNA_paper_code">https://github.com/RichardJActon/tRNA_paper_code</a></p>
<div style="page-break-after: always;"></div>
</div>
<div id="acknowledgements-1" class="section level2">
<h2><span class="header-section-number">4.8</span> Acknowledgements</h2>
<p>We gratefully acknowlege the individuals from TwinsUK, Mavidos and the Hertfordshrie cohort.
TwinsUK received funding from the Wellcome Trust (Ref: 081878/Z/06/Z), European Community’s Seventh Framework Programme (FP7/2007-2013), the National Institute for Health Research (NIHR)-funded BioResource, Clinical Research Facility and Biomedical Research Centre based at Guy’s and St Thomas’ NHS Foundation Trust in partnership with King’s College London.
Further funding support for the EpiTwin project was obtained from the European Research Council (project number 250157) and BGI.
SNP Genotyping was performed by The Wellcome Trust Sanger Institute and National Eye Institute via NIH/CIDR.
The authors would like to thank Nikki Graham for her assistance with the identification and pooling of the MAVIDOS and Hertfordshire DNA samples.
The authors also acknowledge the use of the IRIDIS High Performance Computing Facility, and associated support services at the University of Southampton, in the completion of this work.
The MRC-LEU is supported by the Medical Research Council (MRC).
CGB recieved support from Diabetes UK (16/0005454).
RJA was in receipt of a MRC Doctoral fund (1820097).</p>
</div>
<div id="author-contributions" class="section level2">
<h2><span class="header-section-number">4.9</span> Author Contributions</h2>
<p>RJA designed experiments and analysed all the processed and experimental data.
CGB conceived and designed the experiments.
TDS, KW and JW conceived and provided TwinsUK MeDIP-seq data.
YX, FG and JW produced raw MeDIP-seq data with WY and JB processing and quality controlling these data.
WY contributed an analysis concept.
CC, NH, ED, and KL provided MAVIDOS and Hertfordshire sample data.
EB, EW, and CAM performed the targeted BiS sequencing experiment.
PGH contributed additional data and discussion of results.
RJA and CGB wrote the paper.
All authors reviewed and approved the final manuscript.</p>
</div>
<div id="references" class="section level2">
<h2><span class="header-section-number">4.10</span> References</h2>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Acton2020">
<p>Acton, Richard J, Emma Bourne, Jordana Bell, Karen Lillycrop, Jun Wang, Elaine Dennison, Nicholas Harvey, Tim D Spector, Cyrus Cooper, and Christopher G Bell. 2019. “The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation.” <em>bioRxiv</em>, 1–48. <a href="https://doi.org/10.1101/870352">https://doi.org/10.1101/870352</a>.</p>
</div>
<div id="ref-Amemiya2019">
<p>Amemiya, Haley M., Anshul Kundaje, and Alan P. Boyle. 2019. “The ENCODE Blacklist: Identification of Problematic Regions of the Genome.” <em>Scientific Reports</em> 9 (1): 9354. <a href="https://doi.org/10.1038/s41598-019-45839-z">https://doi.org/10.1038/s41598-019-45839-z</a>.</p>
</div>
<div id="ref-Andrews2010">
<p>Andrews, S. 2010. “FastQC.” Cambridge: Babraham Bioinformatics. <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</p>
</div>
<div id="ref-Bell2017a">
<p>Bell, Christopher G., Fei Gao, Wei Yuan, Leonie Roos, Richard J. Acton, Yudong Xia, Jordana Bell, et al. 2018. “Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci.” <em>Nature Communications</em> 9 (1): 8. <a href="https://doi.org/10.1038/s41467-017-01586-1">https://doi.org/10.1038/s41467-017-01586-1</a>.</p>
</div>
<div id="ref-Bell2012a">
<p>Bell, Christopher G., Gareth A. Wilson, Lee M. Butcher, Christian Roos, Lutz Walter, and Stephan Beck. 2012. “Human-specific CpG "beacons" identify loci associated with human-specific traits and disease.” <em>Epigenetics</em> 7 (10): 1188–99. <a href="https://doi.org/10.4161/epi.22127">https://doi.org/10.4161/epi.22127</a>.</p>
</div>
<div id="ref-Bell2016">
<p>Bell, Christopher G., Yudong Xia, Wei Yuan, Fei Gao, Kirsten Ward, Leonie Roos, Massimo Mangino, et al. 2016. “Novel regional age-associated DNA methylation changes within human common disease-associated loci.” <em>Genome Biology</em> 17 (1): 193. <a href="https://doi.org/10.1186/s13059-016-1051-8">https://doi.org/10.1186/s13059-016-1051-8</a>.</p>
</div>
<div id="ref-Besser1990">
<p>Besser, Daniel, Frank Götz, Kai Schulze-Forster, Herbert Wagner, Hans Kröger, and Dietrich Simon. 1990. “DNA methylation inhibits transcription by RNA polymerase III of a tRNA gene, but not of a 5S rRNA gene.” <em>FEBS Letters</em> 269 (2): 358–62. <a href="https://doi.org/10.1016/0014-5793(90)81193-R">https://doi.org/10.1016/0014-5793(90)81193-R</a>.</p>
</div>
<div id="ref-Booth2016">
<p>Booth, Lauren N., and Anne Brunet. 2016. “The Aging Epigenome.” <em>Molecular Cell</em> 62 (5): 728–44. <a href="https://doi.org/10.1016/j.molcel.2016.05.013">https://doi.org/10.1016/j.molcel.2016.05.013</a>.</p>
</div>
<div id="ref-Campisi2019">
<p>Campisi, Judith, Pankaj Kapahi, Gordon J. Lithgow, Simon Melov, John C. Newman, and Eric Verdin. 2019. “From discoveries in ageing research to therapeutics for healthy ageing.” <em>Nature</em> 571 (7764): 183–92. <a href="https://doi.org/10.1038/s41586-019-1365-2">https://doi.org/10.1038/s41586-019-1365-2</a>.</p>
</div>
<div id="ref-Canella2010">
<p>Canella, Donatella, Viviane Praz, Jaime H. Reina, Pascal Cousin, and Nouria Hernandez. 2010. “Defining the RNA polymerase III transcriptome: Genome-wide localization of the RNA polymerase III transcription machinery in human cells.” <em>Genome Research</em> 20 (6): 710–21. <a href="https://doi.org/10.1101/gr.101337.109">https://doi.org/10.1101/gr.101337.109</a>.</p>
</div>
<div id="ref-Chan2009">
<p>Chan, Patricia P., and Todd M. Lowe. 2009. “GtRNAdb: a database of transfer RNA genes detected in genomic sequence.” <em>Nucleic Acids Research</em> 37 (Database issue): D93–7. <a href="https://doi.org/10.1093/nar/gkn787">https://doi.org/10.1093/nar/gkn787</a>.</p>
</div>
<div id="ref-Christensen2009">
<p>Christensen, Brock C., E. Andres Houseman, Carmen J. Marsit, Shichun Zheng, Margaret R. Wrensch, Joseph L. Wiemels, Heather H. Nelson, et al. 2009. “Aging and Environmental Exposures Alter Tissue-Specific DNA Methylation Dependent upon CpG Island Context.” Edited by Dirk Schübeler. <em>PLoS Genetics</em> 5 (8): e1000602. <a href="https://doi.org/10.1371/journal.pgen.1000602">https://doi.org/10.1371/journal.pgen.1000602</a>.</p>
</div>
<div id="ref-Chuong2017">
<p>Chuong, Edward B., Nels C. Elde, and Cédric Feschotte. 2017. “Regulatory activities of transposable elements: from conflicts to benefits.” <em>Nature Reviews Genetics</em> 18 (2): 71–86. <a href="https://doi.org/10.1038/nrg.2016.139">https://doi.org/10.1038/nrg.2016.139</a>.</p>
</div>
<div id="ref-Crighton2003">
<p>Crighton, Diane. 2003. “p53 represses RNA polymerase III transcription by targeting TBP and inhibiting promoter occupancy by TFIIIB.” <em>The EMBO Journal</em> 22 (11): 2810–20. <a href="https://doi.org/10.1093/emboj/cdg265">https://doi.org/10.1093/emboj/cdg265</a>.</p>
</div>
<div id="ref-Cruickshanks2013">
<p>Cruickshanks, Hazel A, Tony McBryan, David M Nelson, Nathan D. VanderKraats, Parisha P Shah, John van Tuyn, Taranjit Singh Rai, et al. 2013. “Senescent cells harbour features of the cancer epigenome.” <em>Nature Cell Biology</em> 15 (12): 1495–1506. <a href="https://doi.org/10.1038/ncb2879">https://doi.org/10.1038/ncb2879</a>.</p>
</div>
<div id="ref-Darrow2014">
<p>Darrow, Emily M., and Brian P. Chadwick. 2014. “A novel tRNA variable number tandem repeat at human chromosome 1q23.3 is implicated as a boundary element based on conservation of a CTCF motif in mouse.” <em>Nucleic Acids Research</em> 42 (10): 6421–35. <a href="https://doi.org/10.1093/nar/gku280">https://doi.org/10.1093/nar/gku280</a>.</p>
</div>
<div id="ref-Derrien2012">
<p>Derrien, Thomas, Jordi Estellé, Santiago Marco Sola, David G. Knowles, Emanuele Raineri, Roderic Guigó, and Paolo Ribeca. 2012. “Fast Computation and Applications of Genome Mappability.” Edited by Christos A. Ouzounis. <em>PLoS ONE</em> 7 (1): e30377. <a href="https://doi.org/10.1371/journal.pone.0030377">https://doi.org/10.1371/journal.pone.0030377</a>.</p>
</div>
<div id="ref-Dhahbi2013">
<p>Dhahbi, Joseph M, Stephen R Spindler, Hani Atamna, Amy Yamakawa, Dario Boffelli, Patricia Mote, and David IK Martin. 2013. “5’ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction.” <em>BMC Genomics</em> 14 (1): 298. <a href="https://doi.org/10.1186/1471-2164-14-298">https://doi.org/10.1186/1471-2164-14-298</a>.</p>
</div>
<div id="ref-Dittmar2006">
<p>Dittmar, Kimberly A., Jeffrey M. Goodenbour, and Tao Pan. 2006. “Tissue-specific differences in human transfer RNA expression.” <em>PLoS Genetics</em> 2 (12): e221. <a href="https://doi.org/10.1371/journal.pgen.0020221">https://doi.org/10.1371/journal.pgen.0020221</a>.</p>
</div>
<div id="ref-Down2009">
<p>Down, Thomas A, Vardhman K Rakyan, Daniel J Turner, Paul Flicek, Heng Li, Eugene Kulesha, Stefan Gräf, et al. 2008. “A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis.” <em>Nature Biotechnology</em> 26 (7): 779–85. <a href="https://doi.org/10.1038/nbt1414">https://doi.org/10.1038/nbt1414</a>.</p>
</div>
<div id="ref-Du2019">
<p>Du, Qian, Saul A. Bert, Nicola J. Armstrong, C. Elizabeth Caldon, Jenny Z. Song, Shalima S. Nair, Cathryn M. Gould, et al. 2019. “Replication timing and epigenome remodelling are associated with the nature of chromosomal rearrangements in cancer.” <em>Nature Communications</em> 10 (1): 416. <a href="https://doi.org/10.1038/s41467-019-08302-1">https://doi.org/10.1038/s41467-019-08302-1</a>.</p>
</div>
<div id="ref-Ehrlich2019">
<p>Ehrlich, Melanie. 2019. “DNA hypermethylation in disease: mechanisms and clinical relevance.” <em>Epigenetics</em> 14 (12): 1141–63. <a href="https://doi.org/10.1080/15592294.2019.1638701">https://doi.org/10.1080/15592294.2019.1638701</a>.</p>
</div>
<div id="ref-Eigen1989">
<p>Eigen, Manfred, B. Lindemann, Manfred Tietze, R Winkler-Oswatitsch, Andreas Dress, and A von Haeseler. 1989. “How old is the genetic code? Statistical geometry of tRNA provides an answer.” <em>Science</em> 244 (4905): 673–79. <a href="https://doi.org/10.1126/science.2497522">https://doi.org/10.1126/science.2497522</a>.</p>
</div>
<div id="ref-Ernst2011">
<p>Ernst, Jason, Pouya Kheradpour, Tarjei S Mikkelsen, Noam Shoresh, Lucas D Ward, Charles B Epstein, Xiaolan Zhang, et al. 2011. “Mapping and analysis of chromatin state dynamics in nine human cell types.” <em>Nature</em> 473 (7345): 43–49. <a href="https://doi.org/10.1038/nature09906">https://doi.org/10.1038/nature09906</a>.</p>
</div>
<div id="ref-Ewels2016">
<p>Ewels, Philip, Måns Magnusson, Sverker Lundin, and Max Käller. 2016. “MultiQC: summarize analysis results for multiple tools and samples in a single report.” <em>Bioinformatics (Oxford, England)</em> 32 (19): 3047–8. <a href="https://doi.org/10.1093/bioinformatics/btw354">https://doi.org/10.1093/bioinformatics/btw354</a>.</p>
</div>
<div id="ref-Field2018">
<p>Field, Adam E., Neil A. Robertson, Tina Wang, Aaron Havas, Trey Ideker, and Peter D. Adams. 2018. “DNA Methylation Clocks in Aging: Categories, Causes, and Consequences.” <em>Molecular Cell</em> 71 (6): 882–95. <a href="https://doi.org/10.1016/j.molcel.2018.08.008">https://doi.org/10.1016/j.molcel.2018.08.008</a>.</p>
</div>
<div id="ref-Filer2017">
<p>Filer, Danny, Maximillian A. Thompson, Vakil Takhaveev, Adam J. Dobson, Ilektra Kotronaki, James W. M. Green, Matthias Heinemann, Jennifer M. A. Tullet, and Nazif Alic. 2017. “RNA polymerase III limits longevity downstream of TORC1.” <em>Nature</em> 552 (7684): 263–67. <a href="https://doi.org/10.1038/nature25007">https://doi.org/10.1038/nature25007</a>.</p>
</div>
<div id="ref-Fraga2005">
<p>Fraga, Mario F, E. Ballestar, Maria F Paz, Santiago Ropero, Fernando Setien, Maria L Ballestar, D. Heine-Suner, et al. 2005. “From The Cover: Epigenetic differences arise during the lifetime of monozygotic twins.” <em>Proceedings of the National Academy of Sciences</em> 102 (30): 10604–9. <a href="https://doi.org/10.1073/pnas.0500398102">https://doi.org/10.1073/pnas.0500398102</a>.</p>
</div>
<div id="ref-Geiger2013">
<p>Geiger, Hartmut, Gerald de Haan, and M. Carolina Florian. 2013. “The ageing haematopoietic stem cell compartment.” <em>Nature Reviews Immunology</em> 13 (5): 376–89. <a href="https://doi.org/10.1038/nri3433">https://doi.org/10.1038/nri3433</a>.</p>
</div>
<div id="ref-Geslain2010">
<p>Geslain, Renaud, and Tao Pan. 2010. “Functional Analysis of Human tRNA Isodecoders.” <em>Journal of Molecular Biology</em> 396 (3): 821–31. <a href="https://doi.org/10.1016/j.jmb.2009.12.018">https://doi.org/10.1016/j.jmb.2009.12.018</a>.</p>
</div>
<div id="ref-Gingold2012">
<p>Gingold, Hila, Orna Dahan, and Yitzhak Pilpel. 2012. “Dynamic changes in translational efficiency are deduced from codon usage of the transcriptome.” <em>Nucleic Acids Research</em> 40 (20): 10053–63. <a href="https://doi.org/10.1093/nar/gks772">https://doi.org/10.1093/nar/gks772</a>.</p>
</div>
<div id="ref-Gingold2014">
<p>Gingold, Hila, Disa Tehler, Nanna R. Christoffersen, Morten M. Nielsen, Fazila Asmar, Susanne M. Kooistra, Nicolaj S. Christophersen, et al. 2014. “A Dual Program for Translation Regulation in Cellular Proliferation and Differentiation.” <em>Cell</em> 158 (6): 1281–92. <a href="https://doi.org/10.1016/j.cell.2014.08.011">https://doi.org/10.1016/j.cell.2014.08.011</a>.</p>
</div>
<div id="ref-Gomez-Roman2003">
<p>Gomez-Roman, Natividad, Carla Grandori, Robert N. Eisenman, and Robert J. White. 2003. “Direct activation of RNA polymerase III transcription by c-Myc.” <em>Nature</em> 421 (6920): 290–94. <a href="https://doi.org/10.1038/nature01327">https://doi.org/10.1038/nature01327</a>.</p>
</div>
<div id="ref-Goodenbour2006">
<p>Goodenbour, Jeffrey M., and Tao Pan. 2006. “Diversity of tRNA genes in eukaryotes.” <em>Nucleic Acids Research</em> 34 (21): 6137–46. <a href="https://doi.org/10.1093/nar/gkl725">https://doi.org/10.1093/nar/gkl725</a>.</p>
</div>
<div id="ref-Grossman2016">
<p>Grossman, Robert L., Allison P. Heath, Vincent Ferretti, Harold E. Varmus, Douglas R. Lowy, Warren A. Kibbe, and Louis M. Staudt. 2016. “Toward a Shared Vision for Cancer Genomic Data.” <em>New England Journal of Medicine</em> 375 (12): 1109–12. <a href="https://doi.org/10.1056/NEJMp1607591">https://doi.org/10.1056/NEJMp1607591</a>.</p>
</div>
<div id="ref-Gu2016">
<p>Gu, Zuguang, Roland Eils, and Matthias Schlesner. 2016. “Complex heatmaps reveal patterns and correlations in multidimensional genomic data.” <em>Bioinformatics</em> 32 (18): 2847–9. <a href="https://doi.org/10.1093/bioinformatics/btw313">https://doi.org/10.1093/bioinformatics/btw313</a>.</p>
</div>
<div id="ref-Hannum2013">
<p>Hannum, Gregory, Justin Guinney, Ling Zhao, Li Zhang, Guy Hughes, SriniVas Sadda, Brandy Klotzle, et al. 2013. “Genome-wide methylation profiles reveal quantitative views of human aging rates.” <em>Molecular Cell</em> 49 (2): 359–67. <a href="https://doi.org/10.1016/j.molcel.2012.10.016">https://doi.org/10.1016/j.molcel.2012.10.016</a>.</p>
</div>
<div id="ref-Hansen2007">
<p>Hansen, Malene, Stefan Taubert, Douglas Crawford, Nataliya Libina, Seung-Jae Lee, and Cynthia Kenyon. 2007. “Lifespan extension by conditions that inhibit translation in Caenorhabditis elegans.” <em>Aging Cell</em> 6 (1): 95–110. <a href="https://doi.org/10.1111/j.1474-9726.2006.00267.x">https://doi.org/10.1111/j.1474-9726.2006.00267.x</a>.</p>
</div>
<div id="ref-Harvey2012a">
<p>Harvey, Nicholas C, Kassim Javaid, Nicholas Bishop, Stephen Kennedy, Aris T Papageorghiou, Robert Fraser, Saurabh V Gandhi, Inez Schoenmakers, Ann Prentice, and Cyrus Cooper. 2012. “MAVIDOS Maternal Vitamin D Osteoporosis Study: study protocol for a randomized controlled trial. The MAVIDOS Study Group.” <em>Trials</em> 13 (1): 13. <a href="https://doi.org/10.1186/1745-6215-13-13">https://doi.org/10.1186/1745-6215-13-13</a>.</p>
</div>
<div id="ref-Heyn2012">
<p>Heyn, Holger, Ning Li, Humberto J Ferreira, Sebastian Moran, David G Pisano, Antonio Gomez, Javier Diez, et al. 2012. “Distinct DNA methylomes of newborns and centenarians.” <em>Proceedings of the National Academy of Sciences</em> 109 (26): 10522–7. <a href="https://doi.org/10.1073/pnas.1120658109">https://doi.org/10.1073/pnas.1120658109</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>Horvath, Steve. 2013. “DNA methylation age of human tissues and cell types.” <em>Genome Biology</em> 14 (10): R115. <a href="https://doi.org/10.1186/gb-2013-14-10-r115">https://doi.org/10.1186/gb-2013-14-10-r115</a>.</p>
</div>
<div id="ref-Horvath2018a">
<p>Horvath, Steve, Junko Oshima, George M. Martin, Ake T. Lu, Austin Quach, Howard Cohen, Sarah Felton, et al. 2018. “Epigenetic clock for skin and blood cells applied to Hutchinson Gilford Progeria Syndrome and ex vivo studies.” <em>Aging</em> 10 (7): 1758–75. <a href="https://doi.org/10.18632/aging.101508">https://doi.org/10.18632/aging.101508</a>.</p>
</div>
<div id="ref-Horvath2018">
<p>Horvath, Steve, and Kenneth Raj. 2018. “DNA methylation-based biomarkers and the epigenetic clock theory of ageing.” <em>Nature Reviews Genetics</em> 19 (6): 371–84. <a href="https://doi.org/10.1038/s41576-018-0004-3">https://doi.org/10.1038/s41576-018-0004-3</a>.</p>
</div>
<div id="ref-Houseman2012">
<p>Houseman, Eugene Andres, William P Accomando, Devin C Koestler, Brock C Christensen, Carmen J Marsit, Heather H Nelson, John K Wiencke, and Karl T Kelsey. 2012. “DNA methylation arrays as surrogate measures of cell mixture distribution.” <em>BMC Bioinformatics</em> 13 (1): 86. <a href="https://doi.org/10.1186/1471-2105-13-86">https://doi.org/10.1186/1471-2105-13-86</a>.</p>
</div>
<div id="ref-Huang2018">
<p>Huang, Shi-qiong, Bao Sun, Zong-ping Xiong, Yan Shu, Hong-hao Zhou, Wei Zhang, Jing Xiong, and Qing Li. 2018. “The dysregulation of tRNAs and tRNA derivatives in cancer.” <em>Journal of Experimental &amp; Clinical Cancer Research</em> 37 (1): 101. <a href="https://doi.org/10.1186/s13046-018-0745-z">https://doi.org/10.1186/s13046-018-0745-z</a>.</p>
</div>
<div id="ref-Iben2015">
<p>Iben, James R., and Richard J. Maraia. 2014. “tRNA gene copy number variation in humans.” <em>Gene</em> 536 (2): 376–84. <a href="https://doi.org/10.1016/j.gene.2013.11.049">https://doi.org/10.1016/j.gene.2013.11.049</a>.</p>
</div>
<div id="ref-Juzenas2017">
<p>Juzenas, Simonas, Geetha Venkatesh, Matthias Hübenthal, Marc P. Hoeppner, Zhipei Gracie Du, Maren Paulsen, Philip Rosenstiel, et al. 2017. “A comprehensive, cell specific microRNA catalogue of human peripheral blood.” <em>Nucleic Acids Research</em> 45 (16): 9290–9301. <a href="https://doi.org/10.1093/nar/gkx706">https://doi.org/10.1093/nar/gkx706</a>.</p>
</div>
<div id="ref-Keam2014">
<p>Keam, Simon P., Paul E. Young, Alexandra L. McCorkindale, Thurston H. Y. Dang, Jennifer L. Clancy, David T. Humphreys, Thomas Preiss, et al. 2014. “The human Piwi protein Hiwi2 associates with tRNA-derived piRNAs in somatic cells.” <em>Nucleic Acids Research</em> 42 (14): 8984–95. <a href="https://doi.org/10.1093/nar/gku620">https://doi.org/10.1093/nar/gku620</a>.</p>
</div>
<div id="ref-Kennedy2016">
<p>Kennedy, Brian K., and Dudley W. Lamming. 2016. “The Mechanistic Target of Rapamycin: The Grand ConducTOR of Metabolism and Aging.” <em>Cell Metabolism</em> 23 (6): 990–1003. <a href="https://doi.org/10.1016/j.cmet.2016.05.009">https://doi.org/10.1016/j.cmet.2016.05.009</a>.</p>
</div>
<div id="ref-Kim2017">
<p>Kim, Hak Kyun, Gabriele Fuchs, Shengchun Wang, Wei Wei, Yue Zhang, Hyesuk Park, Biswajoy Roy-Chaudhuri, et al. 2017. “A transfer-RNA-derived small RNA regulates ribosome biogenesis.” <em>Nature</em> 552 (7683): 57–62. <a href="https://doi.org/10.1038/nature25005">https://doi.org/10.1038/nature25005</a>.</p>
</div>
<div id="ref-Kirchner2017">
<p>Kirchner, Sebastian, Zhiwei Cai, Robert Rauscher, Nicolai Kastelic, Melanie Anding, Andreas Czech, Bertrand Kleizen, et al. 2017. “Alteration of protein function by a silent polymorphism linked to tRNA abundance.” Edited by Laurence Hurst. <em>PLOS Biology</em> 15 (5): e2000779. <a href="https://doi.org/10.1371/journal.pbio.2000779">https://doi.org/10.1371/journal.pbio.2000779</a>.</p>
</div>
<div id="ref-Kolitz2010">
<p>Kolitz, Sarah E., and Jon R. Lorsch. 2010. “Eukaryotic initiator tRNA: Finely tuned and ready for action.” <em>FEBS Letters</em> 584 (2): 396–404. <a href="https://doi.org/10.1016/j.febslet.2009.11.047">https://doi.org/10.1016/j.febslet.2009.11.047</a>.</p>
</div>
<div id="ref-Krishnan2016">
<p>Krishnan, Preethi, Sunita Ghosh, Bo Wang, Mieke Heyns, Dongping Li, John R. Mackey, Olga Kovalchuk, and Sambasivarao Damaraju. 2016. “Genome-wide profiling of transfer RNAs and their role as novel prognostic markers for breast cancer.” <em>Scientific Reports</em> 6 (1): 32843. <a href="https://doi.org/10.1038/srep32843">https://doi.org/10.1038/srep32843</a>.</p>
</div>
<div id="ref-Krueger2015">
<p>Krueger, Felix. 2015. “Trim Galore.” <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim%7B\_%7Dgalore/">https://www.bioinformatics.babraham.ac.uk/projects/trim{\_}galore/</a>.</p>
</div>
<div id="ref-Krueger2011">
<p>Krueger, Felix, and Simon R. Andrews. 2011. “Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications.” <em>Bioinformatics</em> 27 (11): 1571–2. <a href="https://doi.org/10.1093/bioinformatics/btr167">https://doi.org/10.1093/bioinformatics/btr167</a>.</p>
</div>
<div id="ref-Langmead2012">
<p>Langmead, Ben, and Steven L Salzberg. 2012. “Fast gapped-read alignment with Bowtie 2.” <em>Nature Methods</em> 9 (4): 357–9. <a href="https://doi.org/10.1038/nmeth.1923">https://doi.org/10.1038/nmeth.1923</a>.</p>
</div>
<div id="ref-Lappalainen2017">
<p>Lappalainen, Tuuli, and John M. Greally. 2017. “Associating cellular epigenetic models with human phenotypes.” <em>Nature Reviews Genetics</em> 18 (7): 441–51. <a href="https://doi.org/10.1038/nrg.2017.32">https://doi.org/10.1038/nrg.2017.32</a>.</p>
</div>
<div id="ref-Lee2009">
<p>Lee, Yong Sun, Yoshiyuki Shibata, Ankit Malhotra, and Anindya Dutta. 2009. “A novel class of small RNAs: tRNA-derived RNA fragments (tRFs).” <em>Genes &amp; Development</em> 23 (22): 2639–49. <a href="https://doi.org/10.1101/gad.1837609">https://doi.org/10.1101/gad.1837609</a>.</p>
</div>
<div id="ref-Levine2018">
<p>Levine, Morgan E, Ake T Lu, Austin Quach, Brian H Chen, Themistocles L Assimes, Stefania Bandinelli, Lifang Hou, et al. 2018. “An epigenetic biomarker of aging for lifespan and healthspan.” <em>Aging</em> 10 (4): 573–91. <a href="https://doi.org/10.18632/aging.101414">https://doi.org/10.18632/aging.101414</a>.</p>
</div>
<div id="ref-Li2002">
<p>Li, Long-Cheng, and Rajvir Dahiya. 2002. “MethPrimer: designing primers for methylation PCRs.” <em>Bioinformatics (Oxford, England)</em> 18 (11): 1427–31. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12424112">http://www.ncbi.nlm.nih.gov/pubmed/12424112</a>.</p>
</div>
<div id="ref-Li2019">
<p>Li, Siqi, Xiaoliang Shi, Muxiong Chen, Ningqin Xu, Desen Sun, Rongpan Bai, Haiyan Chen, Kefeng Ding, Jinghao Sheng, and Zhengping Xu. 2019. “Angiogenin promotes colorectal cancer metastasis via tiRNA production.” <em>International Journal of Cancer</em>, March, ijc.32245. <a href="https://doi.org/10.1002/ijc.32245">https://doi.org/10.1002/ijc.32245</a>.</p>
</div>
<div id="ref-Li2018a">
<p>Li, Siqi, Zhengping Xu, and Jinghao Sheng. 2018. “tRNA-Derived Small RNA: A Novel Regulatory Small Non-Coding RNA.” <em>Genes</em> 9 (5): 246. <a href="https://doi.org/10.3390/genes9050246">https://doi.org/10.3390/genes9050246</a>.</p>
</div>
<div id="ref-Lienhard2014">
<p>Lienhard, Matthias, Christina Grimm, Markus Morkel, Ralf Herwig, and Lukas Chavez. 2014. “MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments.” <em>Bioinformatics</em> 30 (2): 284–86. <a href="https://doi.org/10.1093/bioinformatics/btt650">https://doi.org/10.1093/bioinformatics/btt650</a>.</p>
</div>
<div id="ref-Lodish2000">
<p>Lodish, Harvey, Arnold Berk, S Lawrence Zipursky, Paul Matsudaira, David Baltimore, and James Darnell. 2000. <em>Molecular Cell Biology, 4th edition</em>. 4th ed. New York: W. H. Freeman. <a href="https://www.ncbi.nlm.nih.gov/books/NBK21475/">https://www.ncbi.nlm.nih.gov/books/NBK21475/</a>.</p>
</div>
<div id="ref-Loher2017">
<p>Loher, Phillipe, Aristeidis G. Telonis, and Isidore Rigoutsos. 2017. “MINTmap: fast and exhaustive profiling of nuclear and mitochondrial tRNA fragments from short RNA-seq data.” <em>Scientific Reports</em> 7 (1): 41184. <a href="https://doi.org/10.1038/srep41184">https://doi.org/10.1038/srep41184</a>.</p>
</div>
<div id="ref-Lowe2016">
<p>Lowe, Todd M., and Patricia P. Chan. 2016. “tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes.” <em>Nucleic Acids Research</em> 44 (W1): W54–W57. <a href="https://doi.org/10.1093/nar/gkw413">https://doi.org/10.1093/nar/gkw413</a>.</p>
</div>
<div id="ref-Lopez-Otin2013">
<p>López-Otín, Carlos, Maria A. Blasco, Linda Partridge, Manuel Serrano, and Guido Kroemer. 2013. “The hallmarks of aging.” <em>Cell</em> 153 (6): 1194–1217. <a href="https://doi.org/10.1016/j.cell.2013.05.039">https://doi.org/10.1016/j.cell.2013.05.039</a>.</p>
</div>
<div id="ref-Lu2019">
<p>Lu, Ake T., Austin Quach, James G. Wilson, Alex P. Reiner, Abraham Aviv, Kenneth Raj, Lifang Hou, et al. 2019. “DNA methylation GrimAge strongly predicts lifespan and healthspan.” <em>Aging</em> 11 (2): 303–27. <a href="https://doi.org/10.18632/aging.101684">https://doi.org/10.18632/aging.101684</a>.</p>
</div>
<div id="ref-Mange2017">
<p>Mange, François, Viviane Praz, Eugenia Migliavacca, Ian M Willis, Frédéric Schütz, and Nouria Hernandez. 2017. “Diurnal regulation of RNA polymerase III transcription is under the control of both the feeding–fasting response and the circadian clock.” <em>Genome Research</em> 27 (6): 973–84. <a href="https://doi.org/10.1101/gr.217521.116">https://doi.org/10.1101/gr.217521.116</a>.</p>
</div>
<div id="ref-Martin2011">
<p>Martin, Marcel. 2011. “Cutadapt removes adapter sequences from high-throughput sequencing reads.” <em>EMBnet.journal</em> 17 (1): 10. <a href="https://doi.org/10.14806/ej.17.1.200">https://doi.org/10.14806/ej.17.1.200</a>.</p>
</div>
<div id="ref-Martinez2017">
<p>Martinez, German, Sarah G. Choudury, and R. Keith Slotkin. 2017. “tRNA-derived small RNAs target transposable element transcripts.” <em>Nucleic Acids Research</em> 45 (9): 5142–52. <a href="https://doi.org/10.1093/nar/gkx103">https://doi.org/10.1093/nar/gkx103</a>.</p>
</div>
<div id="ref-Meuleman2019">
<p>Meuleman, Wouter. 2019. “Epilogos.” <a href="https://epilogos.altius.org/%20https://github.com/Altius/epilogos">https://epilogos.altius.org/ https://github.com/Altius/epilogos</a>.</p>
</div>
<div id="ref-Moayyeri2013">
<p>Moayyeri, Alireza, Christopher J. Hammond, Ana M. Valdes, and Timothy D. Spector. 2013. “Cohort Profile: TwinsUK and Healthy Ageing Twin Study.” <em>International Journal of Epidemiology</em> 42 (1): 76–85. <a href="https://doi.org/10.1093/ije/dyr207">https://doi.org/10.1093/ije/dyr207</a>.</p>
</div>
<div id="ref-Murawski1994">
<p>Murawski, M., B. Szczesniak, T. Zoladek, A. K. Hopper, N. C. Martin, and M. Boguta. 1994. “maf1 mutation alters the subcellular localization of the Mod5 protein in yeast.” <em>Acta Biochimica Polonica</em> 41 (4): 441–8. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7732762">http://www.ncbi.nlm.nih.gov/pubmed/7732762</a>.</p>
</div>
<div id="ref-Muller2017">
<p>Müller, Carolin A., and Conrad A. Nieduszynski. 2017. “DNA replication timing influences gene expression level.” <em>The Journal of Cell Biology</em> 216 (7): 1907–14. <a href="https://doi.org/10.1083/jcb.201701061">https://doi.org/10.1083/jcb.201701061</a>.</p>
</div>
<div id="ref-Muller2019">
<p>Müller, Fabian, Michael Scherer, Yassen Assenov, Pavlo Lutsik, Jörn Walter, Thomas Lengauer, and Christoph Bock. 2019. “RnBeads 2.0: comprehensive analysis of DNA methylation data.” <em>Genome Biology</em> 20 (1): 55. <a href="https://doi.org/10.1186/s13059-019-1664-9">https://doi.org/10.1186/s13059-019-1664-9</a>.</p>
</div>
<div id="ref-Nazor2012">
<p>Nazor, Kristopher L., Gulsah Altun, Candace Lynch, Ha Tran, Julie V. Harness, Ileana Slavin, Ibon Garitaonandia, et al. 2012. “Recurrent Variations in DNA Methylation in Human Pluripotent Stem Cells and Their Differentiated Derivatives.” <em>Cell Stem Cell</em> 10 (5): 620–34. <a href="https://doi.org/10.1016/j.stem.2012.02.013">https://doi.org/10.1016/j.stem.2012.02.013</a>.</p>
</div>
<div id="ref-Noma2006">
<p>Noma, Ken-ichi, Hugh P. Cam, Richard J. Maraia, and Shiv I S Grewal. 2006. “A role for TFIIIC transcription factor complex in genome organization.” <em>Cell</em> 125 (5): 859–72. <a href="https://doi.org/10.1016/j.cell.2006.04.028">https://doi.org/10.1016/j.cell.2006.04.028</a>.</p>
</div>
<div id="ref-North2003">
<p>North, B. V., D. Curtis, and P. C. Sham. 2003. “A Note on the Calculation of Empirical P Values from Monte Carlo Procedures.” <em>The American Journal of Human Genetics</em> 72 (2): 498–99. <a href="https://doi.org/10.1086/346173">https://doi.org/10.1086/346173</a>.</p>
</div>
<div id="ref-Nwanaji-Enwerem2018">
<p>Nwanaji-Enwerem, Jamaji C., Marc G. Weisskopf, and Andrea A. Baccarelli. 2018. “Multi-tissue DNA methylation age: Molecular relationships and perspectives for advancing biomarker utility.” <em>Ageing Research Reviews</em> 45 (March): 15–23. <a href="https://doi.org/10.1016/j.arr.2018.04.005">https://doi.org/10.1016/j.arr.2018.04.005</a>.</p>
</div>
<div id="ref-Pace2006">
<p>Pace, D. A. 2006. “Fixed metabolic costs for highly variable rates of protein synthesis in sea urchin embryos and larvae.” <em>Journal of Experimental Biology</em> 209 (1): 158–70. <a href="https://doi.org/10.1242/jeb.01962">https://doi.org/10.1242/jeb.01962</a>.</p>
</div>
<div id="ref-Parisien2013">
<p>Parisien, Marc, Xiaoyun Wang, and Tao Pan. 2013. “Diversity of human tRNA genes from the 1000-genomes project.” <em>RNA Biology</em> 10 (12): 1853–67. <a href="https://doi.org/10.4161/rna.27361">https://doi.org/10.4161/rna.27361</a>.</p>
</div>
<div id="ref-Partridge2018">
<p>Partridge, Linda, Joris Deelen, and P. Eline Slagboom. 2018. “Facing up to the global challenges of ageing.” <em>Nature</em> 561 (7721): 45–56. <a href="https://doi.org/10.1038/s41586-018-0457-8">https://doi.org/10.1038/s41586-018-0457-8</a>.</p>
</div>
<div id="ref-Pavon-Eternod2013">
<p>Pavon-Eternod, Mariana, Suzana Gomes, Marsha R Rosner, and Tao Pan. 2013. “Overexpression of initiator methionine tRNA leads to global reprogramming of tRNA expression and increased proliferation in human epithelial cells.” <em>RNA</em> 19 (4): 461–66. <a href="https://doi.org/10.1261/rna.037507.112">https://doi.org/10.1261/rna.037507.112</a>.</p>
</div>
<div id="ref-Petkovich2017">
<p>Petkovich, Daniel A., Dmitriy I. Podolskiy, Alexei V. Lobanov, Sang-Goo Lee, Richard A. Miller, and Vadim N. Gladyshev. 2017. “Using DNA Methylation Profiling to Evaluate Biological Age and Longevity Interventions.” <em>Cell Metabolism</em> 25 (4): 954–960.e6. <a href="https://doi.org/10.1016/j.cmet.2017.03.016">https://doi.org/10.1016/j.cmet.2017.03.016</a>.</p>
</div>
<div id="ref-Pliatsika2018">
<p>Pliatsika, Venetia, Phillipe Loher, Rogan Magee, Aristeidis G. Telonis, Eric Londin, Megumi Shigematsu, Yohei Kirino, and Isidore Rigoutsos. 2018. “MINTbase v2.0: a comprehensive database for tRNA-derived fragments that includes nuclear and mitochondrial fragments from all The Cancer Genome Atlas projects.” <em>Nucleic Acids Research</em> 46 (D1): D152–D159. <a href="https://doi.org/10.1093/nar/gkx1075">https://doi.org/10.1093/nar/gkx1075</a>.</p>
</div>
<div id="ref-Pluta2001">
<p>Pluta, Krzysztof, Olivier Lefebvre, Nancy C Martin, Wieslaw J Smagowicz, David R Stanford, Steven R Ellis, Anita K Hopper, Andre Sentenac, and Magdalena Boguta. 2001. “Maf1p, a Negative Effector of RNA Polymerase III in Saccharomyces cerevisiae.” <em>Molecular and Cellular Biology</em> 21 (15): 5031–40. <a href="https://doi.org/10.1128/MCB.21.15.5031-5040.2001">https://doi.org/10.1128/MCB.21.15.5031-5040.2001</a>.</p>
</div>
<div id="ref-Powell1997">
<p>Powell, J R, and E N Moriyama. 1997. “Evolution of codon usage bias in Drosophila.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 94 (15): 7784–90. <a href="https://doi.org/10.1073/pnas.0404957101">https://doi.org/10.1073/pnas.0404957101</a>.</p>
</div>
<div id="ref-Quinlan2010">
<p>Quinlan, Aaron R., and Ira M. Hall. 2010. “BEDTools: a flexible suite of utilities for comparing genomic features.” <em>Bioinformatics</em> 26 (6): 841–42. <a href="https://doi.org/10.1093/bioinformatics/btq033">https://doi.org/10.1093/bioinformatics/btq033</a>.</p>
</div>
<div id="ref-Raab2011">
<p>Raab, Jesse R, Jonathan Chiu, Jingchun Zhu, Sol Katzman, Sreenivasulu Kurukuti, Paul A Wade, David Haussler, and Rohinton T Kamakaka. 2012. “Human tRNA genes function as chromatin insulators.” <em>The EMBO Journal</em> 31 (2): 330–50. <a href="https://doi.org/10.1038/emboj.2011.406">https://doi.org/10.1038/emboj.2011.406</a>.</p>
</div>
<div id="ref-Rakyan2010">
<p>Rakyan, Vardhman K., Thomas A. Down, Siarhei Maslau, Toby Andrew, Tsun Po Yang, Huriya Beyan, Pamela Whittaker, et al. 2010. “Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains.” <em>Genome Research</em> 20 (4): 434–39. <a href="https://doi.org/10.1101/gr.103101.109">https://doi.org/10.1101/gr.103101.109</a>.</p>
</div>
<div id="ref-Reinius2012">
<p>Reinius, Lovisa E., Nathalie Acevedo, Maaike Joerink, Göran Pershagen, Sven-Erik Dahlén, Dario Greco, Cilla Söderhäll, Annika Scheynius, and Juha Kere. 2012. “Differential DNA Methylation in Purified Human Blood Cells: Implications for Cell Lineage and Studies on Disease Susceptibility.” Edited by Angela H. Ting. <em>PLoS ONE</em> 7 (7): e41361. <a href="https://doi.org/10.1371/journal.pone.0041361">https://doi.org/10.1371/journal.pone.0041361</a>.</p>
</div>
<div id="ref-Rideout2012">
<p>Rideout, E. J., L. Marshall, and S. S. Grewal. 2012. “Drosophila RNA polymerase III repressor Maf1 controls body size and developmental timing by modulating tRNAiMet synthesis and systemic insulin signaling.” <em>Proceedings of the National Academy of Sciences</em> 109 (4): 1139–44. <a href="https://doi.org/10.1073/pnas.1113311109">https://doi.org/10.1073/pnas.1113311109</a>.</p>
</div>
<div id="ref-Ritchie2015">
<p>Ritchie, Matthew E, Belinda Phipson, Di Wu, Yifang Hu, Charity W Law, Wei Shi, and Gordon K Smyth. 2015. “limma powers differential expression analyses for RNA-sequencing and microarray studies.” <em>Nucleic Acids Research</em> 43 (7): e47–e47. <a href="https://doi.org/10.1093/nar/gkv007">https://doi.org/10.1093/nar/gkv007</a>.</p>
</div>
<div id="ref-Rudolph2016">
<p>Rudolph, Konrad L M, Bianca M. Schmitt, Diego Villar, Robert J. White, John C. Marioni, Claudia Kutter, and Duncan T. Odom. 2016. “Codon-Driven Translational Efficiency Is Stable across Diverse Mammalian Cell States.” Edited by Nicolas Galtier. <em>PLoS Genetics</em> 12 (5): e1006024. <a href="https://doi.org/10.1371/journal.pgen.1006024">https://doi.org/10.1371/journal.pgen.1006024</a>.</p>
</div>
<div id="ref-Sagi2016">
<p>Sagi, Dror, Roni Rak, Hila Gingold, Idan Adir, Gadi Maayan, Orna Dahan, Limor Broday, Yitzhak Pilpel, and Oded Rechavi. 2016. “Tissue- and Time-Specific Expression of Otherwise Identical tRNA Genes.” Edited by Andrew D. Chisholm. <em>PLOS Genetics</em> 12 (8): e1006264. <a href="https://doi.org/10.1371/journal.pgen.1006264">https://doi.org/10.1371/journal.pgen.1006264</a>.</p>
</div>
<div id="ref-Sandoval2011">
<p>Sandoval, Juan, Holger Heyn, Sebastian Moran, Jordi Serra-Musach, Miguel A. Pujana, Marina Bibikova, and Manel Esteller. 2011b. “Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome.” <em>Epigenetics</em> 6 (6): 692–702. <a href="https://doi.org/10.4161/epi.6.6.16196">https://doi.org/10.4161/epi.6.6.16196</a>.</p>
</div>
<div id="ref-Schimmel2017">
<p>Schimmel, Paul. 2018. “The emerging complexity of the tRNA world: mammalian tRNAs beyond protein synthesis.” <em>Nature Reviews. Molecular Cell Biology</em> 19 (1): 45–58. <a href="https://doi.org/10.1038/nrm.2017.77">https://doi.org/10.1038/nrm.2017.77</a>.</p>
</div>
<div id="ref-Schmitt2014">
<p>Schmitt, Bianca M, Konrad L M Rudolph, Panagiota Karagianni, Nuno A Fonseca, Robert J White, Iannis Talianidis, Duncan T Odom, John C Marioni, and Claudia Kutter. 2014. “High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA-tRNA interface.” <em>Genome Research</em> 24 (11): 1797–1807. <a href="https://doi.org/10.1101/gr.176784.114">https://doi.org/10.1101/gr.176784.114</a>.</p>
</div>
<div id="ref-Schmitt2014b">
<p>Schmitt, Bianca M., Konrad L. M. Rudolph, Panagiota Karagianni, Nuno A. Fonseca, Robert J. White, Iannis Talianidis, Duncan T. Odom, John C. Marioni, and Claudia Kutter. 2014. “High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA–tRNA interface.” <em>Genome Research</em> 24 (11): 1797–1807. <a href="https://doi.org/10.1101/gr.176784.114">https://doi.org/10.1101/gr.176784.114</a>.</p>
</div>
<div id="ref-Schramm2002">
<p>Schramm, Laura. 2002. “Recruitment of RNA polymerase III to its target promoters.” <em>Genes &amp; Development</em> 16 (20): 2593–2620. <a href="https://doi.org/10.1101/gad.1018902">https://doi.org/10.1101/gad.1018902</a>.</p>
</div>
<div id="ref-Davis2007p">
<p>Sean, Davis, and Paul S. Meltzer. 2007. “GEOquery: A bridge between the Gene Expression Omnibus (GEO) and BioConductor.” <em>Bioinformatics</em> 23 (14): 1846–7. <a href="https://doi.org/10.1093/bioinformatics/btm254">https://doi.org/10.1093/bioinformatics/btm254</a>.</p>
</div>
<div id="ref-Slieker2016">
<p>Slieker, Roderick C., Maarten van Iterson, René Luijk, Marian Beekman, Daria V Zhernakova, Matthijs H. Moed, Hailiang Mei, et al. 2016. “Age-related accrual of methylomic variability is linked to fundamental ageing mechanisms.” <em>Genome Biology</em> 17 (1): 191. <a href="https://doi.org/10.1186/s13059-016-1053-6">https://doi.org/10.1186/s13059-016-1053-6</a>.</p>
</div>
<div id="ref-Sun2018">
<p>Sun, Luyang, Ruofan Yu, and Weiwei Dang. 2018. “Chromatin Architectural Changes during Cellular Senescence and Aging.” <em>Genes</em> 9 (4): 211. <a href="https://doi.org/10.3390/genes9040211">https://doi.org/10.3390/genes9040211</a>.</p>
</div>
<div id="ref-Sutcliffe2000">
<p>Sutcliffe, Josephine E, Timothy R P Brown, Simon J Allison, Pamela H Scott, and Robert J White. 2000. “Retinoblastoma Protein Disrupts Interactions Required for RNA Polymerase III Transcription.” <em>Molecular and Cellular Biology</em> 20 (24): 9192–9202. <a href="https://doi.org/10.1128/MCB.20.24.9192-9202.2000">https://doi.org/10.1128/MCB.20.24.9192-9202.2000</a>.</p>
</div>
<div id="ref-Syddall2005">
<p>Syddall, HE, A Aihie Sayer, EM Dennison, HJ Martin, DJP Barker, and C. Cooper. 2005. “Cohort Profile: The Hertfordshire Cohort Study.” <em>International Journal of Epidemiology</em> 34 (6): 1234–42. <a href="https://doi.org/10.1093/ije/dyi127">https://doi.org/10.1093/ije/dyi127</a>.</p>
</div>
<div id="ref-Tavernarakis2008">
<p>Tavernarakis, Nektarios. 2008. “Ageing and the regulation of protein synthesis: a balancing act?” <em>Trends in Cell Biology</em> 18 (5): 228–35. <a href="https://doi.org/10.1016/j.tcb.2008.02.004">https://doi.org/10.1016/j.tcb.2008.02.004</a>.</p>
</div>
<div id="ref-Teschendorff2010">
<p>Teschendorff, Andrew E., Usha Menon, Aleksandra Gentry-Maharaj, Susan J. Ramus, Daniel J. Weisenberger, Hui Shen, Mihaela Campan, et al. 2010. “Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.” <em>Genome Research</em> 20 (4): 440–6. <a href="https://doi.org/10.1101/gr.103606.109">https://doi.org/10.1101/gr.103606.109</a>.</p>
</div>
<div id="ref-Torres2019">
<p>Torres, Adrian Gabriel, Oscar Reina, Camille Stephan-Otto Attolini, and Lluís Ribas de Pouplana. 2019. “Differential expression of human tRNA genes drives the abundance of tRNA-derived fragments.” <em>Proceedings of the National Academy of Sciences</em> 116 (17): 201821120. <a href="https://doi.org/10.1073/pnas.1821120116">https://doi.org/10.1073/pnas.1821120116</a>.</p>
</div>
<div id="ref-VanBortle2017">
<p>Van Bortle, Kevin, Douglas H. Phanstiel, and Michael P. Snyder. 2017. “Topological organization and dynamic regulation of human tRNA genes during macrophage differentiation.” <em>Genome Biology</em> 18 (1): 180. <a href="https://doi.org/10.1186/s13059-017-1310-3">https://doi.org/10.1186/s13059-017-1310-3</a>.</p>
</div>
<div id="ref-Varshney2015">
<p>Varshney, Dhaval, Jana Vavrova-Anderson, Andrew J Oler, Victoria H Cowling, Bradley R Cairns, and Robert J White. 2015. “SINE transcription by RNA polymerase III is suppressed by histone methylation but not by DNA methylation.” <em>Nature Communications</em> 6 (1): 6569. <a href="https://doi.org/10.1038/ncomms7569">https://doi.org/10.1038/ncomms7569</a>.</p>
</div>
<div id="ref-Vorlander2020">
<p>Vorländer, Matthias K, Florence Baudin, Robyn D Moir, René Wetzel, Wim J H Hagen, Ian M Willis, and Christoph W Müller. 2020. “Structural basis for RNA polymerase III transcription repression by Maf1.” <em>Nature Structural &amp; Molecular Biology</em>, February. <a href="https://doi.org/10.1038/s41594-020-0383-y">https://doi.org/10.1038/s41594-020-0383-y</a>.</p>
</div>
<div id="ref-Wilson1983">
<p>Wilson, V L, and P A Jones. 1983. “DNA methylation decreases in aging but not in immortal cells.” <em>Science (New York, N.Y.)</em> 220 (4601): 1055–7. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.</p>
</div>
<div id="ref-Xu2017a">
<p>Xu, Wei-Lin, Ye Yang, Yi-Dan Wang, Liang-Hu Qu, and Ling-Ling Zheng. 2017. “Computational Approaches to tRNA-Derived Small RNAs.” <em>Non-Coding RNA</em> 3 (1): 2. <a href="https://doi.org/10.3390/ncrna3010002">https://doi.org/10.3390/ncrna3010002</a>.</p>
</div>
<div id="ref-Yang2016">
<p>Yang, Zhen, Andrew Wong, Diana Kuh, Dirk S. Paul, Vardhman K. Rakyan, R. David Leslie, Shijie C. Zheng, Martin Widschwendter, Stephan Beck, and Andrew E. Teschendorff. 2016. “Correlation of an epigenetic mitotic clock with cancer risk.” <em>Genome Biology</em> 17 (1): 205. <a href="https://doi.org/10.1186/s13059-016-1064-3">https://doi.org/10.1186/s13059-016-1064-3</a>.</p>
</div>
<div id="ref-Zhou2017">
<p>Zhou, Wanding, Peter W. Laird, and Hui Shen. 2016. “Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes.” <em>Nucleic Acids Research</em> 45 (4): gkw967. <a href="https://doi.org/10.1093/nar/gkw967">https://doi.org/10.1093/nar/gkw967</a>.</p>
</div>
<div id="ref-Ziller2013">
<p>Ziller, Michael J, Hongcang Gu, Fabian Müller, Julie Donaghey, Linus T-Y Tsai, Oliver Kohlbacher, Philip L De Jager, et al. 2013. “Charting a dynamic DNA methylation landscape of the human genome.” <em>Nature</em> 500 (7463): 477–81. <a href="https://doi.org/10.1038/nature12433">https://doi.org/10.1038/nature12433</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="MAVIDOS.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="results-3-.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/RichardJActon/thesis/04-tRNA.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
