<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 5Zhou, Xiang, and Matthew Stephens. “Genome-wide efficient mixed-model analysis for association studies.” Nature genetics 44.7 (2012): 821-824. | The Epigenomics of Human Ageing</title>
  <meta name="description" content="PhD Thesis" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="6 5Zhou, Xiang, and Matthew Stephens. “Genome-wide efficient mixed-model analysis for association studies.” Nature genetics 44.7 (2012): 821-824. | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhD Thesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 5Zhou, Xiang, and Matthew Stephens. “Genome-wide efficient mixed-model analysis for association studies.” Nature genetics 44.7 (2012): 821-824. | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="PhD Thesis" />
  

<meta name="author" content="Richard J. Acton" />



<meta name="date" content="2020-09-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="Alus.html"/>
<link rel="next" href="discussion.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Research Thesis: Declaration of Authorship</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a><ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#distribution-and-global-trends"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#discussion-and-analysis-of-the-findings-of-dnam-age-studies"><i class="fa fa-check"></i><b>1.5.4</b> Discussion and Analysis of the Findings of DNAm Age Studies</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#GenMethodsMethArray"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#targeted-bisulfite-sequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a><ul>
<li class="chapter" data-level="3.1" data-path="abstract.html"><a href="abstract.html#abstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="introduction.html"><a href="introduction.html#introduction"><i class="fa fa-check"></i><b>3.2</b> Introduction</a><ul>
<li class="chapter" data-level="3.2.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.2.1</b> Outline of EWAS</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>3.3</b> Methods</a><ul>
<li class="chapter" data-level="3.3.1" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.3.1</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.3.2" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.3.2</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.3.3" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.3.3</b> EWAS Models</a></li>
<li class="chapter" data-level="3.3.4" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.3.4</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a><ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.4.1</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.4.2</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.4.3</b> Southampton Women’s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a><ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-abstract"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Introduction"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#tRNAmethods"><i class="fa fa-check"></i><b>4.3</b> Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Participants"><i class="fa fa-check"></i><b>4.3.1</b> Participants</a></li>
<li class="chapter" data-level="4.3.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.3.2</b> tRNA annotation information</a></li>
<li class="chapter" data-level="4.3.3" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>4.3.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.3.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.3.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.3.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.3.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.3.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.3.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.3.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.3.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.3.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.3.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.4.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.4.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-methylation-in-cancer"><i class="fa fa-check"></i><b>4.5</b> tRNA Gene Methylation in Cancer</a><ul>
<li class="chapter" data-level="4.5.1" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.5.1</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.5.2" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.5.2</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Discussion"><i class="fa fa-check"></i><b>4.6</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>5</b> DNA methylation clocks in Alu Elements</a><ul>
<li class="chapter" data-level="5.1" data-path="abstract.html"><a href="abstract.html#abstract"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="methods.html"><a href="methods.html#methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylome-data"><i class="fa fa-check"></i><b>5.3.1</b> DNA methylome data</a></li>
<li class="chapter" data-level="5.3.2" data-path="Alus.html"><a href="Alus.html#alu-element-annotation-and-pre-filtering"><i class="fa fa-check"></i><b>5.3.2</b> Alu element annotation and pre-filtering</a></li>
<li class="chapter" data-level="5.3.3" data-path="Alus.html"><a href="Alus.html#elastic-net-regression"><i class="fa fa-check"></i><b>5.3.3</b> Elastic net regression</a></li>
<li class="chapter" data-level="5.3.4" data-path="Alus.html"><a href="Alus.html#sample-selection"><i class="fa fa-check"></i><b>5.3.4</b> Sample Selection</a></li>
<li class="chapter" data-level="5.3.5" data-path="Alus.html"><a href="Alus.html#binarisation"><i class="fa fa-check"></i><b>5.3.5</b> Binarisation</a></li>
<li class="chapter" data-level="5.3.6" data-path="Alus.html"><a href="Alus.html#AluMethodsArrays"><i class="fa fa-check"></i><b>5.3.6</b> Comparisons to other clocks in TwinsUK Illumina 450k array data</a></li>
<li class="chapter" data-level="5.3.7" data-path="Alus.html"><a href="Alus.html#genetic-association-for-alu-clock"><i class="fa fa-check"></i><b>5.3.7</b> Genetic Association for Alu-Clock</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html"><a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html"><i class="fa fa-check"></i><b>6</b> 5Zhou, Xiang, and Matthew Stephens. “Genome-wide efficient mixed-model analysis for association studies.” Nature genetics 44.7 (2012): 821-824.</a><ul>
<li class="chapter" data-level="6.1" data-path="arrays.html"><a href="arrays.html#results"><i class="fa fa-check"></i><b>6.1</b> Results</a></li>
<li class="chapter" data-level="6.2" data-path="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html"><a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#comparison-to-other-dna-methylation-clocks"><i class="fa fa-check"></i><b>6.2</b> Comparison to other DNA methylation clocks</a><ul>
<li class="chapter" data-level="6.2.1" data-path="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html"><a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#alu-age-acceleration"><i class="fa fa-check"></i><b>6.2.1</b> Alu Age Acceleration</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="arrays.html"><a href="arrays.html#discussion"><i class="fa fa-check"></i><b>7</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="appendices.html"><a href="appendices.html"><i class="fa fa-check"></i>Appendices</a><ul>
<li class="chapter" data-level="7.1" data-path="appendices.html"><a href="appendices.html#biorchi-iv-manuscript-the-genomic-loci-of-specific-human-trna-genes-exhibit-ageing-related-dna-hypermethylation"><i class="fa fa-check"></i><b>7.1</b> <span class="math inline">\(BioR\chi iv\)</span> manuscript: The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Epigenomics of Human Ageing</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="zhou-xiang-and-matthew-stephens.-genome-wide-efficient-mixed-model-analysis-for-association-studies.-nature-genetics-44.7-2012-821-824." class="section level1">
<h1><span class="header-section-number">6</span> 5Zhou, Xiang, and Matthew Stephens. “Genome-wide efficient mixed-model analysis for association studies.” Nature genetics 44.7 (2012): 821-824.</h1>
<div style="page-break-after: always;"></div>
<div id="results" class="section level2">
<h2><span class="header-section-number">6.1</span> Results</h2>
<p>A variety of different initial filtering criteria were applied to the Alu elements included in the models, as well as a number of data transformations.
Pre-filtering steps were performed partially for quality control and partially to see if subsets of the Alu space with biologically interesting differences could yield clocks of equivalent accuracy but which might capture different aspects of biological ageing related to repeat de-repression.</p>
<p>Preliminary analysis were carried out using RPM values, Quantile normalised RPM values, Absolute methylation value estimates from <code>MEDIPS</code> and Binarised DNA methylation data.
Quantile normalised RPM values yielded the best and most consistent results and were used in subsequent analyses.</p>
<p>Whilst Human epigenetic age is logarithmic with time across the lifespan the effects of this are more relevant at younger ages <span class="citation">[<a href="#ref-Snir2019" role="doc-biblioref">396</a>]</span>.
We fitted Alu Clock models on the natural logarithm of age in years this did not improve the quality of predicted to actual age values, likely due in part to the older age distribution of our data.</p>
<p>An Alu element length filter of at least 180bp or approximately two-thirds the length of a full length Alu element (~280bp) was imposed.
This minimises the number of reads with the potential to misalign to other Alu elements, reads which, in this context, could lead to reduced accuracy of positionally specific Alu DNA methylation estimates <span class="citation">[<a href="#ref-Hellen2013" role="doc-biblioref">397</a>]</span>.</p>
<p>CpG density filters were employed to prioritise regions likely to have dynamic and functionally interesting differences in DNA methylation.
Filters were used for CpG densities above the genomic background of 2%, as well as for &gt;5% the approximate threshold for CpG island shores and low methylation regions <span class="citation">[<a href="#ref-Irizarry2009" role="doc-biblioref">55</a>,<a href="#ref-Stadler2011" role="doc-biblioref">62</a>]</span>.</p>
<p>To limit the selection of predictive sites to those likely to have consistent directions of change across samples a clock was constructed with only windows which show nominally significant changes in both members of an MZ twin pair in a consistent direction.
Linear models to predict age based on DNA methylation were fitted for 687 pairs of monozygotic (MZ) twins.
Batch information and blood cell counts (eosinophils, lymphocytes, monocytes, neutrophils) were included as covariates.</p>
<p>To produce a model that specifically captures age as a function of the de-repression of Alu elements clocks were also constructed with only windows which exhibited nominally significant (p &lt; 0.05) decreases in DNAm in both MZ twins.
This filtering was based on the model used in selecting windows with a consistent direction of change.</p>
<p>An age predictor was also constructed using only AluY elements.
AluY is the youngest (Figure <a href="Alus.html#fig:AluAge">5.2</a>), and most CpG dense <a href="Alus.html#fig:AlusWindowsCpGdLVplot">5.3</a> family of Alu elements.
Having a CpG density closer to that of typically de-repressed functional elements may make such elements more readily susceptible to stable de-repression.
In addition the host genome has had less time to adapt to the presence of these elements so their de-repression may be more disruptive than older elements.</p>
<p>Figure <a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#fig:modelSummaryTabsMain">6.1</a> summarises the details of Alu clocks trained with these various pre-filtering criteria applied to their starting sets.
The correlation with the predicted age and the chronological age of samples unrelated to the individuals on which the clock was trained indicate model quality (Figure <a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#fig:corplotsPanelSplit">6.3</a> a).
The Quality of the Alu clock predictions is quite poor in comparison to other DNA methylation clocks with the highest R of 0.65 and a median absolute error of 8.1 years, compared to 0.96 and 3.6 years for Horvath’s multi-tissue clock <span class="citation">[<a href="#ref-Horvath2013" role="doc-biblioref">138</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:modelSummaryTabsMain"></span>
<img src="figs/model_summary_tabs_main.png" alt="Alu Clock Model Summaries A) Models fitted with Training 1 set predicting Replication 1 ages. B) Models fitted with Training 2 set predicting Replication 2 ages. Alu Type Composition Colour code: AluY, AluS, AluJ, FAM, FLAM, FRAM. R is Pearson’s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="100%" />
<p class="caption">
Figure 6.1: <strong>Alu Clock Model Summaries</strong> <em>A)</em> Models fitted with Training 1 set predicting Replication 1 ages. <em>B)</em> Models fitted with Training 2 set predicting Replication 2 ages. Alu Type Composition Colour code: <span style="     color: #e41a1c !important;">AluY</span>, <span style="     color: #377eb8 !important;">AluS</span>, <span style="     color: #4daf4a !important;">AluJ</span>, <span style="     color: #984ea3 !important;">FAM</span>, <span style="     color: #ff7f00 !important;">FLAM</span>, <span style="     color: #ffff33 !important;">FRAM</span>. R is Pearson’s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<p>Figure <a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#fig:modelSummaryTabsCtrl">6.2</a> shows the model summaries when correlations are computed for for the related sets comprised of the twins of individuals in the training set.
The correlations are mostly slightly higher for this prediction than for the prediction of unrelated individuals (Figure <a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#fig:corplotsPanelSplit">6.3</a> b).
This suggests that there are not large genetic effects on the models as their predictive efficacy is only marginally better when employed to predict the age of individuals closely related to the training set compared to unrelated individuals.</p>
<div class="figure" style="text-align: center"><span id="fig:modelSummaryTabsCtrl"></span>
<img src="figs/model_summary_tabs_ctrl.png" alt="Summaries of Alu Clock Models Predicting Twin Groups for the purpose of comparing predictive of age models in related and unrelated samples. A) Models fitted with Training 1 set predicting Replication 2 ages. B) Models fitted with Training 2 set predicting Replication 1 ages. Alu Type Composition Colour code: AluY, AluS, AluJ, FAM, FLAM, FRAM. R is Pearson’s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="100%" />
<p class="caption">
Figure 6.2: <strong>Summaries of Alu Clock Models Predicting Twin Groups</strong> for the purpose of comparing predictive of age models in related and unrelated samples. <em>A)</em> Models fitted with Training 1 set predicting Replication 2 ages. <em>B)</em> Models fitted with Training 2 set predicting Replication 1 ages. Alu Type Composition Colour code: <span style="     color: #e41a1c !important;">AluY</span>, <span style="     color: #377eb8 !important;">AluS</span>, <span style="     color: #4daf4a !important;">AluJ</span>, <span style="     color: #984ea3 !important;">FAM</span>, <span style="     color: #ff7f00 !important;">FLAM</span>, <span style="     color: #ffff33 !important;">FRAM</span>. R is Pearson’s correlation coefficient. RMSE = root mean squared Error, MAE = median absolute error. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:corplotsPanelSplit"></span>
<img src="figs/corplots_panel_split.png" alt="Correlation of Predicted with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="100%" />
<p class="caption">
Figure 6.3: <strong>Correlation of Predicted with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

</div>
<div id="comparison-to-other-dna-methylation-clocks" class="section level2">
<h2><span class="header-section-number">6.2</span> Comparison to other DNA methylation clocks</h2>
<p>To compare the Alu clock results with those of other DNA methylation age clocks DNAm age was predicted for the: Horvath <span class="citation">[<a href="#ref-Horvath2013" role="doc-biblioref">138</a>]</span>, Hannum et al. <span class="citation">[<a href="#ref-Hannum2013" role="doc-biblioref">137</a>]</span> and Levine et al. ‘PhenoAge’ <span class="citation">[<a href="#ref-Levine2018" role="doc-biblioref">157</a>]</span> clocks (<a href="Alus.html#AluMethodsArrays">Methods</a> <a href="Alus.html#AluMethodsArrays">5.3.6</a>) using samples for which both DNA methylation array data and MeDIP-seq data was available (n = 574).
The array based clocks all perform similarly with R values in excess of 0.8, the Alu clock only attains an R of 0.46, though with a narrower error than PhenoAge.
This however, is not entirely surprising given that PhenoAge aims to capture ageing phenotypes and not narrowly chronological age.</p>
<div class="figure" style="text-align: center"><span id="fig:ageVsPredage"></span>
<img src="figs/age_vs_pred_age.png" alt="Correlation of Alu Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set." width="90%" />
<p class="caption">
Figure 6.4: <strong>Correlation of Alu Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:aluclockageVsPredageSingle"></span>
<img src="figs/alu_clock_age_vs_pred_age_single.png" alt="Correlation of Alu Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set." width="90%" />
<p class="caption">
Figure 6.5: <strong>Correlation of Alu Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set.
</p>
</div>

<div id="alu-age-acceleration" class="section level3">
<h3><span class="header-section-number">6.2.1</span> Alu Age Acceleration</h3>
<p>The age acceleration values (Chronological - Predicted Age) for the Alu clocks are strongly correlated with chronological age (Figure <a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#fig:accelCorplotsPanelSplit">6.6</a>), unlike the age acceleration of the Horvath DNA methylation clock and others (Figure <a href="zhou-xiang-and-matthew-stephens-genome-wide-efficient-mixed-model-analysis-for-association-studies-nature-genetics-44-7-2012-821-824-.html#fig:ageVsAgeAccelAluVsOther">6.7</a>).
The older the in individual and more prone their age is to be overestimated and vice versa.
This means that Alu clock age acceleration values cannot be used independently of chronological age to make statements about biological age.</p>
<div class="figure" style="text-align: center"><span id="fig:accelCorplotsPanelSplit"></span>
<img src="figs/accel_corplots_panel_split.png" alt="Correlation of Alu Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="90%" />
<p class="caption">
Figure 6.6: <strong>Correlation of Alu Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:ageVsAgeAccelAluVsOther"></span>
<img src="figs/age_vs_age_accel_alu_vs_other.png" alt="Correlation of Age Acceleration with Chronological Age Across Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="90%" />
<p class="caption">
Figure 6.7: <strong>Correlation of Age Acceleration with Chronological Age Across Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<p>Age acceleration in the Alu clocks is not strongly correlated with age acceleration in the Horvath, Hannun or PhenoAge clocks.</p>
<div class="figure" style="text-align: center"><span id="fig:aluClockAccelVsOthers"></span>
<img src="figs/alu_clock_accel_vs_others.png" alt="Correlation of Alu Age Accelerations with Age Acceleration from Other Models MAE = median absolute error. Orange = High \(R^2\), purple = low \(R^2\). Alu Clock here refers to the unfiltered model trained on the Training 1 set. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus." width="60%" />
<p class="caption">
Figure 6.8: <strong>Correlation of Alu Age Accelerations with Age Acceleration from Other Models</strong> MAE = median absolute error. Orange = High <span class="math inline">\(R^2\)</span>, purple = low <span class="math inline">\(R^2\)</span>. Alu Clock here refers to the unfiltered model trained on the Training 1 set. ‘+’ signifies this filter is in addition to element length &gt;180bp and 99.9% non-zero RPM values at this locus.
</p>
</div>

<div style="page-break-after: always;"></div>
</div>
</div>
<div id="discussion" class="section level2">
<h2><span class="header-section-number">6.3</span> Discussion</h2>
<p>The criteria for being called an epigenetic clock layed out by Horvath &amp; Raj <span class="citation">[<a href="#ref-Horvath2018" role="doc-biblioref">168</a>]</span> are that an r &gt; 0.8 should be achieved by the clock.
By this metric the Alu age predictors generated here should not be considered epigenetic clocks.</p>
<p>The age acceleration of the Alu age predictors in strongly correlate with age.
This may be due the the relatively poor ability of MeDIP-seq to provide absolute quantitation of DNA methylation.
There is substantial technical noise in the value representing DNA methylation at a given locus.
The observed systematic bais towards underestimating the ages of the young and overesimating the ages of the old may be explained by a broadly consistent direction with an inconsistent magnitude.
Simply scaling the value of given locus by a fixed model coefficient when there is a large amount of variation leads to outsized of high magnitude outliers on the age estimates.</p>
<p>It was an aim of this work to investigate the genetic influences on the Alu clock based measures of age-acceleration, as has been with shown previously with an association in the TERT gene with the Horvath clock <span class="citation">[<a href="#ref-Lu2017" role="doc-biblioref">398</a>]</span>, <span class="citation">[<a href="#ref-Gibson2019" role="doc-biblioref">399</a>]</span>.
These results had the potential to reveal mechanisms involved in controlling the epigenetic state of this large portion of the genome.
However the strong relationship with Age acceleration and Chronological age in the Alu clock measures of age acceleration renders this difficult as age would be a potent confounder of age acceleration in this instance.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Irizarry2009">
<p>55. Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P, Cui H, Gabo K, Rongione M, Webster M, Ji H, Potash J, Sabunciyan S, Feinberg AP: <strong>The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.</strong> <em>Nature genetics</em> 2009, <strong>41</strong>:178–186<a href="https://doi.org/10.1038/ng.298">10.1038/ng.298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151715">http://www.ncbi.nlm.nih.gov/pubmed/19151715</a>.</p>
</div>
<div id="ref-Stadler2011">
<p>62. Stadler MB, Murr R, Burger L, Ivanek R, Lienert F, Schöler A, Nimwegen E van, Wirbelauer C, Oakeley EJ, Gaidatzis D, Tiwari VK, Schübeler D: <strong>DNA-binding factors shape the mouse methylome at distal regulatory regions.</strong> <em>Nature</em> 2011, <strong>480</strong>:490–5<a href="https://doi.org/10.1038/nature10716">10.1038/nature10716</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22170606">http://www.ncbi.nlm.nih.gov/pubmed/22170606</a>.</p>
</div>
<div id="ref-Hannum2013">
<p>137. Hannum G, Guinney J, Zhao L, Zhang L, Hughes G, Sadda S, Klotzle B, Bibikova M, Fan J-B, Gao Y, Deconde R, Chen M, Rajapakse I, Friend S, Ideker T, Zhang K: <strong>Genome-wide methylation profiles reveal quantitative views of human aging rates.</strong> <em>Molecular cell</em> 2013, <strong>49</strong>:359–367<a href="https://doi.org/10.1016/j.molcel.2012.10.016">10.1016/j.molcel.2012.10.016</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23177740">http://www.ncbi.nlm.nih.gov/pubmed/23177740</a>.</p>
</div>
<div id="ref-Horvath2013">
<p>138. Horvath S: <strong>DNA methylation age of human tissues and cell types.</strong> <em>Genome biology</em> 2013, <strong>14</strong>:R115<a href="https://doi.org/10.1186/gb-2013-14-10-r115">10.1186/gb-2013-14-10-r115</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24138928">http://www.ncbi.nlm.nih.gov/pubmed/24138928</a>.</p>
</div>
<div id="ref-Levine2018">
<p>157. Levine ME, Lu AT, Quach A, Chen BH, Assimes TL, Bandinelli S, Hou L, Baccarelli AA, Stewart JD, Li Y, Whitsel EA, Wilson JG, Reiner AP, Aviv A, Lohman K, Liu Y, Ferrucci L, Horvath S: <strong>An epigenetic biomarker of aging for lifespan and healthspan</strong>. <em>Aging</em> 2018, <strong>10</strong>:573–591<a href="https://doi.org/10.18632/aging.101414">10.18632/aging.101414</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29676998">http://www.ncbi.nlm.nih.gov/pubmed/29676998</a>.</p>
</div>
<div id="ref-Horvath2018">
<p>168. Horvath S, Raj K: <strong>DNA methylation-based biomarkers and the epigenetic clock theory of ageing</strong>. <em>Nature Reviews Genetics</em> 2018, <strong>19</strong>:371–384<a href="https://doi.org/10.1038/s41576-018-0004-3">10.1038/s41576-018-0004-3</a>Available: <a href="http://www.nature.com/articles/s41576-018-0004-3">http://www.nature.com/articles/s41576-018-0004-3</a>.</p>
</div>
<div id="ref-Snir2019">
<p>396. Snir S, Farrell C, Pellegrini M: <strong>Human epigenetic ageing is logarithmic with time across the entire lifespan</strong>. <em>Epigenetics</em> 2019, <strong>0</strong>:1–15<a href="https://doi.org/10.1080/15592294.2019.1623634">10.1080/15592294.2019.1623634</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634">https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634</a>.</p>
</div>
<div id="ref-Hellen2013">
<p>397. Hellen EH, Brookfield JF: <strong>Alu elements in primates are preferentially lost from areas of high GC content</strong>. <em>PeerJ</em> 2013, <strong>1</strong>:e78<a href="https://doi.org/10.7717/peerj.78">10.7717/peerj.78</a>Available: <a href="https://peerj.com/articles/78">https://peerj.com/articles/78</a>.</p>
</div>
<div id="ref-Lu2017">
<p>398. Lu AT, Hannon E, Levine ME, Crimmins EM, Lunnon K, Mill J, Geschwind DH, Horvath S: <strong>Genetic architecture of epigenetic and neuronal ageing rates in human brain regions</strong>. <em>Nature Communications</em> 2017, <strong>8</strong>:15353<a href="https://doi.org/10.1038/ncomms15353">10.1038/ncomms15353</a>Available: <a href="http://dx.doi.org/10.1038/ncomms15353">http://dx.doi.org/10.1038/ncomms15353</a>.</p>
</div>
<div id="ref-Gibson2019">
<p>399. Gibson J, Russ TC, Clarke T-K, Howard DM, Hillary RF, Evans KL, Walker RM, Bermingham ML, Morris SW, Campbell A, Hayward C, Murray AD, Porteous DJ, Horvath S, Lu AT, McIntosh AM, Whalley HC, Marioni RE: <strong>A meta-analysis of genome-wide association studies of epigenetic age acceleration</strong>. <em>PLOS Genetics</em> 2019, <strong>15</strong>:e1008104<a href="https://doi.org/10.1371/journal.pgen.1008104">10.1371/journal.pgen.1008104</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31738745">http://www.ncbi.nlm.nih.gov/pubmed/31738745</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="Alus.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "night",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
