<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1 Introduction | The Epigenomics of Human Ageing</title>
  <meta name="description" content="Thesis for the degree of Doctor of Philosophy" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="1 Introduction | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis for the degree of Doctor of Philosophy" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1 Introduction | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="Thesis for the degree of Doctor of Philosophy" />
  

<meta name="author" content="Richard J. Acton" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="methods.html"/>
<style type="text/css">
p.abstract{
  text-align: center;
  font-weight: bold;
}
div.abstract{
  margin: auto;
  width: 90%;
}
</style>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#IntroCpGdist"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#dna-methylation-clock-measurements-as-outcomes-for-anti-ageing-interventions"><i class="fa fa-check"></i><b>1.5.4</b> DNA methylation clock measurements as outcomes for anti-ageing interventions</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#GenMethodsMethArray"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#Participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#MethodsTargetedBisulfiteSequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a>
<ul>
<li class="chapter" data-level="3.1" data-path="arrays.html"><a href="arrays.html#arrayAbstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="arrays.html"><a href="arrays.html#arrayIntro"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="arrays.html"><a href="arrays.html#primary-questions"><i class="fa fa-check"></i><b>3.3</b> Primary Questions</a></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#arrayMethods"><i class="fa fa-check"></i><b>3.4</b> Methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.4.1</b> Outline of EWAS</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.4.2</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.4.3</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.4.4" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.4.4</b> EWAS Models</a></li>
<li class="chapter" data-level="3.4.5" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.4.5</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#arrayResults"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="arrays.html"><a href="arrays.html#combined-mavidos-summary-data"><i class="fa fa-check"></i><b>3.5.1</b> Combined MAVIDOS Summary data</a></li>
<li class="chapter" data-level="3.5.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.5.2</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.5.3" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.5.3</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.5.4" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.5.4</b> Southampton Women’s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="arrays.html"><a href="arrays.html#arrayDiscussion"><i class="fa fa-check"></i><b>3.6</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a>
<ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-abstract"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Introduction"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#primary-questions-1"><i class="fa fa-check"></i><b>4.3</b> Primary questions</a></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#tRNAmethods"><i class="fa fa-check"></i><b>4.4</b> Methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Participants"><i class="fa fa-check"></i><b>4.4.1</b> Participants</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Gene coordinates and other annotation information</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-DNA-methylome-Data"><i class="fa fa-check"></i><b>4.4.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.4.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.4.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.4.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.4.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.4.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.4.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.4.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.4.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.5.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.5.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.5.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.5.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.5.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.5.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.5.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.5.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.5.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Discussion"><i class="fa fa-check"></i><b>4.6</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>5</b> DNA methylation clocks in Alu Elements</a>
<ul>
<li class="chapter" data-level="5.1" data-path="Alus.html"><a href="Alus.html#Alu-Abstract"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="Alus.html"><a href="Alus.html#Alu-Methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="Alus.html"><a href="Alus.html#dna-methylome-data"><i class="fa fa-check"></i><b>5.3.1</b> DNA methylome data</a></li>
<li class="chapter" data-level="5.3.2" data-path="Alus.html"><a href="Alus.html#alu-element-annotation-and-pre-filtering"><i class="fa fa-check"></i><b>5.3.2</b> Alu element annotation and pre-filtering</a></li>
<li class="chapter" data-level="5.3.3" data-path="Alus.html"><a href="Alus.html#elastic-net-regression"><i class="fa fa-check"></i><b>5.3.3</b> Elastic net regression</a></li>
<li class="chapter" data-level="5.3.4" data-path="Alus.html"><a href="Alus.html#sampleSelection"><i class="fa fa-check"></i><b>5.3.4</b> Sample Selection</a></li>
<li class="chapter" data-level="5.3.5" data-path="Alus.html"><a href="Alus.html#binarisation"><i class="fa fa-check"></i><b>5.3.5</b> Binarisation</a></li>
<li class="chapter" data-level="5.3.6" data-path="Alus.html"><a href="Alus.html#AluMethodsArrays"><i class="fa fa-check"></i><b>5.3.6</b> Comparisons to other clocks in TwinsUK Illumina 450k array data</a></li>
<li class="chapter" data-level="5.3.7" data-path="Alus.html"><a href="Alus.html#genome-wide-association-studies-of-alu-age-acceleration"><i class="fa fa-check"></i><b>5.3.7</b> Genome Wide Association studies of Alu Age Acceleration</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="Alus.html"><a href="Alus.html#Alu-Results"><i class="fa fa-check"></i><b>5.4</b> Results</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="Alus.html"><a href="Alus.html#comparison-to-other-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.1</b> Comparison to other DNA methylation clocks</a></li>
<li class="chapter" data-level="5.4.2" data-path="Alus.html"><a href="Alus.html#alu-age-acceleration"><i class="fa fa-check"></i><b>5.4.2</b> Alu Age Acceleration</a></li>
<li class="chapter" data-level="5.4.3" data-path="Alus.html"><a href="Alus.html#top-results-genome-wide-association-studies-of-alu-age-acceleration-are-shared-by-alu-age-predictors"><i class="fa fa-check"></i><b>5.4.3</b> Top results Genome Wide Association studies of Alu Age Acceleration are shared by Alu Age predictors</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="Alus.html"><a href="Alus.html#Alu-Discussion"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="discussion.html"><a href="discussion.html#epigenomic-analysis-of-the-developmental-origins-of-long-term-bone-health"><i class="fa fa-check"></i><b>6.1</b> Epigenomic analysis of the developmental origins of long-term bone health</a></li>
<li class="chapter" data-level="6.2" data-path="discussion.html"><a href="discussion.html#the-implications-of-ageing-related-changes-in-the-epigenetic-state-of-trna-genes"><i class="fa fa-check"></i><b>6.2</b> The implications of ageing-related changes in the epigenetic state of tRNA genes</a></li>
<li class="chapter" data-level="6.3" data-path="discussion.html"><a href="discussion.html#assessing-biological-ageing-by-dna-methylation-changes-within-alu-repeat-elements"><i class="fa fa-check"></i><b>6.3</b> Assessing Biological ageing by DNA methylation changes within Alu repeat elements</a></li>
<li class="chapter" data-level="6.4" data-path="discussion.html"><a href="discussion.html#conclusion"><i class="fa fa-check"></i><b>6.4</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="appendices.html"><a href="appendices.html"><i class="fa fa-check"></i>Appendices</a>
<ul>
<li class="chapter" data-level="6.5" data-path="appendices.html"><a href="appendices.html#biorchi-iv-manuscript-the-genomic-loci-of-specific-human-trna-genes-exhibit-ageing-related-dna-hypermethylation"><i class="fa fa-check"></i><b>6.5</b> <span class="math inline">\(BioR\chi iv\)</span> manuscript: The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="6.6" data-path="appendices.html"><a href="appendices.html#apptRNAanno"><i class="fa fa-check"></i><b>6.6</b> Specific tRNA annotations used in the tRNA analyses</a></li>
<li class="chapter" data-level="6.7" data-path="appendices.html"><a href="appendices.html#apptRNAprimers"><i class="fa fa-check"></i><b>6.7</b> tRNA targeted bisulfite sequencing primers</a></li>
<li class="chapter" data-level="6.8" data-path="appendices.html"><a href="appendices.html#apptRNAmods"><i class="fa fa-check"></i><b>6.8</b> Additional tRNA Age modelling results</a></li>
<li class="chapter" data-level="6.9" data-path="appendices.html"><a href="appendices.html#apptRNAmouse"><i class="fa fa-check"></i><b>6.9</b> tRNA mouse models</a></li>
<li class="chapter" data-level="6.10" data-path="appendices.html"><a href="appendices.html#apptRNAprop"><i class="fa fa-check"></i><b>6.10</b> tRNA proportional enrichment fisher’s test count data</a></li>
<li class="chapter" data-level="6.11" data-path="appendices.html"><a href="appendices.html#appTCGA"><i class="fa fa-check"></i><b>6.11</b> TCGA sample used in tRNA analysis</a></li>
<li class="chapter" data-level="6.12" data-path="appendices.html"><a href="appendices.html#apptwins450ktRNA"><i class="fa fa-check"></i><b>6.12</b> Twins UK 450k array tRNA age models</a></li>
<li class="chapter" data-level="6.13" data-path="appendices.html"><a href="appendices.html#appRXRACDKN2A"><i class="fa fa-check"></i><b>6.13</b> MAVIDOS RXRA and CDKN2A probe models</a></li>
<li class="chapter" data-level="6.14" data-path="appendices.html"><a href="appendices.html#appAluageSNPs"><i class="fa fa-check"></i><b>6.14</b> Age acceleration GWAS SNPs</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>The Epigenomics of Human Ageing</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<div id="the-ageing-population-and-the-burden-of-disease" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> The Ageing Population and The Burden of Disease</h2>
<p>The global population is ageing as is evident from the changing shape of the world population pyramid (figure <a href="introduction.html#fig:poppyramid">1.1</a>).
The increase in average lifespan underlying this demographic shift is projected to continue in industrialised nations with a probability of &gt;65% for women and 85% for men <span class="citation">[<a href="references.html#ref-Kontis2017" role="doc-biblioref">2</a>]</span> (figure <a href="introduction.html#fig:Kontis2017-fig1">1.2</a>).
Ageing is the main risk factor for cancer, cardiovascular, neurodegenerative diseases, and many other conditions including osteoarthritis <span class="citation">[<a href="references.html#ref-Niccoli2012" role="doc-biblioref">3</a>]</span>. Furthermore, mortality from these conditions increases according to a logistic function with age <span class="citation">[<a href="references.html#ref-Belikov2019" role="doc-biblioref">4</a>]</span> (figure <a href="introduction.html#fig:Belikov2019-fig1">1.3</a>).
In addition to the non-infectious diseases, immunosenescence and frailty in old age contribute to increased mortality from infectious diseases, as exemplified by dramatically higher hospital admissions and deaths for respiratory disease during seasonal flu epidemics in the over 75s <span class="citation">[<a href="references.html#ref-Fleming2005" role="doc-biblioref">5</a>]</span> (figure <a href="introduction.html#fig:Fleming2005-fig3">1.4</a>).
Ageing is a common underlying risk factor for many conditions and as this would lead one to expect the number of concomitant disorders and the proportion of persons with multiple morbidities increases with age <span class="citation">[<a href="references.html#ref-Barnett2012" role="doc-biblioref">6</a>]</span> (figure <a href="introduction.html#fig:Barnett2012-fig1">1.5</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:poppyramid"></span>
<img src="figs/population-pryamids-1950-2019.png" alt="The Population is Getting Older Population pyramids for global population for A) 1950 and B) 2019 from populationpyramid.net [7]" width="95%" />
<p class="caption">
Figure 1.1: <strong>The Population is Getting Older</strong> Population pyramids for global population for A) 1950 and B) 2019 from <a href="populationpyramid.net">populationpyramid.net</a> <span class="citation">[<a href="references.html#ref-PopulationPyramid2019" role="doc-biblioref">7</a>]</span>
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Kontis2017-fig1"></span>
<img src="figs/Kontis2017_fig1.jpg" alt="The Population is Likely to Continue Getting Older Median projected increase in life expectancy from birth in the period 2010 to 2030 for 35 industrial nations reproduced from Kontis et al. 2017 figure 1 [2]." width="95%" />
<p class="caption">
Figure 1.2: <strong>The Population is Likely to Continue Getting Older</strong> Median projected increase in life expectancy from birth in the period 2010 to 2030 for 35 industrial nations reproduced from Kontis et al. 2017 figure 1 <span class="citation">[<a href="references.html#ref-Kontis2017" role="doc-biblioref">2</a>]</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Belikov2019-fig1"></span>
<img src="figs/Belikov2019_fig1.jpg" alt="Mortality from Age-Related Conditions Increases According to a Logistic Function with Age Mortality rate in deaths per 100,000 at different ages for Atherosclerosis, Essential Hypertension, Insulin-dependent diabetes, Non-insulin-dependent diabetes, Alzheimer’s disease, and Parkinson’s disease. Showing Model fits for exponential, logistic and the sum of 2 logistic functions. Underlying Cause of Death data (1999-2015) were form the Centers for Disease Control and Prevention, Wide-ranging OnLine Data for Epidemiologic Research (CDC WONDER) database. Reproduced from Belikov 2019 figure 1 [4]." width="80%" />
<p class="caption">
Figure 1.3: <strong>Mortality from Age-Related Conditions Increases According to a Logistic Function with Age</strong> Mortality rate in deaths per 100,000 at different ages for Atherosclerosis, Essential Hypertension, Insulin-dependent diabetes, Non-insulin-dependent diabetes, Alzheimer’s disease, and Parkinson’s disease. Showing Model fits for exponential, logistic and the sum of 2 logistic functions. Underlying Cause of Death data (1999-2015) were form the Centers for Disease Control and Prevention, Wide-ranging OnLine Data for Epidemiologic Research (CDC WONDER) database. Reproduced from Belikov 2019 figure 1 <span class="citation">[<a href="references.html#ref-Belikov2019" role="doc-biblioref">4</a>]</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Fleming2005-fig3"></span>
<img src="figs/Fleming2005_fig3.jpg" alt="Mortality from Infectious Diseases Increases With Age Visits to General practitioners, hospital admissions and deaths from respiratory disease in persons aged: (a) 45-64, (b) 65-74 and (c) 75 years and over. Reproduced from Fleming et al. 2005 figure 3 [5]." width="90%" />
<p class="caption">
Figure 1.4: <strong>Mortality from Infectious Diseases Increases With Age</strong> Visits to General practitioners, hospital admissions and deaths from respiratory disease in persons aged: (a) 45-64, (b) 65-74 and (c) 75 years and over. Reproduced from Fleming et al. 2005 figure 3 <span class="citation">[<a href="references.html#ref-Fleming2005" role="doc-biblioref">5</a>]</span>.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Barnett2012-fig1"></span>
<img src="figs/Barnett2012_fig1.jpg" alt="Number and Frequency of Comorbidities increase with Age Number of chronic disorders by age group. reproduced from Barnett et al. 2012 figure 1 [6]." width="80%" />
<p class="caption">
Figure 1.5: <strong>Number and Frequency of Comorbidities increase with Age</strong> Number of chronic disorders by age group. reproduced from Barnett et al. 2012 figure 1 <span class="citation">[<a href="references.html#ref-Barnett2012" role="doc-biblioref">6</a>]</span>.
</p>
</div>

<p>Given the rapid increases in the rates of numerous diseases and the rate of death from those diseases as we age, there is a clear impetus to develop interventions to prevent and treat age-related diseases.
The common factor among all these conditions is their relationship to age, but age is not merely a shared risk factor but frequently the largest; outside of rare genetic conditions which predispose to early-onset <span class="citation">[<a href="references.html#ref-Niccoli2012" role="doc-biblioref">3</a>]</span>.
Thus understanding the underlying biological mechanisms of the ageing process and intervening in those mechanisms, has the potential to be vastly more effective and economical <span class="citation">[<a href="references.html#ref-Goldman2016" role="doc-biblioref">8</a>]</span> than attempting to tackle diseases individually.
The discovery in 1993 by Cynthia Kenyon and others <span class="citation">[<a href="references.html#ref-Kenyon1993" role="doc-biblioref">9</a>]</span> that a single mutation in the <em>daf-2</em> gene of <em>C. elegans</em> could double the lifespan of this organism is often cited as an inflexion point in the confidence of the ageing research field that interventions in the process of ageing itself may be a realistic therapeutic target. What then do we know about the underlying biology of ageing?</p>
</div>
<div id="the-biology-of-ageing" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> The biology of Ageing</h2>
<p>Ageing is characterised by a progressive deterioration of physiological integrity with time, resulting in an increased risk of mortality and morbidity.
Theories of how and why this occurs are numerous with a history in the scientific literature stretching back to the late 1800s <span class="citation">[<a href="references.html#ref-Medvedev1990" role="doc-biblioref">10</a>]</span>, setting aside, for now, theoretical frameworks for the process of ageing we will first consider the nature of these changes.
Changes at the organismal level have their origins in changes at the molecular and cellular scales. Lopez-Otin et al. 2013 <span class="citation">[<a href="references.html#ref-Lopez-Otin2013" role="doc-biblioref">11</a>]</span> identified nine hallmarks of ageing in their 2013 review.
These hallmarks meet the criteria of:
Manifesting during normal ageing;
Accelerating ageing when aggravated;
Retarding ageing when ameliorated.
The hallmarks are further subdivided and hierarchically arranged into primary, antagonistic and integrative categories.
Primary hallmarks are of unequivocally negative effect.
Antagonistic hallmarks arise as a response to the primary and are initially protective against these changes but become problems in their own right as they occur at higher levels over time.
Lastly, the integrative hallmarks directly impact on tissue function and homoeostasis.
The hallmarks are as follows:</p>
<p><strong>Primary Hallmarks - Causes of Damage</strong></p>
<ol style="list-style-type: decimal">
<li><p><em>Genomic instability</em>:
Damage to the genomes of our cells occurs over time and accumulates as we age, this damage ranges from point mutations to large structural mutations, as well as mutations in the mitochondrial genome <span class="citation">[<a href="references.html#ref-Niedernhofer2018" role="doc-biblioref">12</a>]</span>.</p></li>
<li><p><em>Telomere attrition</em>:
Telomere shortening occurs during the ageing process, somatic cells do not generally express telomerase so with each division telomeres grow shorter.
Telomere exhaustion limits the replicative capacity of cells inducing replicative senescence, limiting regenerative potential but also protecting from the uncontrolled growth of cancer <span class="citation">[<a href="references.html#ref-Casagrande2019" role="doc-biblioref">13</a>]</span>.</p></li>
<li><p><strong>Epigenetic alterations</strong>:
Patterns of DNA methylation and of a number of histone modifications exhibit age-related changes. These contribute to the dysregulation of gene expression and derepression of heterochromatin regions which also increases the risk of genomic instability <span class="citation">[<a href="references.html#ref-Saldanha2015" role="doc-biblioref">14</a>]</span>.</p></li>
<li><p><em>Loss of proteostasis</em>:
Production of protein folding chaperones in response to stress decreases with age, as does the activity of the two major proteolytic systems, the ubiquitin-proteasome system and the autophagy-lysosome system.
This leads to increased prevalence of misfolded, aberrant and aggregated proteins with age <span class="citation">[<a href="references.html#ref-Santra2019" role="doc-biblioref">15</a>]</span>.</p></li>
</ol>
<p><strong>Antagonistic Hallmarks - Responses to damage</strong></p>
<ol start="5" style="list-style-type: decimal">
<li><p><em>Deregulated nutrient sensing</em>:
The IIS (insulin &amp; insulin-like growth signalling) and TOR (Target Of Rapamycin) pathways signal nutrient abundance, promoting anabolism.
The AMPK (5’ adenosine monophosphate-activated protein kinase) and Sirtuins signal nutrient scarcity, promoting catabolism.
Broadly speaking inhibiting the pro-anabolic pathways and activating the pro-catabolic pathways can extend lifespan <span class="citation">[<a href="references.html#ref-Johnson2018" role="doc-biblioref">16</a>,<a href="references.html#ref-Soultoukis2016" role="doc-biblioref">17</a>]</span>.</p></li>
<li><p><em>Mitochondrial dysfunction</em>:
Mitochondrial ATP production drops off with and ROS (Reactive Oxygen Species) production increases. Elevated ROS do not appear to accelerate ageing nor do elevated antioxidants retard it.
Indicating the primary function of ROS in ageing is not straightforwardly a result of oxidative damage, but may result from its role in stress signalling.
Dysfunctional mitochondria exhibit an increased propensity to polarise in response to stress, impacting on apoptotic signalling and inflammatory responses.
This contributes to the increases in senescent cells and their chronic inflammatory phenotype <span class="citation">[<a href="references.html#ref-Payne2015" role="doc-biblioref">18</a>]</span>.</p></li>
<li><p><em>Cellular senescence</em>:
Senescent cells are in stable cell-cycle arrest induced by shortened telomeres, other forms of DNA damage, INK4/ARF derepression, or a variety of mitogenic/oncogenic signals.
Induction of a senescent state in pre-cancerous cells protects against cancer and senescent cells are initially cleared by the immune system.
However, these cells accumulate in tissue as we age without contributing to function and with a pro-inflammatory secretory phenotype that promotes chronic inflammation <span class="citation">[<a href="references.html#ref-Avelar2020" role="doc-biblioref">19</a>]</span>.</p></li>
</ol>
<p><strong>Integrative Hallmarks - Proximal Causes of the Ageing phenotype</strong></p>
<ol start="8" style="list-style-type: decimal">
<li><p><em>Stem cell exhaustion</em>:
Adult stem cells capable of producing new cells for the regeneration of tissues undergo asymmetric divisions to replace themselves and produce replicative progenitor cells to replenish tissue.
Stem cells may accumulate damage and cease replicating or leave their quiescent state becoming overly replicative and potentially becoming senescent.
This disrupts the supply of new cells for the renewal of tissue <span class="citation">[<a href="references.html#ref-Ren2017" role="doc-biblioref">20</a>]</span>.</p></li>
<li><p><em>Altered intercellular communication</em>:
The aforementioned cell-autonomous alterations alter external signalling behaviour of cells across intercellular signalling paradigms.
This can lead to feedback loops that exacerbate the problem. The high background of pro-inflammatory signals from senescent cells dilutes already declining immune response and fewer senescent cells are cleared <span class="citation">[<a href="references.html#ref-Franceschi2014" role="doc-biblioref">21</a>]</span>.</p></li>
</ol>
<p>Many genetic changes have been associated with changes in longevity since Cynthia Kenyon’s characterisation of the <em>daf-2</em> mutation which approximately doubled the lifespan of <em>C. elegans</em> <span class="citation">[<a href="references.html#ref-Kenyon1993a" role="doc-biblioref">22</a>–<a href="references.html#ref-Singh2019" role="doc-biblioref">24</a>]</span>.</p>
<p>In Humans Genome Wide Association Studies have identified relatively few genes associated with longevity <span class="citation">[<a href="references.html#ref-Broer2015" role="doc-biblioref">25</a>,<a href="references.html#ref-Zenin2019" role="doc-biblioref">26</a>]</span>.
APOE is perhaps the most consistently identified, followed by FOXO3A, however FOXO3A appears not to be as predictive for the majority of the population and is only associated with longevity in the very long lived <span class="citation">[<a href="references.html#ref-Melzer2019" role="doc-biblioref">27</a>]</span>.
The portion of variance in longevity attributable to genetics may adhere to the emerging pattern of the genetic contibution to common diseases being due to a multitude of common variants with small effects.
With rarer mutations having more extreme effects on longevity <span class="citation">[<a href="references.html#ref-Claussnitzer2020" role="doc-biblioref">28</a>]</span>.
There is also an effect of ultra-rare/private protein-truncating variants, a high burden of which negatively affects longevity <span class="citation">[<a href="references.html#ref-Shindyapina2020" role="doc-biblioref">29</a>]</span>.</p>
<p>This work and therefore the following review of the literature focuses on the 3rd hallmark, epigenetic alterations, specifically the role of DNA methylation in ageing.
Booth &amp; Brunet <span class="citation">[<a href="references.html#ref-Booth2016" role="doc-biblioref">30</a>]</span> contend that epigenetic processes are a hub through which all of the other hallmarks of ageing are mediated and feed-back on one another (Figure <a href="introduction.html#fig:Booth2016fig3">1.6</a>).
One of these interactions is between DNA damage and chromatin state including DNA methylation.
Sinclair and Oberdoerffer <span class="citation">[<a href="references.html#ref-Sinclair2009" role="doc-biblioref">31</a>]</span> make the case that DNA damage and repair processes are disruptive to the epigenome and, with Hayano et al., link double-stranded DNA breaks to accelerated epigenetic ageing in a recent pre-print <span class="citation">[<a href="references.html#ref-Hayano2019" role="doc-biblioref">32</a>]</span>.
Kane &amp; Sinclair <span class="citation">[<a href="references.html#ref-Kane2019" role="doc-biblioref">33</a>]</span> make the case that reprogramming of the epigenome to a ‘younger’ state is a promising mode of intervention in the ageing process.
Whilst loss of genetic information is effectively irreversible, barring gene therapy-like interventions, loss of epigenetic information is not necessarily so.
Dysregulation from the loss of epigenetic information is arguably proximate to age-related changes and may run ahead of significant genetic information loss such that intervening in the former may have beneficial effects in its own right and stave off the latter.</p>
<div class="figure" style="text-align: center"><span id="fig:Booth2016fig3"></span>
<img src="figs/Booth2016_F3_epigenomeHubAgingHallmarks_alpha.png" alt="Epigenetic Changes as a hub for the hallmarks of ageing. (Reproduced from Booth &amp; Brunet [30], figure 3)" width="60%" />
<p class="caption">
Figure 1.6: Epigenetic Changes as a hub for the hallmarks of ageing. (Reproduced from Booth &amp; Brunet <span class="citation">[<a href="references.html#ref-Booth2016" role="doc-biblioref">30</a>]</span>, figure 3)
</p>
</div>

<p>Prior to examining the specific relationship of epigenetic changes to ageing we take a step back to define and outline epigenomics and examine the fundamentals of DNA methylation biology.</p>
</div>
<div id="epigenomics---overview" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Epigenomics - Overview</h2>
<p>Epigenetics generally refers to modifications to DNA and chromatin which do not affect the primary sequence of DNA bases <span class="citation">[<a href="references.html#ref-Bird2007" role="doc-biblioref">34</a>]</span>, but which are to varying degrees stable and heritable.
The term derives from epigenesis and genetics originating with Conrad Waddington <span class="citation">[<a href="references.html#ref-Waddington1942" role="doc-biblioref">35</a>]</span>.
Epigenesis refers to the idea that organisms develop through the progressive differentiation of cells from the egg into adult tissues.
As modern genetics revealed that every cell contained a complete copy of the genome which was differentially utilised by the cells of adult organisms, the terms were fused to reflect the study of how this occurs.
The usage has evolved a little further since molecular biology began to elucidate the mechanisms involved in this process and the term now frequently refers to the study of these mechanisms and their effects in less explicitly developmental contexts.
Arthur Riggs et al. considered that heritability should be a criterion for a mark to be considered epigenetic, but this excludes many phenomena now commonly referred to using the term <span class="citation">[<a href="references.html#ref-Russo1996" role="doc-biblioref">36</a>]</span>.
Requiring heritability results in a further definitional dispute over degrees of heritability; mitotic, meiotic, intergenerational, transgenerational and to what degree of fidelity?
Adrian Bird proposed the definition: “the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.” as a useful compromise <span class="citation">[<a href="references.html#ref-Bird2007" role="doc-biblioref">34</a>]</span>.
Epigenomics refers to the totality of the epigenetic modifications present in a particular cell, tissue type or genome.
This definition appears to have caught on with at least some as: ‘These days, “epigenetics” more generally refers to all non-genomic information storage in cells including gene networks, chromatin structure and post-translational modifications to histones.’ according to Alice Kane and David Sinclair <span class="citation">[<a href="references.html#ref-Kane2019" role="doc-biblioref">33</a>]</span>.</p>
<p>Whilst an organism can generally be thought of as having a single genome, with exceptions such as Somatic mutations and Chimerism, it will have at least as many epigenomes as it has cell types <span class="citation">[<a href="references.html#ref-Tabansky2015" role="doc-biblioref">37</a>]</span>.
There are on the order of <span class="math inline">\(10^{13}\)</span> cells in the human body <span class="citation">[<a href="references.html#ref-Bianconi2013" role="doc-biblioref">38</a>]</span>.
Estimates of the number of cell types vary with the resolution at which one deems cells functionally distinct <span class="citation">[<a href="references.html#ref-Regev2017" role="doc-biblioref">39</a>]</span>.
At this point in time, however, there is no clear definition of what constitutes a distinct cell type, indeed they are beginning to be defined by the distinct patterns of gene expression and epigenetic modifications they exhibit <span class="citation">[<a href="references.html#ref-CellSystemsCellType2017" role="doc-biblioref">40</a>]</span>.
Consequently, the number epigenomes that could be considered distinct will likely ultimately approximate to the number of epigenomes which can usefully distinguish between sub-populations of cells.
A consortium has been established to produce a <a href="www.humancellatlas.org">Human Cell Atlas</a> <span class="citation">[<a href="references.html#ref-Regev2017" role="doc-biblioref">39</a>]</span> which aims to define all human cell types, and a cell type ontology <span class="citation">[<a href="references.html#ref-CellOntologyOBO" role="doc-biblioref">41</a>]</span> exists.</p>
<p>There are a number of epigenetic modifications which can be subdivided into four broad categories:</p>
<ol style="list-style-type: decimal">
<li><p><strong>DNA modifications</strong></p>
<p>In mammals DNA methylation primarily takes place on Cytosine residues.
Cytosine methylation occurs principally in an mCpG sequence context but can also occur in mCpH, mCHG and mCHH (H=A/C/T) contexts; particularly in cells of the nervous system <span class="citation">[<a href="references.html#ref-Guo2014a" role="doc-biblioref">42</a>]</span> and embryonic stem cells where as many as 25% of all cytosines can be methylated in non-canonical contexts.
In contrast to methylation in a differentiated non-neuronal tissue such as foetal lung fibroblasts which is 99.98% in CpG context <span class="citation">[<a href="references.html#ref-Lister2009" role="doc-biblioref">43</a>,<a href="references.html#ref-Schultz2015" role="doc-biblioref">44</a>]</span>.
5-methylcytosine (5mc) can be oxidised to produce another modified DNA base 5-hydroxymethylcytosine (5hmC) <span class="citation">[<a href="references.html#ref-Penn1972" role="doc-biblioref">45</a>,<a href="references.html#ref-Kriaucionis2009" role="doc-biblioref">46</a>]</span>.
5hmC may act as an intermediate to demethylation and potentially has regulatory functions in its own right <span class="citation">[<a href="references.html#ref-Pfeifer2013" role="doc-biblioref">47</a>]</span>.
5hmC can be oxidised further to formylcytosine (5fC) and carboxylcytosine (5caC) <span class="citation">[<a href="references.html#ref-Ito2011" role="doc-biblioref">48</a>]</span>.
The extent to which these modifications are stable and functional is still being explored.
Other DNA bases can be methylated such as N6-methyladenine, but this occurs at a substantially lower frequency that 5mC and much less is known about their potential functions <span class="citation">[<a href="references.html#ref-Wu2016" role="doc-biblioref">49</a>]</span>.
N6-methyladenine may play a role in condensing chromatin <span class="citation">[<a href="references.html#ref-Xie2018" role="doc-biblioref">50</a>]</span>.</p></li>
<li><p><strong>Histone tail modifications</strong></p>
<p>Histone proteins form disk-shaped octamers around which ~150bp of DNA can be wrapped to form a nucleosome.
The ‘tails’ are generally the N termini of the histone proteins, outside of the core globular domains, which protrude from the nucleosome structure <span class="citation">[<a href="references.html#ref-Losick2014" role="doc-biblioref">51</a>]</span>.
Histone tails can be subject to post-translational modification which commonly takes place at lysine residues, other residues are subject to modifications but lysines are among the best characterised.
Examples of modifications include methylation with between one and three methyl groups per lysine, Acetylations, Ubiquitylation, SUMOylations, and phosphorylations <span class="citation">[<a href="references.html#ref-Bannister2011" role="doc-biblioref">52</a>]</span>.
Many of these marks can be generally classified as permissive or repressive but are frequently found in combinations of marks of opposing effect, rendering the interpretation of the ‘Histone Code’ extremely challenging <span class="citation">[<a href="references.html#ref-Strahl2000" role="doc-biblioref">53</a>–<a href="references.html#ref-Voigt2012" role="doc-biblioref">55</a>]</span>.
One class of approaches to interpreting chromatin state is segmentation, this covers a variety of methods for combining mostly ChIP data for an array of histone modifications generated by large consortia projects such as ENCODE <span class="citation">[<a href="references.html#ref-Moore2020" role="doc-biblioref">56</a>]</span>.
Segmentation makes use of pattern discovery techniques such as Hidden Markov Models to divide the genome into discrete sections assigning these sections to a set of categories which can then be compared to existing annotations to provide functional labels <span class="citation">[<a href="references.html#ref-Hoffman2012" role="doc-biblioref">57</a>,<a href="references.html#ref-Carrillo-de-Santa-Pau2017" role="doc-biblioref">58</a>]</span>.
When successful the pattern discovery algorithm independently recapitulates our existing ontology of functional elements and hopefully provides new insights by for example highlighting regions not previously considered to be in a given functional category.
This is a “top-down” approach making use of the “sum” of available chromatin state data rather than trying to interpret the significance of a single type of mark.</p></li>
<li><p><strong>Histone Variants</strong></p>
<p>Alternatives to the canonical histones can be substituted in the nucleosome, examples of such alternative histones include H2A.Z and H3.3 <span class="citation">[<a href="references.html#ref-Talbert2012" role="doc-biblioref">59</a>]</span>.
Alternative histones can alter chromatin structure and dynamics by altering nucleosome stability, binding different factors, and presenting a different substrate for histone modifications, thus altering their pattern <span class="citation">[<a href="references.html#ref-Weber2014" role="doc-biblioref">60</a>]</span>.</p></li>
<li><p><strong>Some non-coding RNAs (ncRNA)</strong></p>
<p>A subset of long ncRNAs, particularly those which persist in close association with chromatin can fall within the definition of epigenetic features <span class="citation">[<a href="references.html#ref-Kaikkonen2011" role="doc-biblioref">61</a>]</span>.
An example of such a long non-coding RNA (lncRNA) is Xist which is a key regulator of X inactivation <span class="citation">[<a href="references.html#ref-Cerase2015" role="doc-biblioref">62</a>]</span>.
The catagorisation of these as epigenetic is somewhat controversial, as they are not direct chemical modifications to chromatin. However Xist, for example, facilitates the mitotically heritable repression of the inactivate X chromosome satisfying other criteria for being considered an epigenetic modification.</p></li>
</ol>
</div>
<div id="fundamentals-of-dna-methylation" class="section level2" number="1.4">
<h2><span class="header-section-number">1.4</span> Fundamentals of DNA Methylation</h2>
<div id="structure-and-context" class="section level3" number="1.4.1">
<h3><span class="header-section-number">1.4.1</span> Structure and Context</h3>
<p>DNA methylation is the most well-studied epigenetic modification <span class="citation">[<a href="references.html#ref-Haluskova2010" role="doc-biblioref">63</a>]</span>.
Specifically CpG methylation, the addition of a Methyl group to the 5 carbon of a cytosine base (figure <a href="introduction.html#fig:5mCvsC">1.7</a>), in a CpG dinucleotide context.
(hereafter ‘DNA Methylation’ or ‘DNAm’ refers to 5mC CpG methylation unless stated otherwise.)
DNA methylation is stable and relatively easy to characterise from biological samples by comparison with other epigenetic marks.
The methylation status of ancient DNA has even been characterised, for example, native American remains ranging in age from 230 to 4500 years were measured using bisulfite sequencing <span class="citation">[<a href="references.html#ref-Smith2015" role="doc-biblioref">64</a>]</span>.
C undergoes spontaneous hydrolytic deamination over time to U, and 5mC to T.
once sufficient time has passed for the deamination process to convert the majority of the 5mC sites, this can be used to infer the methylation state of post-mortem DNA using DNA polymerases that cannot bypass uracils and thus only causes C <span class="math inline">\(\rightarrow\)</span> T misincorporation events at former 5mC loci.
This approach was employed to estimate the age at death of a a 4000 year of old Paleo-Eskimo belonging to the Saqqaq culture <span class="citation">[<a href="references.html#ref-Pedersen2014" role="doc-biblioref">65</a>]</span>.</p>
<p>5mC is quite chemically stable such that for most tissue sample preservation techniques for which DNA can be extracted from the sample DNAm can also be characterised <span class="citation">[<a href="references.html#ref-Briggs2010" role="doc-biblioref">66</a>]</span>.
DNA extracted with normal laboratory methods can be assayed for DNAm.
Histone modification status, on the other hand, is less robust against environmental stresses on samples and often requires considerably more laborious sample preparation to characterise <span class="citation">[<a href="references.html#ref-Bauden2017" role="doc-biblioref">67</a>]</span>.
DNA methylation also offers the appearance of relative simplicity in comparison to the complex picture of the ‘histone code,’ the simple binary nature of DNA methylation state makes it easier to model and possibly to interpret.
However, interaction between DNAm and histone modifications is well documented <span class="citation">[<a href="references.html#ref-Hashimshony2003" role="doc-biblioref">68</a>–<a href="references.html#ref-Rose2014" role="doc-biblioref">70</a>]</span>, this interplay between DNAm and histone modifications renders the interpretation of DNAm more complex, as its effects may be conditional on the chromatin environment.</p>
<div class="figure" style="text-align: center"><span id="fig:5mCvsC"></span>
<img src="figs/5mC_vs_C.png" alt="The structure of 5-methylcytosine contrasted with Cytosine. (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 1.7: The structure of 5-methylcytosine contrasted with Cytosine. (Figure created by the Author.)
</p>
</div>

</div>
<div id="IntroCpGdist" class="section level3" number="1.4.2">
<h3><span class="header-section-number">1.4.2</span> Distribution and Global Trends</h3>
<p>There are 28,299,634 CpGs <span class="citation">[<a href="references.html#ref-Luo2014" role="doc-biblioref">71</a>]</span> in the hg19 assembly of the human genome <span class="citation">[<a href="references.html#ref-Lander2001" role="doc-biblioref">72</a>]</span>.
Given that the GC content of the human genome is <span class="math inline">\(42\%\)</span> the prior probability of getting a CpG dinucleotide is: <span class="math inline">\(0.21 \times 0.21 = 0.0441\)</span>, <span class="math inline">\(4.41\%\)</span>.
CpGs represent <span class="math inline">\(\sim1.8\%\)</span> of the dinucleotides in the human genome ( <span class="math inline">\(28,299,634 \div (3.23\times10^9\div2) \approx 0.0175\)</span>).
Making them <span class="math inline">\(\sim2.5\)</span> fold less frequent than would be expected <em>a priori</em>.
Methylated cytosines are prone to deamination to thymines, resulting in mismatch lesions where a T is opposite a G.
This can result in C to T mutations when DNA repair processes resolve this mismatch by changing the G to an A instead of correcting the T back to a C <span class="citation">[<a href="references.html#ref-Duncan1980" role="doc-biblioref">73</a>]</span>.
This increased mutagenic potential generally means that CpGs are selected against, accounting for at least some part of their under-representation in the genome.
Methylated cytosines had a mutation rate in sperm of 18.5% compared to 5.44% for unmethylated CpGs <span class="citation">[<a href="references.html#ref-Zhou2020" role="doc-biblioref">74</a>]</span>, and C <span class="math inline">\(\rightarrow\)</span> T transitions account for ~33.1% of all SNPs <span class="citation">[<a href="references.html#ref-Guo2005" role="doc-biblioref">75</a>]</span>.
The three major classes of repeat elements SINEs, LINEs and LTRs contain some <span class="math inline">\(46\%\)</span> of all CpG sites, with a further <span class="math inline">\(5\%\)</span> in other repetitive elements <span class="citation">[<a href="references.html#ref-Luo2014" role="doc-biblioref">71</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:CGIs"></span>
<img src="figs/CGI-Shores-Shelves-Seas_2.png" alt="Schematic representation of CpG Islands, shores, shelves and seas. CpG density declines from shores to seas and methylation increases (CpG density and methylation proportion not to scale). (Figure created by the Author.)" width="80%" />
<p class="caption">
Figure 1.8: Schematic representation of CpG Islands, shores, shelves and seas. CpG density declines from shores to seas and methylation increases (CpG density and methylation proportion not to scale). (Figure created by the Author.)
</p>
</div>

<p>CpGs are not uniformly distributed in the genome, they occur at higher frequency in some regions.
“CpG islands” or CGIs are regions of high CpG density.
They are defined somewhat arbitrarily, the definition used by UCSC is a sliding 500bp window <span class="citation">[<a href="references.html#ref-Wu2010" role="doc-biblioref">76</a>]</span>.
There have existed other definitions <span class="citation">[<a href="references.html#ref-Gardiner-Garden1987" role="doc-biblioref">77</a>]</span>.
The total number of CpGs in the UCSC repeat masked CGI annotation list of 28,691 CGIs is 1,990,729.
Therefore, these CpGs comprise <span class="math inline">\(\sim7.0\%\)</span> of the total number of CpGs the genome. The mean percentage of the sequence of these CGIs that is comprised of CpG dinucleotides is <span class="math inline">\(\sim18.5\%\)</span> and their mean length is 761bp.
CGIs overlap the promoters of <span class="math inline">\(\sim70\%\)</span> of genes <span class="citation">[<a href="references.html#ref-Saxonov2006" role="doc-biblioref">78</a>]</span>, this, however, leaves roughly half of all CGIs as “orphans” not associated with a known transcription site.
These orphan CGIs frequently constitute cell-type specific enhancers <span class="citation">[<a href="references.html#ref-Bell2017" role="doc-biblioref">79</a>]</span> and alternate promoters <span class="citation">[<a href="references.html#ref-Illingworth2010" role="doc-biblioref">80</a>,<a href="references.html#ref-Maunakea2010" role="doc-biblioref">81</a>]</span>.
They are found in both intergenic and intragenic regions.
CpGs are distributed quite sparsely through the genome occurring at low density in “seas” and at increasing density in CGIs and their flanking regions, see figure <a href="introduction.html#fig:CGIs">1.8</a>.
It is a characteristic of CGIs that they are generally unmethylated.</p>
<p>Early work showed 70-80% of CpGs are constitutively methylated <span class="citation">[<a href="references.html#ref-Ehrlich1982" role="doc-biblioref">82</a>,<a href="references.html#ref-Bird2002" role="doc-biblioref">83</a>]</span> Stadler et al. produced a more detailed picture of the distribution of CpG methylation in mouse embryonic stem cells (ESCs) <span class="citation">[<a href="references.html#ref-Stadler2011" role="doc-biblioref">84</a>]</span> see figures <a href="introduction.html#fig:Baubec2014fig1">1.9</a> &amp; <a href="introduction.html#fig:CpGfreqVsDNAm">1.10</a>.
Stadler et al. categorised regions of the genome into fully, low and un methylated regions (FMRs, LMRs, UMRs) in mouse embryonic stem cells using a hidden markov model.
LMRs were evolutionarily conserved and highly enriched for the chromatin signatures of distal regulatory regions such as enhancers (High H3K4me1 relative to H3K4me3, p300, H3K27ac over H3K27me3) <span class="citation">[<a href="references.html#ref-Stadler2011" role="doc-biblioref">84</a>]</span>.
Irizarry et al. found that the 2kb regions flanking CpG islands which they termed “CpG island shores” exhibited greater tissue-specific differential methylation than the islands themselves <span class="citation">[<a href="references.html#ref-Irizarry2009" role="doc-biblioref">85</a>]</span>.
This nomenclature has subsequently been expanded further with “CpG island shelves” which are 2kb - 4kb from the CGIs, and “seas” referring to the rest of the genome, see figure <a href="introduction.html#fig:CGIs">1.8</a>.
Ziller et al. identified ~5.6 million CpGs, ~21.8% of autosomal CpGs, which are dynamically regulated across diverse cell-types, these clustered into ~716,000 differentially methylated regions.
More that three quarters of these regions where under 1kb in size and located away from transcription start sites and &gt;70% of which had methylation levels &gt;75% <span class="citation">[<a href="references.html#ref-Ziller2013" role="doc-biblioref">86</a>]</span>.
Data from three different approaches taken by Irizarry et al. <span class="citation">[<a href="references.html#ref-Irizarry2009" role="doc-biblioref">85</a>]</span>, Stadler et al. <span class="citation">[<a href="references.html#ref-Stadler2011" role="doc-biblioref">84</a>]</span>, and Ziller et al. <span class="citation">[<a href="references.html#ref-Ziller2013" role="doc-biblioref">86</a>]</span> indicate that the most dynamic DNA methylation changes are occurring at the level of small distal regulatory features.
A small fraction of CpGs, on the order of 2-6% exhibiting constitutively unmethylated states and on the order of 70-90% constitutively methylated leaving 4-28% of sites subject to dynamic regulation.</p>
<div class="figure" style="text-align: center"><span id="fig:Baubec2014fig1"></span>
<img src="figs/Baubec2014_fig1.png" alt="Stylised representation of the relationship between CpG Density, CpG methylation and functional DNA elements. (Figure reproduced from Baubec &amp; Schubeler [87] figure 1)" width="80%" />
<p class="caption">
Figure 1.9: Stylised representation of the relationship between CpG Density, CpG methylation and functional DNA elements. (Figure reproduced from Baubec &amp; Schubeler <span class="citation">[<a href="references.html#ref-Baubec2014" role="doc-biblioref">87</a>]</span> figure 1)
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:CpGfreqVsDNAm"></span>
<img src="figs/CpGfreqVsDNAm.png" alt="The fraction of CpGs with a given methylation level. Data from Whole-Genome Bisulfite Sequencing (WGBS) of Mouse embryonic stem cells. Colours denote: unmethylated regions (blue), low-methylated regions (red), and fully methylated regions (green). These groups are three segments produced by a hidden markov model. low-methylated regions with 10-50% methylation are evolutionarily conserved distal regulatory elements with dynamic cell-type specific regulation unlike the consistently unmethylated regions at CGIs. (Figure reproduced from Stadler et al. [84] figure 1a)" width="80%" />
<p class="caption">
Figure 1.10: The fraction of CpGs with a given methylation level. Data from Whole-Genome Bisulfite Sequencing (WGBS) of Mouse embryonic stem cells. Colours denote: unmethylated regions (blue), low-methylated regions (red), and fully methylated regions (green). These groups are three segments produced by a hidden markov model. low-methylated regions with 10-50% methylation are evolutionarily conserved distal regulatory elements with dynamic cell-type specific regulation unlike the consistently unmethylated regions at CGIs. (Figure reproduced from Stadler et al. <span class="citation">[<a href="references.html#ref-Stadler2011" role="doc-biblioref">84</a>]</span> figure 1a)
</p>
</div>

<p>The regulatory role played by DNA methylation varies with context and the scale at which it is examined.
For example methylation at a single CpG in a transcription factor binding site can determine whether or not the factor can bind at that site.
There are methylation-sensitive transcription factors which can have their affinity for DNA either increased or decreased by methylation <span class="citation">[<a href="references.html#ref-Yin2017" role="doc-biblioref">88</a>,<a href="references.html#ref-Zuo2017" role="doc-biblioref">89</a>]</span>.
In addition, larger-scale changes in DNAm can through interactions with chromatin modifiers alter DNA compaction and more general accessibility of DNA regions for binding, and/or impacting on their topological organisation.
DNAm also varies with nucleosome occupancy with lower methylation in linker sequences than on nucleosome-associated DNA <span class="citation">[<a href="references.html#ref-Kelly2012" role="doc-biblioref">90</a>,<a href="references.html#ref-Huff2014" role="doc-biblioref">91</a>]</span>.
The oxidation products of 5mC can also affect chromatin organisation, 5-Formylcytosine can impact nucleosome positioning through covalently binding to histones <span class="citation">[<a href="references.html#ref-Raiber2017" role="doc-biblioref">92</a>]</span>.
This raises the question if some DNA methylation occurs specifically to act as an intermediate step for regulatory mechanisms which utilise its oxidation products.</p>
</div>
<div id="pathways-of-dna-methylation-and-demethylation" class="section level3" number="1.4.3">
<h3><span class="header-section-number">1.4.3</span> Pathways of DNA methylation and demethylation</h3>
<p>CpG methylation is produced and maintained by DNA methyltransferase (DNMT) enzymes.
All the DNMT enzymes use S-adenosylmethionine as the source of the methyl donor group.
DNMTs form a covalent intermediate between a conserved cysteine residue and the target base, through a nucleophilic attack on the C6 position in the cytosine ring <span class="citation">[<a href="references.html#ref-Lyko2017" role="doc-biblioref">93</a>]</span>.
This is followed by the transfer of the S-adenosylmethionine methyl group to C5, and deprotonation of the C5 to reform the double bond between C5 and C6; which is mediated by a base provided by the enzyme (Figure <a href="introduction.html#fig:Lyko2017DNAmMechanism">1.11</a>).</p>
<p>DNMT1 is associated with the replication machinery <span class="citation">[<a href="references.html#ref-Vertino2002" role="doc-biblioref">94</a>]</span> and reproduces the methylation state of the parent strand on the daughter strand during replication.
The largest of the DNMT family DNMT1 contains a ‘replication foci targeting sequence’ (RFTS) domain required for its targeting to replication forks.
DNMT1 specialises in recognising hemimethylated DNA and methylating the unmethylated C in a palindromic CpG dinucleotide site, the UHRF1 protein assists in the recognition of these sequences <span class="citation">[<a href="references.html#ref-Bostick2007" role="doc-biblioref">95</a>]</span>.
DNMT3a and DNMT3b are responsible for <em>de novo</em> DNA methylation along with DNMT3L a catalytically inactive, but DNA-binding subunit <span class="citation">[<a href="references.html#ref-Jia2009" role="doc-biblioref">96</a>]</span>.
The location of <em>de novo</em> methylation by the DNMT3s is influenced by a number of factors including the Chromatin state and other DNA binding factors <span class="citation">[<a href="references.html#ref-Lyko2017" role="doc-biblioref">93</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Lyko2017DNAmMechanism"></span>
<img src="figs/Lyko2017DNAmMechanism.png" alt="Motif VI (ENV/PCQ) refers to the conserved catalytic motif in DNMTs. ‘B:’ Represents the enzyme supplied base (Figure reproduced from the review of DNMT biology by Lyko [93] (fig. 1b))" width="80%" />
<p class="caption">
Figure 1.11: Motif VI (ENV/PCQ) refers to the conserved catalytic motif in DNMTs. ‘B:’ Represents the enzyme supplied base (Figure reproduced from the review of DNMT biology by Lyko <span class="citation">[<a href="references.html#ref-Lyko2017" role="doc-biblioref">93</a>]</span> (fig. 1b))
</p>
</div>

<p>DNMT2 is unlike the other members of the DNMT family in that it targets an RNA substrate.
DNMT2 is a tRNA methyltransferase which methylates a specific subset of tRNA genes, mostly Asp isoacceptors, at a site adjacent to the anticodon which protects them from endonucleolytic cleavage under stress conditions.
Unprotected tRNAs produce fragments which compete with small interfering RNAs interfering with their signalling pathways <span class="citation">[<a href="references.html#ref-Kaiser2017" role="doc-biblioref">97</a>]</span>.</p>
<p>The mechanisms of the demethylation of DNA were more recently characterised than those which govern its methylation <span class="citation">[<a href="references.html#ref-Tahiliani2009" role="doc-biblioref">98</a>,<a href="references.html#ref-Ito2011" role="doc-biblioref">48</a>]</span>.
They are more complex and less well understood.
TET2 mutations are quite common in cancer, ranking 65th across 12 major cancer types <span class="citation">[<a href="references.html#ref-Kandoth2013" role="doc-biblioref">99</a>]</span> and more common in diverse myeloid malignancies, where loss of its catalytic activity favours tumourigenesis <span class="citation">[<a href="references.html#ref-Ko2010a" role="doc-biblioref">100</a>]</span>.
A distinction is drawn between active and passive demethylation, in passive demethylation, 5mC bases are diluted out in the process of DNA replication.
In the leading model of active demethylation, they are oxidised one or more times by an enzyme from the TET (Ten-Eleven-translocase) family.
They are then either passively removed by DNA replication or actively removed by a DNA glycosylase (thymine DNA glycosylase TGD) to create an apurinic site which is restored to a C by the Base Excision Repair (BER) pathway. This cycle of cytosine methylation and demethylation is illustrated in Figure <a href="introduction.html#fig:Wu2017TETsDNAmCycle">1.12</a> <span class="citation">[<a href="references.html#ref-Wu2017" role="doc-biblioref">101</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Wu2017TETsDNAmCycle"></span>
<img src="figs/Wu2017TETsDNAmCycle.png" alt="BER = Bases Excision Repair; AM = Active Modification; AR = Active Removal, PD = Passive Dilution (Figure reproduced from the review of TET mediated active demethylation by Wu &amp; Zhang [101] (fig. 1a))" width="60%" />
<p class="caption">
Figure 1.12: BER = Bases Excision Repair; AM = Active Modification; AR = Active Removal, PD = Passive Dilution (Figure reproduced from the review of TET mediated active demethylation by Wu &amp; Zhang <span class="citation">[<a href="references.html#ref-Wu2017" role="doc-biblioref">101</a>]</span> (fig. 1a))
</p>
</div>

</div>
<div id="maintenance-and-fidelity" class="section level3" number="1.4.4">
<h3><span class="header-section-number">1.4.4</span> Maintenance and Fidelity</h3>
<p>DNA methylation is highly dependent on the underlying DNA sequence.
Sequence features of particular importance to determining DNA methylation status are; transcription factor and other DNA binding protein recognition motifs, and CpG density <span class="citation">[<a href="references.html#ref-Lienert2011" role="doc-biblioref">102</a>,<a href="references.html#ref-Ziller2013" role="doc-biblioref">86</a>]</span>.
Alterations in underlying DNA sequence such as SNPs and copy number variants (CNVs) can have a significant impact on methylation level and the susceptibility of the methylation level to change.
In addition, CNVs can result in dosage effects on measures of DNAm, causing regions to appear, respectively, substantially more or less methylated when fewer or greater copies than expected are present <span class="citation">[<a href="references.html#ref-Boks2009" role="doc-biblioref">103</a>–<a href="references.html#ref-Martin-Trujillo2017" role="doc-biblioref">108</a>]</span>.
Despite the strong influence of sequence on methylation, global CpG methylation exhibits change over developmental time as illustrated in figure <a href="introduction.html#fig:DNAmDevTime">1.13</a> and exhibits tissue-specific changes in distribution and amount <span class="citation">[<a href="references.html#ref-Ziller2013" role="doc-biblioref">86</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:DNAmDevTime"></span>
<img src="figs/Ciernia2016f3DNAmTime.PNG" alt="5mC levels over developmental time. 5mCH or 5-methylcytosine-(A, T or C) levels are represented on a separate (purple) Axis from the main 5mCG axis. PGC = Primordial Germ Cell. (Figure reproduced from [109] figure 3)" width="80%" />
<p class="caption">
Figure 1.13: 5mC levels over developmental time. 5mCH or 5-methylcytosine-(A, T or C) levels are represented on a separate (purple) Axis from the main 5mCG axis. PGC = Primordial Germ Cell. (Figure reproduced from <span class="citation">[<a href="references.html#ref-Ciernia2016" role="doc-biblioref">109</a>]</span> figure 3)
</p>
</div>

<p>In mitosis, DNA methylation is inherited by daughter cells with an error rate on the order of <span class="math inline">\(1\times10^{-3}\)</span> per site per generation <span class="citation">[<a href="references.html#ref-Ushijima2005" role="doc-biblioref">110</a>]</span>, several orders of magnitude less than that of DNA replication (error rate of <span class="math inline">\(1\times10^{-7}\)</span> - <span class="math inline">\(1\times10^{-8}\)</span> <span class="citation">[<a href="references.html#ref-Kunkel2004a" role="doc-biblioref">111</a>]</span>).
DNMT1 methylates hemimethylated DNA at CpG sites in S phase copying the DNA methylation state of the parent strand onto the daughter strand.
DNMT1 provides the catalytic activity and is precisely targeted to hemimethylated DNA in conjunction with the UHRF1 (ubiquitin-like, containing PHD and RING finger domains 1) protein which specifically binds hemimethylated CpG sites via its SRA (SET and ring-associated) domain using a flip-out mechanism to recognise 5mC
<span class="citation">[<a href="references.html#ref-Sharif2007" role="doc-biblioref">112</a>,<a href="references.html#ref-Bostick2007" role="doc-biblioref">95</a>,<a href="references.html#ref-Arita2008" role="doc-biblioref">113</a>,<a href="references.html#ref-Liu2013b" role="doc-biblioref">114</a>]</span>.</p>
<p>The fidelity of DNA methylation copying can be assayed by Hairpin-Bisulfite PCR (Polymerase Chain Reaction) <span class="citation">[<a href="references.html#ref-Laird2004" role="doc-biblioref">115</a>]</span>.
Laird et al. looked at two alleles of a portion of the CpG island from the human <em>FMR1</em> gene in uncultured lymphocytes, one hypermethylated and one hypomethylated.
In the hypermethylated allele, they found that 96% of sites methylated in the parent strand remained methylated in the daughter strand and 86% of unmethylated sites remained unmethylated.
By contrast in the hypomethylated allele, there were no methylated sites to be retained and &gt;99% of unmethylated sites remained so following replication.
Laird et al.’s work suggests a higher degree of overall methylation fidelity for hypomethylated DNA and a propensity for unmethylated sites in hypermethylated DNA to become methylated.</p>
<p>Change in DNAm levels over time (divisions) can be modelled using the differential equations <span class="citation">[<a href="references.html#ref-Pfeifer1990a" role="doc-biblioref">116</a>]</span>, which predict that a fully methylated site and a fully unmethylated site will converge on an equilibrium level.
This level is determined by the probability of maintenance of the methylation state and of <em>de novo</em> methylation for a given locus.
This stochastic model of DNAm is in agreement with experimental findings <span class="citation">[<a href="references.html#ref-Laird2004" role="doc-biblioref">115</a>,<a href="references.html#ref-Riggs2004a" role="doc-biblioref">117</a>]</span>, Figure <a href="introduction.html#fig:Riggs2004PNASfig2DNAmFidelity">1.14</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:Riggs2004PNASfig2DNAmFidelity"></span>
<img src="figs/Riggs2004PNASfig2DNAmFidelity.png" alt="\(E_m\) = probability of methylation maintenance, \(E_m\) = probability of de novo methylation. \(M\) &amp; \(U\) = the number of methylated and unmethylated molecules at specific CpG sites respectively. Modelling (Grey points) \(E_m,E_d\) values of 0.90,0.10; 0.95,0.05; 0.97,0.03 (outermost to innermost, two curves for each \(E_m,E_d\) set, one starting at \(M=1,U=0\) and one a \(M=0,U=1\). Experimental data (Black points) from the HpaII locus (site H3) in 17 clones of mouse cell line BML-2 which has a known methylation level of 50%. (Reproduced from Riggs &amp; Xiong [117])" width="70%" />
<p class="caption">
Figure 1.14: <span class="math inline">\(E_m\)</span> = probability of methylation maintenance, <span class="math inline">\(E_m\)</span> = probability of <em>de novo</em> methylation. <span class="math inline">\(M\)</span> &amp; <span class="math inline">\(U\)</span> = the number of methylated and unmethylated molecules at specific CpG sites respectively. Modelling (<strong>Grey points</strong>) <span class="math inline">\(E_m,E_d\)</span> values of 0.90,0.10; 0.95,0.05; 0.97,0.03 (outermost to innermost, two curves for each <span class="math inline">\(E_m,E_d\)</span> set, one starting at <span class="math inline">\(M=1,U=0\)</span> and one a <span class="math inline">\(M=0,U=1\)</span>. Experimental data (<strong>Black points</strong>) from the <em>HpaII</em> locus (site H3) in 17 clones of mouse cell line BML-2 which has a known methylation level of 50%. (Reproduced from Riggs &amp; Xiong <span class="citation">[<a href="references.html#ref-Riggs2004a" role="doc-biblioref">117</a>]</span>)
</p>
</div>

<p>Jenkinson et al. <span class="citation">[<a href="references.html#ref-Jenkinson2017" role="doc-biblioref">118</a>]</span> used an information-theoretic approach, modelling DNAm as a binary communications channel using a 1-dimensional Ising model from the field of statistical physics.
This permitted them to examine properties of DNAm not accessible to conventional means of analysis which typically capture the mean methylation level and perhaps the variability for a given locus.
When considering methylation fidelity using this lens the maintenance of a given methylation state can be seen as an information processing task which requires the consumption of free energy in order to reduce the probability of error in the transmission of that methylation state.
Thus Relative Dissipated Energy (RDE) can serve as a measure of the work expended by a cell at a given locus in order to preserve the current methylation state of that locus.
They found that the Transcription Start Sites (TSS) exhibited high levels of RDE and high information capacity as well as low levels of entropy.
These values trended in the opposite direction as they moved outwards to CGIs, shores and seas.
Jenkinson et al. <span class="citation">[<a href="references.html#ref-Jenkinson2017" role="doc-biblioref">118</a>]</span> also found a global increase in entropy with age, but not with cell passage in culture suggesting an increase in entropy is associated with epigenetic age independent of mitotic age.
This observation is in agreement with the finding that epigenetic state becomes more stochastic and diverges with age <span class="citation">[<a href="references.html#ref-Fraga2005" role="doc-biblioref">119</a>,<a href="references.html#ref-Slieker2016" role="doc-biblioref">120</a>]</span>, indeed genes whose methylation were most divergent with age were enriched for ageing associations.
They also observed a loss of entropic sensitivity with age.
Entropic sensitivity is an indicator of how plastic DNAm state is to extrinsic effects.
Jenkinson et al. noted that there is a general loss of phenotypic plasticity with age, but cited no specific instance of loss of DNAm plasticity to environmental effects with age.
Hahn et al. <span class="citation">[<a href="references.html#ref-Hahn2017" role="doc-biblioref">121</a>]</span> provided a possible example of this effect when they reported that dietary restriction caused fewer differences in the methylation state of older than younger mice.</p>
<p>Zhao et al <span class="citation">[<a href="references.html#ref-Zhao2014" role="doc-biblioref">122</a>]</span> applied Hairpin-Bisulfite PCR genome-wide in mouse embryonic stem cells (ESCs).
They also found high degrees of methylation fidelity in hypomethylated regions such as CGIs and Promoters, as well as a high degree of fidelity in sites bound by transcription factors.
This is in agreement with the findings of Jenkinson et al. <span class="citation">[<a href="references.html#ref-Jenkinson2017" role="doc-biblioref">118</a>]</span> who noted that entropy (methylation stochasticity) was lower and more variable in CGIs and TSS (transcription start sites).</p>
<p>Methylation inheritance fidelity is reduced in cancer <span class="citation">[<a href="references.html#ref-Ushijima2005" role="doc-biblioref">110</a>]</span> and increases with differentiation <span class="citation">[<a href="references.html#ref-Zhao2014" role="doc-biblioref">122</a>]</span>.
The RDE (relative dissipated energy) at CGIs and TSSs is higher in differentiated tissues such as the brain, implying low entropy, and lower in embryonic stem cells, implying greater entropy <span class="citation">[<a href="references.html#ref-Jenkinson2017" role="doc-biblioref">118</a>]</span> (see Figure <a href="introduction.html#fig:Zhao2014fig3ab">1.15</a>).
In addition, the correlation between CpG sites increases in cancer <span class="citation">[<a href="references.html#ref-Jenkinson2017" role="doc-biblioref">118</a>]</span>, suggesting reduced higher-level regulatory control and tendency to fall back on lower level feedback mechanisms leading to a return to the baseline equilibrium described above as the cell is expending less energy to maintain an out of equilibrium methylation state.</p>
<div class="figure" style="text-align: center"><span id="fig:Zhao2014fig3ab"></span>
<img src="figs/Zhao2014-fig3-ab.png" alt="Bimodal Distribution of DNA methylation level and fidelity. Methylation fidelity exhibits a bimodal distribution with most and least methylated regions exhibiting the highest fidelity. Additionally, fidelity for methylation levels of 10-50% is considerably better than fidelity for levels of 50-90%. These data are from the mouse cell line ES-E14TG2a which is capable of self-renewal when cultured with Leukaemia Inhibitory Factor (LIF) and spontaneous differentiation upon removal of LIF (denoted as E14-d0 at day 0 and E14-d6 at day 6 after the withdrawal of LIF, respectively). Fidelity represents the percentage of symmetrically methylated or unmethylated CpG dyads for a given position as determined by Hairpin-Bisulfite PCR. (Reproduced from Zhao et al. 2014 [122] (figure 3 a and b).)" width="80%" />
<p class="caption">
Figure 1.15: Bimodal Distribution of DNA methylation level and fidelity. Methylation fidelity exhibits a bimodal distribution with most and least methylated regions exhibiting the highest fidelity. Additionally, fidelity for methylation levels of 10-50% is considerably better than fidelity for levels of 50-90%. These data are from the mouse cell line ES-E14TG2a which is capable of self-renewal when cultured with Leukaemia Inhibitory Factor (LIF) and spontaneous differentiation upon removal of LIF (denoted as E14-d0 at day 0 and E14-d6 at day 6 after the withdrawal of LIF, respectively). Fidelity represents the percentage of symmetrically methylated or unmethylated CpG dyads for a given position as determined by Hairpin-Bisulfite PCR. (Reproduced from Zhao et al. 2014 <span class="citation">[<a href="references.html#ref-Zhao2014" role="doc-biblioref">122</a>]</span> (figure 3 a and b).)
</p>
</div>

<p>Zhou et al. linked the substantial loss of DNA methylation in proliferative cancer cells, even counterintuitively those cancers with high DNMT1/UHRF1, DNMT3A/B levels and physiological levels of TETs, to a systematic mechanism of DNA methylation loss.
Partially methylated domains (PMDs) are characterised by low CpG density, larger numbers of lone CpGs flanked on both sides by A/T nucleotides, and low levels of H3K36me3 as well as being lamina-associated late replicating domains.
PMDs are prone to becoming hypomethylated in a manner which progressively increases with mitotic division.
Copying of the parent strand methylation state by DNMT1/UHRF1 takes time and may not be fully completed with each round of replication leading to progressive DNA methylation loss in these regions, especially if proliferation is rapid.
Some late replicating regions have greater methylation fidelity through the action of DNMT3A/B at sites with the H3K36me3 mark <span class="citation">[<a href="references.html#ref-Zhou2018" role="doc-biblioref">123</a>]</span>.</p>
</div>
<div id="dna-methylation-assays" class="section level3" number="1.4.5">
<h3><span class="header-section-number">1.4.5</span> DNA Methylation Assays</h3>
<p>There are numerous DNA methylation assays.
Some are designed to assay global DNAm levels producing a single global measurement of the amount of 5mC present in a sample, others assay DNAm in a manner traceable to a specific genomic locus.
These methods can be further subdivided into targeted methods where the loci to be examined are known in advance, and those where the loci covered are quasi-randomly sampled.
Both the targeted and untargeted methods vary in their granularity, from individual CpGs to large regions of the genome, on the order of megabases.
These assays also vary in their coverage of the genome from locus-specific approaches looking at small numbers of individual loci to whole-genome methods providing methylation information on all ~28 million CpG sites.
Between these two extremes are ‘epigenome-wide’ methods which are focused with varying degrees of specificity on genomic regions of interest.</p>
<p>These DNA methylation assays generally exploit one of four following principles:</p>
<ol style="list-style-type: decimal">
<li><p><strong>Methylation sensitive restriction digestion</strong>
A variety of methods exploit enzymes which differentially cut methylated and unmethylated DNA.
An example of such enzymes is: HpaII and MspI which are isoschizomers for the sequence: 5’-C^CGG-3’ but only HpaII is methylation-sensitive and unable to cut the sequence if the internal cytosine is methylated.
The methods which use such enzymes include: Restriction landmark genomic scanning, which gives a roughly megabase resolution indicators of DNAm levels by 2D electrophoresis <span class="citation">[<a href="references.html#ref-Hatada1991" role="doc-biblioref">124</a>]</span>.
MRE-seq (Methylation sensitive restriction enzyme sequencing), which makes use of methylation-sensitive restriction digests to enrich for unmethylated DNA <span class="citation">[<a href="references.html#ref-Maunakea2010" role="doc-biblioref">81</a>]</span>.
This is then sequenced to provide the genomic location of this unmethylated signal.
Reduced Representation Bisulfite Sequencing (RRBS), which uses methylation-sensitive restriction digestion to enrich for unmethylated sequences which are then subject to bisulfite sequencing, (see point 3 below).</p></li>
<li><p><strong>DNA methylation-sensitive binding of DNA by antibodies or other proteins selectively which bind methylated DNA</strong>
Anti-5mC antibodies were first used to isolate methylated DNA in 1985 and were subsequently paired with array-based technologies to assay DNAm levels at specific loci in 2005 <span class="citation">[<a href="references.html#ref-Weber2005" role="doc-biblioref">125</a>,<a href="references.html#ref-Harrison2011" role="doc-biblioref">126</a>]</span>.
This was followed by MeDIP-seq (methylated DNA immunoprecipitation and sequencing) <span class="citation">[<a href="references.html#ref-Down2009" role="doc-biblioref">127</a>,<a href="references.html#ref-Lienhard2014" role="doc-biblioref">128</a>]</span>, which enriches methylated DNA that is then sequenced and the number of reads mapping to a locus is indicative of the relative methylation level.
MBD-seq uses a recombinant Methyl Binding Domain (MBD) protein to enrich for methylated double-stranded DNA, prior to sequencing <span class="citation">[<a href="references.html#ref-Serre2009a" role="doc-biblioref">129</a>]</span>.
Inferring the absolute methylation levels form the number of reads mapping to a locus enriched by one of these pull-down methods is complicated by variation in CpG density in the genome and the fact that CpG dense regions tend to have low methylation levels, and thus tend not to be pulled down by antibodies binding 5mC or MBD proteins.
The best results for estimating absolute methylation levels with these methods come from pairing them with a complementary method to enrich for unmethylated sites such as MRE-seq (described above) <span class="citation">[<a href="references.html#ref-Li2015" role="doc-biblioref">130</a>]</span>.
Though relative methylation levels remain effective in identifying differentially methylated regions when using these methods alone <span class="citation">[<a href="references.html#ref-Bock2010" role="doc-biblioref">131</a>,<a href="references.html#ref-Maunakea2010" role="doc-biblioref">81</a>]</span>.</p></li>
<li><p><strong>Sodium bisulfite conversion of unmethylated cytosines to uracil</strong>
Conversion of cytosine to uracil changes the base complementary to this site from G to A <span class="citation">[<a href="references.html#ref-Hayatsu1970" role="doc-biblioref">132</a>]</span>.
The basic chemistry of the conversion process is illustrated in Figure <a href="introduction.html#fig:BisulfiteConversionNEB">1.16</a>.
This conversion can be detected with a variety of technologies such as:
Sanger sequencing, The Illumina bead chip methylation arrays; WGBS; and Targeted bisulfite PCR or sequencing to examine a small number of selected loci.
It can also be performed on a larger number of loci with microfluidic multiplexing such as that provided by the Fluidigm access array <span class="citation">[<a href="references.html#ref-Adamowicz2018" role="doc-biblioref">133</a>]</span>.
It is worth noting that bisulfite conversion cannot differentiate between methylated and hydroxymethylated cytosines.
Because 5hmC represents a small fraction of modified bases compared to 5mC many analyses have made the working assumption that unconverted bases are methylated, but it is beginning to be recognised that distinguishing between the two may be biologically important especially in tissues where 5hmC is more prevalent such as neurons.
A variant of bisulfite conversion, oxidative bisulfite conversion, exists which can permit 5hmC to be distinguished from 5mC <span class="citation">[<a href="references.html#ref-Skvortsova2017" role="doc-biblioref">134</a>]</span>.
Additionally, New England Biolabs has recently developed an enzymatic alternative to chemical bisulfite conversion <span class="citation">[<a href="references.html#ref-Williams2019" role="doc-biblioref">135</a>]</span></p></li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:BisulfiteConversionNEB"></span>
<img src="figs/BisulfiteConversionNEB.jpg" alt="Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs)." width="80%" />
<p class="caption">
Figure 1.16: Sodium Bisulfite conversion of C to U, 5mC and 5hmC are unaffected. (Image Credit New England Biolabs).
</p>
</div>

<ol start="4" style="list-style-type: decimal">
<li><strong>Physical differences in the methylated base</strong>
These are exploited by the not yet widely used ‘3rd generation’ sequencing technologies such as nanopore and single-molecule real-time sequencing.
These methods respectively exploit the effects of modified bases on ion flow through the nanopore and impact on incorporation time of new bases whilst in the polymerase <span class="citation">[<a href="references.html#ref-Rhoads2015" role="doc-biblioref">136</a>,<a href="references.html#ref-Simpson2017" role="doc-biblioref">137</a>]</span>.</li>
</ol>
</div>
</div>
<div id="dna-methylation-and-ageing" class="section level2" number="1.5">
<h2><span class="header-section-number">1.5</span> DNA Methylation and Ageing</h2>
<p>Returning now to the relationship between DNA methylation and ageing we will review the major discoveries in that field, the roles thought to be played by DNAm in ageing and the utility of DNAm clocks as biomarkers of ageing.</p>
<p>Early work on DNA methylation and ageing used biochemical, chromatographic and radiolabeling techniques to assay global changes in the amount of 5mC.
These studies found differences in 5mC composition with cell type <span class="citation">[<a href="references.html#ref-Ehrlich1982" role="doc-biblioref">82</a>]</span> but did not see changes with age.
Other work around that time however found differences in 5mC levels with the age of cells in culture <span class="citation">[<a href="references.html#ref-Wilson1983" role="doc-biblioref">138</a>,<a href="references.html#ref-Feinberg1983" role="doc-biblioref">139</a>]</span>.
Wilson et al. noted that DNA methylation decreased across several tens of generations of cell lines in culture, but not in immortal cell lines.
Immortal cell lines had lower absolute levels of 5mC to start with but remained constant over time.
Wilson et al. also cite earlier work by Romanov at al. 1981 <span class="citation">[<a href="references.html#ref-Romanov1981" role="doc-biblioref">140</a>]</span> and Berdyshev el al. 1967 <span class="citation">[<a href="references.html#ref-Berdyshev1967" role="doc-biblioref">141</a>]</span> which documented decreases in 5mC with age in cows and salmon, respectively.</p>
<p>Later work by Wilson et al. 1987 <span class="citation">[<a href="references.html#ref-Wilson1987" role="doc-biblioref">142</a>]</span> in mice <em>P. leucopus</em> and <em>M. musculus</em> found decreases in 5mC with age and that the rate of decline was less in the longer-lived <em>P. leucopus</em> than in <em>M. musculus</em>.
Interestingly, a recent study using Reduced Representation Bisulfite Sequencing (RRBS) by Cole et al. found no global differences between young and old mice in short and long-lived strains, however, their other observations would seem to corroborate these initial trends.
For the assayed sites long-lived mice exhibited 10x more hypermethylation than wild-type (WT), and WT mice had 3x more Differentially Methylated Regions significantly associated with age (DMRs/aDMRs) than did long-lived mice.
Notably, the WT and long-lived mice shared many of the same aDMRs which differed in their degree of methylation as opposed to affecting different sites in the genome <span class="citation">[<a href="references.html#ref-Cole2017" role="doc-biblioref">143</a>]</span>.
The lack of apparent global changes may be due to the biases of the reduced representation bisulfite sequencing (RRBS) method used.
RRBS uses a restriction enzyme-based approach to enrich for regions with high GC content such as CGIs which tend to have low levels of methylation, and consequently may not be sensitive to loss of methylation in generally hypermethylated regions which could contribute to a global trend <span class="citation">[<a href="references.html#ref-Meissner2005" role="doc-biblioref">144</a>]</span>.</p>
<p>Wilson et al. also noted that the mitotic index of tissues did not relate to the loss of DNA methylation with age in tension with their earlier observations <em>in vitro</em>.
The persistent loss of 5mC over time and the dramatic changes in methylation seen in cancer cells led Wilson et al. to suggest that dysregulation of DNA methylation may have a substantial role to play in the age-dependency of cancer risk and ageing more generally <span class="citation">[<a href="references.html#ref-Feinberg2004" role="doc-biblioref">145</a>]</span>.
It has been suggested that “Epimutations” may be able to substitute for mutations in the multi-hit model of carcinogenesis <span class="citation">[<a href="references.html#ref-Knudson1971" role="doc-biblioref">146</a>]</span>.
For example; hypermethylation of the <em>BRCA1</em> promoter <span class="citation">[<a href="references.html#ref-Esteller2000" role="doc-biblioref">147</a>]</span>, Or, the development of Wilms’ tumour due to Beckwith–Wiedemann syndrome, a disorder arising from loss of imprinting of the gene encoding insulin-like growth factor 2 (IGF-2) leading to a double dose of IGF-2 protein <span class="citation">[<a href="references.html#ref-Feinberg2018" role="doc-biblioref">148</a>]</span>.
Furthermore, the loss of methylation and increased entropy of closed chromatin regions may lead to increased susceptibility to the types of structural mutations commonly found in cancer cells.
This could result from the exposure of homologous sequences presenting increased opportunities for erroneous breakpoint repair or recombination between similar sequences in different parts of the genome <span class="citation">[<a href="references.html#ref-Feinberg2004" role="doc-biblioref">145</a>,<a href="references.html#ref-Jenkinson2017" role="doc-biblioref">118</a>]</span>.</p>
<p>Methods which allowed the examination of changes in DNA methylation at known loci in the genome permitted a more nuanced picture of changes in DNA methylation over time to develop.
Fraga et al. <span class="citation">[<a href="references.html#ref-Fraga2005" role="doc-biblioref">119</a>]</span> introduced the concept of “epigenetic drift” being the divergence of DNA methylation and other epigenetic modification patterns with time.
Fraga et al. looked at the divergence in DNA methylation along with global Histone H3 and H4 acetylation patterns between monozygotic (MZ) twins over a wide range of ages.
They observed that older twins had greater epigenetic differences with time, in DNA methylation as well as H3, H4 Acetylation.
Fraga et al. also noted that the divergence in epigenetic state was greater in twins who had lived longer apart and had different medical histories.
Figure <a href="introduction.html#fig:Fraga2005fig3">1.17</a> is a useful visual encapsulation of Fraga et al.’s DNA methylation results.
Slieker et al. <span class="citation">[<a href="references.html#ref-Slieker2016" role="doc-biblioref">120</a>]</span> identified 6366 CpGs whose methylation variability increased with age (aVMPs) using the Illumina 450k array platform on whole-blood from 3295 individuals.
Variably upregulated aVMPs were associated with the expression of DNA repair and apoptosis genes and variably downregulated aVMPs with cellular metabolism.
Furthermore
Both the increase in variability with age and the increasing divergence of twins support a narrative of epigenetic dysregulation and increasing entropy with age.</p>
<div class="figure" style="text-align: center"><span id="fig:Fraga2005fig3"></span>
<img src="figs/Fraga2005fig3.png" alt="Mapping Chromosomal Regions of Differential DNA Methylation. Examples from Chromosomes 1, 3, 12 &amp; 17 are shown for a 3 and 50-year-old twin pair. Methylation status is shown by competitive hybridization of AIMS (amplification of inter-methylated sites) products to metaphase chromosomes. Green and Red signals indicate hypermethylation and hypomethylation events between twins, Yellow indicates little difference. Red and Green blocks adjacent to ideograms indicate areas of significant DNA methylation change. (Reproduced from Fraga et al. [119] (figure 3).)" width="80%" />
<p class="caption">
Figure 1.17: Mapping Chromosomal Regions of Differential DNA Methylation. Examples from Chromosomes 1, 3, 12 &amp; 17 are shown for a 3 and 50-year-old twin pair. Methylation status is shown by competitive hybridization of AIMS (amplification of inter-methylated sites) products to metaphase chromosomes. Green and Red signals indicate hypermethylation and hypomethylation events between twins, Yellow indicates little difference. Red and Green blocks adjacent to ideograms indicate areas of significant DNA methylation change. (Reproduced from Fraga et al. <span class="citation">[<a href="references.html#ref-Fraga2005" role="doc-biblioref">119</a>]</span> (figure 3).)
</p>
</div>

<div id="DNAmAgeReview" class="section level3" number="1.5.1">
<h3><span class="header-section-number">1.5.1</span> Age-Related Differential Methylation and Epigenetic Clocks</h3>
<p>With the advent of array-based techniques which permitted the measurement of the methylation status of specific CpGs placed strategically throughout the genome, researchers were able to undertake a more fine-grained analysis of the changes in DNA methylation with age.
This has allowed the prediction of chronological age from a metric of “DNA methylation age” (DNAm age), and interpretation of the differences in predicted and chronological age in terms of the pathophysiology of ageing.</p>
<p>The Illumina Golden gate array <span class="citation">[<a href="references.html#ref-Bibikova2006" role="doc-biblioref">149</a>]</span> with 1505 probes targeted to cancer-related genes was the first of these.
Bjornsson et al. observed DNAm changes with age as well as noting substantial intra-individual differences.
In contrast earlier work by Eckhardt et al. <span class="citation">[<a href="references.html#ref-Eckhardt2006" role="doc-biblioref">150</a>]</span> and Ehrlich et al. <span class="citation">[<a href="references.html#ref-Ehrlich1982" role="doc-biblioref">82</a>]</span> found no age-related changes using averages across individuals.
Bjornsson et al. <span class="citation">[<a href="references.html#ref-Bjornsson2008" role="doc-biblioref">151</a>]</span> also found that the intra-individual differences in differential methylation with ageing were highly heritable by use of familial clustering.
Work by Boks et al. <span class="citation">[<a href="references.html#ref-Boks2009" role="doc-biblioref">103</a>]</span> also using the golden gate array corroborated Bjornsson et al.’s findings in monozygotic twins.</p>
<p>The Bjornsson and Boks studies used the golden gate array on peripheral blood samples, whereas Christensen et al. <span class="citation">[<a href="references.html#ref-Christensen2009" role="doc-biblioref">152</a>]</span> used the golden gate array to look for age-related changes in DNAm in several tissue types.
CpG sites that are differentially methylated with age when searching across tissues were frequently specific to a small number of tissue types.
In addition, CpGs in CGIs tended to be hypermethylated with age and <em>vice versa</em>.
Christensen et al. also found that the methylation profiles for different tissues were highly predictive of tissue type.</p>
<p>Rakyan et al. <span class="citation">[<a href="references.html#ref-Rakyan2010" role="doc-biblioref">153</a>]</span> looked at age-related changes in DNAm using the Illumina 27k array <span class="citation">[<a href="references.html#ref-Bibikova2009" role="doc-biblioref">154</a>]</span> with whole blood samples in a discovery set.
They replicated their findings in sorted cell fractions to see if they could capture changes in DNAm that could be attributed to changes in blood cell-type composition over time, which they did not find.
Teschendorff et al. <span class="citation">[<a href="references.html#ref-Teschendorff2010" role="doc-biblioref">155</a>]</span> found that promoters of targets of the polycomb group proteins (PCGTs) were more likely than non-PCGTs to become methylated with age <span class="citation">[<a href="references.html#ref-Jung2015" role="doc-biblioref">156</a>]</span>.
Repression of PCGTs is required for Stem cells to differentiate.
PCGTs whose methylation status was associated with age were associated with pre-neoplastic conditions in a large cohort using the 27k array with blood and epithelial cell samples.</p>
<p>Whilst Bell et al. <span class="citation">[<a href="references.html#ref-Bell2010" role="doc-biblioref">157</a>]</span> observed a correlation between a major component of variation in DNA methylation in blood and age, Bocklandt et al. <span class="citation">[<a href="references.html#ref-Bocklandt2011" role="doc-biblioref">158</a>]</span> also using the 27k array but with saliva samples created the first “epigenetic clock” used to predict the chronological age of donors based on the DNAm landscape of their cells, the mean error was 5.2 years.
Bocklandt et al. were able to achieve similar predictive accuracy with as few as 3 CpG sites in their model.
Koch et al. <span class="citation">[<a href="references.html#ref-Koch2011" role="doc-biblioref">159</a>]</span> used publicly available 27k array datasets from a variety of different tissues to train a model using a different statistical method but only achieved an error of 11 years.
They did, however, identify a number of CpGs also identified by Bocklandt et al.
Bell et al. <span class="citation">[<a href="references.html#ref-Bell2012" role="doc-biblioref">160</a>]</span> used 27k array data to look for correlations between differential methylation and age-related phenotypes as well as chronological age.
They found that few age-related phenotypes were correlated with differential methylation, only 5 CpG sites were identified for the 16 age-related phenotypes examined. Whereas 490 significant CpGs were associated with chronological age.
Many of the CpGs they identified persisted across tissue types and replicated in a second cohort, several had been identified previously by Rakyan et al. <span class="citation">[<a href="references.html#ref-Rakyan2010" role="doc-biblioref">153</a>]</span> and Bocklandt et al. <span class="citation">[<a href="references.html#ref-Bocklandt2011" role="doc-biblioref">158</a>]</span>.</p>
<p>Heyn et al. <span class="citation">[<a href="references.html#ref-Heyn2012" role="doc-biblioref">161</a>]</span> examined the DNA methylation status of a newborn and a centenarian in CD4+ T cells using whole-genome bisulfite sequencing (WGBS), along with a group of newborns and nonagenarians using the Illumina 450k array <span class="citation">[<a href="references.html#ref-Bibikova2011" role="doc-biblioref">162</a>]</span>.
Heyn et al. observed a global decrease in methylation from newborn to centenarian, as well as an intermediate level of methylation on a sample of intermediate age (see figure <a href="introduction.html#fig:Heyn2012F1b">1.18</a>).
This observation was replicated in 450k data.
In addition, Heyn et al. noted that adjacent CpGs normally exhibit a substantial degree of correlation in methylation status and that they were less well correlated with their neighbours with increasing age.
Interestingly Jenkinson et al. <span class="citation">[<a href="references.html#ref-Jenkinson2017" role="doc-biblioref">118</a>]</span> found that correlation among nearby CpGs increased in cancer tissues.
Modelling work done by Affinito et al. <span class="citation">[<a href="references.html#ref-Affinito2016" role="doc-biblioref">163</a>]</span> agrees with this correlation between neighbouring CpGs under physiological conditions, which is particularly pronounced in CpG dense regions like CGIs.
The greater physical proximity of CpGs in CpG dense regions means that greater correlation is expected.
Due to the fact that these CpGs are more likely to be affected by the same proteins and regulatory features than CpGs with greater distance separating them <span class="citation">[<a href="references.html#ref-Haerter2014" role="doc-biblioref">164</a>]</span>.
Unmethylated CGIs with H3K4me3 histone modifications can act as functional promoter units, this unmethylated state is needed for their interaction with CXXC domain containing proteins like Cfp1 <span class="citation">[<a href="references.html#ref-Thomson2010" role="doc-biblioref">165</a>]</span>.
Garagnani et al. used 450k array data from a cohort of 64 subjects aged 9-83 to identify those CpGs most well correlated with age as had been done with previous array technologies <span class="citation">[<a href="references.html#ref-Garagnani2012" role="doc-biblioref">166</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Heyn2012F1b"></span>
<img src="figs/Heyn2012F1b.png" alt="(ref:Heyn2012F1b-Cap)" width="80%" />
<p class="caption">
Figure 1.18: (ref:Heyn2012F1b-Cap)
</p>
</div>
<p>(ref:Heyn2012F1b-Cap) Global Hypomethylation with Age. NB = Newborn, Y26 = 26 year old, Y103 = 103 year old.
(Reproduced from Heyn et al. <span class="citation">[<a href="references.html#ref-Heyn2012" role="doc-biblioref">161</a>]</span> figure 1 b.)</p>
<p>In January of 2013 Hannum at al. <span class="citation">[<a href="references.html#ref-Hannum2013" role="doc-biblioref">167</a>]</span> published a study using the Illumina 450k array and whole blood samples for 656 participants aged 19-101.
Their optimised model was able to predict age with an error of 3.9 years.
In December of the same year, Horvath <span class="citation">[<a href="references.html#ref-Horvath2013" role="doc-biblioref">168</a>]</span> published a study using multiple datasets, including Hannum’s data, some produced on the 27k and some the 450k array.
Both used ‘elastic net’ penalised multivariate regression models to identify CpGs which collectively provide the best predictor of DNAm age rather than the CpGs which are individually most highly correlated with age <span class="citation">[<a href="references.html#ref-Friedman2010" role="doc-biblioref">169</a>]</span>.</p>
<p>Florath et al. <span class="citation">[<a href="references.html#ref-Florath2014" role="doc-biblioref">170</a>]</span> identified some additional age-associated CpGs from two large cohorts totalling N = 898 and an N = 67 8yr follow-up longitudinal cohort.
More than 3/4 of the CpG sites they identified began as hypomethylated and increased in methylation with age, a proportion likely skewed by the large number of 450k probes in CGIs and other typically low methylation regions.
Bacalini et al. <span class="citation">[<a href="references.html#ref-Bacalini2015" role="doc-biblioref">171</a>]</span> performed a meta-analysis of existing DNAm age datasets and employed a “region-centric” approach to try to identify loci larger than single CpGs which they anticipate will be more biologically meaningful than lone CpGs, despite being limited to just the CpGs for which there are array probes.
They found that their approach increased the number of common features identified using the Hannum et al. <span class="citation">[<a href="references.html#ref-Hannum2013" role="doc-biblioref">167</a>]</span> and Heyn et al. <span class="citation">[<a href="references.html#ref-Heyn2012" role="doc-biblioref">161</a>]</span> datasets.
Zaghlool et al. <span class="citation">[<a href="references.html#ref-Zaghlool2015" role="doc-biblioref">172</a>]</span> performed 450k array DNAm age study in Qatari population they identified 12/88, 23/490 and 102/162 of the CpGs found by Bocklandt et al. <span class="citation">[<a href="references.html#ref-Bocklandt2011" role="doc-biblioref">158</a>]</span>, Bell et al. <span class="citation">[<a href="references.html#ref-Bell2012" role="doc-biblioref">160</a>]</span> and Florath et al. <span class="citation">[<a href="references.html#ref-Florath2014" role="doc-biblioref">170</a>]</span> respectively. Zaghlool et al. <span class="citation">[<a href="references.html#ref-Zaghlool2015" role="doc-biblioref">172</a>]</span> also found that Horvath’s age predictor <span class="citation">[<a href="references.html#ref-Horvath2013" role="doc-biblioref">168</a>]</span> had an error of 3.7 years in their dataset.
This is in agreement with existing findings and indicates that ethnicity has no major effects on the DNAm age signature. Benton et al. <span class="citation">[<a href="references.html#ref-Benton2017" role="doc-biblioref">173</a>]</span> looked at changes in DNA methylation with age in a genetically isolated population on Norfolk Island with similar results to previous studies but identifying some novel age-associated CpGs.</p>
<p>Most DNAm relationships with age noted in previous studies have been linear, Johnson et al. <span class="citation">[<a href="references.html#ref-Johnson2017" role="doc-biblioref">174</a>]</span> used the 450k array in peripheral blood samples.
They identified 21 CpGs whose DNAm changes at a rate that changes with age from an initial pool of 27,723 CpGs which were differentially methylated with age.
Two sites exhibited an increasing rate of increase in DNAm with age, and 18 sites a decreasing rate of increase.
However, these effects could be impacted by changes in blood cell-type composition with age.</p>
</div>
<div id="genetic-influences-on-dna-methylation" class="section level3" number="1.5.2">
<h3><span class="header-section-number">1.5.2</span> Genetic Influences on DNA Methylation</h3>
<p>Epigenetic variation falls on a continuum of genetic influence that can be summarised by three categories:</p>
<ol style="list-style-type: decimal">
<li><p><strong>pure</strong> - DNA sequence has no predictive value for epigenetic state.</p></li>
<li><p><strong>facilitated</strong> - DNA sequence biases the epigenetic state enabling epigenetic variation.</p></li>
<li><p><strong>obligatory</strong> - DNA sequence permits exact prediction of epigenetic state.</p></li>
</ol>
<p>Regional methylation state is strongly influenced by genotype, by single nucleotide polymorphisms (SNPs) <span class="citation">[<a href="references.html#ref-Smith2014a" role="doc-biblioref">175</a>]</span> and by structural variants <span class="citation">[<a href="references.html#ref-Kuehnen2012" role="doc-biblioref">176</a>,<a href="references.html#ref-Gymrek2015" role="doc-biblioref">177</a>]</span>.
An example of an obligatory effect on methylation is a point mutation at a CpG site, a C to T transition precludes methylation at that site in future.
The effect of structural variants on methylation can be hard to determine as changes in sequence dosage often lead to measurement artefacts.
Efforts have been made to correct for the influence of genetic factors in EWAS, when searching for purely epigenetic effects but the potential interaction of the somatic mutations known to accumulate with age (Figure <a href="introduction.html#fig:Jaiswal2014fig1">1.19</a>) and the changes in DNAm with age remain largely unexplored.</p>
<p>Increases in frequency of particular mutations can also be driven by clonal expansion of cellular sub-populations, particularly blood, and possibly in other tissues with a high level of turnover.
The differentiation of blood cells from heamatopoetic stem cells is less linear and hierarchical than initially modelled.
The hematopoetic stem cell compartment is comprised of a pool of cells with characteristics of multiple lineages and are to a greater or lesser extent primed for a particular branch of differentiation <span class="citation">[<a href="references.html#ref-Velten2017" role="doc-biblioref">178</a>]</span>.
The somatic mutations present in these hematopoetic stem cells <span class="citation">[<a href="references.html#ref-Jaiswal2014" role="doc-biblioref">179</a>]</span>, provide the variation necessary for selection processes to act, and the changes in variant allele frequency in blood cells indicates that there are positive selection effects driving the increase in frequency of certain alleles <span class="citation">[<a href="references.html#ref-Watson2020" role="doc-biblioref">180</a>]</span>.
These selection effects favour more highly proloferative cells and may lead to the exhaustion of stem cells able to produce particular lineages and, imbalanced production of different cell types in undesirable proportions <span class="citation">[<a href="references.html#ref-Kirschner2017" role="doc-biblioref">181</a>]</span>.
This can be a pre-cursor to hematalogic malignancies <span class="citation">[<a href="references.html#ref-Jaiswal2014" role="doc-biblioref">179</a>]</span>.</p>
<p>This is of particular relevance to DNAm as the profile of the types of mutation which accumulate with age (Figure <a href="introduction.html#fig:Jaiswal2014fig2c">1.20</a>) distinctly favours C to T transition mutations which can disrupt CpG dinucleotides <span class="citation">[<a href="references.html#ref-Jaiswal2014c" role="doc-biblioref">182</a>]</span>.
CpG dinucleaotides are particularly prone to these mutations through the deamination of 5mC to T.</p>
<div class="figure" style="text-align: center"><span id="fig:Jaiswal2014fig1"></span>
<img src="figs/Jaiswal2014fig1_somaticMutationAge.png" alt="Somatic mutations increase with Age. (Reproduced from Jaiswal et al. [182], figure 1)" width="60%" />
<p class="caption">
Figure 1.19: Somatic mutations increase with Age. (Reproduced from Jaiswal et al. <span class="citation">[<a href="references.html#ref-Jaiswal2014c" role="doc-biblioref">182</a>]</span>, figure 1)
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:Jaiswal2014fig2c"></span>
<img src="figs/Jaiswal2014fig2c.png" alt="C to T transitions are the most common substitutions that occur with age. (Reproduced from Jaiswal et al. [182], figure2 c)" width="60%" />
<p class="caption">
Figure 1.20: C to T transitions are the most common substitutions that occur with age. (Reproduced from Jaiswal et al. <span class="citation">[<a href="references.html#ref-Jaiswal2014c" role="doc-biblioref">182</a>]</span>, figure2 c)
</p>
</div>

</div>
<div id="dnam-age-as-a-biomarker-of-ageing" class="section level3" number="1.5.3">
<h3><span class="header-section-number">1.5.3</span> DNAm Age as a Biomarker of Ageing</h3>
<p>We have seen that DNAm can be a fairly accurate predictor of age, however, there is more to being a good biomarker of ageing than predictive accuracy.
Weidner et al. <span class="citation">[<a href="references.html#ref-Weidner2014" role="doc-biblioref">183</a>]</span> conducted a study with the specific intent of identifying CpG sites that would serve as the best biomarkers of ageing.
Below is a list of specific criteria for a high-quality biomarker of ageing laid out by Johnson et al. <span class="citation">[<a href="references.html#ref-Johnson2006" role="doc-biblioref">184</a>]</span>.</p>
<ol style="list-style-type: decimal">
<li>It must predict the rate of ageing. In other words, it would tell exactly where a person is in their total lifespan. It must be a better predictor of lifespan than chronological age.</li>
<li>It must monitor a basic process that underlies the ageing process, not the effects of disease.</li>
<li>It must be able to be tested repeatedly without harming the person. For example, a blood test or an imaging technique.</li>
<li>It must be something that works in humans and laboratory animals, such as mice. This is so that it can be tested in lab animals before being validated in humans.</li>
</ol>
<p>We have seen that DNA methylation clocks can be good predictors of chronological age, we will revisit the question of whether or not they are superior to chronological age in their ability to predict lifespan below.
Bell et al. <span class="citation">[<a href="references.html#ref-Bell2012" role="doc-biblioref">160</a>]</span> found DNAm did not generally correlate well with ageing phenotypes but did correlate well with chronological age and Teschendorff et al. <span class="citation">[<a href="references.html#ref-Teschendorff2010" role="doc-biblioref">155</a>]</span> found that their age-associated DNAm signature remained fairly constant across several disease states including ovarian cancer and type 1 diabetes.
Suggesting that DNAm age prediction does indeed: “monitor a basic process that underlies the ageing process, not the effects of disease,” meeting criterion number 2.
Several of the DNAm clocks discussed so far have been based on whole peripheral blood samples meeting criterion number 3. DNAm age clocks have been shown to work in Chimpanzees <span class="citation">[<a href="references.html#ref-Horvath2013" role="doc-biblioref">168</a>]</span>, Dogs/Wolves <span class="citation">[<a href="references.html#ref-Thompson2017" role="doc-biblioref">185</a>]</span>, Mice <span class="citation">[<a href="references.html#ref-Stubbs2017" role="doc-biblioref">186</a>]</span>, Naked Mole Rats <span class="citation">[<a href="references.html#ref-Lowe2020" role="doc-biblioref">187</a>]</span>, Rhesus Macaques, Humpback Whales <span class="citation">[<a href="references.html#ref-Lowe2018" role="doc-biblioref">188</a>]</span> and are likely to work in other mammalian model organisms.
Furthermore, a cross-species clock based on conserved ribosomal RNA genes has been developed <span class="citation">[<a href="references.html#ref-Wang2019" role="doc-biblioref">189</a>]</span>.
It should be noted that Horvath’s Human clock uses array data and that the Dog/Wolf and Mouse clocks use RRBS.
Model organisms such as <em>C. elegans</em> which essentially lack DNA methylation would not, however, be likely to be tractable for DNAm based age prediction.
Consequently DNAm age prediction at least partially meets criterion number 4.</p>
<p>Is DNA methylation a better predictor of lifespan than chronological age?
Both Hannum and Horvath identified the difference between DNAm age and chronological age or <span class="math inline">\(\Delta_{Age}\)</span> as a potential indicator of disproportionate biological ageing.
Marioni et al. <span class="citation">[<a href="references.html#ref-Marioni2015" role="doc-biblioref">190</a>]</span> explicitly set forth to test the value of DNAm age as a predictor of mortality.
Adjusting for age, sex, childhood IQ, education, social class, hypertension, diabetes, cardiovascular disease, and <em>APOE</em> e4 status a <span class="math inline">\(\Delta_{Age}\)</span> of +5yr is associated with a 16% increased mortality risk (See figure <a href="introduction.html#fig:Marioni2015f3">1.21</a>).
A longitudinal study of twins found 3.2 fold increase in the risk of dying first per 5yr <span class="math inline">\(\Delta_{Age}\)</span> within twin pairs <span class="citation">[<a href="references.html#ref-Christiansen2016" role="doc-biblioref">191</a>]</span>.
A meta-analysis of DNAm age predictors by Chen et al. <span class="citation">[<a href="references.html#ref-Chen2016a" role="doc-biblioref">192</a>]</span> also found that measures of age acceleration based on Hannum and Horvath clocks were superior predictors of mortality than chronological age before and after correction for various potentially confounding factors.
Chen et al. also noted that correction for blood cell composition improved predictive power.
Indicating that DNAm age is indeed superior to chronological age as a predictor of lifespan meeting criterion number 1 for an ageing biomarker.</p>
<p>A new DNAm age clock explicitly designed to capture phenotypic age and outperform age acceleration as a biomarker of ageing has recently been produced by Levine et al. <span class="citation">[<a href="references.html#ref-Levine2018" role="doc-biblioref">193</a>]</span>.
The ‘PhenoAge’ metric was created by selecting nine biomarkers of ageing from 42 possible metrics using a proportional hazards penalised regression model and combining these with chronological age.
(The nine biomarkers are: Albumin, Creatinine, Serum glucose, C-reactive protein, Lymphocyte per cent, Mean red cell volume, Red cell distribution width, Alkaline phosphatase, White blood cell count.)
DNAm data was then regressed against PhenoAge using elastic-net regression to produce a DNAm based PhenoAge predictor which made use of 513 CpGs.
The PhenoAge predictor outperformed the Horvath and Hannum clocks at predicting all-cause mortality, comorbidities, coronary heart disease risk, and measures of physical functioning.
41 of the 513 CpGs in the PhenoAge clock were present in the original 353 CpG sites used in the Horvath clock.</p>
<p>The ‘GrimAge’ clock <span class="citation">[<a href="references.html#ref-Lu2019" role="doc-biblioref">194</a>]</span> takes a different approach to previous epigenetic clocks in that it has a two-tiered model predicting first seven surrogates for a variety of other biomarkers of ageing then fitting a model to predict time to death using these surrogates.
‘GrimAge’ outperforms age accelerations from the Horvath, Hannum and PhenoAge clocks at predicting time to death <span class="citation">[<a href="references.html#ref-Halloran2020" role="doc-biblioref">195</a>]</span>. Belsky et al. developed a DNA methylation based estimate of the rate of biological ageing called ‘DunedinPoAm’ <span class="citation">[<a href="references.html#ref-Belsky2020" role="doc-biblioref">196</a>]</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:Marioni2015f3"></span>
<img src="figs/Marioni2015f3.png" alt="Survival probability by quartiles of \(\Delta_{Age}\) in LBC 1921 adjusted for sex, and chronological age. LBC = Lothian Birth Cohort. Using the Hannum and Horvath predictors’ values for \(\Delta_{Age}\). (Reproduced from [190] figure 3.)" width="80%" />
<p class="caption">
Figure 1.21: Survival probability by quartiles of <span class="math inline">\(\Delta_{Age}\)</span> in LBC 1921 adjusted for sex, and chronological age. LBC = Lothian Birth Cohort. Using the Hannum and Horvath predictors’ values for <span class="math inline">\(\Delta_{Age}\)</span>. (Reproduced from <span class="citation">[<a href="references.html#ref-Marioni2015" role="doc-biblioref">190</a>]</span> figure 3.)
</p>
</div>

</div>
<div id="dna-methylation-clock-measurements-as-outcomes-for-anti-ageing-interventions" class="section level3" number="1.5.4">
<h3><span class="header-section-number">1.5.4</span> DNA methylation clock measurements as outcomes for anti-ageing interventions</h3>
<p>Fahy et al. showed in a small exploratory study that they could reduce epigenetic age, as measured by four different epigenetic clocks <span class="citation">[<a href="references.html#ref-Fahy2019" role="doc-biblioref">197</a>]</span>.
The clocks used were: Horvath et al. <span class="citation">[<a href="references.html#ref-Horvath2013" role="doc-biblioref">168</a>]</span>, Hannum et al. <span class="citation">[<a href="references.html#ref-Hannum2013" role="doc-biblioref">167</a>]</span>, ‘PhenoAge’ Levine et al. <span class="citation">[<a href="references.html#ref-Levine2018" role="doc-biblioref">193</a>]</span> and ‘GrimAge’ Lu et al. <span class="citation">[<a href="references.html#ref-Lu2019a" role="doc-biblioref">198</a>]</span>.
The study called TRIIM (Thymus Regeneration, Immunorestoration, and Insulin Mitigation) consisted of ten men aged 51-65 who were treated with a regimen of recombinant human growth hormone (rhGH) in combination with dehydroepiandrosterone (DHEA) and metformin.
The aim of this treatment was to counteract the decline in immune function attributed to the decrease in diversity of t-cell receptors that occurs in this age group <span class="citation">[<a href="references.html#ref-Naylor2005" role="doc-biblioref">199</a>]</span>.
rhGH was found to stimulate the reconstitution of the thymus and thereby T-cell levels in HIV patients, hence it was reasoned that it might also counteract the age-related deterioration of the thymus.
DHEA and metformin were included in the regimen to counter the hyperinsulinemia which can be induced by growth hormone <span class="citation">[<a href="references.html#ref-Marcus1990" role="doc-biblioref">200</a>]</span>.
The mean decrease in epigenetic age after 12 months treatment was 2.5 years so a reduction of 1.5 years from baseline, the rate of this decrease appeared still to be increasing in the final 3 months of treatment (9-12 months).
There was incomplete regression towards pre-treatment levels of age acceleration at 18 months from onset, 6 months after cessation of treatment, in 3 of the 4 epigenetic clocks.
There were however no sign of change in the GrimAge <span class="citation">[<a href="references.html#ref-Lu2019a" role="doc-biblioref">198</a>]</span> acceleration estimate which remained at, on average, 2 years lower than age acceleration at onset.
This study sets an important precedent for for the use of DNAm age acceleration as a readout of the effectiveness of anti-ageing interventions, opening the door for their use in future studies.
The challenge faced by studies where the intended outcome is increased lifespan is that this requires a long, expensive, and impractical follow-up period to measure directly.
If DNA methylation based measures of age acceleration can be used as effective proxies for measuring lifespan then this dramatically decreases the cost and inconvenience of studying interventions which effect lifespan.
Clearly however it needs to be established how good a proxy for actual health/lifespan increases DNA methylation age metrics actually are before they can be relied upon.</p>
<p>Ascertaining the effectiveness of using DNA methylation based age acceleration metrics as an outcome in ageing and longevity interventions in model organisms is likely to be informative about how well they can be expected to perform in humans.
Promisingly, differences in interventions known to effect longevity in model organisms such as caloric restriction, rapamycin treatment, ovariectomy in female mice, and genetically long lived mouse strains <span class="citation">[<a href="references.html#ref-Gensous2019" role="doc-biblioref">201</a>,<a href="references.html#ref-Wang2017a" role="doc-biblioref">202</a>,<a href="references.html#ref-Stubbs2017" role="doc-biblioref">186</a>]</span> have consistently shown decreased epigenetic ageing under the pro-longevity conditions.
Horvath et al. recently produced a single epigenetic age predictor which works in both humans and rats using the same assay of conserved CpG sites and the same age prediction algorithm <span class="citation">[<a href="references.html#ref-Horvath2020" role="doc-biblioref">203</a>]</span>.
This unified metric provides greater confidence that an intervention which improves ageing outcomes in rats will also do so in humans.
Horvath et al. also demonstrate the effectiveness of Heterochronic Plasma Exchange (HPE), the replacement of the plasma of old rats by that of young rats at rejuvenating the rat tissues.
They observed an average of a 54.2% reduction in epigenetic age across liver, blood, heart, and hypothalamus with their pan tissue clock in old rats which underwent HPE <span class="citation">[<a href="references.html#ref-Horvath2020" role="doc-biblioref">203</a>]</span>.</p>
<p>These results have laid the groundwork for the first large scale trial of an anti-ageing intervention.
Metformin has a relatively small but well documented effect on lifespan and reduced risk of a number of ageing related conditions in model organisms and in humans with type two diabetes, for which it is widely prescribed <span class="citation">[<a href="references.html#ref-Kulkarni2020" role="doc-biblioref">204</a>]</span>.
The TAME (Targeting Aging with Metformin) trial plans to include over 3,000 individuals aged 65-79 in a double-blind placebo controlled trial to examine the effects of metformin treatment on a composite primary endpoint consisting of stroke, heart failure, dementia, myocardial infarction, cancer and death <span class="citation">[<a href="references.html#ref-TAME2020" role="doc-biblioref">205</a>]</span>.
The biomarker work group indicated that DNA methylation ageing assays may not be among the primary set of biomarkers used in the trial <span class="citation">[<a href="references.html#ref-Justice2018" role="doc-biblioref">206</a>]</span>.
However, TAME will establish a repository of bio-specimens including blood which will likely permit the examination of the effects of metformin on DNA methylation based ageing metrics in humans at a large scale.</p>
</div>
</div>
<div id="aims" class="section level2" number="1.6">
<h2><span class="header-section-number">1.6</span> Aims</h2>
<p>Going forward this work covers several aspects of change in DNA methylation related to ageing phenotypes.
Starting at the beginning of the human life-course EWAS will be performed for <em>in utero</em> markers and interventions thought to impact on long-term bone health.
An examination will be undertaken of the age-related changes in the methylation state of a key functional region of the genome, whose methylation state has not previously been characterised in detail, namely the tRNA genes.
Lastly, new DNA methylation clocks based on a family of repetitive elements, specifically the Alu repeat family, will be generated.
This will make use of another region of the genome the DNA methylation state of which has not previously been characterised in detail, but which represents a substantial proportion of all CpG sites, the derepression of which has long been considered to play a role in ageing.</p>
<ol style="list-style-type: decimal">
<li><p>Identify epigenetic associations with bone health outcomes and of vitamin D supplementation during pregnancy as an intervention to improve bone outcomes <span class="citation">[<a href="references.html#ref-Harvey2012a" role="doc-biblioref">207</a>,<a href="references.html#ref-Cooper2016" role="doc-biblioref">208</a>]</span>. Provide some groundwork for mechanistic studies to extend understanding of what systems influence bone health by identifying what genomic features change epigenetically with bone traits and interventions intended to affect these traits.</p></li>
<li><p>Identify Age-related changes in DNA methylation in regions of the genome characterised in the TwinsUK MeDIP-seq dataset and poorly covered by or covered in small samples by other technologies.
The regions to be examined are:</p>
<ol style="list-style-type: lower-alpha">
<li>The tRNA genes, following up on the previous finding of age-related DNA hypermethylation at tRNA-iMet-CAT-1-4 <span class="citation">[<a href="references.html#ref-Bell2016" role="doc-biblioref">209</a>]</span>.
tRNA genes have a core role in cellular metabolism, many emerging regulatory functions both structural and as signalling molecules.
tRNA genes also interface with many systems the modulation of which impact longevity, making any age-related changes in their epigenetic state potentially very consequential.</li>
<li>Alu repeat elements, a primate-specific family of SINEs present in over 1 million copies in the human genome.
The aim is to construct a DNA methylation clock using only these elements.
Age acceleration based on this clock may yield different information about biological ageing than from previous clocks as the effects of DNA methylation changes differ between repressed repetitive elements and enhancers or promoter proximal regulatory sequences.</li>
</ol></li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection",
"toc_depth": 5,
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
