<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | The Epigenomics of Human Ageing</title>
  <meta name="description" content="Thesis for the degree of Doctor of Philosophy" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="References | The Epigenomics of Human Ageing" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis for the degree of Doctor of Philosophy" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | The Epigenomics of Human Ageing" />
  
  <meta name="twitter:description" content="Thesis for the degree of Doctor of Philosophy" />
  

<meta name="author" content="Richard J. Acton" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="discussion.html"/>
<link rel="next" href="appendices.html"/>
<style type="text/css">
p.abstract{
  text-align: center;
  font-weight: bold;
}
div.abstract{
  margin: auto;
  width: 90%;
}
</style>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#the-ageing-population-and-the-burden-of-disease"><i class="fa fa-check"></i><b>1.1</b> The Ageing Population and The Burden of Disease</a></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#the-biology-of-ageing"><i class="fa fa-check"></i><b>1.2</b> The biology of Ageing</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#epigenomics---overview"><i class="fa fa-check"></i><b>1.3</b> Epigenomics - Overview</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#fundamentals-of-dna-methylation"><i class="fa fa-check"></i><b>1.4</b> Fundamentals of DNA Methylation</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="introduction.html"><a href="introduction.html#structure-and-context"><i class="fa fa-check"></i><b>1.4.1</b> Structure and Context</a></li>
<li class="chapter" data-level="1.4.2" data-path="introduction.html"><a href="introduction.html#IntroCpGdist"><i class="fa fa-check"></i><b>1.4.2</b> Distribution and Global Trends</a></li>
<li class="chapter" data-level="1.4.3" data-path="introduction.html"><a href="introduction.html#pathways-of-dna-methylation-and-demethylation"><i class="fa fa-check"></i><b>1.4.3</b> Pathways of DNA methylation and demethylation</a></li>
<li class="chapter" data-level="1.4.4" data-path="introduction.html"><a href="introduction.html#maintenance-and-fidelity"><i class="fa fa-check"></i><b>1.4.4</b> Maintenance and Fidelity</a></li>
<li class="chapter" data-level="1.4.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-assays"><i class="fa fa-check"></i><b>1.4.5</b> DNA Methylation Assays</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="introduction.html"><a href="introduction.html#dna-methylation-and-ageing"><i class="fa fa-check"></i><b>1.5</b> DNA Methylation and Ageing</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="introduction.html"><a href="introduction.html#DNAmAgeReview"><i class="fa fa-check"></i><b>1.5.1</b> Age-Related Differential Methylation and Epigenetic Clocks</a></li>
<li class="chapter" data-level="1.5.2" data-path="introduction.html"><a href="introduction.html#genetic-influences-on-dna-methylation"><i class="fa fa-check"></i><b>1.5.2</b> Genetic Influences on DNA Methylation</a></li>
<li class="chapter" data-level="1.5.3" data-path="introduction.html"><a href="introduction.html#dnam-age-as-a-biomarker-of-ageing"><i class="fa fa-check"></i><b>1.5.3</b> DNAm Age as a Biomarker of Ageing</a></li>
<li class="chapter" data-level="1.5.4" data-path="introduction.html"><a href="introduction.html#dna-methylation-clock-measurements-as-outcomes-for-anti-ageing-interventions"><i class="fa fa-check"></i><b>1.5.4</b> DNA methylation clock measurements as outcomes for anti-ageing interventions</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="introduction.html"><a href="introduction.html#aims"><i class="fa fa-check"></i><b>1.6</b> Aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#GenMethodsMethArray"><i class="fa fa-check"></i><b>2.1</b> Illumina DNA Methylation arrays</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#MeDIPseq"><i class="fa fa-check"></i><b>2.2</b> MeDIP-seq</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#Participants"><i class="fa fa-check"></i><b>2.2.1</b> Participants</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#MethodsTargetedBisulfiteSequencing"><i class="fa fa-check"></i><b>2.3</b> Targeted Bisulfite Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="arrays.html"><a href="arrays.html"><i class="fa fa-check"></i><b>3</b> Epigenome Wide Association Studies for Bone Health Outcomes in Umbilical Cord Blood and Tissue</a>
<ul>
<li class="chapter" data-level="3.1" data-path="arrays.html"><a href="arrays.html#arrayAbstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="arrays.html"><a href="arrays.html#arrayIntro"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="arrays.html"><a href="arrays.html#primary-questions"><i class="fa fa-check"></i><b>3.3</b> Primary Questions</a></li>
<li class="chapter" data-level="3.4" data-path="arrays.html"><a href="arrays.html#arrayMethods"><i class="fa fa-check"></i><b>3.4</b> Methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="arrays.html"><a href="arrays.html#outline-of-ewas"><i class="fa fa-check"></i><b>3.4.1</b> Outline of EWAS</a></li>
<li class="chapter" data-level="3.4.2" data-path="arrays.html"><a href="arrays.html#functional-normalisation"><i class="fa fa-check"></i><b>3.4.2</b> Functional Normalisation</a></li>
<li class="chapter" data-level="3.4.3" data-path="arrays.html"><a href="arrays.html#genetically-confounded-and-multi-mapping-probes"><i class="fa fa-check"></i><b>3.4.3</b> Genetically Confounded and Multi-mapping Probes</a></li>
<li class="chapter" data-level="3.4.4" data-path="arrays.html"><a href="arrays.html#ewas-models"><i class="fa fa-check"></i><b>3.4.4</b> EWAS Models</a></li>
<li class="chapter" data-level="3.4.5" data-path="arrays.html"><a href="arrays.html#concordance-of-epic-and-450k-ewas-results"><i class="fa fa-check"></i><b>3.4.5</b> Concordance of EPIC and 450k EWAS Results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="arrays.html"><a href="arrays.html#arrayResults"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="arrays.html"><a href="arrays.html#combined-mavidos-summary-data"><i class="fa fa-check"></i><b>3.5.1</b> Combined MAVIDOS Summary data</a></li>
<li class="chapter" data-level="3.5.2" data-path="arrays.html"><a href="arrays.html#mavidos-phase-i"><i class="fa fa-check"></i><b>3.5.2</b> MAVIDOS phase I</a></li>
<li class="chapter" data-level="3.5.3" data-path="arrays.html"><a href="arrays.html#mavidos-phase-ii"><i class="fa fa-check"></i><b>3.5.3</b> MAVIDOS phase II</a></li>
<li class="chapter" data-level="3.5.4" data-path="arrays.html"><a href="arrays.html#southampton-womens-survey-sws"><i class="fa fa-check"></i><b>3.5.4</b> Southampton Women’s Survey (SWS)</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="arrays.html"><a href="arrays.html#arrayDiscussion"><i class="fa fa-check"></i><b>3.6</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="tRNAs.html"><a href="tRNAs.html"><i class="fa fa-check"></i><b>4</b> The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a>
<ul>
<li class="chapter" data-level="4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-abstract"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Introduction"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="tRNAs.html"><a href="tRNAs.html#primary-questions-1"><i class="fa fa-check"></i><b>4.3</b> Primary questions</a></li>
<li class="chapter" data-level="4.4" data-path="tRNAs.html"><a href="tRNAs.html#tRNAmethods"><i class="fa fa-check"></i><b>4.4</b> Methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Participants"><i class="fa fa-check"></i><b>4.4.1</b> Participants</a></li>
<li class="chapter" data-level="4.4.2" data-path="tRNAs.html"><a href="tRNAs.html#tRNAanno"><i class="fa fa-check"></i><b>4.4.2</b> tRNA Gene coordinates and other annotation information</a></li>
<li class="chapter" data-level="4.4.3" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-DNA-methylome-Data"><i class="fa fa-check"></i><b>4.4.3</b> DNA methylome data</a></li>
<li class="chapter" data-level="4.4.4" data-path="tRNAs.html"><a href="tRNAs.html#chromatin-segmentation-data"><i class="fa fa-check"></i><b>4.4.4</b> Chromatin Segmentation Data</a></li>
<li class="chapter" data-level="4.4.5" data-path="tRNAs.html"><a href="tRNAs.html#sortedbloodmethods"><i class="fa fa-check"></i><b>4.4.5</b> Isolated Blood Cell Type Specific Data</a></li>
<li class="chapter" data-level="4.4.6" data-path="tRNAs.html"><a href="tRNAs.html#TCGAmethods"><i class="fa fa-check"></i><b>4.4.6</b> Cancer and Tissue Specific Methylation Data</a></li>
<li class="chapter" data-level="4.4.7" data-path="tRNAs.html"><a href="tRNAs.html#mintmapmethods"><i class="fa fa-check"></i><b>4.4.7</b> Assaying tRNA expression in blood with MINTmap</a></li>
<li class="chapter" data-level="4.4.8" data-path="tRNAs.html"><a href="tRNAs.html#mousemethods"><i class="fa fa-check"></i><b>4.4.8</b> Mouse RRBS Analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="tRNAs.html"><a href="tRNAs.html#res"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="tRNAs.html"><a href="tRNAs.html#dna-methylation-of-specific-trna-gene-loci-changes-with-age"><i class="fa fa-check"></i><b>4.5.1</b> DNA Methylation of Specific tRNA Gene Loci Changes with Age</a></li>
<li class="chapter" data-level="4.5.2" data-path="tRNAs.html"><a href="tRNAs.html#trna-genes-are-enriched-for-age-related-dna-hypermethylation"><i class="fa fa-check"></i><b>4.5.2</b> tRNA Genes are Enriched for Age Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="4.5.3" data-path="tRNAs.html"><a href="tRNAs.html#trna-gene-dna-methylation-in-other-tissues"><i class="fa fa-check"></i><b>4.5.3</b> tRNA Gene DNA Methylation in Other Tissues</a></li>
<li class="chapter" data-level="4.5.4" data-path="tRNAs.html"><a href="tRNAs.html#expression-of-trnas-in-blood-with-age"><i class="fa fa-check"></i><b>4.5.4</b> Expression of tRNAs in Blood with Age</a></li>
<li class="chapter" data-level="4.5.5" data-path="tRNAs.html"><a href="tRNAs.html#mice-also-show-age-related-trna-gene-dna-hypermethylation"><i class="fa fa-check"></i><b>4.5.5</b> Mice also show age-related tRNA gene DNA hypermethylation</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="tRNAs.html"><a href="tRNAs.html#tRNA-Discussion"><i class="fa fa-check"></i><b>4.6</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="Alus.html"><a href="Alus.html"><i class="fa fa-check"></i><b>5</b> DNA methylation clocks in Alu Elements</a>
<ul>
<li class="chapter" data-level="5.1" data-path="Alus.html"><a href="Alus.html#Alu-Abstract"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="Alus.html"><a href="Alus.html#AluIntro"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="Alus.html"><a href="Alus.html#Alu-Methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="Alus.html"><a href="Alus.html#dna-methylome-data"><i class="fa fa-check"></i><b>5.3.1</b> DNA methylome data</a></li>
<li class="chapter" data-level="5.3.2" data-path="Alus.html"><a href="Alus.html#alu-element-annotation-and-pre-filtering"><i class="fa fa-check"></i><b>5.3.2</b> Alu element annotation and pre-filtering</a></li>
<li class="chapter" data-level="5.3.3" data-path="Alus.html"><a href="Alus.html#elastic-net-regression"><i class="fa fa-check"></i><b>5.3.3</b> Elastic net regression</a></li>
<li class="chapter" data-level="5.3.4" data-path="Alus.html"><a href="Alus.html#sampleSelection"><i class="fa fa-check"></i><b>5.3.4</b> Sample Selection</a></li>
<li class="chapter" data-level="5.3.5" data-path="Alus.html"><a href="Alus.html#binarisation"><i class="fa fa-check"></i><b>5.3.5</b> Binarisation</a></li>
<li class="chapter" data-level="5.3.6" data-path="Alus.html"><a href="Alus.html#AluMethodsArrays"><i class="fa fa-check"></i><b>5.3.6</b> Comparisons to other clocks in TwinsUK Illumina 450k array data</a></li>
<li class="chapter" data-level="5.3.7" data-path="Alus.html"><a href="Alus.html#genome-wide-association-studies-of-alu-age-acceleration"><i class="fa fa-check"></i><b>5.3.7</b> Genome Wide Association studies of Alu Age Acceleration</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="Alus.html"><a href="Alus.html#Alu-Results"><i class="fa fa-check"></i><b>5.4</b> Results</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="Alus.html"><a href="Alus.html#comparison-to-other-dna-methylation-clocks"><i class="fa fa-check"></i><b>5.4.1</b> Comparison to other DNA methylation clocks</a></li>
<li class="chapter" data-level="5.4.2" data-path="Alus.html"><a href="Alus.html#alu-age-acceleration"><i class="fa fa-check"></i><b>5.4.2</b> Alu Age Acceleration</a></li>
<li class="chapter" data-level="5.4.3" data-path="Alus.html"><a href="Alus.html#top-results-genome-wide-association-studies-of-alu-age-acceleration-are-shared-by-alu-age-predictors"><i class="fa fa-check"></i><b>5.4.3</b> Top results Genome Wide Association studies of Alu Age Acceleration are shared by Alu Age predictors</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="Alus.html"><a href="Alus.html#Alu-Discussion"><i class="fa fa-check"></i><b>5.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a>
<ul>
<li class="chapter" data-level="6.1" data-path="discussion.html"><a href="discussion.html#epigenomic-analysis-of-the-developmental-origins-of-long-term-bone-health"><i class="fa fa-check"></i><b>6.1</b> Epigenomic analysis of the developmental origins of long-term bone health</a></li>
<li class="chapter" data-level="6.2" data-path="discussion.html"><a href="discussion.html#the-implications-of-ageing-related-changes-in-the-epigenetic-state-of-trna-genes"><i class="fa fa-check"></i><b>6.2</b> The implications of ageing-related changes in the epigenetic state of tRNA genes</a></li>
<li class="chapter" data-level="6.3" data-path="discussion.html"><a href="discussion.html#assessing-biological-ageing-by-dna-methylation-changes-within-alu-repeat-elements"><i class="fa fa-check"></i><b>6.3</b> Assessing Biological ageing by DNA methylation changes within Alu repeat elements</a></li>
<li class="chapter" data-level="6.4" data-path="discussion.html"><a href="discussion.html#conclusion"><i class="fa fa-check"></i><b>6.4</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="appendices.html"><a href="appendices.html"><i class="fa fa-check"></i>Appendices</a>
<ul>
<li class="chapter" data-level="6.5" data-path="appendices.html"><a href="appendices.html#biorchi-iv-manuscript-the-genomic-loci-of-specific-human-trna-genes-exhibit-ageing-related-dna-hypermethylation"><i class="fa fa-check"></i><b>6.5</b> <span class="math inline">\(BioR\chi iv\)</span> manuscript: The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</a></li>
<li class="chapter" data-level="6.6" data-path="appendices.html"><a href="appendices.html#apptRNAanno"><i class="fa fa-check"></i><b>6.6</b> Specific tRNA annotations used in the tRNA analyses</a></li>
<li class="chapter" data-level="6.7" data-path="appendices.html"><a href="appendices.html#apptRNAprimers"><i class="fa fa-check"></i><b>6.7</b> tRNA targeted bisulfite sequencing primers</a></li>
<li class="chapter" data-level="6.8" data-path="appendices.html"><a href="appendices.html#apptRNAmods"><i class="fa fa-check"></i><b>6.8</b> Additional tRNA Age modelling results</a></li>
<li class="chapter" data-level="6.9" data-path="appendices.html"><a href="appendices.html#apptRNAmouse"><i class="fa fa-check"></i><b>6.9</b> tRNA mouse models</a></li>
<li class="chapter" data-level="6.10" data-path="appendices.html"><a href="appendices.html#apptRNAprop"><i class="fa fa-check"></i><b>6.10</b> tRNA proportional enrichment fisher’s test count data</a></li>
<li class="chapter" data-level="6.11" data-path="appendices.html"><a href="appendices.html#appTCGA"><i class="fa fa-check"></i><b>6.11</b> TCGA sample used in tRNA analysis</a></li>
<li class="chapter" data-level="6.12" data-path="appendices.html"><a href="appendices.html#apptwins450ktRNA"><i class="fa fa-check"></i><b>6.12</b> Twins UK 450k array tRNA age models</a></li>
<li class="chapter" data-level="6.13" data-path="appendices.html"><a href="appendices.html#appRXRACDKN2A"><i class="fa fa-check"></i><b>6.13</b> MAVIDOS RXRA and CDKN2A probe models</a></li>
<li class="chapter" data-level="6.14" data-path="appendices.html"><a href="appendices.html#appAluageSNPs"><i class="fa fa-check"></i><b>6.14</b> Age acceleration GWAS SNPs</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><strong>The Epigenomics of Human Ageing</strong></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references csl-bib-body">
<div id="ref-Acton2020" class="csl-entry">
1. Acton RJ, Bourne E, Bell J, Lillycrop K, Wang J, Dennison E, Harvey N, Spector TD, Cooper C, Bell CG: <strong><span class="nocase">The Genomic Loci of Specific Human tRNA Genes Exhibit Ageing-Related DNA Hypermethylation</span></strong>. <em>bioRxiv</em> 2019, :1–48<a href="https://doi.org/10.1101/870352">10.1101/870352</a>Available: <a href="https://www.biorxiv.org/content/10.1101/870352v1">https://www.biorxiv.org/content/10.1101/870352v1</a>.
</div>
<div id="ref-Kontis2017" class="csl-entry">
2. Kontis V, Bennett JE, Mathers CD, Li G, Foreman K, Ezzati M: <strong><span class="nocase">Future life expectancy in 35 industrialised countries: projections with a Bayesian model ensemble</span></strong>. <em>The Lancet</em> 2017, <strong>389</strong>:1323–1335<a href="https://doi.org/10.1016/S0140-6736(16)32381-9">10.1016/S0140-6736(16)32381-9</a>Available: <a href="http://dx.doi.org/10.1016/S0140-6736(16)32381-9">http://dx.doi.org/10.1016/S0140-6736(16)32381-9</a>.
</div>
<div id="ref-Niccoli2012" class="csl-entry">
3. Niccoli T, Partridge L: <strong><span class="nocase">Ageing as a risk factor for disease</span></strong>. <em>Current Biology</em> 2012, <strong>22</strong>:R741–R752<a href="https://doi.org/10.1016/j.cub.2012.07.024">10.1016/j.cub.2012.07.024</a>Available: <a href="http://dx.doi.org/10.1016/j.cub.2012.07.024">http://dx.doi.org/10.1016/j.cub.2012.07.024</a>.
</div>
<div id="ref-Belikov2019" class="csl-entry">
4. Belikov AV: <strong><span class="nocase">Age-related diseases as vicious cycles</span></strong>. <em>Ageing Research Reviews</em> 2019, <strong>49</strong>:11–26<a href="https://doi.org/10.1016/j.arr.2018.11.002">10.1016/j.arr.2018.11.002</a>Available: <a href="https://doi.org/10.1016/j.arr.2018.11.002">https://doi.org/10.1016/j.arr.2018.11.002</a>.
</div>
<div id="ref-Fleming2005" class="csl-entry">
5. Fleming DM, Elliot AJ: <strong><span class="nocase">The impact of influenza on the health and health care utilisation of elderly people</span></strong>. <em>Vaccine</em> 2005, <strong>23</strong>:S1<a href="https://doi.org/10.1016/j.vaccine.2005.04.018">10.1016/j.vaccine.2005.04.018</a>.
</div>
<div id="ref-Barnett2012" class="csl-entry">
6. Barnett K, Mercer SW, Norbury M, Watt G, Wyke S, Guthrie B: <strong><span class="nocase">Epidemiology of multimorbidity and implications for health care, research, and medical education: a cross-sectional study</span></strong>. <em>The Lancet</em> 2012, <strong>380</strong>:37–43<a href="https://doi.org/10.1016/S0140-6736(12)60240-2">10.1016/S0140-6736(12)60240-2</a>Available: <a href="http://dx.doi.org/10.1016/S0140-6736(12)60240-2">http://dx.doi.org/10.1016/S0140-6736(12)60240-2</a>.
</div>
<div id="ref-PopulationPyramid2019" class="csl-entry">
7. United Nations, Department of Economic and Social Affairs PDivision: <strong><span class="nocase">PopulationPyramid.net</span></strong>. Available: <a href="https://PopulationPyramid.net">PopulationPyramid.net</a>.
</div>
<div id="ref-Goldman2016" class="csl-entry">
8. Goldman D: <strong><span class="nocase">The Economic Promise of Delayed Aging</span></strong>. <em>Cold Spring Harbor Perspectives in Medicine</em> 2016, <strong>6</strong>:a025072<a href="https://doi.org/10.1101/cshperspect.a025072">10.1101/cshperspect.a025072</a>Available: <a href="http://perspectivesinmedicine.cshlp.org/lookup/doi/10.1101/cshperspect.a025072">http://perspectivesinmedicine.cshlp.org/lookup/doi/10.1101/cshperspect.a025072</a>.
</div>
<div id="ref-Kenyon1993" class="csl-entry">
9. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R: <strong><span class="nocase">A C. elegans mutant that lives twice as long as wild type</span></strong>. <em>Nature</em> 1993, <strong>366</strong>:461–464<a href="https://doi.org/10.1038/366461a0">10.1038/366461a0</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/8247153">http://www.ncbi.nlm.nih.gov/pubmed/8247153</a>.
</div>
<div id="ref-Medvedev1990" class="csl-entry">
10. Medvedev ZA: <strong><span class="nocase">An Attempt at a Rational Classification of Theories of Ageing</span></strong>. <em>Biological Reviews</em> 1990, <strong>65</strong>:375–398<a href="https://doi.org/10.1111/j.1469-185X.1990.tb01428.x">10.1111/j.1469-185X.1990.tb01428.x</a>Available: <a href="http://doi.wiley.com/10.1111/j.1469-185X.1990.tb01428.x">http://doi.wiley.com/10.1111/j.1469-185X.1990.tb01428.x</a>.
</div>
<div id="ref-Lopez-Otin2013" class="csl-entry">
11. López-Otín C, Blasco MA, Partridge L, Serrano M, Kroemer G: <strong><span class="nocase">The hallmarks of aging.</span></strong> <em>Cell</em> 2013, <strong>153</strong>:1194–217<a href="https://doi.org/10.1016/j.cell.2013.05.039">10.1016/j.cell.2013.05.039</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23746838">http://www.ncbi.nlm.nih.gov/pubmed/23746838</a>.
</div>
<div id="ref-Niedernhofer2018" class="csl-entry">
12. Niedernhofer LJ, Gurkar AU, Wang Y, Vijg J, Hoeijmakers JHJ, Robbins PD: <strong><span class="nocase">Nuclear Genomic Instability and Aging</span></strong>. <em>Annual Review of Biochemistry</em> 2018, <strong>87</strong>:295–322<a href="https://doi.org/10.1146/annurev-biochem-062917-012239">10.1146/annurev-biochem-062917-012239</a>Available: <a href="https://www.annualreviews.org/doi/10.1146/annurev-biochem-062917-012239">https://www.annualreviews.org/doi/10.1146/annurev-biochem-062917-012239</a>.
</div>
<div id="ref-Casagrande2019" class="csl-entry">
13. Casagrande S, Hau M: <strong><span class="nocase">Telomere attrition: metabolic regulation and signalling function?</span></strong> <em>Biology Letters</em> 2019, <strong>15</strong>:20180885<a href="https://doi.org/10.1098/rsbl.2018.0885">10.1098/rsbl.2018.0885</a>Available: <a href="https://royalsocietypublishing.org/doi/10.1098/rsbl.2018.0885">https://royalsocietypublishing.org/doi/10.1098/rsbl.2018.0885</a>.
</div>
<div id="ref-Saldanha2015" class="csl-entry">
14. Pal S, Tyler JK: <strong><span class="nocase">Epigenetics and aging</span></strong>. <em>Science Advances</em> 2016, <strong>2</strong>:e1600584<a href="https://doi.org/10.1126/sciadv.1600584">10.1126/sciadv.1600584</a>Available: <a href="https://advances.sciencemag.org/lookup/doi/10.1126/sciadv.1600584">https://advances.sciencemag.org/lookup/doi/10.1126/sciadv.1600584</a>.
</div>
<div id="ref-Santra2019" class="csl-entry">
15. Santra M, Dill KA, Graff AMR de: <strong><span class="nocase">Proteostasis collapse is a driver of cell aging and death</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2019, <strong>116</strong>:22173–22178<a href="https://doi.org/10.1073/pnas.1906592116">10.1073/pnas.1906592116</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1906592116">http://www.pnas.org/lookup/doi/10.1073/pnas.1906592116</a>.
</div>
<div id="ref-Johnson2018" class="csl-entry">
16. Johnson SC: <strong><span class="nocase">Nutrient Sensing, Signaling and Ageing: The Role of IGF-1 and mTOR in Ageing and Age-Related Disease</span></strong>. In <em>Subcellular biochemistry</em> Vol. 90 2018:49–97. Available: <a href="http://link.springer.com/10.1007/978-981-13-2835-0{\_}3">http://link.springer.com/10.1007/978-981-13-2835-0{\_}3</a>.
</div>
<div id="ref-Soultoukis2016" class="csl-entry">
17. Soultoukis GA, Partridge L: <strong><span class="nocase">Dietary Protein, Metabolism, and Aging</span></strong>. <em>Annual Review of Biochemistry</em> 2016, <strong>85</strong>:5–34<a href="https://doi.org/10.1146/annurev-biochem-060815-014422">10.1146/annurev-biochem-060815-014422</a>Available: <a href="http://www.annualreviews.org/doi/10.1146/annurev-biochem-060815-014422">http://www.annualreviews.org/doi/10.1146/annurev-biochem-060815-014422</a>.
</div>
<div id="ref-Payne2015" class="csl-entry">
18. Payne BAI, Chinnery PF: <strong><span class="nocase">Mitochondrial dysfunction in aging: Much progress but many unresolved questions</span></strong>. <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 2015, <strong>1847</strong>:1347–1353<a href="https://doi.org/10.1016/j.bbabio.2015.05.022">10.1016/j.bbabio.2015.05.022</a>Available: <a href="http://dx.doi.org/10.1016/j.bbabio.2015.05.022">http://dx.doi.org/10.1016/j.bbabio.2015.05.022</a>.
</div>
<div id="ref-Avelar2020" class="csl-entry">
19. Avelar RA, Ortega JG, Tacutu R, Tyler EJ, Bennett D, Binetti P, Budovsky A, Chatsirisupachai K, Johnson E, Murray A, Shields S, Tejada-Martinez D, Thornton D, Fraifeld VE, Bishop CL, Magalhães JP de: <strong><span class="nocase">A multidimensional systems biology analysis of cellular senescence in aging and disease</span></strong>. <em>Genome Biology</em> 2020, <strong>21</strong>:91<a href="https://doi.org/10.1186/s13059-020-01990-9">10.1186/s13059-020-01990-9</a>Available: <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01990-9">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01990-9</a>.
</div>
<div id="ref-Ren2017" class="csl-entry">
20. Ren R, Ocampo A, Liu G-H, Izpisua Belmonte JC: <strong><span class="nocase">Regulation of Stem Cell Aging by Metabolism and Epigenetics</span></strong>. <em>Cell Metabolism</em> 2017, <strong>26</strong>:460–474<a href="https://doi.org/10.1016/j.cmet.2017.07.019">10.1016/j.cmet.2017.07.019</a>Available: <a href="http://dx.doi.org/10.1016/j.cmet.2017.07.019 https://linkinghub.elsevier.com/retrieve/pii/S1550413117304849">http://dx.doi.org/10.1016/j.cmet.2017.07.019 https://linkinghub.elsevier.com/retrieve/pii/S1550413117304849</a>.
</div>
<div id="ref-Franceschi2014" class="csl-entry">
21. Franceschi C, Campisi J: <strong><span class="nocase">Chronic Inflammation (Inflammaging) and Its Potential Contribution to Age-Associated Diseases</span></strong>. <em>The Journals of Gerontology Series A: Biological Sciences and Medical Sciences</em> 2014, <strong>69</strong>:S4–S9<a href="https://doi.org/10.1093/gerona/glu057">10.1093/gerona/glu057</a>Available: <a href="https://academic.oup.com/biomedgerontology/article-lookup/doi/10.1093/gerona/glu057">https://academic.oup.com/biomedgerontology/article-lookup/doi/10.1093/gerona/glu057</a>.
</div>
<div id="ref-Kenyon1993a" class="csl-entry">
22. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R: <strong><span class="nocase">A C. elegans mutant that lives twice as long as wild type</span></strong>. <em>Nature</em> 1993, <strong>366</strong>:461–464<a href="https://doi.org/10.1038/366461a0">10.1038/366461a0</a>Available: <a href="http://www.nature.com/articles/366461a0">http://www.nature.com/articles/366461a0</a>.
</div>
<div id="ref-Kenyon2010a" class="csl-entry">
23. Kenyon CJ: <strong><span class="nocase">The genetics of ageing</span></strong>. <em>Nature</em> 2010, <strong>464</strong>:504–512<a href="https://doi.org/10.1038/nature08980">10.1038/nature08980</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20336132">http://www.ncbi.nlm.nih.gov/pubmed/20336132</a>.
</div>
<div id="ref-Singh2019" class="csl-entry">
24. Singh PP, Demmitt BA, Nath RD, Brunet A: <strong><span class="nocase">The Genetics of Aging: A Vertebrate Perspective</span></strong>. <em>Cell</em> 2019, <strong>177</strong>:200–220<a href="https://doi.org/10.1016/j.cell.2019.02.038">10.1016/j.cell.2019.02.038</a>Available: <a href="https://doi.org/10.1016/j.cell.2019.02.038">https://doi.org/10.1016/j.cell.2019.02.038</a>.
</div>
<div id="ref-Broer2015" class="csl-entry">
25. Broer L, Buchman AS, Deelen J, Evans DS, Faul JD, Lunetta KL, Sebastiani P, Smith JA, Smith AV, Tanaka T, Yu L, Arnold AM, Aspelund T, Benjamin EJ, De Jager PL, Eirkisdottir G, Evans DA, Garcia ME, Hofman A, Kaplan RC, Kardia SLR, Kiel DP, Oostra BA, Orwoll ES, Parimi N, Psaty BM, Rivadeneira F, Rotter JI, Seshadri S, Singleton A, et al.: <strong><span class="nocase">GWAS of Longevity in CHARGE Consortium Confirms APOE and FOXO3 Candidacy</span></strong>. <em>The Journals of Gerontology: Series A</em> 2015, <strong>70</strong>:110–118<a href="https://doi.org/10.1093/gerona/glu166">10.1093/gerona/glu166</a>Available: <a href="https://academic.oup.com/biomedgerontology/article/70/1/110/2947666">https://academic.oup.com/biomedgerontology/article/70/1/110/2947666</a>.
</div>
<div id="ref-Zenin2019" class="csl-entry">
26. Zenin A, Tsepilov Y, Sharapov S, Getmantsev E, Menshikov LI, Fedichev PO, Aulchenko Y: <strong><span class="nocase">Identification of 12 genetic loci associated with human healthspan</span></strong>. <em>Communications Biology</em> 2019, <strong>2</strong>:41<a href="https://doi.org/10.1038/s42003-019-0290-0">10.1038/s42003-019-0290-0</a>Available: <a href="http://dx.doi.org/10.1038/s42003-019-0290-0 http://www.nature.com/articles/s42003-019-0290-0">http://dx.doi.org/10.1038/s42003-019-0290-0 http://www.nature.com/articles/s42003-019-0290-0</a>.
</div>
<div id="ref-Melzer2019" class="csl-entry">
27. Melzer D, Pilling LC, Ferrucci L: <strong><span class="nocase">The genetics of human ageing</span></strong>. <em>Nature Reviews Genetics</em> 2019, <a href="https://doi.org/10.1038/s41576-019-0183-6">10.1038/s41576-019-0183-6</a>Available: <a href="http://dx.doi.org/10.1038/s41576-019-0183-6">http://dx.doi.org/10.1038/s41576-019-0183-6</a>.
</div>
<div id="ref-Claussnitzer2020" class="csl-entry">
28. Claussnitzer M, Cho JH, Collins R, Cox NJ, Dermitzakis ET, Hurles ME, Kathiresan S, Kenny EE, Lindgren CM, MacArthur DG, North KN, Plon SE, Rehm HL, Risch N, Rotimi CN, Shendure J, Soranzo N, McCarthy MI: <strong><span class="nocase">A brief history of human disease genetics</span></strong>. <em>Nature</em> 2020, <strong>577</strong>:179–189<a href="https://doi.org/10.1038/s41586-019-1879-7">10.1038/s41586-019-1879-7</a>.
</div>
<div id="ref-Shindyapina2020" class="csl-entry">
29. Shindyapina AV, Zenin AA, Tarkhov AE, Santesmasses D, Fedichev PO, Gladyshev VN: <strong><span class="nocase">Germline burden of rare damaging variants negatively affects human healthspan and lifespan</span></strong>. <em>eLife</em> 2020, <strong>9</strong>:1–18<a href="https://doi.org/10.7554/eLife.53449">10.7554/eLife.53449</a>Available: <a href="https://elifesciences.org/articles/53449">https://elifesciences.org/articles/53449</a>.
</div>
<div id="ref-Booth2016" class="csl-entry">
30. Booth LN, Brunet A: <strong><span>The Aging Epigenome</span></strong>. <em>Molecular Cell</em> 2016, <strong>62</strong>:728–744<a href="https://doi.org/10.1016/j.molcel.2016.05.013">10.1016/j.molcel.2016.05.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2016.05.013">http://dx.doi.org/10.1016/j.molcel.2016.05.013</a>.
</div>
<div id="ref-Sinclair2009" class="csl-entry">
31. Sinclair DA, Oberdoerffer P: <strong><span class="nocase">The ageing epigenome: Damaged beyond repair?</span></strong> <em>Ageing Research Reviews</em> 2009, <strong>8</strong>:189–198<a href="https://doi.org/10.1016/j.arr.2009.04.004">10.1016/j.arr.2009.04.004</a>.
</div>
<div id="ref-Hayano2019" class="csl-entry">
32. Hayano M, Yang J-H, Bonkowski MS, Amorim JA, Ross JM, Coppotelli G, Griffin P, Chew YC, Guo W, Yang X, Vera DL, Salfati EL, Das A, Thakur S, Kane AE, Mitchell SJ, Mohri Y, Nishimura EK, Schaevitz L, Garg N, Balta A-M, Rego MA, Gregory-Ksander M, Jakobs TC, Zhong L, Wakimoto H, Mostoslavsky R, Wagers AJ, Tsubota K, Bonasera SJ, et al.: <strong><span class="nocase">DNA Break-Induced Epigenetic Drift as a Cause of Mammalian Aging</span></strong>. <em>bioRxiv</em> 2019, <a href="https://doi.org/10.1101/808659">10.1101/808659</a>Available: <a href="https://doi.org/10.1101/808659">https://doi.org/10.1101/808659</a>.
</div>
<div id="ref-Kane2019" class="csl-entry">
33. Kane AE, Sinclair DA: <strong><span class="nocase">Epigenetic changes during aging and their reprogramming potential</span></strong>. <em>Critical Reviews in Biochemistry and Molecular Biology</em> 2019, <strong>54</strong>:61–83<a href="https://doi.org/10.1080/10409238.2019.1570075">10.1080/10409238.2019.1570075</a>Available: <a href="https://doi.org/10.1080/10409238.2019.1570075">https://doi.org/10.1080/10409238.2019.1570075</a>.
</div>
<div id="ref-Bird2007" class="csl-entry">
34. Bird A: <strong><span class="nocase">Perceptions of epigenetics.</span></strong> <em>Nature</em> 2007, <strong>447</strong>:396–8<a href="https://doi.org/10.1038/nature05913">10.1038/nature05913</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17522671">http://www.ncbi.nlm.nih.gov/pubmed/17522671</a>.
</div>
<div id="ref-Waddington1942" class="csl-entry">
35. Waddington CH: <strong><span class="nocase">The epigenotype. 1942.</span></strong> <em>International journal of epidemiology</em> 2012, <strong>41</strong>:10–3<a href="https://doi.org/10.1093/ije/dyr184">10.1093/ije/dyr184</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22186258">http://www.ncbi.nlm.nih.gov/pubmed/22186258</a>.
</div>
<div id="ref-Russo1996" class="csl-entry">
36. Russo V, Martienssen R, Riggs AD: <em><span class="nocase">Epigenetic Mechanisms of Gene Regulation</span></em>. Cold Spring Harbor Laboratory Press; 1996 Available: <a href="https://cshmonographs.org/index.php/monographs/issue/view/087969490.32">https://cshmonographs.org/index.php/monographs/issue/view/087969490.32</a>.
</div>
<div id="ref-Tabansky2015" class="csl-entry">
37. Tabansky I, Stern JNH, Pfaff DW: <strong><span class="nocase">Implications of Epigenetic Variability within a Cell Population for <span>“Cell Type”</span> Classification</span></strong>. <em>Frontiers in Behavioral Neuroscience</em> 2015, <strong>9</strong>:1–11<a href="https://doi.org/10.3389/fnbeh.2015.00342">10.3389/fnbeh.2015.00342</a>Available: <a href="http://journal.frontiersin.org/Article/10.3389/fnbeh.2015.00342/abstract">http://journal.frontiersin.org/Article/10.3389/fnbeh.2015.00342/abstract</a>.
</div>
<div id="ref-Bianconi2013" class="csl-entry">
38. Bianconi E, Piovesan A, Facchin F, Beraudi A, Casadei R, Frabetti F, Vitale L, Pelleri MC, Tassani S, Piva F, Perez-Amodio S, Strippoli P, Canaider S: <strong><span class="nocase">An estimation of the number of cells in the human body.</span></strong> <em>Annals of human biology</em> 2013, <strong>40</strong>:463–71<a href="https://doi.org/10.3109/03014460.2013.807878">10.3109/03014460.2013.807878</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23829164">http://www.ncbi.nlm.nih.gov/pubmed/23829164</a>.
</div>
<div id="ref-Regev2017" class="csl-entry">
39. Regev A, Teichmann SA, Lander ES, Amit I, Benoist C, Birney E, Bodenmiller B, Campbell P, Carninci P, Clatworthy M, Clevers H, Deplancke B, Dunham I, Eberwine J, Eils R, Enard W, Farmer A, Fugger L, Göttgens B, Hacohen N, Haniffa M, Hemberg M, Kim S, Klenerman P, Kriegstein A, Lein E, Linnarsson S, Lundberg E, Lundeberg J, Majumder P, et al.: <strong><span>The Human Cell Atlas</span></strong>. <em>eLife</em> 2017, <strong>6</strong>:1–30<a href="https://doi.org/10.7554/eLife.27041">10.7554/eLife.27041</a>Available: <a href="https://elifesciences.org/articles/27041">https://elifesciences.org/articles/27041</a>.
</div>
<div id="ref-CellSystemsCellType2017" class="csl-entry">
40. CellSystemsVoices: <strong><span class="nocase">What Is Your Conceptual Definition of <span>“Cell Type”</span> in the Context of a Mature Organism?</span></strong> <em>Cell Systems</em> 2017, <strong>4</strong>:255–259<a href="https://doi.org/10.1016/j.cels.2017.03.006">10.1016/j.cels.2017.03.006</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S2405471217300911">https://linkinghub.elsevier.com/retrieve/pii/S2405471217300911</a>.
</div>
<div id="ref-CellOntologyOBO" class="csl-entry">
41. CellOntology: <strong><span>Cell Ontology</span></strong>. 2017, <a href="https://doi.org/10.5281/zenodo.168254">10.5281/zenodo.168254</a>Available: <a href="http://obofoundry.org/ontology/cl.html">http://obofoundry.org/ontology/cl.html</a>.Accessed 5 April 2017.
</div>
<div id="ref-Guo2014a" class="csl-entry">
42. Guo JU, Su Y, Shin JH, Shin J, Li H, Xie B, Zhong C, Hu S, Le T, Fan G, Zhu H, Chang Q, Gao Y, Ming G, Song H: <strong><span class="nocase">Distribution, recognition and regulation of non-CpG methylation in the adult mammalian brain.</span></strong> <em>Nature neuroscience</em> 2014, <strong>17</strong>:215–22<a href="https://doi.org/10.1038/nn.3607">10.1038/nn.3607</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24362762">http://www.ncbi.nlm.nih.gov/pubmed/24362762</a>.
</div>
<div id="ref-Lister2009" class="csl-entry">
43. Lister R, Pelizzola M, Dowen RH, Hawkins RD, Hon G, Tonti-Filippini J, Nery JR, Lee L, Ye Z, Ngo Q, Edsall L, Antosiewicz-Bourget J, Stewart R, Ruotti V, Millar AH, Thomson JA, Ren B, Ecker JR: <strong><span class="nocase">Human DNA methylomes at base resolution show widespread epigenomic differences.</span></strong> <em>Nature</em> 2009, <strong>462</strong>:315–22<a href="https://doi.org/10.1038/nature08514">10.1038/nature08514</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19829295">http://www.ncbi.nlm.nih.gov/pubmed/19829295</a>.
</div>
<div id="ref-Schultz2015" class="csl-entry">
44. Schultz MD, He Y, Whitaker JW, Hariharan M, Mukamel EA, Leung D, Rajagopal N, Nery JR, Urich MA, Chen H, Lin S, Lin Y, Jung I, Schmitt AD, Selvaraj S, Ren B, Sejnowski TJ, Wang W, Ecker JR: <strong><span class="nocase">Human body epigenome maps reveal noncanonical DNA methylation variation.</span></strong> <em>Nature</em> 2015, <strong>523</strong>:212–6<a href="https://doi.org/10.1038/nature14465">10.1038/nature14465</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26030523">http://www.ncbi.nlm.nih.gov/pubmed/26030523</a>.
</div>
<div id="ref-Penn1972" class="csl-entry">
45. Penn NW, Suwalski R, O’Riley C, Bojanowski K, Yura R: <strong><span class="nocase">The presence of 5-hydroxymethylcytosine in animal deoxyribonucleic acid</span></strong>. <em>Biochemical Journal</em> 1972, <strong>126</strong>:781–790<a href="https://doi.org/10.1042/bj1260781">10.1042/bj1260781</a>Available: <a href="https://portlandpress.com/biochemj/article/126/4/781/6593/The-presence-of-5hydroxymethylcytosine-in-animal">https://portlandpress.com/biochemj/article/126/4/781/6593/The-presence-of-5hydroxymethylcytosine-in-animal</a>.
</div>
<div id="ref-Kriaucionis2009" class="csl-entry">
46. Kriaucionis S, Heintz N: <strong><span class="nocase">The Nuclear DNA Base 5-Hydroxymethylcytosine Is Present in Purkinje Neurons and the Brain</span></strong>. <em>Science</em> 2009, <strong>324</strong>:929–930<a href="https://doi.org/10.1126/science.1169786">10.1126/science.1169786</a>Available: <a href="https://www.sciencemag.org/lookup/doi/10.1126/science.1169786">https://www.sciencemag.org/lookup/doi/10.1126/science.1169786</a>.
</div>
<div id="ref-Pfeifer2013" class="csl-entry">
47. Pfeifer GP, Kadam S, Jin S-G: <strong>5-hydroxymethylcytosine and its potential roles in development and cancer.</strong> <em>Epigenetics <span>&amp;</span> chromatin</em> 2013, <strong>6</strong>:10<a href="https://doi.org/10.1186/1756-8935-6-10">10.1186/1756-8935-6-10</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23634848">http://www.ncbi.nlm.nih.gov/pubmed/23634848</a>.
</div>
<div id="ref-Ito2011" class="csl-entry">
48. Ito S, Shen L, Dai Q, Wu SC, Collins LB, Swenberg JA, He C, Zhang Y: <strong><span class="nocase">Tet Proteins Can Convert 5-Methylcytosine to 5-Formylcytosine and 5-Carboxylcytosine</span></strong>. <em>Science</em> 2011, <strong>333</strong>:1300–1303<a href="https://doi.org/10.1126/science.1210597">10.1126/science.1210597</a>Available: <a href="https://www.sciencemag.org/lookup/doi/10.1126/science.1210597">https://www.sciencemag.org/lookup/doi/10.1126/science.1210597</a>.
</div>
<div id="ref-Wu2016" class="csl-entry">
49. Wu TP, Wang T, Seetin MG, Lai Y, Zhu S, Lin K, Liu Y, Byrum SD, Mackintosh SG, Zhong M, Tackett A, Wang G, Hon LS, Fang G, Swenberg Ja, Xiao AZ: <strong><span class="nocase">DNA methylation on N(6)-adenine in mammalian embryonic stem cells.</span></strong> <em>Nature</em> 2016, <strong>532</strong>:329–33<a href="https://doi.org/10.1038/nature17640">10.1038/nature17640</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27027282">http://www.ncbi.nlm.nih.gov/pubmed/27027282</a>.
</div>
<div id="ref-Xie2018" class="csl-entry">
50. Xie Q, Wu TP, Gimple RC, Li Z, Prager BC, Wu Q, Yu Y, Wang P, Wang Y, Gorkin DU, Zhang C, Dowiak AV, Lin K, Zeng C, Sui Y, Kim LJY, Miller TE, Jiang L, Lee CH, Huang Z, Fang X, Zhai K, Mack SC, Sander M, Bao S, Kerstetter-Fogle AE, Sloan AE, Xiao AZ, Rich JN: <strong><span class="nocase">N-methyladenine DNA Modification in Glioblastoma</span></strong>. <em>Cell</em> 2018, <strong>175</strong>:1228–1243.e20<a href="https://doi.org/10.1016/j.cell.2018.10.006">10.1016/j.cell.2018.10.006</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S009286741831314X">https://linkinghub.elsevier.com/retrieve/pii/S009286741831314X</a>.
</div>
<div id="ref-Losick2014" class="csl-entry">
51. Losick JDWTABSPBAGMLR: <em><span class="nocase">Molecular Biology of the Gene</span></em>. 7th ed. Pearson; 2014.
</div>
<div id="ref-Bannister2011" class="csl-entry">
52. Bannister AJ, Kouzarides T: <strong><span class="nocase">Regulation of chromatin by histone modifications</span></strong>. <em>Cell Research</em> 2011, <strong>21</strong>:381–395<a href="https://doi.org/10.1038/cr.2011.22">10.1038/cr.2011.22</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21321607">http://www.ncbi.nlm.nih.gov/pubmed/21321607</a>.
</div>
<div id="ref-Strahl2000" class="csl-entry">
53. Strahl BD, Allis CD: <strong><span class="nocase">The language of covalent histone modifications</span></strong>. <em>Nature</em> 2000, <strong>403</strong>:41–45<a href="https://doi.org/10.1038/47412">10.1038/47412</a>Available: <a href="http://www.nature.com/articles/47412">http://www.nature.com/articles/47412</a>.
</div>
<div id="ref-Schreiber2002" class="csl-entry">
54. Schreiber SL, Bernstein BE: <strong><span class="nocase">Signaling Network Model of Chromatin</span></strong>. <em>Cell</em> 2002, <strong>111</strong>:771–778<a href="https://doi.org/10.1016/S0092-8674(02)01196-0">10.1016/S0092-8674(02)01196-0</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0092867402011960">https://linkinghub.elsevier.com/retrieve/pii/S0092867402011960</a>.
</div>
<div id="ref-Voigt2012" class="csl-entry">
55. Voigt P, LeRoy G, Drury WJ, Zee BM, Son J, Beck DB, Young NL, Garcia BA, Reinberg D: <strong><span class="nocase">Asymmetrically modified nucleosomes.</span></strong> <em>Cell</em> 2012, <strong>151</strong>:181–93<a href="https://doi.org/10.1016/j.cell.2012.09.002">10.1016/j.cell.2012.09.002</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23021224">http://www.ncbi.nlm.nih.gov/pubmed/23021224</a>.
</div>
<div id="ref-Moore2020" class="csl-entry">
56. Moore JE, Purcaro MJ, Pratt HE, Epstein CB, Shoresh N, Adrian J, Kawli T, Davis CA, Dobin A, Kaul R, Halow J, Van Nostrand EL, Freese P, Gorkin DU, Shen Y, He Y, Mackiewicz M, Pauli-Behn F, Williams BA, Mortazavi A, Keller CA, Zhang X-O, Elhajjajy SI, Huey J, Dickel DE, Snetkova V, Wei X, Wang X, Rivera-Mulia JC, Rozowsky J, et al.: <strong><span class="nocase">Expanded encyclopaedias of DNA elements in the human and mouse genomes</span></strong>. <em>Nature</em> 2020, <strong>583</strong>:699–710<a href="https://doi.org/10.1038/s41586-020-2493-4">10.1038/s41586-020-2493-4</a>Available: <a href="http://www.nature.com/articles/s41586-020-2493-4">http://www.nature.com/articles/s41586-020-2493-4</a>.
</div>
<div id="ref-Hoffman2012" class="csl-entry">
57. Hoffman MM, Buske OJ, Wang J, Weng Z, Bilmes Ja, Noble WS: <strong><span class="nocase">Unsupervised pattern discovery in human chromatin structure through genomic segmentation.</span></strong> <em>Nature methods</em> 2012, <strong>9</strong>:473–6<a href="https://doi.org/10.1038/nmeth.1937">10.1038/nmeth.1937</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22426492">http://www.ncbi.nlm.nih.gov/pubmed/22426492</a>.
</div>
<div id="ref-Carrillo-de-Santa-Pau2017" class="csl-entry">
58. Carrillo-de-Santa-Pau E, Juan D, Pancaldi V, Were F, Martin-Subero I, Rico D, Valencia A: <strong><span class="nocase">Automatic identification of informative regions with epigenomic changes associated to hematopoiesis</span></strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:9244–9259<a href="https://doi.org/10.1093/nar/gkx618">10.1093/nar/gkx618</a>Available: <a href="https://academic.oup.com/nar/article/45/16/9244/3976483">https://academic.oup.com/nar/article/45/16/9244/3976483</a>.
</div>
<div id="ref-Talbert2012" class="csl-entry">
59. Talbert PB, Ahmad K, Almouzni G, Ausió J, Berger F, Bhalla PL, Bonner WM, Cande W, Chadwick BP, Chan SWL, Cross GAM, Cui L, Dimitrov SI, Doenecke D, Eirin-López JM, Gorovsky MA, Hake SB, Hamkalo BA, Holec S, Jacobsen SE, Kamieniarz K, Khochbin S, Ladurner AG, Landsman D, Latham JA, Loppin B, Malik HS, Marzluff WF, Pehrson JR, Postberg J, et al.: <strong><span class="nocase">A unified phylogeny-based nomenclature for histone variants</span></strong>. <em>Epigenetics <span>&amp;</span> Chromatin</em> 2012, <strong>5</strong>:7<a href="https://doi.org/10.1186/1756-8935-5-7">10.1186/1756-8935-5-7</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/1756-8935-5-7">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/1756-8935-5-7</a>.
</div>
<div id="ref-Weber2014" class="csl-entry">
60. Weber CM, Henikoff S: <strong><span class="nocase">Histone variants: dynamic punctuation in transcription.</span></strong> <em>Genes <span>&amp;</span> development</em> 2014, <strong>28</strong>:672–82<a href="https://doi.org/10.1101/gad.238873.114">10.1101/gad.238873.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24696452">http://www.ncbi.nlm.nih.gov/pubmed/24696452</a>.
</div>
<div id="ref-Kaikkonen2011" class="csl-entry">
61. Kaikkonen MU, Lam MTY, Glass CK: <strong><span class="nocase">Non-coding RNAs as regulators of gene expression and epigenetics.</span></strong> <em>Cardiovascular research</em> 2011, <strong>90</strong>:430–40<a href="https://doi.org/10.1093/cvr/cvr097">10.1093/cvr/cvr097</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21558279">http://www.ncbi.nlm.nih.gov/pubmed/21558279</a>.
</div>
<div id="ref-Cerase2015" class="csl-entry">
62. Cerase A, Pintacuda G, Tattermusch A, Avner P: <strong><span class="nocase">Xist localization and function: new insights from multiple levels</span></strong>. <em>Genome Biology</em> 2015, <strong>16</strong>:166<a href="https://doi.org/10.1186/s13059-015-0733-y">10.1186/s13059-015-0733-y</a>Available: <a href="http://genomebiology.com/2015/16/1/166">http://genomebiology.com/2015/16/1/166</a>.
</div>
<div id="ref-Haluskova2010" class="csl-entry">
63. Jenuwein T: <strong><span class="nocase">Translating the Histone Code</span></strong>. <em>Science</em> 2001, <strong>293</strong>:1074–1080<a href="https://doi.org/10.1126/science.1063127">10.1126/science.1063127</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20653993">http://www.ncbi.nlm.nih.gov/pubmed/20653993</a>.
</div>
<div id="ref-Smith2015" class="csl-entry">
64. Smith RWA, Monroe C, Bolnick DA: <strong><span class="nocase">Detection of Cytosine methylation in ancient DNA from five native american populations using bisulfite sequencing.</span></strong> <em>PloS one</em> 2015, <strong>10</strong>:e0125344<a href="https://doi.org/10.1371/journal.pone.0125344">10.1371/journal.pone.0125344</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26016479">http://www.ncbi.nlm.nih.gov/pubmed/26016479</a>.
</div>
<div id="ref-Pedersen2014" class="csl-entry">
65. Pedersen JS, Valen E, Velazquez AMV, Parker BJ, Rasmussen M, Lindgreen S, Lilje B, Tobin DJ, Kelly TK, Vang S, Andersson R, Jones PA, Hoover CA, Tikhonov A, Prokhortchouk E, Rubin EM, Sandelin A, Gilbert MTP, Krogh A, Willerslev E, Orlando L: <strong><span class="nocase">Genome-wide nucleosome map and cytosine methylation levels of an ancient human genome</span></strong>. <em>Genome Research</em> 2014, <strong>24</strong>:454–466<a href="https://doi.org/10.1101/gr.163592.113">10.1101/gr.163592.113</a>Available: <a href="http://genome.cshlp.org/cgi/doi/10.1101/gr.163592.113">http://genome.cshlp.org/cgi/doi/10.1101/gr.163592.113</a>.
</div>
<div id="ref-Briggs2010" class="csl-entry">
66. Briggs AW, Stenzel U, Meyer M, Krause J, Kircher M, Pääbo S: <strong><span class="nocase">Removal of deaminated cytosines and detection of in vivo methylation in ancient DNA.</span></strong> <em>Nucleic acids research</em> 2010, <strong>38</strong>:e87<a href="https://doi.org/10.1093/nar/gkp1163">10.1093/nar/gkp1163</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20028723">http://www.ncbi.nlm.nih.gov/pubmed/20028723</a>.
</div>
<div id="ref-Bauden2017" class="csl-entry">
67. Bauden M, Kristl T, Andersson R, Marko-Varga G, Ansari D: <strong><span class="nocase">Characterization of histone-related chemical modifications in formalin-fixed paraffin-embedded and fresh-frozen human pancreatic cancer xenografts using LC-MS/MS</span></strong>. <em>Laboratory Investigation</em> 2017, <strong>97</strong>:279–288<a href="https://doi.org/10.1038/labinvest.2016.134">10.1038/labinvest.2016.134</a>Available: <a href="http://www.nature.com/doifinder/10.1038/labinvest.2016.134">http://www.nature.com/doifinder/10.1038/labinvest.2016.134</a>.
</div>
<div id="ref-Hashimshony2003" class="csl-entry">
68. Hashimshony T, Zhang J, Keshet I, Bustin M, Cedar H: <strong><span class="nocase">The role of DNA methylation in setting up chromatin structure during development</span></strong>. <em>Nature Genetics</em> 2003, <strong>34</strong>:187–192<a href="https://doi.org/10.1038/ng1158">10.1038/ng1158</a>Available: <a href="http://www.nature.com/articles/ng1158">http://www.nature.com/articles/ng1158</a>.
</div>
<div id="ref-Esteve2006" class="csl-entry">
69. Estève P-O, Chin HG, Smallwood A, Feehery GR, Gangisetty O, Karpf AR, Carey MF, Pradhan S: <strong><span class="nocase">Direct interaction between DNMT1 and G9a coordinates DNA and histone methylation during replication.</span></strong> <em>Genes <span>&amp;</span> development</em> 2006, <strong>20</strong>:3089–103<a href="https://doi.org/10.1101/gad.1463706">10.1101/gad.1463706</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17085482">http://www.ncbi.nlm.nih.gov/pubmed/17085482</a>.
</div>
<div id="ref-Rose2014" class="csl-entry">
70. Rose NR, Klose RJ: <strong><span class="nocase">Understanding the relationship between DNA methylation and histone lysine methylation</span></strong>. <em>Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</em> 2014, <strong>1839</strong>:1362–1372<a href="https://doi.org/10.1016/j.bbagrm.2014.02.007">10.1016/j.bbagrm.2014.02.007</a>Available: <a href="http://dx.doi.org/10.1016/j.bbagrm.2014.02.007">http://dx.doi.org/10.1016/j.bbagrm.2014.02.007</a>.
</div>
<div id="ref-Luo2014" class="csl-entry">
71. Luo Y, Lu X, Xie H: <strong><span class="nocase">Dynamic Alu Methylation during Normal Development, Aging, and Tumorigenesis</span></strong>. <em>BioMed Research International</em> 2014, <strong>2014</strong>:1–12<a href="https://doi.org/10.1155/2014/784706">10.1155/2014/784706</a>Available: <a href="http://www.hindawi.com/journals/bmri/2014/784706/">http://www.hindawi.com/journals/bmri/2014/784706/</a>.
</div>
<div id="ref-Lander2001" class="csl-entry">
72. Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, Baldwin J, Devon K, Dewar K, Doyle M, FitzHugh W, Funke R, Gage D, Harris K, Heaford A, Howland J, Kann L, Lehoczky J, LeVine R, McEwan P, McKernan K, Meldrim J, Mesirov JP, Miranda C, Morris W, Naylor J, Raymond C, Rosetti M, Santos R, Sheridan A, Sougnez C, et al.: <strong><span class="nocase">Initial sequencing and analysis of the human genome.</span></strong> <em>Nature</em> 2001, <strong>409</strong>:860–921<a href="https://doi.org/10.1038/35057062">10.1038/35057062</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011">http://www.ncbi.nlm.nih.gov/pubmed/11237011</a>.
</div>
<div id="ref-Duncan1980" class="csl-entry">
73. Duncan BK, Miller JH: <strong><span class="nocase">Mutagenic deamination of cytosine residues in DNA.</span></strong> <em>Nature</em> 1980, <strong>287</strong>:560–1Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6999365">http://www.ncbi.nlm.nih.gov/pubmed/6999365</a>.
</div>
<div id="ref-Zhou2020" class="csl-entry">
74. Zhou Y, He F, Pu W, Gu X, Wang J, Su Z: <strong><span class="nocase">The Impact of DNA Methylation Dynamics on the Mutation Rate During Human Germline Development</span></strong>. <em>G3<span>&amp;</span><span>#</span>58; Genes|Genomes|Genetics</em> 2020, <strong>10</strong>:3337–3346<a href="https://doi.org/10.1534/g3.120.401511">10.1534/g3.120.401511</a>Available: <a href="http://g3journal.org/lookup/doi/10.1534/g3.120.401511">http://g3journal.org/lookup/doi/10.1534/g3.120.401511</a>.
</div>
<div id="ref-Guo2005" class="csl-entry">
75. Guo Y, Jamison DC: <strong><span class="nocase">The distribution of SNPs in human gene regulatory regions</span></strong>. <em>BMC Genomics</em> 2005, <strong>6</strong>:140<a href="https://doi.org/10.1186/1471-2164-6-140">10.1186/1471-2164-6-140</a>Available: <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-6-140">https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-6-140</a>.
</div>
<div id="ref-Wu2010" class="csl-entry">
76. Wu H, Caffo B, Jaffee HA, Irizarry RA, Feinberg AP: <strong><span class="nocase">Redefining CpG islands using hidden Markov models</span></strong>. <em>Biostatistics</em> 2010, <strong>11</strong>:499–514<a href="https://doi.org/10.1093/biostatistics/kxq005">10.1093/biostatistics/kxq005</a>Available: <a href="https://academic.oup.com/biostatistics/article-lookup/doi/10.1093/biostatistics/kxq005">https://academic.oup.com/biostatistics/article-lookup/doi/10.1093/biostatistics/kxq005</a>.
</div>
<div id="ref-Gardiner-Garden1987" class="csl-entry">
77. Gardiner-Garden M, Frommer M: <strong><span class="nocase">CpG Islands in vertebrate genomes</span></strong>. <em>Journal of Molecular Biology</em> 1987, <strong>196</strong>:261–282<a href="https://doi.org/10.1016/0022-2836(87)90689-9">10.1016/0022-2836(87)90689-9</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/0022283687906899">https://linkinghub.elsevier.com/retrieve/pii/0022283687906899</a>.
</div>
<div id="ref-Saxonov2006" class="csl-entry">
78. Saxonov S, Berg P, Brutlag DL: <strong><span class="nocase">A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2006, <strong>103</strong>:1412–1417<a href="https://doi.org/10.1073/pnas.0510310103">10.1073/pnas.0510310103</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.0510310103">http://www.pnas.org/cgi/doi/10.1073/pnas.0510310103</a>.
</div>
<div id="ref-Bell2017" class="csl-entry">
79. Bell JSK, Vertino PM: <strong><span class="nocase">Orphan CpG islands define a novel class of highly active enhancers.</span></strong> <em>Epigenetics</em> 2017, <strong>12</strong>:449–464<a href="https://doi.org/10.1080/15592294.2017.1297910">10.1080/15592294.2017.1297910</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28448736">http://www.ncbi.nlm.nih.gov/pubmed/28448736</a>.
</div>
<div id="ref-Illingworth2010" class="csl-entry">
80. Illingworth RS, Gruenewald-Schneider U, Webb S, Kerr ARW, James KD, Turner DJ, Smith C, Harrison DJ, Andrews R, Bird AP: <strong><span class="nocase">Orphan CpG islands identify numerous conserved promoters in the mammalian genome.</span></strong> <em>PLoS genetics</em> 2010, <strong>6</strong>:e1001134<a href="https://doi.org/10.1371/journal.pgen.1001134">10.1371/journal.pgen.1001134</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20885785">http://www.ncbi.nlm.nih.gov/pubmed/20885785</a>.
</div>
<div id="ref-Maunakea2010" class="csl-entry">
81. Maunakea AK, Nagarajan RP, Bilenky M, Ballinger TJ, D’Souza C, Fouse SD, Johnson BE, Hong C, Nielsen C, Zhao Y, Turecki G, Delaney A, Varhol R, Thiessen N, Shchors K, Heine VM, Rowitch DH, Xing X, Fiore C, Schillebeeckx M, Jones SJM, Haussler D, Marra MA, Hirst M, Wang T, Costello JF: <strong><span class="nocase">Conserved role of intragenic DNA methylation in regulating alternative promoters.</span></strong> <em>Nature</em> 2010, <strong>466</strong>:253–7<a href="https://doi.org/10.1038/nature09165">10.1038/nature09165</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20613842">http://www.ncbi.nlm.nih.gov/pubmed/20613842</a>.
</div>
<div id="ref-Ehrlich1982" class="csl-entry">
82. Ehrlich M, Gama-Sosa MA, Huang L-H, Midgett RM, Kuo KC, McCune RA, Gehrke C: <strong><span class="nocase">Amount and distribution of 5-methylcytosine in human DNA from different types of tissues or cells</span></strong>. <em>Nucleic Acids Research</em> 1982, <strong>10</strong>:2709–2721<a href="https://doi.org/10.1093/nar/10.8.2709">10.1093/nar/10.8.2709</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/10.8.2709">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/10.8.2709</a>.
</div>
<div id="ref-Bird2002" class="csl-entry">
83. Bird A: <strong><span class="nocase">DNA methylation patterns and epigenetic memory.</span></strong> <em>Genes <span>&amp;</span> development</em> 2002, <strong>16</strong>:6–21<a href="https://doi.org/10.1101/gad.947102">10.1101/gad.947102</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11782440">http://www.ncbi.nlm.nih.gov/pubmed/11782440</a>.
</div>
<div id="ref-Stadler2011" class="csl-entry">
84. Stadler MB, Murr R, Burger L, Ivanek R, Lienert F, Schöler A, Nimwegen E van, Wirbelauer C, Oakeley EJ, Gaidatzis D, Tiwari VK, Schübeler D: <strong><span class="nocase">DNA-binding factors shape the mouse methylome at distal regulatory regions.</span></strong> <em>Nature</em> 2011, <strong>480</strong>:490–5<a href="https://doi.org/10.1038/nature10716">10.1038/nature10716</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22170606">http://www.ncbi.nlm.nih.gov/pubmed/22170606</a>.
</div>
<div id="ref-Irizarry2009" class="csl-entry">
85. Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, Onyango P, Cui H, Gabo K, Rongione M, Webster M, Ji H, Potash J, Sabunciyan S, Feinberg AP: <strong><span class="nocase">The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores.</span></strong> <em>Nature genetics</em> 2009, <strong>41</strong>:178–186<a href="https://doi.org/10.1038/ng.298">10.1038/ng.298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151715">http://www.ncbi.nlm.nih.gov/pubmed/19151715</a>.
</div>
<div id="ref-Ziller2013" class="csl-entry">
86. Ziller MJ, Gu H, Müller F, Donaghey J, Tsai LT-Y, Kohlbacher O, De Jager PL, Rosen ED, Bennett DA, Bernstein BE, Gnirke A, Meissner A: <strong><span class="nocase">Charting a dynamic DNA methylation landscape of the human genome.</span></strong> <em>Nature</em> 2013, <strong>500</strong>:477–81<a href="https://doi.org/10.1038/nature12433">10.1038/nature12433</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23925113">http://www.ncbi.nlm.nih.gov/pubmed/23925113</a>.
</div>
<div id="ref-Baubec2014" class="csl-entry">
87. Baubec T, Schübeler D: <strong><span class="nocase">Genomic patterns and context specific interpretation of DNA methylation</span></strong>. <em>Current Opinion in Genetics <span>&amp;</span> Development</em> 2014, <strong>25</strong>:85–92<a href="https://doi.org/10.1016/j.gde.2013.11.015">10.1016/j.gde.2013.11.015</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24614011">http://www.ncbi.nlm.nih.gov/pubmed/24614011</a>.
</div>
<div id="ref-Yin2017" class="csl-entry">
88. Yin Y, Morgunova E, Jolma A, Kaasinen E, Sahu B, Khund-Sayeed S, Das PK, Kivioja T, Dave K, Zhong F, Nitta KR, Taipale M, Popov A, Ginno PA, Domcke S, Yan J, Schübeler D, Vinson C, Taipale J: <strong><span class="nocase">Impact of cytosine methylation on DNA binding specificities of human transcription factors.</span></strong> <em>Science (New York, N.Y.)</em> 2017, <strong>356</strong>:eaaj2239<a href="https://doi.org/10.1126/science.aaj2239">10.1126/science.aaj2239</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28473536">http://www.ncbi.nlm.nih.gov/pubmed/28473536</a>.
</div>
<div id="ref-Zuo2017" class="csl-entry">
89. Zuo Z, Roy B, Chang YK, Granas D, Stormo GD: <strong><span class="nocase">Measuring quantitative effects of methylation on transcription factor–DNA binding affinity</span></strong>. <em>Science Advances</em> 2017, <strong>3</strong>:eaao1799<a href="https://doi.org/10.1126/sciadv.aao1799">10.1126/sciadv.aao1799</a>Available: <a href="http://advances.sciencemag.org/lookup/doi/10.1126/sciadv.aao1799">http://advances.sciencemag.org/lookup/doi/10.1126/sciadv.aao1799</a>.
</div>
<div id="ref-Kelly2012" class="csl-entry">
90. Kelly TK, Liu Y, Lay FD, Liang G, Berman BP, Jones Pa: <strong><span class="nocase">Genome-wide mapping of nucleosome positioning and DNA methylation within individual DNA molecules</span></strong>. <em>Genome Research</em> 2012, <strong>22</strong>:2497–2506<a href="https://doi.org/10.1101/gr.143008.112">10.1101/gr.143008.112</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22960375">http://www.ncbi.nlm.nih.gov/pubmed/22960375</a>.
</div>
<div id="ref-Huff2014" class="csl-entry">
91. Huff JT, Zilberman D: <strong><span class="nocase">Dnmt1-Independent CG Methylation Contributes to Nucleosome Positioning in Diverse Eukaryotes</span></strong>. <em>Cell</em> 2014, <strong>156</strong>:1286–1297<a href="https://doi.org/10.1016/j.cell.2014.01.029">10.1016/j.cell.2014.01.029</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24630728">http://www.ncbi.nlm.nih.gov/pubmed/24630728</a>.
</div>
<div id="ref-Raiber2017" class="csl-entry">
92. Raiber E, Portella G, Martínez Cuesta S, Hardisty R, Murat P, Li Z, Iurlaro M, Dean W, Spindel J, Beraldi D, Liu Z, Dawson MA, Reik W, Balasubramanian S: <strong><span class="nocase">5-Formylcytosine organizes nucleosomes and forms Schiff base interactions with histones in mouse embryonic stem cells</span></strong>. <em>Nature Chemistry</em> 2018, <strong>10</strong>:1258–1266<a href="https://doi.org/10.1038/s41557-018-0149-x">10.1038/s41557-018-0149-x</a>Available: <a href="http://www.nature.com/articles/s41557-018-0149-x">http://www.nature.com/articles/s41557-018-0149-x</a>.
</div>
<div id="ref-Lyko2017" class="csl-entry">
93. Lyko F: <strong><span class="nocase">The DNA methyltransferase family: a versatile toolkit for epigenetic regulation</span></strong>. <em>Nature Reviews Genetics</em> 2017, <strong>19</strong>:81–92<a href="https://doi.org/10.1038/nrg.2017.80">10.1038/nrg.2017.80</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.80">http://www.nature.com/doifinder/10.1038/nrg.2017.80</a>.
</div>
<div id="ref-Vertino2002" class="csl-entry">
94. Vertino PM, Sekowski JA, Coll JM, Applegren N, Han S, Hickey RJ, Malkas LH: <strong><span class="nocase">DNMT1 is a component of a multiprotein DNA replication complex.</span></strong> <em>Cell cycle (Georgetown, Tex.)</em> 2002, <strong>1</strong>:416–23<a href="https://doi.org/10.4161/cc.1.6.270">10.4161/cc.1.6.270</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12548018">http://www.ncbi.nlm.nih.gov/pubmed/12548018</a>.
</div>
<div id="ref-Bostick2007" class="csl-entry">
95. Bostick M, Kim JK, Esteve P-O, Clark A, Pradhan S, Jacobsen SE: <strong><span class="nocase">UHRF1 Plays a Role in Maintaining DNA Methylation in Mammalian Cells</span></strong>. <em>Science</em> 2007, <strong>317</strong>:1760–1764<a href="https://doi.org/10.1126/science.1147939">10.1126/science.1147939</a>Available: <a href="https://www.sciencemag.org/lookup/doi/10.1126/science.1147939">https://www.sciencemag.org/lookup/doi/10.1126/science.1147939</a>.
</div>
<div id="ref-Jia2009" class="csl-entry">
96. Jia D, Jurkowska RZ, Zhang X, Jeltsch A, Cheng X: <strong><span class="nocase">Structure of Dnmt3a bound to Dnmt3L suggests a model for de novo DNA methylation</span></strong>. <em>Nature</em> 2007, <strong>449</strong>:248–251<a href="https://doi.org/10.1038/nature06146">10.1038/nature06146</a>Available: <a href="http://www.nature.com/articles/nature06146">http://www.nature.com/articles/nature06146</a>.
</div>
<div id="ref-Kaiser2017" class="csl-entry">
97. Kaiser S, Jurkowski TP, Kellner S, Schneider D, Jeltsch A, Helm M: <strong><span class="nocase">The RNA methyltransferase Dnmt2 methylates DNA in the structural context of a tRNA</span></strong>. <em>RNA Biology</em> 2017, <strong>14</strong>:1241–1251<a href="https://doi.org/10.1080/15476286.2016.1236170">10.1080/15476286.2016.1236170</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15476286.2016.1236170">https://www.tandfonline.com/doi/full/10.1080/15476286.2016.1236170</a>.
</div>
<div id="ref-Tahiliani2009" class="csl-entry">
98. Tahiliani M, Koh KP, Shen Y, Pastor Wa, Bandukwala H, Brudno Y, Agarwal S, Iyer LM, Liu DR, Aravind L, Rao A: <strong><span class="nocase">Conversion of 5-Methylcytosine to 5-Hydroxymethylcytosine in Mammalian DNA by MLL Partner TET1</span></strong>. <em>Science</em> 2009, <strong>324</strong>:930–935<a href="https://doi.org/10.1126/science.1170116">10.1126/science.1170116</a>Available: <a href="http://www.sciencemag.org/cgi/doi/10.1126/science.1170116">http://www.sciencemag.org/cgi/doi/10.1126/science.1170116</a>.
</div>
<div id="ref-Kandoth2013" class="csl-entry">
99. Kandoth C, McLellan MD, Vandin F, Ye K, Niu B, Lu C, Xie M, Zhang Q, McMichael JF, Wyczalkowski MA, Leiserson MDM, Miller CA, Welch JS, Walter MJ, Wendl MC, Ley TJ, Wilson RK, Raphael BJ, Ding L: <strong><span class="nocase">Mutational landscape and significance across 12 major cancer types</span></strong>. <em>Nature</em> 2013, <strong>502</strong>:333–339<a href="https://doi.org/10.1038/nature12634">10.1038/nature12634</a>.
</div>
<div id="ref-Ko2010a" class="csl-entry">
100. Ko M, Huang Y, Jankowska AM, Pape UJ, Tahiliani M, Bandukwala HS, An J, Lamperti ED, Koh KP, Ganetzky R, Liu XS, Aravind L, Agarwal S, Maciejewski JP, Rao A: <strong><span class="nocase">Impaired hydroxylation of 5-methylcytosine in myeloid cancers with mutant TET2</span></strong>. <em>Nature</em> 2010, <strong>468</strong>:839–843<a href="https://doi.org/10.1038/nature09586">10.1038/nature09586</a>Available: <a href="http://www.nature.com/articles/nature09586">http://www.nature.com/articles/nature09586</a>.
</div>
<div id="ref-Wu2017" class="csl-entry">
101. Wu X, Zhang Y: <strong><span class="nocase">TET-mediated active DNA demethylation: mechanism, function and beyond</span></strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:517–534<a href="https://doi.org/10.1038/nrg.2017.33">10.1038/nrg.2017.33</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.33">http://www.nature.com/doifinder/10.1038/nrg.2017.33</a>.
</div>
<div id="ref-Lienert2011" class="csl-entry">
102. Lienert F, Wirbelauer C, Som I, Dean A, Mohn F, Schübeler D: <strong><span class="nocase">Identification of genetic elements that autonomously determine DNA methylation states.</span></strong> <em>Nature genetics</em> 2011, <strong>43</strong>:1091–7<a href="https://doi.org/10.1038/ng.946">10.1038/ng.946</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21964573">http://www.ncbi.nlm.nih.gov/pubmed/21964573</a>.
</div>
<div id="ref-Boks2009" class="csl-entry">
103. Boks MP, Derks EM, Weisenberger DJ, Strengman E, Janson E, Sommer IE, Kahn RS, Ophoff RA: <strong><span class="nocase">The relationship of DNA methylation with age, gender and genotype in twins and healthy controls.</span></strong> <em>PloS one</em> 2009, <strong>4</strong>:e6767<a href="https://doi.org/10.1371/journal.pone.0006767">10.1371/journal.pone.0006767</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19774229">http://www.ncbi.nlm.nih.gov/pubmed/19774229</a>.
</div>
<div id="ref-Kerkel2008" class="csl-entry">
104. Kerkel K, Spadola A, Yuan E, Kosek J, Jiang L, Hod E, Li K, Murty VV, Schupf N, Vilain E, Morris M, Haghighi F, Tycko B: <strong><span class="nocase">Genomic surveys by methylation-sensitive SNP analysis identify sequence-dependent allele-specific DNA methylation.</span></strong> <em>Nature genetics</em> 2008, <strong>40</strong>:904–8<a href="https://doi.org/10.1038/ng.174">10.1038/ng.174</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18568024">http://www.ncbi.nlm.nih.gov/pubmed/18568024</a>.
</div>
<div id="ref-Schalkwyk2010" class="csl-entry">
105. Schalkwyk LC, Meaburn EL, Smith R, Dempster EL, Jeffries AR, Davies MN, Plomin R, Mill J: <strong><span class="nocase">Allelic skewing of DNA methylation is widespread across the genome.</span></strong> <em>American journal of human genetics</em> 2010, <strong>86</strong>:196–212<a href="https://doi.org/10.1016/j.ajhg.2010.01.014">10.1016/j.ajhg.2010.01.014</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20159110">http://www.ncbi.nlm.nih.gov/pubmed/20159110</a>.
</div>
<div id="ref-Shoemaker2010" class="csl-entry">
106. Shoemaker R, Deng J, Wang W, Zhang K: <strong><span class="nocase">Allele-specific methylation is prevalent and is contributed by CpG-SNPs in the human genome.</span></strong> <em>Genome research</em> 2010, <strong>20</strong>:883–9<a href="https://doi.org/10.1101/gr.104695.109">10.1101/gr.104695.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20418490">http://www.ncbi.nlm.nih.gov/pubmed/20418490</a>.
</div>
<div id="ref-Bell2017a" class="csl-entry">
107. Bell CG, Gao F, Yuan W, Roos L, Acton RJ, Xia Y, Bell J, Ward K, Mangino M, Hysi PG, Wang J, Spector TD: <strong><span class="nocase">Obligatory and facilitative allelic variation in the DNA methylome within common disease-associated loci</span></strong>. <em>Nature Communications</em> 2018, <strong>9</strong>:8<a href="https://doi.org/10.1038/s41467-017-01586-1">10.1038/s41467-017-01586-1</a>Available: <a href="http://www.nature.com/articles/s41467-017-01586-1">http://www.nature.com/articles/s41467-017-01586-1</a>.
</div>
<div id="ref-Martin-Trujillo2017" class="csl-entry">
108. Martin-Trujillo A, Vidal E, Monteagudo-Sánchez A, Sanchez-Delgado M, Moran S, Hernandez Mora JR, Heyn H, Guitart M, Esteller M, Monk D: <strong><span class="nocase">Copy number rather than epigenetic alterations are the major dictator of imprinted methylation in tumors</span></strong>. <em>Nature Communications</em> 2017, <strong>8</strong>:467<a href="https://doi.org/10.1038/s41467-017-00639-9">10.1038/s41467-017-00639-9</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28883545">http://www.ncbi.nlm.nih.gov/pubmed/28883545</a>.
</div>
<div id="ref-Ciernia2016" class="csl-entry">
109. Ciernia AV, LaSalle J: <strong><span class="nocase">The landscape of DNA methylation amid a perfect storm of autism aetiologies</span></strong>. <em>Nature Reviews Neuroscience</em> 2016, <strong>17</strong>:411–423<a href="https://doi.org/10.1038/nrn.2016.41">10.1038/nrn.2016.41</a>Available: <a href="http://dx.doi.org/10.1038/nrn.2016.41">http://dx.doi.org/10.1038/nrn.2016.41</a>.
</div>
<div id="ref-Ushijima2005" class="csl-entry">
110. Ushijima T, Watanabe N, Shimizu K, Miyamoto K, Sugimura T, Kaneda A: <strong><span class="nocase">Decreased fidelity in replicating CpG methylation patterns in cancer cells.</span></strong> <em>Cancer research</em> 2005, <strong>65</strong>:11–7Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15665274">http://www.ncbi.nlm.nih.gov/pubmed/15665274</a>.
</div>
<div id="ref-Kunkel2004a" class="csl-entry">
111. Kunkel TA: <strong><span>DNA Replication Fidelity</span></strong>. <em>Journal of Biological Chemistry</em> 2004, <strong>279</strong>:16895–16898<a href="https://doi.org/10.1074/jbc.R400006200">10.1074/jbc.R400006200</a>Available: <a href="http://www.jbc.org/lookup/doi/10.1074/jbc.R400006200">http://www.jbc.org/lookup/doi/10.1074/jbc.R400006200</a>.
</div>
<div id="ref-Sharif2007" class="csl-entry">
112. Sharif J, Muto M, Takebayashi S, Suetake I, Iwamatsu A, Endo TA, Shinga J, Mizutani-Koseki Y, Toyoda T, Okamura K, Tajima S, Mitsuya K, Okano M, Koseki H: <strong><span class="nocase">The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA</span></strong>. <em>Nature</em> 2007, <strong>450</strong>:908–912<a href="https://doi.org/10.1038/nature06397">10.1038/nature06397</a>Available: <a href="http://www.nature.com/articles/nature06397">http://www.nature.com/articles/nature06397</a>.
</div>
<div id="ref-Arita2008" class="csl-entry">
113. Arita K, Ariyoshi M, Tochio H, Nakamura Y, Shirakawa M: <strong><span class="nocase">Recognition of hemi-methylated DNA by the SRA protein UHRF1 by a base-flipping mechanism</span></strong>. <em>Nature</em> 2008, <strong>455</strong>:818–821<a href="https://doi.org/10.1038/nature07249">10.1038/nature07249</a>Available: <a href="http://www.nature.com/articles/nature07249">http://www.nature.com/articles/nature07249</a>.
</div>
<div id="ref-Liu2013b" class="csl-entry">
114. Liu X, Gao Q, Li P, Zhao Q, Zhang J, Li J, Koseki H, Wong J: <strong><span class="nocase">UHRF1 targets DNMT1 for DNA methylation through cooperative binding of hemi-methylated DNA and methylated H3K9</span></strong>. <em>Nature Communications</em> 2013, <strong>4</strong>:1563<a href="https://doi.org/10.1038/ncomms2562">10.1038/ncomms2562</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23463006">http://www.ncbi.nlm.nih.gov/pubmed/23463006</a>.
</div>
<div id="ref-Laird2004" class="csl-entry">
115. Laird CD, Pleasant ND, Clark AD, Sneeden JL, Hassan KMA, Manley NC, Vary JC, Morgan T, Hansen RS, Stoger R: <strong><span class="nocase">Hairpin-bisulfite PCR: Assessing epigenetic methylation patterns on complementary strands of individual DNA molecules</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2004, <strong>101</strong>:204–209<a href="https://doi.org/10.1073/pnas.2536758100">10.1073/pnas.2536758100</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.2536758100">http://www.pnas.org/cgi/doi/10.1073/pnas.2536758100</a>.
</div>
<div id="ref-Pfeifer1990a" class="csl-entry">
116. Pfeifer GP, Steigerwald SD, Hansen RS, Gartler SM, Riggs AD: <strong><span class="nocase">Polymerase chain reaction-aided genomic sequencing of an X chromosome-linked CpG island: methylation patterns suggest clonal inheritance, CpG site autonomy, and an explanation of activity state stability.</span></strong> <em>Proceedings of the National Academy of Sciences</em> 1990, <strong>87</strong>:8252–8256<a href="https://doi.org/10.1073/pnas.87.21.8252">10.1073/pnas.87.21.8252</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.87.21.8252">http://www.pnas.org/cgi/doi/10.1073/pnas.87.21.8252</a>.
</div>
<div id="ref-Riggs2004a" class="csl-entry">
117. Riggs AD, Xiong Z: <strong><span class="nocase">Methylation and epigenetic fidelity</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2004, <strong>101</strong>:4–5<a href="https://doi.org/10.1073/pnas.0307781100">10.1073/pnas.0307781100</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/14695893">http://www.ncbi.nlm.nih.gov/pubmed/14695893</a>.
</div>
<div id="ref-Jenkinson2017" class="csl-entry">
118. Jenkinson G, Pujadas E, Goutsias J, Feinberg AP: <strong><span class="nocase">Potential energy landscapes identify the information-theoretic nature of the epigenome.</span></strong> <em>Nature genetics</em> 2017, <strong>49</strong>:719–729<a href="https://doi.org/10.1038/ng.3811">10.1038/ng.3811</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28346445">http://www.ncbi.nlm.nih.gov/pubmed/28346445</a>.
</div>
<div id="ref-Fraga2005" class="csl-entry">
119. Fraga MF, Ballestar E, Paz MF, Ropero S, Setien F, Ballestar ML, Heine-Suner D, Cigudosa JC, Urioste M, Benitez J, Boix-Chornet M, Sanchez-Aguilera A, Ling C, Carlsson E, Poulsen P, Vaag A, Stephan Z, Spector TD, Wu Y-Z, Plass C, Esteller M: <strong><span class="nocase">From The Cover: Epigenetic differences arise during the lifetime of monozygotic twins</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2005, <strong>102</strong>:10604–10609<a href="https://doi.org/10.1073/pnas.0500398102">10.1073/pnas.0500398102</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16009939">http://www.ncbi.nlm.nih.gov/pubmed/16009939</a>.
</div>
<div id="ref-Slieker2016" class="csl-entry">
120. Slieker RC, Iterson M van, Luijk R, Beekman M, Zhernakova DV, Moed MH, Mei H, Galen M van, Deelen P, Bonder MJ, Zhernakova A, Uitterlinden AG, Tigchelaar EF, Stehouwer CDA, Schalkwijk CG, Kallen CJH van der, Hofman A, Heemst D van, Geus EJ de, Dongen J van, Deelen J, Berg LH van den, Meurs J van, Jansen R, ’t Hoen PAC, Franke L, Wijmenga C, Veldink JH, Swertz MA, Greevenbroek MMJ van, et al.: <strong><span class="nocase">Age-related accrual of methylomic variability is linked to fundamental ageing mechanisms.</span></strong> <em>Genome biology</em> 2016, <strong>17</strong>:191<a href="https://doi.org/10.1186/s13059-016-1053-6">10.1186/s13059-016-1053-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27654999">http://www.ncbi.nlm.nih.gov/pubmed/27654999</a>.
</div>
<div id="ref-Hahn2017" class="csl-entry">
121. Hahn O, Grönke S, Stubbs TM, Ficz G, Hendrich O, Krueger F, Andrews S, Zhang Q, Wakelam MJ, Beyer A, Reik W, Partridge L: <strong><span class="nocase">Dietary restriction protects from age-associated DNA methylation and induces epigenetic reprogramming of lipid metabolism</span></strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:56<a href="https://doi.org/10.1186/s13059-017-1187-1">10.1186/s13059-017-1187-1</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1187-1">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1187-1</a>.
</div>
<div id="ref-Zhao2014" class="csl-entry">
122. Zhao L, Sun M, Li Z, Bai X, Yu M, Wang M, Liang L, Shao X, Arnovitz S, Wang Q, He C, Lu X, Chen J, Xie H: <strong><span class="nocase">The dynamics of DNA methylation fidelity during mouse embryonic stem cell self-renewal and differentiation</span></strong>. <em>Genome Research</em> 2014, <strong>24</strong>:1296–1307<a href="https://doi.org/10.1101/gr.163147.113">10.1101/gr.163147.113</a>Available: <a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.163147.113">http://genome.cshlp.org/lookup/doi/10.1101/gr.163147.113</a>.
</div>
<div id="ref-Zhou2018" class="csl-entry">
123. Zhou W, Dinh HQ, Ramjan Z, Weisenberger DJ, Nicolet CM, Shen H, Laird PW, Berman BP: <strong><span class="nocase">DNA methylation loss in late-replicating domains is linked to mitotic cell division</span></strong>. <em>Nature Genetics</em> 2018, <strong>50</strong>:591–602<a href="https://doi.org/10.1038/s41588-018-0073-4">10.1038/s41588-018-0073-4</a>Available: <a href="http://dx.doi.org/10.1038/s41588-018-0073-4">http://dx.doi.org/10.1038/s41588-018-0073-4</a>.
</div>
<div id="ref-Hatada1991" class="csl-entry">
124. Hatada I, Hayashizaki Y, Hirotsune S, Komatsubara H, Mukai T: <strong><span class="nocase">A genomic scanning method for higher organisms using restriction sites as landmarks.</span></strong> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 1991, <strong>88</strong>:9523–7<a href="https://doi.org/10.1073/pnas.88.21.9523">10.1073/pnas.88.21.9523</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/1946366">http://www.ncbi.nlm.nih.gov/pubmed/1946366</a>.
</div>
<div id="ref-Weber2005" class="csl-entry">
125. Weber M, Davies JJ, Wittig D, Oakeley EJ, Haase M, Lam WL, Schübeler D: <strong><span class="nocase">Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells</span></strong>. <em>Nature Genetics</em> 2005, <strong>37</strong>:853–862<a href="https://doi.org/10.1038/ng1598">10.1038/ng1598</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16007088">http://www.ncbi.nlm.nih.gov/pubmed/16007088</a>.
</div>
<div id="ref-Harrison2011" class="csl-entry">
126. Harrison A, Parle-McDermott A: <strong><span class="nocase">DNA Methylation: A Timeline of Methods and Applications</span></strong>. <em>Frontiers in Genetics</em> 2011, <strong>2</strong>:632–649<a href="https://doi.org/10.3389/fgene.2011.00074">10.3389/fgene.2011.00074</a>Available: <a href="http://journal.frontiersin.org/article/10.3389/fgene.2011.00074/abstract">http://journal.frontiersin.org/article/10.3389/fgene.2011.00074/abstract</a>.
</div>
<div id="ref-Down2009" class="csl-entry">
127. Down TA, Rakyan VK, Turner DJ, Flicek P, Li H, Kulesha E, Gräf S, Johnson N, Herrero J, Tomazou EM, Thorne NP, Bäckdahl L, Herberth M, Howe KL, Jackson DK, Miretti MM, Marioni JC, Birney E, Hubbard TJP, Durbin R, Tavaré S, Beck S: <strong><span class="nocase">A Bayesian deconvolution strategy for immunoprecipitation-based DNA methylome analysis</span></strong>. <em>Nature Biotechnology</em> 2008, <strong>26</strong>:779–785<a href="https://doi.org/10.1038/nbt1414">10.1038/nbt1414</a>Available: <a href="http://www.nature.com/articles/nbt1414">http://www.nature.com/articles/nbt1414</a>.
</div>
<div id="ref-Lienhard2014" class="csl-entry">
128. Lienhard M, Grimm C, Morkel M, Herwig R, Chavez L: <strong><span class="nocase">MEDIPS: genome-wide differential coverage analysis of sequencing data derived from DNA enrichment experiments</span></strong>. <em>Bioinformatics</em> 2014, <strong>30</strong>:284–286<a href="https://doi.org/10.1093/bioinformatics/btt650">10.1093/bioinformatics/btt650</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24227674">http://www.ncbi.nlm.nih.gov/pubmed/24227674</a>.
</div>
<div id="ref-Serre2009a" class="csl-entry">
129. Serre D, Lee BH, Ting AH: <strong><span class="nocase">MBD-isolated Genome Sequencing provides a high-throughput and comprehensive survey of DNA methylation in the human genome</span></strong>. <em>Nucleic Acids Research</em> 2010, <strong>38</strong>:391–399<a href="https://doi.org/10.1093/nar/gkp992">10.1093/nar/gkp992</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp992">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkp992</a>.
</div>
<div id="ref-Li2015" class="csl-entry">
130. Li J, Yang J, Zhou P, Le Y, Zhou C, Wang S, Xu D, Lin H-K, Gong Z: <strong><span class="nocase">Circular RNAs in cancer: novel insights into origins, properties, functions and implications.</span></strong> <em>American journal of cancer research</em> 2015, <strong>5</strong>:472–80Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25973291">http://www.ncbi.nlm.nih.gov/pubmed/25973291</a>.
</div>
<div id="ref-Bock2010" class="csl-entry">
131. Bock C, Tomazou EM, Brinkman AB, Müller F, Simmer F, Gu H, Jäger N, Gnirke A, Stunnenberg HG, Meissner A: <strong><span class="nocase">Quantitative comparison of genome-wide DNA methylation mapping technologies</span></strong>. <em>Nature Biotechnology</em> 2010, <strong>28</strong>:1106–1114<a href="https://doi.org/10.1038/nbt.1681">10.1038/nbt.1681</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20852634">http://www.ncbi.nlm.nih.gov/pubmed/20852634</a>.
</div>
<div id="ref-Hayatsu1970" class="csl-entry">
132. Hayatsu H, Wataya Y, Kai K, Iida S: <strong><span class="nocase">Reaction of sodium bisulfite with uracil, cytosine, and their derivatives</span></strong>. <em>Biochemistry</em> 1970, <strong>9</strong>:2858–2865<a href="https://doi.org/10.1021/bi00816a016">10.1021/bi00816a016</a>Available: <a href="http://pubs.acs.org/doi/abs/10.1021/bi00816a016">http://pubs.acs.org/doi/abs/10.1021/bi00816a016</a>.
</div>
<div id="ref-Adamowicz2018" class="csl-entry">
133. Adamowicz M, Maratou K, Aitman TJ: <strong><span class="nocase">Multiplexed DNA Methylation Analysis of Target Regions Using Microfluidics (Fluidigm)</span></strong>. In <em>Methods in molecular biology (clifton, n.j.)</em> Vol. 1708 2018:349–363. Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29224153">http://www.ncbi.nlm.nih.gov/pubmed/29224153</a>.
</div>
<div id="ref-Skvortsova2017" class="csl-entry">
134. Skvortsova K, Zotenko E, Luu P-L, Gould CM, Nair SS, Clark SJ, Stirzaker C: <strong><span class="nocase">Comprehensive evaluation of genome-wide 5-hydroxymethylcytosine profiling approaches in human DNA</span></strong>. <em>Epigenetics <span>&amp;</span> Chromatin</em> 2017, <strong>10</strong>:16<a href="https://doi.org/10.1186/s13072-017-0123-7">10.1186/s13072-017-0123-7</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-017-0123-7">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-017-0123-7</a>.
</div>
<div id="ref-Williams2019" class="csl-entry">
135. Williams L, Bei Y, Church HE, Dai N, Dimalanta ET, Ettwiller LM, Evans TC, Langhorst BW, Borgaro JG, Guan S, Marks K, Menin JF, Nichols NM, Chaithanya Ponnaluri VK, Saleh L, Samaranayake M, Sexton BS, Sun Z, Tamanaha E, Vaisvila R, Yigit E, Davis TB: <strong><span class="nocase">Enzymatic Methyl-seq: The Next Generation of Methylome Analysis BISULFITE SEQUENCING ALTERNATIVE METHODS FOR DETECTING 5mC AND 5hmC</span></strong>. 2019, :2–5Available: <a href="https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis">https://international.neb.com/tools-and-resources/feature-articles/enzymatic-methyl-seq-the-next-generation-of-methylome-analysis</a>.
</div>
<div id="ref-Rhoads2015" class="csl-entry">
136. Rhoads A, Au KF: <strong><span class="nocase">PacBio Sequencing and Its Applications</span></strong>. <em>Genomics, Proteomics <span>&amp;</span> Bioinformatics</em> 2015, <strong>13</strong>:278–289<a href="https://doi.org/10.1016/j.gpb.2015.08.002">10.1016/j.gpb.2015.08.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1672022915001345">https://linkinghub.elsevier.com/retrieve/pii/S1672022915001345</a>.
</div>
<div id="ref-Simpson2017" class="csl-entry">
137. Simpson JT, Workman RE, Zuzarte PC, David M, Dursi LJ, Timp W: <strong><span class="nocase">Detecting DNA cytosine methylation using nanopore sequencing</span></strong>. <em>Nature Methods</em> 2017, <strong>14</strong>:407–410<a href="https://doi.org/10.1038/nmeth.4184">10.1038/nmeth.4184</a>Available: <a href="http://www.nature.com/articles/nmeth.4184">http://www.nature.com/articles/nmeth.4184</a>.
</div>
<div id="ref-Wilson1983" class="csl-entry">
138. Wilson VL, Jones PA: <strong><span class="nocase">DNA methylation decreases in aging but not in immortal cells.</span></strong> <em>Science (New York, N.Y.)</em> 1983, <strong>220</strong>:1055–7Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6844925">http://www.ncbi.nlm.nih.gov/pubmed/6844925</a>.
</div>
<div id="ref-Feinberg1983" class="csl-entry">
139. Feinberg AP, Vogelstein B: <strong><span class="nocase">Hypomethylation distinguishes genes of some human cancers from their normal counterparts</span></strong>. <em>Nature</em> 1983, <strong>301</strong>:89–92<a href="https://doi.org/10.1038/301089a0">10.1038/301089a0</a>Available: <a href="http://www.nature.com/articles/301089a0">http://www.nature.com/articles/301089a0</a>.
</div>
<div id="ref-Romanov1981" class="csl-entry">
140. Romanov GA, Vanyushin BF: <strong><span class="nocase">Methylation of reiterated sequences in mammalian DNAs. Effects of the tissue type, age, malignancy and hormonal induction.</span></strong> <em>Biochimica et biophysica acta</em> 1981, <strong>653</strong>:204–18Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7225396">http://www.ncbi.nlm.nih.gov/pubmed/7225396</a>.
</div>
<div id="ref-Berdyshev1967" class="csl-entry">
141. Berdyshev GD, Korotaev GK, Boiarskikh GV, Vaniushin BF: <strong><span class="nocase">Nucleotide composition of DNA and RNA from somatic tissues of humpback and its changes during spawning.</span></strong> <em>Biokhimiia (Moscow, Russia)</em> 1967, <strong>32</strong>:988–93Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5628601">http://www.ncbi.nlm.nih.gov/pubmed/5628601</a>.
</div>
<div id="ref-Wilson1987" class="csl-entry">
142. Wilson VL, Smith RA, Ma S, Cutler RG: <strong><span class="nocase">Genomic 5-methyldeoxycytidine decreases with age.</span></strong> <em>The Journal of biological chemistry</em> 1987, <strong>262</strong>:9948–51Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3611071">http://www.ncbi.nlm.nih.gov/pubmed/3611071</a>.
</div>
<div id="ref-Cole2017" class="csl-entry">
143. Cole JJ, Robertson NA, Rather MI, Thomson JP, McBryan T, Sproul D, Wang T, Brock C, Clark W, Ideker T, Meehan RR, Miller RA, Brown-Borg HM, Adams PD: <strong><span class="nocase">Diverse interventions that extend mouse lifespan suppress shared age-associated epigenetic changes at critical gene regulatory regions</span></strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:58<a href="https://doi.org/10.1186/s13059-017-1185-3">10.1186/s13059-017-1185-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1185-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1185-3</a>.
</div>
<div id="ref-Meissner2005" class="csl-entry">
144. Meissner A, Gnirke A, Bell GW, Ramsahoye B, Lander ES, Jaenisch R: <strong><span class="nocase">Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis.</span></strong> <em>Nucleic acids research</em> 2005, <strong>33</strong>:5868–77<a href="https://doi.org/10.1093/nar/gki901">10.1093/nar/gki901</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16224102">http://www.ncbi.nlm.nih.gov/pubmed/16224102</a>.
</div>
<div id="ref-Feinberg2004" class="csl-entry">
145. Feinberg AP, Tycko B: <strong><span class="nocase">The history of cancer epigenetics</span></strong>. <em>Nature Reviews Cancer</em> 2004, <strong>4</strong>:143–153<a href="https://doi.org/10.1038/nrc1279">10.1038/nrc1279</a>Available: <a href="http://www.nature.com/articles/nrc1279">http://www.nature.com/articles/nrc1279</a>.
</div>
<div id="ref-Knudson1971" class="csl-entry">
146. Knudson AG: <strong><span class="nocase">Mutation and cancer: statistical study of retinoblastoma.</span></strong> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 1971, <strong>68</strong>:820–3<a href="https://doi.org/10.1073/pnas.68.4.820">10.1073/pnas.68.4.820</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5279523">http://www.ncbi.nlm.nih.gov/pubmed/5279523</a>.
</div>
<div id="ref-Esteller2000" class="csl-entry">
147. Esteller M, Silva JM, Dominguez G, Bonilla F, Matias-Guiu X, Lerma E, Bussaglia E, Prat J, Harkes IC, Repasky EA, Gabrielson E, Schutte M, Baylin SB, Herman JG: <strong><span class="nocase">Promoter hypermethylation and BRCA1 inactivation in sporadic breast and ovarian tumors.</span></strong> <em>Journal of the National Cancer Institute</em> 2000, <strong>92</strong>:564–9<a href="https://doi.org/10.1093/jnci/92.7.564">10.1093/jnci/92.7.564</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10749912">http://www.ncbi.nlm.nih.gov/pubmed/10749912</a>.
</div>
<div id="ref-Feinberg2018" class="csl-entry">
148. Feinberg AP: <strong><span class="nocase">The Key Role of Epigenetics in Human Disease Prevention and Mitigation</span></strong>. <em>New England Journal of Medicine</em> 2018, <strong>378</strong>:1323–1334<a href="https://doi.org/10.1056/NEJMra1402513">10.1056/NEJMra1402513</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29617578">http://www.ncbi.nlm.nih.gov/pubmed/29617578</a>.
</div>
<div id="ref-Bibikova2006" class="csl-entry">
149. Bibikova M: <strong><span class="nocase">High-throughput DNA methylation profiling using universal bead arrays</span></strong>. <em>Genome Research</em> 2006, <strong>16</strong>:383–393<a href="https://doi.org/10.1101/gr.4410706">10.1101/gr.4410706</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.4410706">http://www.genome.org/cgi/doi/10.1101/gr.4410706</a>.
</div>
<div id="ref-Eckhardt2006" class="csl-entry">
150. Eckhardt F, Lewin J, Cortese R, Rakyan VK, Attwood J, Burger M, Burton J, Cox TV, Davies R, Down TA, Haefliger C, Horton R, Howe K, Jackson DK, Kunde J, Koenig C, Liddle J, Niblett D, Otto T, Pettett R, Seemann S, Thompson C, West T, Rogers J, Olek A, Berlin K, Beck S: <strong><span class="nocase">DNA methylation profiling of human chromosomes 6, 20 and 22</span></strong>. <em>Nature Genetics</em> 2006, <strong>38</strong>:1378–1385<a href="https://doi.org/10.1038/ng1909">10.1038/ng1909</a>Available: <a href="http://www.nature.com/doifinder/10.1038/ng1909">http://www.nature.com/doifinder/10.1038/ng1909</a>.
</div>
<div id="ref-Bjornsson2008" class="csl-entry">
151. Bjornsson HT: <strong><span class="nocase">Intra-individual Change Over Time in DNA Methylation With Familial Clustering</span></strong>. <em>JAMA</em> 2008, <strong>299</strong>:2877<a href="https://doi.org/10.1001/jama.299.24.2877">10.1001/jama.299.24.2877</a>Available: <a href="http://jama.jamanetwork.com/article.aspx?doi=10.1001/jama.299.24.2877">http://jama.jamanetwork.com/article.aspx?doi=10.1001/jama.299.24.2877</a>.
</div>
<div id="ref-Christensen2009" class="csl-entry">
152. Christensen BC, Houseman EA, Marsit CJ, Zheng S, Wrensch MR, Wiemels JL, Nelson HH, Karagas MR, Padbury JF, Bueno R, Sugarbaker DJ, Yeh R-F, Wiencke JK, Kelsey KT: <strong><span class="nocase">Aging and Environmental Exposures Alter Tissue-Specific DNA Methylation Dependent upon CpG Island Context</span></strong>. <em>PLoS Genetics</em> 2009, <strong>5</strong>:e1000602<a href="https://doi.org/10.1371/journal.pgen.1000602">10.1371/journal.pgen.1000602</a>Available: <a href="https://dx.plos.org/10.1371/journal.pgen.1000602">https://dx.plos.org/10.1371/journal.pgen.1000602</a>.
</div>
<div id="ref-Rakyan2010" class="csl-entry">
153. Rakyan VK, Down TA, Maslau S, Andrew T, Yang TP, Beyan H, Whittaker P, McCann OT, Finer S, Valdes AM, Leslie RD, Deloukas P, Spector TD: <strong><span class="nocase">Human aging-associated DNA hypermethylation occurs preferentially at bivalent chromatin domains</span></strong>. <em>Genome Research</em> 2010, <strong>20</strong>:434–439<a href="https://doi.org/10.1101/gr.103101.109">10.1101/gr.103101.109</a>Available: <a href="http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109">http://genome.cshlp.org/cgi/doi/10.1101/gr.103101.109</a>.
</div>
<div id="ref-Bibikova2009" class="csl-entry">
154. Bibikova M, Le J, Barnes B, Saedinia-Melnyk S, Zhou L, Shen R, Gunderson KL: <strong><span class="nocase">Genome-wide DNA methylation profiling using Infinium <span><span></span></span> assay</span></strong>. <em>Epigenomics</em> 2009, <strong>1</strong>:177–200<a href="https://doi.org/10.2217/epi.09.14">10.2217/epi.09.14</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi.09.14">https://www.futuremedicine.com/doi/10.2217/epi.09.14</a>.
</div>
<div id="ref-Teschendorff2010" class="csl-entry">
155. Teschendorff AE, Menon U, Gentry-Maharaj A, Ramus SJ, Weisenberger DJ, Shen H, Campan M, Noushmehr H, Bell CG, Maxwell AP, Savage DA, Mueller-Holzner E, Marth C, Kocjan G, Gayther SA, Jones A, Beck S, Wagner W, Laird PW, Jacobs IJ, Widschwendter M: <strong><span class="nocase">Age-dependent DNA methylation of genes that are suppressed in stem cells is a hallmark of cancer.</span></strong> <em>Genome research</em> 2010, <strong>20</strong>:440–6<a href="https://doi.org/10.1101/gr.103606.109">10.1101/gr.103606.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20219944">http://www.ncbi.nlm.nih.gov/pubmed/20219944</a>.
</div>
<div id="ref-Jung2015" class="csl-entry">
156. Jung M, Pfeifer GP: <strong><span class="nocase">Aging and DNA methylation</span></strong>. <em>BMC Biology</em> 2015, <strong>13</strong>:7<a href="https://doi.org/10.1186/s12915-015-0118-4">10.1186/s12915-015-0118-4</a>Available: <a href="http://www.biomedcentral.com/1741-7007/13/7">http://www.biomedcentral.com/1741-7007/13/7</a>.
</div>
<div id="ref-Bell2010" class="csl-entry">
157. Bell CG, Teschendorff AE, Rakyan VK, Maxwell AP, Beck S, Savage Da: <strong><span class="nocase">Genome-wide DNA methylation analysis for diabetic nephropathy in type 1 diabetes mellitus</span></strong>. <em>BMC Medical Genomics</em> 2010, <strong>3</strong>:33<a href="https://doi.org/10.1186/1755-8794-3-33">10.1186/1755-8794-3-33</a>Available: <a href="http://bmcmedgenomics.biomedcentral.com/articles/10.1186/1755-8794-3-33">http://bmcmedgenomics.biomedcentral.com/articles/10.1186/1755-8794-3-33</a>.
</div>
<div id="ref-Bocklandt2011" class="csl-entry">
158. Bocklandt S, Lin W, Sehl ME, Sánchez FJ, Sinsheimer JS, Horvath S, Vilain E: <strong><span class="nocase">Epigenetic Predictor of Age</span></strong>. <em>PLoS ONE</em> 2011, <strong>6</strong>:e14821<a href="https://doi.org/10.1371/journal.pone.0014821">10.1371/journal.pone.0014821</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0014821">https://dx.plos.org/10.1371/journal.pone.0014821</a>.
</div>
<div id="ref-Koch2011" class="csl-entry">
159. Koch CM, Wagner W: <strong><span class="nocase">Epigenetic-aging-signature to determine age in different tissues.</span></strong> <em>Aging</em> 2011, <strong>3</strong>:1018–27<a href="https://doi.org/10.18632/aging.100395">10.18632/aging.100395</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22067257">http://www.ncbi.nlm.nih.gov/pubmed/22067257</a>.
</div>
<div id="ref-Bell2012" class="csl-entry">
160. Bell JT, Tsai P-C, Yang T-P, Pidsley R, Nisbet J, Glass D, Mangino M, Zhai G, Zhang F, Valdes A, Shin S-Y, Dempster EL, Murray RM, Grundberg E, Hedman AK, Nica A, Small KS, MuTHER Consortium, Dermitzakis ET, McCarthy MI, Mill J, Spector TD, Deloukas P: <strong><span class="nocase">Epigenome-wide scans identify differentially methylated regions for age and age-related phenotypes in a healthy ageing population.</span></strong> <em>PLoS genetics</em> 2012, <strong>8</strong>:e1002629<a href="https://doi.org/10.1371/journal.pgen.1002629">10.1371/journal.pgen.1002629</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22532803">http://www.ncbi.nlm.nih.gov/pubmed/22532803</a>.
</div>
<div id="ref-Heyn2012" class="csl-entry">
161. Heyn H, Li N, Ferreira HJ, Moran S, Pisano DG, Gomez A, Diez J, Sanchez-Mut JV, Setien F, Carmona FJ, Puca AA, Sayols S, Pujana MA, Serra-Musach J, Iglesias-Platas I, Formiga F, Fernandez AF, Fraga MF, Heath SC, Valencia A, Gut IG, Wang J, Esteller M: <strong><span class="nocase">Distinct DNA methylomes of newborns and centenarians</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2012, <strong>109</strong>:10522–10527<a href="https://doi.org/10.1073/pnas.1120658109">10.1073/pnas.1120658109</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1120658109">http://www.pnas.org/cgi/doi/10.1073/pnas.1120658109</a>.
</div>
<div id="ref-Bibikova2011" class="csl-entry">
162. Bibikova M, Barnes B, Tsan C, Ho V, Klotzle B, Le JM, Delano D, Zhang L, Schroth GP, Gunderson KL, Fan J-B, Shen R: <strong><span class="nocase">High density DNA methylation array with single CpG site resolution.</span></strong> <em>Genomics</em> 2011, <strong>98</strong>:288–95<a href="https://doi.org/10.1016/j.ygeno.2011.07.007">10.1016/j.ygeno.2011.07.007</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21839163">http://www.ncbi.nlm.nih.gov/pubmed/21839163</a>.
</div>
<div id="ref-Affinito2016" class="csl-entry">
163. Affinito O, Scala G, Palumbo D, Florio E, Monticelli A, Miele G, Avvedimento VE, Usiello A, Chiariotti L, Cocozza S: <strong><span class="nocase">Modeling DNA methylation by analyzing the individual configurations of single molecules.</span></strong> <em>Epigenetics</em> 2016, <strong>11</strong>:881–888<a href="https://doi.org/10.1080/15592294.2016.1246108">10.1080/15592294.2016.1246108</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27748645">http://www.ncbi.nlm.nih.gov/pubmed/27748645</a>.
</div>
<div id="ref-Haerter2014" class="csl-entry">
164. Haerter JO, Lövkvist C, Dodd IB, Sneppen K: <strong><span class="nocase">Collaboration between CpG sites is needed for stable somatic inheritance of DNA methylation states</span></strong>. <em>Nucleic Acids Research</em> 2014, <strong>42</strong>:2235–2244<a href="https://doi.org/10.1093/nar/gkt1235">10.1093/nar/gkt1235</a>Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3936770">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3936770</a>.
</div>
<div id="ref-Thomson2010" class="csl-entry">
165. Thomson JP, Skene PJ, Selfridge J, Clouaire T, Guy J, Webb S, Kerr ARW, Deaton A, Andrews R, James KD, Turner DJ, Illingworth R, Bird A: <strong><span class="nocase">CpG islands influence chromatin structure via the CpG-binding protein Cfp1.</span></strong> <em>Nature</em> 2010, <strong>464</strong>:1082–6<a href="https://doi.org/10.1038/nature08924">10.1038/nature08924</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20393567">http://www.ncbi.nlm.nih.gov/pubmed/20393567</a>.
</div>
<div id="ref-Garagnani2012" class="csl-entry">
166. Garagnani P, Bacalini MG, Pirazzini C, Gori D, Giuliani C, Mari D, Di Blasio AM, Gentilini D, Vitale G, Collino S, Rezzi S, Castellani G, Capri M, Salvioli S, Franceschi C: <strong><span class="nocase">Methylation of ELOVL2 gene as a new epigenetic marker of age.</span></strong> <em>Aging cell</em> 2012, <strong>11</strong>:1132–4<a href="https://doi.org/10.1111/acel.12005">10.1111/acel.12005</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23061750">http://www.ncbi.nlm.nih.gov/pubmed/23061750</a>.
</div>
<div id="ref-Hannum2013" class="csl-entry">
167. Hannum G, Guinney J, Zhao L, Zhang L, Hughes G, Sadda S, Klotzle B, Bibikova M, Fan J-B, Gao Y, Deconde R, Chen M, Rajapakse I, Friend S, Ideker T, Zhang K: <strong><span class="nocase">Genome-wide methylation profiles reveal quantitative views of human aging rates.</span></strong> <em>Molecular cell</em> 2013, <strong>49</strong>:359–367<a href="https://doi.org/10.1016/j.molcel.2012.10.016">10.1016/j.molcel.2012.10.016</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23177740">http://www.ncbi.nlm.nih.gov/pubmed/23177740</a>.
</div>
<div id="ref-Horvath2013" class="csl-entry">
168. Horvath S: <strong><span class="nocase">DNA methylation age of human tissues and cell types.</span></strong> <em>Genome biology</em> 2013, <strong>14</strong>:R115<a href="https://doi.org/10.1186/gb-2013-14-10-r115">10.1186/gb-2013-14-10-r115</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24138928">http://www.ncbi.nlm.nih.gov/pubmed/24138928</a>.
</div>
<div id="ref-Friedman2010" class="csl-entry">
169. Friedman J, Hastie T, Tibshirani R: <strong><span class="nocase">Regularization Paths for Generalized Linear Models via Coordinate Descent</span></strong>. <em>Journal of Statistical Software</em> 2010, <strong>33</strong>:7–10<a href="https://doi.org/10.18637/jss.v033.i01">10.18637/jss.v033.i01</a>Available: <a href="http://www.jstatsoft.org/v33/i01/">http://www.jstatsoft.org/v33/i01/</a>.
</div>
<div id="ref-Florath2014" class="csl-entry">
170. Florath I, Butterbach K, Muller H, Bewerunge-Hudler M, Brenner H: <strong><span class="nocase">Cross-sectional and longitudinal changes in DNA methylation with age: an epigenome-wide analysis revealing over 60 novel age-associated CpG sites</span></strong>. <em>Human Molecular Genetics</em> 2014, <strong>23</strong>:1186–1201<a href="https://doi.org/10.1093/hmg/ddt531">10.1093/hmg/ddt531</a>Available: <a href="https://academic.oup.com/hmg/article-lookup/doi/10.1093/hmg/ddt531">https://academic.oup.com/hmg/article-lookup/doi/10.1093/hmg/ddt531</a>.
</div>
<div id="ref-Bacalini2015" class="csl-entry">
171. Bacalini MG, Boattini A, Gentilini D, Giampieri E, Pirazzini C, Giuliani C, Fontanesi E, Remondini D, Capri M, Del Rio A, Luiselli D, Vitale G, Mari D, Castellani G, Di Blasio AM, Salvioli S, Franceschi C, Garagnani P: <strong><span class="nocase">A meta-analysis on age-associated changes in blood DNA methylation: results from an original analysis pipeline for Infinium 450k data</span></strong>. <em>Aging</em> 2015, <strong>7</strong>:97–109<a href="https://doi.org/10.18632/aging.100718">10.18632/aging.100718</a>Available: <a href="http://www.aging-us.com/article/100718">http://www.aging-us.com/article/100718</a>.
</div>
<div id="ref-Zaghlool2015" class="csl-entry">
172. Zaghlool SB, Al-Shafai M, Al Muftah WA, Kumar P, Falchi M, Suhre K: <strong><span class="nocase">Association of DNA methylation with age, gender, and smoking in an Arab population</span></strong>. <em>Clinical Epigenetics</em> 2015, <strong>7</strong>:6<a href="https://doi.org/10.1186/s13148-014-0040-6">10.1186/s13148-014-0040-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25663950">http://www.ncbi.nlm.nih.gov/pubmed/25663950</a>.
</div>
<div id="ref-Benton2017" class="csl-entry">
173. Benton MC, Sutherland HG, Macartney-Coxson D, Haupt LM, Lea RA, Griffiths LR: <strong><span class="nocase">Methylome-wide association study of whole blood DNA in the Norfolk Island isolate identifies robust loci associated with age</span></strong>. <em>Aging</em> 2017, <strong>9</strong>:753–768<a href="https://doi.org/10.18632/aging.101187">10.18632/aging.101187</a>Available: <a href="http://www.aging-us.com/article/101187/text">http://www.aging-us.com/article/101187/text</a>.
</div>
<div id="ref-Johnson2017" class="csl-entry">
174. Johnson ND, Wiener HW, Smith AK, Nishitani S, Absher DM, Arnett DK, Aslibekyan S, Conneely KN: <strong><span class="nocase">Non-linear patterns in age-related DNA methylation may reflect CD4 + T cell differentiation</span></strong>. <em>Epigenetics</em> 2017, <strong>12</strong>:492–503<a href="https://doi.org/10.1080/15592294.2017.1314419">10.1080/15592294.2017.1314419</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28387568">http://www.ncbi.nlm.nih.gov/pubmed/28387568</a>.
</div>
<div id="ref-Smith2014a" class="csl-entry">
175. Smith AK, Kilaru V, Kocak M, Almli LM, Mercer KB, Ressler KJ, Tylavsky FA, Conneely KN: <strong><span class="nocase">Methylation quantitative trait loci (meQTLs) are consistently detected across ancestry, developmental stage, and tissue type</span></strong>. <em>BMC Genomics</em> 2014, <strong>15</strong>:145<a href="https://doi.org/10.1186/1471-2164-15-145">10.1186/1471-2164-15-145</a>Available: <a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-145">http://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-145</a>.
</div>
<div id="ref-Kuehnen2012" class="csl-entry">
176. Kuehnen P, Mischke M, Wiegand S, Sers C, Horsthemke B, Lau S, Keil T, Lee YA, Grueters A, Krude H: <strong><span class="nocase">An alu element-associated hypermethylation variant of the POMC gene is associated with childhood obesity</span></strong>. <em>PLoS Genetics</em> 2012, <strong>8</strong>:1–12<a href="https://doi.org/10.1371/journal.pgen.1002543">10.1371/journal.pgen.1002543</a>.
</div>
<div id="ref-Gymrek2015" class="csl-entry">
177. Gymrek M, Willems T, Guilmatre A, Zeng H, Markus B, Georgiev S, Daly MJ, Price AL, Pritchard JK, Sharp AJ, Erlich Y: <strong><span class="nocase">Abundant contribution of short tandem repeats to gene expression variation in humans</span></strong>. <em>Nature Genetics</em> 2016, <strong>48</strong>:22–29<a href="https://doi.org/10.1038/ng.3461">10.1038/ng.3461</a>Available: <a href="http://dx.doi.org/10.1038/ng.3461">http://dx.doi.org/10.1038/ng.3461</a>.
</div>
<div id="ref-Velten2017" class="csl-entry">
178. Velten L, Haas SF, Raffel S, Blaszkiewicz S, Islam S, Hennig BP, Hirche C, Lutz C, Buss EC, Nowak D, Boch T, Hofmann W, Ho AD, Huber W, Trumpp A, Essers MAG, Steinmetz LM: <strong><span class="nocase">Human haematopoietic stem cell lineage commitment is a continuous process</span></strong>. <em>Nature Cell Biology</em> 2017, <strong>19</strong>:271–281<a href="https://doi.org/10.1038/ncb3493">10.1038/ncb3493</a>Available: <a href="http://www.nature.com/articles/ncb3493">http://www.nature.com/articles/ncb3493</a>.
</div>
<div id="ref-Jaiswal2014" class="csl-entry">
179. Jaiswal S, Fontanillas P, Flannick J, Manning A, Grauman PV, Mar BG, Lindsley RC, Mermel CH, Burtt N, Chavez A, Higgins JM, Moltchanov V, Kuo FC, Kluk MJ, Henderson B, Kinnunen L, Koistinen Ha, Ladenvall C, Getz G, Correa A, Banahan BF, Gabriel S, Kathiresan S, Stringham HM, McCarthy MI, Boehnke M, Tuomilehto J, Haiman C, Groop L, Atzmon G, et al.: <strong><span class="nocase">Age-Related Clonal Hematopoiesis Associated with Adverse Outcomes</span></strong>. <em>New England Journal of Medicine</em> 2014, <strong>371</strong>:2488–2498<a href="https://doi.org/10.1056/NEJMoa1408617">10.1056/NEJMoa1408617</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25426837">http://www.ncbi.nlm.nih.gov/pubmed/25426837</a>.
</div>
<div id="ref-Watson2020" class="csl-entry">
180. Watson CJ, Papula AL, Poon GYP, Wong WH, Young AL, Druley TE, Fisher DS, Blundell JR: <strong><span class="nocase">The evolutionary dynamics and fitness landscape of clonal hematopoiesis</span></strong>. <em>Science</em> 2020, <strong>367</strong>:1449–1454<a href="https://doi.org/10.1126/science.aay9333">10.1126/science.aay9333</a>Available: <a href="https://www.sciencemag.org/lookup/doi/10.1126/science.aay9333">https://www.sciencemag.org/lookup/doi/10.1126/science.aay9333</a>.
</div>
<div id="ref-Kirschner2017" class="csl-entry">
181. Kirschner K, Chandra T, Kiselev V, Flores-Santa Cruz D, Macaulay IC, Park HJ, Li J, Kent DG, Kumar R, Pask DC, Hamilton TL, Hemberg M, Reik W, Green AR: <strong><span class="nocase">Proliferation Drives Aging-Related Functional Decline in a Subpopulation of the Hematopoietic Stem Cell Compartment</span></strong>. <em>Cell Reports</em> 2017, <strong>19</strong>:1503–1511<a href="https://doi.org/10.1016/j.celrep.2017.04.074">10.1016/j.celrep.2017.04.074</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S2211124717306034">https://linkinghub.elsevier.com/retrieve/pii/S2211124717306034</a>.
</div>
<div id="ref-Jaiswal2014c" class="csl-entry">
182. Jaiswal S, Fontanillas P, Flannick J, Manning A, Grauman PV, Mar BG, Lindsley RC, Mermel CH, Burtt N, Chavez A, Higgins JM, Moltchanov V, Kuo FC, Kluk MJ, Henderson B, Kinnunen L, Koistinen HA, Ladenvall C, Getz G, Correa A, Banahan BF, Gabriel S, Kathiresan S, Stringham HM, McCarthy MI, Boehnke M, Tuomilehto J, Haiman C, Groop L, Atzmon G, et al.: <strong><span class="nocase">Age-Related Clonal Hematopoiesis Associated with Adverse Outcomes</span></strong>. <em>New England Journal of Medicine</em> 2014, <strong>371</strong>:2488–2498<a href="https://doi.org/10.1056/NEJMoa1408617">10.1056/NEJMoa1408617</a>Available: <a href="http://www.nejm.org/doi/10.1056/NEJMoa1408617">http://www.nejm.org/doi/10.1056/NEJMoa1408617</a>.
</div>
<div id="ref-Weidner2014" class="csl-entry">
183. Weidner CI, Lin Q, Koch CM, Eisele L, Beier F, Ziegler P, Bauerschlag DO, Jöckel K-H, Erbel R, Mühleisen TW, Zenke M, Brümmendorf TH, Wagner W: <strong><span class="nocase">Aging of blood can be tracked by DNA methylation changes at just three CpG sites.</span></strong> <em>Genome biology</em> 2014, <strong>15</strong>:R24<a href="https://doi.org/10.1186/gb-2014-15-2-r24">10.1186/gb-2014-15-2-r24</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24490752">http://www.ncbi.nlm.nih.gov/pubmed/24490752</a>.
</div>
<div id="ref-Johnson2006" class="csl-entry">
184. Johnson TE: <strong><span class="nocase">Recent results: Biomarkers of aging</span></strong>. <em>Experimental Gerontology</em> 2006, <strong>41</strong>:1243–1246<a href="https://doi.org/10.1016/j.exger.2006.09.006">10.1016/j.exger.2006.09.006</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0531556506002865">https://linkinghub.elsevier.com/retrieve/pii/S0531556506002865</a>.
</div>
<div id="ref-Thompson2017" class="csl-entry">
185. Thompson MJ, VonHoldt B, Horvath S, Pellegrini M: <strong><span class="nocase">An epigenetic aging clock for dogs and wolves.</span></strong> <em>Aging</em> 2017, <strong>9</strong>:1055–1068<a href="https://doi.org/10.18632/aging.101211">10.18632/aging.101211</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28373601">http://www.ncbi.nlm.nih.gov/pubmed/28373601</a>.
</div>
<div id="ref-Stubbs2017" class="csl-entry">
186. Stubbs TM, Bonder MJ, Stark A-K, Krueger F, Meyenn F von, Stegle O, Reik W: <strong><span class="nocase">Multi-tissue DNA methylation age predictor in mouse</span></strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:68<a href="https://doi.org/10.1186/s13059-017-1203-5">10.1186/s13059-017-1203-5</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28399939">http://www.ncbi.nlm.nih.gov/pubmed/28399939</a>.
</div>
<div id="ref-Lowe2020" class="csl-entry">
187. Lowe R, Danson AF, Rakyan VK, Yildizoglu S, Saldmann F, Viltard M, Friedlander G, Faulkes CG: <strong><span class="nocase">DNA methylation clocks as a predictor for ageing and age estimation in naked mole-rats , Heterocephalus glaber</span></strong>. 2020, <strong>12</strong>:1–13<a href="https://doi.org/10.18632/aging.102892">10.18632/aging.102892</a>.
</div>
<div id="ref-Lowe2018" class="csl-entry">
188. Lowe R, Barton C, Jenkins CA, Ernst C, Forman O, Fernandez-Twinn DS, Bock C, Rossiter SJ, Faulkes CG, Ozanne SE, Walter L, Odom DT, Mellersh C, Rakyan VK: <strong><span class="nocase">Ageing-associated DNA methylation dynamics are a molecular readout of lifespan variation among mammalian species</span></strong>. <em>Genome Biology</em> 2018, <strong>19</strong>:22<a href="https://doi.org/10.1186/s13059-018-1397-1">10.1186/s13059-018-1397-1</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29452591">http://www.ncbi.nlm.nih.gov/pubmed/29452591</a>.
</div>
<div id="ref-Wang2019" class="csl-entry">
189. Wang M, Lemos B: <strong><span class="nocase">Ribosomal DNA harbors an evolutionarily conserved clock of biological aging.</span></strong> <em>Genome research</em> 2019, <strong>29</strong>:325–333<a href="https://doi.org/10.1101/gr.241745.118">10.1101/gr.241745.118</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30765617">http://www.ncbi.nlm.nih.gov/pubmed/30765617</a>.
</div>
<div id="ref-Marioni2015" class="csl-entry">
190. Marioni RE, Shah S, McRae AF, Chen BH, Colicino E, Harris SE, Gibson J, Henders AK, Redmond P, Cox SR, Pattie A, Corley J, Murphy L, Martin NG, Montgomery GW, Feinberg AP, Fallin MD, Multhaup ML, Jaffe AE, Joehanes R, Schwartz J, Just AC, Lunetta KL, Murabito JM, Starr JM, Horvath S, Baccarelli AA, Levy D, Visscher PM, Wray NR, et al.: <strong><span class="nocase">DNA methylation age of blood predicts all-cause mortality in later life.</span></strong> <em>Genome biology</em> 2015, <strong>16</strong>:25<a href="https://doi.org/10.1186/s13059-015-0584-6">10.1186/s13059-015-0584-6</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25633388">http://www.ncbi.nlm.nih.gov/pubmed/25633388</a>.
</div>
<div id="ref-Christiansen2016" class="csl-entry">
191. Christiansen L, Lenart A, Tan Q, Vaupel JW, Aviv A, McGue M, Christensen K: <strong><span class="nocase">DNA methylation age is associated with mortality in a longitudinal Danish twin study</span></strong>. <em>Aging Cell</em> 2016, <strong>15</strong>:149–154<a href="https://doi.org/10.1111/acel.12421">10.1111/acel.12421</a>Available: <a href="http://doi.wiley.com/10.1111/acel.12421">http://doi.wiley.com/10.1111/acel.12421</a>.
</div>
<div id="ref-Chen2016a" class="csl-entry">
192. Chen BH, Marioni RE, Colicino E, Peters MJ, Ward-Caviness CK, Tsai P-C, Roetker NS, Just AC, Demerath EW, Guan W, Bressler J, Fornage M, Studenski S, Vandiver AR, Moore AZ, Tanaka T, Kiel DP, Liang L, Vokonas P, Schwartz J, Lunetta KL, Murabito JM, Bandinelli S, Hernandez DG, Melzer D, Nalls M, Pilling LC, Price TR, Singleton AB, Gieger C, et al.: <strong><span class="nocase">DNA methylation-based measures of biological age: meta-analysis predicting time to death.</span></strong> <em>Aging</em> 2016, <strong>8</strong>:1844–1865<a href="https://doi.org/10.18632/aging.101020">10.18632/aging.101020</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27690265">http://www.ncbi.nlm.nih.gov/pubmed/27690265</a>.
</div>
<div id="ref-Levine2018" class="csl-entry">
193. Levine ME, Lu AT, Quach A, Chen BH, Assimes TL, Bandinelli S, Hou L, Baccarelli AA, Stewart JD, Li Y, Whitsel EA, Wilson JG, Reiner AP, Aviv A, Lohman K, Liu Y, Ferrucci L, Horvath S: <strong><span class="nocase">An epigenetic biomarker of aging for lifespan and healthspan</span></strong>. <em>Aging</em> 2018, <strong>10</strong>:573–591<a href="https://doi.org/10.18632/aging.101414">10.18632/aging.101414</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/29676998">http://www.ncbi.nlm.nih.gov/pubmed/29676998</a>.
</div>
<div id="ref-Lu2019" class="csl-entry">
194. Lu AT, Quach A, Wilson JG, Reiner AP, Aviv A, Raj K, Hou L, Baccarelli AA, Li Y, Stewart JD, Whitsel EA, Assimes TL, Ferrucci L, Horvath S: <strong><span class="nocase">DNA methylation GrimAge strongly predicts lifespan and healthspan</span></strong>. <em>Aging</em> 2019, <strong>11</strong>:303–327<a href="https://doi.org/10.18632/aging.101684">10.18632/aging.101684</a>Available: <a href="http://www.aging-us.com/article/101684/text">http://www.aging-us.com/article/101684/text</a>.
</div>
<div id="ref-Halloran2020" class="csl-entry">
195. Halloran AMO, Hever A, Ni C, Ake C, Horvath S: <strong><span class="nocase">Association of 4 epigenetic clocks with measures of functional health, cognition, and all- cause mortality in The Irish Longitudinal Study on Ageing (TILDA)</span></strong>. 2020, :1–35<a href="https://doi.org/10.1101/2020.04.27.063164">10.1101/2020.04.27.063164</a>Available: <a href="https://www.biorxiv.org/content/10.1101/2020.04.27.063164v1">https://www.biorxiv.org/content/10.1101/2020.04.27.063164v1</a>.
</div>
<div id="ref-Belsky2020" class="csl-entry">
196. Belsky DW, Caspi A, Arseneault L, Baccarelli A, Corcoran DL, Gao X, Hannon E, Harrington HL, Rasmussen LJH, Houts R, Huffman K, Kraus WE, Kwon D, Mill J, Pieper CF, Prinz JA, Poulton R, Schwartz J, Sugden K, Vokonas P, Williams BS, Moffitt TE: <strong><span class="nocase">Quantification of the pace of biological aging in humans through a blood test, the DunedinPoAm DNA methylation algorithm</span></strong>. <em>eLife</em> 2020, <strong>9</strong>:1–56<a href="https://doi.org/10.7554/eLife.54870">10.7554/eLife.54870</a>Available: <a href="https://elifesciences.org/articles/54870">https://elifesciences.org/articles/54870</a>.
</div>
<div id="ref-Fahy2019" class="csl-entry">
197. Fahy GM, Brooke RT, Watson JP, Good Z, Vasanawala SS, Maecker H, Leipold MD, Lin DTS, Kobor MS, Horvath S: <strong><span class="nocase">Reversal of epigenetic aging and immunosenescent trends in humans</span></strong>. <em>Aging Cell</em> 2019, :1–12<a href="https://doi.org/10.1111/acel.13028">10.1111/acel.13028</a>.
</div>
<div id="ref-Lu2019a" class="csl-entry">
198. Lu AT, Seeboth A, Tsai PC, Sun D, Quach A, Reiner AP, Kooperberg C, Ferrucci L, Hou L, Baccarelli AA, Li Y, Harris SE, Corley J, Taylor A, Deary IJ, Stewart JD, Whitsel EA, Assimes TL, Chen W, Li S, Mangino M, Bell JT, Wilson JG, Aviv A, Marioni RE, Raj K, Horvath S: <strong><span class="nocase">DNA methylation-based estimator of telomere length</span></strong>. <em>Aging</em> 2019, <strong>11</strong>:5895–5923<a href="https://doi.org/10.18632/aging.102173">10.18632/aging.102173</a>.
</div>
<div id="ref-Naylor2005" class="csl-entry">
199. Naylor K, Li G, Vallejo AN, Lee W-W, Koetz K, Bryl E, Witkowski J, Fulbright J, Weyand CM, Goronzy JJ: <strong><span class="nocase">The Influence of Age on T Cell Generation and TCR Diversity</span></strong>. <em>The Journal of Immunology</em> 2005, <strong>174</strong>:7446–7452<a href="https://doi.org/10.4049/jimmunol.174.11.7446">10.4049/jimmunol.174.11.7446</a>Available: <a href="http://www.jimmunol.org/lookup/doi/10.4049/jimmunol.174.11.7446">http://www.jimmunol.org/lookup/doi/10.4049/jimmunol.174.11.7446</a>.
</div>
<div id="ref-Marcus1990" class="csl-entry">
200. MARCUS R, BUTTERFIELD G, HOLLOWAY L, GILLILAND L, BAYLINK DJ, HINTZ RL, SHERMAN BM: <strong><span class="nocase">Effects of Short Term Administration of Recombinant Human Growth Hormone to Elderly People*</span></strong>. <em>The Journal of Clinical Endocrinology <span>&amp;</span> Metabolism</em> 1990, <strong>70</strong>:519–527<a href="https://doi.org/10.1210/jcem-70-2-519">10.1210/jcem-70-2-519</a>Available: <a href="https://academic.oup.com/jcem/article-lookup/doi/10.1210/jcem-70-2-519">https://academic.oup.com/jcem/article-lookup/doi/10.1210/jcem-70-2-519</a>.
</div>
<div id="ref-Gensous2019" class="csl-entry">
201. Gensous N, Franceschi C, Santoro A, Milazzo M, Garagnani P, Bacalini MG: <strong><span class="nocase">The Impact of Caloric Restriction on the Epigenetic Signatures of Aging</span></strong>. <em>International Journal of Molecular Sciences</em> 2019, <strong>20</strong>:2022<a href="https://doi.org/10.3390/ijms20082022">10.3390/ijms20082022</a>Available: <a href="https://www.mdpi.com/1422-0067/20/8/2022">https://www.mdpi.com/1422-0067/20/8/2022</a>.
</div>
<div id="ref-Wang2017a" class="csl-entry">
202. Wang T, Tsui B, Kreisberg JF, Robertson NA, Gross AM, Yu MK, Carter H, Brown-Borg HM, Adams PD, Ideker T: <strong><span class="nocase">Epigenetic aging signatures in mice livers are slowed by dwarfism, calorie restriction and rapamycin treatment</span></strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:57<a href="https://doi.org/10.1186/s13059-017-1186-2">10.1186/s13059-017-1186-2</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1186-2">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1186-2</a>.
</div>
<div id="ref-Horvath2020" class="csl-entry">
203. Horvath S, Singh K, Raj K, Khairnar S, Sanghavi A, Shrivastava A, Zoller JA, Li CZ, Herenu CB, Canatelli-Mallat M, Lehmann M, Woods LCS, Martinez AG, Wang T, Chiavellini P, Levine AJ, Chen H, Goya RG, Katcher HL: <strong><span class="nocase">Reversing age: dual species measurement of epigenetic age with a single clock</span></strong>. <em>bioRxiv</em> 2020, :2020.05.07.082917<a href="https://doi.org/10.1101/2020.05.07.082917">10.1101/2020.05.07.082917</a>Available: <a href="https://www.biorxiv.org/content/10.1101/2020.05.07.082917v1">https://www.biorxiv.org/content/10.1101/2020.05.07.082917v1</a>.
</div>
<div id="ref-Kulkarni2020" class="csl-entry">
204. Kulkarni AS, Gubbi S, Barzilai N: <strong><span class="nocase">Benefits of Metformin in Attenuating the Hallmarks of Aging</span></strong>. <em>Cell Metabolism</em> 2020, <strong>32</strong>:15–30<a href="https://doi.org/10.1016/j.cmet.2020.04.001">10.1016/j.cmet.2020.04.001</a>Available: <a href="https://doi.org/10.1016/j.cmet.2020.04.001">https://doi.org/10.1016/j.cmet.2020.04.001</a>.
</div>
<div id="ref-TAME2020" class="csl-entry">
205. <span class="nocase">TAME trial website</span>: 2020, Available: <a href="https://www.afar.org/tame-trial">https://www.afar.org/tame-trial</a>.
</div>
<div id="ref-Justice2018" class="csl-entry">
206. Justice JN, Ferrucci L, Newman AB, Aroda VR, Bahnson JL, Divers J, Espeland MA, Marcovina S, Pollak MN, Kritchevsky SB, Barzilai N, Kuchel GA: <strong><span class="nocase">A framework for selection of blood-based biomarkers for geroscience-guided clinical trials: report from the TAME Biomarkers Workgroup</span></strong>. <em>GeroScience</em> 2018, <strong>40</strong>:419–436<a href="https://doi.org/10.1007/s11357-018-0042-y">10.1007/s11357-018-0042-y</a>Available: <a href="http://link.springer.com/10.1007/s11357-018-0042-y">http://link.springer.com/10.1007/s11357-018-0042-y</a>.
</div>
<div id="ref-Harvey2012a" class="csl-entry">
207. Harvey NC, Javaid K, Bishop N, Kennedy S, Papageorghiou AT, Fraser R, Gandhi SV, Schoenmakers I, Prentice A, Cooper C: <strong><span class="nocase">MAVIDOS Maternal Vitamin D Osteoporosis Study: study protocol for a randomized controlled trial. The MAVIDOS Study Group</span></strong>. <em>Trials</em> 2012, <strong>13</strong>:13<a href="https://doi.org/10.1186/1745-6215-13-13">10.1186/1745-6215-13-13</a>Available: <a href="http://trialsjournal.biomedcentral.com/articles/10.1186/1745-6215-13-13">http://trialsjournal.biomedcentral.com/articles/10.1186/1745-6215-13-13</a>.
</div>
<div id="ref-Cooper2016" class="csl-entry">
208. Cooper C, Harvey NC, Bishop NJ, Kennedy S, Papageorghiou AT, Schoenmakers I, Fraser R, Gandhi SV, Carr A, D’Angelo S, Crozier SR, Moon RJ, Arden NK, Dennison EM, Godfrey KM, Inskip HM, Prentice A, Mughal MZ, Eastell R, Reid DM, Javaid MK: <strong><span class="nocase">Maternal gestational vitamin D supplementation and offspring bone health (MAVIDOS): a multicentre, double-blind, randomised placebo-controlled trial</span></strong>. <em>The Lancet Diabetes <span>&amp;</span> Endocrinology</em> 2016, <strong>4</strong>:393–402<a href="https://doi.org/10.1016/S2213-8587(16)00044-9">10.1016/S2213-8587(16)00044-9</a>Available: <a href="http://dx.doi.org/10.1016/S2213-8587(16)00044-9">http://dx.doi.org/10.1016/S2213-8587(16)00044-9</a>.
</div>
<div id="ref-Bell2016" class="csl-entry">
209. Bell CG, Xia Y, Yuan W, Gao F, Ward K, Roos L, Mangino M, Hysi PG, Bell J, Wang J, Spector TD: <strong><span class="nocase">Novel regional age-associated DNA methylation changes within human common disease-associated loci</span></strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:193<a href="https://doi.org/10.1186/s13059-016-1051-8">10.1186/s13059-016-1051-8</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27663977">http://www.ncbi.nlm.nih.gov/pubmed/27663977</a>.
</div>
<div id="ref-Moran2015" class="csl-entry">
210. Moran S, Arribas C, Esteller M: <strong><span class="nocase">Validation of a DNA methylation microarray for 850,000 CpG sites of the human genome enriched in enhancer sequences.</span></strong> <em>Epigenomics</em> 2016, <strong>8</strong>:389–99<a href="https://doi.org/10.2217/epi.15.114">10.2217/epi.15.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26673039">http://www.ncbi.nlm.nih.gov/pubmed/26673039</a>.
</div>
<div id="ref-Gunderson2004" class="csl-entry">
211. Gunderson KL: <strong><span>Decoding Randomly Ordered DNA Arrays</span></strong>. <em>Genome Research</em> 2004, <strong>14</strong>:870–877<a href="https://doi.org/10.1101/gr.2255804">10.1101/gr.2255804</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.2255804">http://www.genome.org/cgi/doi/10.1101/gr.2255804</a>.
</div>
<div id="ref-Michael1998" class="csl-entry">
212. Michael KL, Taylor LC, Schultz SL, Walt DR: <strong><span>Randomly Ordered Addressable High-Density Optical Sensor Arrays</span></strong>. <em>Analytical Chemistry</em> 1998, <strong>70</strong>:1242–1248<a href="https://doi.org/10.1021/ac971343r">10.1021/ac971343r</a>Available: <a href="https://pubs.acs.org/doi/10.1021/ac971343r">https://pubs.acs.org/doi/10.1021/ac971343r</a>.
</div>
<div id="ref-Clark2012" class="csl-entry">
213. Clark C, Palta P, Joyce CJ, Scott C, Grundberg E, Deloukas P, Palotie A, Coffey AJ: <strong><span class="nocase">A Comparison of the Whole Genome Approach of MeDIP-Seq to the Targeted Approach of the Infinium HumanMethylation450 BeadChip<span><span></span></span> for Methylome Profiling</span></strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e50233<a href="https://doi.org/10.1371/journal.pone.0050233">10.1371/journal.pone.0050233</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0050233">https://dx.plos.org/10.1371/journal.pone.0050233</a>.
</div>
<div id="ref-Smith2013" class="csl-entry">
214. Smith ML, Baggerly KA, Bengtsson H, Ritchie ME, Hansen KD: <strong><span class="nocase">illuminaio: An open source IDAT parsing tool for Illumina microarrays</span></strong>. <em>F1000Research</em> 2013, <strong>2</strong>:1–8<a href="https://doi.org/10.12688/f1000research.2-264.v1">10.12688/f1000research.2-264.v1</a>.
</div>
<div id="ref-Dedeurwaerder2011" class="csl-entry">
215. Dedeurwaerder S, Defrance M, Calonne E, Denis H, Sotiriou C, Fuks F: <strong><span class="nocase">Evaluation of the Infinium Methylation 450K technology</span></strong>. <em>Epigenomics</em> 2011, <strong>3</strong>:771–784<a href="https://doi.org/10.2217/epi.11.105">10.2217/epi.11.105</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi.11.105">https://www.futuremedicine.com/doi/10.2217/epi.11.105</a>.
</div>
<div id="ref-Fortin2016" class="csl-entry">
216. Fortin J-P, Triche TJ, Hansen KD: <strong><span class="nocase">Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi</span></strong>. <em>Bioinformatics</em> 2016, :btw691<a href="https://doi.org/10.1093/bioinformatics/btw691">10.1093/bioinformatics/btw691</a>Available: <a href="http://biorxiv.org/lookup/doi/10.1101/065490">http://biorxiv.org/lookup/doi/10.1101/065490</a>.
</div>
<div id="ref-Teschendorff2013" class="csl-entry">
217. Teschendorff AE, Marabita F, Lechner M, Bartlett T, Tegner J, Gomez-Cabrero D, Beck S: <strong><span class="nocase">A beta-mixture quantile normalization method for correcting probe design bias in Illumina Infinium 450 k DNA methylation data</span></strong>. <em>Bioinformatics</em> 2013, <strong>29</strong>:189–196<a href="https://doi.org/10.1093/bioinformatics/bts680">10.1093/bioinformatics/bts680</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bts680">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bts680</a>.
</div>
<div id="ref-Leek2010" class="csl-entry">
218. Leek JT, Scharpf RB, Bravo HC, Simcha D, Langmead B, Johnson WE, Geman D, Baggerly K, Irizarry RA: <strong><span class="nocase">Tackling the widespread and critical impact of batch effects in high-throughput data</span></strong>. <em>Nature Reviews Genetics</em> 2010, <strong>11</strong>:733–739<a href="https://doi.org/10.1038/nrg2825">10.1038/nrg2825</a>Available: <a href="http://www.nature.com/articles/nrg2825">http://www.nature.com/articles/nrg2825</a>.
</div>
<div id="ref-Fortin2014" class="csl-entry">
219. Fortin J-P, Labbe A, Lemire M, Zanke BW, Hudson TJ, Fertig EJ, Greenwood CMT, Hansen KD: <strong><span class="nocase">Functional normalization of 450k methylation array data improves replication in large cancer studies</span></strong>. <em>Genome Biology</em> 2014, <strong>15</strong>:503<a href="https://doi.org/10.1186/s13059-014-0503-2">10.1186/s13059-014-0503-2</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0503-2">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0503-2</a>.
</div>
<div id="ref-Ehrich2007" class="csl-entry">
220. Ehrich M, Zoll S, Sur S, Boom D van den: <strong><span class="nocase">A new method for accurate assessment of DNA quality after bisulfite treatment.</span></strong> <em>Nucleic acids research</em> 2007, <strong>35</strong>:e29<a href="https://doi.org/10.1093/nar/gkl1134">10.1093/nar/gkl1134</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkl1134 http://www.ncbi.nlm.nih.gov/pubmed/17259213 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC1865059">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkl1134 http://www.ncbi.nlm.nih.gov/pubmed/17259213 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC1865059</a>.
</div>
<div id="ref-Illumina2015" class="csl-entry">
221. Illumina: <em><span>Infinium HD Assay Methylation Protocol Guide</span></em>. Illumina; 2015 Available: <a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry{\_}documentation/infinium{\_}assays/infinium{\_}hd{\_}methylation/infinium-hd-methylation-guide-15019519-01.pdf">https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry{\_}documentation/infinium{\_}assays/infinium{\_}hd{\_}methylation/infinium-hd-methylation-guide-15019519-01.pdf</a>.
</div>
<div id="ref-Pidsley2016" class="csl-entry">
222. Pidsley R, Zotenko E, Peters TJ, Lawrence MG, Risbridger GP, Molloy P, Van Djik S, Muhlhausler B, Stirzaker C, Clark SJ: <strong><span class="nocase">Critical evaluation of the Illumina MethylationEPIC BeadChip microarray for whole-genome DNA methylation profiling.</span></strong> <em>Genome biology</em> 2016, <strong>17</strong>:208<a href="https://doi.org/10.1186/s13059-016-1066-1">10.1186/s13059-016-1066-1</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27717381">http://www.ncbi.nlm.nih.gov/pubmed/27717381</a>.
</div>
<div id="ref-Zhou2017" class="csl-entry">
223. Zhou W, Laird PW, Shen H: <strong><span class="nocase">Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes</span></strong>. <em>Nucleic Acids Research</em> 2016, <strong>45</strong>:gkw967<a href="https://doi.org/10.1093/nar/gkw967">10.1093/nar/gkw967</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27924034">http://www.ncbi.nlm.nih.gov/pubmed/27924034</a>.
</div>
<div id="ref-Birney2016" class="csl-entry">
224. Birney E, Smith GD, Greally JM: <strong><span class="nocase">Epigenome-wide Association Studies and the Interpretation of Disease -Omics</span></strong>. <em>PLOS Genetics</em> 2016, <strong>12</strong>:e1006105<a href="https://doi.org/10.1371/journal.pgen.1006105">10.1371/journal.pgen.1006105</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27336614">http://www.ncbi.nlm.nih.gov/pubmed/27336614</a>.
</div>
<div id="ref-Lappalainen2017" class="csl-entry">
225. Lappalainen T, Greally JM: <strong><span class="nocase">Associating cellular epigenetic models with human phenotypes</span></strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:441–451<a href="https://doi.org/10.1038/nrg.2017.32">10.1038/nrg.2017.32</a>Available: <a href="http://www.nature.com/doifinder/10.1038/nrg.2017.32">http://www.nature.com/doifinder/10.1038/nrg.2017.32</a>.
</div>
<div id="ref-Andrews2016" class="csl-entry">
226. Andrews SV, Ladd-Acosta C, Feinberg AP, Hansen KD, Fallin MD: <strong><span class="nocase"><span>“Gap hunting”</span> to characterize clustered probe signals in Illumina methylation array data</span></strong>. <em>Epigenetics <span>&amp;</span> Chromatin</em> 2016, <strong>9</strong>:56<a href="https://doi.org/10.1186/s13072-016-0107-z">10.1186/s13072-016-0107-z</a>Available: <a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z">http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0107-z</a>.
</div>
<div id="ref-Taiwo2012" class="csl-entry">
227. Taiwo O, Wilson GA, Morris T, Seisenberger S, Reik W, Pearce D, Beck S, Butcher LM: <strong><span class="nocase">Methylome analysis using MeDIP-seq with low DNA concentrations</span></strong>. <em>Nature Protocols</em> 2012, <strong>7</strong>:617–636<a href="https://doi.org/10.1038/nprot.2012.012">10.1038/nprot.2012.012</a>Available: <a href="http://www.nature.com/articles/nprot.2012.012">http://www.nature.com/articles/nprot.2012.012</a>.
</div>
<div id="ref-Chavez2010" class="csl-entry">
228. Chavez L, Jozefczuk J, Grimm C, Dietrich J, Timmermann B, Lehrach H, Herwig R, Adjaye J: <strong><span class="nocase">Computational analysis of genome-wide DNA methylation during the differentiation of human embryonic stem cells along the endodermal lineage</span></strong>. <em>Genome Research</em> 2010, <strong>20</strong>:1441–1450<a href="https://doi.org/10.1101/gr.110114.110">10.1101/gr.110114.110</a>Available: <a href="http://genome.cshlp.org/cgi/doi/10.1101/gr.110114.110">http://genome.cshlp.org/cgi/doi/10.1101/gr.110114.110</a>.
</div>
<div id="ref-Andrews2010" class="csl-entry">
229. Andrews S: <strong><span>FastQC</span></strong>. 2010, Available: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.
</div>
<div id="ref-Li2009b" class="csl-entry">
230. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R: <strong><span class="nocase">The Sequence Alignment/Map format and SAMtools</span></strong>. <em>Bioinformatics</em> 2009, <strong>25</strong>:2078–2079<a href="https://doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btp352">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btp352</a>.
</div>
<div id="ref-Quinlan2010" class="csl-entry">
231. Quinlan AR, Hall IM: <strong><span class="nocase">BEDTools: a flexible suite of utilities for comparing genomic features</span></strong>. <em>Bioinformatics</em> 2010, <strong>26</strong>:841–842<a href="https://doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a>Available: <a href="http://bedtools.readthedocs.io/en/latest/index.html">http://bedtools.readthedocs.io/en/latest/index.html</a>.
</div>
<div id="ref-Moayyeri2013" class="csl-entry">
232. Moayyeri A, Hammond CJ, Valdes AM, Spector TD: <strong><span class="nocase">Cohort Profile: TwinsUK and Healthy Ageing Twin Study</span></strong>. <em>International Journal of Epidemiology</em> 2013, <strong>42</strong>:76–85<a href="https://doi.org/10.1093/ije/dyr207">10.1093/ije/dyr207</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22253318">http://www.ncbi.nlm.nih.gov/pubmed/22253318</a>.
</div>
<div id="ref-Korbie2015" class="csl-entry">
233. Korbie D, Lin E, Wall D, Nair SS, Stirzaker C, Clark SJ, Trau M: <strong><span class="nocase">Multiplex bisulfite PCR resequencing of clinical FFPE DNA</span></strong>. <em>Clinical Epigenetics</em> 2015, <strong>7</strong>:28<a href="https://doi.org/10.1186/s13148-015-0067-3">10.1186/s13148-015-0067-3</a>Available: <a href="https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-015-0067-3">https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-015-0067-3</a>.
</div>
<div id="ref-Tran2014" class="csl-entry">
234. Tran H, Porter J, Sun M, Xie H, Zhang L: <strong><span class="nocase">Objective and Comprehensive Evaluation of Bisulfite Short Read Mapping Tools</span></strong>. <em>Advances in Bioinformatics</em> 2014, <strong>2014</strong>:1–11<a href="https://doi.org/10.1155/2014/472045">10.1155/2014/472045</a>Available: <a href="http://www.hindawi.com/journals/abi/2014/472045/">http://www.hindawi.com/journals/abi/2014/472045/</a>.
</div>
<div id="ref-Bottcher2012" class="csl-entry">
235. Böttcher R, Amberg R, Ruzius FP, Guryev V, Verhaegh WFJ, Beyerlein P, Zaag PJ van der: <strong><span class="nocase">Using a priori knowledge to align sequencing reads to their exact genomic position</span></strong>. <em>Nucleic Acids Research</em> 2012, <strong>40</strong>:e125–e125<a href="https://doi.org/10.1093/nar/gks393">10.1093/nar/gks393</a>Available: <a href="https://academic.oup.com/nar/article/40/16/e125/1026881">https://academic.oup.com/nar/article/40/16/e125/1026881</a>.
</div>
<div id="ref-Krueger2011" class="csl-entry">
236. Krueger F, Andrews SR: <strong><span class="nocase">Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications</span></strong>. <em>Bioinformatics</em> 2011, <strong>27</strong>:1571–1572<a href="https://doi.org/10.1093/bioinformatics/btr167">10.1093/bioinformatics/btr167</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr167">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr167</a>.
</div>
<div id="ref-Li2002" class="csl-entry">
237. Li L-C, Dahiya R: <strong><span class="nocase">MethPrimer: designing primers for methylation PCRs.</span></strong> <em>Bioinformatics (Oxford, England)</em> 2002, <strong>18</strong>:1427–31Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12424112">http://www.ncbi.nlm.nih.gov/pubmed/12424112</a>.
</div>
<div id="ref-VanStaa2001" class="csl-entry">
238. Staa TP van, Dennison EM, Leufkens HGM, Cooper C: <strong><span class="nocase">Epidemiology of fractures in England and Wales</span></strong>. <em>Bone</em> 2001, <strong>29</strong>:517–522<a href="https://doi.org/10.1016/S8756-3282(01)00614-7">10.1016/S8756-3282(01)00614-7</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S8756328201006147">https://linkinghub.elsevier.com/retrieve/pii/S8756328201006147</a>.
</div>
<div id="ref-Streubel2011" class="csl-entry">
239. Streubel PN, Ricci WM, Wong A, Gardner MJ: <strong><span class="nocase">Mortality After Distal Femur Fractures in Elderly Patients</span></strong>. <em>Clinical Orthopaedics and Related Research</em> 2011, <strong>469</strong>:1188–1196<a href="https://doi.org/10.1007/s11999-010-1530-2">10.1007/s11999-010-1530-2</a>Available: <a href="http://link.springer.com/10.1007/s11999-010-1530-2">http://link.springer.com/10.1007/s11999-010-1530-2</a>.
</div>
<div id="ref-Thompson2018" class="csl-entry">
240. Thompson E, Greenspan S: <em><span>National Osteoporosis Foundation Annual Report (2018)</span></em>. National Osteoporosis Foundation; 2018 Available: <a href="https://cdn.nof.org/wp-content/uploads/2018{\_}NOF{\_}Annual{\_}report{\_}FINAL.pdf">https://cdn.nof.org/wp-content/uploads/2018{\_}NOF{\_}Annual{\_}report{\_}FINAL.pdf</a>.
</div>
<div id="ref-Hernandez2003" class="csl-entry">
241. Hernandez CJ, Beaupré GS, Carter DR: <strong><span class="nocase">A theoretical analysis of the relative influences of peak BMD, age-related bone loss and menopause on the development of osteoporosis</span></strong>. <em>Osteoporosis International</em> 2003, <strong>14</strong>:843–847<a href="https://doi.org/10.1007/s00198-003-1454-8">10.1007/s00198-003-1454-8</a>.
</div>
<div id="ref-Nguyen2005" class="csl-entry">
242. Nguyen TV, Center JR, Eisman JA: <strong><span class="nocase">Femoral Neck Bone Loss Predicts Fracture Risk Independent of Baseline BMD</span></strong>. <em>Journal of Bone and Mineral Research</em> 2005, <strong>20</strong>:1195–1201<a href="https://doi.org/10.1359/JBMR.050215">10.1359/JBMR.050215</a>Available: <a href="http://doi.wiley.com/10.1359/JBMR.050215">http://doi.wiley.com/10.1359/JBMR.050215</a>.
</div>
<div id="ref-Finkelstein2008" class="csl-entry">
243. Finkelstein JS, Brockwell SE, Mehta V, Greendale GA, Sowers MR, Ettinger B, Lo JC, Johnston JM, Cauley JA, Danielson ME, Neer RM: <strong><span class="nocase">Bone Mineral Density Changes during the Menopause Transition in a Multiethnic Cohort of Women</span></strong>. <em>The Journal of Clinical Endocrinology <span>&amp;</span> Metabolism</em> 2008, <strong>93</strong>:861–868<a href="https://doi.org/10.1210/jc.2007-1876">10.1210/jc.2007-1876</a>Available: <a href="https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2007-1876">https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2007-1876</a>.
</div>
<div id="ref-Hui1990" class="csl-entry">
244. Hui SL, Slemenda CW, Johnston CC: <strong><span class="nocase">The contribution of bone loss to postmenopausal osteoporosis</span></strong>. <em>Osteoporosis International</em> 1990, <strong>1</strong>:30–34<a href="https://doi.org/10.1007/BF01880413">10.1007/BF01880413</a>.
</div>
<div id="ref-Harvey2014b" class="csl-entry">
245. Harvey N, Dennison E, Cooper C: <strong><span>Osteoporosis: A Lifecourse Approach</span></strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:1917–1925<a href="https://doi.org/10.1002/jbmr.2286">10.1002/jbmr.2286</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2286">http://doi.wiley.com/10.1002/jbmr.2286</a>.
</div>
<div id="ref-Bikle2012" class="csl-entry">
246. Bikle DD: <strong><span class="nocase">Vitamin D and Bone</span></strong>. <em>Current Osteoporosis Reports</em> 2012, <strong>10</strong>:151–159<a href="https://doi.org/10.1007/s11914-012-0098-z">10.1007/s11914-012-0098-z</a>Available: <a href="http://link.springer.com/10.1007/s11914-012-0098-z">http://link.springer.com/10.1007/s11914-012-0098-z</a>.
</div>
<div id="ref-Christakos2011" class="csl-entry">
247. Christakos S, Dhawan P, Porta A, Mady LJ, Seth T: <strong><span class="nocase">Vitamin D and intestinal calcium absorption</span></strong>. <em>Molecular and Cellular Endocrinology</em> 2011, <strong>347</strong>:25–29<a href="https://doi.org/10.1016/j.mce.2011.05.038">10.1016/j.mce.2011.05.038</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0303720711002930">https://linkinghub.elsevier.com/retrieve/pii/S0303720711002930</a>.
</div>
<div id="ref-Fukumoto2014" class="csl-entry">
248. Fukumoto S: <strong><span class="nocase">Phosphate metabolism and vitamin D</span></strong>. <em>BoneKEy Reports</em> 2014, <strong>3</strong>:1–5<a href="https://doi.org/10.1038/bonekey.2013.231">10.1038/bonekey.2013.231</a>Available: <a href="http://dx.doi.org/10.1038/bonekey.2013.231">http://dx.doi.org/10.1038/bonekey.2013.231</a>.
</div>
<div id="ref-Baird2011" class="csl-entry">
249. Baird J, Kurshid MA, Kim M, Harvey N, Dennison E, Cooper C: <strong><span class="nocase">Does birthweight predict bone mass in adulthood? A systematic review and meta-analysis</span></strong>. <em>Osteoporosis International</em> 2011, <strong>22</strong>:1323–1334<a href="https://doi.org/10.1007/s00198-010-1344-9">10.1007/s00198-010-1344-9</a>.
</div>
<div id="ref-Cooper2001" class="csl-entry">
250. Cooper C, Eriksson JG, Forsén T, Osmond C, Tuomilehto J, Barker DJP: <strong><span class="nocase">Maternal height, childhood growth and risk of hip fracture in later life: A longitudinal study</span></strong>. <em>Osteoporosis International</em> 2001, <strong>12</strong>:623–629<a href="https://doi.org/10.1007/s001980170061">10.1007/s001980170061</a>.
</div>
<div id="ref-Inskip2006" class="csl-entry">
251. Inskip HM, Godfrey KM, Robinson SM, Law CM, Jp D: <strong><span class="nocase">Europe PMC Funders Group Cohort Profile : The Southampton Women ’ s Survey</span></strong>. <em>International Journal of Epidemiology</em> 2006, <strong>35</strong>:42–48<a href="https://doi.org/10.1093/ije/dyi202">10.1093/ije/dyi202</a>.
</div>
<div id="ref-Harvey2010a" class="csl-entry">
252. Harvey NC, Mahon PA, Robinson SM, Nisbet CE, Javaid MK, Crozier SR, Inskip HM, Godfrey KM, Arden NK, Dennison EM, Cooper C, Taylor P, Greenaway LJ, Hanson M, Barker DJP, Law CM: <strong><span class="nocase">Different indices of fetal growth predict bone size and volumetric density at 4 years of age</span></strong>. <em>Journal of Bone and Mineral Research</em> 2010, <strong>25</strong>:920–927<a href="https://doi.org/10.1359/jbmr.091022">10.1359/jbmr.091022</a>.
</div>
<div id="ref-Harvey2013" class="csl-entry">
253. Harvey NC, Cole ZA, Crozier SR, Ntani G, Mahon PA, Robinson SM, Inskip HM, Godfrey KM, Dennison EM, Cooper C: <strong><span class="nocase">Fetal and infant growth predict hip geometry at 6 y old: Findings from the Southampton Women’s Survey</span></strong>. <em>Pediatric Research</em> 2013, <strong>74</strong>:450–456<a href="https://doi.org/10.1038/pr.2013.119">10.1038/pr.2013.119</a>.
</div>
<div id="ref-Mahon2010a" class="csl-entry">
254. Mahon P, Harvey N, Crozier S, Inskip H, Robinson S, Arden N, Swaminathan R, Cooper C, Godfrey K: <strong><span class="nocase">Low maternal vitamin D status and fetal bone development: Cohort study</span></strong>. <em>Journal of Bone and Mineral Research</em> 2010, <strong>25</strong>:14–19<a href="https://doi.org/10.1359/jbmr.090701">10.1359/jbmr.090701</a>Available: <a href="http://doi.wiley.com/10.1359/jbmr.090701">http://doi.wiley.com/10.1359/jbmr.090701</a>.
</div>
<div id="ref-Viljakainen2010" class="csl-entry">
255. Viljakainen HT, Saarnio E, Hytinantti T, Miettinen M, Surcel H, Mäkitie O, Andersson S, Laitinen K, Lamberg-Allardt C: <strong><span class="nocase">Maternal Vitamin D Status Determines Bone Variables in the Newborn</span></strong>. <em>The Journal of Clinical Endocrinology <span>&amp;</span> Metabolism</em> 2010, <strong>95</strong>:1749–1757<a href="https://doi.org/10.1210/jc.2009-1391">10.1210/jc.2009-1391</a>Available: <a href="https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2009-1391">https://academic.oup.com/jcem/article-lookup/doi/10.1210/jc.2009-1391</a>.
</div>
<div id="ref-Viljakainen2011" class="csl-entry">
256. Viljakainen HT, Korhonen T, Hytinantti T, Laitinen EKA, Andersson S, Mäkitie O, Lamberg-Allardt C: <strong><span class="nocase">Maternal vitamin D status affects bone growth in early childhood—a prospective cohort study</span></strong>. <em>Osteoporosis International</em> 2011, <strong>22</strong>:883–891<a href="https://doi.org/10.1007/s00198-010-1499-4">10.1007/s00198-010-1499-4</a>Available: <a href="http://link.springer.com/10.1007/s00198-010-1499-4">http://link.springer.com/10.1007/s00198-010-1499-4</a>.
</div>
<div id="ref-Zhu2014a" class="csl-entry">
257. Zhu K, Whitehouse AJ, Hart PH, Kusel M, Mountain J, Lye S, Pennell C, Walsh JP: <strong><span class="nocase">Maternal Vitamin D Status During Pregnancy and Bone Mass in Offspring at 20 Years of Age: A Prospective Cohort Study</span></strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:1088–1095<a href="https://doi.org/10.1002/jbmr.2138">10.1002/jbmr.2138</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2138">http://doi.wiley.com/10.1002/jbmr.2138</a>.
</div>
<div id="ref-Kip2004" class="csl-entry">
258. Kip SN, Strehler EE: <strong><span class="nocase">Vitamin D 3 upregulates plasma membrane Ca 2+ -ATPase expression and potentiates apico-basal Ca 2+ flux in MDCK cells</span></strong>. <em>American Journal of Physiology-Renal Physiology</em> 2004, <strong>286</strong>:F363–F369<a href="https://doi.org/10.1152/ajprenal.00076.2003">10.1152/ajprenal.00076.2003</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/14583431">http://www.ncbi.nlm.nih.gov/pubmed/14583431</a>.
</div>
<div id="ref-Burdge2007a" class="csl-entry">
259. Burdge GC, Slater-Jefferies J, Torrens C, Phillips ES, Hanson MA, Lillycrop KA: <strong><span class="nocase">Dietary protein restriction of pregnant rats in the F 0 generation induces altered methylation of hepatic gene promoters in the adult male offspring in the F 1 and F 2 generations</span></strong>. <em>British Journal of Nutrition</em> 2007, <strong>97</strong>:435–439<a href="https://doi.org/10.1017/S0007114507352392">10.1017/S0007114507352392</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507352392/type/journal{\_}article">https://www.cambridge.org/core/product/identifier/S0007114507352392/type/journal{\_}article</a>.
</div>
<div id="ref-Lillycrop2008" class="csl-entry">
260. Lillycrop KA, Phillips ES, Torrens C, Hanson MA, Jackson AA, Burdge GC: <strong><span class="nocase">Feeding pregnant rats a protein-restricted diet persistently alters the methylation of specific cytosines in the hepatic PPAR<span class="math inline">\(\alpha\)</span> promoter of the offspring</span></strong>. <em>British Journal of Nutrition</em> 2008, <strong>100</strong>:278–282<a href="https://doi.org/10.1017/S0007114507894438">10.1017/S0007114507894438</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507894438/type/journal{\_}article">https://www.cambridge.org/core/product/identifier/S0007114507894438/type/journal{\_}article</a>.
</div>
<div id="ref-Burdge2007" class="csl-entry">
261. Burdge GC, Hanson MA, Slater-Jefferies JL, Lillycrop KA: <strong><span class="nocase">Epigenetic regulation of transcription: a mechanism for inducing variations in phenotype (fetal programming) by differences in nutrition during early life?</span></strong> <em>British Journal of Nutrition</em> 2007, <strong>97</strong>:1036–1046<a href="https://doi.org/10.1017/S0007114507682920">10.1017/S0007114507682920</a>Available: <a href="https://www.cambridge.org/core/product/identifier/S0007114507682920/type/journal{\_}article">https://www.cambridge.org/core/product/identifier/S0007114507682920/type/journal{\_}article</a>.
</div>
<div id="ref-Holroyd2012" class="csl-entry">
262. Holroyd C, Harvey N, Dennison E, Cooper C: <strong><span class="nocase">Epigenetic influences in the developmental origins of osteoporosis</span></strong>. <em>Osteoporosis International</em> 2012, <strong>23</strong>:401–410<a href="https://doi.org/10.1007/s00198-011-1671-5">10.1007/s00198-011-1671-5</a>.
</div>
<div id="ref-Harvey2014d" class="csl-entry">
263. Harvey NC, Sheppard A, Godfrey KM, McLean C, Garratt E, Ntani G, Davies L, Murray R, Inskip HM, Gluckman PD, Hanson MA, Lillycrop KA, Cooper C: <strong><span class="nocase">Childhood Bone Mineral Content Is Associated With Methylation Status of the RXRA Promoter at Birth</span></strong>. <em>Journal of Bone and Mineral Research</em> 2014, <strong>29</strong>:600–607<a href="https://doi.org/10.1002/jbmr.2056">10.1002/jbmr.2056</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.2056">http://doi.wiley.com/10.1002/jbmr.2056</a>.
</div>
<div id="ref-Curtis2017" class="csl-entry">
264. Curtis EM, Murray R, Titcombe P, Cook E, Clarke-Harris R, Costello P, Garratt E, Holbrook JD, Barton S, Inskip H, Godfrey KM, Bell CG, Cooper C, Lillycrop KA, Harvey NC: <strong><span class="nocase">Perinatal DNA Methylation at CDKN2A Is Associated With Offspring Bone Mass: Findings From the Southampton Women’s Survey</span></strong>. <em>Journal of Bone and Mineral Research</em> 2017, <strong>32</strong>:2030–2040<a href="https://doi.org/10.1002/jbmr.3153">10.1002/jbmr.3153</a>Available: <a href="http://doi.wiley.com/10.1002/jbmr.3153">http://doi.wiley.com/10.1002/jbmr.3153</a>.
</div>
<div id="ref-Richards2006" class="csl-entry">
265. Richards EJ: <strong><span class="nocase">Inherited epigenetic variation–revisiting soft inheritance.</span></strong> <em>Nature reviews. Genetics</em> 2006, <strong>7</strong>:395–401<a href="https://doi.org/10.1038/nrg1834">10.1038/nrg1834</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16534512">http://www.ncbi.nlm.nih.gov/pubmed/16534512</a>.
</div>
<div id="ref-Holland2017" class="csl-entry">
266. Holland ML, Lowe R, Caton PW, Gemma C, Carbajosa G, Danson AF, Carpenter AAM, Loche E, Ozanne SE, Rakyan VK: <strong><span class="nocase">Early-life nutrition modulates the epigenetic state of specific rDNA genetic variants in mice</span></strong>. <em>Science</em> 2016, <strong>353</strong>:495–498<a href="https://doi.org/10.1126/science.aaf7040">10.1126/science.aaf7040</a>Available: <a href="http://www.sciencemag.org/lookup/doi/10.1126/science.aaf7040">http://www.sciencemag.org/lookup/doi/10.1126/science.aaf7040</a>.
</div>
<div id="ref-Dogan2017" class="csl-entry">
267. Dogan MV, Beach SRH, Philibert RA: <strong><span class="nocase">Genetically contextual effects of smoking on genome wide DNA methylation</span></strong>. <em>American Journal of Medical Genetics Part B: Neuropsychiatric Genetics</em> 2017, <strong>174</strong>:595–607<a href="https://doi.org/10.1002/ajmg.b.32565">10.1002/ajmg.b.32565</a>Available: <a href="http://doi.wiley.com/10.1002/ajmg.b.32565">http://doi.wiley.com/10.1002/ajmg.b.32565</a>.
</div>
<div id="ref-Dai2016" class="csl-entry">
268. Dai L, Mehta A, Mordukhovich I, Just AC, Shen J, Hou L, Koutrakis P, Sparrow D, Vokonas PS, Baccarelli AA, Schwartz JD: <strong><span class="nocase">Differential DNA methylation and PM 2.5 species in a 450K epigenome-wide association study</span></strong>. <em>Epigenetics</em> 2017, <strong>12</strong>:139–148<a href="https://doi.org/10.1080/15592294.2016.1271853">10.1080/15592294.2016.1271853</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2016.1271853">https://www.tandfonline.com/doi/full/10.1080/15592294.2016.1271853</a>.
</div>
<div id="ref-Wahl2016" class="csl-entry">
269. Wahl S, Drong A, Lehne B, Loh M, Scott WR, Kunze S, Tsai P-C, Ried JS, Zhang W, Yang Y, Tan S, Fiorito G, Franke L, Guarrera S, Kasela S, Kriebel J, Richmond RC, Adamo M, Afzal U, Ala-Korpela M, Albetti B, Ammerpohl O, Apperley JF, Beekman M, Bertazzi PA, Black SL, Blancher C, Bonder M-J, Brosch M, Carstensen-Kirberg M, et al.: <strong><span class="nocase">Epigenome-wide association study of body mass index, and the adverse outcomes of adiposity.</span></strong> <em>Nature</em> 2017, <strong>541</strong>:81–86<a href="https://doi.org/10.1038/nature20784">10.1038/nature20784</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28002404">http://www.ncbi.nlm.nih.gov/pubmed/28002404</a>.
</div>
<div id="ref-Min2017" class="csl-entry">
270. Min J, Hemani G, Smith GD, Relton CL, Suderman M: <strong><span class="nocase">Meffil: efficient normalisation and analysis of very large DNA methylation samples</span></strong>. <em>bioRxiv</em> 2017, <strong>44</strong>:125963<a href="https://doi.org/10.1101/125963">10.1101/125963</a>Available: <a href="https://www.biorxiv.org/content/early/2017/04/27/125963">https://www.biorxiv.org/content/early/2017/04/27/125963</a>.
</div>
<div id="ref-Jaffe2014" class="csl-entry">
271. Jaffe AE, Irizarry RA: <strong><span class="nocase">Accounting for cellular heterogeneity is critical in epigenome-wide association studies.</span></strong> <em>Genome biology</em> 2014, <strong>15</strong>:R31<a href="https://doi.org/10.1186/gb-2014-15-2-r31">10.1186/gb-2014-15-2-r31</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24495553">http://www.ncbi.nlm.nih.gov/pubmed/24495553</a>.
</div>
<div id="ref-Houseman2012" class="csl-entry">
272. Houseman EA, Accomando WP, Koestler DC, Christensen BC, Marsit CJ, Nelson HH, Wiencke JK, Kelsey KT: <strong><span class="nocase">DNA methylation arrays as surrogate measures of cell mixture distribution</span></strong>. <em>BMC Bioinformatics</em> 2012, <strong>13</strong>:86<a href="https://doi.org/10.1186/1471-2105-13-86">10.1186/1471-2105-13-86</a>Available: <a href="http://www.biomedcentral.com/1471-2105/13/86">http://www.biomedcentral.com/1471-2105/13/86</a>.
</div>
<div id="ref-Cardenas2016" class="csl-entry">
273. Cardenas A, Allard C, Doyon M, Houseman EA, Bakulski KM, Perron P, Bouchard L, Hivert M-F: <strong><span class="nocase">Validation of a DNA methylation reference panel for the estimation of nucleated cells types in cord blood.</span></strong> <em>Epigenetics</em> 2016, <strong>11</strong>:773–779<a href="https://doi.org/10.1080/15592294.2016.1233091">10.1080/15592294.2016.1233091</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27668573">http://www.ncbi.nlm.nih.gov/pubmed/27668573</a>.
</div>
<div id="ref-DeGoede2015" class="csl-entry">
274. Goede OM de, Razzaghian HR, Price EM, Jones MJ, Kobor MS, Robinson WP, Lavoie PM: <strong><span class="nocase">Nucleated red blood cells impact DNA methylation and expression analyses of cord blood hematopoietic cells</span></strong>. <em>Clinical Epigenetics</em> 2015, <strong>7</strong>:95<a href="https://doi.org/10.1186/s13148-015-0129-6">10.1186/s13148-015-0129-6</a>Available: <a href="http://dx.doi.org/10.1186/s13148-015-0129-6">http://dx.doi.org/10.1186/s13148-015-0129-6</a>.
</div>
<div id="ref-Bakulski2016" class="csl-entry">
275. Bakulski KM, Feinberg JI, Andrews SV, Yang J, Brown S, L. McKenney S, Witter F, Walston J, Feinberg AP, Fallin MD: <strong><span class="nocase">DNA methylation of cord blood cell types: Applications for mixed cell birth studies</span></strong>. <em>Epigenetics</em> 2016, <strong>11</strong>:354–362<a href="https://doi.org/10.1080/15592294.2016.1161875">10.1080/15592294.2016.1161875</a>Available: <a href="http://dx.doi.org/10.1080/15592294.2016.1161875">http://dx.doi.org/10.1080/15592294.2016.1161875</a>.
</div>
<div id="ref-Gervin2016" class="csl-entry">
276. Gervin K, Page CM, Aass HCD, Jansen MA, Fjeldstad HE, Andreassen BK, Duijts L, Meurs JB van, Zelm MC van, Jaddoe VW, Nordeng H, Knudsen GP, Magnus P, Nystad W, Staff AC, Felix JF, Lyle R: <strong><span class="nocase">Cell type specific DNA methylation in cord blood: A 450K-reference data set and cell count-based validation of estimated cell type composition</span></strong>. <em>Epigenetics</em> 2016, <strong>11</strong>:690–698<a href="https://doi.org/10.1080/15592294.2016.1214782">10.1080/15592294.2016.1214782</a>Available: <a href="http://dx.doi.org/10.1080/15592294.2016.1214782">http://dx.doi.org/10.1080/15592294.2016.1214782</a>.
</div>
<div id="ref-Leek2007" class="csl-entry">
277. Leek JT, Storey JD: <strong><span class="nocase">Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis</span></strong>. <em>PLoS Genetics</em> 2007, <strong>3</strong>:e161<a href="https://doi.org/10.1371/journal.pgen.0030161">10.1371/journal.pgen.0030161</a>Available: <a href="https://dx.plos.org/10.1371/journal.pgen.0030161">https://dx.plos.org/10.1371/journal.pgen.0030161</a>.
</div>
<div id="ref-Teschendorff2011" class="csl-entry">
278. Teschendorff AE, Zhuang J, Widschwendter M: <strong><span class="nocase">Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies</span></strong>. <em>Bioinformatics</em> 2011, <strong>27</strong>:1496–1505<a href="https://doi.org/10.1093/bioinformatics/btr171">10.1093/bioinformatics/btr171</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr171">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btr171</a>.
</div>
<div id="ref-McGregor2016" class="csl-entry">
279. McGregor K, Bernatsky S, Colmegna I, Hudson M, Pastinen T, Labbe A, Greenwood CMT: <strong><span class="nocase">An evaluation of methods correcting for cell-type heterogeneity in DNA methylation studies</span></strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:84<a href="https://doi.org/10.1186/s13059-016-0935-y">10.1186/s13059-016-0935-y</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0935-y">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0935-y</a>.
</div>
<div id="ref-Teschendorff2017" class="csl-entry">
280. Teschendorff AE, Zheng SC: <strong><span class="nocase">Cell-type deconvolution in epigenome-wide association studies: a review and recommendations</span></strong>. <em>Epigenomics</em> 2017, <strong>9</strong>:757–768<a href="https://doi.org/10.2217/epi-2016-0153">10.2217/epi-2016-0153</a>Available: <a href="https://www.futuremedicine.com/doi/10.2217/epi-2016-0153">https://www.futuremedicine.com/doi/10.2217/epi-2016-0153</a>.
</div>
<div id="ref-Lin2018" class="csl-entry">
281. Lin X, Tan JYL, Teh AL, Lim IY, Liew SJ, MacIsaac JL, Chong YS, Gluckman PD, Kobor MS, Cheong CY, Karnani N: <strong><span class="nocase">Cell type-specific DNA methylation in neonatal cord tissue and cord blood: a 850K-reference panel and comparison of cell types</span></strong>. <em>Epigenetics</em> 2018, <strong>13</strong>:941–958<a href="https://doi.org/10.1080/15592294.2018.1522929">10.1080/15592294.2018.1522929</a>Available: <a href="https://doi.org/10.1080/15592294.2018.1522929">https://doi.org/10.1080/15592294.2018.1522929</a>.
</div>
<div id="ref-Wang2011" class="csl-entry">
282. Wang S: <strong><span class="nocase">Method to detect differentially methylated loci with case-control designs using Illumina arrays</span></strong>. <em>Genetic Epidemiology</em> 2011, <strong>35</strong>:686–694<a href="https://doi.org/10.1002/gepi.20619">10.1002/gepi.20619</a>Available: <a href="http://doi.wiley.com/10.1002/gepi.20619">http://doi.wiley.com/10.1002/gepi.20619</a>.
</div>
<div id="ref-Tsai2015" class="csl-entry">
283. Tsai PC, Bell JT: <strong><span class="nocase">Power and sample size estimation for epigenome-wide association scans to detect differential DNA methylation</span></strong>. <em>International Journal of Epidemiology</em> 2015, <strong>44</strong>:1429–1441<a href="https://doi.org/10.1093/ije/dyv041">10.1093/ije/dyv041</a>.
</div>
<div id="ref-Graw2019" class="csl-entry">
284. Graw S, Henn R, Thompson JA, Koestler DC: <strong><span class="nocase">PwrEWAS: A user-friendly tool for comprehensive power estimation for epigenome wide association studies (EWAS)</span></strong>. <em>BMC Bioinformatics</em> 2019, <strong>20</strong>:1–11<a href="https://doi.org/10.1186/s12859-019-2804-7">10.1186/s12859-019-2804-7</a>.
</div>
<div id="ref-Cohen1988" class="csl-entry">
285. Cohen J: <em><span class="nocase">Statisticsl Power Analysis for the Behavioural Sciences</span></em>. 2nd ed. New York: Lawrence Erlbaum Associates; 1988.
</div>
<div id="ref-Joubert2016" class="csl-entry">
286. Joubert BR, Felix JF, Yousefi P, Bakulski KM, Just AC, Breton C, Reese SE, Markunas CA, Richmond RC, Xu C-J, Küpers LK, Oh SS, Hoyo C, Gruzieva O, Söderhäll C, Salas LA, Baïz N, Zhang H, Lepeule J, Ruiz C, Ligthart S, Wang T, Taylor JA, Duijts L, Sharp GC, Jankipersadsing SA, Nilsen RM, Vaez A, Fallin MD, Hu D, et al.: <strong><span class="nocase">DNA Methylation in Newborns and Maternal Smoking in Pregnancy: Genome-wide Consortium Meta-analysis</span></strong>. <em>The American Journal of Human Genetics</em> 2016, <strong>98</strong>:680–696<a href="https://doi.org/10.1016/j.ajhg.2016.02.019">10.1016/j.ajhg.2016.02.019</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27040690">http://www.ncbi.nlm.nih.gov/pubmed/27040690</a>.
</div>
<div id="ref-Breton2017" class="csl-entry">
287. Breton CV, Marsit CJ, Faustman E, Nadeau K, Goodrich JM, Dolinoy DC, Herbstman J, Holland N, LaSalle JM, Schmidt R, Yousefi P, Perera F, Joubert BR, Wiemels J, Taylor M, Yang IV, Chen R, Hew KM, Freeland DMH, Miller R, Murphy SK: <strong><span class="nocase">Small-Magnitude Effect Sizes in Epigenetic End Points are Important in Children’s Environmental Health Studies: The Children’s Environmental Health and Disease Prevention Research Center’s Epigenetics Working Group</span></strong>. <em>Environmental Health Perspectives</em> 2017, <strong>125</strong>:511–526<a href="https://doi.org/10.1289/EHP595">10.1289/EHP595</a>Available: <a href="https://ehp.niehs.nih.gov/doi/10.1289/EHP595">https://ehp.niehs.nih.gov/doi/10.1289/EHP595</a>.
</div>
<div id="ref-Partridge2018" class="csl-entry">
288. Partridge L, Deelen J, Slagboom PE: <strong><span class="nocase">Facing up to the global challenges of ageing</span></strong>. <em>Nature</em> 2018, <strong>561</strong>:45–56<a href="https://doi.org/10.1038/s41586-018-0457-8">10.1038/s41586-018-0457-8</a>Available: <a href="http://www.nature.com/articles/s41586-018-0457-8">http://www.nature.com/articles/s41586-018-0457-8</a>.
</div>
<div id="ref-Campisi2019" class="csl-entry">
289. Campisi J, Kapahi P, Lithgow GJ, Melov S, Newman JC, Verdin E: <strong><span class="nocase">From discoveries in ageing research to therapeutics for healthy ageing</span></strong>. <em>Nature</em> 2019, <strong>571</strong>:183–192<a href="https://doi.org/10.1038/s41586-019-1365-2">10.1038/s41586-019-1365-2</a>Available: <a href="http://www.nature.com/articles/s41586-019-1365-2">http://www.nature.com/articles/s41586-019-1365-2</a>.
</div>
<div id="ref-Chuong2017" class="csl-entry">
290. Chuong EB, Elde NC, Feschotte C: <strong><span class="nocase">Regulatory activities of transposable elements: from conflicts to benefits</span></strong>. <em>Nature Reviews Genetics</em> 2017, <strong>18</strong>:71–86<a href="https://doi.org/10.1038/nrg.2016.139">10.1038/nrg.2016.139</a>Available: <a href="http://www.nature.com/articles/nrg.2016.139">http://www.nature.com/articles/nrg.2016.139</a>.
</div>
<div id="ref-Sandoval2011" class="csl-entry">
291. Sandoval J, Heyn H, Moran S, Serra-Musach J, Pujana MA, Bibikova M, Esteller M: <strong><span class="nocase">Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome</span></strong>. <em>Epigenetics</em> 2011, <strong>6</strong>:692–702<a href="https://doi.org/10.4161/epi.6.6.16196">10.4161/epi.6.6.16196</a>Available: <a href="http://www.tandfonline.com/doi/abs/10.4161/epi.6.6.16196">http://www.tandfonline.com/doi/abs/10.4161/epi.6.6.16196</a>.
</div>
<div id="ref-Horvath2018a" class="csl-entry">
292. Horvath S, Oshima J, Martin GM, Lu AT, Quach A, Cohen H, Felton S, Matsuyama M, Lowe D, Kabacik S, Wilson JG, Reiner AP, Maierhofer A, Flunkert J, Aviv A, Hou L, Baccarelli AA, Li Y, Stewart JD, Whitsel EA, Ferrucci L, Matsuyama S, Raj K: <strong><span class="nocase">Epigenetic clock for skin and blood cells applied to Hutchinson Gilford Progeria Syndrome and ex vivo studies</span></strong>. <em>Aging</em> 2018, <strong>10</strong>:1758–1775<a href="https://doi.org/10.18632/aging.101508">10.18632/aging.101508</a>Available: <a href="http://www.aging-us.com/article/101508/text">http://www.aging-us.com/article/101508/text</a>.
</div>
<div id="ref-Horvath2018" class="csl-entry">
293. Horvath S, Raj K: <strong><span class="nocase">DNA methylation-based biomarkers and the epigenetic clock theory of ageing</span></strong>. <em>Nature Reviews Genetics</em> 2018, <strong>19</strong>:371–384<a href="https://doi.org/10.1038/s41576-018-0004-3">10.1038/s41576-018-0004-3</a>Available: <a href="http://www.nature.com/articles/s41576-018-0004-3">http://www.nature.com/articles/s41576-018-0004-3</a>.
</div>
<div id="ref-Field2018" class="csl-entry">
294. Field AE, Robertson NA, Wang T, Havas A, Ideker T, Adams PD: <strong><span class="nocase">DNA Methylation Clocks in Aging: Categories, Causes, and Consequences</span></strong>. <em>Molecular Cell</em> 2018, <strong>71</strong>:882–895<a href="https://doi.org/10.1016/j.molcel.2018.08.008">10.1016/j.molcel.2018.08.008</a>Available: <a href="https://doi.org/10.1016/j.molcel.2018.08.008">https://doi.org/10.1016/j.molcel.2018.08.008</a>.
</div>
<div id="ref-Bhattacharyya2016" class="csl-entry">
295. Bhattacharyya S, Varshney U: <strong><span class="nocase">Evolution of initiator tRNAs and selection of methionine as the initiating amino acid</span></strong>. <em>RNA Biology</em> 2016, <strong>13</strong>:810–819<a href="https://doi.org/10.1080/15476286.2016.1195943">10.1080/15476286.2016.1195943</a>Available: <a href="http://dx.doi.org/10.1080/15476286.2016.1195943">http://dx.doi.org/10.1080/15476286.2016.1195943</a>.
</div>
<div id="ref-Kamhi2010" class="csl-entry">
296. Kamhi E, Raitskin O, Sperling R, Sperling J: <strong><span class="nocase">A potential role for initiator-tRNA in pre-mRNA splicing regulation</span></strong>. <em>Proceedings of the National Academy of Sciences of the United States of America</em> 2010, <strong>107</strong>:11319–11324<a href="https://doi.org/10.1073/pnas.0911561107">10.1073/pnas.0911561107</a>.
</div>
<div id="ref-Birch2016" class="csl-entry">
297. Birch J, Clarke CJ, Campbell AD, Campbell K, Mitchell L, Liko D, Kalna G, Strathdee D, Sansom OJ, Neilson M, Blyth K, Norman JC: <strong><span class="nocase">The initiator methionine tRNA drives cell migration and invasion leading to increased metastatic potential in melanoma</span></strong>. <em>Biology Open</em> 2016, <strong>5</strong>:1371–1379<a href="https://doi.org/10.1242/bio.019075">10.1242/bio.019075</a>.
</div>
<div id="ref-Rideout2012" class="csl-entry">
298. Rideout EJ, Marshall L, Grewal SS: <strong><span class="nocase">Drosophila RNA polymerase III repressor Maf1 controls body size and developmental timing by modulating tRNAiMet synthesis and systemic insulin signaling</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2012, <strong>109</strong>:1139–1144<a href="https://doi.org/10.1073/pnas.1113311109">10.1073/pnas.1113311109</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1113311109">http://www.pnas.org/cgi/doi/10.1073/pnas.1113311109</a>.
</div>
<div id="ref-Kolitz2010" class="csl-entry">
299. Kolitz SE, Lorsch JR: <strong><span class="nocase">Eukaryotic initiator tRNA: Finely tuned and ready for action</span></strong>. <em>FEBS Letters</em> 2010, <strong>584</strong>:396–404<a href="https://doi.org/10.1016/j.febslet.2009.11.047">10.1016/j.febslet.2009.11.047</a>Available: <a href="http://doi.wiley.com/10.1016/j.febslet.2009.11.047">http://doi.wiley.com/10.1016/j.febslet.2009.11.047</a>.
</div>
<div id="ref-Pavon-Eternod2013" class="csl-entry">
300. Pavon-Eternod M, Gomes S, Rosner MR, Pan T: <strong><span class="nocase">Overexpression of initiator methionine tRNA leads to global reprogramming of tRNA expression and increased proliferation in human epithelial cells</span></strong>. <em>RNA</em> 2013, <strong>19</strong>:461–466<a href="https://doi.org/10.1261/rna.037507.112">10.1261/rna.037507.112</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23431330">http://www.ncbi.nlm.nih.gov/pubmed/23431330</a>.
</div>
<div id="ref-Eigen1989" class="csl-entry">
301. Eigen M, Lindemann B, Tietze M, Winkler-Oswatitsch R, Dress A, Haeseler A von: <strong><span class="nocase">How old is the genetic code? Statistical geometry of tRNA provides an answer</span></strong>. <em>Science</em> 1989, <strong>244</strong>:673–679<a href="https://doi.org/10.1126/science.2497522">10.1126/science.2497522</a>Available: <a href="http://www.sciencemag.org/cgi/doi/10.1126/science.2497522">http://www.sciencemag.org/cgi/doi/10.1126/science.2497522</a>.
</div>
<div id="ref-Tavernarakis2008" class="csl-entry">
302. Tavernarakis N: <strong><span class="nocase">Ageing and the regulation of protein synthesis: a balancing act?</span></strong> <em>Trends in Cell Biology</em> 2008, <strong>18</strong>:228–235<a href="https://doi.org/10.1016/j.tcb.2008.02.004">10.1016/j.tcb.2008.02.004</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0962892408000792">https://linkinghub.elsevier.com/retrieve/pii/S0962892408000792</a>.
</div>
<div id="ref-Parisien2013" class="csl-entry">
303. Parisien M, Wang X, Pan T: <strong><span class="nocase">Diversity of human tRNA genes from the 1000-genomes project</span></strong>. <em>RNA Biology</em> 2013, <strong>10</strong>:1853–1867<a href="https://doi.org/10.4161/rna.27361">10.4161/rna.27361</a>Available: <a href="http://www.tandfonline.com/doi/full/10.4161/rna.27361">http://www.tandfonline.com/doi/full/10.4161/rna.27361</a>.
</div>
<div id="ref-Chan2009" class="csl-entry">
304. Chan PP, Lowe TM: <strong><span class="nocase">GtRNAdb: a database of transfer RNA genes detected in genomic sequence.</span></strong> <em>Nucleic acids research</em> 2009, <strong>37</strong>:D93–7<a href="https://doi.org/10.1093/nar/gkn787">10.1093/nar/gkn787</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/18984615">http://www.ncbi.nlm.nih.gov/pubmed/18984615</a>.
</div>
<div id="ref-Torres2019a" class="csl-entry">
305. Torres AG: <strong><span class="nocase">Enjoy the Silence: Nearly Half of Human tRNA Genes Are Silent</span></strong>. <em>Bioinformatics and Biology Insights</em> 2019, <strong>13</strong>:117793221986845<a href="https://doi.org/10.1177/1177932219868454">10.1177/1177932219868454</a>Available: <a href="http://journals.sagepub.com/doi/10.1177/1177932219868454">http://journals.sagepub.com/doi/10.1177/1177932219868454</a>.
</div>
<div id="ref-Lodish2000" class="csl-entry">
306. Lodish H, Berk A, Zipursky SL, Matsudaira P, Baltimore D, Darnell J: <em><span class="nocase">Molecular Cell Biology, 4th edition</span></em>. 4th ed. New York: W. H. Freeman; 2000 Available: <a href="https://www.ncbi.nlm.nih.gov/books/NBK21475/">https://www.ncbi.nlm.nih.gov/books/NBK21475/</a>.
</div>
<div id="ref-Schramm2002" class="csl-entry">
307. Schramm L: <strong><span class="nocase">Recruitment of RNA polymerase III to its target promoters</span></strong>. <em>Genes <span>&amp;</span> Development</em> 2002, <strong>16</strong>:2593–2620<a href="https://doi.org/10.1101/gad.1018902">10.1101/gad.1018902</a>Available: <a href="http://www.genesdev.org/cgi/doi/10.1101/gad.1018902">http://www.genesdev.org/cgi/doi/10.1101/gad.1018902</a>.
</div>
<div id="ref-Canella2010" class="csl-entry">
308. Canella D, Praz V, Reina JH, Cousin P, Hernandez N: <strong><span class="nocase">Defining the RNA polymerase III transcriptome: Genome-wide localization of the RNA polymerase III transcription machinery in human cells.</span></strong> <em>Genome research</em> 2010, <strong>20</strong>:710–21<a href="https://doi.org/10.1101/gr.101337.109">10.1101/gr.101337.109</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20413673">http://www.ncbi.nlm.nih.gov/pubmed/20413673</a>.
</div>
<div id="ref-Dieci1996" class="csl-entry">
309. Dieci G, Sentenac A: <strong><span class="nocase">Facilitated Recycling Pathway for RNA Polymerase III</span></strong>. <em>Cell</em> 1996, <strong>84</strong>:245–252<a href="https://doi.org/10.1016/S0092-8674(00)80979-4">10.1016/S0092-8674(00)80979-4</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0092867400809794">https://linkinghub.elsevier.com/retrieve/pii/S0092867400809794</a>.
</div>
<div id="ref-Murawski1994" class="csl-entry">
310. Murawski M, Szczesniak B, Zoladek T, Hopper AK, Martin NC, Boguta M: <strong><span class="nocase">maf1 mutation alters the subcellular localization of the Mod5 protein in yeast.</span></strong> <em>Acta biochimica Polonica</em> 1994, <strong>41</strong>:441–8Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7732762">http://www.ncbi.nlm.nih.gov/pubmed/7732762</a>.
</div>
<div id="ref-Pluta2001" class="csl-entry">
311. Pluta K, Lefebvre O, Martin NC, Smagowicz WJ, Stanford DR, Ellis SR, Hopper AK, Sentenac A, Boguta M: <strong><span class="nocase">Maf1p, a Negative Effector of RNA Polymerase III in Saccharomyces cerevisiae</span></strong>. <em>Molecular and Cellular Biology</em> 2001, <strong>21</strong>:5031–5040<a href="https://doi.org/10.1128/MCB.21.15.5031-5040.2001">10.1128/MCB.21.15.5031-5040.2001</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11438659">http://www.ncbi.nlm.nih.gov/pubmed/11438659</a>.
</div>
<div id="ref-Vorlander2020" class="csl-entry">
312. Vorländer MK, Baudin F, Moir RD, Wetzel R, Hagen WJH, Willis IM, Müller CW: <strong><span class="nocase">Structural basis for RNA polymerase III transcription repression by Maf1</span></strong>. <em>Nature Structural <span>&amp;</span> Molecular Biology</em> 2020, <a href="https://doi.org/10.1038/s41594-020-0383-y">10.1038/s41594-020-0383-y</a>Available: <a href="http://www.nature.com/articles/s41594-020-0383-y">http://www.nature.com/articles/s41594-020-0383-y</a>.
</div>
<div id="ref-Mange2017" class="csl-entry">
313. Mange F, Praz V, Migliavacca E, Willis IM, Schütz F, Hernandez N: <strong><span class="nocase">Diurnal regulation of RNA polymerase III transcription is under the control of both the feeding–fasting response and the circadian clock</span></strong>. <em>Genome Research</em> 2017, <strong>27</strong>:973–984<a href="https://doi.org/10.1101/gr.217521.116">10.1101/gr.217521.116</a>Available: <a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.217521.116">http://genome.cshlp.org/lookup/doi/10.1101/gr.217521.116</a>.
</div>
<div id="ref-Kennedy2016" class="csl-entry">
314. Kennedy BK, Lamming DW: <strong><span class="nocase">The Mechanistic Target of Rapamycin: The Grand ConducTOR of Metabolism and Aging</span></strong>. <em>Cell Metabolism</em> 2016, <strong>23</strong>:990–1003<a href="https://doi.org/10.1016/j.cmet.2016.05.009">10.1016/j.cmet.2016.05.009</a>Available: <a href="http://dx.doi.org/10.1016/j.cmet.2016.05.009">http://dx.doi.org/10.1016/j.cmet.2016.05.009</a>.
</div>
<div id="ref-Crighton2003" class="csl-entry">
315. Crighton D: <strong><span class="nocase">p53 represses RNA polymerase III transcription by targeting TBP and inhibiting promoter occupancy by TFIIIB</span></strong>. <em>The EMBO Journal</em> 2003, <strong>22</strong>:2810–2820<a href="https://doi.org/10.1093/emboj/cdg265">10.1093/emboj/cdg265</a>Available: <a href="http://emboj.embopress.org/cgi/doi/10.1093/emboj/cdg265">http://emboj.embopress.org/cgi/doi/10.1093/emboj/cdg265</a>.
</div>
<div id="ref-Sutcliffe2000" class="csl-entry">
316. Sutcliffe JE, Brown TRP, Allison SJ, Scott PH, White RJ: <strong><span class="nocase">Retinoblastoma Protein Disrupts Interactions Required for RNA Polymerase III Transcription</span></strong>. <em>Molecular and Cellular Biology</em> 2000, <strong>20</strong>:9192–9202<a href="https://doi.org/10.1128/MCB.20.24.9192-9202.2000">10.1128/MCB.20.24.9192-9202.2000</a>Available: <a href="http://mcb.asm.org/cgi/doi/10.1128/MCB.20.24.9192-9202.2000">http://mcb.asm.org/cgi/doi/10.1128/MCB.20.24.9192-9202.2000</a>.
</div>
<div id="ref-Gomez-Roman2003" class="csl-entry">
317. Gomez-Roman N, Grandori C, Eisenman RN, White RJ: <strong><span class="nocase">Direct activation of RNA polymerase III transcription by c-Myc</span></strong>. <em>Nature</em> 2003, <strong>421</strong>:290–294<a href="https://doi.org/10.1038/nature01327">10.1038/nature01327</a>Available: <a href="http://www.nature.com/articles/nature01327">http://www.nature.com/articles/nature01327</a>.
</div>
<div id="ref-Krishnan2016" class="csl-entry">
318. Krishnan P, Ghosh S, Wang B, Heyns M, Li D, Mackey JR, Kovalchuk O, Damaraju S: <strong><span class="nocase">Genome-wide profiling of transfer RNAs and their role as novel prognostic markers for breast cancer</span></strong>. <em>Scientific Reports</em> 2016, <strong>6</strong>:32843<a href="https://doi.org/10.1038/srep32843">10.1038/srep32843</a>Available: <a href="http://dx.doi.org/10.1038/srep32843">http://dx.doi.org/10.1038/srep32843</a>.
</div>
<div id="ref-Huang2018" class="csl-entry">
319. Huang S, Sun B, Xiong Z, Shu Y, Zhou H, Zhang W, Xiong J, Li Q: <strong><span class="nocase">The dysregulation of tRNAs and tRNA derivatives in cancer</span></strong>. <em>Journal of Experimental <span>&amp;</span> Clinical Cancer Research</em> 2018, <strong>37</strong>:101<a href="https://doi.org/10.1186/s13046-018-0745-z">10.1186/s13046-018-0745-z</a>Available: <a href="https://jeccr.biomedcentral.com/articles/10.1186/s13046-018-0745-z">https://jeccr.biomedcentral.com/articles/10.1186/s13046-018-0745-z</a>.
</div>
<div id="ref-Besser1990" class="csl-entry">
320. Besser D, Götz F, Schulze-Forster K, Wagner H, Kröger H, Simon D: <strong><span class="nocase">DNA methylation inhibits transcription by RNA polymerase III of a tRNA gene, but not of a 5S rRNA gene.</span></strong> <em>FEBS letters</em> 1990, <strong>269</strong>:358–62<a href="https://doi.org/10.1016/0014-5793(90)81193-R">10.1016/0014-5793(90)81193-R</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2401361">http://www.ncbi.nlm.nih.gov/pubmed/2401361</a>.
</div>
<div id="ref-Varshney2015" class="csl-entry">
321. Varshney D, Vavrova-Anderson J, Oler AJ, Cowling VH, Cairns BR, White RJ: <strong><span class="nocase">SINE transcription by RNA polymerase III is suppressed by histone methylation but not by DNA methylation.</span></strong> <em>Nature communications</em> 2015, <strong>6</strong>:6569<a href="https://doi.org/10.1038/ncomms7569">10.1038/ncomms7569</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25798578">http://www.ncbi.nlm.nih.gov/pubmed/25798578</a>.
</div>
<div id="ref-Gerber2020" class="csl-entry">
322. Gerber A, Ito K, Chu C-S, Roeder RG: <strong><span class="nocase">Gene-Specific Control of tRNA Expression by RNA Polymerase II</span></strong>. <em>Molecular Cell</em> 2020, <strong>78</strong>:765–778.e7<a href="https://doi.org/10.1016/j.molcel.2020.03.023">10.1016/j.molcel.2020.03.023</a>Available: <a href="https://doi.org/10.1016/j.molcel.2020.03.023">https://doi.org/10.1016/j.molcel.2020.03.023</a>.
</div>
<div id="ref-Gingold2012" class="csl-entry">
323. Gingold H, Dahan O, Pilpel Y: <strong><span class="nocase">Dynamic changes in translational efficiency are deduced from codon usage of the transcriptome.</span></strong> <em>Nucleic acids research</em> 2012, <strong>40</strong>:10053–63<a href="https://doi.org/10.1093/nar/gks772">10.1093/nar/gks772</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22941644">http://www.ncbi.nlm.nih.gov/pubmed/22941644</a>.
</div>
<div id="ref-Athey2017" class="csl-entry">
324. Athey J, Alexaki A, Osipova E, Rostovtsev A, Santana-Quintero LV, Katneni U, Simonyan V, Kimchi-Sarfaty C: <strong><span class="nocase">A new and updated resource for codon usage tables</span></strong>. <em>BMC Bioinformatics</em> 2017, <strong>18</strong>:391<a href="https://doi.org/10.1186/s12859-017-1793-7">10.1186/s12859-017-1793-7</a>Available: <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1793-7">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1793-7</a>.
</div>
<div id="ref-Dittmar2006" class="csl-entry">
325. Dittmar KA, Goodenbour JM, Pan T: <strong><span class="nocase">Tissue-specific differences in human transfer RNA expression.</span></strong> <em>PLoS genetics</em> 2006, <strong>2</strong>:e221<a href="https://doi.org/10.1371/journal.pgen.0020221">10.1371/journal.pgen.0020221</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17194224">http://www.ncbi.nlm.nih.gov/pubmed/17194224</a>.
</div>
<div id="ref-Sagi2016" class="csl-entry">
326. Sagi D, Rak R, Gingold H, Adir I, Maayan G, Dahan O, Broday L, Pilpel Y, Rechavi O: <strong><span class="nocase">Tissue- and Time-Specific Expression of Otherwise Identical tRNA Genes</span></strong>. <em>PLOS Genetics</em> 2016, <strong>12</strong>:e1006264<a href="https://doi.org/10.1371/journal.pgen.1006264">10.1371/journal.pgen.1006264</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27560950">http://www.ncbi.nlm.nih.gov/pubmed/27560950</a>.
</div>
<div id="ref-Kirchner2017" class="csl-entry">
327. Kirchner S, Cai Z, Rauscher R, Kastelic N, Anding M, Czech A, Kleizen B, Ostedgaard LS, Braakman I, Sheppard DN, Ignatova Z: <strong><span class="nocase">Alteration of protein function by a silent polymorphism linked to tRNA abundance</span></strong>. <em>PLOS Biology</em> 2017, <strong>15</strong>:e2000779<a href="https://doi.org/10.1371/journal.pbio.2000779">10.1371/journal.pbio.2000779</a>Available: <a href="http://dx.plos.org/10.1371/journal.pbio.2000779">http://dx.plos.org/10.1371/journal.pbio.2000779</a>.
</div>
<div id="ref-Ishimura2015" class="csl-entry">
328. Ishimura R, Nagy G, Dotu I, Zhou H, Yang X-L, Schimmel P, Senju S, Nishimura Y, Chuang JH, Ackerman SL: <strong><span class="nocase">RNA function. Ribosome stalling induced by mutation of a CNS-specific tRNA causes neurodegeneration.</span></strong> <em>Science (New York, N.Y.)</em> 2014, <strong>345</strong>:455–9<a href="https://doi.org/10.1126/science.1249749">10.1126/science.1249749</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25061210">http://www.ncbi.nlm.nih.gov/pubmed/25061210</a>.
</div>
<div id="ref-Pliatsika2018" class="csl-entry">
329. Pliatsika V, Loher P, Magee R, Telonis AG, Londin E, Shigematsu M, Kirino Y, Rigoutsos I: <strong><span class="nocase">MINTbase v2.0: a comprehensive database for tRNA-derived fragments that includes nuclear and mitochondrial fragments from all The Cancer Genome Atlas projects</span></strong>. <em>Nucleic Acids Research</em> 2018, <strong>46</strong>:D152–D159<a href="https://doi.org/10.1093/nar/gkx1075">10.1093/nar/gkx1075</a>Available: <a href="http://academic.oup.com/nar/article/46/D1/D152/4653530">http://academic.oup.com/nar/article/46/D1/D152/4653530</a>.
</div>
<div id="ref-Lee2009" class="csl-entry">
330. Lee YS, Shibata Y, Malhotra A, Dutta A: <strong><span class="nocase">A novel class of small RNAs: tRNA-derived RNA fragments (tRFs).</span></strong> <em>Genes <span>&amp;</span> development</em> 2009, <strong>23</strong>:2639–49<a href="https://doi.org/10.1101/gad.1837609">10.1101/gad.1837609</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19933153">http://www.ncbi.nlm.nih.gov/pubmed/19933153</a>.
</div>
<div id="ref-Torres2019" class="csl-entry">
331. Torres AG, Reina O, Stephan-Otto Attolini C, Ribas de Pouplana L: <strong><span class="nocase">Differential expression of human tRNA genes drives the abundance of tRNA-derived fragments</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2019, <strong>116</strong>:201821120<a href="https://doi.org/10.1073/pnas.1821120116">10.1073/pnas.1821120116</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1821120116">http://www.pnas.org/lookup/doi/10.1073/pnas.1821120116</a>.
</div>
<div id="ref-Li2018a" class="csl-entry">
332. Li S, Xu Z, Sheng J: <strong><span class="nocase">tRNA-Derived Small RNA: A Novel Regulatory Small Non-Coding RNA</span></strong>. <em>Genes</em> 2018, <strong>9</strong>:246<a href="https://doi.org/10.3390/genes9050246">10.3390/genes9050246</a>Available: <a href="http://www.mdpi.com/2073-4425/9/5/246">http://www.mdpi.com/2073-4425/9/5/246</a>.
</div>
<div id="ref-Xu2017a" class="csl-entry">
333. Xu W-L, Yang Y, Wang Y-D, Qu L-H, Zheng L-L: <strong><span class="nocase">Computational Approaches to tRNA-Derived Small RNAs</span></strong>. <em>Non-Coding RNA</em> 2017, <strong>3</strong>:2<a href="https://doi.org/10.3390/ncrna3010002">10.3390/ncrna3010002</a>Available: <a href="http://www.mdpi.com/2311-553X/3/1/2">http://www.mdpi.com/2311-553X/3/1/2</a>.
</div>
<div id="ref-Keam2014" class="csl-entry">
334. Keam SP, Young PE, McCorkindale AL, Dang THY, Clancy JL, Humphreys DT, Preiss T, Hutvagner G, Martin DIK, Cropley JE, Suter CM: <strong><span class="nocase">The human Piwi protein Hiwi2 associates with tRNA-derived piRNAs in somatic cells</span></strong>. <em>Nucleic Acids Research</em> 2014, <strong>42</strong>:8984–8995<a href="https://doi.org/10.1093/nar/gku620">10.1093/nar/gku620</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gku620">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gku620</a>.
</div>
<div id="ref-Honda2017" class="csl-entry">
335. Honda S, Kawamura T, Loher P, Morichika K, Rigoutsos I, Kirino Y: <strong><span class="nocase">The biogenesis pathway of tRNA-derived piRNAs in Bombyx germ cells</span></strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:9108–9120<a href="https://doi.org/10.1093/nar/gkx537">10.1093/nar/gkx537</a>Available: <a href="http://academic.oup.com/nar/article/45/15/9108/3883741/The-biogenesis-pathway-of-tRNAderived-piRNAs-in">http://academic.oup.com/nar/article/45/15/9108/3883741/The-biogenesis-pathway-of-tRNAderived-piRNAs-in</a>.
</div>
<div id="ref-Tosar2018" class="csl-entry">
336. Tosar JP, Rovira C, Cayota A: <strong><span class="nocase">Non-coding RNA fragments account for the majority of annotated piRNAs expressed in somatic non-gonadal tissues</span></strong>. <em>Communications Biology</em> 2018, <strong>1</strong>:2<a href="https://doi.org/10.1038/s42003-017-0001-7">10.1038/s42003-017-0001-7</a>Available: <a href="http://www.nature.com/articles/s42003-017-0001-7">http://www.nature.com/articles/s42003-017-0001-7</a>.
</div>
<div id="ref-Loher2017" class="csl-entry">
337. Loher P, Telonis AG, Rigoutsos I: <strong><span class="nocase">MINTmap: fast and exhaustive profiling of nuclear and mitochondrial tRNA fragments from short RNA-seq data.</span></strong> <em>Scientific reports</em> 2017, <strong>7</strong>:41184<a href="https://doi.org/10.1038/srep41184">10.1038/srep41184</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28220888">http://www.ncbi.nlm.nih.gov/pubmed/28220888</a>.
</div>
<div id="ref-Kim2017" class="csl-entry">
338. Kim HK, Fuchs G, Wang S, Wei W, Zhang Y, Park H, Roy-Chaudhuri B, Li P, Xu J, Chu K, Zhang F, Chua M-S, So S, Zhang QC, Sarnow P, Kay MA: <strong><span class="nocase">A transfer-RNA-derived small RNA regulates ribosome biogenesis</span></strong>. <em>Nature</em> 2017, <strong>552</strong>:57–62<a href="https://doi.org/10.1038/nature25005">10.1038/nature25005</a>Available: <a href="http://www.nature.com/articles/nature25005">http://www.nature.com/articles/nature25005</a>.
</div>
<div id="ref-Martinez2017" class="csl-entry">
339. Martinez G, Choudury SG, Slotkin RK: <strong><span class="nocase">tRNA-derived small RNAs target transposable element transcripts</span></strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:5142–5152<a href="https://doi.org/10.1093/nar/gkx103">10.1093/nar/gkx103</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx103">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx103</a>.
</div>
<div id="ref-Schimmel2017" class="csl-entry">
340. Schimmel P: <strong><span class="nocase">The emerging complexity of the tRNA world: mammalian tRNAs beyond protein synthesis.</span></strong> <em>Nature reviews. Molecular cell biology</em> 2018, <strong>19</strong>:45–58<a href="https://doi.org/10.1038/nrm.2017.77">10.1038/nrm.2017.77</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28875994">http://www.ncbi.nlm.nih.gov/pubmed/28875994</a>.
</div>
<div id="ref-Cristodero2017" class="csl-entry">
341. Cristodero M, Polacek N: <strong><span class="nocase">The multifaceted regulatory potential of tRNA-derived fragments</span></strong>. <em>Non-coding RNA Investigation</em> 2017, <strong>1</strong>:7–7<a href="https://doi.org/10.21037/ncri.2017.08.07">10.21037/ncri.2017.08.07</a>Available: <a href="http://ncri.amegroups.com/article/view/3820/4459">http://ncri.amegroups.com/article/view/3820/4459</a>.
</div>
<div id="ref-Pace2006" class="csl-entry">
342. Pace DA: <strong><span class="nocase">Fixed metabolic costs for highly variable rates of protein synthesis in sea urchin embryos and larvae</span></strong>. <em>Journal of Experimental Biology</em> 2006, <strong>209</strong>:158–170<a href="https://doi.org/10.1242/jeb.01962">10.1242/jeb.01962</a>Available: <a href="http://jeb.biologists.org/cgi/doi/10.1242/jeb.01962">http://jeb.biologists.org/cgi/doi/10.1242/jeb.01962</a>.
</div>
<div id="ref-Nwanaji-Enwerem2018" class="csl-entry">
343. Nwanaji-Enwerem JC, Weisskopf MG, Baccarelli AA: <strong><span class="nocase">Multi-tissue DNA methylation age: Molecular relationships and perspectives for advancing biomarker utility</span></strong>. <em>Ageing Research Reviews</em> 2018, <strong>45</strong>:15–23<a href="https://doi.org/10.1016/j.arr.2018.04.005">10.1016/j.arr.2018.04.005</a>Available: <a href="https://doi.org/10.1016/j.arr.2018.04.005">https://doi.org/10.1016/j.arr.2018.04.005</a>.
</div>
<div id="ref-Hansen2007" class="csl-entry">
344. Hansen M, Taubert S, Crawford D, Libina N, Lee S-J, Kenyon C: <strong><span class="nocase">Lifespan extension by conditions that inhibit translation in Caenorhabditis elegans</span></strong>. <em>Aging Cell</em> 2007, <strong>6</strong>:95–110<a href="https://doi.org/10.1111/j.1474-9726.2006.00267.x">10.1111/j.1474-9726.2006.00267.x</a>Available: <a href="http://doi.wiley.com/10.1111/j.1474-9726.2006.00267.x">http://doi.wiley.com/10.1111/j.1474-9726.2006.00267.x</a>.
</div>
<div id="ref-Filer2017" class="csl-entry">
345. Filer D, Thompson MA, Takhaveev V, Dobson AJ, Kotronaki I, Green JWM, Heinemann M, Tullet JMA, Alic N: <strong><span class="nocase">RNA polymerase III limits longevity downstream of TORC1</span></strong>. <em>Nature</em> 2017, <strong>552</strong>:263–267<a href="https://doi.org/10.1038/nature25007">10.1038/nature25007</a>Available: <a href="http://www.nature.com/articles/nature25007">http://www.nature.com/articles/nature25007</a>.
</div>
<div id="ref-Dhahbi2013" class="csl-entry">
346. Dhahbi JM, Spindler SR, Atamna H, Yamakawa A, Boffelli D, Mote P, Martin DI: <strong><span class="nocase">5’ tRNA halves are present as abundant complexes in serum, concentrated in blood cells, and modulated by aging and calorie restriction</span></strong>. <em>BMC Genomics</em> 2013, <strong>14</strong>:298<a href="https://doi.org/10.1186/1471-2164-14-298">10.1186/1471-2164-14-298</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23638709">http://www.ncbi.nlm.nih.gov/pubmed/23638709</a>.
</div>
<div id="ref-Bhargava2013a" class="csl-entry">
347. Bhargava P: <strong><span class="nocase">Epigenetic regulation of transcription by RNA polymerase III</span></strong>. <em>Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</em> 2013, <strong>1829</strong>:1015–1025<a href="https://doi.org/10.1016/j.bbagrm.2013.05.005">10.1016/j.bbagrm.2013.05.005</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1874939913001004">https://linkinghub.elsevier.com/retrieve/pii/S1874939913001004</a>.
</div>
<div id="ref-Park2017" class="csl-entry">
348. Bhargava P: <strong><span class="nocase">Epigenetic regulation of transcription by RNA polymerase III.</span></strong> <em>Biochimica et biophysica acta</em> 2013, <strong>1829</strong>:1015–25<a href="https://doi.org/10.1016/j.bbagrm.2013.05.005">10.1016/j.bbagrm.2013.05.005</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23732820">http://www.ncbi.nlm.nih.gov/pubmed/23732820</a>.
</div>
<div id="ref-Wilusz2015" class="csl-entry">
349. Wilusz JE: <strong><span class="nocase">Controlling translation via modulation of tRNA levels</span></strong>. <em>Wiley Interdisciplinary Reviews: RNA</em> 2015, <strong>6</strong>:453–470<a href="https://doi.org/10.1002/wrna.1287">10.1002/wrna.1287</a>Available: <a href="http://doi.wiley.com/10.1002/wrna.1287">http://doi.wiley.com/10.1002/wrna.1287</a>.
</div>
<div id="ref-Cozen2015" class="csl-entry">
350. Cozen AE, Quartley E, Holmes AD, Hrabeta-Robinson E, Phizicky EM, Lowe TM: <strong><span class="nocase">ARM-seq: AlkB-facilitated RNA methylation sequencing reveals a complex landscape of modified tRNA fragments</span></strong>. <em>Nature Methods</em> 2015, <strong>12</strong>:879–884<a href="https://doi.org/10.1038/nmeth.3508">10.1038/nmeth.3508</a>Available: <a href="http://www.nature.com/articles/nmeth.3508">http://www.nature.com/articles/nmeth.3508</a>.
</div>
<div id="ref-Shigematsu2017a" class="csl-entry">
351. Shigematsu M, Honda S, Loher P, Telonis AG, Rigoutsos I, Kirino Y: <strong><span class="nocase">YAMAT-seq: an efficient method for high-throughput sequencing of mature transfer RNAs.</span></strong> <em>Nucleic acids research</em> 2017, <strong>45</strong>:e70<a href="https://doi.org/10.1093/nar/gkx005">10.1093/nar/gkx005</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28108659">http://www.ncbi.nlm.nih.gov/pubmed/28108659</a>.
</div>
<div id="ref-Gogakos2017" class="csl-entry">
352. Gogakos T, Brown M, Garzia A, Meyer C, Hafner M, Tuschl T: <strong><span class="nocase">Characterizing Expression and Processing of Precursor and Mature Human tRNAs by Hydro-tRNAseq and PAR-CLIP</span></strong>. <em>Cell Reports</em> 2017, <strong>20</strong>:1463–1475<a href="https://doi.org/10.1016/j.celrep.2017.07.029">10.1016/j.celrep.2017.07.029</a>Available: <a href="http://dx.doi.org/10.1016/j.celrep.2017.07.029">http://dx.doi.org/10.1016/j.celrep.2017.07.029</a>.
</div>
<div id="ref-Raab2011" class="csl-entry">
353. Raab JR, Chiu J, Zhu J, Katzman S, Kurukuti S, Wade PA, Haussler D, Kamakaka RT: <strong><span class="nocase">Human tRNA genes function as chromatin insulators.</span></strong> <em>The EMBO journal</em> 2012, <strong>31</strong>:330–50<a href="https://doi.org/10.1038/emboj.2011.406">10.1038/emboj.2011.406</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22085927">http://www.ncbi.nlm.nih.gov/pubmed/22085927</a>.
</div>
<div id="ref-VanBortle2017" class="csl-entry">
354. Van Bortle K, Phanstiel DH, Snyder MP: <strong><span class="nocase">Topological organization and dynamic regulation of human tRNA genes during macrophage differentiation</span></strong>. <em>Genome Biology</em> 2017, <strong>18</strong>:180<a href="https://doi.org/10.1186/s13059-017-1310-3">10.1186/s13059-017-1310-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1310-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1310-3</a>.
</div>
<div id="ref-Noma2006" class="csl-entry">
355. Noma K, Cam HP, Maraia RJ, Grewal SIS: <strong><span class="nocase">A role for TFIIIC transcription factor complex in genome organization.</span></strong> <em>Cell</em> 2006, <strong>125</strong>:859–72<a href="https://doi.org/10.1016/j.cell.2006.04.028">10.1016/j.cell.2006.04.028</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16751097">http://www.ncbi.nlm.nih.gov/pubmed/16751097</a>.
</div>
<div id="ref-Syddall2005" class="csl-entry">
356. Syddall H, Aihie Sayer A, Dennison E, Martin H, Barker D, Cooper C: <strong><span>Cohort Profile: The Hertfordshire Cohort Study</span></strong>. <em>International Journal of Epidemiology</em> 2005, <strong>34</strong>:1234–1242<a href="https://doi.org/10.1093/ije/dyi127">10.1093/ije/dyi127</a>Available: <a href="http://academic.oup.com/ije/article/34/6/1234/707357/Cohort-Profile-The-Hertfordshire-Cohort-Study">http://academic.oup.com/ije/article/34/6/1234/707357/Cohort-Profile-The-Hertfordshire-Cohort-Study</a>.
</div>
<div id="ref-Amemiya2019" class="csl-entry">
357. Amemiya HM, Kundaje A, Boyle AP: <strong><span class="nocase">The ENCODE Blacklist: Identification of Problematic Regions of the Genome</span></strong>. <em>Scientific Reports</em> 2019, <strong>9</strong>:9354<a href="https://doi.org/10.1038/s41598-019-45839-z">10.1038/s41598-019-45839-z</a>Available: <a href="http://www.nature.com/articles/s41598-019-45839-z">http://www.nature.com/articles/s41598-019-45839-z</a>.
</div>
<div id="ref-Derrien2012" class="csl-entry">
358. Derrien T, Estellé J, Marco Sola S, Knowles DG, Raineri E, Guigó R, Ribeca P: <strong><span class="nocase">Fast Computation and Applications of Genome Mappability</span></strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e30377<a href="https://doi.org/10.1371/journal.pone.0030377">10.1371/journal.pone.0030377</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0030377">https://dx.plos.org/10.1371/journal.pone.0030377</a>.
</div>
<div id="ref-Ernst2011" class="csl-entry">
359. Ernst J, Kheradpour P, Mikkelsen TS, Shoresh N, Ward LD, Epstein CB, Zhang X, Wang L, Issner R, Coyne M, Ku M, Durham T, Kellis M, Bernstein BE: <strong><span class="nocase">Mapping and analysis of chromatin state dynamics in nine human cell types</span></strong>. <em>Nature</em> 2011, <strong>473</strong>:43–49<a href="https://doi.org/10.1038/nature09906">10.1038/nature09906</a>Available: <a href="http://www.nature.com/articles/nature09906">http://www.nature.com/articles/nature09906</a>.
</div>
<div id="ref-Bell2012a" class="csl-entry">
360. Bell CG, Wilson GA, Butcher LM, Roos C, Walter L, Beck S: <strong><span class="nocase">Human-specific CpG "beacons" identify loci associated with human-specific traits and disease.</span></strong> <em>Epigenetics</em> 2012, <strong>7</strong>:1188–99<a href="https://doi.org/10.4161/epi.22127">10.4161/epi.22127</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22968434">http://www.ncbi.nlm.nih.gov/pubmed/22968434</a>.
</div>
<div id="ref-North2003" class="csl-entry">
361. North BV, Curtis D, Sham PC: <strong><span class="nocase">A Note on the Calculation of Empirical P Values from Monte Carlo Procedures</span></strong>. <em>The American Journal of Human Genetics</em> 2003, <strong>72</strong>:498–499<a href="https://doi.org/10.1086/346173">10.1086/346173</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002929707605618">https://linkinghub.elsevier.com/retrieve/pii/S0002929707605618</a>.
</div>
<div id="ref-Ewels2016" class="csl-entry">
362. Ewels P, Magnusson M, Lundin S, Käller M: <strong><span class="nocase">MultiQC: summarize analysis results for multiple tools and samples in a single report.</span></strong> <em>Bioinformatics (Oxford, England)</em> 2016, <strong>32</strong>:3047–8<a href="https://doi.org/10.1093/bioinformatics/btw354">10.1093/bioinformatics/btw354</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27312411">http://www.ncbi.nlm.nih.gov/pubmed/27312411</a>.
</div>
<div id="ref-Krueger2015" class="csl-entry">
363. Krueger F: <strong><span>Trim Galore</span></strong>. 2015, Available: <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim{\_}galore/">https://www.bioinformatics.babraham.ac.uk/projects/trim{\_}galore/</a>.
</div>
<div id="ref-Martin2011" class="csl-entry">
364. Martin M: <strong><span class="nocase">Cutadapt removes adapter sequences from high-throughput sequencing reads</span></strong>. <em>EMBnet.journal</em> 2011, <strong>17</strong>:10<a href="https://doi.org/10.14806/ej.17.1.200">10.14806/ej.17.1.200</a>Available: <a href="http://journal.embnet.org/index.php/embnetjournal/article/view/200">http://journal.embnet.org/index.php/embnetjournal/article/view/200</a>.
</div>
<div id="ref-Langmead2012" class="csl-entry">
365. Langmead B, Salzberg SL: <strong><span class="nocase">Fast gapped-read alignment with Bowtie 2</span></strong>. <em>Nature Methods</em> 2012, <strong>9</strong>:357–359<a href="https://doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">http://www.ncbi.nlm.nih.gov/pubmed/22388286</a>.
</div>
<div id="ref-Muller2019" class="csl-entry">
366. Müller F, Scherer M, Assenov Y, Lutsik P, Walter J, Lengauer T, Bock C: <strong><span class="nocase">RnBeads 2.0: comprehensive analysis of DNA methylation data</span></strong>. <em>Genome Biology</em> 2019, <strong>20</strong>:55<a href="https://doi.org/10.1186/s13059-019-1664-9">10.1186/s13059-019-1664-9</a>Available: <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1664-9">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1664-9</a>.
</div>
<div id="ref-Ritchie2015" class="csl-entry">
367. Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK: <strong><span class="nocase">limma powers differential expression analyses for RNA-sequencing and microarray studies</span></strong>. <em>Nucleic Acids Research</em> 2015, <strong>43</strong>:e47–e47<a href="https://doi.org/10.1093/nar/gkv007">10.1093/nar/gkv007</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25605792">http://www.ncbi.nlm.nih.gov/pubmed/25605792</a>.
</div>
<div id="ref-Meuleman2019" class="csl-entry">
368. Meuleman W: <strong>Epilogos</strong>. 2019, Available: <a href="https://epilogos.altius.org/ https://github.com/Altius/epilogos">https://epilogos.altius.org/ https://github.com/Altius/epilogos</a>.
</div>
<div id="ref-Reinius2012" class="csl-entry">
369. Reinius LE, Acevedo N, Joerink M, Pershagen G, Dahlén S-E, Greco D, Söderhäll C, Scheynius A, Kere J: <strong><span class="nocase">Differential DNA Methylation in Purified Human Blood Cells: Implications for Cell Lineage and Studies on Disease Susceptibility</span></strong>. <em>PLoS ONE</em> 2012, <strong>7</strong>:e41361<a href="https://doi.org/10.1371/journal.pone.0041361">10.1371/journal.pone.0041361</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0041361">https://dx.plos.org/10.1371/journal.pone.0041361</a>.
</div>
<div id="ref-Davis2007p" class="csl-entry">
370. Sean D, Meltzer PS: <strong><span class="nocase">GEOquery: A bridge between the Gene Expression Omnibus (GEO) and BioConductor</span></strong>. <em>Bioinformatics</em> 2007, <strong>23</strong>:1846–1847<a href="https://doi.org/10.1093/bioinformatics/btm254">10.1093/bioinformatics/btm254</a>.
</div>
<div id="ref-Grossman2016" class="csl-entry">
371. Grossman RL, Heath AP, Ferretti V, Varmus HE, Lowy DR, Kibbe WA, Staudt LM: <strong><span class="nocase">Toward a Shared Vision for Cancer Genomic Data</span></strong>. <em>New England Journal of Medicine</em> 2016, <strong>375</strong>:1109–1112<a href="https://doi.org/10.1056/NEJMp1607591">10.1056/NEJMp1607591</a>Available: <a href="http://www.nejm.org/doi/10.1056/NEJMp1607591">http://www.nejm.org/doi/10.1056/NEJMp1607591</a>.
</div>
<div id="ref-Yang2016" class="csl-entry">
372. Yang Z, Wong A, Kuh D, Paul DS, Rakyan VK, Leslie RD, Zheng SC, Widschwendter M, Beck S, Teschendorff AE: <strong><span class="nocase">Correlation of an epigenetic mitotic clock with cancer risk</span></strong>. <em>Genome Biology</em> 2016, <strong>17</strong>:205<a href="https://doi.org/10.1186/s13059-016-1064-3">10.1186/s13059-016-1064-3</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3</a>.
</div>
<div id="ref-Nazor2012" class="csl-entry">
373. Nazor KL, Altun G, Lynch C, Tran H, Harness JV, Slavin I, Garitaonandia I, Müller F-J, Wang Y-C, Boscolo FS, Fakunle E, Dumevska B, Lee S, Park HS, Olee T, D’Lima DD, Semechkin R, Parast MM, Galat V, Laslett AL, Schmidt U, Keirstead HS, Loring JF, Laurent LC: <strong><span class="nocase">Recurrent Variations in DNA Methylation in Human Pluripotent Stem Cells and Their Differentiated Derivatives</span></strong>. <em>Cell Stem Cell</em> 2012, <strong>10</strong>:620–634<a href="https://doi.org/10.1016/j.stem.2012.02.013">10.1016/j.stem.2012.02.013</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/10449618">http://www.ncbi.nlm.nih.gov/pubmed/10449618</a>.
</div>
<div id="ref-Juzenas2017" class="csl-entry">
374. Juzenas S, Venkatesh G, Hübenthal M, Hoeppner MP, Du ZG, Paulsen M, Rosenstiel P, Senger P, Hofmann-Apitius M, Keller A, Kupcinskas L, Franke A, Hemmrich-Stanisak G: <strong><span class="nocase">A comprehensive, cell specific microRNA catalogue of human peripheral blood</span></strong>. <em>Nucleic Acids Research</em> 2017, <strong>45</strong>:9290–9301<a href="https://doi.org/10.1093/nar/gkx706">10.1093/nar/gkx706</a>Available: <a href="http://academic.oup.com/nar/article/45/16/9290/4080663">http://academic.oup.com/nar/article/45/16/9290/4080663</a>.
</div>
<div id="ref-Petkovich2017" class="csl-entry">
375. Petkovich DA, Podolskiy DI, Lobanov AV, Lee S-G, Miller RA, Gladyshev VN: <strong><span class="nocase">Using DNA Methylation Profiling to Evaluate Biological Age and Longevity Interventions</span></strong>. <em>Cell Metabolism</em> 2017, <strong>25</strong>:954–960.e6<a href="https://doi.org/10.1016/j.cmet.2017.03.016">10.1016/j.cmet.2017.03.016</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1550413117301687">https://linkinghub.elsevier.com/retrieve/pii/S1550413117301687</a>.
</div>
<div id="ref-Lowe2016" class="csl-entry">
376. Lowe TM, Chan PP: <strong><span class="nocase">tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes</span></strong>. <em>Nucleic Acids Research</em> 2016, <strong>44</strong>:W54–W57<a href="https://doi.org/10.1093/nar/gkw413">10.1093/nar/gkw413</a>Available: <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw413">https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkw413</a>.
</div>
<div id="ref-Geiger2013" class="csl-entry">
377. Geiger H, Haan G de, Florian MC: <strong><span class="nocase">The ageing haematopoietic stem cell compartment</span></strong>. <em>Nature Reviews Immunology</em> 2013, <strong>13</strong>:376–389<a href="https://doi.org/10.1038/nri3433">10.1038/nri3433</a>Available: <a href="http://dx.doi.org/10.1038/nri3433">http://dx.doi.org/10.1038/nri3433</a>.
</div>
<div id="ref-Schmitt2014b" class="csl-entry">
378. Schmitt BM, Rudolph KLM, Karagianni P, Fonseca NA, White RJ, Talianidis I, Odom DT, Marioni JC, Kutter C: <strong><span class="nocase">High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA–tRNA interface</span></strong>. <em>Genome Research</em> 2014, <strong>24</strong>:1797–1807<a href="https://doi.org/10.1101/gr.176784.114">10.1101/gr.176784.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25122613">http://www.ncbi.nlm.nih.gov/pubmed/25122613</a>.
</div>
<div id="ref-Gu2016" class="csl-entry">
379. Gu Z, Eils R, Schlesner M: <strong><span class="nocase">Complex heatmaps reveal patterns and correlations in multidimensional genomic data</span></strong>. <em>Bioinformatics</em> 2016, <strong>32</strong>:2847–2849<a href="https://doi.org/10.1093/bioinformatics/btw313">10.1093/bioinformatics/btw313</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btw313">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btw313</a>.
</div>
<div id="ref-Thornlow2020" class="csl-entry">
380. Thornlow BP, Armstrong J, Holmes AD, Howard JM, Corbett-Detig RB, Lowe TM: <strong><span class="nocase">Predicting transfer RNA gene activity from sequence and genome context</span></strong>. <em>Genome Research</em> 2020, <strong>30</strong>:85–94<a href="https://doi.org/10.1101/gr.256164.119">10.1101/gr.256164.119</a>.
</div>
<div id="ref-Ehrlich2019" class="csl-entry">
381. Ehrlich M: <strong><span class="nocase">DNA hypermethylation in disease: mechanisms and clinical relevance.</span></strong> <em>Epigenetics</em> 2019, <strong>14</strong>:1141–1163<a href="https://doi.org/10.1080/15592294.2019.1638701">10.1080/15592294.2019.1638701</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31284823">http://www.ncbi.nlm.nih.gov/pubmed/31284823</a>.
</div>
<div id="ref-Xu2014b" class="csl-entry">
382. Xu Z, Taylor JA: <strong><span class="nocase">Genome-wide age-related DNA methylation changes in blood and other tissues relate to histone modification, expression and cancer</span></strong>. <em>Carcinogenesis</em> 2014, <strong>35</strong>:356–364<a href="https://doi.org/10.1093/carcin/bgt391">10.1093/carcin/bgt391</a>Available: <a href="https://academic.oup.com/carcin/article-lookup/doi/10.1093/carcin/bgt391">https://academic.oup.com/carcin/article-lookup/doi/10.1093/carcin/bgt391</a>.
</div>
<div id="ref-Slieker2018" class="csl-entry">
383. Slieker RC, Relton CL, Gaunt TR, Slagboom PE, Heijmans BT: <strong><span class="nocase">Age-related DNA methylation changes are tissue-specific with ELOVL2 promoter methylation as exception</span></strong>. <em>Epigenetics and Chromatin</em> 2018, <strong>11</strong>:1–11<a href="https://doi.org/10.1186/s13072-018-0191-3">10.1186/s13072-018-0191-3</a>Available: <a href="https://doi.org/10.1186/s13072-018-0191-3">https://doi.org/10.1186/s13072-018-0191-3</a>.
</div>
<div id="ref-Zhu2018" class="csl-entry">
384. Zhu T, Zheng SC, Paul DS, Horvath S, Teschendorff AE: <strong><span class="nocase">Cell and tissue type independent age-associated DNA methylation changes are not rare but common.</span></strong> <em>Aging</em> 2018, <strong>10</strong>:3541–3557<a href="https://doi.org/10.18632/aging.101666">10.18632/aging.101666</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30482885">http://www.ncbi.nlm.nih.gov/pubmed/30482885</a>.
</div>
<div id="ref-Goodenbour2006" class="csl-entry">
385. Goodenbour JM, Pan T: <strong><span class="nocase">Diversity of tRNA genes in eukaryotes.</span></strong> <em>Nucleic acids research</em> 2006, <strong>34</strong>:6137–46<a href="https://doi.org/10.1093/nar/gkl725">10.1093/nar/gkl725</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/17088292">http://www.ncbi.nlm.nih.gov/pubmed/17088292</a>.
</div>
<div id="ref-Geslain2010" class="csl-entry">
386. Geslain R, Pan T: <strong><span class="nocase">Functional Analysis of Human tRNA Isodecoders</span></strong>. <em>Journal of Molecular Biology</em> 2010, <strong>396</strong>:821–831<a href="https://doi.org/10.1016/j.jmb.2009.12.018">10.1016/j.jmb.2009.12.018</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S002228360901523X">https://linkinghub.elsevier.com/retrieve/pii/S002228360901523X</a>.
</div>
<div id="ref-Li2019" class="csl-entry">
387. Li S, Shi X, Chen M, Xu N, Sun D, Bai R, Chen H, Ding K, Sheng J, Xu Z: <strong><span class="nocase">Angiogenin promotes colorectal cancer metastasis via tiRNA production</span></strong>. <em>International Journal of Cancer</em> 2019, :ijc.32245<a href="https://doi.org/10.1002/ijc.32245">10.1002/ijc.32245</a>Available: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.32245">https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.32245</a>.
</div>
<div id="ref-Plotkin2004" class="csl-entry">
388. Plotkin JB, Robins H, Levine AJ: <strong><span class="nocase">Tissue-specific codon usage and the expression of human genes.</span></strong> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 2004, <strong>101</strong>:12588–91<a href="https://doi.org/10.1073/pnas.0404957101">10.1073/pnas.0404957101</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15314228">http://www.ncbi.nlm.nih.gov/pubmed/15314228</a>.
</div>
<div id="ref-Schmitt2014" class="csl-entry">
389. Schmitt BM, Rudolph KLM, Karagianni P, Fonseca NA, White RJ, Talianidis I, Odom DT, Marioni JC, Kutter C: <strong><span class="nocase">High-resolution mapping of transcriptional dynamics across tissue development reveals a stable mRNA-tRNA interface.</span></strong> <em>Genome research</em> 2014, <strong>24</strong>:1797–807<a href="https://doi.org/10.1101/gr.176784.114">10.1101/gr.176784.114</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25122613">http://www.ncbi.nlm.nih.gov/pubmed/25122613</a>.
</div>
<div id="ref-Powell1997" class="csl-entry">
390. Powell JR, Moriyama EN: <strong><span class="nocase">Evolution of codon usage bias in Drosophila</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 1997, <strong>94</strong>:7784–7790<a href="https://doi.org/10.1073/pnas.94.15.7784">10.1073/pnas.94.15.7784</a>Available: <a href="http://www.pnas.org/cgi/doi/10.1073/pnas.94.15.7784">http://www.pnas.org/cgi/doi/10.1073/pnas.94.15.7784</a>.
</div>
<div id="ref-Rudolph2016" class="csl-entry">
391. Rudolph KLM, Schmitt BM, Villar D, White RJ, Marioni JC, Kutter C, Odom DT: <strong><span class="nocase">Codon-Driven Translational Efficiency Is Stable across Diverse Mammalian Cell States.</span></strong> <em>PLoS genetics</em> 2016, <strong>12</strong>:e1006024<a href="https://doi.org/10.1371/journal.pgen.1006024">10.1371/journal.pgen.1006024</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27166679">http://www.ncbi.nlm.nih.gov/pubmed/27166679</a>.
</div>
<div id="ref-Gu2018" class="csl-entry">
392. Gu T, Lin X, Cullen SM, Luo M, Jeong M, Estecio M, Shen J, Hardikar S, Sun D, Su J, Rux D, Guzman A, Lee M, Qi LS, Chen J-J, Kyba M, Huang Y, Chen T, Li W, Goodell MA: <strong><span class="nocase">DNMT3A and TET1 cooperate to regulate promoter epigenetic landscapes in mouse embryonic stem cells</span></strong>. <em>Genome Biology</em> 2018, <strong>19</strong>:88<a href="https://doi.org/10.1186/s13059-018-1464-7">10.1186/s13059-018-1464-7</a>Available: <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1464-7">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1464-7</a>.
</div>
<div id="ref-Wang2019a" class="csl-entry">
393. Wang Y, Brady KS, Caiello BP, Ackerson SM, Stewart JA: <strong><span class="nocase">Human CST suppresses origin licensing and promotes AND-1/Ctf4 chromatin association</span></strong>. <em>Life Science Alliance</em> 2019, <strong>2</strong>:e201800270<a href="https://doi.org/10.26508/lsa.201800270">10.26508/lsa.201800270</a>Available: <a href="http://www.life-science-alliance.org/lookup/doi/10.26508/lsa.201800270">http://www.life-science-alliance.org/lookup/doi/10.26508/lsa.201800270</a>.
</div>
<div id="ref-Sargolzaeiaval2018" class="csl-entry">
394. Sargolzaeiaval F, Zhang J, Schleit J, Lessel D, Kubisch C, Precioso DR, Sillence D, Hisama FM, Dorschner M, Martin GM, Oshima J: <strong><span class="nocase">CTC1 mutations in a Brazilian family with progeroid features and recurrent bone fractures</span></strong>. <em>Molecular Genetics <span>&amp;</span> Genomic Medicine</em> 2018, <strong>6</strong>:1148–1156<a href="https://doi.org/10.1002/mgg3.495">10.1002/mgg3.495</a>Available: <a href="http://doi.wiley.com/10.1002/mgg3.495">http://doi.wiley.com/10.1002/mgg3.495</a>.
</div>
<div id="ref-Thornlow2018" class="csl-entry">
395. Thornlow BP, Hough J, Roger JM, Gong H, Lowe TM, Corbett-Detig RB: <strong><span class="nocase">Transfer RNA genes experience exceptionally elevated mutation rates</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2018, <strong>115</strong>:8996–9001<a href="https://doi.org/10.1073/pnas.1801240115">10.1073/pnas.1801240115</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1801240115">http://www.pnas.org/lookup/doi/10.1073/pnas.1801240115</a>.
</div>
<div id="ref-Iben2015" class="csl-entry">
396. Iben JR, Maraia RJ: <strong><span class="nocase">tRNA gene copy number variation in humans</span></strong>. <em>Gene</em> 2014, <strong>536</strong>:376–384<a href="https://doi.org/10.1016/j.gene.2013.11.049">10.1016/j.gene.2013.11.049</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0378111913015758">https://linkinghub.elsevier.com/retrieve/pii/S0378111913015758</a>.
</div>
<div id="ref-Darrow2014" class="csl-entry">
397. Darrow EM, Chadwick BP: <strong><span class="nocase">A novel tRNA variable number tandem repeat at human chromosome 1q23.3 is implicated as a boundary element based on conservation of a CTCF motif in mouse</span></strong>. <em>Nucleic Acids Research</em> 2014, <strong>42</strong>:6421–6435<a href="https://doi.org/10.1093/nar/gku280">10.1093/nar/gku280</a>Available: <a href="http://academic.oup.com/nar/article/42/10/6421/2435926/A-novel-tRNA-variable-number-tandem-repeat-at">http://academic.oup.com/nar/article/42/10/6421/2435926/A-novel-tRNA-variable-number-tandem-repeat-at</a>.
</div>
<div id="ref-Muller2017" class="csl-entry">
398. Müller CA, Nieduszynski CA: <strong><span class="nocase">DNA replication timing influences gene expression level</span></strong>. <em>The Journal of Cell Biology</em> 2017, <strong>216</strong>:1907–1914<a href="https://doi.org/10.1083/jcb.201701061">10.1083/jcb.201701061</a>Available: <a href="http://www.jcb.org/lookup/doi/10.1083/jcb.201701061">http://www.jcb.org/lookup/doi/10.1083/jcb.201701061</a>.
</div>
<div id="ref-Du2019" class="csl-entry">
399. Du Q, Bert SA, Armstrong NJ, Caldon CE, Song JZ, Nair SS, Gould CM, Luu P-L, Peters T, Khoury A, Qu W, Zotenko E, Stirzaker C, Clark SJ: <strong><span class="nocase">Replication timing and epigenome remodelling are associated with the nature of chromosomal rearrangements in cancer</span></strong>. <em>Nature Communications</em> 2019, <strong>10</strong>:416<a href="https://doi.org/10.1038/s41467-019-08302-1">10.1038/s41467-019-08302-1</a>Available: <a href="http://dx.doi.org/10.1038/s41467-019-08302-1">http://dx.doi.org/10.1038/s41467-019-08302-1</a>.
</div>
<div id="ref-Cruickshanks2013" class="csl-entry">
400. Cruickshanks HA, McBryan T, Nelson DM, VanderKraats ND, Shah PP, Tuyn J van, Singh Rai T, Brock C, Donahue G, Dunican DS, Drotar ME, Meehan RR, Edwards JR, Berger SL, Adams PD: <strong><span class="nocase">Senescent cells harbour features of the cancer epigenome</span></strong>. <em>Nature Cell Biology</em> 2013, <strong>15</strong>:1495–1506<a href="https://doi.org/10.1038/ncb2879">10.1038/ncb2879</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24270890">http://www.ncbi.nlm.nih.gov/pubmed/24270890</a>.
</div>
<div id="ref-Sun2018" class="csl-entry">
401. Sun L, Yu R, Dang W: <strong><span class="nocase">Chromatin Architectural Changes during Cellular Senescence and Aging</span></strong>. <em>Genes</em> 2018, <strong>9</strong>:211<a href="https://doi.org/10.3390/genes9040211">10.3390/genes9040211</a>Available: <a href="http://www.mdpi.com/2073-4425/9/4/211">http://www.mdpi.com/2073-4425/9/4/211</a>.
</div>
<div id="ref-Moskowitz2017" class="csl-entry">
402. Moskowitz DM, Zhang DW, Hu B, Le Saux S, Yanes RE, Ye Z, Buenrostro JD, Weyand CM, Greenleaf WJ, Goronzy JJ: <strong><span class="nocase">Epigenomics of human CD8 T cell differentiation and aging</span></strong>. <em>Science Immunology</em> 2017, <strong>2</strong>:eaag0192<a href="https://doi.org/10.1126/sciimmunol.aag0192">10.1126/sciimmunol.aag0192</a>Available: <a href="https://immunology.sciencemag.org/lookup/doi/10.1126/sciimmunol.aag0192">https://immunology.sciencemag.org/lookup/doi/10.1126/sciimmunol.aag0192</a>.
</div>
<div id="ref-Hashimoto2019" class="csl-entry">
403. Hashimoto K, Kouno T, Ikawa T, Hayatsu N, Miyajima Y, Yabukami H, Terooatea T, Sasaki T, Suzuki T, Valentine M, Pascarella G, Okazaki Y, Suzuki H, Shin JW, Minoda A, Taniuchi I, Okano H, Arai Y, Hirose N, Carninci P: <strong><span class="nocase">Single-cell transcriptomics reveals expansion of cytotoxic CD4 T cells in supercentenarians</span></strong>. <em>Proceedings of the National Academy of Sciences of the United States of America</em> 2019, <strong>116</strong>:24242–24251<a href="https://doi.org/10.1073/pnas.1907883116">10.1073/pnas.1907883116</a>.
</div>
<div id="ref-Deniz2019" class="csl-entry">
404. Deniz Ö, Frost JM, Branco MR: <strong><span class="nocase">Regulation of transposable elements by DNA modifications</span></strong>. <em>Nature Reviews Genetics</em> 2019, <strong>20</strong>:417–431<a href="https://doi.org/10.1038/s41576-019-0106-6">10.1038/s41576-019-0106-6</a>.
</div>
<div id="ref-Kazazian2017" class="csl-entry">
405. Kazazian HH, Moran JV: <strong><span class="nocase">Mobile DNA in health and disease</span></strong>. <em>New England Journal of Medicine</em> 2017, <strong>377</strong>:361–370<a href="https://doi.org/10.1056/NEJMra1510092">10.1056/NEJMra1510092</a>.
</div>
<div id="ref-Gregory2005" class="csl-entry">
406. Gregory TR: <strong><span class="nocase">Synergy between sequence and size in large-scale genomics</span></strong>. <em>Nature Reviews Genetics</em> 2005, <strong>6</strong>:699–708<a href="https://doi.org/10.1038/nrg1674">10.1038/nrg1674</a>.
</div>
<div id="ref-Deaton2011" class="csl-entry">
407. Deaton AM, Bird A: <strong><span class="nocase">CpG islands and the regulation of transcription</span></strong>. <em>Genes <span>&amp;</span> development</em> 2011, <strong>25</strong>:1010–22<a href="https://doi.org/10.1101/gad.2037511">10.1101/gad.2037511</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21576262">http://www.ncbi.nlm.nih.gov/pubmed/21576262</a>.
</div>
<div id="ref-Hellman2007" class="csl-entry">
408. Hellman A, Chess A: <strong><span class="nocase">Gene body-specific methylation on the active X chromosome</span></strong>. <em>Science</em> 2007, <strong>315</strong>:1141–1143<a href="https://doi.org/10.1126/science.1136352">10.1126/science.1136352</a>.
</div>
<div id="ref-Shukla2011" class="csl-entry">
409. Shukla S, Kavak E, Gregory M, Imashimizu M, Shutinoski B, Kashlev M, Oberdoerffer P, Sandberg R, Oberdoerffer S: <strong><span class="nocase">CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing</span></strong>. <em>Nature</em> 2011, <strong>479</strong>:74–79<a href="https://doi.org/10.1038/nature10442">10.1038/nature10442</a>Available: <a href="http://dx.doi.org/10.1038/nature10442">http://dx.doi.org/10.1038/nature10442</a>.
</div>
<div id="ref-Jorda2017" class="csl-entry">
410. Jordà M, Díez-Villanueva A, Mallona I, Martín B, Lois S, Barrera V, Esteller M, Vavouri T, Peinado MA: <strong><span class="nocase">The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells</span></strong>. <em>Genome Research</em> 2017, <strong>27</strong>:118–132<a href="https://doi.org/10.1101/gr.207522.116">10.1101/gr.207522.116</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27999094">http://www.ncbi.nlm.nih.gov/pubmed/27999094</a>.
</div>
<div id="ref-Bollati2010" class="csl-entry">
411. Bollati V, Schwartz J, Wright R, Litonjua A, Tarantini L, Suh H, Sparrow D, Vokonas P, Baccarelli A: <strong><span class="nocase">Decline in genomic DNA methylation through aging in a cohort of elderly subjects</span></strong>. <em>Mechanisms of Ageing and Development</em> 2009, <strong>130</strong>:234–239<a href="https://doi.org/10.1016/j.mad.2008.12.003">10.1016/j.mad.2008.12.003</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820">https://linkinghub.elsevier.com/retrieve/pii/S0047637408002820</a>.
</div>
<div id="ref-Dewannieux2003" class="csl-entry">
412. Dewannieux M, Esnault C, Heidmann T: <strong><span class="nocase">LINE-mediated retrotransposition of marked Alu sequences</span></strong>. <em>Nature Genetics</em> 2003, <strong>35</strong>:41–48<a href="https://doi.org/10.1038/ng1223">10.1038/ng1223</a>Available: <a href="http://www.nature.com/articles/ng1223">http://www.nature.com/articles/ng1223</a>.
</div>
<div id="ref-Deininger2011" class="csl-entry">
413. Deininger P: <strong><span class="nocase">Alu elements: know the SINEs</span></strong>. <em>Genome Biology</em> 2011, <strong>12</strong>:236<a href="https://doi.org/10.1186/gb-2011-12-12-236">10.1186/gb-2011-12-12-236</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236">http://genomebiology.biomedcentral.com/articles/10.1186/gb-2011-12-12-236</a>.
</div>
<div id="ref-Ullu1984" class="csl-entry">
414. Ullu E, Tschudi C: <strong><span class="nocase">Alu sequences are processed 7SL RNA genes</span></strong>. <em>Nature</em> 1984, <strong>312</strong>:171–172<a href="https://doi.org/10.1038/312171a0">10.1038/312171a0</a>Available: <a href="http://www.nature.com/articles/312171a0">http://www.nature.com/articles/312171a0</a>.
</div>
<div id="ref-Cordaux2009" class="csl-entry">
415. Cordaux R, Batzer MA: <strong><span class="nocase">The impact of retrotransposons on human genome evolution</span></strong>. <em>Nature Reviews Genetics</em> 2009, <strong>10</strong>:691–703<a href="https://doi.org/10.1038/nrg2640">10.1038/nrg2640</a>Available: <a href="http://www.nature.com/articles/nrg2640">http://www.nature.com/articles/nrg2640</a>.
</div>
<div id="ref-Quentin1992" class="csl-entry">
416. Quentin Y: <strong><span class="nocase">Origin of the Alu family:</span></strong> <em>Nucleic Acids Research</em> 1992, <strong>20</strong>:3397–3401.
</div>
<div id="ref-Chen2008" class="csl-entry">
417. Chen L-L, Carmichael GG: <strong><span class="nocase">Gene regulation by SINES and inosines: biological consequences of A-to-I editing of Alu element inverted repeats</span></strong>. <em>Cell Cycle</em> 2008, <strong>7</strong>:3294–3301<a href="https://doi.org/10.4161/cc.7.21.6927">10.4161/cc.7.21.6927</a>Available: <a href="http://www.tandfonline.com/doi/abs/10.4161/cc.7.21.6927">http://www.tandfonline.com/doi/abs/10.4161/cc.7.21.6927</a>.
</div>
<div id="ref-Batzer2002" class="csl-entry">
418. Batzer MA, Deininger PL: <strong><span class="nocase">Alu repeats and human genomic diversity</span></strong>. <em>Nature Reviews Genetics</em> 2002, <strong>3</strong>:370–379<a href="https://doi.org/10.1038/nrg798">10.1038/nrg798</a>Available: <a href="http://www.nature.com/articles/nrg798">http://www.nature.com/articles/nrg798</a>.
</div>
<div id="ref-Ade2013" class="csl-entry">
419. Ade C, Roy-Engel AM, Deininger PL: <strong><span class="nocase">Alu elements: an intrinsic source of human genome instability</span></strong>. <em>Current Opinion in Virology</em> 2013, <strong>3</strong>:639–645<a href="https://doi.org/10.1016/j.coviro.2013.09.002">10.1016/j.coviro.2013.09.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533">https://linkinghub.elsevier.com/retrieve/pii/S1879625713001533</a>.
</div>
<div id="ref-Belancio2008" class="csl-entry">
420. Belancio VP, Hedges DJ, Deininger P: <strong><span class="nocase">Mammalian non-LTR retrotransposons: For better or worse, in sickness and in health</span></strong>. <em>Genome Research</em> 2008, <strong>18</strong>:343–358<a href="https://doi.org/10.1101/gr.5558208">10.1101/gr.5558208</a>Available: <a href="http://www.genome.org/cgi/doi/10.1101/gr.5558208">http://www.genome.org/cgi/doi/10.1101/gr.5558208</a>.
</div>
<div id="ref-Chen2017b" class="csl-entry">
421. Chen L-L, Yang L: <strong><span class="nocase">ALU ternative Regulation for Gene Expression</span></strong>. <em>Trends in Cell Biology</em> 2017, <strong>27</strong>:480–490<a href="https://doi.org/10.1016/j.tcb.2017.01.002">10.1016/j.tcb.2017.01.002</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028">https://linkinghub.elsevier.com/retrieve/pii/S0962892417300028</a>.
</div>
<div id="ref-RobertoFerrari2019" class="csl-entry">
422. Ferrari R, de Llobet Cucalon LI, Di Vona C, Le Dilly F, Vidal E, Lioutas A, Oliete JQ, Jochem L, Cutts E, Dieci G, Vannini A, Teichmann M, Luna S de la, Beato M: <strong><span class="nocase">TFIIIC Binding to Alu Elements Controls Gene Expression via Chromatin Looping and Histone Acetylation</span></strong>. <em>Molecular Cell</em> 2019, :1–13<a href="https://doi.org/10.1016/j.molcel.2019.10.020">10.1016/j.molcel.2019.10.020</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981">https://linkinghub.elsevier.com/retrieve/pii/S1097276519307981</a>.
</div>
<div id="ref-Polak2006" class="csl-entry">
423. Polak P, Domany E: <strong><span class="nocase">Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes.</span></strong> <em>BMC genomics</em> 2006, <strong>7</strong>:133<a href="https://doi.org/10.1186/1471-2164-7-133">10.1186/1471-2164-7-133</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/16740159">http://www.ncbi.nlm.nih.gov/pubmed/16740159</a>.
</div>
<div id="ref-Su2014" class="csl-entry">
424. Su M, Han D, Boyd-Kirkup J, Yu X, Han J-DJ: <strong><span class="nocase">Evolution of Alu Elements toward Enhancers</span></strong>. <em>Cell Reports</em> 2014, <strong>7</strong>:376–385<a href="https://doi.org/10.1016/j.celrep.2014.03.011">10.1016/j.celrep.2014.03.011</a>Available: <a href="http://dx.doi.org/10.1016/j.celrep.2014.03.011">http://dx.doi.org/10.1016/j.celrep.2014.03.011</a>.
</div>
<div id="ref-Blattler2014" class="csl-entry">
425. Blattler A, Yao L, Witt H, Guo Y, Nicolet CM, Berman BP, Farnham PJ: <strong><span class="nocase">Global loss of DNA methylation uncovers intronic enhancers in genes showing expression changes</span></strong>. <em>Genome Biology</em> 2014, <strong>15</strong>:469<a href="https://doi.org/10.1186/s13059-014-0469-0">10.1186/s13059-014-0469-0</a>Available: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0">http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0469-0</a>.
</div>
<div id="ref-Ward2013" class="csl-entry">
426. Ward MC, Wilson MD, Barbosa-Morais NL, Schmidt D, Stark R, Pan Q, Schwalie PC, Menon S, Lukk M, Watt S, Thybert D, Kutter C, Kirschner K, Flicek P, Blencowe BJ, Odom DT: <strong><span class="nocase">Latent Regulatory Potential of Human-Specific Repetitive Elements</span></strong>. <em>Molecular Cell</em> 2013, <strong>49</strong>:262–272<a href="https://doi.org/10.1016/j.molcel.2012.11.013">10.1016/j.molcel.2012.11.013</a>Available: <a href="http://dx.doi.org/10.1016/j.molcel.2012.11.013">http://dx.doi.org/10.1016/j.molcel.2012.11.013</a>.
</div>
<div id="ref-Xie2013" class="csl-entry">
427. Xie M, Hong C, Zhang B, Lowdon RF, Xing X, Li D, Zhou X, Lee HJ, Maire CL, Ligon KL, Gascard P, Sigaroudinia M, Tlsty TD, Kadlecek T, Weiss A, O’Geen H, Farnham PJ, Madden PAF, Mungall AJ, Tam A, Kamoh B, Cho S, Moore R, Hirst M, Marra MA, Costello JF, Wang T: <strong><span class="nocase">DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape</span></strong>. <em>Nature Genetics</em> 2013, <strong>45</strong>:836–841<a href="https://doi.org/10.1038/ng.2649">10.1038/ng.2649</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23708189">http://www.ncbi.nlm.nih.gov/pubmed/23708189</a>.
</div>
<div id="ref-Zhang2019" class="csl-entry">
428. Zhang X-O, Gingeras TR, Weng Z: <strong><span class="nocase">Genome-wide analysis of polymerase III–transcribed Alu elements suggests cell-type–specific enhancer function</span></strong>. <em>Genome Research</em> 2019, <strong>29</strong>:1402–1414<a href="https://doi.org/10.1101/gr.249789.119">10.1101/gr.249789.119</a>Available: <a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.249789.119">http://genome.cshlp.org/lookup/doi/10.1101/gr.249789.119</a>.
</div>
<div id="ref-Jang2019" class="csl-entry">
429. Jang HS, Shah NM, Du AY, Dailey ZZ, Pehrsson EC, Godoy PM, Zhang D, Li D, Xing X, Kim S, O’Donnell D, Gordon JI, Wang T: <strong><span class="nocase">Transposable elements drive widespread expression of oncogenes in human cancers</span></strong>. <em>Nature Genetics</em> 2019, <strong>51</strong>:611–617<a href="https://doi.org/10.1038/s41588-019-0373-3">10.1038/s41588-019-0373-3</a>Available: <a href="http://dx.doi.org/10.1038/s41588-019-0373-3">http://dx.doi.org/10.1038/s41588-019-0373-3</a>.
</div>
<div id="ref-Rajendiran2016" class="csl-entry">
430. Rajendiran S, Gibbs LD, Van Treuren T, Klinkebiel DL, Vishwanatha JK: <strong><span class="nocase">MIEN1 is tightly regulated by SINE Alu methylation in its promoter</span></strong>. <em>Oncotarget</em> 2016, <strong>7</strong>:65307–65319<a href="https://doi.org/10.18632/oncotarget.11675">10.18632/oncotarget.11675</a>Available: <a href="http://www.oncotarget.com/fulltext/11675">http://www.oncotarget.com/fulltext/11675</a>.
</div>
<div id="ref-Jintaridth2013" class="csl-entry">
431. Jintaridth P, Tungtrongchitr R, Preutthipan S, Mutirangura A: <strong><span class="nocase">Hypomethylation of Alu Elements in Post-Menopausal Women with Osteoporosis</span></strong>. <em>PLoS ONE</em> 2013, <strong>8</strong>:e70386<a href="https://doi.org/10.1371/journal.pone.0070386">10.1371/journal.pone.0070386</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0070386">https://dx.plos.org/10.1371/journal.pone.0070386</a>.
</div>
<div id="ref-Lu2018a" class="csl-entry">
432. Lu S, Niu Z, Chen Y, Tu Q, Zhang Y, Chen W, Tong W, Zhang Z: <strong><span class="nocase">Repetitive Element DNA Methylation is Associated with Menopausal Age</span></strong>. <em>Aging and Disease</em> 2018, <strong>9</strong>:435<a href="https://doi.org/10.14336/AD.2017.0810">10.14336/AD.2017.0810</a>Available: <a href="http://www.aginganddisease.org/EN/10.14336/AD.2017.0810">http://www.aginganddisease.org/EN/10.14336/AD.2017.0810</a>.
</div>
<div id="ref-Payer2017" class="csl-entry">
433. Payer LM, Steranka JP, Yang WR, Kryatova M, Medabalimi S, Ardeljan D, Liu C, Boeke JD, Avramopoulos D, Burns KH: <strong><span class="nocase">Structural variants caused by Alu insertions are associated with risks for many human diseases</span></strong>. <em>Proceedings of the National Academy of Sciences</em> 2017, <strong>114</strong>:E3984–E3992<a href="https://doi.org/10.1073/pnas.1704117114">10.1073/pnas.1704117114</a>Available: <a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114">http://www.pnas.org/lookup/doi/10.1073/pnas.1704117114</a>.
</div>
<div id="ref-Bell2019" class="csl-entry">
434. Bell CG, Lowe R, Adams PD, Baccarelli AA, Beck S, Bell JT, Christensen BC, Gladyshev VN, Heijmans BT, Horvath S, Ideker T, Issa J-PJ, Kelsey KT, Marioni RE, Reik W, Relton CL, Schalkwyk LC, Teschendorff AE, Wagner W, Zhang K, Rakyan VK: <strong><span class="nocase">DNA methylation aging clocks: challenges and recommendations.</span></strong> <em>Genome biology</em> 2019, <strong>20</strong>:249<a href="https://doi.org/10.1186/s13059-019-1824-y">10.1186/s13059-019-1824-y</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31767039">http://www.ncbi.nlm.nih.gov/pubmed/31767039</a>.
</div>
<div id="ref-Mozhui2017" class="csl-entry">
435. Mozhui K, Pandey AK: <strong><span class="nocase">Conserved effect of aging on DNA methylation and association with EZH2 polycomb protein in mice and humans</span></strong>. <em>Mechanisms of Ageing and Development</em> 2017, <strong>162</strong>:27–37<a href="https://doi.org/10.1016/j.mad.2017.02.006">10.1016/j.mad.2017.02.006</a>Available: <a href="http://dx.doi.org/10.1016/j.mad.2017.02.006">http://dx.doi.org/10.1016/j.mad.2017.02.006</a>.
</div>
<div id="ref-Zhang2017" class="csl-entry">
436. Zhang Y, Hapala J, Brenner H, Wagner W: <strong><span class="nocase">Individual CpG sites that are associated with age and life expectancy become hypomethylated upon aging</span></strong>. <em>Clinical Epigenetics</em> 2017, <strong>9</strong>:9<a href="https://doi.org/10.1186/s13148-017-0315-9">10.1186/s13148-017-0315-9</a>Available: <a href="http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9">http://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-017-0315-9</a>.
</div>
<div id="ref-Smit2009" class="csl-entry">
437. Smit A, Hubley R, Green: <strong><span>Repeat Masker</span></strong>. 2009, Available: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>.
</div>
<div id="ref-Prive2019" class="csl-entry">
438. Privé F, Aschard H, Blum MGB: <strong><span class="nocase">Efficient Implementation of Penalized Regression for Genetic Risk Prediction</span></strong>. <em>Genetics</em> 2019, <strong>212</strong>:65–74<a href="https://doi.org/10.1534/genetics.119.302019">10.1534/genetics.119.302019</a>Available: <a href="http://www.genetics.org/lookup/doi/10.1534/genetics.119.302019 https://figshare.com/articles/code/7178750 https://gsajournals.figshare.com/articles/Supplemental{\_}Material{\_}for{\_}Priv{\_}Aschard{\_}and{\_}Blum{\_}2019/7851470">http://www.genetics.org/lookup/doi/10.1534/genetics.119.302019 https://figshare.com/articles/code/7178750 https://gsajournals.figshare.com/articles/Supplemental{\_}Material{\_}for{\_}Priv{\_}Aschard{\_}and{\_}Blum{\_}2019/7851470</a>.
</div>
<div id="ref-Fairley2020" class="csl-entry">
439. Fairley S, Lowy-Gallego E, Perry E, Flicek P: <strong><span class="nocase">The International Genome Sample Resource (IGSR) collection of open human genomic variation resources</span></strong>. <em>Nucleic Acids Research</em> 2020, <strong>48</strong>:D941–D947<a href="https://doi.org/10.1093/nar/gkz836">10.1093/nar/gkz836</a>Available: <a href="https://academic.oup.com/nar/article/48/D1/D941/5580898">https://academic.oup.com/nar/article/48/D1/D941/5580898</a>.
</div>
<div id="ref-Payer2019" class="csl-entry">
440. Payer LM, Steranka JP, Ardeljan D, Walker J, Fitzgerald KC, Calabresi PA, Cooper TA, Burns KH: <strong><span class="nocase">Alu insertion variants alter mRNA splicing</span></strong>. <em>Nucleic Acids Research</em> 2019, <strong>47</strong>:421–431<a href="https://doi.org/10.1093/nar/gky1086">10.1093/nar/gky1086</a>.
</div>
<div id="ref-Tibshirani2012" class="csl-entry">
441. Tibshirani R, Bien J, Friedman J, Hastie T, Simon N, Taylor J, Tibshirani RJ: <strong><span class="nocase">Strong rules for discarding predictors in lasso-type problems</span></strong>. <em>Journal of the Royal Statistical Society. Series B: Statistical Methodology</em> 2012, <strong>74</strong>:245–266<a href="https://doi.org/10.1111/j.1467-9868.2011.01004.x">10.1111/j.1467-9868.2011.01004.x</a>Available: <a href="http://arxiv.org/abs/1011.2234">http://arxiv.org/abs/1011.2234</a>.
</div>
<div id="ref-Snir2019" class="csl-entry">
442. Snir S, Farrell C, Pellegrini M: <strong><span class="nocase">Human epigenetic ageing is logarithmic with time across the entire lifespan</span></strong>. <em>Epigenetics</em> 2019, <strong>0</strong>:1–15<a href="https://doi.org/10.1080/15592294.2019.1623634">10.1080/15592294.2019.1623634</a>Available: <a href="https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634">https://www.tandfonline.com/doi/full/10.1080/15592294.2019.1623634</a>.
</div>
<div id="ref-Hellen2013" class="csl-entry">
443. Hellen EH, Brookfield JF: <strong><span class="nocase">Alu elements in primates are preferentially lost from areas of high GC content</span></strong>. <em>PeerJ</em> 2013, <strong>1</strong>:e78<a href="https://doi.org/10.7717/peerj.78">10.7717/peerj.78</a>Available: <a href="https://peerj.com/articles/78">https://peerj.com/articles/78</a>.
</div>
<div id="ref-Meyer2020" class="csl-entry">
444. Meyer DH: <strong><span class="nocase">A transcriptome based aging clock near the theoretical limit of accuracy</span></strong>. 2020, <a href="https://doi.org/10.1101/2020.05.29.123430">10.1101/2020.05.29.123430</a>Available: <a href="https://doi.org/10.1101/2020.05.29.123430">https://doi.org/10.1101/2020.05.29.123430</a>.
</div>
<div id="ref-Hysi2010" class="csl-entry">
445. Hysi PG, Young TL, Mackey DA, Andrew T, Fernández-Medarde A, Solouki AM, Hewitt AW, Macgregor S, Vingerling JR, Li Y, Ikram MK, Fai LY, Sham PC, Manyes L, Porteros A, Lopes MC, Carbonaro F, Fahy SJ, Martin NG, Duijn CM van, Spector TD, Rahi JS, Santos E, Klaver CCW, Hammond CJ: <strong><span class="nocase">A genome-wide association study for myopia and refractive error identifies a susceptibility locus at 15q25</span></strong>. <em>Nature Genetics</em> 2010, <strong>42</strong>:902–905<a href="https://doi.org/10.1038/ng.664">10.1038/ng.664</a>Available: <a href="http://www.nature.com/articles/ng.664">http://www.nature.com/articles/ng.664</a>.
</div>
<div id="ref-Loh2016" class="csl-entry">
446. Loh P, Palamara PF, Price AL: <strong><span class="nocase">Fast and accurate long-range phasing in a UK Biobank cohort</span></strong>. <em>Nature Genetics</em> 2016, <strong>48</strong>:811–816<a href="https://doi.org/10.1038/ng.3571">10.1038/ng.3571</a>Available: <a href="http://www.nature.com/articles/ng.3571">http://www.nature.com/articles/ng.3571</a>.
</div>
<div id="ref-Das2016" class="csl-entry">
447. Das S, Forer L, Schönherr S, Sidore C, Locke AE, Kwong A, Vrieze SI, Chew EY, Levy S, McGue M, Schlessinger D, Stambolian D, Loh P-R, Iacono WG, Swaroop A, Scott LJ, Cucca F, Kronenberg F, Boehnke M, Abecasis GR, Fuchsberger C: <strong><span class="nocase">Next-generation genotype imputation service and methods</span></strong>. <em>Nature Genetics</em> 2016, <strong>48</strong>:1284–1287<a href="https://doi.org/10.1038/ng.3656">10.1038/ng.3656</a>Available: <a href="http://www.nature.com/articles/ng.3656">http://www.nature.com/articles/ng.3656</a>.
</div>
<div id="ref-McCarthy2017" class="csl-entry">
448. McCarthy S: <strong><span class="nocase">A reference panel of 64,976 haplotypes for genotype imputation</span></strong>. <em>Nature Genetics</em> 2016, <strong>48</strong>:1279–1283<a href="https://doi.org/10.1038/ng.3643">10.1038/ng.3643</a>Available: <a href="http://www.nature.com/articles/ng.3643">http://www.nature.com/articles/ng.3643</a>.
</div>
<div id="ref-Zhou2012a" class="csl-entry">
449. Zhou X, Stephens M: <strong><span class="nocase">Genome-wide efficient mixed-model analysis for association studies</span></strong>. <em>Nature Genetics</em> 2012, <strong>44</strong>:821–824<a href="https://doi.org/10.1038/ng.2310">10.1038/ng.2310</a>Available: <a href="http://www.nature.com/articles/ng.2310">http://www.nature.com/articles/ng.2310</a>.
</div>
<div id="ref-Pruim2011" class="csl-entry">
450. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ: <strong><span class="nocase">LocusZoom: regional visualization of genome-wide association scan results</span></strong>. <em>Bioinformatics</em> 2010, <strong>26</strong>:2336–2337<a href="https://doi.org/10.1093/bioinformatics/btq419">10.1093/bioinformatics/btq419</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btq419">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btq419</a>.
</div>
<div id="ref-Staley2016" class="csl-entry">
451. Staley JR, Blackshaw J, Kamat MA, Ellis S, Surendran P, Sun BB, Paul DS, Freitag D, Burgess S, Danesh J, Young R, Butterworth AS: <strong><span class="nocase">PhenoScanner: a database of human genotype–phenotype associations</span></strong>. <em>Bioinformatics</em> 2016, <strong>32</strong>:3207–3209<a href="https://doi.org/10.1093/bioinformatics/btw373">10.1093/bioinformatics/btw373</a>Available: <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btw373">https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btw373</a>.
</div>
<div id="ref-Kamat2019" class="csl-entry">
452. Staley JR, Blackshaw J, Kamat MA, Ellis S, Surendran P, Sun BB, Paul DS, Freitag D, Burgess S, Danesh J, Young R, Butterworth AS: <strong><span class="nocase">PhenoScanner: a database of human genotype-phenotype associations.</span></strong> <em>Bioinformatics (Oxford, England)</em> 2016, <strong>32</strong>:3207–3209<a href="https://doi.org/10.1093/bioinformatics/btw373">10.1093/bioinformatics/btw373</a>Available: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5048068">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5048068</a>.
</div>
<div id="ref-GaglianoTaliun2020" class="csl-entry">
453. Gagliano Taliun SA, VandeHaar P, Boughton AP, Welch RP, Taliun D, Schmidt EM, Zhou W, Nielsen JB, Willer CJ, Lee S, Fritsche LG, Boehnke M, Abecasis GR: <strong><span class="nocase">Exploring and visualizing large-scale genetic associations by using PheWeb</span></strong>. <em>Nature Genetics</em> 2020, <strong>52</strong>:550–552<a href="https://doi.org/10.1038/s41588-020-0622-5">10.1038/s41588-020-0622-5</a>.
</div>
<div id="ref-CancerResearchUK2020" class="csl-entry">
454. Cancer Research UK: <strong><span class="nocase">Gallbladder cancer incidence statistics</span></strong>. 2020, Available: <a href="https://www.cancerresearchuk.org/health-professional/cancer-statistics/statistics-by-cancer-type/gallbladder-cancer/incidence{\#}heading-One">https://www.cancerresearchuk.org/health-professional/cancer-statistics/statistics-by-cancer-type/gallbladder-cancer/incidence{\#}heading-One</a>.Accessed 17 September 2020.
</div>
<div id="ref-Lohler2019" class="csl-entry">
455. Löhler J, Cebulla M, Shehata-Dieler W, Volkenstein S, Völter C, Walther LE: <strong><span class="nocase">Hearing impairment in old age</span></strong>. <em>Deutsches Aerzteblatt Online</em> 2019, <strong>116</strong>:301–310<a href="https://doi.org/10.3238/arztebl.2019.0301">10.3238/arztebl.2019.0301</a>Available: <a href="https://www.aerzteblatt.de/10.3238/arztebl.2019.0301">https://www.aerzteblatt.de/10.3238/arztebl.2019.0301</a>.
</div>
<div id="ref-Hileeto2005" class="csl-entry">
456. Hileeto D, Fadare O, Martel M, Zheng W: <strong><span class="nocase">Age dependent association of endometrial polyps with increased risk of cancer involvement.</span></strong> <em>World journal of surgical oncology</em> 2005, <strong>3</strong>:8<a href="https://doi.org/10.1186/1477-7819-3-8">10.1186/1477-7819-3-8</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/15703068">http://www.ncbi.nlm.nih.gov/pubmed/15703068</a>.
</div>
<div id="ref-Abhishekh2013" class="csl-entry">
457. Abhishekh HA, Nisarga P, Kisan R, Meghana A, Chandran S, Trichur Raju, Sathyaprabha TN: <strong><span class="nocase">Influence of age and gender on autonomic regulation of heart</span></strong>. <em>Journal of Clinical Monitoring and Computing</em> 2013, <strong>27</strong>:259–264<a href="https://doi.org/10.1007/s10877-012-9424-3">10.1007/s10877-012-9424-3</a>Available: <a href="http://link.springer.com/10.1007/s10877-012-9424-3">http://link.springer.com/10.1007/s10877-012-9424-3</a>.
</div>
<div id="ref-Nikitin2005" class="csl-entry">
458. Nikitin NP, Witte KKA, Ingle L, Clark AL, Farnsworth TA, Cleland JGF: <strong><span class="nocase">Longitudinal myocardial dysfunction in healthy older subjects as a manifestation of cardiac ageing</span></strong>. <em>Age and Ageing</em> 2005, <strong>34</strong>:343–349<a href="https://doi.org/10.1093/ageing/afi043">10.1093/ageing/afi043</a>Available: <a href="http://academic.oup.com/ageing/article/34/4/343/10217/Longitudinal-myocardial-dysfunction-in-healthy">http://academic.oup.com/ageing/article/34/4/343/10217/Longitudinal-myocardial-dysfunction-in-healthy</a>.
</div>
<div id="ref-Lu2017" class="csl-entry">
459. Lu AT, Hannon E, Levine ME, Crimmins EM, Lunnon K, Mill J, Geschwind DH, Horvath S: <strong><span class="nocase">Genetic architecture of epigenetic and neuronal ageing rates in human brain regions</span></strong>. <em>Nature Communications</em> 2017, <strong>8</strong>:15353<a href="https://doi.org/10.1038/ncomms15353">10.1038/ncomms15353</a>Available: <a href="http://dx.doi.org/10.1038/ncomms15353">http://dx.doi.org/10.1038/ncomms15353</a>.
</div>
<div id="ref-Gibson2019" class="csl-entry">
460. Gibson J, Russ TC, Clarke T-K, Howard DM, Hillary RF, Evans KL, Walker RM, Bermingham ML, Morris SW, Campbell A, Hayward C, Murray AD, Porteous DJ, Horvath S, Lu AT, McIntosh AM, Whalley HC, Marioni RE: <strong><span class="nocase">A meta-analysis of genome-wide association studies of epigenetic age acceleration</span></strong>. <em>PLOS Genetics</em> 2019, <strong>15</strong>:e1008104<a href="https://doi.org/10.1371/journal.pgen.1008104">10.1371/journal.pgen.1008104</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31738745">http://www.ncbi.nlm.nih.gov/pubmed/31738745</a>.
</div>
<div id="ref-Mccartney2020" class="csl-entry">
461. Mccartney DL, Min JL, Richmond RC, Lu AT, Maria K, Davies G, Broer L, Guo X, Jeong A, Kasela S, Katrinli S, Kuo P, Matias- PR, Mishra PP, Nygaard M, Palviainen T, Soerensen M, Sun D, Tsai P, Matthijs D, Xu Z, Yao J, Zhao W, Correa A, Boerwinkle E, Durda P, Elliott HR, Gieger C, Genetics T, Consortium M, et al.: <strong><span class="nocase">Genome-wide association studies identify 137 loci for DNA methylation biomarkers of ageing</span></strong>. <em>bioRxiv</em> 2020, :1–50<a href="https://doi.org/10.1101/2020.06.29.133702">10.1101/2020.06.29.133702</a>Available: <a href="https://doi.org/10.1101/2020.06.29.133702">https://doi.org/10.1101/2020.06.29.133702</a>.
</div>
<div id="ref-VanDeursen2019" class="csl-entry">
462. Deursen JM van: <strong><span class="nocase">Senolytic therapies for healthy longevity</span></strong>. <em>Science</em> 2019, <strong>364</strong>:636–637<a href="https://doi.org/10.1126/science.aaw1299">10.1126/science.aaw1299</a>Available: <a href="https://www.sciencemag.org/lookup/doi/10.1126/science.aaw1299">https://www.sciencemag.org/lookup/doi/10.1126/science.aaw1299</a>.
</div>
<div id="ref-Gjaltema2020" class="csl-entry">
463. Gjaltema RAF, Rots MG: <strong><span class="nocase">Advances of epigenetic editing</span></strong>. <em>Current Opinion in Chemical Biology</em> 2020, <strong>57</strong>:75–81<a href="https://doi.org/10.1016/j.cbpa.2020.04.020">10.1016/j.cbpa.2020.04.020</a>Available: <a href="https://doi.org/10.1016/j.cbpa.2020.04.020">https://doi.org/10.1016/j.cbpa.2020.04.020</a>.
</div>
<div id="ref-Ni2019" class="csl-entry">
464. Ni P, Huang N, Zhang Z, Wang D-P, Liang F, Miao Y, Xiao C-L, Luo F, Wang J: <strong><span class="nocase">DeepSignal: detecting DNA methylation state from Nanopore sequencing reads using deep-learning</span></strong>. <em>Bioinformatics</em> 2019, <strong>35</strong>:4586–4595<a href="https://doi.org/10.1093/bioinformatics/btz276">10.1093/bioinformatics/btz276</a>Available: <a href="https://academic.oup.com/bioinformatics/article/35/22/4586/5474907">https://academic.oup.com/bioinformatics/article/35/22/4586/5474907</a>.
</div>
<div id="ref-Zhu2018a" class="csl-entry">
465. Zhu P, Guo H, Ren Y, Hou Y, Dong J, Li R, Lian Y, Fan X, Hu B, Gao Y, Wang X, Wei Y, Liu P, Yan J, Ren X, Yuan P, Yuan Y, Yan Z, Wen L, Yan L, Qiao J, Tang F: <strong><span class="nocase">Single-cell DNA methylome sequencing of human preimplantation embryos</span></strong>. <em>Nature Genetics</em> 2018, <strong>50</strong>:12–19<a href="https://doi.org/10.1038/s41588-017-0007-6">10.1038/s41588-017-0007-6</a>Available: <a href="http://dx.doi.org/10.1038/s41588-017-0007-6">http://dx.doi.org/10.1038/s41588-017-0007-6</a>.
</div>
<div id="ref-Boyle2017" class="csl-entry">
466. Boyle EA, Li YI, Pritchard JK: <strong><span class="nocase">An Expanded View of Complex Traits: From Polygenic to Omnigenic</span></strong>. <em>Cell</em> 2017, <strong>169</strong>:1177–1186<a href="https://doi.org/10.1016/j.cell.2017.05.038">10.1016/j.cell.2017.05.038</a>Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0092867417306293">https://linkinghub.elsevier.com/retrieve/pii/S0092867417306293</a>.
</div>
<div id="ref-Langfelder2008" class="csl-entry">
467. Langfelder P, Horvath S: <strong><span class="nocase">WGCNA: An R package for weighted correlation network analysis</span></strong>. <em>BMC Bioinformatics</em> 2008, <strong>9</strong><a href="https://doi.org/10.1186/1471-2105-9-559">10.1186/1471-2105-9-559</a>.
</div>
<div id="ref-Blackledge2013" class="csl-entry">
468. Blackledge NP, Thomson JP, Skene PJ: <strong><span class="nocase">CpG Island Chromatin Is Shaped by Recruitment of ZF-CxxC Proteins</span></strong>. <em>Cold Spring Harbor Perspectives in Biology</em> 2013, <strong>5</strong>:a018648–a018648<a href="https://doi.org/10.1101/cshperspect.a018648">10.1101/cshperspect.a018648</a>Available: <a href="http://cshperspectives.cshlp.org/lookup/doi/10.1101/cshperspect.a018648">http://cshperspectives.cshlp.org/lookup/doi/10.1101/cshperspect.a018648</a>.
</div>
<div id="ref-Sijacic2019" class="csl-entry">
469. Sijacic P, Holder DH, Bajic M, Deal RB: <strong><span class="nocase">Methyl-CpG-binding domain 9 (MBD9) is required for H2A.Z incorporation into chromatin at a subset of H2A.Z-enriched regions in the Arabidopsis genome</span></strong>. <em>PLOS Genetics</em> 2019, <strong>15</strong>:e1008326<a href="https://doi.org/10.1371/journal.pgen.1008326">10.1371/journal.pgen.1008326</a>Available: <a href="https://dx.plos.org/10.1371/journal.pgen.1008326">https://dx.plos.org/10.1371/journal.pgen.1008326</a>.
</div>
<div id="ref-Hamdani2019" class="csl-entry">
470. Hamdani O, Dhillon N, Hsieh T-HS, Fujita T, Ocampo J, Kirkland JG, Lawrimore J, Kobayashi TJ, Friedman B, Fulton D, Wu KY, Chereji RV, Oki M, Bloom K, Clark DJ, Rando OJ, Kamakaka RT: <strong><span class="nocase">tRNA Genes Affect Chromosome Structure and Function via Local Effects.</span></strong> <em>Molecular and cellular biology</em> 2019, <strong>39</strong>:1–26<a href="https://doi.org/10.1128/MCB.00432-18">10.1128/MCB.00432-18</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30718362">http://www.ncbi.nlm.nih.gov/pubmed/30718362</a>.
</div>
<div id="ref-Rounge2018" class="csl-entry">
471. Rounge TB, Umu SU, Keller A, Meese E, Ursin G, Tretli S, Lyle R, Langseth H: <strong><span class="nocase">Circulating small non-coding RNAs associated with age, sex, smoking, body mass and physical activity</span></strong>. <em>Scientific Reports</em> 2018, <strong>8</strong>:1–13<a href="https://doi.org/10.1038/s41598-018-35974-4">10.1038/s41598-018-35974-4</a>Available: <a href="http://dx.doi.org/10.1038/s41598-018-35974-4">http://dx.doi.org/10.1038/s41598-018-35974-4</a>.
</div>
<div id="ref-Tellier2020" class="csl-entry">
472. Tellier M, Maudlin I, Murphy S: <strong><span class="nocase">Transcription and splicing: A two-way street</span></strong>. <em>Wiley Interdisciplinary Reviews: RNA</em> 2020, <strong>11</strong><a href="https://doi.org/10.1002/wrna.1593">10.1002/wrna.1593</a>.
</div>
<div id="ref-Wieben2019" class="csl-entry">
473. Wieben ED, Aleff RA, Basu S, Sarangi V, Bowman B, McLaughlin IJ, Mills JR, Butz ML, Highsmith EW, Ida CM, Ekholm JM, Baratz KH, Fautsch MP: <strong><span class="nocase">Amplification-free long-read sequencing of TCF4 expanded trinucleotide repeats in Fuchs Endothelial Corneal Dystrophy</span></strong>. <em>PLOS ONE</em> 2019, <strong>14</strong>:e0219446<a href="https://doi.org/10.1371/journal.pone.0219446">10.1371/journal.pone.0219446</a>Available: <a href="https://dx.plos.org/10.1371/journal.pone.0219446">https://dx.plos.org/10.1371/journal.pone.0219446</a>.
</div>
<div id="ref-Flusberg2010" class="csl-entry">
474. Flusberg BA, Webster DR, Lee JH, Travers KJ, Olivares EC, Clark TA, Korlach J, Turner SW: <strong><span class="nocase">Direct detection of DNA methylation during single-molecule, real-time sequencing</span></strong>. <em>Nature Methods</em> 2010, <strong>7</strong>:461–465<a href="https://doi.org/10.1038/nmeth.1459">10.1038/nmeth.1459</a>Available: <a href="http://www.nature.com/articles/nmeth.1459">http://www.nature.com/articles/nmeth.1459</a>.
</div>
<div id="ref-Price2010" class="csl-entry">
475. Price AL, Zaitlen NA, Reich D, Patterson N: <strong><span class="nocase">New approaches to population stratification in genome-wide association studies</span></strong>. <em>Nature Reviews Genetics</em> 2010, <strong>11</strong>:459–463<a href="https://doi.org/10.1038/nrg2813">10.1038/nrg2813</a>Available: <a href="http://www.nature.com/articles/nrg2813">http://www.nature.com/articles/nrg2813</a>.
</div>
<div id="ref-Jjingo2014" class="csl-entry">
476. Jjingo D, Conley AB, Wang J, Mariño-Ramírez L, Lunyak VV, Jordan I: <strong><span class="nocase">Mammalian-wide interspersed repeat (MIR)-derived enhancers and the regulation of human gene expression</span></strong>. <em>Mobile DNA</em> 2014, <strong>5</strong>:14<a href="https://doi.org/10.1186/1759-8753-5-14">10.1186/1759-8753-5-14</a>Available: <a href="http://mobilednajournal.biomedcentral.com/articles/10.1186/1759-8753-5-14">http://mobilednajournal.biomedcentral.com/articles/10.1186/1759-8753-5-14</a>.
</div>
<div id="ref-Leung2015" class="csl-entry">
477. Leung D, Jung I, Rajagopal N, Schmitt A, Selvaraj S, Lee AY, Yen C-A, Lin S, Lin Y, Qiu Y, Xie W, Yue F, Hariharan M, Ray P, Kuan S, Edsall L, Yang H, Chi NC, Zhang MQ, Ecker JR, Ren B: <strong><span class="nocase">Integrative analysis of haplotype-resolved epigenomes across human tissues.</span></strong> <em>Nature</em> 2015, <strong>518</strong>:350–354<a href="https://doi.org/10.1038/nature14217">10.1038/nature14217</a>Available: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25693566">http://www.ncbi.nlm.nih.gov/pubmed/25693566</a>.
</div>
<div id="ref-Billon2012" class="csl-entry">
478. Billon P, Côté J: <strong><span class="nocase">Precise deposition of histone H2A.Z in chromatin for genome expression and maintenance</span></strong>. <em>Biochimica et Biophysica Acta - Gene Regulatory Mechanisms</em> 2012, <strong>1819</strong>:290–302<a href="https://doi.org/10.1016/j.bbagrm.2011.10.004">10.1016/j.bbagrm.2011.10.004</a>Available: <a href="http://dx.doi.org/10.1016/j.bbagrm.2011.10.004">http://dx.doi.org/10.1016/j.bbagrm.2011.10.004</a>.
</div>
<div id="ref-Tudge2016" class="csl-entry">
479. Tudge SJ, Watson RA, Brede M: <strong><span class="nocase">Game theoretic treatments for the differentiation of functional roles in the transition to multicellularity</span></strong>. <em>Journal of Theoretical Biology</em> 2016, <strong>395</strong>:161–173<a href="https://doi.org/10.1016/j.jtbi.2016.01.041">10.1016/j.jtbi.2016.01.041</a>Available: <a href="http://dx.doi.org/10.1016/j.jtbi.2016.01.041">http://dx.doi.org/10.1016/j.jtbi.2016.01.041</a>.
</div>
</div>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="discussion.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="appendices.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["thesis.pdf", "thesis.epub"],
"toc": {
"collapse": "subsection",
"toc_depth": 5,
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
